{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<img src=\"https://www.ikiam.edu.ec/img/logo-ikiam-grey.png\" width=300 height=240 />\n",
    "\n",
    "# EXAMEN PARCIAL PYTHON\n",
    "    \n",
    "\n",
    "## GBI6-2021II: BIOINFORMÁTICA\n",
    "\n",
    "**Salcán Castillo Shirley Michell\n",
    "\n",
    "**18-02-2021**"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<font color='red'>NO AGREGUE NUEVAS CELDAS, UTILICE ÚNICAMENTE LAS QUE ESTÁN CREADAS</font>\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### REQUERIMIENTOS PARA EL EXAMEN\n",
    "Para este examen se requiere dos documentos: \n",
    "\n",
    "1. Archivo ```miningscience.py``` donde tendrá dos funciones:\n",
    "\n",
    "2. Archivo ```2021II_GBI6_ExamenParcialPython``` donde se llamará las funciones y se obtendrá resultados."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Ejercicio 1 [2 puntos]\n",
    "Cree el archivo ```miningscience.py``` con las siguientes dosfunciones:\n",
    "    \n",
    "i. ```download_pubmed```: para descargar la data de PubMed utilizando el **ENTREZ** de Biopython. El parámetro de entrada para la función es el ```keyword```. \n",
    "    \n",
    "ii. ```mining_pubs```: esta función debe utilizar el módulo **re** y utilizar el párametro ```tipo``` para realizar lo siguiente:\n",
    "        \n",
    "* Si el tipo es \"DP\" recupera el año de publicación del artículo. El retorno es un *dataframe* con el **PMID** y el **DP_year**.\n",
    "      \n",
    "* Si el tipo es \"AU\" recupera el número de autores por **PMID**. El retorno es un *dataframe* con el **PMID** y el **num_auth**. \n",
    "      \n",
    "* Si el tipo es \"AD\" recupera el conteo de autores por país. El retorno es un *dataframe* con el **country** y el **num_auth**. \n",
    "\n",
    "* *Puede utilizar un ```if``` dentro de la función.* \n",
    "\n",
    "* *Cree un docstring para cada función.* \n",
    "\n",
    "Luego de crear las funciones, cargue el módulo ```miningscience``` como ```msc``` e imprima docstring de cada función.\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Help on function download_pubmed in module miningscience:\n",
      "\n",
      "download_pubmed(keyword)\n",
      "    La función download_pubmed recibe un argumento, que corresponde a palabras claves con las cuales se realizará una búsqueda \n",
      "    en Pubmed y  retorna la búsqueda en la base de datos antes nombrada, tal cual una descarga tipo PubMed.\n",
      "    keyword: son palabras claves de búsqueda\n",
      "\n",
      "Help on function mining_pubs in module miningscience:\n",
      "\n",
      "mining_pubs(tipo, archivo)\n",
      "    La función mining_pubs recibe dos argumentos, que corresponde al tipo de minado de datos que se desea y el documento del \n",
      "    cual se va a extraer los datos.\n",
      "    Tipo: puede ser DP, AU o AD.\n",
      "           \"DP\" --> recupera el año de publicación del artículo. El retorno es un dataframe con el PMID y el DP_year.\n",
      "           \"AU\" --> recupera el número de autores por PMID. El retorno es un dataframe con el PMID y el num_auth.\n",
      "           \"AD\" -->recupera el conteo de autores por país. El retorno es un dataframe con el country y el num_auth.\n",
      "    Archivo: corresponde al archivo del cual se extraerá la infomarción\n",
      "\n"
     ]
    }
   ],
   "source": [
    "import miningscience as msc\n",
    "\n",
    "help(msc.download_pubmed)\n",
    "help(msc.mining_pubs)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "## Ejercicio 2 [1 punto]\n",
    "\n",
    "\n",
    "Utilice la función ```download_pubmed``` para descargar la data con el keyword ```Ecuador genomics``` e imprima lo siguiente: \n",
    "\n",
    "```python\n",
    "'El número de artículos es: xx' # xx corresponde al número de artículos recuperados\n",
    "\n",
    "'El PMID del primer artículo es: xxxxxxxx' # xxxxxxxx corresponde al ID del artículo en PUDMED\n",
    "\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "El número de artículos es:  193\n",
      "El PMID del primero artículo es:  PMID- 35121084\n",
      "\n",
      "\n",
      "PMID- 35121084\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "LR  - 20220210\n",
      "IS  - 1873-2615 (Electronic)\n",
      "IS  - 1050-1738 (Linking)\n",
      "DP  - 2022 Feb 2\n",
      "TI  - A systematic review of electrocardiographic changes in healthy high-altitudepopulations.\n",
      "LID - S1050-1738(22)00015-9 [pii]\n",
      "LID - 10.1016/j.tcm.2022.01.013 [doi]\n",
      "AB  - High-altitude environments are characterized by decreased atmospheric pressuresat which individuals exhibit a reduced volume of maximal oxygen uptake andarterial partial pressure of oxygen, both of which lead to hypobaric hypoxia.While acute exposure may temporarily offset cardiovascular homeostasis insea-level residents, native highlanders have become accustomed to thesehigh-altitude conditions and often exhibit variations in normal ECG parameters.As part of the \"Altitude Non-differentiated ECG Study\" (ANDES) project, thispaper aims to systematically review the available literature regarding ECGchanges in healthy highlander populations. After searching the PubMed, Medline,and Embase databases, 286 abstracts were screened, of which 13 full-texts wereultimately included. This process was completed in accordance with the Preferred Reporting Items for Systematic Reviews and Meta-analyses (PRISMA) guidelines.Major ECG deviations in native healthy highlanders include right QRS axisdeviation, right ventricular hypertrophy signs, and more prevalent T-waveinversion in the right precordial leads. Notably, they exhibit a prolonged QTccompared to sea-level residents, although within normal limits. Evidence aboutincreased P-wave amplitude or duration, variations in PR interval, or greaterprevalence of complete right bundle branch block is not conclusive. This reviewprovides ECG reference standards that can be used by clinicians, who should beaware of the effects of high-altitude residence on cardiovascular health and how these may change according to age, ethnicity, and other factors.\n",
      "CI  - Copyright (c) 2022 Elsevier Ltd. All rights reserved.\n",
      "FAU - Parodi, Josefina B\n",
      "AU  - Parodi JB\n",
      "AD  - Cardiology Division, CEMIC, Ciudad Autonoma de Buenos Aires, Argentina.\n",
      "FAU - Ramchandani, Rashi\n",
      "AU  - Ramchandani R\n",
      "AD  - Department of Medicine, Kingston Health Sciences Center, Queen's University,Kingston, Ontario, Canada.\n",
      "FAU - Zhou, Zier\n",
      "AU  - Zhou Z\n",
      "AD  - Atherosclerosis, Genomics and Vascular Biology Division, University of OttawaHeart Institute, Ottawa, Ontario, Canada.\n",
      "FAU - Chango, Diego X\n",
      "AU  - Chango DX\n",
      "AD  - Cardiology and Advanced Cardiac Imaging Division, Hospital Universitario del Rio,Cuenca, Azuay, Ecuador.\n",
      "FAU - Acunzo, Rafael\n",
      "AU  - Acunzo R\n",
      "AD  - Department of Medicine, University of Buenos Aires, Ciudad Autonoma de BuenosAires, Argentina.\n",
      "FAU - Liblik, Kiera\n",
      "AU  - Liblik K\n",
      "AD  - Department of Medicine, Kingston Health Sciences Center, Queen's University,Kingston, Ontario, Canada.\n",
      "FAU - Farina, Juan M\n",
      "AU  - Farina JM\n",
      "AD  - Cardiovascular and Thoracic Surgery Department, Mayo Clinic, Phoenix, AZ, United States.\n",
      "FAU - Zaidel, Ezequiel J\n",
      "AU  - Zaidel EJ\n",
      "AD  - Cardiology Department, Sanatorio Guemes, Ciudad Autonoma de Buenos Aires,Argentina.\n",
      "FAU - Ruiz-Mori, Enrique\n",
      "AU  - Ruiz-Mori E\n",
      "AD  - Cardiology Department, Instituto de Enfermedades neoplasicas, Lima, Peru.\n",
      "FAU - Carreon, Jose M Arce\n",
      "AU  - Carreon JMA\n",
      "AD  - Cardiology Department, Instituto Nacional del Torax, La Paz, Bolivia.\n",
      "FAU - Liprandi, Alvaro Sosa\n",
      "AU  - Liprandi AS\n",
      "AD  - Cardiology Department, Sanatorio Guemes, Ciudad Autonoma de Buenos Aires,Argentina.\n",
      "FAU - Baranchuk, Adrian\n",
      "AU  - Baranchuk A\n",
      "AD  - Department of Medicine, Kingston Health Sciences Center, Queen's University,Kingston, Ontario, Canada. Electronic address: barancha@kgh.kari.net.\n",
      "CN  - ANDES (Altitude Non-specific Distributed ECG Screening) Project Investigators\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "DEP - 20220202\n",
      "PL  - United States\n",
      "TA  - Trends Cardiovasc Med\n",
      "JT  - Trends in cardiovascular medicine\n",
      "JID - 9108337\n",
      "SB  - IM\n",
      "OTO - NOTNLM\n",
      "OT  - Acclimatization\n",
      "OT  - Altitude\n",
      "OT  - Cardiovascular system\n",
      "OT  - Electrocardiography\n",
      "OT  - Vectorcardiography\n",
      "COIS- Declarations of interest The authors declare that there are no declarations ofinterest.\n",
      "EDAT- 2022/02/06 06:00\n",
      "MHDA- 2022/02/06 06:00\n",
      "CRDT- 2022/02/05 05:38\n",
      "PHST- 2021/10/04 00:00 [received]\n",
      "PHST- 2022/01/17 00:00 [revised]\n",
      "PHST- 2022/01/27 00:00 [accepted]\n",
      "PHST- 2022/02/06 06:00 [pubmed]\n",
      "PHST- 2022/02/06 06:00 [medline]\n",
      "PHST- 2022/02/05 05:38 [entrez]\n",
      "AID - S1050-1738(22)00015-9 [pii]\n",
      "AID - 10.1016/j.tcm.2022.01.013 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Trends Cardiovasc Med. 2022 Feb 2. pii: S1050-1738(22)00015-9. doi:10.1016/j.tcm.2022.01.013.\n",
      "\n",
      "PMID- 35120121\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "LR  - 20220218\n",
      "IS  - 1553-7404 (Electronic)\n",
      "IS  - 1553-7390 (Linking)\n",
      "VI  - 18\n",
      "IP  - 2\n",
      "DP  - 2022 Feb\n",
      "TI  - Population genomics and geographic dispersal in Chagas disease vectors: Landscapedrivers and evidence of possible adaptation to the domestic setting.\n",
      "PG  - e1010019\n",
      "LID - 10.1371/journal.pgen.1010019 [doi]\n",
      "AB  - Accurate prediction of vectors dispersal, as well as identification ofadaptations that allow blood-feeding vectors to thrive in built environments, area basis for effective disease control. Here we adopted a landscape genomicsapproach to assay gene flow, possible local adaptation, and drivers of populationstructure in Rhodnius ecuadoriensis, an important vector of Chagas disease. Weused a reduced-representation sequencing technique (2b-RADseq) to obtain 2,552SNP markers across 272 R. ecuadoriensis samples from 25 collection sites insouthern Ecuador. Evidence of high and directional gene flow between seven wildand domestic population pairs across our study site indicates insecticide-basedcontrol will be hindered by repeated re-infestation of houses from the forest.Preliminary genome scans across multiple population pairs revealed shared outlierloci potentially consistent with local adaptation to the domestic setting, which we mapped to genes involved with embryogenesis and saliva production. Landscapegenomic models showed elevation is a key barrier to R. ecuadoriensis dispersal.Together our results shed early light on the genomic adaptation in triatominevectors and facilitate vector control by predicting that spatially-targeted,proactive interventions would be more efficacious than current, reactiveapproaches.\n",
      "FAU - Hernandez-Castro, Luis E\n",
      "AU  - Hernandez-Castro LE\n",
      "AUID- ORCID: https://orcid.org/0000-0002-2342-1655\n",
      "AD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of Glasgow, Glasgow, United Kingdom.\n",
      "AD  - The Epidemiology, Economics and Risk Assessment Group, The Roslin Institute,Easter Bush Campus, The University of Edinburgh, Midlothian, United Kingdom.\n",
      "FAU - Villacis, Anita G\n",
      "AU  - Villacis AG\n",
      "AD  - Centro de Investigacion para la Salud en America Latina, Facultad de CienciasExactas y Naturales, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Jacobs, Arne\n",
      "AU  - Jacobs A\n",
      "AUID- ORCID: https://orcid.org/0000-0001-7635-5447\n",
      "AD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of Glasgow, Glasgow, United Kingdom.\n",
      "AD  - Department of Natural Resources and the Environment, Cornell University, Ithaca, New York, United States of America.\n",
      "FAU - Cheaib, Bachar\n",
      "AU  - Cheaib B\n",
      "AUID- ORCID: https://orcid.org/0000-0002-0325-823X\n",
      "AD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of Glasgow, Glasgow, United Kingdom.\n",
      "FAU - Day, Casey C\n",
      "AU  - Day CC\n",
      "AUID- ORCID: https://orcid.org/0000-0001-6096-3448\n",
      "AD  - Computational Ecology Lab, School of Public and Community Health Sciences,University of Montana, Missoula, Montana, United States of America.\n",
      "FAU - Ocana-Mayorga, Sofia\n",
      "AU  - Ocana-Mayorga S\n",
      "AUID- ORCID: https://orcid.org/0000-0003-3861-9784\n",
      "AD  - Centro de Investigacion para la Salud en America Latina, Facultad de CienciasExactas y Naturales, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Yumiseva, Cesar A\n",
      "AU  - Yumiseva CA\n",
      "AD  - Centro de Investigacion para la Salud en America Latina, Facultad de CienciasExactas y Naturales, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Bacigalupo, Antonella\n",
      "AU  - Bacigalupo A\n",
      "AUID- ORCID: https://orcid.org/0000-0003-1661-8916\n",
      "AD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of Glasgow, Glasgow, United Kingdom.\n",
      "FAU - Andersson, Bjorn\n",
      "AU  - Andersson B\n",
      "AUID- ORCID: https://orcid.org/0000-0002-4624-0259\n",
      "AD  - Department of Cell and Molecular Biology, Karolinska Institutet, Stockholm,Sweden.\n",
      "FAU - Matthews, Louise\n",
      "AU  - Matthews L\n",
      "AUID- ORCID: https://orcid.org/0000-0003-4420-8367\n",
      "AD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of Glasgow, Glasgow, United Kingdom.\n",
      "FAU - Landguth, Erin L\n",
      "AU  - Landguth EL\n",
      "AUID- ORCID: https://orcid.org/0000-0002-7044-346X\n",
      "AD  - Computational Ecology Lab, School of Public and Community Health Sciences,University of Montana, Missoula, Montana, United States of America.\n",
      "AD  - Center for Population Health Research, School of Public and Community HealthSciences, University of Montana, Missoula, Montana, United States of America.\n",
      "FAU - Costales, Jaime A\n",
      "AU  - Costales JA\n",
      "AUID- ORCID: https://orcid.org/0000-0001-9418-6640\n",
      "AD  - Centro de Investigacion para la Salud en America Latina, Facultad de CienciasExactas y Naturales, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Llewellyn, Martin S\n",
      "AU  - Llewellyn MS\n",
      "AUID- ORCID: https://orcid.org/0000-0001-9856-1591\n",
      "AD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of Glasgow, Glasgow, United Kingdom.\n",
      "FAU - Grijalva, Mario J\n",
      "AU  - Grijalva MJ\n",
      "AUID- ORCID: https://orcid.org/0000-0003-1964-1425\n",
      "AD  - Centro de Investigacion para la Salud en America Latina, Facultad de CienciasExactas y Naturales, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\n",
      "AD  - Infectious and Tropical Disease Institute, Department of Biomedical Sciences,Heritage College of Osteopathic Medicine, Ohio University, Athens, Ohio, UnitedStates of America.\n",
      "LA  - eng\n",
      "GR  - P20 GM130418/GM/NIGMS NIH HHS/United States\n",
      "GR  - R15 AI105749/AI/NIAID NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "DEP - 20220204\n",
      "PL  - United States\n",
      "TA  - PLoS Genet\n",
      "JT  - PLoS genetics\n",
      "JID - 101239074\n",
      "SB  - IM\n",
      "PMC - PMC8849464\n",
      "COIS- The authors have declared that no competing interests exist.\n",
      "EDAT- 2022/02/05 06:00\n",
      "MHDA- 2022/02/05 06:00\n",
      "CRDT- 2022/02/04 17:11\n",
      "PHST- 2021/07/12 00:00 [received]\n",
      "PHST- 2022/01/06 00:00 [accepted]\n",
      "PHST- 2022/02/16 00:00 [revised]\n",
      "PHST- 2022/02/05 06:00 [pubmed]\n",
      "PHST- 2022/02/05 06:00 [medline]\n",
      "PHST- 2022/02/04 17:11 [entrez]\n",
      "AID - 10.1371/journal.pgen.1010019 [doi]\n",
      "AID - PGENETICS-D-21-00937 [pii]\n",
      "PST - epublish\n",
      "SO  - PLoS Genet. 2022 Feb 4;18(2):e1010019. doi: 10.1371/journal.pgen.1010019.eCollection 2022 Feb.\n",
      "\n",
      "PMID- 35113400\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20220207\n",
      "LR  - 20220208\n",
      "IS  - 0065-2598 (Print)\n",
      "IS  - 0065-2598 (Linking)\n",
      "VI  - 1346\n",
      "DP  - 2021\n",
      "TI  - Cataloging Posttranslational Modifications in Plant Histones.\n",
      "PG  - 131-154\n",
      "LID - 10.1007/978-3-030-80352-0_8 [doi]\n",
      "AB  - Eukaryotic DNA exist in the nuclei in the form of a complex with proteins called chromatin. Access to the information encoded in the DNA requires the opening ofthe chromatin. Modulation of the chromatin structure is therefore an importantlayer of regulation for DNA-templated processes. The basic unit of the chromatin is the nucleosome, which contains DNA wrapped around an octamer of histones, H2A,H2B, H3, and H4. Because histones are a structural part of the nucleosome, itsmodification can lead to changes in chromatin structure. Amino acid residues inhistones could be modified with at least 20 different types of functional groups leading to a vast number of modified residues. Here, an overview of the histonemodifications found in plants is provided. We focus mainly in proteomic-basedstudies either aimed to identify PTMs on purified histones or proteome-wideanalysis of particular modifications. The strategies used for catalogingmodifications in plants are also described. Profiling of histone modifications isimportant to begin to understand their functions as mediators of gene regulation in plant biological systems.\n",
      "CI  - (c) 2021. Springer Nature Switzerland AG.\n",
      "FAU - Zacarias, Ericka\n",
      "AU  - Zacarias E\n",
      "AD  - School of Biological Sciences and Engineering, Yachay Tech University, San Miguelde Urcuqui, Ecuador.\n",
      "FAU - Casas-Mollano, J Armando\n",
      "AU  - Casas-Mollano JA\n",
      "AD  - School of Biological Sciences and Engineering, Yachay Tech University, San Miguelde Urcuqui, Ecuador. acasasmollano@gmail.com.\n",
      "AD  - The BioTechnology Institute, College of Biological Sciences, University ofMinnesota, Saint Paul, MN, USA. acasasmollano@gmail.com.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - United States\n",
      "TA  - Adv Exp Med Biol\n",
      "JT  - Advances in experimental medicine and biology\n",
      "JID - 0121103\n",
      "RN  - 0 (Chromatin)\n",
      "RN  - 0 (Histones)\n",
      "RN  - 0 (Nucleosomes)\n",
      "SB  - IM\n",
      "MH  - Chromatin\n",
      "MH  - *Histones/genetics/metabolism\n",
      "MH  - Nucleosomes/genetics\n",
      "MH  - Plants/*genetics\n",
      "MH  - *Protein Processing, Post-Translational\n",
      "MH  - *Proteomics\n",
      "OTO - NOTNLM\n",
      "OT  - Acetylation\n",
      "OT  - Acylation\n",
      "OT  - Chromatin\n",
      "OT  - Core histones\n",
      "OT  - Histone modifications\n",
      "OT  - Mass spectrometry\n",
      "OT  - Methylation\n",
      "OT  - Proteomics\n",
      "EDAT- 2022/02/04 06:00\n",
      "MHDA- 2022/02/08 06:00\n",
      "CRDT- 2022/02/03 12:18\n",
      "PHST- 2022/02/03 12:18 [entrez]\n",
      "PHST- 2022/02/04 06:00 [pubmed]\n",
      "PHST- 2022/02/08 06:00 [medline]\n",
      "AID - 10.1007/978-3-030-80352-0_8 [doi]\n",
      "PST - ppublish\n",
      "SO  - Adv Exp Med Biol. 2021;1346:131-154. doi: 10.1007/978-3-030-80352-0_8.\n",
      "\n",
      "PMID- 35104837\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "LR  - 20220201\n",
      "IS  - 1476-4687 (Electronic)\n",
      "IS  - 0028-0836 (Linking)\n",
      "DP  - 2022 Feb 1\n",
      "TI  - Altered TMPRSS2 usage by SARS-CoV-2 Omicron impacts tropism and fusogenicity.\n",
      "LID - 10.1038/s41586-022-04474-x [doi]\n",
      "AB  - The SARS-CoV-2 Omicron BA.1 variant emerged in 2021(1) and bears multiple spikemutations(2). Here we show that Omicron spike has higher affinity for ACE2compared to Delta as well as a marked change of antigenicity conferringsignificant evasion of therapeutic monoclonal and vaccine-elicited polyclonalneutralising antibodies after two doses. mRNA vaccination as a third vaccine doserescues and broadens neutralisation. Importantly, antiviral drugs remdesivir and molnupiravir retain efficacy against Omicron BA.1. Replication was similar forOmicron and Delta virus isolates in human nasal epithelial cultures. However, in lower airway organoids, lung cells and gut cells, Omicron demonstrated lowerreplication. Omicron spike protein was less efficiently cleaved compared toDelta. Replication differences mapped to entry efficiency using spike pseudotypedvirus (PV) assays. The defect for Omicron PV to enter specific cell typeseffectively correlated with higher cellular RNA expression of TMPRSS2, and knock down of TMPRSS2 impacted Delta entry to a greater extent than Omicron.Furthermore, drug inhibitors targeting specific entry pathways(3) demonstratedthat the Omicron spike inefficiently utilises the cellular protease TMPRSS2 that promotes cell entry via plasma membrane fusion, with greater dependency on cellentry via the endocytic pathway. Consistent with suboptimal S1/S2 cleavage andinability to utilise TMPRSS2, syncytium formation by the Omicron spike wasmarkedly impaired compared to the Delta spike. Omicron's less efficient spikecleavage at S1/S2 is associated with shift in cellular tropism away from TMPRSS2 expressing cells, with implications for altered pathogenesis.\n",
      "CI  - (c) 2022. The Author(s), under exclusive licence to Springer Nature Limited.\n",
      "FAU - Meng, Bo\n",
      "AU  - Meng B\n",
      "AD  - Cambridge Institute of Therapeutic Immunology & Infectious Disease (CITIID),Cambridge, UK.\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "FAU - Abdullahi, Adam\n",
      "AU  - Abdullahi A\n",
      "AD  - Cambridge Institute of Therapeutic Immunology & Infectious Disease (CITIID),Cambridge, UK.\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "FAU - Ferreira, Isabella A T M\n",
      "AU  - Ferreira IATM\n",
      "AD  - Cambridge Institute of Therapeutic Immunology & Infectious Disease (CITIID),Cambridge, UK.\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "FAU - Goonawardane, Niluka\n",
      "AU  - Goonawardane N\n",
      "AD  - Cambridge Institute of Therapeutic Immunology & Infectious Disease (CITIID),Cambridge, UK.\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "FAU - Saito, Akatsuki\n",
      "AU  - Saito A\n",
      "AD  - Department of Veterinary Science, Faculty of Agriculture, University of Miyazaki,Miyazaki, Japan.\n",
      "FAU - Kimura, Izumi\n",
      "AU  - Kimura I\n",
      "AD  - Division of Systems Virology, Department of Infectious Disease Control,International Research Center for Infectious Diseases, The Institute of MedicalScience, The University of Tokyo, Tokyo, Japan.\n",
      "FAU - Yamasoba, Daichi\n",
      "AU  - Yamasoba D\n",
      "AD  - Division of Systems Virology, Department of Infectious Disease Control,International Research Center for Infectious Diseases, The Institute of MedicalScience, The University of Tokyo, Tokyo, Japan.\n",
      "FAU - Gerber, Pehuen Pereyra\n",
      "AU  - Gerber PP\n",
      "AD  - Cambridge Institute of Therapeutic Immunology & Infectious Disease (CITIID),Cambridge, UK.\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "FAU - Fatihi, Saman\n",
      "AU  - Fatihi S\n",
      "AD  - CSIR Institute of Genomics and Integrative Biology, Delhi, India.\n",
      "FAU - Rathore, Surabhi\n",
      "AU  - Rathore S\n",
      "AD  - CSIR Institute of Genomics and Integrative Biology, Delhi, India.\n",
      "FAU - Zepeda, Samantha K\n",
      "AU  - Zepeda SK\n",
      "AD  - Department of Biochemistry, University of Washington, Seattle, USA.\n",
      "FAU - Papa, Guido\n",
      "AU  - Papa G\n",
      "AD  - MRC - Laboratory of Molecular Biology, Cambridge, UK.\n",
      "FAU - Kemp, Steven A\n",
      "AU  - Kemp SA\n",
      "AD  - Cambridge Institute of Therapeutic Immunology & Infectious Disease (CITIID),Cambridge, UK.\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "FAU - Ikeda, Terumasa\n",
      "AU  - Ikeda T\n",
      "AD  - Division of Infection and Immunity, Joint Research Center for Human Retrovirusinfection, Kumamoto University, Kumamoto, Japan.\n",
      "FAU - Toyoda, Mako\n",
      "AU  - Toyoda M\n",
      "AD  - Kuramochi Clinic Interpark, Utsunomiya, Tochigi, Japan.\n",
      "FAU - Tan, Toong Seng\n",
      "AU  - Tan TS\n",
      "AD  - Kuramochi Clinic Interpark, Utsunomiya, Tochigi, Japan.\n",
      "FAU - Kuramochi, Jin\n",
      "AU  - Kuramochi J\n",
      "AD  - Human Genetics Laboratory, National Institute of Genetics, Mishima, Shizuoka,Japan.\n",
      "FAU - Mitsunaga, Shigeki\n",
      "AU  - Mitsunaga S\n",
      "AD  - Department of Hematology and Oncology, Graduate School of Medicine, KyotoUniversity, Kyoto, Japan.\n",
      "FAU - Ueno, Takamasa\n",
      "AU  - Ueno T\n",
      "AD  - Kuramochi Clinic Interpark, Utsunomiya, Tochigi, Japan.\n",
      "FAU - Shirakawa, Kotaro\n",
      "AU  - Shirakawa K\n",
      "AD  - Department of Hematology and Oncology, Graduate School of Medicine, KyotoUniversity, Kyoto, Japan.\n",
      "FAU - Takaori-Kondo, Akifumi\n",
      "AU  - Takaori-Kondo A\n",
      "AD  - Department of Hematology and Oncology, Graduate School of Medicine, KyotoUniversity, Kyoto, Japan.\n",
      "FAU - Brevini, Teresa\n",
      "AU  - Brevini T\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "FAU - Mallery, Donna L\n",
      "AU  - Mallery DL\n",
      "AD  - MRC - Laboratory of Molecular Biology, Cambridge, UK.\n",
      "FAU - Charles, Oscar J\n",
      "AU  - Charles OJ\n",
      "AD  - Division of Infection and Immunity, UCL, London, UK.\n",
      "CN  - CITIID-NIHR BioResource COVID- Collaboration\n",
      "CN  - Genotype to Phenotype Japan (GP-Japan) Consortium members\n",
      "CN  - Ecuador-COVID19 Consortium\n",
      "FAU - Bowen, John E\n",
      "AU  - Bowen JE\n",
      "AD  - Department of Biochemistry, University of Washington, Seattle, USA.\n",
      "FAU - Joshi, Anshu\n",
      "AU  - Joshi A\n",
      "AD  - Department of Biochemistry, University of Washington, Seattle, USA.\n",
      "FAU - Walls, Alexandra C\n",
      "AU  - Walls AC\n",
      "AD  - Department of Biochemistry, University of Washington, Seattle, USA.\n",
      "AD  - Department of Virology, University of Cambridge, Cambridge, UK.\n",
      "FAU - Jackson, Laurelle\n",
      "AU  - Jackson L\n",
      "AD  - Africa Health Research Institute, Durban, South Africa.\n",
      "FAU - Martin, Darren\n",
      "AU  - Martin D\n",
      "AD  - University of Cape Town, Cape Town, South Africa.\n",
      "FAU - Smith, Kenneth G C\n",
      "AU  - Smith KGC\n",
      "AD  - Cambridge Institute of Therapeutic Immunology & Infectious Disease (CITIID),Cambridge, UK.\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "FAU - Bradley, John\n",
      "AU  - Bradley J\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "FAU - Briggs, John A G\n",
      "AU  - Briggs JAG\n",
      "AD  - Max Planck Institute of Biochemistry, Martinsried, Germany.\n",
      "FAU - Choi, Jinwook\n",
      "AU  - Choi J\n",
      "AD  - Wellcome-MRC Cambridge Stem Cell Institute, Cambridge, UK.\n",
      "FAU - Madissoon, Elo\n",
      "AU  - Madissoon E\n",
      "AD  - Welcome Sanger Institute, Wellcome Genome Campus, Hinxton, Cambridge, UK.\n",
      "AD  - European Molecular Biology Laboratory, European Bioinformatics Institute,EMBL-EBI, Wellcome Trust Genome Campus, Hinxton, UK.\n",
      "FAU - Meyer, Kerstin\n",
      "AU  - Meyer K\n",
      "AD  - Welcome Sanger Institute, Wellcome Genome Campus, Hinxton, Cambridge, UK.\n",
      "FAU - Mlcochova, Petra\n",
      "AU  - Mlcochova P\n",
      "AD  - Cambridge Institute of Therapeutic Immunology & Infectious Disease (CITIID),Cambridge, UK.\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "FAU - Ceron-Gutierrez, Lourdes\n",
      "AU  - Ceron-Gutierrez L\n",
      "AD  - Cambridge University Hospitals NHS Foundation Trust, Addenbrookes Hospital,Cambridge, UK.\n",
      "FAU - Doffinger, Rainer\n",
      "AU  - Doffinger R\n",
      "AD  - Cambridge University Hospitals NHS Foundation Trust, Addenbrookes Hospital,Cambridge, UK.\n",
      "FAU - Teichmann, Sarah A\n",
      "AU  - Teichmann SA\n",
      "AD  - Welcome Sanger Institute, Wellcome Genome Campus, Hinxton, Cambridge, UK.\n",
      "AD  - Cavendish Laboratory/Dept Physics, University of Cambridge, JJ Thomson Ave,Cambridge, UK.\n",
      "FAU - Fisher, Andrew J\n",
      "AU  - Fisher AJ\n",
      "AD  - Transplant and Regenerative Medicine Laboratory, Translational and ClinicalResearch Institute, Faculty of Medical Sciences, Newcastle University, Newcastle Upon Tyne, UK.\n",
      "FAU - Pizzuto, Matteo S\n",
      "AU  - Pizzuto MS\n",
      "AD  - Humabs Biomed SA, a subsidiary of Vir Biotechnology, Bellinzona, Switzerland.\n",
      "FAU - de Marco, Anna\n",
      "AU  - de Marco A\n",
      "AD  - Humabs Biomed SA, a subsidiary of Vir Biotechnology, Bellinzona, Switzerland.\n",
      "FAU - Corti, Davide\n",
      "AU  - Corti D\n",
      "AD  - Humabs Biomed SA, a subsidiary of Vir Biotechnology, Bellinzona, Switzerland.\n",
      "FAU - Hosmillo, Myra\n",
      "AU  - Hosmillo M\n",
      "AD  - Department of Virology, University of Cambridge, Cambridge, UK.\n",
      "FAU - Lee, Joo Hyeon\n",
      "AU  - Lee JH\n",
      "AD  - Wellcome-MRC Cambridge Stem Cell Institute, Cambridge, UK.\n",
      "AD  - Department of Physiology, Development and Neuroscience, University of Cambridge, Cambridge, UK.\n",
      "FAU - James, Leo C\n",
      "AU  - James LC\n",
      "AD  - MRC - Laboratory of Molecular Biology, Cambridge, UK.\n",
      "FAU - Thukral, Lipi\n",
      "AU  - Thukral L\n",
      "AD  - Department of Biochemistry, University of Washington, Seattle, USA.\n",
      "FAU - Veesler, David\n",
      "AU  - Veesler D\n",
      "AD  - Department of Biochemistry, University of Washington, Seattle, USA.\n",
      "AD  - Howard Hughes Medical Institute, Seattle, WA, USA.\n",
      "FAU - Sigal, Alex\n",
      "AU  - Sigal A\n",
      "AD  - University of Cape Town, Cape Town, South Africa.\n",
      "AD  - Max Planck Institute for Infection Biology, Berlin, Germany.\n",
      "FAU - Sampaziotis, Fotios\n",
      "AU  - Sampaziotis F\n",
      "AD  - Cambridge Institute of Therapeutic Immunology & Infectious Disease (CITIID),Cambridge, UK.\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "AD  - Wellcome-MRC Cambridge Stem Cell Institute, Cambridge, UK.\n",
      "AD  - Cambridge University Hospitals NHS Foundation Trust, Addenbrookes Hospital,Cambridge, UK.\n",
      "FAU - Goodfellow, Ian G\n",
      "AU  - Goodfellow IG\n",
      "AD  - Department of Virology, University of Cambridge, Cambridge, UK.\n",
      "FAU - Matheson, Nicholas J\n",
      "AU  - Matheson NJ\n",
      "AD  - Cambridge Institute of Therapeutic Immunology & Infectious Disease (CITIID),Cambridge, UK.\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK.\n",
      "AD  - Cambridge University Hospitals NHS Foundation Trust, Addenbrookes Hospital,Cambridge, UK.\n",
      "AD  - NHS Blood and Transplant, Cambridge, UK.\n",
      "FAU - Sato, Kei\n",
      "AU  - Sato K\n",
      "AD  - Division of Systems Virology, Department of Infectious Disease Control,International Research Center for Infectious Diseases, The Institute of MedicalScience, The University of Tokyo, Tokyo, Japan. keisato@g.ecc.u-tokyo.ac.jp.\n",
      "AD  - CREST, Japan Science and Technology Agency, Saitama, Japan.keisato@g.ecc.u-tokyo.ac.jp.\n",
      "FAU - Gupta, Ravindra K\n",
      "AU  - Gupta RK\n",
      "AUID- ORCID: http://orcid.org/0000-0001-9751-1808\n",
      "AD  - Cambridge Institute of Therapeutic Immunology & Infectious Disease (CITIID),Cambridge, UK. rkg20@cam.ac.uk.\n",
      "AD  - Department of Medicine, University of Cambridge, Cambridge, UK. rkg20@cam.ac.uk.\n",
      "AD  - Africa Health Research Institute, Durban, South Africa. rkg20@cam.ac.uk.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20220201\n",
      "PL  - England\n",
      "TA  - Nature\n",
      "JT  - Nature\n",
      "JID - 0410462\n",
      "SB  - IM\n",
      "IR  - Baker S\n",
      "FIR - Baker, Stephen\n",
      "IR  - Dougan G\n",
      "FIR - Dougan, Gordon\n",
      "IR  - Hess C\n",
      "FIR - Hess, Christoph\n",
      "IR  - Kingston N\n",
      "FIR - Kingston, Nathalie\n",
      "IR  - Lehner PJ\n",
      "FIR - Lehner, Paul J\n",
      "IR  - Lyons PA\n",
      "FIR - Lyons, Paul A\n",
      "IR  - Matheson NJ\n",
      "FIR - Matheson, Nicholas J\n",
      "IR  - Owehand WH\n",
      "FIR - Owehand, Willem H\n",
      "IR  - Saunders C\n",
      "FIR - Saunders, Caroline\n",
      "IR  - Summers C\n",
      "FIR - Summers, Charlotte\n",
      "IR  - Thaventhiran JED\n",
      "FIR - Thaventhiran, James E D\n",
      "IR  - Toshner M\n",
      "FIR - Toshner, Mark\n",
      "IR  - Weekes MP\n",
      "FIR - Weekes, Michael P\n",
      "IR  - Maxwell P\n",
      "FIR - Maxwell, Patrick\n",
      "IR  - Shaw A\n",
      "FIR - Shaw, Ashley\n",
      "IR  - Bucke A\n",
      "FIR - Bucke, Ashlea\n",
      "IR  - Calder J\n",
      "FIR - Calder, Jo\n",
      "IR  - Canna L\n",
      "FIR - Canna, Laura\n",
      "IR  - Domingo J\n",
      "FIR - Domingo, Jason\n",
      "IR  - Elmer A\n",
      "FIR - Elmer, Anne\n",
      "IR  - Fuller S\n",
      "FIR - Fuller, Stewart\n",
      "IR  - Harris J\n",
      "FIR - Harris, Julie\n",
      "IR  - Hewitt S\n",
      "FIR - Hewitt, Sarah\n",
      "IR  - Kennet J\n",
      "FIR - Kennet, Jane\n",
      "IR  - Jose S\n",
      "FIR - Jose, Sherly\n",
      "IR  - Kourampa J\n",
      "FIR - Kourampa, Jenny\n",
      "IR  - Meadows A\n",
      "FIR - Meadows, Anne\n",
      "IR  - O'Brien C\n",
      "FIR - O'Brien, Criona\n",
      "IR  - Price J\n",
      "FIR - Price, Jane\n",
      "IR  - Publico C\n",
      "FIR - Publico, Cherry\n",
      "IR  - Rastall R\n",
      "FIR - Rastall, Rebecca\n",
      "IR  - Ribeiro C\n",
      "FIR - Ribeiro, Carla\n",
      "IR  - Rowlands J\n",
      "FIR - Rowlands, Jane\n",
      "IR  - Ruffolo V\n",
      "FIR - Ruffolo, Valentina\n",
      "IR  - Tordesillas H\n",
      "FIR - Tordesillas, Hugo\n",
      "IR  - Bullman B\n",
      "FIR - Bullman, Ben\n",
      "IR  - Dunmore BJ\n",
      "FIR - Dunmore, Benjamin J\n",
      "IR  - Fawke S\n",
      "FIR - Fawke, Stuart\n",
      "IR  - Graf S\n",
      "FIR - Graf, Stefan\n",
      "IR  - Hodgson J\n",
      "FIR - Hodgson, Josh\n",
      "IR  - Huang C\n",
      "FIR - Huang, Christopher\n",
      "IR  - Hunter K\n",
      "FIR - Hunter, Kelvin\n",
      "IR  - Jones E\n",
      "FIR - Jones, Emma\n",
      "IR  - Legchenko E\n",
      "FIR - Legchenko, Ekaterina\n",
      "IR  - Matara C\n",
      "FIR - Matara, Cecilia\n",
      "IR  - Martin J\n",
      "FIR - Martin, Jennifer\n",
      "IR  - Mescia F\n",
      "FIR - Mescia, Federica\n",
      "IR  - O'Donnell C\n",
      "FIR - O'Donnell, Ciara\n",
      "IR  - Pointon L\n",
      "FIR - Pointon, Linda\n",
      "IR  - Shih J\n",
      "FIR - Shih, Joy\n",
      "IR  - Sutcliffe R\n",
      "FIR - Sutcliffe, Rachel\n",
      "IR  - Tilly T\n",
      "FIR - Tilly, Tobias\n",
      "IR  - Treacy C\n",
      "FIR - Treacy, Carmen\n",
      "IR  - Tong Z\n",
      "FIR - Tong, Zhen\n",
      "IR  - Wood J\n",
      "FIR - Wood, Jennifer\n",
      "IR  - Wylot M\n",
      "FIR - Wylot, Marta\n",
      "IR  - Betancourt A\n",
      "FIR - Betancourt, Ariana\n",
      "IR  - Bower G\n",
      "FIR - Bower, Georgie\n",
      "IR  - Cossetti C\n",
      "FIR - Cossetti, Chiara\n",
      "IR  - De Sa A\n",
      "FIR - De Sa, Aloka\n",
      "IR  - Epping M\n",
      "FIR - Epping, Madeline\n",
      "IR  - Fawke S\n",
      "FIR - Fawke, Stuart\n",
      "IR  - Gleadall N\n",
      "FIR - Gleadall, Nick\n",
      "IR  - Grenfell R\n",
      "FIR - Grenfell, Richard\n",
      "IR  - Hinch A\n",
      "FIR - Hinch, Andrew\n",
      "IR  - Jackson S\n",
      "FIR - Jackson, Sarah\n",
      "IR  - Jarvis I\n",
      "FIR - Jarvis, Isobel\n",
      "IR  - Krishna B\n",
      "FIR - Krishna, Ben\n",
      "IR  - Nice F\n",
      "FIR - Nice, Francesca\n",
      "IR  - Omarjee O\n",
      "FIR - Omarjee, Ommar\n",
      "IR  - Perera M\n",
      "FIR - Perera, Marianne\n",
      "IR  - Potts M\n",
      "FIR - Potts, Martin\n",
      "IR  - Richoz N\n",
      "FIR - Richoz, Nathan\n",
      "IR  - Romashova V\n",
      "FIR - Romashova, Veronika\n",
      "IR  - Stefanucci L\n",
      "FIR - Stefanucci, Luca\n",
      "IR  - Strezlecki M\n",
      "FIR - Strezlecki, Mateusz\n",
      "IR  - Turner L\n",
      "FIR - Turner, Lori\n",
      "IR  - De Bie EMDD\n",
      "FIR - De Bie, Eckart M D D\n",
      "IR  - Bunclark K\n",
      "FIR - Bunclark, Katherine\n",
      "IR  - Josipovic M\n",
      "FIR - Josipovic, Masa\n",
      "IR  - Mackay M\n",
      "FIR - Mackay, Michael\n",
      "IR  - Allison J\n",
      "FIR - Allison, John\n",
      "IR  - Butcher H\n",
      "FIR - Butcher, Helen\n",
      "IR  - Caputo D\n",
      "FIR - Caputo, Daniela\n",
      "IR  - Clapham-Riley D\n",
      "FIR - Clapham-Riley, Debbie\n",
      "IR  - Dewhurst E\n",
      "FIR - Dewhurst, Eleanor\n",
      "IR  - Furlong A\n",
      "FIR - Furlong, Anita\n",
      "IR  - Graves B\n",
      "FIR - Graves, Barbara\n",
      "IR  - Gray J\n",
      "FIR - Gray, Jennifer\n",
      "IR  - Ivers T\n",
      "FIR - Ivers, Tasmin\n",
      "IR  - Le Gresley E\n",
      "FIR - Le Gresley, Emma\n",
      "IR  - Linger R\n",
      "FIR - Linger, Rachel\n",
      "IR  - Meloy S\n",
      "FIR - Meloy, Sarah\n",
      "IR  - Muldoon F\n",
      "FIR - Muldoon, Francesca\n",
      "IR  - Ovington N\n",
      "FIR - Ovington, Nigel\n",
      "IR  - Papadia S\n",
      "FIR - Papadia, Sofia\n",
      "IR  - Phelan I\n",
      "FIR - Phelan, Isabel\n",
      "IR  - Stark H\n",
      "FIR - Stark, Hannah\n",
      "IR  - Stirrups KE\n",
      "FIR - Stirrups, Kathleen E\n",
      "IR  - Townsend P\n",
      "FIR - Townsend, Paul\n",
      "IR  - Walker N\n",
      "FIR - Walker, Neil\n",
      "IR  - Webster J\n",
      "FIR - Webster, Jennifer\n",
      "IR  - Scholtes I\n",
      "FIR - Scholtes, Ingrid\n",
      "IR  - Hein S\n",
      "FIR - Hein, Sabine\n",
      "IR  - King R\n",
      "FIR - King, Rebecca\n",
      "IR  - Butlertanaka EP\n",
      "FIR - Butlertanaka, Erika P\n",
      "IR  - Tanaka YL\n",
      "FIR - Tanaka, Yuri L\n",
      "IR  - Ikeda T\n",
      "FIR - Ikeda, Terumasa\n",
      "IR  - Ito J\n",
      "FIR - Ito, Jumpei\n",
      "IR  - Uriu K\n",
      "FIR - Uriu, Keiya\n",
      "IR  - Kosugi Y\n",
      "FIR - Kosugi, Yusuke\n",
      "IR  - Suganami M\n",
      "FIR - Suganami, Mai\n",
      "IR  - Oide A\n",
      "FIR - Oide, Akiko\n",
      "IR  - Yokoyama M\n",
      "FIR - Yokoyama, Miyabishara\n",
      "IR  - Chiba M\n",
      "FIR - Chiba, Mika\n",
      "IR  - Motozono C\n",
      "FIR - Motozono, Chihiro\n",
      "IR  - Nasser H\n",
      "FIR - Nasser, Hesham\n",
      "IR  - Shimizu R\n",
      "FIR - Shimizu, Ryo\n",
      "IR  - Yuan Y\n",
      "FIR - Yuan, Yue\n",
      "IR  - Kitazato K\n",
      "FIR - Kitazato, Kazuko\n",
      "IR  - Hasebe H\n",
      "FIR - Hasebe, Haruyo\n",
      "IR  - Nakagawa S\n",
      "FIR - Nakagawa, So\n",
      "IR  - Wu J\n",
      "FIR - Wu, Jiaqi\n",
      "IR  - Takahashi M\n",
      "FIR - Takahashi, Miyoko\n",
      "IR  - Fukuhara T\n",
      "FIR - Fukuhara, Takasuke\n",
      "IR  - Shimizu K\n",
      "FIR - Shimizu, Kenta\n",
      "IR  - Tsushima K\n",
      "FIR - Tsushima, Kana\n",
      "IR  - Kubo H\n",
      "FIR - Kubo, Haruko\n",
      "IR  - Kazuma Y\n",
      "FIR - Kazuma, Yasuhiro\n",
      "IR  - Nomura R\n",
      "FIR - Nomura, Ryosuke\n",
      "IR  - Horisawa Y\n",
      "FIR - Horisawa, Yoshihito\n",
      "IR  - Nagata K\n",
      "FIR - Nagata, Kayoko\n",
      "IR  - Kawai Y\n",
      "FIR - Kawai, Yugo\n",
      "IR  - Yanagida Y\n",
      "FIR - Yanagida, Yohei\n",
      "IR  - Tashiro Y\n",
      "FIR - Tashiro, Yusuke\n",
      "IR  - Tokunaga K\n",
      "FIR - Tokunaga, Kenzo\n",
      "IR  - Ozono S\n",
      "FIR - Ozono, Seiya\n",
      "IR  - Kawabata R\n",
      "FIR - Kawabata, Ryoko\n",
      "IR  - Morizako N\n",
      "FIR - Morizako, Nanami\n",
      "IR  - Sadamasu K\n",
      "FIR - Sadamasu, Kenji\n",
      "IR  - Asakura H\n",
      "FIR - Asakura, Hiroyuki\n",
      "IR  - Nagashima M\n",
      "FIR - Nagashima, Mami\n",
      "IR  - Yoshimura K\n",
      "FIR - Yoshimura, Kazuhisa\n",
      "IR  - Cardenas P\n",
      "FIR - Cardenas, Paul\n",
      "IR  - Munoz E\n",
      "FIR - Munoz, Erika\n",
      "IR  - Barragan V\n",
      "FIR - Barragan, Veronica\n",
      "IR  - Marquez S\n",
      "FIR - Marquez, Sully\n",
      "IR  - Prado-Vivar B\n",
      "FIR - Prado-Vivar, Belen\n",
      "IR  - Becerra-Wong M\n",
      "FIR - Becerra-Wong, Monica\n",
      "IR  - Caravajal M\n",
      "FIR - Caravajal, Mateo\n",
      "IR  - Trueba G\n",
      "FIR - Trueba, Gabriel\n",
      "IR  - Rojas-Silva P\n",
      "FIR - Rojas-Silva, Patricio\n",
      "IR  - Grunauer M\n",
      "FIR - Grunauer, Michelle\n",
      "IR  - Gutierrez B\n",
      "FIR - Gutierrez, Bernardo\n",
      "IR  - Guadalupe JJ\n",
      "FIR - Guadalupe, Juan Jose\n",
      "IR  - Fernandez-Cadena JC\n",
      "FIR - Fernandez-Cadena, Juan Carlos\n",
      "IR  - Andrade-Molina D\n",
      "FIR - Andrade-Molina, Derly\n",
      "IR  - Baldeon M\n",
      "FIR - Baldeon, Manuel\n",
      "IR  - Pinos A\n",
      "FIR - Pinos, Andrea\n",
      "EDAT- 2022/02/02 06:00\n",
      "MHDA- 2022/02/02 06:00\n",
      "CRDT- 2022/02/01 20:12\n",
      "PHST- 2022/02/01 20:12 [entrez]\n",
      "PHST- 2022/02/02 06:00 [pubmed]\n",
      "PHST- 2022/02/02 06:00 [medline]\n",
      "AID - 10.1038/s41586-022-04474-x [doi]\n",
      "AID - 10.1038/s41586-022-04474-x [pii]\n",
      "PST - aheadofprint\n",
      "SO  - Nature. 2022 Feb 1. pii: 10.1038/s41586-022-04474-x. doi:10.1038/s41586-022-04474-x.\n",
      "\n",
      "PMID- 35064679\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "LR  - 20220208\n",
      "IS  - 1469-8137 (Electronic)\n",
      "IS  - 0028-646X (Linking)\n",
      "DP  - 2022 Jan 22\n",
      "TI  - Discovery and characterization of sweetpotato's closest tetraploid relative.\n",
      "LID - 10.1111/nph.17991 [doi]\n",
      "AB  - The origin of sweetpotato, a hexaploid species, is poorly understood, partlybecause the identity of its tetraploid progenitor remains unknown. In this study,we identify, describe and characterize a new species of Ipomoea that issweetpotato's closest tetraploid relative known to date and probably a directdescendant of its tetraploid progenitor. We integrate morphological,phylogenetic, and genomic analyses of herbarium and germplasm accessions of thehexaploid sweetpotato, its closest known diploid relative Ipomoea trifida, andvarious tetraploid plants closely related to them from across the Americancontinent. We identify wild autotetraploid plants from Ecuador that aremorphologically distinct from Ipomoea batatas and I. trifida, but monophyleticand sister to I. batatas in phylogenetic analysis of nuclear data. We describethis new species as Ipomoea aequatoriensis T. Wells & P. Munoz sp. nov.,distinguish it from hybrid tetraploid material collected in Mexico; and show thatit likely played a direct role in the origin of sweetpotato's hexaploid genome.This discovery transforms our understanding of sweetpotato's origin.\n",
      "CI  - (c) 2022 The Authors. New Phytologist (c) 2022 New Phytologist Foundation.\n",
      "FAU - Munoz-Rodriguez, Pablo\n",
      "AU  - Munoz-Rodriguez P\n",
      "AUID- ORCID: https://orcid.org/0000-0002-3580-8136\n",
      "AD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford, OX13RB, UK.\n",
      "FAU - Wells, Tom\n",
      "AU  - Wells T\n",
      "AUID- ORCID: https://orcid.org/0000-0002-4664-7868\n",
      "AD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford, OX13RB, UK.\n",
      "FAU - Wood, John R I\n",
      "AU  - Wood JRI\n",
      "AUID- ORCID: https://orcid.org/0000-0001-5102-3729\n",
      "AD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford, OX13RB, UK.\n",
      "AD  - Royal Botanic Gardens, Kew, Richmond, Surrey, TW9 3AB, UK.\n",
      "FAU - Carruthers, Tom\n",
      "AU  - Carruthers T\n",
      "AUID- ORCID: https://orcid.org/0000-0003-1586-3557\n",
      "AD  - Royal Botanic Gardens, Kew, Richmond, Surrey, TW9 3AB, UK.\n",
      "FAU - Anglin, Noelle L\n",
      "AU  - Anglin NL\n",
      "AUID- ORCID: https://orcid.org/0000-0002-3454-1142\n",
      "AD  - International Potato Center, Avenida La Molina 1895, Distrito de La Molina, Lima,15023, Peru.\n",
      "AD  - United States Department of Agriculture, 1109 Experiment Street, Griffin, GA,3022, USA.\n",
      "FAU - Jarret, Robert L\n",
      "AU  - Jarret RL\n",
      "AUID- ORCID: https://orcid.org/0000-0002-0426-6186\n",
      "AD  - United States Department of Agriculture, 1109 Experiment Street, Griffin, GA,3022, USA.\n",
      "FAU - Scotland, Robert W\n",
      "AU  - Scotland RW\n",
      "AUID- ORCID: https://orcid.org/0000-0002-6371-2238\n",
      "AD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford, OX13RB, UK.\n",
      "LA  - eng\n",
      "GR  - T001445/1/BB_/Biotechnology and Biological Sciences Research Council/UnitedKingdom\n",
      "PT  - Journal Article\n",
      "DEP - 20220122\n",
      "PL  - England\n",
      "TA  - New Phytol\n",
      "JT  - The New phytologist\n",
      "JID - 9882884\n",
      "SB  - IM\n",
      "OTO - NOTNLM\n",
      "OT  - Ipomoea aequatoriensis\n",
      "OT  - Ecuador\n",
      "OT  - crop wild relatives\n",
      "OT  - genomics\n",
      "OT  - herbarium specimens\n",
      "OT  - new species\n",
      "OT  - tetraploid\n",
      "EDAT- 2022/01/23 06:00\n",
      "MHDA- 2022/01/23 06:00\n",
      "CRDT- 2022/01/22 08:36\n",
      "PHST- 2021/11/23 00:00 [received]\n",
      "PHST- 2022/01/16 00:00 [accepted]\n",
      "PHST- 2022/01/23 06:00 [pubmed]\n",
      "PHST- 2022/01/23 06:00 [medline]\n",
      "PHST- 2022/01/22 08:36 [entrez]\n",
      "AID - 10.1111/nph.17991 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - New Phytol. 2022 Jan 22. doi: 10.1111/nph.17991.\n",
      "\n",
      "PMID- 35047841\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20220121\n",
      "IS  - 2666-2477 (Electronic)\n",
      "IS  - 2666-2477 (Linking)\n",
      "VI  - 2\n",
      "IP  - 4\n",
      "DP  - 2021 Oct 14\n",
      "TI  - Genetic ancestry and ethnic identity in Ecuador.\n",
      "PG  - 100050\n",
      "LID - 10.1016/j.xhgg.2021.100050 [doi]\n",
      "AB  - We investigated the ancestral origins of four Ecuadorian ethnicgroups-Afro-Ecuadorian, Mestizo, Montubio, and the Indigenous Tsachila-in aneffort to gain insight on the relationship between ancestry, culture, and theformation of ethnic identities in Latin America. The observed patterns of geneticancestry are largely concordant with ethnic identities and historical records of conquest and colonization in Ecuador. Nevertheless, a number of exceptionalfindings highlight the complex relationship between genetic ancestry andethnicity in Ecuador. Afro-Ecuadorians show far less African ancestry, and thehighest levels of Native American ancestry, seen for any Afro-descendantpopulation in the Americas. Mestizos in Ecuador show high levels of NativeAmerican ancestry, with substantially less European ancestry, despite therelatively low Indigenous population in the country. The recently recognizedMontubio ethnic group is highly admixed, with substantial contributions from all three continental ancestries. The Tsachila show two distinct ancestry subgroups, with most individuals showing almost exclusively Native American ancestry and asmaller group showing a Mestizo characteristic pattern. Considered together with historical data and sociological studies, our results indicate the extent towhich ancestry and culture interact, often in unexpected ways, to shape ethnicidentity in Ecuador.\n",
      "CI  - (c) 2021 The Authors.\n",
      "FAU - Nagar, Shashwat Deepali\n",
      "AU  - Nagar SD\n",
      "AD  - School of Biological Sciences, Georgia Institute of Technology, Atlanta, GA, USA.\n",
      "AD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\n",
      "FAU - Conley, Andrew B\n",
      "AU  - Conley AB\n",
      "AD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\n",
      "AD  - IHRC-Georgia Tech Applied Bioinformatics Laboratory, Atlanta, GA, USA.\n",
      "AD  - National Institute on Minority Health and Health Disparities, National Institutesof Health, Bethesda, MD, USA.\n",
      "FAU - Chande, Aroon T\n",
      "AU  - Chande AT\n",
      "AD  - School of Biological Sciences, Georgia Institute of Technology, Atlanta, GA, USA.\n",
      "AD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\n",
      "AD  - IHRC-Georgia Tech Applied Bioinformatics Laboratory, Atlanta, GA, USA.\n",
      "FAU - Rishishwar, Lavanya\n",
      "AU  - Rishishwar L\n",
      "AD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\n",
      "AD  - IHRC-Georgia Tech Applied Bioinformatics Laboratory, Atlanta, GA, USA.\n",
      "AD  - National Institute on Minority Health and Health Disparities, National Institutesof Health, Bethesda, MD, USA.\n",
      "FAU - Sharma, Shivam\n",
      "AU  - Sharma S\n",
      "AD  - National Institute on Minority Health and Health Disparities, National Institutesof Health, Bethesda, MD, USA.\n",
      "FAU - Marino-Ramirez, Leonardo\n",
      "AU  - Marino-Ramirez L\n",
      "AD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\n",
      "AD  - National Institute on Minority Health and Health Disparities, National Institutesof Health, Bethesda, MD, USA.\n",
      "FAU - Aguinaga-Romero, Gabriela\n",
      "AU  - Aguinaga-Romero G\n",
      "AD  - Faculty of Medical Sciences, Central University of Ecuador, Quito, Ecuador.\n",
      "FAU - Gonzalez-Andrade, Fabricio\n",
      "AU  - Gonzalez-Andrade F\n",
      "AD  - Faculty of Medical Sciences, Central University of Ecuador, Quito, Ecuador.\n",
      "FAU - Jordan, I King\n",
      "AU  - Jordan IK\n",
      "AD  - School of Biological Sciences, Georgia Institute of Technology, Atlanta, GA, USA.\n",
      "AD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\n",
      "AD  - IHRC-Georgia Tech Applied Bioinformatics Laboratory, Atlanta, GA, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20210820\n",
      "PL  - United States\n",
      "TA  - HGG Adv\n",
      "JT  - HGG advances\n",
      "JID - 101772885\n",
      "PMC - PMC8756502\n",
      "OTO - NOTNLM\n",
      "OT  - Afro-Ecuadorian\n",
      "OT  - Ecuador\n",
      "OT  - Indigenous\n",
      "OT  - Latin America\n",
      "OT  - Mestizo\n",
      "OT  - Montubio\n",
      "OT  - ethnicity\n",
      "OT  - genetic ancestry\n",
      "OT  - genomics\n",
      "OT  - population genetics\n",
      "COIS- The authors declare no competing interests.\n",
      "EDAT- 2022/01/21 06:00\n",
      "MHDA- 2022/01/21 06:01\n",
      "CRDT- 2022/01/20 06:09\n",
      "PHST- 2021/01/19 00:00 [received]\n",
      "PHST- 2021/08/09 00:00 [accepted]\n",
      "PHST- 2022/01/20 06:09 [entrez]\n",
      "PHST- 2022/01/21 06:00 [pubmed]\n",
      "PHST- 2022/01/21 06:01 [medline]\n",
      "AID - 10.1016/j.xhgg.2021.100050 [doi]\n",
      "AID - S2666-2477(21)00031-2 [pii]\n",
      "PST - epublish\n",
      "SO  - HGG Adv. 2021 Aug 20;2(4):100050. doi: 10.1016/j.xhgg.2021.100050. eCollection2021 Oct 14.\n",
      "\n",
      "PMID- 35043846\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20220120\n",
      "LR  - 20220120\n",
      "IS  - 1678-2690 (Electronic)\n",
      "IS  - 0001-3765 (Linking)\n",
      "VI  - 94\n",
      "IP  - 1\n",
      "DP  - 2022\n",
      "TI  - When the tail shakes the snake: phylogenetic affinities and morphology ofAtractus badius (Serpentes: Dipsadidae), reveals some current pitfalls on thesnake's genomic age.\n",
      "PG  - e20191254\n",
      "LID - S0001-37652022000101010 [pii]\n",
      "LID - 10.1590/0001-3765202220191254 [doi]\n",
      "AB  - Atractus badius has a long and controversial nomenclatural history due to bothits antiquity and the brevity of its original description. This species wasdescribed based on two syntypes from Java. Later, a lectotype was designated and the distribution range restricted to the Guiana Shield. Although this species hasbeen repeatedly recorded throughout Amazonia and the Andes, these records havebeen erroneously assigned to A. badius because of a considerable level ofconfusion in the literature. We found 13 additional specimens of this poorlyknown snake, expanding our knowledge on its morphological variability,phylogenetic relationships and distribution. In this paper, we report newlocalities and data on meristic, morphometric, coloration, scalesmicro-ornamentation, osteology and hemipenial morphology to A. badius. We alsodetermine the phylogenetic position of A. badius as nested in a composite Guiana Shield clade. In addition, we discuss recent advances on the systematics ofAtractus and comment on some 'new arrangements' with respect to previouslyrecognized species groups and available morphological evidence. Finally, wehighlight the importance of accurate voucher identification before promotingtaxonomic changes or implementing nomenclatural acts derived from newphylogenetic hypotheses, and notably propose the synonymy of the recentlydescribed A. pyroni with A. roulei.\n",
      "FAU - Passos, Paulo\n",
      "AU  - Passos P\n",
      "AUID- ORCID: http://orcid.org/0000-0002-1775-0970\n",
      "AD  - Universidade Federal do Rio de Janeiro, Museu Nacional, Departamento deVertebrados, Quinta da Boa Vista, s/n, 20940-040 Rio de Janeiro, RJ, Brazil.\n",
      "FAU - Melo-Sampaio, Paulo R\n",
      "AU  - Melo-Sampaio PR\n",
      "AUID- ORCID: http://orcid.org/0000-0003-1858-1643\n",
      "AD  - Universidade Federal do Rio de Janeiro, Museu Nacional, Departamento deVertebrados, Quinta da Boa Vista, s/n, 20940-040 Rio de Janeiro, RJ, Brazil.\n",
      "FAU - Ramos, Luciana O\n",
      "AU  - Ramos LO\n",
      "AUID- ORCID: http://orcid.org/0000-0002-5190-5163\n",
      "AD  - Universidade Federal do Rio de Janeiro, Museu Nacional, Departamento deVertebrados, Quinta da Boa Vista, s/n, 20940-040 Rio de Janeiro, RJ, Brazil.\n",
      "FAU - Grazziotin, Felipe G\n",
      "AU  - Grazziotin FG\n",
      "AUID- ORCID: http://orcid.org/0000-0001-9896-9722\n",
      "AD  - Instituto Butantan, Laboratorio Especial de Colecoes Zoologicas, Avenida VitalBrazil, 1500, 05508-040 Sao Paulo, SP, Brazil.\n",
      "FAU - Fouquet, Antoine\n",
      "AU  - Fouquet A\n",
      "AUID- ORCID: http://orcid.org/0000-0003-4060-0281\n",
      "AD  - Laboratoire Evolution & Diversite Biologique, Universite Toulouse III PaulSabatier, Batiment 4R1 31062 cedex 9, 118 Route de Narbonne, 31077 Toulouse,France.\n",
      "AD  - Laboratoire Ecologie, Evolution, Interactions des Systemes Amazoniens (LEEISA),USR 3456 (CNRS, IFREMER, Universite de Guyane), Centre de Recherche de Montabo,Cayenne, French Guiana.\n",
      "FAU - Torres-Carvajal, Omar\n",
      "AU  - Torres-Carvajal O\n",
      "AUID- ORCID: http://orcid.org/0000-0003-0041-9250\n",
      "AD  - Museo de Zoologia, Escuela de Biologia, Pontificia Universidad Catolica delEcuador, Avenida 12 de Octubre y Roca, 1076, Apartado 17-01-2184, Quito, Ecuador.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20220117\n",
      "PL  - Brazil\n",
      "TA  - An Acad Bras Cienc\n",
      "JT  - Anais da Academia Brasileira de Ciencias\n",
      "JID - 7503280\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Brazil\n",
      "MH  - *Genomics\n",
      "MH  - Osteology\n",
      "MH  - Phylogeny\n",
      "MH  - *Snakes/genetics\n",
      "EDAT- 2022/01/20 06:00\n",
      "MHDA- 2022/01/21 06:00\n",
      "CRDT- 2022/01/19 08:52\n",
      "PHST- 2019/10/10 00:00 [received]\n",
      "PHST- 2020/06/21 00:00 [accepted]\n",
      "PHST- 2022/01/19 08:52 [entrez]\n",
      "PHST- 2022/01/20 06:00 [pubmed]\n",
      "PHST- 2022/01/21 06:00 [medline]\n",
      "AID - S0001-37652022000101010 [pii]\n",
      "AID - 10.1590/0001-3765202220191254 [doi]\n",
      "PST - epublish\n",
      "SO  - An Acad Bras Cienc. 2022 Jan 17;94(1):e20191254. doi:10.1590/0001-3765202220191254. eCollection 2022.\n",
      "\n",
      "PMID- 35018222\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "LR  - 20220112\n",
      "IS  - 2052-2975 (Print)\n",
      "IS  - 2052-2975 (Linking)\n",
      "DP  - 2022 Jan 7\n",
      "TI  - First detection of SARS-CoV-2 variant B.1.1.529 (Omicron) in Ecuador.\n",
      "PG  - 100951\n",
      "LID - 10.1016/j.nmni.2022.100951 [doi]\n",
      "AB  - The National Institute of Research and Public Health reported the first localrecord of the Omicron variant detected in Ecuador. A fully vaccinated subjectreturned from South Africa with a negative RT-PCR. We present the cumulativefrequency of the variants in Ecuador and the Phylogeny of this new Omicron.\n",
      "CI  - (c) 2022 Published by Elsevier Ltd.\n",
      "FAU - Carrazco-Montalvo, Andres\n",
      "AU  - Carrazco-Montalvo A\n",
      "AD  - Direccion de Investigacion, Desarrollo e Innovacion. Instituto Nacional deInvestigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\n",
      "FAU - Armendariz-Castillo, Isaac\n",
      "AU  - Armendariz-Castillo I\n",
      "AD  - Coordinacion Zonal 9. Instituto Nacional de investigacion en Salud Publica\"Leopoldo Izquieta Perez\", Quito, Ecuador.\n",
      "FAU - Tello, Clara Lucia\n",
      "AU  - Tello CL\n",
      "AD  - Direccion de Investigacion, Desarrollo e Innovacion. Instituto Nacional deInvestigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\n",
      "FAU - Morales, Diego\n",
      "AU  - Morales D\n",
      "AD  - Centro de Referencia Nacional de Influenza y otros Virus Respiratorios. InstitutoNacional de Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\n",
      "FAU - Armas-Gonzalez, Ruben\n",
      "AU  - Armas-Gonzalez R\n",
      "AD  - Centro de Referencia Nacional de Influenza y otros Virus Respiratorios. InstitutoNacional de Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\n",
      "FAU - Guizado-Herrera, David\n",
      "AU  - Guizado-Herrera D\n",
      "AD  - Direccion de Investigacion, Desarrollo e Innovacion. Instituto Nacional deInvestigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\n",
      "FAU - Leon-Sosa, Ariana\n",
      "AU  - Leon-Sosa A\n",
      "AD  - Direccion de Investigacion, Desarrollo e Innovacion. Instituto Nacional deInvestigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\n",
      "FAU - Ramos-Sarmiento, Daniel\n",
      "AU  - Ramos-Sarmiento D\n",
      "AD  - Centro de Referencia Nacional de Influenza y otros Virus Respiratorios. InstitutoNacional de Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\n",
      "FAU - Fuertes, Byron\n",
      "AU  - Fuertes B\n",
      "AD  - Centro de Referencia Nacional de Influenza y otros Virus Respiratorios. InstitutoNacional de Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\n",
      "FAU - Patino, Leandro\n",
      "AU  - Patino L\n",
      "AD  - Direccion de Investigacion, Desarrollo e Innovacion. Instituto Nacional deInvestigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\n",
      "CN  - USFQ-Consortium\n",
      "LA  - eng\n",
      "PT  - Case Reports\n",
      "DEP - 20220107\n",
      "PL  - England\n",
      "TA  - New Microbes New Infect\n",
      "JT  - New microbes and new infections\n",
      "JID - 101624750\n",
      "PMC - PMC8739794\n",
      "OTO - NOTNLM\n",
      "OT  - COVID-19\n",
      "OT  - Genomics\n",
      "OT  - Omicron\n",
      "OT  - SARS-CoV-2\n",
      "OT  - variant\n",
      "IR  - Cardenas P\n",
      "FIR - Cardenas, Paul\n",
      "IR  - Marquez S\n",
      "FIR - Marquez, Sully\n",
      "IR  - Prado-Vivar B\n",
      "FIR - Prado-Vivar, Belen\n",
      "IR  - Guadalupe JJ\n",
      "FIR - Guadalupe, Juan Jose\n",
      "IR  - Gutierrez B\n",
      "FIR - Gutierrez, Bernardo\n",
      "IR  - Wong MB\n",
      "FIR - Wong, Monica Becerra\n",
      "IR  - Grunauer M\n",
      "FIR - Grunauer, Michelle\n",
      "IR  - Trueba G\n",
      "FIR - Trueba, Gabriel\n",
      "IR  - Rojas-Silva P\n",
      "FIR - Rojas-Silva, Patricio\n",
      "IR  - Barragan V\n",
      "FIR - Barragan, Veronica\n",
      "EDAT- 2022/01/13 06:00\n",
      "MHDA- 2022/01/13 06:00\n",
      "CRDT- 2022/01/12 06:32\n",
      "PHST- 2021/12/29 00:00 [received]\n",
      "PHST- 2022/01/04 00:00 [accepted]\n",
      "PHST- 2022/01/12 06:32 [entrez]\n",
      "PHST- 2022/01/13 06:00 [pubmed]\n",
      "PHST- 2022/01/13 06:00 [medline]\n",
      "AID - 10.1016/j.nmni.2022.100951 [doi]\n",
      "AID - S2052-2975(22)00003-8 [pii]\n",
      "PST - aheadofprint\n",
      "SO  - New Microbes New Infect. 2022 Jan 7:100951. doi: 10.1016/j.nmni.2022.100951.\n",
      "\n",
      "PMID- 34962985\n",
      "OWN - NLM\n",
      "STAT- In-Process\n",
      "LR  - 20220216\n",
      "IS  - 1759-6653 (Electronic)\n",
      "IS  - 1759-6653 (Linking)\n",
      "VI  - 14\n",
      "IP  - 1\n",
      "DP  - 2022 Jan 4\n",
      "TI  - De Novo Genome Assembly and Annotation of an Andean Caddisfly, Atopsychedavidsoni Sykora, 1991, a Model for Genome Research of High-ElevationAdaptations.\n",
      "LID - evab286 [pii]\n",
      "LID - 10.1093/gbe/evab286 [doi]\n",
      "AB  - We sequence, assemble, and annotate the genome of Atopsyche davidsoni Sykora,1991, the first whole-genome assembly for the caddisfly family Hydrobiosidae.This free-living and predatory caddisfly inhabits streams in the high-elevationAndes and is separated by more than 200 Myr of evolutionary history from the mostclosely related caddisfly species with genome assemblies available. Wedemonstrate the promise of PacBio HiFi reads by assembling the most contiguouscaddisfly genome assembly to date with a contig N50 of 14 Mb, which is more than 6x more contiguous than the current most contiguous assembly for a caddisfly(Hydropsyche tenuis). We recover 98.8% of insect BUSCO genes indicating a highlevel of gene completeness. We also provide a genome annotation of 12,232annotated proteins. This new genome assembly provides an important new resourcefor studying genomic adaptation of aquatic insects to harsh, high-altitudeenvironments.\n",
      "CI  - (c) The Author(s) 2021. Published by Oxford University Press on behalf of theSociety for Molecular Biology and Evolution.\n",
      "FAU - Rios-Touma, Blanca\n",
      "AU  - Rios-Touma B\n",
      "AD  - Facultad de Ingenierias y Ciencias Aplicadas, Ingenieria Ambiental, Grupo deInvestigacion en Biodiversidad, Medio Ambiente y Salud (BIOMAS), Universidad delas Americas, Quito, Ecuador.\n",
      "FAU - Holzenthal, Ralph W\n",
      "AU  - Holzenthal RW\n",
      "AD  - Department of Entomology, University of Minnesota, St. Paul, Minnesota, USA.\n",
      "FAU - Razuri-Gonzales, Ernesto\n",
      "AU  - Razuri-Gonzales E\n",
      "AD  - Department of Entomology, University of Minnesota, St. Paul, Minnesota, USA.\n",
      "AD  - Department of Terrestrial Zoology, Entomology III, Senckenberg Research Instituteand Natural History Museum Frankfurt, Germany.\n",
      "FAU - Heckenhauer, Jacqueline\n",
      "AU  - Heckenhauer J\n",
      "AD  - Department of Terrestrial Zoology, Entomology III, Senckenberg Research Instituteand Natural History Museum Frankfurt, Germany.\n",
      "AD  - LOEWE Centre for Translational Biodiversity Genomics (LOEWE-TBG), Frankfurt,Germany.\n",
      "FAU - Pauls, Steffen U\n",
      "AU  - Pauls SU\n",
      "AD  - Department of Terrestrial Zoology, Entomology III, Senckenberg Research Instituteand Natural History Museum Frankfurt, Germany.\n",
      "AD  - LOEWE Centre for Translational Biodiversity Genomics (LOEWE-TBG), Frankfurt,Germany.\n",
      "AD  - Institute of Insect Biotechnology, Justus-Liebig University, Giessen, Germany.\n",
      "FAU - Storer, Caroline G\n",
      "AU  - Storer CG\n",
      "AD  - McGuire Center for Lepidoptera and Biodiversity, Florida Museum of NaturalHistory, University of Florida, Gainesville, Florida, USA.\n",
      "FAU - Frandsen, Paul B\n",
      "AU  - Frandsen PB\n",
      "AUID- ORCID: 0000-0002-4801-7579\n",
      "AD  - LOEWE Centre for Translational Biodiversity Genomics (LOEWE-TBG), Frankfurt,Germany.\n",
      "AD  - Department of Plant and Wildlife Sciences, Brigham Young University, Provo, Utah,USA.\n",
      "LA  - eng\n",
      "SI  - figshare/10.6084/m9.figshare.16836871.v1\n",
      "SI  - figshare/10.6084/m9.figshare.16837963.v1\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PL  - England\n",
      "TA  - Genome Biol Evol\n",
      "JT  - Genome biology and evolution\n",
      "JID - 101509707\n",
      "SB  - IM\n",
      "PMC - PMC8767365\n",
      "OTO - NOTNLM\n",
      "OT  - *Insecta\n",
      "OT  - *PacBio HiFi\n",
      "OT  - *Trichoptera\n",
      "OT  - *caddisfly genomics\n",
      "OT  - *extreme environments\n",
      "OT  - *high altitude\n",
      "EDAT- 2021/12/29 06:00\n",
      "MHDA- 2021/12/29 06:00\n",
      "CRDT- 2021/12/28 17:18\n",
      "PHST- 2021/12/20 00:00 [accepted]\n",
      "PHST- 2021/12/29 06:00 [pubmed]\n",
      "PHST- 2021/12/29 06:00 [medline]\n",
      "PHST- 2021/12/28 17:18 [entrez]\n",
      "AID - 6486422 [pii]\n",
      "AID - 10.1093/gbe/evab286 [doi]\n",
      "PST - ppublish\n",
      "SO  - Genome Biol Evol. 2022 Jan 4;14(1). pii: 6486422. doi: 10.1093/gbe/evab286.\n",
      "\n",
      "PMID- 34960145\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20211231\n",
      "IS  - 2076-393X (Print)\n",
      "IS  - 2076-393X (Linking)\n",
      "VI  - 9\n",
      "IP  - 12\n",
      "DP  - 2021 Nov 26\n",
      "TI  - Major Complex Trait for Early De Novo Programming 'CoV-MAC-TED' Detected in HumanNasal Epithelial Cells Infected by Two SARS-CoV-2 Variants Is Promising to Helpin Designing Therapeutic Strategies.\n",
      "LID - 1399 [pii]\n",
      "LID - 10.3390/vaccines9121399 [doi]\n",
      "AB  - BACKGROUND: Early metabolic reorganization was only recently recognized as anessentially integrated part of immunology. In this context, unbalanced ROS/RNSlevels connected to increased aerobic fermentation, which is linked toalpha-tubulin-based cell restructuring and control of cell cycle progression,were identified as a major complex trait for early de novo programming('CoV-MAC-TED') during SARS-CoV-2 infection. This trait was highlighted as acritical target for developing early anti-viral/anti-SARS-CoV-2 strategies. Toobtain this result, analyses had been performed on transcriptome data fromdiverse experimental cell systems. A call was released for wide data collectionof the defined set of genes for transcriptome analyses, named 'ReprogVirus',which should be based on strictly standardized protocols and data entry fromdiverse virus types and variants into the 'ReprogVirus Platform'. This platformis currently under development. However, so far, an in vitro cell system fromprimary target cells for virus attacks that could ideally serve for standardizingthe data collection of early SARS-CoV-2 infection responses has not been defined.RESULTS: Here, we demonstrate transcriptome-level profiles of the most critical'ReprogVirus' gene sets for identifying 'CoV-MAC-TED' in cultured human nasalepithelial cells infected by two SARS-CoV-2 variants differing in diseaseseverity. Our results (a) validate 'Cov-MAC-TED' as a crucial trait for earlySARS-CoV-2 reprogramming for the tested virus variants and (b) demonstrate itsrelevance in cultured human nasal epithelial cells. CONCLUSION: In vitro-culturedhuman nasal epithelial cells proved to be appropriate for standardizedtranscriptome data collection in the 'ReprogVirus Platform'. Thus, this cellsystem is highly promising to advance integrative data analyses with the help of artificial intelligence methodologies for designing anti-SARS-CoV-2 strategies.\n",
      "FAU - Costa, Jose Helio\n",
      "AU  - Costa JH\n",
      "AUID- ORCID: 0000-0001-7914-0150\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza 60451-970, Brazil.\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,7050-704 Montemor-o-Novo, Portugal.\n",
      "FAU - Aziz, Shahid\n",
      "AU  - Aziz S\n",
      "AUID- ORCID: 0000-0002-4657-1066\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza 60451-970, Brazil.\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,7050-704 Montemor-o-Novo, Portugal.\n",
      "FAU - Noceda, Carlos\n",
      "AU  - Noceda C\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,7050-704 Montemor-o-Novo, Portugal.\n",
      "AD  - Plant Molecular and Cellular Biotechnology/Industrial Biotechnology andBioproducts, Department of Life and Agricultural Sciences, Universidad de lasFuerzas Armadas-ESPE, Quito 171103, Ecuador.\n",
      "FAU - Arnholdt-Schmitt, Birgit\n",
      "AU  - Arnholdt-Schmitt B\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza 60451-970, Brazil.\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,7050-704 Montemor-o-Novo, Portugal.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20211126\n",
      "PL  - Switzerland\n",
      "TA  - Vaccines (Basel)\n",
      "JT  - Vaccines\n",
      "JID - 101629355\n",
      "PMC - PMC8708361\n",
      "OTO - NOTNLM\n",
      "OT  - ADH5\n",
      "OT  - E2F1\n",
      "OT  - SARS-CoV-2 RdRp\n",
      "OT  - SARS-CoV-2 helicase\n",
      "OT  - SARS-CoV-2 Delta382\n",
      "OT  - anti-viral diagnosis and strategies\n",
      "OT  - cell cycle\n",
      "OT  - immunology paradigm shift\n",
      "OT  - melatonin\n",
      "OT  - microbiota\n",
      "OT  - repurposing drugs\n",
      "EDAT- 2021/12/29 06:00\n",
      "MHDA- 2021/12/29 06:01\n",
      "CRDT- 2021/12/28 01:05\n",
      "PHST- 2021/09/15 00:00 [received]\n",
      "PHST- 2021/11/15 00:00 [revised]\n",
      "PHST- 2021/11/22 00:00 [accepted]\n",
      "PHST- 2021/12/28 01:05 [entrez]\n",
      "PHST- 2021/12/29 06:00 [pubmed]\n",
      "PHST- 2021/12/29 06:01 [medline]\n",
      "AID - vaccines9121399 [pii]\n",
      "AID - 10.3390/vaccines9121399 [doi]\n",
      "PST - epublish\n",
      "SO  - Vaccines (Basel). 2021 Nov 26;9(12). pii: vaccines9121399. doi:10.3390/vaccines9121399.\n",
      "\n",
      "PMID- 34938321\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20211224\n",
      "IS  - 1664-8021 (Print)\n",
      "IS  - 1664-8021 (Linking)\n",
      "VI  - 12\n",
      "DP  - 2021\n",
      "TI  - S-RNase Alleles Associated With Self-Compatibility in the Tomato Clade:Structure, Origins, and Expression Plasticity.\n",
      "PG  - 780793\n",
      "LID - 10.3389/fgene.2021.780793 [doi]\n",
      "AB  - The self-incompatibility (SI) system in the Solanaceae is comprised of cytotoxic pistil S-RNases which are countered by S-locus F-box (SLF) resistance factorsfound in pollen. Under this barrier-resistance architecture, mating systemtransitions from SI to self-compatibility (SC) typically result fromloss-of-function mutations in genes encoding pistil SI factors such as S-RNase.However, the nature of these mutations is often not well characterized. Here weuse a combination of S-RNase sequence analysis, transcript profiling, proteinexpression and reproductive phenotyping to better understand different mechanismsthat result in loss of S-RNase function. Our analysis focuses on 12 S-RNasealleles identified in SC species and populations across the tomato clade. In six cases, the reason for gene dysfunction due to mutations is evident. The six otheralleles potentially encode functional S-RNase proteins but are typicallytranscriptionally silenced. We identified three S-RNase alleles which aretranscriptionally silenced under some conditions but actively expressed inothers. In one case, expression of the S-RNase is associated with SI. In another case, S-RNase expression does not lead to SI, but instead confers a reproductive barrier against pollen tubes from other tomato species. In the third case,expression of S-RNase does not affect self, interspecific or inter-populationreproductive barriers. Our results indicate that S-RNase expression is moredynamic than previously thought, and that changes in expression can impactdifferent reproductive barriers within or between natural populations.\n",
      "CI  - Copyright (c) 2021 Broz, Miller, Baek, Tovar-Mendez, Acosta-Quezada,Riofrio-Cuenca, Rusch and Bedinger.\n",
      "FAU - Broz, Amanda K\n",
      "AU  - Broz AK\n",
      "AD  - Department of Biology, Colorado State University, Fort Collins, CO, UnitedStates.\n",
      "FAU - Miller, Christopher M\n",
      "AU  - Miller CM\n",
      "AD  - Department of Biology, Colorado State University, Fort Collins, CO, UnitedStates.\n",
      "FAU - Baek, You Soon\n",
      "AU  - Baek YS\n",
      "AD  - Department of Biology, Colorado State University, Fort Collins, CO, UnitedStates.\n",
      "FAU - Tovar-Mendez, Alejandro\n",
      "AU  - Tovar-Mendez A\n",
      "AD  - Department of Biochemistry, University of Missouri, Columbia, MO, United States.\n",
      "FAU - Acosta-Quezada, Pablo Geovanny\n",
      "AU  - Acosta-Quezada PG\n",
      "AD  - Departamento de Ciencias Biologicas y Agropecuarias, Universidad TecnicaParticular de Loja, Loja, Ecuador.\n",
      "FAU - Riofrio-Cuenca, Tanya Elizabet\n",
      "AU  - Riofrio-Cuenca TE\n",
      "AD  - Departamento de Ciencias Biologicas y Agropecuarias, Universidad TecnicaParticular de Loja, Loja, Ecuador.\n",
      "FAU - Rusch, Douglas B\n",
      "AU  - Rusch DB\n",
      "AD  - Center for Genomics and Bioinformatics, Indiana University, Bloomington, IN,United States.\n",
      "FAU - Bedinger, Patricia A\n",
      "AU  - Bedinger PA\n",
      "AD  - Department of Biology, Colorado State University, Fort Collins, CO, UnitedStates.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20211206\n",
      "PL  - Switzerland\n",
      "TA  - Front Genet\n",
      "JT  - Frontiers in genetics\n",
      "JID - 101560621\n",
      "PMC - PMC8685505\n",
      "OTO - NOTNLM\n",
      "OT  - S-RNase\n",
      "OT  - mating system transitions\n",
      "OT  - reproductive barriers\n",
      "OT  - self-compatibility (compatible)\n",
      "OT  - self-incompatibility (incompatible)\n",
      "COIS- The authors declare that the research was conducted in the absence of anycommercial or financial relationships that could be construed as a potentialconflict of interest.\n",
      "EDAT- 2021/12/24 06:00\n",
      "MHDA- 2021/12/24 06:01\n",
      "CRDT- 2021/12/23 05:48\n",
      "PHST- 2021/09/21 00:00 [received]\n",
      "PHST- 2021/11/09 00:00 [accepted]\n",
      "PHST- 2021/12/23 05:48 [entrez]\n",
      "PHST- 2021/12/24 06:00 [pubmed]\n",
      "PHST- 2021/12/24 06:01 [medline]\n",
      "AID - 10.3389/fgene.2021.780793 [doi]\n",
      "AID - 780793 [pii]\n",
      "PST - epublish\n",
      "SO  - Front Genet. 2021 Dec 6;12:780793. doi: 10.3389/fgene.2021.780793. eCollection2021.\n",
      "\n",
      "PMID- 34878103\n",
      "OWN - NLM\n",
      "STAT- In-Data-Review\n",
      "LR  - 20220209\n",
      "IS  - 2160-1836 (Electronic)\n",
      "IS  - 2160-1836 (Linking)\n",
      "VI  - 12\n",
      "IP  - 2\n",
      "DP  - 2022 Feb 4\n",
      "TI  - The genome sequence of the avian vampire fly (Philornis downsi), an invasive nestparasite of Darwin's finches in Galapagos.\n",
      "LID - jkab414 [pii]\n",
      "LID - 10.1093/g3journal/jkab414 [doi]\n",
      "AB  - The invasive avian vampire fly (Philornis downsi, Diptera: Muscidae) isconsidered one of the greatest threats to the endemic avifauna of the GalapagosIslands. The fly larvae parasitize nearly every passerine species, includingDarwin's finches. Most P. downsi research to date has focused on the effects ofthe fly on avian host fitness and mitigation methods. A lag in research relatedto the genetics of this invasion demonstrates, in part, the need to developfull-scale genomic resources with which to address further questions within this system. In this study, an adult female P. downsi was sequenced to generate ahigh-quality genome assembly. We examined various features of the genome (e.g.,coding regions and noncoding transposable elements) and carried out comparativegenomics analysis against other dipteran genomes. We identified lists of genefamilies that are significantly expanding or contracting in P. downsi that arerelated to insecticide resistance, detoxification, and counter defense againsthost immune responses. The P. downsi genome assembly provides an importantresource for studying the molecular basis of successful invasion in the Galapagosand the dynamics of its population across multiple islands. The findings ofsignificantly changing gene families associated with insecticide resistance andimmune responses highlight the need for further investigations into the role ofdifferent gene families in aiding the fly's successful invasion. Furthermore,this genomic resource provides a necessary tool to better inform future research studies and mitigation strategies aimed at minimizing the fly's impact onGalapagos birds.\n",
      "CI  - (c) The Author(s) 2021. Published by Oxford University Press on behalf ofGenetics Society of America.\n",
      "FAU - Romine, Melia G\n",
      "AU  - Romine MG\n",
      "AD  - School of Biomedical Sciences, Kent State University, Kent, OH 44240, USA.\n",
      "FAU - Knutie, Sarah A\n",
      "AU  - Knutie SA\n",
      "AD  - Department of Ecology and Evolutionary Biology, University of Connecticut,Storrs, CT 06269, USA.\n",
      "AD  - Institute for Systems Genomics, University of Connecticut, Storrs, CT 06269, USA.\n",
      "FAU - Crow, Carly M\n",
      "AU  - Crow CM\n",
      "AD  - Department of Biological Sciences, Northern Illinois University, DeKalb, IL60115, USA.\n",
      "FAU - Vaziri, Grace J\n",
      "AU  - Vaziri GJ\n",
      "AD  - Department of Ecology and Evolutionary Biology, University of Connecticut,Storrs, CT 06269, USA.\n",
      "FAU - Chaves, Jaime A\n",
      "AU  - Chaves JA\n",
      "AD  - Department of Biology, San Francisco State University, San Francisco, CA 94132,USA.\n",
      "AD  - Colegio de Ciencias Biologicas y Ambientales, Universidad San Francisco de Quito,Quito 170901, Ecuador.\n",
      "FAU - Koop, Jennifer A H\n",
      "AU  - Koop JAH\n",
      "AD  - Department of Biological Sciences, Northern Illinois University, DeKalb, IL60115, USA.\n",
      "FAU - Lamichhaney, Sangeet\n",
      "AU  - Lamichhaney S\n",
      "AUID- ORCID: 0000-0003-4826-0349\n",
      "AD  - School of Biomedical Sciences, Kent State University, Kent, OH 44240, USA.\n",
      "AD  - Department of Biological Sciences, Kent State University, Kent, OH 44240, USA.\n",
      "LA  - eng\n",
      "GR  - Department of Biological Sciences, Kent State University\n",
      "GR  - Northern Illinois University Division of Research and Innovation Partnerships\n",
      "GR  - University Library\n",
      "GR  - University of Connecticut\n",
      "PT  - Journal Article\n",
      "PL  - England\n",
      "TA  - G3 (Bethesda)\n",
      "JT  - G3 (Bethesda, Md.)\n",
      "JID - 101566598\n",
      "SB  - IM\n",
      "OTO - NOTNLM\n",
      "OT  - Darwin's finches\n",
      "OT  - Galapagos\n",
      "OT  - avian vampire fly genome\n",
      "OT  - insecticide resistance\n",
      "OT  - parasitic invasion\n",
      "EDAT- 2021/12/09 06:00\n",
      "MHDA- 2021/12/09 06:00\n",
      "CRDT- 2021/12/08 09:20\n",
      "PHST- 2021/10/06 00:00 [received]\n",
      "PHST- 2021/11/15 00:00 [accepted]\n",
      "PHST- 2021/12/09 06:00 [pubmed]\n",
      "PHST- 2021/12/09 06:00 [medline]\n",
      "PHST- 2021/12/08 09:20 [entrez]\n",
      "AID - 6456303 [pii]\n",
      "AID - 10.1093/g3journal/jkab414 [doi]\n",
      "PST - ppublish\n",
      "SO  - G3 (Bethesda). 2022 Feb 4;12(2). pii: 6456303. doi: 10.1093/g3journal/jkab414.\n",
      "\n",
      "PMID- 34810701\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20211124\n",
      "LR  - 20211124\n",
      "IS  - 1175-5334 (Electronic)\n",
      "IS  - 1175-5326 (Linking)\n",
      "VI  - 5068\n",
      "IP  - 3\n",
      "DP  - 2021 Nov 17\n",
      "TI  - Gamasiphis species (Acari: Mesostigmata: Ologamasidae) from Ecuador, withdescription of a new species and new records.\n",
      "PG  - 410-418\n",
      "LID - 10.11646/zootaxa.5068.3.5 [doi]\n",
      "AB  - Gamasiphis Berlese is the most diverse genus of Ologamasidae, with 74 describedspecies, seven of which were described from Ecuador. The main objective of thispaper is to report the Gamasiphis species found in surveys recently conducted in Ecuador, including a new species here described, Gamasiphis vikkiae Melo-MolinaSantos n. sp. Complementary descriptions of two previously described species,also collected in this study, are presented.\n",
      "FAU - Melo-Molina, Elsa L\n",
      "AU  - Melo-Molina EL\n",
      "AD  - Escola Superior de Agricultura Luiz de Queiroz (ESALQ), Universidade de So Paulo (USP), 13418-900 Piracicaba, So Paulo, Brazil. meloelsa@gmail.com.\n",
      "FAU - Santos, Jandir C\n",
      "AU  - Santos JC\n",
      "AD  - Centre for Biodiversity Genomics, University of Guelph, Guelph, Ontario.user@example.com.\n",
      "FAU - DE Moraes, Gilberto J\n",
      "AU  - DE Moraes GJ\n",
      "AD  - Escola Superior de Agricultura Luiz de Queiroz (ESALQ), Universidade de So Paulo (USP), 13418-900 Piracicaba, So Paulo, Brazil. CNPq researcher. . moraesg@usp.br.\n",
      "FAU - Castilho, Raphael C\n",
      "AU  - Castilho RC\n",
      "AD  - Escola Superior de Agricultura Luiz de Queiroz (ESALQ), Universidade de So Paulo (USP), 13418-900 Piracicaba, So Paulo, Brazil. raphael.castilho@usp.br.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20211117\n",
      "PL  - New Zealand\n",
      "TA  - Zootaxa\n",
      "JT  - Zootaxa\n",
      "JID - 101179386\n",
      "SB  - IM\n",
      "MH  - *Acari\n",
      "MH  - *Amphipoda\n",
      "MH  - Animals\n",
      "MH  - Ecuador\n",
      "MH  - *Gastropoda\n",
      "MH  - *Mites\n",
      "EDAT- 2021/11/24 06:00\n",
      "MHDA- 2021/11/25 06:00\n",
      "CRDT- 2021/11/23 05:56\n",
      "PHST- 2021/11/15 00:00 [received]\n",
      "PHST- 2021/11/23 05:56 [entrez]\n",
      "PHST- 2021/11/24 06:00 [pubmed]\n",
      "PHST- 2021/11/25 06:00 [medline]\n",
      "AID - 10.11646/zootaxa.5068.3.5 [doi]\n",
      "PST - epublish\n",
      "SO  - Zootaxa. 2021 Nov 17;5068(3):410-418. doi: 10.11646/zootaxa.5068.3.5.\n",
      "\n",
      "PMID- 34741053\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20220126\n",
      "LR  - 20220126\n",
      "IS  - 2045-2322 (Electronic)\n",
      "IS  - 2045-2322 (Linking)\n",
      "VI  - 11\n",
      "IP  - 1\n",
      "DP  - 2021 Nov 5\n",
      "TI  - Assessing the origin, genetic structure and demographic history of the commonpheasant (Phasianus colchicus) in the introduced European range.\n",
      "PG  - 21721\n",
      "LID - 10.1038/s41598-021-00567-1 [doi]\n",
      "AB  - The common pheasant, a game species widely introduced throughout the world, canbe considered as an ideal model to study the effects of introduction events onlocal adaptations, biogeographic patterns, and genetic divergence processes. Weaimed to assess the origin, spatial patterns of genetic variation, anddemographic history of the introduced populations in the contact zone of Central and Southeast Europe, using mitochondrial DNA control region sequences andmicrosatellite loci. Both types of molecular markers indicated relatively low to moderate levels of genetic variation. The mtDNA analyses revealed that commonpheasants across the study area are divided into two distinct clades: B(mongolicus group) and F (colchicus group). Analyses of the microsatellite dataconsistently suggested a differentiation between Hungary and Serbia, with thepheasant population in Hungary being much more genetically homogeneous, whilethat of Serbia has much more genetic mixture and admixture. This crypticdifferentiation was not detected using a non-spatial Bayesian clustering model.The analyses also provided strong evidence for a recent population expansion.This fundamental information is essential for adequate and effective conservationmanagement of populations of a game species of great economic and ecologicalimportance in the studied geographical region.\n",
      "CI  - (c) 2021. The Author(s).\n",
      "FAU - Ashrafzadeh, Mohammad Reza\n",
      "AU  - Ashrafzadeh MR\n",
      "AD  - Department of Fisheries and Environmental Sciences, Faculty of Natural Resources and Earth Sciences, Shahrekord University, 8818634141, Shahrekord, Iran.\n",
      "FAU - Khosravi, Rasoul\n",
      "AU  - Khosravi R\n",
      "AD  - Department of Natural Resources and Environmental Engineering, School ofAgriculture, Shiraz University, 71441-13131, Shiraz, Iran.\n",
      "FAU - Fernandes, Carlos\n",
      "AU  - Fernandes C\n",
      "AD  - cE3c - Centre for Ecology, Evolution and Environmental Changes, Departamento deBiologia Animal, Fac-Uldade de Ciencias, Universidade de Lisboa, 1749-016,Lisboa, Portugal.\n",
      "AD  - Faculdade de Psicologia, Universidade de Lisboa, Alameda da Universidade,1649-013, Lisboa, Portugal.\n",
      "FAU - Aguayo, Cecilia\n",
      "AU  - Aguayo C\n",
      "AD  - Agrarian University of Ecuador, Guayaquil, 090104, Ecuador.\n",
      "FAU - Bagi, Zoltan\n",
      "AU  - Bagi Z\n",
      "AD  - Centre for Agricultural Genomics and Biotechnology, University of Debrecen,Debrecen, 4032, Hungary.\n",
      "FAU - Lavadinovic, Vukan M\n",
      "AU  - Lavadinovic VM\n",
      "AD  - Faculty of Forestry, University of Belgrade, 11030, Belgrade, Serbia.\n",
      "FAU - Szendrei, Laszlo\n",
      "AU  - Szendrei L\n",
      "AD  - Department of Nature Conservation Zoology and Game Management, University ofDebrecen, Debrecen, 4032, Hungary.\n",
      "FAU - Beukovic, Dejan\n",
      "AU  - Beukovic D\n",
      "AD  - Faculty of Agriculture, University of Novi Sad, 21000, Novi Sad, Serbia.\n",
      "FAU - Mihalik, Bendeguz\n",
      "AU  - Mihalik B\n",
      "AD  - Centre for Agricultural Genomics and Biotechnology, University of Debrecen,Debrecen, 4032, Hungary.\n",
      "AD  - Institute of Genetics and Biotechnology, Godollo Campus, Hungarian University of Agriculture and Life Sciences, Godollo, 2100, Hungary.\n",
      "FAU - Kusza, Szilvia\n",
      "AU  - Kusza S\n",
      "AD  - Centre for Agricultural Genomics and Biotechnology, University of Debrecen,Debrecen, 4032, Hungary. kusza@agr.unideb.hu.\n",
      "LA  - eng\n",
      "PT  - Comparative Study\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20211105\n",
      "PL  - England\n",
      "TA  - Sci Rep\n",
      "JT  - Scientific reports\n",
      "JID - 101563288\n",
      "SB  - IM\n",
      "MH  - Animal Distribution\n",
      "MH  - Animals\n",
      "MH  - Europe\n",
      "MH  - Founder Effect\n",
      "MH  - Galliformes/*genetics\n",
      "MH  - *Genetic Variation\n",
      "MH  - *Introduced Species\n",
      "MH  - Phylogeography\n",
      "PMC - PMC8571287\n",
      "EDAT- 2021/11/07 06:00\n",
      "MHDA- 2022/01/27 06:00\n",
      "CRDT- 2021/11/06 05:47\n",
      "PHST- 2021/06/01 00:00 [received]\n",
      "PHST- 2021/10/06 00:00 [accepted]\n",
      "PHST- 2021/11/06 05:47 [entrez]\n",
      "PHST- 2021/11/07 06:00 [pubmed]\n",
      "PHST- 2022/01/27 06:00 [medline]\n",
      "AID - 10.1038/s41598-021-00567-1 [doi]\n",
      "AID - 10.1038/s41598-021-00567-1 [pii]\n",
      "PST - epublish\n",
      "SO  - Sci Rep. 2021 Nov 5;11(1):21721. doi: 10.1038/s41598-021-00567-1.\n",
      "\n",
      "PMID- 34731497\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20211220\n",
      "LR  - 20211220\n",
      "IS  - 1530-6860 (Electronic)\n",
      "IS  - 0892-6638 (Linking)\n",
      "VI  - 35\n",
      "IP  - 12\n",
      "DP  - 2021 Dec\n",
      "TI  - Characterization and profiling analysis of bovine oviduct and uterineextracellular vesicles and their miRNA cargo through the estrous cycle.\n",
      "PG  - e22000\n",
      "LID - 10.1096/fj.202101023R [doi]\n",
      "AB  - Extracellular vesicles (EVs) found in various biological fluids and particularly in reproductive fluids, have gained considerable attention for their possiblerole in cell- to- cell communication. Among, the different bioactive moleculescargos of EVs, MicroRNAs (miRNAs) are emerging as promising diagnostic biomarkerswith high clinical potential. Aiming to understand the roles of EVs in bovinereproductive tract, we intended to characterize and profile the EVs of oviductand uterine fluids (OF-EVs, UF-EVs) and their miRNA across the estrous cycle.Nanoparticle tracking analysis and transmission electron microscopy confirmed theexistence of small EV population in OF and UF at all stages, (size between 30 and200 nm; concentration: 3.4 x 10(10) EVs/ml and 6.0 x 10(10) EVs/ml for OF and UF,respectively, regardless of stage). The identification of EV markers (CD9, HSP70,and ALIX proteins) was confirmed by western blot. The miRNA analysis revealed theabundance of 310 and 351 miRNAs in OF-EVs and UF-EVs, respectively. Nine miRNAswere differentially abundant in OF-EVs between stages of the cycle, eight of themdisplayed a progressive increase from S1 to S4 (p < .05). In UF-EVs, a total of14 miRNAs were differentially abundant between stages. Greater differences wereobserved between stage 1 (S1) and stage 3 (S3), with 11 miRNAs enriched in S3compared to S1. Functional enrichment analysis revealed the involvement of these miRNAs in relevant pathways such as cell signaling, intercellular junctions, and reproductive functions that may be implicated in oviduct and uterus modulationacross the cycle, but also in their preparation for embryo/conceptus presence anddevelopment.\n",
      "CI  - (c) 2021 Federation of American Societies for Experimental Biology.\n",
      "FAU - Hamdi, Meriem\n",
      "AU  - Hamdi M\n",
      "AUID- ORCID: 0000-0002-5210-4915\n",
      "AD  - Department of Animal Reproduction, INIA-CSIC, Madrid, Spain.\n",
      "AD  - Genetics and Functional Genomics, Clinic of Reproductive Medicine, Department forFarm Animals, Vetsuisse Faculty, University of Zurich, Zurich, Switzerland.\n",
      "FAU - Canon-Beltran, Karina\n",
      "AU  - Canon-Beltran K\n",
      "AD  - Department of Animal Reproduction, INIA-CSIC, Madrid, Spain.\n",
      "AD  - Facultad de Ciencias Agrarias y Ambientales, Programa de Medicina Veterinaria,Fundacion Universitaria Juan de Castellanos, Tunja, Colombia.\n",
      "FAU - Mazzarella, Rosane\n",
      "AU  - Mazzarella R\n",
      "AD  - Department of Veterinary Medicine, Faculty of Animal Science and FoodEngineering, University of Sao Paulo, Pirassununga, Brazil.\n",
      "FAU - Cajas, Yulia N\n",
      "AU  - Cajas YN\n",
      "AD  - Department of Animal Reproduction, INIA-CSIC, Madrid, Spain.\n",
      "AD  - Laboratorio de Biotecnologia de la Reproduccion Animal, Facultad de CienciasAgropecuarias, Universidad de Cuenca (UC), Cuenca, Ecuador.\n",
      "FAU - Leal, Claudia L V\n",
      "AU  - Leal CLV\n",
      "AD  - Department of Veterinary Medicine, Faculty of Animal Science and FoodEngineering, University of Sao Paulo, Pirassununga, Brazil.\n",
      "FAU - Gutierrez-Adan, Alfonso\n",
      "AU  - Gutierrez-Adan A\n",
      "AD  - Department of Animal Reproduction, INIA-CSIC, Madrid, Spain.\n",
      "FAU - Gonzalez, Encina M\n",
      "AU  - Gonzalez EM\n",
      "AD  - Department of Anatomy and Embryology, Veterinary Faculty, Complutense University of Madrid (UCM), Madrid, Spain.\n",
      "FAU - Da Silveira, Juliano C\n",
      "AU  - Da Silveira JC\n",
      "AD  - Department of Veterinary Medicine, Faculty of Animal Science and FoodEngineering, University of Sao Paulo, Pirassununga, Brazil.\n",
      "FAU - Rizos, Dimitrios\n",
      "AU  - Rizos D\n",
      "AUID- ORCID: 0000-0001-6813-3940\n",
      "AD  - Department of Animal Reproduction, INIA-CSIC, Madrid, Spain.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PL  - United States\n",
      "TA  - FASEB J\n",
      "JT  - FASEB journal : official publication of the Federation of American Societies for Experimental Biology\n",
      "JID - 8804484\n",
      "RN  - 0 (MicroRNAs)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Cattle\n",
      "MH  - *Cell Communication\n",
      "MH  - Estrous Cycle/genetics/*metabolism\n",
      "MH  - Extracellular Vesicles/genetics/*metabolism\n",
      "MH  - Female\n",
      "MH  - MicroRNAs/*genetics/metabolism\n",
      "MH  - Oviducts/*metabolism\n",
      "MH  - Phagocytosis\n",
      "MH  - Uterus/*metabolism\n",
      "OTO - NOTNLM\n",
      "OT  - *estrous cycle\n",
      "OT  - *extracellular vesicles\n",
      "OT  - *microRNA\n",
      "OT  - *oviduct\n",
      "OT  - *uterus\n",
      "EDAT- 2021/11/04 06:00\n",
      "MHDA- 2021/12/21 06:00\n",
      "CRDT- 2021/11/03 17:33\n",
      "PHST- 2021/09/19 00:00 [revised]\n",
      "PHST- 2021/06/23 00:00 [received]\n",
      "PHST- 2021/10/05 00:00 [accepted]\n",
      "PHST- 2021/11/03 17:33 [entrez]\n",
      "PHST- 2021/11/04 06:00 [pubmed]\n",
      "PHST- 2021/12/21 06:00 [medline]\n",
      "AID - 10.1096/fj.202101023R [doi]\n",
      "PST - ppublish\n",
      "SO  - FASEB J. 2021 Dec;35(12):e22000. doi: 10.1096/fj.202101023R.\n",
      "\n",
      "PMID- 34659277\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20211022\n",
      "IS  - 1664-462X (Print)\n",
      "IS  - 1664-462X (Linking)\n",
      "VI  - 12\n",
      "DP  - 2021\n",
      "TI  - Adaptive Reprogramming During Early Seed Germination Requires TemporarilyEnhanced Fermentation-A Critical Role for Alternative Oxidase Regulation ThatConcerns Also Microbiota Effectiveness.\n",
      "PG  - 686274\n",
      "LID - 10.3389/fpls.2021.686274 [doi]\n",
      "AB  - Plants respond to environmental cues via adaptive cell reprogramming that canaffect whole plant and ecosystem functionality. Microbiota constitutes part ofthe inner and outer environment of the plant. This Umwelt underlies steadydynamics, due to complex local and global biotic and abiotic changes. Hence,adaptive plant holobiont responses are crucial for continuous metabolicadjustment at the systems level. Plants require oxygen-dependent respiration for energy-dependent adaptive morphology, such as germination, root and shoot growth,and formation of adventitious, clonal, and reproductive organs, fruits, andseeds. Fermentative paths can help in acclimation and, to our view, the role ofalternative oxidase (AOX) in coordinating complex metabolic and physiologicaladjustments is underestimated. Cellular levels of sucrose are an important sensorof environmental stress. We explored the role of exogenous sucrose and itsinterplay with AOX during early seed germination. We found that sucrose-dependentinitiation of fermentation during the first 12 h after imbibition (HAI) wasbeneficial to germination. However, parallel upregulated AOX expression wasessential to control negative effects by prolonged sucrose treatment. Earlydownregulated AOX activity until 12 HAI improved germination efficiency in theabsence of sucrose but suppressed early germination in its presence. The results also suggest that seeds inoculated with arbuscular mycorrhizal fungi (AMF) canbuffer sucrose stress during germination to restore normal respiration moreefficiently. Following this approach, we propose a simple method to identifyorganic seeds and low-cost on-farm perspectives for early identifying diseasetolerance, predicting plant holobiont behavior, and improving germination.Furthermore, the research strengthens the view that AOX can serve as a powerfulfunctional marker source for seed hologenomes.\n",
      "CI  - Copyright (c) 2021 Bharadwaj, Noceda, Mohanapriya, Kumar, Thiers, Costa, Macedo, Kumari, Gupta, Srivastava, Adholeya, Oliveira, Velada, Sircar, Sathishkumar andArnholdt-Schmitt.\n",
      "FAU - Bharadwaj, Revuru\n",
      "AU  - Bharadwaj R\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "FAU - Noceda, Carlos\n",
      "AU  - Noceda C\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Cell and Molecular Biology of Plants (BIOCEMP)/Industrial Biotechnology andBioproducts, Departamento de Ciencias de la Vida y de la Agricultura, Universidadde las Fuerzas Armadas-ESPE, Sangolqui, Ecuador.\n",
      "FAU - Mohanapriya, Gunasekharan\n",
      "AU  - Mohanapriya G\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "FAU - Kumar, Sarma Rajeev\n",
      "AU  - Kumar SR\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "FAU - Thiers, Karine Leitao Lima\n",
      "AU  - Thiers KLL\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "FAU - Costa, Jose Helio\n",
      "AU  - Costa JH\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "FAU - Macedo, Elisete Santos\n",
      "AU  - Macedo ES\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "FAU - Kumari, Aprajita\n",
      "AU  - Kumari A\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - National Institute of Plant Genome Research, New Delhi, India.\n",
      "FAU - Gupta, Kapuganti Jagadis\n",
      "AU  - Gupta KJ\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - National Institute of Plant Genome Research, New Delhi, India.\n",
      "FAU - Srivastava, Shivani\n",
      "AU  - Srivastava S\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Centre for Mycorrhizal Research, Sustainable Agriculture Division, The Energy andResources Institute (TERI), TERI Gram, Gurugram, India.\n",
      "FAU - Adholeya, Alok\n",
      "AU  - Adholeya A\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Centre for Mycorrhizal Research, Sustainable Agriculture Division, The Energy andResources Institute (TERI), TERI Gram, Gurugram, India.\n",
      "FAU - Oliveira, Manuela\n",
      "AU  - Oliveira M\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Department of Mathematics and CIMA - Center for Research on Mathematics and ItsApplications, Universidade de Evora, Evora, Portugal.\n",
      "FAU - Velada, Isabel\n",
      "AU  - Velada I\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - MED-Mediterranean Institute for Agriculture, Environment and Development,Instituto de Investigacao e Formacao Avancada, Universidade de Evora, Evora,Portugal.\n",
      "FAU - Sircar, Debabrata\n",
      "AU  - Sircar D\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Department of Biotechnology, Indian Institute of Technology Roorkee, Roorkee,India.\n",
      "FAU - Sathishkumar, Ramalingam\n",
      "AU  - Sathishkumar R\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "FAU - Arnholdt-Schmitt, Birgit\n",
      "AU  - Arnholdt-Schmitt B\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20211001\n",
      "PL  - Switzerland\n",
      "TA  - Front Plant Sci\n",
      "JT  - Frontiers in plant science\n",
      "JID - 101568200\n",
      "PMC - PMC8518632\n",
      "OTO - NOTNLM\n",
      "OT  - ROS\n",
      "OT  - Warburg effect\n",
      "OT  - bacterial endophytes and mycorrhizal fungi\n",
      "OT  - biotic stress\n",
      "OT  - on-farm seed selection\n",
      "OT  - organic seeds\n",
      "OT  - seed quality\n",
      "COIS- The authors declare that the research was conducted in the absence of anycommercial or financial relationships that could be construed as a potentialconflict of interest.\n",
      "EDAT- 2021/10/19 06:00\n",
      "MHDA- 2021/10/19 06:01\n",
      "CRDT- 2021/10/18 08:55\n",
      "PHST- 2021/04/20 00:00 [received]\n",
      "PHST- 2021/08/10 00:00 [accepted]\n",
      "PHST- 2021/10/18 08:55 [entrez]\n",
      "PHST- 2021/10/19 06:00 [pubmed]\n",
      "PHST- 2021/10/19 06:01 [medline]\n",
      "AID - 10.3389/fpls.2021.686274 [doi]\n",
      "PST - epublish\n",
      "SO  - Front Plant Sci. 2021 Oct 1;12:686274. doi: 10.3389/fpls.2021.686274. eCollection2021.\n",
      "\n",
      "PMID- 34534714\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20211102\n",
      "LR  - 20211103\n",
      "IS  - 1876-7737 (Electronic)\n",
      "IS  - 1874-3919 (Linking)\n",
      "VI  - 249\n",
      "DP  - 2021 Oct 30\n",
      "TI  - Venomics of the poorly studied hognosed pitvipers Porthidium arcosae andPorthidium volcanicum.\n",
      "PG  - 104379\n",
      "LID - S1874-3919(21)00278-5 [pii]\n",
      "LID - 10.1016/j.jprot.2021.104379 [doi]\n",
      "AB  - We report the first proteomics analyses of the venoms of two poorly studiedsnakes, the Manabi hognosed pitviper Porthidium arcosae endemic to the westerncoastal province of Manabi (Ecuador), and the Costa Rican hognosed pitviper P.volcanicum with distribution restricted to South Pacific Costa Rica and westernPanama. These venom proteomes share a conserved compositional pattern reported infour other congeneric species within the clade of South American Porthidiumspecies, P. nasutum, P. lansbergii, P. ophryomegas, and P. porrasi. Theparaspecific immunorecognition profile of antivenoms produced in Costa Rica (ICP polyvalent), Peru (Instituto Nacional de Salud) and Brazil (soro antibotropicopentavalente, SAB, from Instituto Butantan) against the venom of P. arcosae wasinvestigated through a third-generation antivenomics approach. The maximalvenom-binding capacities of the investigated antivenoms were 97.1 mg, 21.8 mg,and 25.7 mg of P. arcosae venom proteins per gram of SAB, ICP, and INS-PERUantibody molecules, respectively, which translate into 28.4 mg, 13.1 mg, and 15.2mg of total venom proteins bound per vial of SAB, ICP, and INS-PERU AV. Theantivenomics results suggest that 21.8%, 7.8% and 6.1% of the SAB, ICP, andINS-PERU antibody molecules recognized P. arcosae venom toxins. The SAB antivenomneutralized P. arcosae venom's lethality in mice with an ED50 of 31.3 mgV/g SABAV. This preclinical neutralization paraspecificity points to Brazilian SAB as a promising candidate for the treatment of envenomings by Ecuadorian P. arcosae.BIOLOGICAL SIGNIFICANCE: Assessing the preclinical efficacy profile of antivenomsagainst homologous and heterologous medically relevant snake venoms represents animportant goal towards defining the biogeographic range of their clinicalutility. This is particularly relevant in regions, such as Mesoamerica, where asmall number of pharmaceutical companies produce antivenoms against the venoms ofa small number of species of maximum medical relevance among the local richherpetofauna, leaving a wide range of snakes of secondary medical relevance, but also causing life-threatening human envenomings without nominal clinicalcoverage. This work is part of a larger project aiming at mapping theimmunological characteristics of antivenoms generated in Latin American countriestowards venoms of such poorly studied snakes of the local and neighboringcountries' herpetofauna. Here we report the proteomics characterization of theManabi hognosed pitviper Porthidium arcosae endemic to the western coastalprovince of Manabi (Ecuador), and the Costa Rican hognosed pitviper P. volcanicumwith distribution restricted to southwestern Costa Rica, the antivenomicsassessment of three bothropoid commercial antivenoms produced in Costa Rica,Peru, and Brazil against the venom components of P. arcosae, and the in vivocapacity of the Brazilian soro antibotropico pentavalente (SAB) from InstitutoButantan to neutralize the murine lethality of P. arcosae venom. The preclinical paraspecific ED50 of 31.3 mg of P. arcosae venom per gram of antivenom points to Brazilian SAB as a promising candidate for the treatment of envenomings by theManabi hognosed pitviper P. arcosae.\n",
      "CI  - Copyright (c) 2021 Elsevier B.V. All rights reserved.\n",
      "FAU - Ruiz-Campos, Marco\n",
      "AU  - Ruiz-Campos M\n",
      "AD  - Instituto Clodomiro Picado, Facultad de Microbiologia, Universidad de Costa Rica,San Jose 11501, Costa Rica.\n",
      "FAU - Sanz, Libia\n",
      "AU  - Sanz L\n",
      "AD  - Laboratorio de Venomica Evolutiva y Traslacional, Consejo Superior deInvestigaciones Cientificas, Valencia, Spain. Electronic address:libia.sanz@ibv.csic.es.\n",
      "FAU - Bonilla, Fabian\n",
      "AU  - Bonilla F\n",
      "AD  - Instituto Clodomiro Picado, Facultad de Microbiologia, Universidad de Costa Rica,San Jose 11501, Costa Rica. Electronic address: Fabian.Bonilla@ucr.ac.cr.\n",
      "FAU - Sasa, Mahmood\n",
      "AU  - Sasa M\n",
      "AD  - Instituto Clodomiro Picado, Facultad de Microbiologia, Universidad de Costa Rica,San Jose 11501, Costa Rica; Centro Investigaciones, Biodiversidad y EcologiaTropical, Universidad de Costa Rica, San Jose, Costa Rica. Electronic address:mahmood.sasa@ucr.ac.cr.\n",
      "FAU - Lomonte, Bruno\n",
      "AU  - Lomonte B\n",
      "AD  - Instituto Clodomiro Picado, Facultad de Microbiologia, Universidad de Costa Rica,San Jose 11501, Costa Rica. Electronic address: bruno.lomonte@ucr.ac.cr.\n",
      "FAU - Zaruma-Torres, Fausto\n",
      "AU  - Zaruma-Torres F\n",
      "AD  - College of Biochemistry and Pharmacy, Faculty of Chemical Sciences, University ofCuenca, Cuenca, Ecuador. Electronic address: fausto.zaruma@ucuenca.edu.ec.\n",
      "FAU - Teran, Maria\n",
      "AU  - Teran M\n",
      "AD  - Instituto Nacional de Investigacion en Salud Publica (INSPI), Guayaquil, Ecuador.Electronic address: mteran@inspi.gob.ec.\n",
      "FAU - Fernandez, Julian\n",
      "AU  - Fernandez J\n",
      "AD  - Instituto Clodomiro Picado, Facultad de Microbiologia, Universidad de Costa Rica,San Jose 11501, Costa Rica.\n",
      "FAU - Calvete, Juan J\n",
      "AU  - Calvete JJ\n",
      "AD  - Laboratorio de Venomica Evolutiva y Traslacional, Consejo Superior deInvestigaciones Cientificas, Valencia, Spain. Electronic address:jcalvete@ibv.csic.es.\n",
      "FAU - Caldeira, Cleopatra A S\n",
      "AU  - Caldeira CAS\n",
      "AD  - Centro de Estudos de Biomoleculas Aplicadas a Saude CEBio/FIOCRUZ/UNIR, Rua daBeira 7671, Lagoa, CEP 76820-245 Porto Velho, Rondonia, Brazil; Programa dePos-graduacao em Biologia Experimental (PGBIOEXP), Universidade Federal deRondonia (UNIR), Porto Velho, Brazil; Programa de Pos-graduacao em Biodiversidadee Biotecnologia, rede BIONORTE, Porto Velho, RO, Brazil. Electronic address:cleopatra@unir.br.\n",
      "FAU - Da Silva, Saulo L\n",
      "AU  - Da Silva SL\n",
      "AD  - College of Biochemistry and Pharmacy, Faculty of Chemical Sciences, University ofCuenca, Cuenca, Ecuador. Electronic address: saulo.da.silva@ucuenca.edu.ec.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20210915\n",
      "PL  - Netherlands\n",
      "TA  - J Proteomics\n",
      "JT  - Journal of proteomics\n",
      "JID - 101475056\n",
      "RN  - 0 (Antivenins)\n",
      "RN  - 0 (Crotalid Venoms)\n",
      "RN  - 0 (Proteome)\n",
      "RN  - 0 (Snake Venoms)\n",
      "SB  - IM\n",
      "EIN - J Proteomics. 2021 Oct 5;:104391. PMID: 34636725\n",
      "MH  - Animals\n",
      "MH  - Antivenins\n",
      "MH  - *Crotalid Venoms\n",
      "MH  - *Crotalinae\n",
      "MH  - Mice\n",
      "MH  - Proteome\n",
      "MH  - Proteomics\n",
      "MH  - Snake Venoms\n",
      "OTO - NOTNLM\n",
      "OT  - *Antivenomics\n",
      "OT  - *Genus Porthidium\n",
      "OT  - *Latin American antivenoms\n",
      "OT  - *Porthidium arcosae\n",
      "OT  - *Porthidium volcanicum\n",
      "OT  - *Snake venomics\n",
      "EDAT- 2021/09/18 06:00\n",
      "MHDA- 2021/11/03 06:00\n",
      "CRDT- 2021/09/17 20:15\n",
      "PHST- 2021/08/11 00:00 [received]\n",
      "PHST- 2021/09/07 00:00 [revised]\n",
      "PHST- 2021/09/07 00:00 [accepted]\n",
      "PHST- 2021/09/18 06:00 [pubmed]\n",
      "PHST- 2021/11/03 06:00 [medline]\n",
      "PHST- 2021/09/17 20:15 [entrez]\n",
      "AID - S1874-3919(21)00278-5 [pii]\n",
      "AID - 10.1016/j.jprot.2021.104379 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Proteomics. 2021 Oct 30;249:104379. doi: 10.1016/j.jprot.2021.104379. Epub 2021Sep 15.\n",
      "\n",
      "PMID- 34448248\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20211110\n",
      "LR  - 20211129\n",
      "IS  - 1525-1470 (Electronic)\n",
      "IS  - 0736-8046 (Linking)\n",
      "VI  - 38\n",
      "IP  - 5\n",
      "DP  - 2021 Sep\n",
      "TI  - CARD14-associated papulosquamous eruption (CAPE) in pediatric patients: Threeadditional cases and review of the literature.\n",
      "PG  - 1237-1242\n",
      "LID - 10.1111/pde.14779 [doi]\n",
      "AB  - CARD14-associated papulosquamous eruption (CAPE) is a proposed term thatencompasses features ranging from psoriasis to pityriasis rubra pilaris (PRP) in association with CARD14 mutations. The early onset of the disease, prominentfacial involvement, family history of an autosomal dominant trait, and poorresponse to conventional treatment are characteristics of CAPE that distinguishit from classical psoriasis and PRP. We describe the clinical features, familyhistory, and response to therapy in three unrelated children with CAPE andcompare these characteristics with those of previously described pediatricpatients. Testing for CARD14 mutations in children with early onset of featuresof psoriasis or pityriasis rubra pilaris and resistance to conventional therapyshould be considered.\n",
      "CI  - (c) 2021 Wiley Periodicals LLC.\n",
      "FAU - Frare, Cindy P\n",
      "AU  - Frare CP\n",
      "AUID- ORCID: https://orcid.org/0000-0001-8488-5675\n",
      "AD  - Department of Dermatology, Hospital Aleman, Buenos Aires, Argentina.\n",
      "FAU - Blumstein, Alli J\n",
      "AU  - Blumstein AJ\n",
      "AD  - Departments of Dermatology and Pediatrics, Northwestern University FeinbergSchool of Medicine, Chicago, IL, USA.\n",
      "FAU - Paller, Amy S\n",
      "AU  - Paller AS\n",
      "AUID- ORCID: https://orcid.org/0000-0001-6187-6549\n",
      "AD  - Departments of Dermatology and Pediatrics, Northwestern University FeinbergSchool of Medicine, Chicago, IL, USA.\n",
      "FAU - Pieretti, Lia\n",
      "AU  - Pieretti L\n",
      "AD  - Dr Uraga Private Center for Skin Diseases, Guayaquil, Ecuador.\n",
      "FAU - Choate, Keith A\n",
      "AU  - Choate KA\n",
      "AD  - Department of Genetics, Yale University School of Medicine, New Haven, CT, USA.\n",
      "FAU - Bowcock, Anne M\n",
      "AU  - Bowcock AM\n",
      "AD  - Dermatology, Oncological Sciences and Genetics & Genomics, Icahn School ofMedicine at Mount Sinai, New York, NY, USA.\n",
      "FAU - Larralde, Margarita\n",
      "AU  - Larralde M\n",
      "AUID- ORCID: https://orcid.org/0000-0002-3801-194X\n",
      "AD  - Department of Dermatology, Hospital Aleman, Buenos Aires, Argentina.\n",
      "AD  - Department of Pediatric Dermatology, Ramos Mejia Hospital, Buenos Aires,Argentina.\n",
      "LA  - eng\n",
      "GR  - R01 AR050266/AR/NIAMS NIH HHS/United States\n",
      "GR  - Foundation for Ichthyosis and Related Skin Types\n",
      "GR  - R01 AR050266/NH/NIH HHS/United States\n",
      "GR  - R01 AR068392/NH/NIH HHS/United States\n",
      "GR  - R01 AR068392/AR/NIAMS NIH HHS/United States\n",
      "PT  - Case Reports\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "DEP - 20210826\n",
      "PL  - United States\n",
      "TA  - Pediatr Dermatol\n",
      "JT  - Pediatric dermatology\n",
      "JID - 8406799\n",
      "RN  - 0 (CARD Signaling Adaptor Proteins)\n",
      "RN  - 0 (Membrane Proteins)\n",
      "RN  - EC 4.6.1.- (CARD14 protein, human)\n",
      "RN  - EC 4.6.1.2 (Guanylate Cyclase)\n",
      "SB  - IM\n",
      "MH  - CARD Signaling Adaptor Proteins/genetics\n",
      "MH  - Child\n",
      "MH  - *Exanthema\n",
      "MH  - Guanylate Cyclase\n",
      "MH  - Humans\n",
      "MH  - Membrane Proteins\n",
      "MH  - *Pityriasis Rubra Pilaris/diagnosis/drug therapy/genetics\n",
      "PMC - PMC8578307\n",
      "MID - NIHMS1733848\n",
      "OTO - NOTNLM\n",
      "OT  - CARD14\n",
      "OT  - child\n",
      "OT  - pityriasis rubra pilaris\n",
      "OT  - psoriasis\n",
      "OT  - ustekinumab\n",
      "EDAT- 2021/08/28 06:00\n",
      "MHDA- 2021/11/11 06:00\n",
      "CRDT- 2021/08/27 07:12\n",
      "PMCR- 2022/09/01 00:00\n",
      "PHST- 2022/09/01 00:00 [pmc-release]\n",
      "PHST- 2021/08/28 06:00 [pubmed]\n",
      "PHST- 2021/11/11 06:00 [medline]\n",
      "PHST- 2021/08/27 07:12 [entrez]\n",
      "AID - 10.1111/pde.14779 [doi]\n",
      "PST - ppublish\n",
      "SO  - Pediatr Dermatol. 2021 Sep;38(5):1237-1242. doi: 10.1111/pde.14779. Epub 2021 Aug26.\n",
      "\n",
      "PMID- 34446793\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20211108\n",
      "LR  - 20211108\n",
      "IS  - 2045-2322 (Electronic)\n",
      "IS  - 2045-2322 (Linking)\n",
      "VI  - 11\n",
      "IP  - 1\n",
      "DP  - 2021 Aug 26\n",
      "TI  - Identification of key proteins in the signaling crossroads between wound healing and cancer hallmark phenotypes.\n",
      "PG  - 17245\n",
      "LID - 10.1038/s41598-021-96750-5 [doi]\n",
      "AB  - Wound healing (WH) and cancer seem to share common cellular and molecularprocesses that could work in a tight balance to maintain tissue homeostasis or,when unregulated, drive tumor progression. The \"Cancer Hallmarks\" comprisecrucial biological properties that mediate the advancement of the disease andaffect patient prognosis. These hallmarks have been proposed to overlap withessential features of the WH process. However, common hallmarks and proteinsactively participating in both processes have yet to be described. In this workwe identify 21 WH proteins strongly linked with solid tumors by integrated TCGAPan-Cancer and multi-omics analyses. These proteins were associated with eight ofthe ten described cancer hallmarks, especially avoiding immune destruction. Theseresults show that WH and cancer's common proteins are involved in themicroenvironment modification of solid tissues and immune system regulation. Thisset of proteins, between WH and cancer, could represent key targets fordeveloping therapies.\n",
      "CI  - (c) 2021. The Author(s).\n",
      "FAU - Lopez-Cortes, Andres\n",
      "AU  - Lopez-Cortes A\n",
      "AD  - Facultad de Ciencias de la Salud Eugenio Espejo, Universidad UTE, Quito, Ecuador.aalc84@gmail.com.\n",
      "AD  - Latin American Network for the Implementation and Validation of ClinicalPharmacogenomics Guidelines (RELIVAF-CYTED), Madrid, Spain. aalc84@gmail.com.\n",
      "AD  - RNASA-IMEDIR, Computer Science Faculty, Universidad of A Coruna, A Coruna, Spain.aalc84@gmail.com.\n",
      "FAU - Abarca, Estefania\n",
      "AU  - Abarca E\n",
      "AD  - Carrera de Biotecnologia, Universidad Politecnica Salesiana UPS, Quito, Ecuador.\n",
      "FAU - Silva, Leonardo\n",
      "AU  - Silva L\n",
      "AD  - Carrera de Biotecnologia, Universidad Politecnica Salesiana UPS, Quito, Ecuador.\n",
      "FAU - Velastegui, Erick\n",
      "AU  - Velastegui E\n",
      "AD  - Carrera de Biotecnologia, Universidad Politecnica Salesiana UPS, Quito, Ecuador.\n",
      "FAU - Leon-Sosa, Ariana\n",
      "AU  - Leon-Sosa A\n",
      "AD  - Instituto de Investigaciones en Biomedicina iBioMed, Universidad San Francisco deQuito USFQ, Quito, Ecuador.\n",
      "FAU - Karolys, Germania\n",
      "AU  - Karolys G\n",
      "AD  - Carrera de Biotecnologia, Universidad Politecnica Salesiana UPS, Quito, Ecuador.\n",
      "AD  - Grupo de Investigacion y Desarrollo en Ciencias Aplicadas a los RecursosBiologicos, Universidad Politecnica Salesiana, Quito, Ecuador.\n",
      "FAU - Cabrera, Francisco\n",
      "AU  - Cabrera F\n",
      "AD  - Instituto de Investigaciones en Biomedicina iBioMed, Universidad San Francisco deQuito USFQ, Quito, Ecuador.\n",
      "AD  - Colegio de Ciencias de la Salud, Escuela de Medicina Veterinaria, Universidad SanFrancisco de Quito USFQ, Quito, Ecuador.\n",
      "FAU - Caicedo, Andres\n",
      "AU  - Caicedo A\n",
      "AD  - Instituto de Investigaciones en Biomedicina iBioMed, Universidad San Francisco deQuito USFQ, Quito, Ecuador. acaicedo@usfq.edu.ec.\n",
      "AD  - Colegio de Ciencias de la Salud, Escuela de Medicina, Universidad San Franciscode Quito USFQ, Quito, Ecuador. acaicedo@usfq.edu.ec.\n",
      "AD  - Mito-Act Research Consortium, Quito, Ecuador. acaicedo@usfq.edu.ec.\n",
      "AD  - Sistemas Medicos SIME, Universidad San Francisco de Quito USFQ, Quito, Ecuador.acaicedo@usfq.edu.ec.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210826\n",
      "PL  - England\n",
      "TA  - Sci Rep\n",
      "JT  - Scientific reports\n",
      "JID - 101563288\n",
      "RN  - 0 (Proteome)\n",
      "SB  - IM\n",
      "MH  - Gene Expression Profiling/methods\n",
      "MH  - Genetic Predisposition to Disease/genetics\n",
      "MH  - Genomics/methods\n",
      "MH  - Homeostasis/genetics/physiology\n",
      "MH  - Humans\n",
      "MH  - Mutation\n",
      "MH  - Neoplasms/genetics/*metabolism\n",
      "MH  - Phenotype\n",
      "MH  - Protein Interaction Maps/genetics\n",
      "MH  - Proteome/genetics/*metabolism\n",
      "MH  - Proteomics/*methods\n",
      "MH  - Signal Transduction/genetics/*physiology\n",
      "MH  - Tumor Microenvironment/genetics\n",
      "MH  - Wound Healing/genetics/*physiology\n",
      "PMC - PMC8390472\n",
      "EDAT- 2021/08/28 06:00\n",
      "MHDA- 2021/11/09 06:00\n",
      "CRDT- 2021/08/27 06:35\n",
      "PHST- 2021/03/20 00:00 [received]\n",
      "PHST- 2021/07/20 00:00 [accepted]\n",
      "PHST- 2021/08/27 06:35 [entrez]\n",
      "PHST- 2021/08/28 06:00 [pubmed]\n",
      "PHST- 2021/11/09 06:00 [medline]\n",
      "AID - 10.1038/s41598-021-96750-5 [doi]\n",
      "AID - 10.1038/s41598-021-96750-5 [pii]\n",
      "PST - epublish\n",
      "SO  - Sci Rep. 2021 Aug 26;11(1):17245. doi: 10.1038/s41598-021-96750-5.\n",
      "\n",
      "PMID- 34416565\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20211109\n",
      "LR  - 20211109\n",
      "IS  - 1877-9603 (Electronic)\n",
      "IS  - 1877-959X (Linking)\n",
      "VI  - 12\n",
      "IP  - 6\n",
      "DP  - 2021 Nov\n",
      "TI  - Potential distribution of Amblyomma mixtum (Koch, 1844) in climate changescenarios in the Americas.\n",
      "PG  - 101812\n",
      "LID - S1877-959X(21)00165-5 [pii]\n",
      "LID - 10.1016/j.ttbdis.2021.101812 [doi]\n",
      "AB  - Amblyomma mixtum is a Neotropical generalist tick of medical and veterinaryimportance which is widely distributed from United States of America to Ecuador. The aim of this study was to evaluate changes in the geographic projections ofthe ecological niche models of A. mixtum in climate change scenarios in America. We constructed a database of published scientific publications, personalcollections, personal communications, and online databases. Ecological nichemodelling was performed with 15 Bioclimatic variables using kuenm in R and wasprojected to three time periods (Last Glacial Maximum, Current and 2050) forAmerica. Our model indicated a wide distribution for A. mixtum, with higherprobability of occurrence along the Gulf of Mexico and occurring in a lesserproportion in the Pacific states, Central America, and the northern part of SouthAmerica. The areas of new invasion are located mainly on the border of Mexicowith Guatemala and Belize, some regions of Central America and Colombia. Weconclude that the ecological niche modelling are effective tools to infer thepotential distribution of A. mixtum in America, in addition to helping to proposefuture measures of epidemiological control and surveillance in the new potential areas of invasion.\n",
      "CI  - Copyright (c) 2021. Published by Elsevier GmbH.\n",
      "FAU - Aguilar-Dominguez, Mariel\n",
      "AU  - Aguilar-Dominguez M\n",
      "AD  - Laboratorio de Parasitologia, Rancho Torreon del Molino, Facultad de MedicinaVeterinaria y Zootecnia, Universidad Veracruzana, Circunvalacion y Yanez s/n,C.P. 91710 Veracruz, Mexico.\n",
      "FAU - Moo-Llanes, David A\n",
      "AU  - Moo-Llanes DA\n",
      "AD  - Centro Regional de Investigacion en Salud Publica, Instituto Nacional de SaludPublica, Tapachula 30700, Chiapas, Mexico.\n",
      "FAU - Sanchez-Montes, Sokani\n",
      "AU  - Sanchez-Montes S\n",
      "AD  - Facultad de Ciencias Biologicas y Agropecuarias, Region Tuxpan, UniversidadVeracruzana, Tuxpan de Rodriguez Cano, 92870 Veracruz, Mexico; Centro de MedicinaTropical, Division de Investigacion, Facultad de Medicina, Universidad NacionalAutonoma de Mexico, Mexico City, 06726, Mexico.\n",
      "FAU - Becker, Ingeborg\n",
      "AU  - Becker I\n",
      "AD  - Centro de Medicina Tropical, Division de Investigacion, Facultad de Medicina,Universidad Nacional Autonoma de Mexico, Mexico City, 06726, Mexico.\n",
      "FAU - Feria-Arroyo, Teresa Patricia\n",
      "AU  - Feria-Arroyo TP\n",
      "AD  - Department of Biology, The University of Texas Rio Grande Valley, 1201WUniversity Drive Edinburg, TX 78539, USA.\n",
      "FAU - de Leon, Adalberto Perez\n",
      "AU  - de Leon AP\n",
      "AD  - USDA-ARS Knipling-Bushland U.S. Livestock Insects Research Laboratory andVeterinary Pest Genomics Center, Kerrville, TX, USA; USDA-ARS San Joaquin Valley Agricultural Sciences Center, Parlier, CA 93648, USA.\n",
      "FAU - Romero-Salas, Dora\n",
      "AU  - Romero-Salas D\n",
      "AD  - Laboratorio de Parasitologia, Rancho Torreon del Molino, Facultad de MedicinaVeterinaria y Zootecnia, Universidad Veracruzana, Circunvalacion y Yanez s/n,C.P. 91710 Veracruz, Mexico. Electronic address: dromero@uv.mx.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210813\n",
      "PL  - Netherlands\n",
      "TA  - Ticks Tick Borne Dis\n",
      "JT  - Ticks and tick-borne diseases\n",
      "JID - 101522599\n",
      "SB  - IM\n",
      "MH  - Amblyomma/*physiology\n",
      "MH  - *Animal Distribution\n",
      "MH  - Animals\n",
      "MH  - Central America\n",
      "MH  - *Climate Change\n",
      "MH  - *Ecosystem\n",
      "MH  - Models, Biological\n",
      "MH  - North America\n",
      "MH  - South America\n",
      "OTO - NOTNLM\n",
      "OT  - *Amblyomma mixtum\n",
      "OT  - *America\n",
      "OT  - *Ecological niche modelling\n",
      "OT  - *Ectoparasite\n",
      "OT  - *Kuenm\n",
      "OT  - *Potential distribution\n",
      "EDAT- 2021/08/21 06:00\n",
      "MHDA- 2021/11/10 06:00\n",
      "CRDT- 2021/08/20 20:24\n",
      "PHST- 2020/07/21 00:00 [received]\n",
      "PHST- 2021/07/27 00:00 [revised]\n",
      "PHST- 2021/07/28 00:00 [accepted]\n",
      "PHST- 2021/08/21 06:00 [pubmed]\n",
      "PHST- 2021/11/10 06:00 [medline]\n",
      "PHST- 2021/08/20 20:24 [entrez]\n",
      "AID - S1877-959X(21)00165-5 [pii]\n",
      "AID - 10.1016/j.ttbdis.2021.101812 [doi]\n",
      "PST - ppublish\n",
      "SO  - Ticks Tick Borne Dis. 2021 Nov;12(6):101812. doi: 10.1016/j.ttbdis.2021.101812.Epub 2021 Aug 13.\n",
      "\n",
      "PMID- 34406852\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20220112\n",
      "LR  - 20220112\n",
      "IS  - 2379-5042 (Electronic)\n",
      "IS  - 2379-5042 (Linking)\n",
      "VI  - 6\n",
      "IP  - 4\n",
      "DP  - 2021 Aug 25\n",
      "TI  - Proteorhodopsin Phototrophy in Antarctic Coastal Waters.\n",
      "PG  - e0052521\n",
      "LID - 10.1128/mSphere.00525-21 [doi]\n",
      "AB  - Microbial proton-pumping rhodopsins are considered the simplest strategy amongphototrophs to conserve energy from light. Proteorhodopsins are the most studied rhodopsins thus far because of their ubiquitous presence in the ocean, except in Antarctica, where they remain understudied. We analyzed proteorhodopsin abundanceand transcriptional activity in the Western Antarctic coastal seawaters.Combining quantitative PCR (qPCR) and metagenomics, the relative abundance ofproteorhodopsin-bearing bacteria accounted on average for 17, 3.5, and 29.7% ofthe bacterial community in Chile Bay (South Shetland Islands) during 2014, 2016, and 2017 summer-autumn, respectively. The abundance of proteorhodopsin-bearingbacteria changed in relation to environmental conditions such as chlorophyll aand temperature. Alphaproteobacteria, Gammaproteobacteria, and Flavobacteriiawere the main bacteria that transcribed the proteorhodopsin gene during day andnight. Although green light-absorbing proteorhodopsin genes were more abundantthan blue-absorbing ones, the latter were transcribed more intensely, resultingin >50% of the proteorhodopsin transcripts during the day and night.Flavobacteriia were the most abundant proteorhodopsin-bearing bacteria in themetagenomes; however, Alphaproteobacteria and Gammaproteobacteria were morerepresented in the metatranscriptomes, with qPCR quantification suggesting thedominance of the active SAR11 clade. Our results show thatproteorhodopsin-bearing bacteria are prevalent in Antarctic coastal waters inlate austral summer and early autumn, and their ecological relevance needs to be elucidated to better understand how sunlight energy is used in this marineecosystem. IMPORTANCE Proteorhodopsin-bearing microorganisms in the SouthernOcean have been overlooked since their discovery in 2000. The present studyidentify taxonomy and quantify the relative abundance of proteorhodopsin-bearing bacteria and proteorhodopsin gene transcription in the West Antarctic Peninsula'scoastal waters. This information is crucial to understand better how sunlightenters this marine environment through alternative ways unrelated tochlorophyll-based strategies. The relative abundance of proteorhodopsin-bearingbacteria seems to be related to environmental parameters (e.g., chlorophyll a,temperature) that change yearly at the coastal water of the West AntarcticPeninsula during the austral late summers and early autumns.Proteorhodopsin-bearing bacteria from Antarctic coastal waters are potentiallyable to exploit both the green and blue spectrum of sunlight and are a prevalent group during the summer in this polar environment.\n",
      "FAU - Cifuentes-Anticevic, Jeronimo\n",
      "AU  - Cifuentes-Anticevic J\n",
      "AD  - Department of Molecular Genetics and Microbiology, Pontificia UniversidadCatolica de Chilegrid.7870.8, Santiago, Chile.\n",
      "FAU - Alcaman-Arias, Maria E\n",
      "AU  - Alcaman-Arias ME\n",
      "AD  - Department of Oceanography, Universidad de Concepcion, Concepcion, Chile.\n",
      "AD  - Center for Climate and Resilience Research (CR)2, Santiago, Chile.\n",
      "AD  - Escuela de Medicina, Universidad Espiritu Santo, Samborondon, Ecuador.\n",
      "FAU - Alarcon-Schumacher, Tomas\n",
      "AU  - Alarcon-Schumacher T\n",
      "AD  - Department of Molecular Genetics and Microbiology, Pontificia UniversidadCatolica de Chilegrid.7870.8, Santiago, Chile.\n",
      "AD  - Max Planck Institute for Marine Microbiology, Bremen, Germany.\n",
      "FAU - Tamayo-Leiva, Javier\n",
      "AU  - Tamayo-Leiva J\n",
      "AD  - Department of Molecular Genetics and Microbiology, Pontificia UniversidadCatolica de Chilegrid.7870.8, Santiago, Chile.\n",
      "AD  - Center for Climate and Resilience Research (CR)2, Santiago, Chile.\n",
      "FAU - Pedros-Alio, Carlos\n",
      "AU  - Pedros-Alio C\n",
      "AD  - Departamento de Biologia de Sistemas, Centro Nacional de Biotecnologia (CSIC),Madrid, Spain.\n",
      "FAU - Farias, Laura\n",
      "AU  - Farias L\n",
      "AD  - Department of Oceanography, Universidad de Concepcion, Concepcion, Chile.\n",
      "AD  - Center for Climate and Resilience Research (CR)2, Santiago, Chile.\n",
      "FAU - Diez, Beatriz\n",
      "AU  - Diez B\n",
      "AD  - Department of Molecular Genetics and Microbiology, Pontificia UniversidadCatolica de Chilegrid.7870.8, Santiago, Chile.\n",
      "AD  - Center for Climate and Resilience Research (CR)2, Santiago, Chile.\n",
      "AD  - Center for Genome Regulation (CRG), Santiago, Chile.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210818\n",
      "PL  - United States\n",
      "TA  - mSphere\n",
      "JT  - mSphere\n",
      "JID - 101674533\n",
      "RN  - 0 (Rhodopsins, Microbial)\n",
      "RN  - 0 (proteorhodopsin)\n",
      "RN  - 9009-81-8 (Rhodopsin)\n",
      "SB  - IM\n",
      "MH  - Alphaproteobacteria/chemistry/classification/genetics\n",
      "MH  - Antarctic Regions\n",
      "MH  - Ecosystem\n",
      "MH  - Flavobacteriaceae/chemistry/classification/genetics\n",
      "MH  - Metagenomics/*methods\n",
      "MH  - Microbiota/*genetics\n",
      "MH  - *Phototrophic Processes\n",
      "MH  - Phylogeny\n",
      "MH  - Rhodopsin/metabolism\n",
      "MH  - Rhodopsins, Microbial/analysis/*genetics\n",
      "MH  - Seawater/*microbiology\n",
      "PMC - PMC8386455\n",
      "OTO - NOTNLM\n",
      "OT  - *Antarctica\n",
      "OT  - *marine microbiology\n",
      "OT  - *metagenomics\n",
      "OT  - *metatranscriptomes\n",
      "OT  - *photoheterotrophy\n",
      "OT  - *proteorhodopsin\n",
      "OT  - *sunlight\n",
      "EDAT- 2021/08/19 06:00\n",
      "MHDA- 2022/01/13 06:00\n",
      "CRDT- 2021/08/18 17:14\n",
      "PHST- 2021/08/19 06:00 [pubmed]\n",
      "PHST- 2022/01/13 06:00 [medline]\n",
      "PHST- 2021/08/18 17:14 [entrez]\n",
      "AID - 10.1128/mSphere.00525-21 [doi]\n",
      "PST - ppublish\n",
      "SO  - mSphere. 2021 Aug 25;6(4):e0052521. doi: 10.1128/mSphere.00525-21. Epub 2021 Aug 18.\n",
      "\n",
      "PMID- 34348791\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20220204\n",
      "LR  - 20220204\n",
      "IS  - 1479-7364 (Electronic)\n",
      "IS  - 1473-9542 (Linking)\n",
      "VI  - 15\n",
      "IP  - 1\n",
      "DP  - 2021 Aug 4\n",
      "TI  - The limits of clinical findings in similar phenotypes, from Carpenter to ATRXsyndrome using a whole exome sequencing approach: a case review.\n",
      "PG  - 49\n",
      "LID - 10.1186/s40246-021-00348-x [doi]\n",
      "AB  - BACKGROUND: The diagnostic process for uncommon disorders with similarmanifestations is complicated and requires newer technology, like gene sequencingfor a correct diagnosis. MAIN BODY: We described two brothers clinicallydiagnosed with Carpenter syndrome, which is a condition characterized by thepremature fusion of certain skull bones (craniosynostosis), abnormalities of the fingers and toes, and other developmental problems, for which they underwentcraniotomies. However, whole exome sequencing analysis concluded a novelpathological variation in the ATRX chromatin remodeler gene and proteinremodeling demonstrated structural variations that decreased the function, givinga completely different diagnosis to these patients. CONCLUSION: Our study focuseson the importance of using newer technologies, such as whole exome sequencinganalysis, in patients with ambiguous phenotypes.\n",
      "CI  - (c) 2021. The Author(s).\n",
      "FAU - Saenz, Samantha S\n",
      "AU  - Saenz SS\n",
      "AD  - School of Medicine, Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Arias, Benjamin\n",
      "AU  - Arias B\n",
      "AD  - Onelabt S.A., Ballenita, Ecuador.\n",
      "FAU - Hosomichi, Kazuyoshi\n",
      "AU  - Hosomichi K\n",
      "AD  - Department of Bioinformatics and Genomics, Kanazawa University, Kanazawa, Japan.\n",
      "FAU - Romero, Vanessa I\n",
      "AU  - Romero VI\n",
      "AUID- ORCID: 0000-0003-3048-9081\n",
      "AD  - School of Medicine, Universidad San Francisco de Quito, Quito, Ecuador.vromero@usfq.edu.ec.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "DEP - 20210804\n",
      "PL  - England\n",
      "TA  - Hum Genomics\n",
      "JT  - Human genomics\n",
      "JID - 101202210\n",
      "RN  - 0 (Nuclear Proteins)\n",
      "RN  - EC 3.6.4.- (DNA Helicases)\n",
      "RN  - EC 3.6.4.12 (ATRX protein, human)\n",
      "RN  - EC 3.6.4.12 (X-linked Nuclear Protein)\n",
      "RN  - ATR-X syndrome\n",
      "RN  - Acrocephalopolysyndactyly Type II\n",
      "SB  - IM\n",
      "MH  - Acrocephalosyndactylia/*genetics/pathology\n",
      "MH  - DNA Helicases/genetics\n",
      "MH  - Exome/genetics\n",
      "MH  - Humans\n",
      "MH  - Mental Retardation, X-Linked/*genetics/pathology\n",
      "MH  - Mutation/genetics\n",
      "MH  - Nuclear Proteins/*genetics\n",
      "MH  - Phenotype\n",
      "MH  - Whole Exome Sequencing\n",
      "MH  - X-linked Nuclear Protein/*genetics\n",
      "MH  - alpha-Thalassemia/*genetics/pathology\n",
      "PMC - PMC8336023\n",
      "OTO - NOTNLM\n",
      "OT  - *ATRX syndrome\n",
      "OT  - *Rare diseases\n",
      "OT  - *Whole exome sequencing\n",
      "EDAT- 2021/08/06 06:00\n",
      "MHDA- 2022/02/05 06:00\n",
      "CRDT- 2021/08/05 05:37\n",
      "PHST- 2021/03/08 00:00 [received]\n",
      "PHST- 2021/07/20 00:00 [accepted]\n",
      "PHST- 2021/08/05 05:37 [entrez]\n",
      "PHST- 2021/08/06 06:00 [pubmed]\n",
      "PHST- 2022/02/05 06:00 [medline]\n",
      "AID - 10.1186/s40246-021-00348-x [doi]\n",
      "AID - 10.1186/s40246-021-00348-x [pii]\n",
      "PST - epublish\n",
      "SO  - Hum Genomics. 2021 Aug 4;15(1):49. doi: 10.1186/s40246-021-00348-x.\n",
      "\n",
      "PMID- 34345025\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20211111\n",
      "LR  - 20220210\n",
      "IS  - 1530-0366 (Electronic)\n",
      "IS  - 1098-3600 (Linking)\n",
      "VI  - 23\n",
      "IP  - 11\n",
      "DP  - 2021 Nov\n",
      "TI  - Delineating the molecular and phenotypic spectrum of the SETD1B-related syndrome.\n",
      "PG  - 2122-2137\n",
      "LID - 10.1038/s41436-021-01246-2 [doi]\n",
      "AB  - PURPOSE: Pathogenic variants in SETD1B have been associated with a syndromicneurodevelopmental disorder including intellectual disability, language delay,and seizures. To date, clinical features have been described for 11 patients with(likely) pathogenic SETD1B sequence variants. This study aims to furtherdelineate the spectrum of the SETD1B-related syndrome based on characterizing an expanded patient cohort. METHODS: We perform an in-depth clinicalcharacterization of a cohort of 36 unpublished individuals with SETD1B sequencevariants, describing their molecular and phenotypic spectrum. Selected variantswere functionally tested using in vitro and genome-wide methylation assays.RESULTS: Our data present evidence for a loss-of-function mechanism of SETD1Bvariants, resulting in a core clinical phenotype of global developmental delay,language delay including regression, intellectual disability, autism and otherbehavioral issues, and variable epilepsy phenotypes. Developmental delay appearedto precede seizure onset, suggesting SETD1B dysfunction impacts physiologicalneurodevelopment even in the absence of epileptic activity. Males aresignificantly overrepresented and more severely affected, and we speculate thatsex-linked traits could affect susceptibility to penetrance and the clinicalspectrum of SETD1B variants. CONCLUSION: Insights from this extensive cohort willfacilitate the counseling regarding the molecular and phenotypic landscape ofnewly diagnosed patients with the SETD1B-related syndrome.\n",
      "CI  - (c) 2021. The Author(s).\n",
      "FAU - Weerts, Marjolein J A\n",
      "AU  - Weerts MJA\n",
      "AD  - Department of Clinical Genetics, Erasmus MC University Medical Center, Rotterdam,The Netherlands.\n",
      "FAU - Lanko, Kristina\n",
      "AU  - Lanko K\n",
      "AD  - Department of Clinical Genetics, Erasmus MC University Medical Center, Rotterdam,The Netherlands.\n",
      "FAU - Guzman-Vega, Francisco J\n",
      "AU  - Guzman-Vega FJ\n",
      "AD  - King Abdullah University of Science and Technology (KAUST), ComputationalBioscience Research Center (CBRC), Division of Biological and EnvironmentalSciences and Engineering (BESE), Thuwal, Saudi Arabia.\n",
      "FAU - Jackson, Adam\n",
      "AU  - Jackson A\n",
      "AD  - Manchester Centre for Genomic Medicine, St. Mary's Hospital, ManchesterUniversity NHS Foundation Trust, Health Innovation Manchester, Manchester, UK.\n",
      "AD  - Division of Evolution & Genomic Sciences, School of Biological Sciences, Faculty of Biology, Medicine and Health, University of Manchester, Manchester, UK.\n",
      "FAU - Ramakrishnan, Reshmi\n",
      "AU  - Ramakrishnan R\n",
      "AD  - King Abdullah University of Science and Technology (KAUST), ComputationalBioscience Research Center (CBRC), Division of Biological and EnvironmentalSciences and Engineering (BESE), Thuwal, Saudi Arabia.\n",
      "FAU - Cardona-Londono, Kelly J\n",
      "AU  - Cardona-Londono KJ\n",
      "AD  - King Abdullah University of Science and Technology (KAUST), ComputationalBioscience Research Center (CBRC), Division of Biological and EnvironmentalSciences and Engineering (BESE), Thuwal, Saudi Arabia.\n",
      "FAU - Pena-Guerra, Karla A\n",
      "AU  - Pena-Guerra KA\n",
      "AD  - King Abdullah University of Science and Technology (KAUST), ComputationalBioscience Research Center (CBRC), Division of Biological and EnvironmentalSciences and Engineering (BESE), Thuwal, Saudi Arabia.\n",
      "FAU - van Bever, Yolande\n",
      "AU  - van Bever Y\n",
      "AD  - Department of Clinical Genetics, Erasmus MC University Medical Center, Rotterdam,The Netherlands.\n",
      "FAU - van Paassen, Barbara W\n",
      "AU  - van Paassen BW\n",
      "AD  - Department of Clinical Genetics, Erasmus MC University Medical Center, Rotterdam,The Netherlands.\n",
      "FAU - Kievit, Anneke\n",
      "AU  - Kievit A\n",
      "AD  - Department of Clinical Genetics, Erasmus MC University Medical Center, Rotterdam,The Netherlands.\n",
      "FAU - van Slegtenhorst, Marjon\n",
      "AU  - van Slegtenhorst M\n",
      "AD  - Department of Clinical Genetics, Erasmus MC University Medical Center, Rotterdam,The Netherlands.\n",
      "FAU - Allen, Nicholas M\n",
      "AU  - Allen NM\n",
      "AD  - Department of Paediatrics, National University of Ireland Galway, Galway,Ireland.\n",
      "FAU - Kehoe, Caroline M\n",
      "AU  - Kehoe CM\n",
      "AD  - Department of Paediatrics, National University of Ireland Galway, Galway,Ireland.\n",
      "FAU - Robinson, Hannah K\n",
      "AU  - Robinson HK\n",
      "AD  - Exeter Genomics Laboratory, RILD Building, Royal Devon and Exeter NHS Foundation Trust, Exeter, UK.\n",
      "FAU - Pang, Lewis\n",
      "AU  - Pang L\n",
      "AD  - Exeter Genomics Laboratory, RILD Building, Royal Devon and Exeter NHS Foundation Trust, Exeter, UK.\n",
      "FAU - Banu, Selina H\n",
      "AU  - Banu SH\n",
      "AD  - Department of Pediatric Neurology, Dr. M.R. Khan Shishu (Children) Hospital andICH, Mirpur, Dhaka, Bangladesh.\n",
      "FAU - Zaman, Mashaya\n",
      "AU  - Zaman M\n",
      "AD  - Department of Pediatric Neurology, Dr. M.R. Khan Shishu (Children) Hospital andICH, Mirpur, Dhaka, Bangladesh.\n",
      "FAU - Efthymiou, Stephanie\n",
      "AU  - Efthymiou S\n",
      "AD  - Department of Neuromuscular Disorders, Queen Square Institute of Neurology,University College London, London, UK.\n",
      "FAU - Houlden, Henry\n",
      "AU  - Houlden H\n",
      "AD  - Department of Neuromuscular Disorders, Queen Square Institute of Neurology,University College London, London, UK.\n",
      "FAU - Jarvela, Irma\n",
      "AU  - Jarvela I\n",
      "AD  - Department of Medical Genetics, University of Helsinki, Helsinki, Finland.\n",
      "FAU - Lauronen, Leena\n",
      "AU  - Lauronen L\n",
      "AD  - Department of Clinical Neurophysiology, New Children s Hospital, HUS DiagnosticCenter, University of Helsinki and Helsinki University Hospital (HUH), Helsinki, Finland.\n",
      "FAU - Maatta, Tuomo\n",
      "AU  - Maatta T\n",
      "AD  - Disability Services, Joint Authority for Kainuu, Kajaani, Finland.\n",
      "FAU - Schrauwen, Isabelle\n",
      "AU  - Schrauwen I\n",
      "AD  - Center for Statistical Genetics, Sergievsky Center, Taub Institute forAlzheimer's Disease and the Aging Brain, Department of Neurology, ColumbiaUniversity Medical Center, New York, NY, USA.\n",
      "FAU - Leal, Suzanne M\n",
      "AU  - Leal SM\n",
      "AD  - Center for Statistical Genetics, Sergievsky Center, Taub Institute forAlzheimer's Disease and the Aging Brain, Department of Neurology, ColumbiaUniversity Medical Center, New York, NY, USA.\n",
      "FAU - Ruivenkamp, Claudia A L\n",
      "AU  - Ruivenkamp CAL\n",
      "AD  - Department of Clinical Genetics, Leiden University Medical Center, Leiden, TheNetherlands.\n",
      "FAU - Barge-Schaapveld, Daniela Q C M\n",
      "AU  - Barge-Schaapveld DQCM\n",
      "AD  - Department of Clinical Genetics, Leiden University Medical Center, Leiden, TheNetherlands.\n",
      "FAU - Peeters-Scholte, Cacha M P C D\n",
      "AU  - Peeters-Scholte CMPCD\n",
      "AD  - Department of Neurology, Leiden University Medical Center, Leiden, TheNetherlands.\n",
      "FAU - Galehdari, Hamid\n",
      "AU  - Galehdari H\n",
      "AD  - Department of Genetics, Faculty of Science, Shahid Chamran University of Ahvaz,Ahvaz, Iran.\n",
      "FAU - Mazaheri, Neda\n",
      "AU  - Mazaheri N\n",
      "AD  - Department of Genetics, Faculty of Science, Shahid Chamran University of Ahvaz,Ahvaz, Iran.\n",
      "FAU - Sisodiya, Sanjay M\n",
      "AU  - Sisodiya SM\n",
      "AD  - Department of Clinical and Experimental Epilepsy, UCL Queen Square Institute ofNeurology, London, UK.\n",
      "AD  - Chalfont Centre for Epilepsy, London, Bucks, UK.\n",
      "FAU - Harrison, Victoria\n",
      "AU  - Harrison V\n",
      "AD  - Wessex Clinical Genetics Service, Princess Anne Hospital, Southampton, UK.\n",
      "FAU - Sun, Angela\n",
      "AU  - Sun A\n",
      "AD  - Department of Pediatrics, Division of Genetic Medicine, University of Washington School of Medicine, Seattle, Washington, USA.\n",
      "FAU - Thies, Jenny\n",
      "AU  - Thies J\n",
      "AD  - Department of Pediatrics, Division of Genetic Medicine, Seattle Children'sHospital, Seattle, WA, USA.\n",
      "FAU - Pedroza, Luis Alberto\n",
      "AU  - Pedroza LA\n",
      "AD  - Department of Pediatrics, Vagelos College of Physicians and Surgeons, ColumbiaUniversity Irving Medical Center, New York, New York, USA.\n",
      "FAU - Lara-Taranchenko, Yana\n",
      "AU  - Lara-Taranchenko Y\n",
      "AD  - Universidad San Francisco de Quito, Colegio de ciencias de la salud-Hospital delos Valles, Quito, Ecuador.\n",
      "FAU - Chinn, Ivan K\n",
      "AU  - Chinn IK\n",
      "AD  - Department of Pediatrics, Section of Immunology, Allergy, and Retrovirology,Baylor College of Medicine and Texas Children's Hospital, Houston, TX, USA.\n",
      "AD  - Center for Human Immunobiology of Texas Children's Hospital/Department ofPediatrics, Baylor College of Medicine, Houston, TX, USA.\n",
      "FAU - Lupski, James R\n",
      "AU  - Lupski JR\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics, Department of Molecular and HumanGenetics, Baylor College of Medicine, Houston, TX, USA.\n",
      "AD  - Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX, USA.\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Texas Children's Hospital, Houston, TX, USA.\n",
      "FAU - Garza-Flores, Alexandra\n",
      "AU  - Garza-Flores A\n",
      "AD  - Cook Children's Genetics, Cook Children's Physician Network, Cook Children'sHospital, Fort Worth, TX, USA.\n",
      "FAU - McGlothlin, Jeffery\n",
      "AU  - McGlothlin J\n",
      "AD  - Cook Children's Neurosciences, Cook Children's Physician Network, Cook Children'sHospital, Fort Worth, TX, USA.\n",
      "FAU - Yang, Lin\n",
      "AU  - Yang L\n",
      "AD  - Department of Pediatrics, The Second Affiliated Hospital of Xi 'an JiaotongUniversity, Xi'an, China.\n",
      "FAU - Huang, Shaoping\n",
      "AU  - Huang S\n",
      "AD  - Department of Pediatrics, The Second Affiliated Hospital of Xi 'an JiaotongUniversity, Xi'an, China.\n",
      "FAU - Wang, Xiaodong\n",
      "AU  - Wang X\n",
      "AD  - Cipher Gene Ltd, Beijing, China.\n",
      "FAU - Jewett, Tamison\n",
      "AU  - Jewett T\n",
      "AD  - Department of Pediatrics, Section on Medical Genetics, Wake Forest School ofMedicine, Winston-Salem, NC, USA.\n",
      "FAU - Rosso, Gretchen\n",
      "AU  - Rosso G\n",
      "AD  - Department of Pediatrics, Section on Medical Genetics, Wake Forest School ofMedicine, Winston-Salem, NC, USA.\n",
      "FAU - Lin, Xi\n",
      "AU  - Lin X\n",
      "AD  - Pediatrics Department, The First Affiliated Hospital of Fujian MedicalUniversity, Fuzhou, China.\n",
      "FAU - Mohammed, Shehla\n",
      "AU  - Mohammed S\n",
      "AD  - Clinical Genetics, Guy's and St Thomas NHS Foundation Trust, London, UK.\n",
      "FAU - Merritt, J Lawrence 2nd\n",
      "AU  - Merritt JL 2nd\n",
      "AD  - Department of Pediatrics, Division of Genetic Medicine, University of Washington School of Medicine, Seattle, Washington, USA.\n",
      "FAU - Mirzaa, Ghayda M\n",
      "AU  - Mirzaa GM\n",
      "AD  - Department of Pediatrics, Division of Genetic Medicine, University of Washington School of Medicine, Seattle, Washington, USA.\n",
      "AD  - Center for Integrative Brain Research, Seattle Children's Research Institute,Seattle, WA, USA.\n",
      "AD  - Brotman Baty Institute for Precision Medicine, Seattle, WA, USA.\n",
      "FAU - Timms, Andrew E\n",
      "AU  - Timms AE\n",
      "AD  - Center for Developmental Biology and Regenerative Medicine, Seattle Children'sResearch Institute, Seattle, WA, USA.\n",
      "FAU - Scheck, Joshua\n",
      "AU  - Scheck J\n",
      "AD  - Center for Integrative Brain Research, Seattle Children's Research Institute,Seattle, WA, USA.\n",
      "FAU - Elting, Mariet W\n",
      "AU  - Elting MW\n",
      "AD  - Department of Clinical Genetics, VU University Medical Center, Amsterdam, TheNetherlands.\n",
      "FAU - Polstra, Abeltje M\n",
      "AU  - Polstra AM\n",
      "AD  - Department of Clinical Genetics, VU University Medical Center, Amsterdam, TheNetherlands.\n",
      "FAU - Schenck, Lauren\n",
      "AU  - Schenck L\n",
      "AD  - Department of Pediatrics, Division of Medical Genetics, Stanford Medicine,Stanford, CA, USA.\n",
      "FAU - Ruzhnikov, Maura R Z\n",
      "AU  - Ruzhnikov MRZ\n",
      "AD  - Department of Pediatrics, Division of Medical Genetics, Stanford Medicine,Stanford, CA, USA.\n",
      "AD  - Department of Neurology and Neurological Sciences, Stanford Medicine, Stanford,CA, USA.\n",
      "FAU - Vetro, Annalisa\n",
      "AU  - Vetro A\n",
      "AD  - Pediatric Neurology, Neurogenetics and Neurobiology Unit and Laboratories, Meyer Children's Hospital-University of Florence, Florence, Italy.\n",
      "FAU - Montomoli, Martino\n",
      "AU  - Montomoli M\n",
      "AD  - Pediatric Neurology, Neurogenetics and Neurobiology Unit and Laboratories, Meyer Children's Hospital-University of Florence, Florence, Italy.\n",
      "FAU - Guerrini, Renzo\n",
      "AU  - Guerrini R\n",
      "AD  - Pediatric Neurology, Neurogenetics and Neurobiology Unit and Laboratories, Meyer Children's Hospital-University of Florence, Florence, Italy.\n",
      "FAU - Koboldt, Daniel C\n",
      "AU  - Koboldt DC\n",
      "AD  - Nationwide Children's Hospital, Columbus, OH, USA.\n",
      "FAU - Mosher, Theresa Mihalic\n",
      "AU  - Mosher TM\n",
      "AD  - Nationwide Children's Hospital, Columbus, OH, USA.\n",
      "FAU - Pastore, Matthew T\n",
      "AU  - Pastore MT\n",
      "AD  - Nationwide Children's Hospital, Columbus, OH, USA.\n",
      "FAU - McBride, Kim L\n",
      "AU  - McBride KL\n",
      "AD  - Nationwide Children's Hospital, Columbus, OH, USA.\n",
      "FAU - Peng, Jing\n",
      "AU  - Peng J\n",
      "AD  - Xiangya Hospital of Central South University, Changsha, China.\n",
      "FAU - Pan, Zou\n",
      "AU  - Pan Z\n",
      "AD  - Xiangya Hospital of Central South University, Changsha, China.\n",
      "FAU - Willemsen, Marjolein\n",
      "AU  - Willemsen M\n",
      "AD  - Department of Human Genetics, Radboud university medical center, Nijmegen, TheNetherlands.\n",
      "FAU - Koning, Susanne\n",
      "AU  - Koning S\n",
      "AD  - Department of Clinical Genetics, Maastricht University Medical Center,Maastricht, The Netherlands.\n",
      "FAU - Turnpenny, Peter D\n",
      "AU  - Turnpenny PD\n",
      "AD  - Clinical Genetics, Royal Devon & Exeter NHS Foundation Trust, Exeter, UK.\n",
      "FAU - de Vries, Bert B A\n",
      "AU  - de Vries BBA\n",
      "AD  - Department of Human Genetics, Radboud university medical center, Nijmegen, TheNetherlands.\n",
      "FAU - Gilissen, Christian\n",
      "AU  - Gilissen C\n",
      "AD  - Department of Human Genetics, Radboud university medical center, Nijmegen, TheNetherlands.\n",
      "FAU - Pfundt, Rolph\n",
      "AU  - Pfundt R\n",
      "AD  - Department of Human Genetics, Radboud university medical center, Nijmegen, TheNetherlands.\n",
      "FAU - Lees, Melissa\n",
      "AU  - Lees M\n",
      "AD  - NE Thames Regional Genetics Service, Great Ormond Street Hospital for ChildrenNHS Foundation Trust, London, UK.\n",
      "FAU - Braddock, Stephen R\n",
      "AU  - Braddock SR\n",
      "AD  - Division of Medical Genetics, Department of Pediatrics, SSM Health CardinalGlennon Children's Hospital, Saint Louis University School of Medicine, SaintLouis, MO, USA.\n",
      "FAU - Klemp, Kara C\n",
      "AU  - Klemp KC\n",
      "AD  - Division of Medical Genetics, Department of Pediatrics, SSM Health CardinalGlennon Children's Hospital, Saint Louis University School of Medicine, SaintLouis, MO, USA.\n",
      "FAU - Vansenne, Fleur\n",
      "AU  - Vansenne F\n",
      "AD  - Department of Clinical Genetics, University Medical Center Groningen, Groningen, The Netherlands.\n",
      "FAU - van Gijn, Marielle E\n",
      "AU  - van Gijn ME\n",
      "AD  - Department of Clinical Genetics, University Medical Center Groningen, Groningen, The Netherlands.\n",
      "FAU - Quindipan, Catherine\n",
      "AU  - Quindipan C\n",
      "AD  - Center for Personalized Medicine, Department of Pathology and LaboratoryMedicine, Children's Hospital Los Angeles, Los Angeles, CA, USA.\n",
      "FAU - Deardorff, Matthew A\n",
      "AU  - Deardorff MA\n",
      "AD  - Center for Personalized Medicine, Department of Pathology and LaboratoryMedicine, Children's Hospital Los Angeles, Los Angeles, CA, USA.\n",
      "AD  - Departments of Pathology and Pediatrics, Keck School of Medicine of the,University of Southern California, Los Angeles, CA, USA.\n",
      "FAU - Hamm, J Austin\n",
      "AU  - Hamm JA\n",
      "AD  - East Tennessee Children's Hospital Genetics Center, Knoxville, TN, USA.\n",
      "FAU - Putnam, Abbey M\n",
      "AU  - Putnam AM\n",
      "AD  - East Tennessee Children's Hospital Genetics Center, Knoxville, TN, USA.\n",
      "FAU - Baud, Rebecca\n",
      "AU  - Baud R\n",
      "AD  - Department of Medical and Molecular Genetics, Indiana University School ofMedicine, Indianapolis, IN, USA.\n",
      "FAU - Walsh, Laurence\n",
      "AU  - Walsh L\n",
      "AD  - Department of Medical and Molecular Genetics, Indiana University School ofMedicine, Indianapolis, IN, USA.\n",
      "AD  - Department of Neurology, Indiana University School of Medicine and Riley Hospitalfor Children, Indianapolis, IN, USA.\n",
      "FAU - Lynch, Sally A\n",
      "AU  - Lynch SA\n",
      "AD  - Department of Clinical Genetics, Children's Health Ireland at Temple St.Children's Hospital and Our Lady's Children's Hospital, Crumlin, Dublin, Ireland.\n",
      "FAU - Baptista, Julia\n",
      "AU  - Baptista J\n",
      "AD  - Exeter Genomics Laboratory, RILD Building, Royal Devon and Exeter NHS Foundation Trust, Exeter, UK.\n",
      "AD  - Institute of Biomedical and Clinical Science, University of Exeter MedicalSchool, Exeter, UK.\n",
      "FAU - Person, Richard E\n",
      "AU  - Person RE\n",
      "AD  - GeneDx, Gaithersburg, MD, USA.\n",
      "FAU - Monaghan, Kristin G\n",
      "AU  - Monaghan KG\n",
      "AD  - GeneDx, Gaithersburg, MD, USA.\n",
      "FAU - Crunk, Amy\n",
      "AU  - Crunk A\n",
      "AD  - GeneDx, Gaithersburg, MD, USA.\n",
      "FAU - Keller-Ramey, Jennifer\n",
      "AU  - Keller-Ramey J\n",
      "AD  - GeneDx, Gaithersburg, MD, USA.\n",
      "FAU - Reich, Adi\n",
      "AU  - Reich A\n",
      "AD  - GeneDx, Gaithersburg, MD, USA.\n",
      "FAU - Elloumi, Houda Zghal\n",
      "AU  - Elloumi HZ\n",
      "AD  - GeneDx, Gaithersburg, MD, USA.\n",
      "FAU - Alders, Marielle\n",
      "AU  - Alders M\n",
      "AD  - Amsterdam UMC, Department of Clinical Genetics, Amsterdam Reproduction &Development Research Institute, University of Amsterdam, Amsterdam, TheNetherlands.\n",
      "FAU - Kerkhof, Jennifer\n",
      "AU  - Kerkhof J\n",
      "AD  - Molecular Genetics Laboratory, Molecular Diagnostics Division, London HealthSciences Centre, London, ON, Canada.\n",
      "FAU - McConkey, Haley\n",
      "AU  - McConkey H\n",
      "AD  - Molecular Genetics Laboratory, Molecular Diagnostics Division, London HealthSciences Centre, London, ON, Canada.\n",
      "FAU - Haghshenas, Sadegheh\n",
      "AU  - Haghshenas S\n",
      "AD  - Department of Pathology and Laboratory Medicine, Western University, London, ON, Canada.\n",
      "CN  - Genomics England Research Consortium\n",
      "FAU - Maroofian, Reza\n",
      "AU  - Maroofian R\n",
      "AD  - Department of Neuromuscular Disorders, Queen Square Institute of Neurology,University College London, London, UK.\n",
      "FAU - Sadikovic, Bekim\n",
      "AU  - Sadikovic B\n",
      "AD  - Molecular Genetics Laboratory, Molecular Diagnostics Division, London HealthSciences Centre, London, ON, Canada.\n",
      "AD  - Department of Pathology and Laboratory Medicine, Western University, London, ON, Canada.\n",
      "FAU - Banka, Siddharth\n",
      "AU  - Banka S\n",
      "AD  - Manchester Centre for Genomic Medicine, St. Mary's Hospital, ManchesterUniversity NHS Foundation Trust, Health Innovation Manchester, Manchester, UK.\n",
      "AD  - Division of Evolution & Genomic Sciences, School of Biological Sciences, Faculty of Biology, Medicine and Health, University of Manchester, Manchester, UK.\n",
      "FAU - Arold, Stefan T\n",
      "AU  - Arold ST\n",
      "AD  - King Abdullah University of Science and Technology (KAUST), ComputationalBioscience Research Center (CBRC), Division of Biological and EnvironmentalSciences and Engineering (BESE), Thuwal, Saudi Arabia.\n",
      "AD  - Centre de Biologie Structurale, CNRS, INSERM, Universite de Montpellier,Montpellier, France.\n",
      "FAU - Barakat, Tahsin Stefan\n",
      "AU  - Barakat TS\n",
      "AUID- ORCID: http://orcid.org/0000-0003-1231-1562\n",
      "AD  - Department of Clinical Genetics, Erasmus MC University Medical Center, Rotterdam,The Netherlands. t.barakat@erasmusmc.nl.\n",
      "LA  - eng\n",
      "GR  - Veni 91617021/ZONMW_/ZonMw/Netherlands\n",
      "GR  - MR/S005021/1/MRC_/Medical Research Council/United Kingdom\n",
      "GR  - UM1 HG006542/HG/NHGRI NIH HHS/United States\n",
      "GR  - 91617021/None/None\n",
      "GR  - MR/S01165X/1/MRC_/Medical Research Council/United Kingdom\n",
      "GR  - WT_/Wellcome Trust/United Kingdom\n",
      "GR  - G0601943/MRC_/Medical Research Council/United Kingdom\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210803\n",
      "PL  - United States\n",
      "TA  - Genet Med\n",
      "JT  - Genetics in medicine : official journal of the American College of MedicalGenetics\n",
      "JID - 9815831\n",
      "RN  - EC 2.1.1.43 (Histone-Lysine N-Methyltransferase)\n",
      "RN  - EC 2.1.1.43 (SETD1B protein, human)\n",
      "SB  - IM\n",
      "MH  - *Epilepsy/diagnosis/genetics\n",
      "MH  - *Histone-Lysine N-Methyltransferase/genetics\n",
      "MH  - Humans\n",
      "MH  - *Intellectual Disability/diagnosis/genetics\n",
      "MH  - Male\n",
      "MH  - *Neurodevelopmental Disorders/diagnosis/genetics\n",
      "MH  - Phenotype\n",
      "MH  - Seizures/diagnosis/genetics\n",
      "PMC - PMC8553606\n",
      "MID - EMS133143\n",
      "EDAT- 2021/08/05 06:00\n",
      "MHDA- 2021/11/12 06:00\n",
      "CRDT- 2021/08/04 07:00\n",
      "PHST- 2021/03/05 00:00 [received]\n",
      "PHST- 2021/06/02 00:00 [accepted]\n",
      "PHST- 2021/06/02 00:00 [revised]\n",
      "PHST- 2021/08/05 06:00 [pubmed]\n",
      "PHST- 2021/11/12 06:00 [medline]\n",
      "PHST- 2021/08/04 07:00 [entrez]\n",
      "AID - 10.1038/s41436-021-01246-2 [doi]\n",
      "AID - S1098-3600(21)05172-8 [pii]\n",
      "PST - ppublish\n",
      "SO  - Genet Med. 2021 Nov;23(11):2122-2137. doi: 10.1038/s41436-021-01246-2. Epub 2021 Aug 3.\n",
      "\n",
      "PMID- 34305903\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210805\n",
      "LR  - 20210805\n",
      "IS  - 1664-3224 (Electronic)\n",
      "IS  - 1664-3224 (Linking)\n",
      "VI  - 12\n",
      "DP  - 2021\n",
      "TI  - ROS/RNS Balancing, Aerobic Fermentation Regulation and Cell Cycle Control - aComplex Early Trait ('CoV-MAC-TED') for Combating SARS-CoV-2-Induced CellReprogramming.\n",
      "PG  - 673692\n",
      "LID - 10.3389/fimmu.2021.673692 [doi]\n",
      "AB  - In a perspective entitled 'From plant survival under severe stress to anti-viral human defense' we raised and justified the hypothesis that transcript levelprofiles of justified target genes established from in vitro somaticembryogenesis (SE) induction in plants as a reference compared to virus-inducedprofiles can identify differential virus signatures that link to harmfulreprogramming. A standard profile of selected genes named 'ReprogVirus' wasproposed for in vitro-scanning of early virus-induced reprogramming in criticalprimary infected cells/tissues as target trait. For data collection, the'ReprogVirus platform' was initiated. This initiative aims to identify in acommon effort across scientific boundaries critical virus footprints from diversevirus origins and variants as a basis for anti-viral strategy design. Thisapproach is open for validation and extension. In the present study, we initiatedvalidation by experimental transcriptome data available in public domain combinedwith advancing plant wet lab research. We compared plant-adapted transcriptomesaccording to 'RegroVirus' complemented by alternative oxidase (AOX) genes during de novo programming under SE-inducing conditions with in vitro coronavirus-induced transcriptome profiles. This approach enabled identifying a majorcomplex trait for early de novo programming during SARS-CoV-2 infection, called'CoV-MAC-TED'. It consists of unbalanced ROS/RNS levels, which are connected toincreased aerobic fermentation that links to alpha-tubulin-based cellrestructuration and progression of cell cycle. We conclude thatanti-viral/anti-SARS-CoV-2 strategies need to rigorously target 'CoV-MAC-TED' in primary infected nose and mouth cells through prophylactic and very earlytherapeutic strategies. We also discuss potential strategies in the view of thebeneficial role of AOX for resilient behavior in plants. Furthermore, followingthe general observation that ROS/RNS equilibration/redox homeostasis is of utmostimportance at the very beginning of viral infection, we highlight that'de-stressing' disease and social handling should be seen as essential part ofanti-viral/anti-SARS-CoV-2 strategies.\n",
      "CI  - Copyright (c) 2021 Costa, Mohanapriya, Bharadwaj, Noceda, Thiers, Aziz,Srivastava, Oliveira, Gupta, Kumari, Sircar, Kumar, Achra, Sathishkumar, Adholeyaand Arnholdt-Schmitt.\n",
      "FAU - Costa, Jose Helio\n",
      "AU  - Costa JH\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "FAU - Mohanapriya, Gunasekaran\n",
      "AU  - Mohanapriya G\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "FAU - Bharadwaj, Revuru\n",
      "AU  - Bharadwaj R\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "FAU - Noceda, Carlos\n",
      "AU  - Noceda C\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Cell and Molecular Biotechnology of Plants (BIOCEMP)/Industrial Biotechnology andBioproducts, Departamento de Ciencias de la Vida y de la Agricultura, Universidadde las Fuerzas Armadas-ESPE, Sangolqui, Ecuador.\n",
      "FAU - Thiers, Karine Leitao Lima\n",
      "AU  - Thiers KLL\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "FAU - Aziz, Shahid\n",
      "AU  - Aziz S\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "FAU - Srivastava, Shivani\n",
      "AU  - Srivastava S\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Centre for Mycorrhizal Research, Sustainable Agriculture Division, The Energy andResources, Institute (TERI), TERI Gram, Gurugram, India.\n",
      "FAU - Oliveira, Manuela\n",
      "AU  - Oliveira M\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Department of Mathematics and CIMA - Center for Research on Mathematics and ItsApplications, Universidade de Evora, Evora, Portugal.\n",
      "FAU - Gupta, Kapuganti Jagadis\n",
      "AU  - Gupta KJ\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - National Institute of Plant Genome Research, Aruna Asaf Ali Marg, New Delhi,India.\n",
      "FAU - Kumari, Aprajita\n",
      "AU  - Kumari A\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - National Institute of Plant Genome Research, Aruna Asaf Ali Marg, New Delhi,India.\n",
      "FAU - Sircar, Debabrata\n",
      "AU  - Sircar D\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Department of Biotechnology, Indian Institute of Technology Roorkee, Uttarakhand,India.\n",
      "FAU - Kumar, Sarma Rajeev\n",
      "AU  - Kumar SR\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "FAU - Achra, Arvind\n",
      "AU  - Achra A\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Department of Microbiology, Atal Bihari Vajpayee Institute of Medical Sciences & Dr Ram Manohar Lohia Hospital, New Delhi, India.\n",
      "FAU - Sathishkumar, Ramalingam\n",
      "AU  - Sathishkumar R\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "FAU - Adholeya, Alok\n",
      "AU  - Adholeya A\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Centre for Mycorrhizal Research, Sustainable Agriculture Division, The Energy andResources, Institute (TERI), TERI Gram, Gurugram, India.\n",
      "FAU - Arnholdt-Schmitt, Birgit\n",
      "AU  - Arnholdt-Schmitt B\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210707\n",
      "PL  - Switzerland\n",
      "TA  - Front Immunol\n",
      "JT  - Frontiers in immunology\n",
      "JID - 101560960\n",
      "RN  - 0 (Mitochondrial Proteins)\n",
      "RN  - 0 (Plant Proteins)\n",
      "RN  - 0 (Reactive Nitrogen Species)\n",
      "RN  - 0 (Reactive Oxygen Species)\n",
      "RN  - 0 (Tubulin)\n",
      "RN  - EC 1.- (Oxidoreductases)\n",
      "RN  - EC 1.- (alternative oxidase)\n",
      "RN  - EC 2.1.1.4 (ASMT protein, human)\n",
      "RN  - EC 2.1.1.4 (Acetylserotonin O-Methyltransferase)\n",
      "SB  - IM\n",
      "MH  - Acetylserotonin O-Methyltransferase/genetics\n",
      "MH  - Arabidopsis/genetics/growth & development\n",
      "MH  - Cell Cycle/genetics\n",
      "MH  - Cellular Reprogramming/*genetics\n",
      "MH  - Databases, Genetic\n",
      "MH  - Daucus carota/genetics/growth & development\n",
      "MH  - Fermentation\n",
      "MH  - Gene Expression Profiling\n",
      "MH  - Humans\n",
      "MH  - Mitochondrial Proteins/genetics/metabolism\n",
      "MH  - Multifactorial Inheritance/*genetics\n",
      "MH  - Oxidoreductases/genetics/metabolism\n",
      "MH  - Plant Proteins/genetics/metabolism\n",
      "MH  - Reactive Nitrogen Species/metabolism\n",
      "MH  - Reactive Oxygen Species/metabolism\n",
      "MH  - SARS-CoV-2/*pathogenicity\n",
      "MH  - Tubulin/genetics\n",
      "MH  - Viruses/pathogenicity\n",
      "PMC - PMC8293103\n",
      "OTO - NOTNLM\n",
      "OT  - *SARS-CoV-2\n",
      "OT  - *alternative oxidase\n",
      "OT  - *mTOR\n",
      "OT  - *melatonin\n",
      "OT  - *redox biology\n",
      "OT  - *repurposing drugs\n",
      "OT  - *tubulin\n",
      "COIS- The authors declare that the research was conducted in the absence of anycommercial or financial relationships that could be construed as a potentialconflict of interest.\n",
      "EDAT- 2021/07/27 06:00\n",
      "MHDA- 2021/08/06 06:00\n",
      "CRDT- 2021/07/26 06:24\n",
      "PHST- 2021/02/28 00:00 [received]\n",
      "PHST- 2021/06/17 00:00 [accepted]\n",
      "PHST- 2021/07/26 06:24 [entrez]\n",
      "PHST- 2021/07/27 06:00 [pubmed]\n",
      "PHST- 2021/08/06 06:00 [medline]\n",
      "AID - 10.3389/fimmu.2021.673692 [doi]\n",
      "PST - epublish\n",
      "SO  - Front Immunol. 2021 Jul 7;12:673692. doi: 10.3389/fimmu.2021.673692. eCollection 2021.\n",
      "\n",
      "PMID- 34233044\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20220128\n",
      "LR  - 20220128\n",
      "IS  - 1365-294X (Electronic)\n",
      "IS  - 0962-1083 (Linking)\n",
      "VI  - 30\n",
      "IP  - 23\n",
      "DP  - 2021 Dec\n",
      "TI  - Genomics of altitude-associated wing shape in two tropical butterflies.\n",
      "PG  - 6387-6402\n",
      "LID - 10.1111/mec.16067 [doi]\n",
      "AB  - Understanding how organisms adapt to their local environment is central toevolution. With new whole-genome sequencing technologies and the explosion ofdata, deciphering the genomic basis of complex traits that are ecologicallyrelevant is becoming increasingly feasible. Here, we studied the genomic basis ofwing shape in two Neotropical butterflies that inhabit large geographical ranges.Heliconius butterflies at high elevations have been shown to generally haverounder wings than those in the lowlands. We reared over 1,100 butterflies from71 broods of H. erato and H. melpomene in common-garden conditions and showedthat wing aspect ratio, that is, elongatedness, is highly heritable in bothspecies and that elevation-associated wing aspect ratio differences aremaintained. Genome-wide associations with a published data set of 666 wholegenomes from across a hybrid zone, uncovered a highly polygenic basis to wingaspect ratio variation in the wild. We identified several genes that have rolesin wing morphogenesis or wing aspect ratio variation in Drosophila flies, making them promising candidates for future studies. There was little evidence formolecular parallelism in the two species, with only one shared candidate gene,nor for a role of the four known colour pattern loci, except for optix in H.erato. Thus, we present the first insights into the heritability and genomicbasis of within-species wing aspect ratio in two Heliconius species, adding to a growing body of evidence that polygenic adaptation may underlie many ecologicallyrelevant traits.\n",
      "CI  - (c) 2021 The Authors. Molecular Ecology published by John Wiley & Sons Ltd.\n",
      "FAU - Montejo-Kovacevich, Gabriela\n",
      "AU  - Montejo-Kovacevich G\n",
      "AUID- ORCID: 0000-0003-3716-9929\n",
      "AD  - Department of Zoology, University of Cambridge, Cambridge, UK.\n",
      "FAU - Salazar, Patricio A\n",
      "AU  - Salazar PA\n",
      "AUID- ORCID: 0000-0001-8988-0769\n",
      "AD  - Animal and Plant Sciences, University of Sheffield, Sheffield, UK.\n",
      "FAU - Smith, Sophie H\n",
      "AU  - Smith SH\n",
      "AUID- ORCID: 0000-0001-9530-3896\n",
      "AD  - Animal and Plant Sciences, University of Sheffield, Sheffield, UK.\n",
      "FAU - Gavilanes, Kimberly\n",
      "AU  - Gavilanes K\n",
      "AD  - Universidad Regional Amazonica de Ikiam, Tena, Ecuador.\n",
      "FAU - Bacquet, Caroline N\n",
      "AU  - Bacquet CN\n",
      "AUID- ORCID: 0000-0002-1954-1806\n",
      "AD  - Universidad Regional Amazonica de Ikiam, Tena, Ecuador.\n",
      "FAU - Chan, Yingguang Frank\n",
      "AU  - Chan YF\n",
      "AUID- ORCID: 0000-0001-6292-9681\n",
      "AD  - Friedrich Miescher Laboratory of the Max Planck Society, Tubingen, Germany.\n",
      "FAU - Jiggins, Chris D\n",
      "AU  - Jiggins CD\n",
      "AUID- ORCID: 0000-0002-7809-062X\n",
      "AD  - Department of Zoology, University of Cambridge, Cambridge, UK.\n",
      "FAU - Meier, Joana I\n",
      "AU  - Meier JI\n",
      "AUID- ORCID: 0000-0001-7726-2875\n",
      "AD  - Department of Zoology, University of Cambridge, Cambridge, UK.\n",
      "AD  - St John's College, University of Cambridge, Cambridge, UK.\n",
      "FAU - Nadeau, Nicola J\n",
      "AU  - Nadeau NJ\n",
      "AUID- ORCID: 0000-0002-9319-921X\n",
      "AD  - Animal and Plant Sciences, University of Sheffield, Sheffield, UK.\n",
      "LA  - eng\n",
      "GR  - 639096 \"HybridMiX\" and 339873 \"SpeciationGenetics\"/European Research CouncilStarting Grant\n",
      "GR  - 2018SPE0000400194/Agencia Espanola de Cooperacion Internacional para elDesarrollo\n",
      "GR  - NE/L002507/1/Natural Environment Research Council\n",
      "GR  - NE/R010331/1/Natural Environment Research Council\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210728\n",
      "PL  - England\n",
      "TA  - Mol Ecol\n",
      "JT  - Molecular ecology\n",
      "JID - 9214478\n",
      "SB  - IM\n",
      "MH  - Altitude\n",
      "MH  - Animals\n",
      "MH  - *Butterflies/genetics\n",
      "MH  - Female\n",
      "MH  - Genomics\n",
      "MH  - Phenotype\n",
      "MH  - Pigmentation\n",
      "MH  - Pregnancy\n",
      "MH  - Wings, Animal\n",
      "OTO - NOTNLM\n",
      "OT  - * Heliconius\n",
      "OT  - *GWAS\n",
      "OT  - *altitude\n",
      "OT  - *local adaptation\n",
      "OT  - *polygenic\n",
      "OT  - *wing shape\n",
      "EDAT- 2021/07/08 06:00\n",
      "MHDA- 2022/01/29 06:00\n",
      "CRDT- 2021/07/07 17:25\n",
      "PHST- 2020/12/04 00:00 [received]\n",
      "PHST- 2021/07/01 00:00 [accepted]\n",
      "PHST- 2021/07/08 06:00 [pubmed]\n",
      "PHST- 2022/01/29 06:00 [medline]\n",
      "PHST- 2021/07/07 17:25 [entrez]\n",
      "AID - 10.1111/mec.16067 [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Ecol. 2021 Dec;30(23):6387-6402. doi: 10.1111/mec.16067. Epub 2021 Jul 28.\n",
      "\n",
      "PMID- 34229490\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210803\n",
      "LR  - 20210830\n",
      "IS  - 1471-2954 (Electronic)\n",
      "IS  - 0962-8452 (Linking)\n",
      "VI  - 288\n",
      "IP  - 1954\n",
      "DP  - 2021 Jul 14\n",
      "TI  - Green, yellow or black? Genetic differentiation and adaptation signatures in ahighly migratory marine turtle.\n",
      "PG  - 20210754\n",
      "LID - 10.1098/rspb.2021.0754 [doi]\n",
      "AB  - Marine species may exhibit genetic structure accompanied by phenotypicdifferentiation related to adaptation despite their high mobility. Twoshape-based morphotypes have been identified for the green turtle (Cheloniamydas) in the Pacific Ocean: the south-central/western or yellow turtle andnorth-central/eastern or black turtle. The genetic differentiation between these morphotypes and the adaptation of the black turtle to environmentally contrastingconditions of the eastern Pacific region has remained a mystery for decades. Herewe addressed both questions using a reduced-representation genome approach(Dartseq; 9473 neutral SNPs) and identifying candidate outlier loci (67 outlierSNPs) of biological relevance between shape-based morphotypes from eight Pacific foraging grounds (n = 158). Our results support genetic divergence betweenmorphotypes, probably arising from strong natal homing behaviour. Genes andenriched biological functions linked to thermoregulation, hypoxia, melanism,morphogenesis, osmoregulation, diet and reproduction were found to be outliersfor differentiation, providing evidence for adaptation of C. mydas to the easternPacific region and suggesting independent evolutionary trajectories of theshape-based morphotypes. Our findings support the evolutionary distinctness ofthe enigmatic black turtle and contribute to the adaptive research andconservation genomics of a long-lived and highly mobile vertebrate.\n",
      "FAU - Alvarez-Varas, Rocio\n",
      "AU  - Alvarez-Varas R\n",
      "AD  - Departamento de Ciencias Ecologicas, Facultad de Ciencias, Universidad de Chile, Santiago, Chile.\n",
      "AD  - Nucleo Milenio de Ecologia y Manejo Sustentable de Islas Oceanicas (ESMOI),Departamento de Biologia Marina, Universidad Catolica del Norte, Coquimbo, Chile.\n",
      "AD  - Qarapara Tortugas Marinas Chile NGO, Santiago, Chile.\n",
      "FAU - Rojas-Hernandez, Noemi\n",
      "AU  - Rojas-Hernandez N\n",
      "AD  - Departamento de Ciencias Ecologicas, Facultad de Ciencias, Universidad de Chile, Santiago, Chile.\n",
      "FAU - Heidemeyer, Maike\n",
      "AU  - Heidemeyer M\n",
      "AD  - Centro de Investigacion en Biologia Celular y Molecular (CIBCM), Universidad deCosta Rica, San Jose, Costa Rica.\n",
      "FAU - Riginos, Cynthia\n",
      "AU  - Riginos C\n",
      "AD  - School of Biological Sciences, The University of Queensland, Brisbane, Australia.\n",
      "FAU - Benitez, Hugo A\n",
      "AU  - Benitez HA\n",
      "AD  - Laboratorio de Ecologia y Morfometria Evolutiva, Centro de Investigacion deEstudios Avanzados del Maule, Universidad Catolica del Maule, Talca, Chile.\n",
      "FAU - Araya-Donoso, Raul\n",
      "AU  - Araya-Donoso R\n",
      "AD  - School of Life Sciences, Arizona State University, Tempe, AZ, USA.\n",
      "FAU - Resendiz, Eduardo\n",
      "AU  - Resendiz E\n",
      "AD  - Departamento Academico de Ciencias Marinas y Costeras, Universidad Autonoma deBaja California Sur, Mexico.\n",
      "FAU - Lara-Uc, Monica\n",
      "AU  - Lara-Uc M\n",
      "AD  - Departamento Academico de Ciencias Marinas y Costeras, Universidad Autonoma deBaja California Sur, Mexico.\n",
      "FAU - Godoy, Daniel A\n",
      "AU  - Godoy DA\n",
      "AD  - Coastal-Marine Research Group, Institute of Natural and Mathematical Sciences,Massey University, Auckland, New Zealand.\n",
      "FAU - Munoz-Perez, Juan Pablo\n",
      "AU  - Munoz-Perez JP\n",
      "AD  - Galapagos Science Center GSC (Universidad San Francisco de Quito USFQ-University of North Carolina at Chapel Hill UNC), Isla San Cristobal, Galapagos, Ecuador.\n",
      "AD  - University of the Sunshine Coast USC, 90 Sippy Downs Dr, Sippy Downs, Queensland 4556, Australia.\n",
      "FAU - Alarcon-Ruales, Daniela E\n",
      "AU  - Alarcon-Ruales DE\n",
      "AD  - Galapagos Science Center GSC (Universidad San Francisco de Quito USFQ-University of North Carolina at Chapel Hill UNC), Isla San Cristobal, Galapagos, Ecuador.\n",
      "FAU - Alfaro-Shigueto, Joanna\n",
      "AU  - Alfaro-Shigueto J\n",
      "AD  - ProDelphinus, Lima, Peru.\n",
      "AD  - Facultad de Biologia Marina, Universidad Cientifica del Peru, Lima, Peru.\n",
      "FAU - Ortiz-Alvarez, Clara\n",
      "AU  - Ortiz-Alvarez C\n",
      "AD  - ProDelphinus, Lima, Peru.\n",
      "FAU - Mangel, Jeffrey C\n",
      "AU  - Mangel JC\n",
      "AD  - ProDelphinus, Lima, Peru.\n",
      "FAU - Vianna, Juliana A\n",
      "AU  - Vianna JA\n",
      "AD  - Departamento de Ecosistemas y Medio Ambiente, Facultad de Agronomia e Ingenieria Forestal, Pontificia Universidad Catolica de Chile, Santiago, Chile.\n",
      "FAU - Veliz, David\n",
      "AU  - Veliz D\n",
      "AD  - Departamento de Ciencias Ecologicas, Facultad de Ciencias, Universidad de Chile, Santiago, Chile.\n",
      "AD  - Nucleo Milenio de Ecologia y Manejo Sustentable de Islas Oceanicas (ESMOI),Departamento de Biologia Marina, Universidad Catolica del Norte, Coquimbo, Chile.\n",
      "LA  - eng\n",
      "SI  - figshare/10.6084/m9.figshare.c.5477735\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210707\n",
      "PL  - England\n",
      "TA  - Proc Biol Sci\n",
      "JT  - Proceedings. Biological sciences\n",
      "JID - 101245157\n",
      "SB  - IM\n",
      "MH  - Adaptation, Physiological/genetics\n",
      "MH  - Animals\n",
      "MH  - Genetic Drift\n",
      "MH  - Pacific Ocean\n",
      "MH  - *Turtles/genetics\n",
      "PMC - PMC8261206\n",
      "OTO - NOTNLM\n",
      "OT  - *Chelonia mydas\n",
      "OT  - *candidate genes\n",
      "OT  - *conservation genomics\n",
      "OT  - *eastern Pacific\n",
      "OT  - *genetic structure\n",
      "OT  - *melanism\n",
      "EDAT- 2021/07/08 06:00\n",
      "MHDA- 2021/08/04 06:00\n",
      "CRDT- 2021/07/07 05:33\n",
      "PMCR- 2022/07/14 00:00\n",
      "PHST- 2022/07/14 00:00 [pmc-release]\n",
      "PHST- 2021/07/07 05:33 [entrez]\n",
      "PHST- 2021/07/08 06:00 [pubmed]\n",
      "PHST- 2021/08/04 06:00 [medline]\n",
      "AID - 10.1098/rspb.2021.0754 [doi]\n",
      "PST - ppublish\n",
      "SO  - Proc Biol Sci. 2021 Jul 14;288(1954):20210754. doi: 10.1098/rspb.2021.0754. Epub 2021 Jul 7.\n",
      "\n",
      "PMID- 34211468\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210714\n",
      "LR  - 20210714\n",
      "IS  - 1664-3224 (Electronic)\n",
      "IS  - 1664-3224 (Linking)\n",
      "VI  - 12\n",
      "DP  - 2021\n",
      "TI  - From Plant Survival Under Severe Stress to Anti-Viral Human Defense - APerspective That Calls for Common Efforts.\n",
      "PG  - 673723\n",
      "LID - 10.3389/fimmu.2021.673723 [doi]\n",
      "AB  - Reprogramming of primary virus-infected cells is the critical step that turnsviral attacks harmful to humans by initiating super-spreading at cell, organismand population levels. To develop early anti-viral therapies and proactiveadministration, it is important to understand the very first steps of thisprocess. Plant somatic embryogenesis (SE) is the earliest and most studied model for de novo programming upon severe stress that, in contrast to virus attacks,promotes individual cell and organism survival. We argued that transcript levelprofiles of target genes established from in vitro SE induction as referencecompared to virus-induced profiles can identify differential virus traits thatlink to harmful reprogramming. To validate this hypothesis, we selected astandard set of genes named 'ReprogVirus'. This approach was recently applied andpublished. It resulted in identifying 'CoV-MAC-TED', a complex trait that ispromising to support combating SARS-CoV-2-induced cell reprogramming in primaryinfected nose and mouth cells. In this perspective, we aim to explain therationale of our scientific approach. We are highlighting relevant backgroundknowledge on SE, emphasize the role of alternative oxidase in plant reprogrammingand resilience as a learning tool for designing human virus-defense strategiesand, present the list of selected genes. As an outlook, we announce wider datacollection in a 'ReprogVirus Platform' to support anti-viral strategy designthrough common efforts.\n",
      "CI  - Copyright (c) 2021 Arnholdt-Schmitt, Mohanapriya, Bharadwaj, Noceda, Macedo,Sathishkumar, Gupta, Sircar, Kumar, Srivastava, Adholeya, Thiers, Aziz, Velada,Oliveira, Quaresma, Achra, Gupta, Kumar and Costa.\n",
      "FAU - Arnholdt-Schmitt, Birgit\n",
      "AU  - Arnholdt-Schmitt B\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "FAU - Mohanapriya, Gunasekaran\n",
      "AU  - Mohanapriya G\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "FAU - Bharadwaj, Revuru\n",
      "AU  - Bharadwaj R\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "FAU - Noceda, Carlos\n",
      "AU  - Noceda C\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Cell and Molecular Biotechnology of Plants (BIOCEMP)/Industrial Biotechnology andBioproducts, Departamento de Ciencias de la Vida y de la Agricultura, Universidadde las Fuerzas Armadas-ESPE, Sangolqui, Ecuador.\n",
      "FAU - Macedo, Elisete Santos\n",
      "AU  - Macedo ES\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "FAU - Sathishkumar, Ramalingam\n",
      "AU  - Sathishkumar R\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "FAU - Gupta, Kapuganti Jagadis\n",
      "AU  - Gupta KJ\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - National Institute of Plant Genome Research, Aruna Asaf Ali Marg, New Delhi,India.\n",
      "FAU - Sircar, Debabrata\n",
      "AU  - Sircar D\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Department of Biotechnology, Indian Institute of Technology, Roorkee,Uttarakhand, India.\n",
      "FAU - Kumar, Sarma Rajeev\n",
      "AU  - Kumar SR\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "FAU - Srivastava, Shivani\n",
      "AU  - Srivastava S\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Centre for Mycorrhizal Research, Sustainable Agriculture Division, The Energy andResources Institute (TERI), TERI Gram, Gual Pahari, Gurugram, India.\n",
      "FAU - Adholeya, Alok\n",
      "AU  - Adholeya A\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Centre for Mycorrhizal Research, Sustainable Agriculture Division, The Energy andResources Institute (TERI), TERI Gram, Gual Pahari, Gurugram, India.\n",
      "FAU - Thiers, KarineLeitao Lima\n",
      "AU  - Thiers KL\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "FAU - Aziz, Shahid\n",
      "AU  - Aziz S\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "FAU - Velada, Isabel\n",
      "AU  - Velada I\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - MED-Mediterranean Institute for Agriculture, Environment and Development,Instituto de Investigacao e Formacao Avancada, Universidade de Evora, Evora,Portugal.\n",
      "FAU - Oliveira, Manuela\n",
      "AU  - Oliveira M\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Department of Mathematics and CIMA - Center for Research on Mathematics and itsApplications, Universidade de Evora, Evora, Portugal.\n",
      "FAU - Quaresma, Paulo\n",
      "AU  - Quaresma P\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - NOVA LINCS - Laboratory for Informatics and Computer Science, University ofEvora, Evora, Portugal.\n",
      "FAU - Achra, Arvind\n",
      "AU  - Achra A\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Department of Microbiology, Atal Bihari Vajpayee Institute of Medical Sciences & Dr Ram Manohar Lohia Hospital, New Delhi, India.\n",
      "FAU - Gupta, Nidhi\n",
      "AU  - Gupta N\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "FAU - Kumar, Ashwani\n",
      "AU  - Kumar A\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Hargovind Khorana Chair, Jayoti Vidyapeeth Womens University, Jaipur, India.\n",
      "FAU - Costa, Jose Helio\n",
      "AU  - Costa JH\n",
      "AD  - Non-Institutional Competence Focus (NICFocus) 'Functional Cell Reprogramming and Organism Plasticity' (FunCROP), Coordinated from Foros de Vale de Figueira,Alentejo, Portugal.\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20210615\n",
      "PL  - Switzerland\n",
      "TA  - Front Immunol\n",
      "JT  - Frontiers in immunology\n",
      "JID - 101560960\n",
      "RN  - 0 (Mitochondrial Proteins)\n",
      "RN  - 0 (Plant Proteins)\n",
      "RN  - 0 (Reactive Oxygen Species)\n",
      "RN  - EC 1.- (Oxidoreductases)\n",
      "RN  - EC 1.- (alternative oxidase)\n",
      "SB  - IM\n",
      "MH  - COVID-19/pathology/*prevention & control\n",
      "MH  - Cellular Reprogramming Techniques/*methods\n",
      "MH  - Gene Expression Regulation, Developmental/genetics\n",
      "MH  - Gene Expression Regulation, Plant/genetics\n",
      "MH  - Humans\n",
      "MH  - Mitochondrial Proteins/metabolism\n",
      "MH  - Oxidoreductases/metabolism\n",
      "MH  - Plant Development/genetics\n",
      "MH  - Plant Proteins/metabolism\n",
      "MH  - Plant Somatic Embryogenesis Techniques/*methods\n",
      "MH  - Plants/embryology/genetics\n",
      "MH  - Reactive Oxygen Species/metabolism\n",
      "MH  - SARS-CoV-2/*genetics\n",
      "PMC - PMC8240590\n",
      "OTO - NOTNLM\n",
      "OT  - ReprogVirus\n",
      "OT  - SARS-CoV-2\n",
      "OT  - aerobic fermentation\n",
      "OT  - alternative oxidase (AOX)\n",
      "OT  - early cell reprogramming\n",
      "OT  - somatic embryogenesis\n",
      "OT  - stress tolerance\n",
      "OT  - viral diseases\n",
      "COIS- The authors declare that the research was conducted in the absence of anycommercial or financial relationships that could be construed as a potentialconflict of interest.\n",
      "EDAT- 2021/07/03 06:00\n",
      "MHDA- 2021/07/15 06:00\n",
      "CRDT- 2021/07/02 06:37\n",
      "PHST- 2021/02/28 00:00 [received]\n",
      "PHST- 2021/05/13 00:00 [accepted]\n",
      "PHST- 2021/07/02 06:37 [entrez]\n",
      "PHST- 2021/07/03 06:00 [pubmed]\n",
      "PHST- 2021/07/15 06:00 [medline]\n",
      "AID - 10.3389/fimmu.2021.673723 [doi]\n",
      "PST - epublish\n",
      "SO  - Front Immunol. 2021 Jun 15;12:673723. doi: 10.3389/fimmu.2021.673723. eCollection2021.\n",
      "\n",
      "PMID- 34188846\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20210701\n",
      "IS  - 2045-7758 (Print)\n",
      "IS  - 2045-7758 (Linking)\n",
      "VI  - 11\n",
      "IP  - 12\n",
      "DP  - 2021 Jun\n",
      "TI  - Epigenetic effects of parasites and pesticides on captive and wild nestlingbirds.\n",
      "PG  - 7713-7729\n",
      "LID - 10.1002/ece3.7606 [doi]\n",
      "AB  - Anthropogenic changes to the environment challenge animal populations to adapt tonew conditions and unique threats. While the study of adaptation has focused ongenetic variation, epigenetic mechanisms may also be important. DNA methylationis sensitive to environmental stressors, such as parasites and pesticides, which may affect gene expression and phenotype. We studied the effects of an invasiveectoparasite, Philornis downsi, on DNA methylation of Galapagos mockingbirds(Mimus parvulus). We used the insecticide permethrin to manipulate P. downsipresence in nests of free-living mockingbirds and tested for effects ofparasitism on nestling mockingbirds using epiGBS, a reduced-representationbisulfite sequencing (RRBS) approach. To distinguish the confounding effects ofinsecticide exposure, we conducted a matching experiment exposing captivenestling zebra finches (Taeniopygia guttata) to permethrin. We used zebra finchesbecause they were the closest model organism to mockingbirds that we could breed in controlled conditions. We identified a limited number of differentiallymethylated cytosines (DMCs) in parasitized versus nonparasitized mockingbirds,but the number was not more than expected by chance. In contrast, we saw cleareffects of permethrin on methylation in captive zebra finches. DMCs in zebrafinches paralleled documented effects of permethrin exposure on vertebratecellular signaling and endocrine function. Our results from captive birdsindicate a role for epigenetic processes in mediating sublethal nontarget effectsof pyrethroid exposure in vertebrates. Environmental conditions in the field weremore variable than the laboratory, which may have made effects of both parasitismand permethrin harder to detect in mockingbirds. RRBS approaches such as epiGBSmay be a cost-effective way to characterize genome-wide methylation profiles.However, our results indicate that ecological epigenetic studies in naturalpopulations should consider the number of cytosines interrogated and the depth ofsequencing in order to have adequate power to detect small and variable effects.\n",
      "CI  - (c) 2021 The Authors. Ecology and Evolution published by John Wiley & Sons Ltd.\n",
      "FAU - McNew, Sabrina M\n",
      "AU  - McNew SM\n",
      "AUID- ORCID: https://orcid.org/0000-0002-1345-1674\n",
      "AD  - School of Biological Sciences University of Utah Salt Lake City UT USA.\n",
      "AD  - Cornell Lab of Ornithology Cornell University Ithaca NY USA.\n",
      "AD  - Department of Ecology and Evolutionary Biology Cornell University Ithaca NY USA.\n",
      "FAU - Boquete, M Teresa\n",
      "AU  - Boquete MT\n",
      "AUID- ORCID: https://orcid.org/0000-0002-5886-7374\n",
      "AD  - Department of Integrative Biology University of South Florida Tampa FL USA.\n",
      "AD  - Department of Evolutionary Ecology Estacion Biologica de Donana CSIC SevillaSpain.\n",
      "FAU - Espinoza-Ulloa, Sebastian\n",
      "AU  - Espinoza-Ulloa S\n",
      "AUID- ORCID: https://orcid.org/0000-0003-1497-356X\n",
      "AD  - Department of Biology University of Saskatchewan Saskatoon SK Canada.\n",
      "AD  - Facultad de Medicina Pontifica Universidad Catolica del Ecuador Quito Ecuador.\n",
      "FAU - Andres, Jose A\n",
      "AU  - Andres JA\n",
      "AD  - Department of Ecology and Evolutionary Biology Cornell University Ithaca NY USA.\n",
      "AD  - Department of Biology University of Saskatchewan Saskatoon SK Canada.\n",
      "FAU - Wagemaker, Niels C A M\n",
      "AU  - Wagemaker NCAM\n",
      "AD  - Faculty of Science Radboud University Nijmegen The Netherlands.\n",
      "FAU - Knutie, Sarah A\n",
      "AU  - Knutie SA\n",
      "AUID- ORCID: https://orcid.org/0000-0001-6423-9561\n",
      "AD  - School of Biological Sciences University of Utah Salt Lake City UT USA.\n",
      "AD  - Department of Ecology and Evolutionary Biology University of Connecticut StorrsCT USA.\n",
      "AD  - Institute for Systems Genomics University of Connecticut Storrs CT USA.\n",
      "FAU - Richards, Christina L\n",
      "AU  - Richards CL\n",
      "AUID- ORCID: https://orcid.org/0000-0001-7848-5165\n",
      "AD  - Department of Integrative Biology University of South Florida Tampa FL USA.\n",
      "FAU - Clayton, Dale H\n",
      "AU  - Clayton DH\n",
      "AUID- ORCID: https://orcid.org/0000-0003-1698-3542\n",
      "AD  - School of Biological Sciences University of Utah Salt Lake City UT USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20210503\n",
      "PL  - England\n",
      "TA  - Ecol Evol\n",
      "JT  - Ecology and evolution\n",
      "JID - 101566408\n",
      "PMC - PMC8216931\n",
      "OTO - NOTNLM\n",
      "OT  - DNA methylation\n",
      "OT  - Galapagos mockingbirds\n",
      "OT  - Philornis downsi\n",
      "OT  - epiGBS\n",
      "OT  - permethrin\n",
      "OT  - pyrethroid\n",
      "COIS- The authors declare no conflict of interest.\n",
      "EDAT- 2021/07/01 06:00\n",
      "MHDA- 2021/07/01 06:01\n",
      "CRDT- 2021/06/30 06:39\n",
      "PHST- 2021/02/12 00:00 [received]\n",
      "PHST- 2021/04/06 00:00 [revised]\n",
      "PHST- 2021/04/07 00:00 [accepted]\n",
      "PHST- 2021/06/30 06:39 [entrez]\n",
      "PHST- 2021/07/01 06:00 [pubmed]\n",
      "PHST- 2021/07/01 06:01 [medline]\n",
      "AID - 10.1002/ece3.7606 [doi]\n",
      "AID - ECE37606 [pii]\n",
      "PST - epublish\n",
      "SO  - Ecol Evol. 2021 May 3;11(12):7713-7729. doi: 10.1002/ece3.7606. eCollection 2021 Jun.\n",
      "\n",
      "PMID- 34165082\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20211020\n",
      "LR  - 20211020\n",
      "IS  - 2050-084X (Electronic)\n",
      "IS  - 2050-084X (Linking)\n",
      "VI  - 10\n",
      "DP  - 2021 Jun 24\n",
      "TI  - Introgression shapes fruit color convergence in invasive Galapagos tomato.\n",
      "LID - 10.7554/eLife.64165 [doi]\n",
      "LID - e64165 [pii]\n",
      "AB  - Invasive species represent one of the foremost risks to global biodiversity.Here, we use population genomics to evaluate the history and consequences of aninvasion of wild tomato-Solanum pimpinellifolium-onto the Galapagos Islands from continental South America. Using >300 archipelago and mainland collections, weinfer this invasion was recent and largely the result of a single event fromcentral Ecuador. Patterns of ancestry within the genomes of invasive plants also reveal post-colonization hybridization and introgression between S.pimpinellifolium and the closely related Galapagos endemic Solanum cheesmaniae.Of admixed invasive individuals, those that carry endemic alleles at one of twodifferent carotenoid biosynthesis loci also have orange fruits-characteristic of the endemic species-instead of typical red S. pimpinellifolium fruits. We inferthat introgression of two independent fruit color loci explains this observedtrait convergence, suggesting that selection has favored repeated transitions of red to orange fruits on the Galapagos.\n",
      "CI  - (c) 2021, Gibson et al.\n",
      "FAU - Gibson, Matthew Js\n",
      "AU  - Gibson MJ\n",
      "AUID- ORCID: 0000-0001-7855-1628\n",
      "AD  - Department of Biology, Indiana University, Bloomington, United States.\n",
      "FAU - Torres, Maria de Lourdes\n",
      "AU  - Torres ML\n",
      "AUID- ORCID: 0000-0001-7207-4568\n",
      "AD  - Universidad San Francisco de Quito (USFQ). Colegio de Ciencias Biologicas yAmbientales, Laboratorio de Biotecnologia Vegetal. Campus Cumbaya, Quito,Ecuador.\n",
      "AD  - Galapagos Science Center, Universidad San Francisco de Quito and University ofNorth Carolina at Chapel Hill, Galapagos, Ecuador.\n",
      "FAU - Brandvain, Yaniv\n",
      "AU  - Brandvain Y\n",
      "AD  - Department of Plant Biology, University of Minnesota-Twin Cities, St. Paul,United States.\n",
      "FAU - Moyle, Leonie C\n",
      "AU  - Moyle LC\n",
      "AUID- ORCID: 0000-0003-4960-8001\n",
      "AD  - Department of Biology, Indiana University, Bloomington, United States.\n",
      "LA  - eng\n",
      "SI  - Dryad/10.5061/dryad.2v6wwpzkm\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20210624\n",
      "PL  - England\n",
      "TA  - Elife\n",
      "JT  - eLife\n",
      "JID - 101579614\n",
      "RN  - 0 (Pigments, Biological)\n",
      "SB  - IM\n",
      "MH  - Ecuador\n",
      "MH  - Fruit/*genetics/*physiology\n",
      "MH  - *Genetic Introgression\n",
      "MH  - Genetic Variation\n",
      "MH  - *Introduced Species\n",
      "MH  - Pigments, Biological\n",
      "MH  - Solanum/*genetics/*physiology\n",
      "MH  - Species Specificity\n",
      "PMC - PMC8294854\n",
      "OTO - NOTNLM\n",
      "OT  - *Galapagos\n",
      "OT  - *convergence\n",
      "OT  - *evolutionary biology\n",
      "OT  - *genetics\n",
      "OT  - *genomics\n",
      "OT  - *in situ hybridization\n",
      "OT  - *introgression\n",
      "OT  - *invasion\n",
      "OT  - *tomato\n",
      "COIS- MG, MT, YB, LM No competing interests declared\n",
      "EDAT- 2021/06/25 06:00\n",
      "MHDA- 2021/10/21 06:00\n",
      "CRDT- 2021/06/24 08:50\n",
      "PHST- 2020/10/20 00:00 [received]\n",
      "PHST- 2021/06/23 00:00 [accepted]\n",
      "PHST- 2021/06/25 06:00 [pubmed]\n",
      "PHST- 2021/10/21 06:00 [medline]\n",
      "PHST- 2021/06/24 08:50 [entrez]\n",
      "AID - 10.7554/eLife.64165 [doi]\n",
      "AID - 64165 [pii]\n",
      "PST - epublish\n",
      "SO  - Elife. 2021 Jun 24;10. pii: 64165. doi: 10.7554/eLife.64165.\n",
      "\n",
      "PMID- 34025990\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20210526\n",
      "IS  - 2045-7758 (Print)\n",
      "IS  - 2045-7758 (Linking)\n",
      "VI  - 11\n",
      "IP  - 10\n",
      "DP  - 2021 May\n",
      "TI  - Urban living influences the nesting success of Darwin's finches in the Galapagos Islands.\n",
      "PG  - 5038-5048\n",
      "LID - 10.1002/ece3.7360 [doi]\n",
      "AB  - Urbanization is expanding worldwide with major consequences for organisms.Anthropogenic factors can reduce the fitness of animals but may have benefits,such as consistent human food availability. Understanding anthropogenictrade-offs is critical in environments with variable levels of natural foodavailability, such as the Galapagos Islands, an area of rapid urbanization. Forexample, during dry years, the reproductive success of bird species, such asDarwin's finches, is low because reduced precipitation impacts food availability.Urban areas provide supplemental human food to finches, which could improve theirreproductive success during years with low natural food availability. However,urban finches might face trade-offs, such as the incorporation of anthropogenicdebris (e.g., string, plastic) into their nests, which may increase mortality. Inour study, we determined the effect of urbanization on the nesting success ofsmall ground finches (Geospiza fuliginosa; a species of Darwin's finch) during a dry year on San Cristobal Island. We quantified nest building, egg laying andhatching, and fledging in an urban and nonurban area and characterized theanthropogenic debris in nests. We also documented mortalities including nesttrash-related deaths and whether anthropogenic materials directly led toentanglement- or ingestion-related nest mortalities. Overall, urban finches builtmore nests, laid more eggs, and produced more fledglings than nonurban finches.However, every nest in the urban area contained anthropogenic material, whichresulted in 18% nestling mortality while nonurban nests had no anthropogenicdebris. Our study showed that urban living has trade-offs: urban birds haveoverall higher nesting success during a dry year than nonurban birds, but urbanbirds can suffer mortality from anthropogenic-related nest-materials. Theseresults suggest that despite potential costs, finches benefit overall from urban living and urbanization may buffer the effects of limited resource availabilityin the Galapagos Islands.\n",
      "CI  - (c) 2021 The Authors. Ecology and Evolution published by John Wiley & Sons Ltd.\n",
      "FAU - Harvey, Johanna A\n",
      "AU  - Harvey JA\n",
      "AUID- ORCID: https://orcid.org/0000-0003-4504-6777\n",
      "AD  - Department of Ecology and Evolutionary Biology University of Connecticut StorrsCT USA.\n",
      "AD  - Present address: Division of Invertebrate Zoology American Museum of NaturalHistory New York NY USA.\n",
      "FAU - Chernicky, Kiley\n",
      "AU  - Chernicky K\n",
      "AD  - Department of Ecology and Evolutionary Biology University of Connecticut StorrsCT USA.\n",
      "FAU - Simons, Shelby R\n",
      "AU  - Simons SR\n",
      "AD  - Department of Ecology and Evolutionary Biology University of Connecticut StorrsCT USA.\n",
      "FAU - Verrett, Taylor B\n",
      "AU  - Verrett TB\n",
      "AD  - Department of Ecology and Evolutionary Biology University of Connecticut StorrsCT USA.\n",
      "FAU - Chaves, Jaime A\n",
      "AU  - Chaves JA\n",
      "AUID- ORCID: https://orcid.org/0000-0003-0679-2603\n",
      "AD  - Department of Biology San Francisco State University San Francisco CA USA.\n",
      "AD  - Colegio de Ciencias Biologicas y Ambientales Universidad San Francisco de QuitoLaboratorio de Biologia Evolutiva Diego de Robles y Pampite Quito Ecuador.\n",
      "FAU - Knutie, Sarah A\n",
      "AU  - Knutie SA\n",
      "AUID- ORCID: https://orcid.org/0000-0001-6423-9561\n",
      "AD  - Department of Ecology and Evolutionary Biology University of Connecticut StorrsCT USA.\n",
      "AD  - Institute for Systems Genomics University of Connecticut Storrs CT USA.\n",
      "LA  - eng\n",
      "SI  - figshare/10.6084/m9.figshare.12928838.v2\n",
      "PT  - Journal Article\n",
      "DEP - 20210316\n",
      "PL  - England\n",
      "TA  - Ecol Evol\n",
      "JT  - Ecology and evolution\n",
      "JID - 101566408\n",
      "PMC - PMC8131787\n",
      "OTO - NOTNLM\n",
      "OT  - Galapagos Islands\n",
      "OT  - Geospiza fuliginosa\n",
      "OT  - La Nina\n",
      "OT  - anthropogenic debris\n",
      "OT  - dry year\n",
      "OT  - entanglement\n",
      "OT  - nest material\n",
      "OT  - trash\n",
      "OT  - urban ecology\n",
      "COIS- The authors declare that they have no conflict of interest.\n",
      "EDAT- 2021/05/25 06:00\n",
      "MHDA- 2021/05/25 06:01\n",
      "CRDT- 2021/05/24 08:07\n",
      "PHST- 2020/09/08 00:00 [received]\n",
      "PHST- 2021/01/27 00:00 [revised]\n",
      "PHST- 2021/01/29 00:00 [accepted]\n",
      "PHST- 2021/05/24 08:07 [entrez]\n",
      "PHST- 2021/05/25 06:00 [pubmed]\n",
      "PHST- 2021/05/25 06:01 [medline]\n",
      "AID - 10.1002/ece3.7360 [doi]\n",
      "AID - ECE37360 [pii]\n",
      "PST - epublish\n",
      "SO  - Ecol Evol. 2021 Mar 16;11(10):5038-5048. doi: 10.1002/ece3.7360. eCollection 2021May.\n",
      "\n",
      "PMID- 34025776\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20210526\n",
      "IS  - 1752-4571 (Print)\n",
      "IS  - 1752-4571 (Linking)\n",
      "VI  - 14\n",
      "IP  - 5\n",
      "DP  - 2021 May\n",
      "TI  - Population genomics of invasive rodents on islands: Genetic consequences ofcolonization and prospects for localized synthetic gene drive.\n",
      "PG  - 1421-1435\n",
      "LID - 10.1111/eva.13210 [doi]\n",
      "AB  - Introduced rodent populations pose significant threats worldwide, withparticularly severe impacts on islands. Advancements in genome editing havemotivated interest in synthetic gene drives that could potentially provideefficient and localized suppression of invasive rodent populations. Applicationof such technologies will require rigorous population genomic surveys to evaluatepopulation connectivity, taxonomic identification, and to inform design of genedrive localization mechanisms. One proposed approach leverages the predictedshifts in genetic variation that accompany island colonization, wherein foundereffects, genetic drift, and island-specific selection are expected to result inlocally fixed alleles (LFA) that are variable in neighboring nontargetpopulations. Engineering of guide RNAs that target LFA may thus yield gene drivesthat spread within invasive island populations, but would have limited impacts onnontarget populations in the event of an escape. Here we used pooled whole-genomesequencing of invasive mouse (Mus musculus) populations on four islands alongwith paired putative source populations to test genetic predictions of islandcolonization and characterize locally fixed Cas9 genomic targets. Patterns ofvariation across the genome reflected marked reductions in allelic diversity inisland populations and moderate to high degrees of differentiation from nearbysource populations despite relatively recent colonization. Locally fixed Cas9sites in female fertility genes were observed in all island populations,including a small number with multiplexing potential. In practice, rigoroussampling of presumptive LFA will be essential to fully assess risk of resistance alleles. These results should serve to guide development of improved, spatiallylimited gene drive design in future applications.\n",
      "CI  - (c) 2021 The Authors. Evolutionary Applications published by John Wiley & SonsLtd. This article has been contributed to by US Government employees and theirwork is in the public domain in the USA.\n",
      "FAU - Oh, Kevin P\n",
      "AU  - Oh KP\n",
      "AUID- ORCID: https://orcid.org/0000-0001-5094-5510\n",
      "AD  - National Wildlife Research Center USDA APHIS Wildlife Services Fort CollinsColorado USA.\n",
      "AD  - Department of Microbiology, Immunology and Pathology Colorado State UniversityFort Collins Colorado USA.\n",
      "FAU - Shiels, Aaron B\n",
      "AU  - Shiels AB\n",
      "AD  - National Wildlife Research Center USDA APHIS Wildlife Services Fort CollinsColorado USA.\n",
      "FAU - Shiels, Laura\n",
      "AU  - Shiels L\n",
      "AD  - National Wildlife Research Center USDA APHIS Wildlife Services Fort CollinsColorado USA.\n",
      "FAU - Blondel, Dimitri V\n",
      "AU  - Blondel DV\n",
      "AD  - Department of Biological Sciences North Carolina State University Raleigh NorthCarolina USA.\n",
      "FAU - Campbell, Karl J\n",
      "AU  - Campbell KJ\n",
      "AD  - Island Conservation Puerto Ayora Ecuador.\n",
      "AD  - School of Agriculture and Food Sciences The University of Queensland GattonQueensland Australia.\n",
      "FAU - Saah, J Royden\n",
      "AU  - Saah JR\n",
      "AD  - Island Conservation Puerto Ayora Ecuador.\n",
      "AD  - Genetic Engineering and Society Center North Carolina State University RaleighNorth Carolina USA.\n",
      "FAU - Lloyd, Alun L\n",
      "AU  - Lloyd AL\n",
      "AD  - Genetic Engineering and Society Center North Carolina State University RaleighNorth Carolina USA.\n",
      "AD  - Biomathematics Graduate Program and Department of Mathematics North CarolinaState University Raleigh North Carolina USA.\n",
      "FAU - Thomas, Paul Q\n",
      "AU  - Thomas PQ\n",
      "AD  - The Robinson Research Institute and School of Medicine The University of AdelaideAdelaide South Australia Australia.\n",
      "FAU - Gould, Fred\n",
      "AU  - Gould F\n",
      "AD  - Genetic Engineering and Society Center North Carolina State University RaleighNorth Carolina USA.\n",
      "AD  - Department of Entomology and Plant Pathology North Carolina State UniversityRaleigh North Carolina USA.\n",
      "FAU - Abdo, Zaid\n",
      "AU  - Abdo Z\n",
      "AD  - Department of Microbiology, Immunology and Pathology Colorado State UniversityFort Collins Colorado USA.\n",
      "FAU - Godwin, John R\n",
      "AU  - Godwin JR\n",
      "AD  - Department of Biological Sciences North Carolina State University Raleigh NorthCarolina USA.\n",
      "AD  - Genetic Engineering and Society Center North Carolina State University RaleighNorth Carolina USA.\n",
      "FAU - Piaggio, Antoinette J\n",
      "AU  - Piaggio AJ\n",
      "AD  - National Wildlife Research Center USDA APHIS Wildlife Services Fort CollinsColorado USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20210310\n",
      "PL  - England\n",
      "TA  - Evol Appl\n",
      "JT  - Evolutionary applications\n",
      "JID - 101461828\n",
      "PMC - PMC8127709\n",
      "OTO - NOTNLM\n",
      "OT  - CRISPR\n",
      "OT  - Cas9\n",
      "OT  - Mus musculus\n",
      "OT  - genetic biocontrol\n",
      "OT  - genome editing\n",
      "OT  - synthetic biology\n",
      "EDAT- 2021/05/25 06:00\n",
      "MHDA- 2021/05/25 06:01\n",
      "CRDT- 2021/05/24 08:05\n",
      "PHST- 2020/07/16 00:00 [received]\n",
      "PHST- 2021/02/15 00:00 [revised]\n",
      "PHST- 2021/02/16 00:00 [accepted]\n",
      "PHST- 2021/05/24 08:05 [entrez]\n",
      "PHST- 2021/05/25 06:00 [pubmed]\n",
      "PHST- 2021/05/25 06:01 [medline]\n",
      "AID - 10.1111/eva.13210 [doi]\n",
      "AID - EVA13210 [pii]\n",
      "PST - epublish\n",
      "SO  - Evol Appl. 2021 Mar 10;14(5):1421-1435. doi: 10.1111/eva.13210. eCollection 2021 May.\n",
      "\n",
      "PMID- 33913659\n",
      "OWN - NLM\n",
      "STAT- Publisher\n",
      "LR  - 20210429\n",
      "IS  - 1827-1669 (Electronic)\n",
      "IS  - 0026-4806 (Linking)\n",
      "DP  - 2021 Apr 29\n",
      "TI  - The clock diet: a practical nutritional guide to manage obesity throughchrononutrition.\n",
      "LID - 10.23736/S0026-4806.21.07207-4 [doi]\n",
      "AB  - Chronobiology studies the biological rhythms or circadian cycles of livingorganisms and their adaptation to external changes. Biological rhythms can affecthormone production cycles such as sleep/wake, and nutrition/fasting, but thesefactors can also alter the circadian rhythm (CR). In recent years, numerousstudies have highlighted how feeding times and frequency can influence biologicalrhythms. Additionally, individuals' chronotype, working shifts, and food intakecan make a deep impact on people's tendency to develop obesity and metabolicdiseases. In this context, a single food and a specific combination of these, canalso affect the CR and fasting cycle and consequently body weight and viceversa. The purpose of the review is to propose practical nutritional recommendations to help in resynchronizing the circadian rhythm as a tool in weight control.\n",
      "FAU - Barrea, Luigi\n",
      "AU  - Barrea L\n",
      "AD  - Dipartimento di Scienze Umanistiche, Universita Telematica Pegaso, Naples, Italy - luigi.barrea@unipegaso.it.\n",
      "AD  - Endocrinology Unit, Department of Clinical Medicine and Surgery, Centro Italiano per la cura e il Benessere del paziente con Obesita (C.I.B.O), University MedicalSchool of Naples, Naples, Italy - luigi.barrea@unipegaso.it.\n",
      "FAU - Frias-Toral, Evelyn\n",
      "AU  - Frias-Toral E\n",
      "AD  - Research Committee, SOLCA Guayaquil, Guayaquil, Ecuador.\n",
      "AD  - Palliative Care Residency, Universidad Catolica Santiago de Guayaquil, Guayaquil,Ecuador.\n",
      "FAU - Aprano, Sara\n",
      "AU  - Aprano S\n",
      "AD  - Endocrinology Unit, Department of Clinical Medicine and Surgery, Centro Italiano per la cura e il Benessere del paziente con Obesita (C.I.B.O), University MedicalSchool of Naples, Naples, Italy.\n",
      "AD  - Unit of Endocrinology, Dipartimento di Medicina Clinica e Chirurgia, Federico II University Medical School of Naples, Naples, Italy.\n",
      "FAU - Castellucci, Bianca\n",
      "AU  - Castellucci B\n",
      "AD  - Endocrinology Unit, Department of Clinical Medicine and Surgery, Centro Italiano per la cura e il Benessere del paziente con Obesita (C.I.B.O), University MedicalSchool of Naples, Naples, Italy.\n",
      "AD  - Unit of Endocrinology, Dipartimento di Medicina Clinica e Chirurgia, Federico II University Medical School of Naples, Naples, Italy.\n",
      "FAU - Pugliese, Gabriella\n",
      "AU  - Pugliese G\n",
      "AD  - Endocrinology Unit, Department of Clinical Medicine and Surgery, Centro Italiano per la cura e il Benessere del paziente con Obesita (C.I.B.O), University MedicalSchool of Naples, Naples, Italy.\n",
      "AD  - Unit of Endocrinology, Dipartimento di Medicina Clinica e Chirurgia, Federico II University Medical School of Naples, Naples, Italy.\n",
      "FAU - Rodriguez-Veintimilla, Dolores\n",
      "AU  - Rodriguez-Veintimilla D\n",
      "AD  - Clinical Nutrition and Dietetics Service, SOLCA Guayaquil, Guayaquil, Ecuador.\n",
      "FAU - Vitale, Giovanni\n",
      "AU  - Vitale G\n",
      "AD  - Laboratory of Geriatric and Oncologic Neuroendocrinology Research, IstitutoAuxologico Italiano IRCCS, Cusano Milanino, Milan, Italy.\n",
      "AD  - Department of Medical Biotechnologies and Translational Medicine, University ofMilan, Milan, Italy.\n",
      "FAU - Gentilini, Davide\n",
      "AU  - Gentilini D\n",
      "AD  - Department of Brain and Behavioral Sciences, University of Pavia, Pavia, Italy.\n",
      "AD  - Bioinformatics and Statistical Genomics Unit, Istituto Auxologico Italiano IRCCS,Cusano Milanino, Milan, Italy.\n",
      "FAU - Colao, Annamaria\n",
      "AU  - Colao A\n",
      "AD  - Endocrinology Unit, Department of Clinical Medicine and Surgery, Centro Italiano per la cura e il Benessere del paziente con Obesita (C.I.B.O), University MedicalSchool of Naples, Naples, Italy.\n",
      "AD  - Unit of Endocrinology, Dipartimento di Medicina Clinica e Chirurgia, Federico II University Medical School of Naples, Naples, Italy.\n",
      "AD  - Cattedra Unesco Educazione alla salute e allo sviluppo sostenibile, UniversityFederico II, Naples, Italy.\n",
      "FAU - Savastano, Silvia\n",
      "AU  - Savastano S\n",
      "AD  - Endocrinology Unit, Department of Clinical Medicine and Surgery, Centro Italiano per la cura e il Benessere del paziente con Obesita (C.I.B.O), University MedicalSchool of Naples, Naples, Italy.\n",
      "AD  - Unit of Endocrinology, Dipartimento di Medicina Clinica e Chirurgia, Federico II University Medical School of Naples, Naples, Italy.\n",
      "FAU - Muscogiuri, Giovanna\n",
      "AU  - Muscogiuri G\n",
      "AD  - Endocrinology Unit, Department of Clinical Medicine and Surgery, Centro Italiano per la cura e il Benessere del paziente con Obesita (C.I.B.O), University MedicalSchool of Naples, Naples, Italy.\n",
      "AD  - Unit of Endocrinology, Dipartimento di Medicina Clinica e Chirurgia, Federico II University Medical School of Naples, Naples, Italy.\n",
      "AD  - Cattedra Unesco Educazione alla salute e allo sviluppo sostenibile, UniversityFederico II, Naples, Italy.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20210429\n",
      "PL  - Italy\n",
      "TA  - Minerva Med\n",
      "JT  - Minerva medica\n",
      "JID - 0400732\n",
      "SB  - IM\n",
      "EDAT- 2021/04/30 06:00\n",
      "MHDA- 2021/04/30 06:00\n",
      "CRDT- 2021/04/29 08:56\n",
      "PHST- 2021/04/29 08:56 [entrez]\n",
      "PHST- 2021/04/30 06:00 [pubmed]\n",
      "PHST- 2021/04/30 06:00 [medline]\n",
      "AID - S0026-4806.21.07207-4 [pii]\n",
      "AID - 10.23736/S0026-4806.21.07207-4 [doi]\n",
      "PST - aheadofprint\n",
      "SO  - Minerva Med. 2021 Apr 29. pii: S0026-4806.21.07207-4. doi:10.23736/S0026-4806.21.07207-4.\n",
      "\n",
      "PMID- 33872206\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20220201\n",
      "LR  - 20220201\n",
      "IS  - 1476-1645 (Electronic)\n",
      "IS  - 0002-9637 (Linking)\n",
      "VI  - 104\n",
      "IP  - 6\n",
      "DP  - 2021 Apr 19\n",
      "TI  - Gut Microbiome Changes with Acute Diarrheal Disease in Urban Versus RuralSettings in Northern Ecuador.\n",
      "PG  - 2275-2285\n",
      "LID - 10.4269/ajtmh.20-0831 [doi]\n",
      "LID - tpmd200831 [pii]\n",
      "AB  - Previous studies have reported lower fecal bacterial diversity in urbanpopulations compared with those living in rural settings. However, most of these studies compare geographically distant populations from different countries andeven continents. The extent of differences in the gut microbiome in adjacentrural versus urban populations, and the role of such differences, if any, during enteric infections remain poorly understood. To provide new insights into theseissues, we sampled the gut microbiome of young children with and without acutediarrheal disease (ADD) living in rural and urban areas in northern Ecuador.Shotgun metagenomic analyses of non-ADD samples revealed small but significantdifferences in the abundance of microbial taxa, including a greater abundance of Prevotella and a lower abundance of Bacteroides and Alistipes in ruralpopulations. Greater and more significant shifts in taxon abundance, metabolicpathway abundance, and diversity were observed between ADD and non-ADD statuswhen comparing urban to rural sites (Welch's t-test, P < 0.05). Collectively our data show substantial functional, diversity, and taxonomic shifts in the gutmicrobiome of urban populations with ADD, supporting the idea that the microbiomeof rural populations may be more resilient to ADD episodes.\n",
      "FAU - Soto-Giron, Maria J\n",
      "AU  - Soto-Giron MJ\n",
      "AD  - 1School of Biological Sciences, Georgia Institute of Technology, Atlanta,Georgia.\n",
      "FAU - Pena-Gonzalez, Angela\n",
      "AU  - Pena-Gonzalez A\n",
      "AD  - 1School of Biological Sciences, Georgia Institute of Technology, Atlanta,Georgia.\n",
      "FAU - Hatt, Janet K\n",
      "AU  - Hatt JK\n",
      "AD  - 2School of Civil and Environmental Engineering, Georgia Institute of Technology, Atlanta, Georgia.\n",
      "FAU - Montero, Lorena\n",
      "AU  - Montero L\n",
      "AD  - 3Instituto de Microbiologia, Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Paez, Maritza\n",
      "AU  - Paez M\n",
      "AD  - 3Instituto de Microbiologia, Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Ortega, Estefania\n",
      "AU  - Ortega E\n",
      "AD  - 3Instituto de Microbiologia, Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Smith, Shanon\n",
      "AU  - Smith S\n",
      "AD  - 4Rollins School of Public Health, Emory University, Atlanta, Georgia.\n",
      "FAU - Cevallos, William\n",
      "AU  - Cevallos W\n",
      "AD  - 5Centro de Biomedicina, Universidad Central del Ecuador, Quito, Ecuador.\n",
      "FAU - Trueba, Gabriel\n",
      "AU  - Trueba G\n",
      "AD  - 3Instituto de Microbiologia, Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Konstantinidis, Konstantinos T\n",
      "AU  - Konstantinidis KT\n",
      "AD  - 1School of Biological Sciences, Georgia Institute of Technology, Atlanta,Georgia.\n",
      "AD  - 2School of Civil and Environmental Engineering, Georgia Institute of Technology, Atlanta, Georgia.\n",
      "FAU - Levy, Karen\n",
      "AU  - Levy K\n",
      "AD  - 6School of Public Health, University of Washington, Seattle, Washington.\n",
      "LA  - eng\n",
      "GR  - K01 AI103544/AI/NIAID NIH HHS/United States\n",
      "GR  - R01 AI137679/AI/NIAID NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210419\n",
      "PL  - United States\n",
      "TA  - Am J Trop Med Hyg\n",
      "JT  - The American journal of tropical medicine and hygiene\n",
      "JID - 0370507\n",
      "SB  - IM\n",
      "MH  - Acute Disease/epidemiology\n",
      "MH  - Adolescent\n",
      "MH  - Adult\n",
      "MH  - Aged\n",
      "MH  - Bacteria/*classification/*genetics/isolation & purification\n",
      "MH  - Child\n",
      "MH  - Child, Preschool\n",
      "MH  - Diarrhea/epidemiology/*microbiology\n",
      "MH  - Ecuador/epidemiology\n",
      "MH  - Feces/microbiology\n",
      "MH  - *Gastrointestinal Microbiome\n",
      "MH  - Humans\n",
      "MH  - Infant\n",
      "MH  - Infant, Newborn\n",
      "MH  - Metagenomics\n",
      "MH  - Middle Aged\n",
      "MH  - Rural Population/*statistics & numerical data\n",
      "MH  - Urban Population/*statistics & numerical data\n",
      "MH  - Young Adult\n",
      "PMC - PMC8176484\n",
      "EDAT- 2021/04/20 06:00\n",
      "MHDA- 2022/02/02 06:00\n",
      "CRDT- 2021/04/19 17:19\n",
      "PMCR- 2022/06/01 00:00\n",
      "PHST- 2020/07/09 00:00 [received]\n",
      "PHST- 2021/02/15 00:00 [accepted]\n",
      "PHST- 2022/06/01 00:00 [pmc-release]\n",
      "PHST- 2021/04/20 06:00 [pubmed]\n",
      "PHST- 2022/02/02 06:00 [medline]\n",
      "PHST- 2021/04/19 17:19 [entrez]\n",
      "AID - 10.4269/ajtmh.20-0831 [doi]\n",
      "AID - tpmd200831 [pii]\n",
      "PST - epublish\n",
      "SO  - Am J Trop Med Hyg. 2021 Apr 19;104(6):2275-2285. doi: 10.4269/ajtmh.20-0831.\n",
      "\n",
      "PMID- 33846529\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210804\n",
      "LR  - 20210804\n",
      "IS  - 2399-3642 (Electronic)\n",
      "IS  - 2399-3642 (Linking)\n",
      "VI  - 4\n",
      "IP  - 1\n",
      "DP  - 2021 Apr 12\n",
      "TI  - Population genomics and antimicrobial resistance dynamics of Escherichia coli in wastewater and river environments.\n",
      "PG  - 457\n",
      "LID - 10.1038/s42003-021-01949-x [doi]\n",
      "AB  - Aquatic environments are key niches for the emergence, evolution anddissemination of antimicrobial resistance. However, the population diversity and the genetic elements that drive the dynamics of resistant bacteria in differentaquatic environments are still largely unknown. The aim of this study was tounderstand the population genomics and evolutionary events of Escherichia coliresistant to clinically important antibiotics including aminoglycosides, inanthropogenic and natural water ecosystems. Here we show that less different E.coli sequence types (STs) are identified in wastewater than in rivers, albeitmore resistant to antibiotics, and with significantly more plasmids/cell (6.36 vs3.72). However, the genomic diversity within E. coli STs in both aquaticenvironments is similar. Wastewater environments favor the selection of conservedchromosomal structures associated with diverse flexible plasmids, unravelingpromiscuous interplasmidic resistance genes flux. On the contrary, the key driverfor river E. coli adaptation is a mutable chromosome along with few plasmid typesshared between diverse STs harboring a limited resistance gene content.\n",
      "FAU - Delgado-Blas, Jose F\n",
      "AU  - Delgado-Blas JF\n",
      "AUID- ORCID: 0000-0001-6447-7113\n",
      "AD  - Antimicrobial Resistance Unit (ARU), Animal Health Department, Faculty ofVeterinary Medicine and VISAVET Health Surveillance Centre, ComplutenseUniversity of Madrid, Madrid, Spain.\n",
      "FAU - Ovejero, Cristina M\n",
      "AU  - Ovejero CM\n",
      "AD  - Antimicrobial Resistance Unit (ARU), Animal Health Department, Faculty ofVeterinary Medicine and VISAVET Health Surveillance Centre, ComplutenseUniversity of Madrid, Madrid, Spain.\n",
      "FAU - David, Sophia\n",
      "AU  - David S\n",
      "AD  - Centre for Genomic Pathogen Surveillance (CGPS), Wellcome Sanger Institute,Hinxton, UK.\n",
      "FAU - Montero, Natalia\n",
      "AU  - Montero N\n",
      "AD  - Antimicrobial Resistance Unit (ARU), Animal Health Department, Faculty ofVeterinary Medicine and VISAVET Health Surveillance Centre, ComplutenseUniversity of Madrid, Madrid, Spain.\n",
      "FAU - Calero-Caceres, William\n",
      "AU  - Calero-Caceres W\n",
      "AUID- ORCID: 0000-0003-3819-9206\n",
      "AD  - Department of Genetics, Microbiology and Statistics, Faculty of Biology,University of Barcelona, Barcelona, Spain.\n",
      "AD  - UTA RAM One Health, Faculty of Food Science, Engineering and Biotechnology,Technical University of Ambato, Ambato, Ecuador.\n",
      "FAU - Garcillan-Barcia, M Pilar\n",
      "AU  - Garcillan-Barcia MP\n",
      "AUID- ORCID: 0000-0001-7058-5428\n",
      "AD  - Institute of Biomedicine and Biotechnology (IBBTEC), CSIC, University ofCantabria, Santander, Spain.\n",
      "FAU - de la Cruz, Fernando\n",
      "AU  - de la Cruz F\n",
      "AUID- ORCID: 0000-0003-4758-6857\n",
      "AD  - Institute of Biomedicine and Biotechnology (IBBTEC), CSIC, University ofCantabria, Santander, Spain.\n",
      "FAU - Muniesa, Maite\n",
      "AU  - Muniesa M\n",
      "AUID- ORCID: 0000-0001-5549-1394\n",
      "AD  - Department of Genetics, Microbiology and Statistics, Faculty of Biology,University of Barcelona, Barcelona, Spain.\n",
      "FAU - Aanensen, David M\n",
      "AU  - Aanensen DM\n",
      "AD  - Centre for Genomic Pathogen Surveillance (CGPS), Wellcome Sanger Institute,Hinxton, UK.\n",
      "FAU - Gonzalez-Zorn, Bruno\n",
      "AU  - Gonzalez-Zorn B\n",
      "AUID- ORCID: 0000-0002-6958-4716\n",
      "AD  - Antimicrobial Resistance Unit (ARU), Animal Health Department, Faculty ofVeterinary Medicine and VISAVET Health Surveillance Centre, ComplutenseUniversity of Madrid, Madrid, Spain. bgzorn@ucm.es.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210412\n",
      "PL  - England\n",
      "TA  - Commun Biol\n",
      "JT  - Communications biology\n",
      "JID - 101719179\n",
      "RN  - 0 (Anti-Bacterial Agents)\n",
      "RN  - 0 (Waste Water)\n",
      "SB  - IM\n",
      "MH  - Anti-Bacterial Agents/*pharmacology\n",
      "MH  - *Drug Resistance, Bacterial\n",
      "MH  - Escherichia coli/*genetics\n",
      "MH  - *Genetic Variation\n",
      "MH  - *Genome, Bacterial\n",
      "MH  - Metagenomics\n",
      "MH  - Plasmids/physiology\n",
      "MH  - Rivers/*microbiology\n",
      "MH  - Spain\n",
      "MH  - Waste Water/*microbiology\n",
      "PMC - PMC8041779\n",
      "EDAT- 2021/04/14 06:00\n",
      "MHDA- 2021/08/05 06:00\n",
      "CRDT- 2021/04/13 06:22\n",
      "PHST- 2020/08/05 00:00 [received]\n",
      "PHST- 2021/03/02 00:00 [accepted]\n",
      "PHST- 2021/04/13 06:22 [entrez]\n",
      "PHST- 2021/04/14 06:00 [pubmed]\n",
      "PHST- 2021/08/05 06:00 [medline]\n",
      "AID - 10.1038/s42003-021-01949-x [doi]\n",
      "AID - 10.1038/s42003-021-01949-x [pii]\n",
      "PST - epublish\n",
      "SO  - Commun Biol. 2021 Apr 12;4(1):457. doi: 10.1038/s42003-021-01949-x.\n",
      "\n",
      "PMID- 33845910\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210506\n",
      "LR  - 20210506\n",
      "IS  - 2049-2618 (Electronic)\n",
      "IS  - 2049-2618 (Linking)\n",
      "VI  - 9\n",
      "IP  - 1\n",
      "DP  - 2021 Apr 12\n",
      "TI  - Microbial community characterization of shrimp survivors to AHPND challenge test treated with an effective shrimp probiotic (Vibrio diabolicus).\n",
      "PG  - 88\n",
      "LID - 10.1186/s40168-021-01043-8 [doi]\n",
      "AB  - BACKGROUND: Acute hepatopancreatic necrosis disease (AHPND) is an importantshrimp bacterial disease caused by some Vibrio species. The severity of theimpact of this disease on aquaculture worldwide has made it necessary to develop alternatives to prophylactic antibiotics use, such as the application ofprobiotics. To assess the potential to use probiotics in order to limit thedetrimental effects of AHNPD, we evaluated the effect of the ILI strain, a Vibriosp. bacterium and efficient shrimp probiotic, using metabarcoding (16S rRNA gene)on the gastrointestinal microbiota of shrimp after being challenged withAHPND-causing V. parahaemolyticus. RESULTS: We showed how the gastrointestinalmicrobiome of shrimp varied between healthy and infected organisms. Nevertheless,a challenge of working with AHPND-causing Vibrio pathogens and Vibrio-relatedbacteria as probiotics is the potential risk of the probiotic strain becomingpathogenic. Consequently, we evaluated whether ILI strain can acquire the plasmidpV-AHPND via horizontal transfer and further cause the disease in shrimp.Conjugation assays were performed resulting in a high frequency (70%) of coloniesharboring the pv-AHPND. However, no shrimp mortality was observed whentransconjugant colonies of the ILI strain were used in a challenge test usinghealthy shrimp. We sequenced the genome of the ILI strain and performedcomparative genomics analyses using AHPND and non-AHPND Vibrio isolates. Usingavailable phylogenetic and phylogenomics analyses, we reclassified the ILI strainas Vibrio diabolicus. In summary, this work represents an effort to study therole that probiotics play in the normal gastrointestinal shrimp microbiome and inAHPND-infected shrimp, showing that the ILI probiotic was able to controlpathogenic bacterial populations in the host's gastrointestinal tract andstimulate the shrimp's survival. The identification of probiotic bacterialspecies that are effective in the host's colonization is important to promoteanimal health and prevent disease. CONCLUSIONS: This study describes probioticbacteria capable of controlling pathogenic populations of bacteria in the shrimp gastrointestinal tract. Our work provides new insights into the complex dynamics between shrimp and the changes in the microbiota. It also addresses the practicalapplication of probiotics to solve problems with pathogens that cause highmortality-rate in shrimp farming around the world. Video Abstract.\n",
      "FAU - Restrepo, Leda\n",
      "AU  - Restrepo L\n",
      "AD  - Department of Biological Sciences, Universidad de los Andes, Bogota, Colombia.\n",
      "AD  - Max Planck Tandem Group in Computational Biology, Universidad de los Andes,Bogota, Colombia.\n",
      "AD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura eInvestigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,P.O. Box 09-01-5863, Guayaquil, Ecuador.\n",
      "FAU - Dominguez-Borbor, Cristobal\n",
      "AU  - Dominguez-Borbor C\n",
      "AD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura eInvestigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,P.O. Box 09-01-5863, Guayaquil, Ecuador.\n",
      "FAU - Bajana, Leandro\n",
      "AU  - Bajana L\n",
      "AD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura eInvestigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,P.O. Box 09-01-5863, Guayaquil, Ecuador.\n",
      "FAU - Betancourt, Irma\n",
      "AU  - Betancourt I\n",
      "AD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura eInvestigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,P.O. Box 09-01-5863, Guayaquil, Ecuador.\n",
      "FAU - Rodriguez, Jenny\n",
      "AU  - Rodriguez J\n",
      "AD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura eInvestigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,P.O. Box 09-01-5863, Guayaquil, Ecuador.\n",
      "AD  - Escuela Superior Politecnica del Litoral, ESPOL, Facultad de Ciencias de la Vida,FCV, Campus Gustavo Galindo Km 30.5 Via Perimetral, P.O. Box 09-01-5863,Guayaquil, Ecuador.\n",
      "FAU - Bayot, Bonny\n",
      "AU  - Bayot B\n",
      "AD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura eInvestigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,P.O. Box 09-01-5863, Guayaquil, Ecuador. bbayot@espol.edu.ec.\n",
      "AD  - Escuela Superior Politecnica del Litoral, ESPOL, Facultad de Ingenieria Maritima y Ciencias del Mar, FIMCM, Campus Gustavo Galindo Km 30.5 Via Perimetral, P.O.Box 09-01-5863, Guayaquil, Ecuador. bbayot@espol.edu.ec.\n",
      "FAU - Reyes, Alejandro\n",
      "AU  - Reyes A\n",
      "AUID- ORCID: 0000-0003-2907-3265\n",
      "AD  - Department of Biological Sciences, Universidad de los Andes, Bogota, Colombia.a.reyes@uniandes.edu.co.\n",
      "AD  - Max Planck Tandem Group in Computational Biology, Universidad de los Andes,Bogota, Colombia. a.reyes@uniandes.edu.co.\n",
      "AD  - Center for Genome Sciences and Systems Biology, Department of Pathology andImmunology, Washington University in Saint Louis, Saint Louis, MO, USA.a.reyes@uniandes.edu.co.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Video-Audio Media\n",
      "DEP - 20210412\n",
      "PL  - England\n",
      "TA  - Microbiome\n",
      "JT  - Microbiome\n",
      "JID - 101615147\n",
      "RN  - 0 (RNA, Ribosomal, 16S)\n",
      "RN  - Vibrio diabolicus\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Humans\n",
      "MH  - *Microbiota\n",
      "MH  - Necrosis\n",
      "MH  - *Penaeidae\n",
      "MH  - Phylogeny\n",
      "MH  - *Probiotics\n",
      "MH  - RNA, Ribosomal, 16S/genetics\n",
      "MH  - Survivors\n",
      "MH  - Vibrio\n",
      "MH  - *Vibrio parahaemolyticus/genetics\n",
      "PMC - PMC8042889\n",
      "OTO - NOTNLM\n",
      "OT  - *Acute hepatopancreatic necrosis disease (AHPND)\n",
      "OT  - *Comparative genomics\n",
      "OT  - *Horizontal gene transfer (HGT)\n",
      "OT  - *Metagenomics\n",
      "OT  - *Microbiome\n",
      "OT  - *Probiotics\n",
      "OT  - *Vibrio diabolicus\n",
      "EDAT- 2021/04/14 06:00\n",
      "MHDA- 2021/05/07 06:00\n",
      "CRDT- 2021/04/13 05:38\n",
      "PHST- 2020/12/07 00:00 [received]\n",
      "PHST- 2021/03/05 00:00 [accepted]\n",
      "PHST- 2021/04/13 05:38 [entrez]\n",
      "PHST- 2021/04/14 06:00 [pubmed]\n",
      "PHST- 2021/05/07 06:00 [medline]\n",
      "AID - 10.1186/s40168-021-01043-8 [doi]\n",
      "AID - 10.1186/s40168-021-01043-8 [pii]\n",
      "PST - epublish\n",
      "SO  - Microbiome. 2021 Apr 12;9(1):88. doi: 10.1186/s40168-021-01043-8.\n",
      "\n",
      "PMID- 33822137\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210920\n",
      "LR  - 20210920\n",
      "IS  - 1537-1719 (Electronic)\n",
      "IS  - 0737-4038 (Linking)\n",
      "VI  - 38\n",
      "IP  - 8\n",
      "DP  - 2021 Jul 29\n",
      "TI  - Migration through a Major Andean Ecogeographic Disruption as a Driver of Genetic and Phenotypic Diversity in a Wild Tomato Species.\n",
      "PG  - 3202-3219\n",
      "LID - 10.1093/molbev/msab092 [doi]\n",
      "AB  - Evolutionary dynamics at the population level play a central role in creating thediversity of life on our planet. In this study, we sought to understand theorigins of such population-level variation in mating systems and defensiveacylsugar chemistry in Solanum habrochaites-a wild tomato species found indiverse Andean habitats in Ecuador and Peru. UsingRestriction-site-Associated-DNA-Sequencing (RAD-seq) of 50 S. habrochaitesaccessions, we identified eight population clusters generated via isolation andhybridization dynamics of 4-6 ancestral populations. Detailed characterization ofmating systems of these clusters revealed emergence of multiple self-compatible(SC) groups from progenitor self-incompatible populations in the northern part ofthe species range. Emergence of these SC groups was also associated with fixationof deleterious alleles inactivating acylsugar acetylation. TheAmotape-Huancabamba Zone-a geographical landmark in the Andes with high endemism and isolated microhabitats-was identified as a major driver of differentiation inthe northern species range, whereas large geographical distances contributed topopulation structure and evolution of a novel SC group in the central andsouthern parts of the range, where the species was also inferred to haveoriginated. Findings presented here highlight the role of the diverseecogeography of Peru and Ecuador in generating population differentiation, andenhance our understanding of the microevolutionary processes that createbiological diversity.\n",
      "CI  - (c) The Author(s) 2021. Published by Oxford University Press on behalf of theSociety for Molecular Biology and Evolution.\n",
      "FAU - Landis, Jacob B\n",
      "AU  - Landis JB\n",
      "AD  - Department of Botany and Plant Sciences, University of California, Riverside,Riverside, CA, USA.\n",
      "AD  - Plant Biology Section, School of Integrative Plant Science, Cornell University,Ithaca, NY, USA.\n",
      "FAU - Miller, Christopher M\n",
      "AU  - Miller CM\n",
      "AD  - Department of Biology, Colorado State University, Fort Collins, CO, USA.\n",
      "FAU - Broz, Amanda K\n",
      "AU  - Broz AK\n",
      "AD  - Department of Biology, Colorado State University, Fort Collins, CO, USA.\n",
      "FAU - Bennett, Alexandra A\n",
      "AU  - Bennett AA\n",
      "AD  - Plant Biology Section, School of Integrative Plant Science, Cornell University,Ithaca, NY, USA.\n",
      "FAU - Carrasquilla-Garcia, Noelia\n",
      "AU  - Carrasquilla-Garcia N\n",
      "AD  - Department of Plant Pathology, University of California, Davis, Davis, CA, USA.\n",
      "FAU - Cook, Douglas R\n",
      "AU  - Cook DR\n",
      "AD  - Department of Plant Pathology, University of California, Davis, Davis, CA, USA.\n",
      "FAU - Last, Robert L\n",
      "AU  - Last RL\n",
      "AD  - Department of Biochemistry and Molecular Biology, Michigan State University, EastLansing, MI, USA.\n",
      "AD  - Department of Plant Biology, Michigan State University, East Lansing, MI, USA.\n",
      "FAU - Bedinger, Patricia A\n",
      "AU  - Bedinger PA\n",
      "AD  - Department of Biology, Colorado State University, Fort Collins, CO, USA.\n",
      "FAU - Moghe, Gaurav D\n",
      "AU  - Moghe GD\n",
      "AUID- ORCID: 0000-0002-8761-064X\n",
      "AD  - Plant Biology Section, School of Integrative Plant Science, Cornell University,Ithaca, NY, USA.\n",
      "AD  - Department of Biochemistry and Molecular Biology, Michigan State University, EastLansing, MI, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "PL  - United States\n",
      "TA  - Mol Biol Evol\n",
      "JT  - Molecular biology and evolution\n",
      "JID - 8501455\n",
      "SB  - IM\n",
      "MH  - Acetylation\n",
      "MH  - Ecuador\n",
      "MH  - *Gene Flow\n",
      "MH  - Lycopersicon esculentum/*genetics/metabolism\n",
      "MH  - Peru\n",
      "MH  - Phylogeography\n",
      "MH  - Self-Fertilization\n",
      "MH  - Self-Incompatibility in Flowering Plants/*genetics\n",
      "MH  - Solanum/*genetics/metabolism\n",
      "PMC - PMC8321546\n",
      "OTO - NOTNLM\n",
      "OT  - *RAD-seq\n",
      "OT  - *biodiversity\n",
      "OT  - *mating systems\n",
      "OT  - *metabolic evolution\n",
      "OT  - *plant evolution\n",
      "OT  - *population genomics\n",
      "EDAT- 2021/04/07 06:00\n",
      "MHDA- 2021/09/21 06:00\n",
      "CRDT- 2021/04/06 12:37\n",
      "PHST- 2021/04/07 06:00 [pubmed]\n",
      "PHST- 2021/09/21 06:00 [medline]\n",
      "PHST- 2021/04/06 12:37 [entrez]\n",
      "AID - 6209742 [pii]\n",
      "AID - 10.1093/molbev/msab092 [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Biol Evol. 2021 Jul 29;38(8):3202-3219. doi: 10.1093/molbev/msab092.\n",
      "\n",
      "PMID- 33747978\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210616\n",
      "LR  - 20210617\n",
      "IS  - 2235-2988 (Electronic)\n",
      "IS  - 2235-2988 (Linking)\n",
      "VI  - 11\n",
      "DP  - 2021\n",
      "TI  - Repeat-Driven Generation of Antigenic Diversity in a Major Human Pathogen,Trypanosoma cruzi.\n",
      "PG  - 614665\n",
      "LID - 10.3389/fcimb.2021.614665 [doi]\n",
      "AB  - Trypanosoma cruzi, a zoonotic kinetoplastid protozoan parasite, is the causative agent of American trypanosomiasis (Chagas disease). Having a very plastic,repetitive and complex genome, the parasite displays a highly diverse repertoire of surface molecules, with pivotal roles in cell invasion, immune evasion andpathogenesis. Before 2016, the complexity of the genomic regions containing thesegenes impaired the assembly of a genome at chromosomal level, making itimpossible to study the structure and function of the several thousand repetitivegenes encoding the surface molecules of the parasite. We here describe the genomeassembly of the Sylvio X10/1 genome sequence, which since 2016 has been used as areference genome sequence for T. cruzi clade I (TcI), produced using highcoverage PacBio single-molecule sequencing. It was used to analyze deep Illumina sequence data from 34 T. cruzi TcI isolates and clones from different geographic locations, sample sources and clinical outcomes. Resolution of the surfacemolecule gene distribution showed the unusual duality in the organization of the parasite genome, a synteny of the core genomic region with related protozoaflanked by unique and highly plastic multigene family clusters encoding surfaceantigens. The presence of abundant interspersed retrotransposons in thesemultigene family clusters suggests that these elements are involved in arecombination mechanism for the generation of antigenic variation and evasion of the host immune response on these TcI strains. The comparative genomic analysisof the cohort of TcI strains revealed multiple cases of such recombination eventsinvolving surface molecule genes and has provided new insights into T. cruzipopulation structure.\n",
      "CI  - Copyright (c) 2021 Talavera-Lopez, Messenger, Lewis, Yeo, Reis-Cunha, Matos,Bartholomeu, Calzada, Saldana, Ramirez, Guhl, Ocana-Mayorga, Costales, Gorchakov,Jones, Nolan, Teixeira, Carrasco, Bottazzi, Hotez, Murray, Grijalva, Burleigh,Grisard, Miles and Andersson.\n",
      "FAU - Talavera-Lopez, Carlos\n",
      "AU  - Talavera-Lopez C\n",
      "AD  - Department of Cell and Molecular Biology, Karolinska Institutet, Stockholm,Sweden.\n",
      "AD  - European Bioinformatics Institute, Wellcome Sanger Institute, Hinxton, UnitedKingdom.\n",
      "FAU - Messenger, Louisa A\n",
      "AU  - Messenger LA\n",
      "AD  - Faculty of Infectious and Tropical Diseases, London School of Hygiene andTropical Medicine, London, United Kingdom.\n",
      "FAU - Lewis, Michael D\n",
      "AU  - Lewis MD\n",
      "AD  - Faculty of Infectious and Tropical Diseases, London School of Hygiene andTropical Medicine, London, United Kingdom.\n",
      "FAU - Yeo, Matthew\n",
      "AU  - Yeo M\n",
      "AD  - Faculty of Infectious and Tropical Diseases, London School of Hygiene andTropical Medicine, London, United Kingdom.\n",
      "FAU - Reis-Cunha, Joao Luis\n",
      "AU  - Reis-Cunha JL\n",
      "AD  - Departamento de Parasitologia, Universidade Federal de Minas Gerais, BeloHorizonte, Brazil.\n",
      "FAU - Matos, Gabriel Machado\n",
      "AU  - Matos GM\n",
      "AD  - Departamento de Biologia Celular, Embriologia e Genetica, Universidade FederalSanta Catarina, Florianopolis, Brazil.\n",
      "FAU - Bartholomeu, Daniella C\n",
      "AU  - Bartholomeu DC\n",
      "AD  - Departamento de Parasitologia, Universidade Federal de Minas Gerais, BeloHorizonte, Brazil.\n",
      "FAU - Calzada, Jose E\n",
      "AU  - Calzada JE\n",
      "AD  - Departamento de Parasitologia, Instituto Conmemorativo Gorgas de Estudios de laSalud, Ciudad de Panama, Panama.\n",
      "FAU - Saldana, Azael\n",
      "AU  - Saldana A\n",
      "AD  - Departamento de Parasitologia, Instituto Conmemorativo Gorgas de Estudios de laSalud, Ciudad de Panama, Panama.\n",
      "FAU - Ramirez, Juan David\n",
      "AU  - Ramirez JD\n",
      "AD  - Grupo de Investigaciones Microbiologicas-UR (GIMUR), Departamento de Biologia,Facultad de Ciencias Naturales, Universidad del Rosario, Bogota, Colombia.\n",
      "FAU - Guhl, Felipe\n",
      "AU  - Guhl F\n",
      "AD  - Grupo de Investigaciones en Microbiologia y Parasitologia Tropical (CIMPAT),Tropical Parasitology Research Center, Universidad de Los Andes, Bogota,Colombia.\n",
      "FAU - Ocana-Mayorga, Sofia\n",
      "AU  - Ocana-Mayorga S\n",
      "AD  - Centro de Investigacion para la Salud en America Latina (CISeAL), Escuela deCiencias Biologicas, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Costales, Jaime A\n",
      "AU  - Costales JA\n",
      "AD  - Centro de Investigacion para la Salud en America Latina (CISeAL), Escuela deCiencias Biologicas, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Gorchakov, Rodion\n",
      "AU  - Gorchakov R\n",
      "AD  - Sabin Vaccine Institute and Texas Children's Hospital Center for VaccineDevelopment, National School of Tropical Medicine, Department of Pediatrics -Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\n",
      "FAU - Jones, Kathryn\n",
      "AU  - Jones K\n",
      "AD  - Sabin Vaccine Institute and Texas Children's Hospital Center for VaccineDevelopment, National School of Tropical Medicine, Department of Pediatrics -Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\n",
      "FAU - Nolan, Melissa S\n",
      "AU  - Nolan MS\n",
      "AD  - Sabin Vaccine Institute and Texas Children's Hospital Center for VaccineDevelopment, National School of Tropical Medicine, Department of Pediatrics -Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\n",
      "FAU - Teixeira, Santuza M R\n",
      "AU  - Teixeira SMR\n",
      "AD  - Departamento de Bioquimica e Imunologia, Universidade Federal de Minas Gerais,Belo Horizonte, Brazil.\n",
      "FAU - Carrasco, Hernan Jose\n",
      "AU  - Carrasco HJ\n",
      "AD  - Laboratorio de Biologia Molecular de Protozoarios, Instituto de MedicinaTropical, Facultad de Medicina, Universidad Central de Venezuela, Caracas,Venezuela.\n",
      "FAU - Bottazzi, Maria Elena\n",
      "AU  - Bottazzi ME\n",
      "AD  - Sabin Vaccine Institute and Texas Children's Hospital Center for VaccineDevelopment, National School of Tropical Medicine, Department of Pediatrics -Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\n",
      "FAU - Hotez, Peter J\n",
      "AU  - Hotez PJ\n",
      "AD  - Sabin Vaccine Institute and Texas Children's Hospital Center for VaccineDevelopment, National School of Tropical Medicine, Department of Pediatrics -Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\n",
      "FAU - Murray, Kristy O\n",
      "AU  - Murray KO\n",
      "AD  - Sabin Vaccine Institute and Texas Children's Hospital Center for VaccineDevelopment, National School of Tropical Medicine, Department of Pediatrics -Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\n",
      "FAU - Grijalva, Mario J\n",
      "AU  - Grijalva MJ\n",
      "AD  - Centro de Investigacion para la Salud en America Latina (CISeAL), Escuela deCiencias Biologicas, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\n",
      "AD  - Department of Biomedical Sciences, Heritage College of Osteopathic Medicine,Infectious and Tropical Disease Institute, Ohio University, Athens, OH, UnitedStates.\n",
      "FAU - Burleigh, Barbara\n",
      "AU  - Burleigh B\n",
      "AD  - Department of Immunology and Infectious Diseases, T.H. Chan School of PublicHealth, Harvard University, Boston, MA, United States.\n",
      "FAU - Grisard, Edmundo C\n",
      "AU  - Grisard EC\n",
      "AD  - Departamento de Microbiologia, Imunologia e Parasitologia, Universidade FederalSanta Catarina, Florianopolis, Brazil.\n",
      "FAU - Miles, Michael A\n",
      "AU  - Miles MA\n",
      "AD  - Faculty of Infectious and Tropical Diseases, London School of Hygiene andTropical Medicine, London, United Kingdom.\n",
      "FAU - Andersson, Bjorn\n",
      "AU  - Andersson B\n",
      "AD  - Department of Cell and Molecular Biology, Karolinska Institutet, Stockholm,Sweden.\n",
      "LA  - eng\n",
      "GR  - R01 AI107028/AI/NIAID NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210303\n",
      "PL  - Switzerland\n",
      "TA  - Front Cell Infect Microbiol\n",
      "JT  - Frontiers in cellular and infection microbiology\n",
      "JID - 101585359\n",
      "SB  - IM\n",
      "MH  - *Antigenic Variation\n",
      "MH  - Multigene Family\n",
      "MH  - Synteny\n",
      "MH  - *Trypanosoma cruzi/genetics\n",
      "PMC - PMC7966520\n",
      "OTO - NOTNLM\n",
      "OT  - *Trypanosoma cruzi\n",
      "OT  - *antigenic variation\n",
      "OT  - *genome sequence\n",
      "OT  - *microbial genomics\n",
      "OT  - *parasitology\n",
      "OT  - *pathology of infectious diseases\n",
      "OT  - *population genetics\n",
      "OT  - *tropical medicine\n",
      "COIS- The authors declare that the research was conducted in the absence of anycommercial or financial relationships that could be construed as a potentialconflict of interest.\n",
      "EDAT- 2021/03/23 06:00\n",
      "MHDA- 2021/06/17 06:00\n",
      "CRDT- 2021/03/22 08:20\n",
      "PHST- 2020/10/06 00:00 [received]\n",
      "PHST- 2021/01/22 00:00 [accepted]\n",
      "PHST- 2021/03/22 08:20 [entrez]\n",
      "PHST- 2021/03/23 06:00 [pubmed]\n",
      "PHST- 2021/06/17 06:00 [medline]\n",
      "AID - 10.3389/fcimb.2021.614665 [doi]\n",
      "PST - epublish\n",
      "SO  - Front Cell Infect Microbiol. 2021 Mar 3;11:614665. doi:10.3389/fcimb.2021.614665. eCollection 2021.\n",
      "\n",
      "PMID- 33742029\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "DCOM- 20210914\n",
      "LR  - 20210914\n",
      "IS  - 2045-2322 (Electronic)\n",
      "IS  - 2045-2322 (Linking)\n",
      "VI  - 11\n",
      "IP  - 1\n",
      "DP  - 2021 Mar 19\n",
      "TI  - Microplastic pollution in seawater and marine organisms across the TropicalEastern Pacific and Galapagos.\n",
      "PG  - 6424\n",
      "LID - 10.1038/s41598-021-85939-3 [doi]\n",
      "AB  - Detection of plastic debris degrading into micro particles across all oceanicenvironments and inside of marine organisms is no longer surprising news.Microplastic contamination now appears as one of the world's environmental mainconcerns. To determine the levels of microplastic pollution at sea, water sampleswere collected across a 4000 km-trajectory in the Tropical Eastern Pacific andthe Galapagos archipelago, covering an area of 453,000 square kilometres.Furthermore, 240 specimens of 16 different species of fish, squid, and shrimp,all of human consumption, were collected along the continental coast.Microplastic particles were found in 100% of the water samples and marineorganisms. Microplastic particles ranging from 150 to 500 microm in size were themost predominant. This is one of the first reports simultaneously detecting andquantifying microplastic particles abundance and their impact on marine organismsof this region.\n",
      "FAU - Alfaro-Nunez, Alonzo\n",
      "AU  - Alfaro-Nunez A\n",
      "AD  - Virus Research & Development Laboratory, Statens Serum Institut, Artillerivej 5, 2300, Kobenhavn S, Denmark. alonzoalfaro@gmail.com.\n",
      "AD  - Section for Evolutionary Genomics, GLOBE Institute, University of Copenhagen,Oster Farimagsgade 5, 1353, Copenhagen K, Denmark. alonzoalfaro@gmail.com.\n",
      "FAU - Astorga, Diana\n",
      "AU  - Astorga D\n",
      "AD  - Escuela de Ciencias Biologicas, Pontificia Universidad Catolica del Ecuador, Av. 12 de Octubre 1076, 17-01-2184, Apartado, Quito, Ecuador.\n",
      "FAU - Caceres-Farias, Lenin\n",
      "AU  - Caceres-Farias L\n",
      "AD  - Grupo de Investigacion en Sanidad Acuicola, Inocuidad y Salud Ambiental, Escuela de Acuicultura y Pesqueria, Facultad de Ciencias Veterinarias, UniversidadTecnica de Manabi, Ciudadela Universitaria, Leonidas Plaza EC131402, Bahia deCaraquez, Ecuador.\n",
      "AD  - AquaCEAL Corp. Urb, Las Palmeras, av. Capitan Byron Palacios y General Quisquis, #8 EC230101, Santo Domingo de los Colorados, Ecuador.\n",
      "FAU - Bastidas, Lisandra\n",
      "AU  - Bastidas L\n",
      "AD  - School of Biological Science and Engineering, Yachay Tech University, Urcuqui,Imbabura, Ecuador.\n",
      "FAU - Soto Villegas, Cynthia\n",
      "AU  - Soto Villegas C\n",
      "AD  - School of Biological Science and Engineering, Yachay Tech University, Urcuqui,Imbabura, Ecuador.\n",
      "FAU - Macay, Kewrin\n",
      "AU  - Macay K\n",
      "AD  - School of Biological Science and Engineering, Yachay Tech University, Urcuqui,Imbabura, Ecuador.\n",
      "FAU - Christensen, Jan H\n",
      "AU  - Christensen JH\n",
      "AD  - Department of Plant and Environmental Sciences, University of Copenhagen,Thorvaldsensvej 40, 1871, Frederiksberg, Denmark.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210319\n",
      "PL  - England\n",
      "TA  - Sci Rep\n",
      "JT  - Scientific reports\n",
      "JID - 101563288\n",
      "SB  - IM\n",
      "PMC - PMC7979831\n",
      "EDAT- 2021/03/21 06:00\n",
      "MHDA- 2021/03/21 06:01\n",
      "CRDT- 2021/03/20 06:26\n",
      "PHST- 2020/10/06 00:00 [received]\n",
      "PHST- 2021/03/09 00:00 [accepted]\n",
      "PHST- 2021/03/20 06:26 [entrez]\n",
      "PHST- 2021/03/21 06:00 [pubmed]\n",
      "PHST- 2021/03/21 06:01 [medline]\n",
      "AID - 10.1038/s41598-021-85939-3 [doi]\n",
      "AID - 10.1038/s41598-021-85939-3 [pii]\n",
      "PST - epublish\n",
      "SO  - Sci Rep. 2021 Mar 19;11(1):6424. doi: 10.1038/s41598-021-85939-3.\n",
      "\n",
      "PMID- 33668747\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20210413\n",
      "IS  - 2076-2615 (Print)\n",
      "IS  - 2076-2615 (Linking)\n",
      "VI  - 11\n",
      "IP  - 3\n",
      "DP  - 2021 Feb 25\n",
      "TI  - Genomic Analysis, Progress and Future Perspectives in Dairy Cattle Selection: AReview.\n",
      "LID - 599 [pii]\n",
      "LID - 10.3390/ani11030599 [doi]\n",
      "AB  - Genomics comprises a set of current and valuable technologies implemented asselection tools in dairy cattle commercial breeding programs. The intensiveprogeny testing for production and reproductive traits based on genomic breeding values (GEBVs) has been crucial to increasing dairy cattle productivity. Theknowledge of key genes and haplotypes, including their regulation mechanisms, as markers for productivity traits, may improve the strategies on the present andfuture for dairy cattle selection. Genome-wide association studies (GWAS) such asquantitative trait loci (QTL), single nucleotide polymorphisms (SNPs), orsingle-step genomic best linear unbiased prediction (ssGBLUP) methods havealready been included in global dairy programs for the estimation ofmarker-assisted selection-derived effects. The increase in genetic progress basedon genomic predicting accuracy has also contributed to the understanding ofgenetic effects in dairy cattle offspring. However, the crossing withininbred-lines critically increased homozygosis with accumulated negative effectsof inbreeding like a decline in reproductive performance. Thus, inaccurate-biasedestimations based on empirical-conventional models of dairy production systemsface an increased risk of providing suboptimal results derived from errors in theselection of candidates of high genetic merit-based just on low-heritabilityphenotypic traits. This extends the generation intervals and increases costs due to the significant reduction of genetic gains. The remarkable progress of genomicprediction increases the accurate selection of superior candidates. The scope of the present review is to summarize and discuss the advances and challenges ofgenomic tools for dairy cattle selection for optimizing breeding programs andcontrolling negative inbreeding depression effects on productivity andconsequently, achieving economic-effective advances in food productionefficiency. Particular attention is given to the potential genomicselection-derived results to facilitate precision management on modern dairyfarms, including an overview of novel genome editing methodologies asperspectives toward the future.\n",
      "FAU - Gutierrez-Reinoso, Miguel A\n",
      "AU  - Gutierrez-Reinoso MA\n",
      "AD  - Facultad de Ciencias Agropecuarias y Recursos Naturales, Carrera de MedicinaVeterinaria, Universidad Tecnica de Cotopaxi (UTC), Latacunga 05-0150, Ecuador.\n",
      "AD  - Laboratorio de Biotecnologia Animal, Departamento de Ciencia Animal, Facultad de Ciencias Veterinarias, Universidad de Concepcion (UdeC), Chillan 3780000, Chile.\n",
      "FAU - Aponte, Pedro M\n",
      "AU  - Aponte PM\n",
      "AD  - Colegio de Ciencias Biologicas y Ambientales (COCIBA), Universidad San Francisco de Quito (USFQ), Quito 170157, Ecuador.\n",
      "AD  - Campus Cumbaya, Instituto de Investigaciones en Biomedicina \"One-health\",Universidad San Francisco de Quito (USFQ), Quito 170157, Ecuador.\n",
      "FAU - Garcia-Herreros, Manuel\n",
      "AU  - Garcia-Herreros M\n",
      "AUID- ORCID: 0000-0002-0911-6689\n",
      "AD  - Instituto Nacional de Investigacao Agraria e Veterinaria (INIAV), 2005-048Santarem, Portugal.\n",
      "LA  - eng\n",
      "GR  - 20/10/UTC2020/Universidad Tecnica de Cotopaxi (UTC), Ecuador.\n",
      "GR  - 2020 - 21201280/Universidad San Francisco de Quito (USFQ), Ecuador; Secretariatof Higher Education, Science, Technology and Innovation (SENESCYT) of theGovernment of Ecuador; Agencia Nacional de Investigacion y Desarrollo (ANID);Programa de Becas / Doctorado Nacional\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "DEP - 20210225\n",
      "PL  - Switzerland\n",
      "TA  - Animals (Basel)\n",
      "JT  - Animals : an open access journal from MDPI\n",
      "JID - 101635614\n",
      "PMC - PMC7996307\n",
      "OTO - NOTNLM\n",
      "OT  - dairy cattle\n",
      "OT  - environment\n",
      "OT  - gene edition\n",
      "OT  - genomic analysis\n",
      "OT  - health\n",
      "OT  - linear types\n",
      "OT  - nutrition\n",
      "OT  - production\n",
      "OT  - reproduction\n",
      "OT  - welfare\n",
      "EDAT- 2021/03/07 06:00\n",
      "MHDA- 2021/03/07 06:01\n",
      "CRDT- 2021/03/06 01:02\n",
      "PHST- 2020/10/26 00:00 [received]\n",
      "PHST- 2020/11/23 00:00 [revised]\n",
      "PHST- 2020/11/24 00:00 [accepted]\n",
      "PHST- 2021/03/06 01:02 [entrez]\n",
      "PHST- 2021/03/07 06:00 [pubmed]\n",
      "PHST- 2021/03/07 06:01 [medline]\n",
      "AID - ani11030599 [pii]\n",
      "AID - 10.3390/ani11030599 [doi]\n",
      "PST - epublish\n",
      "SO  - Animals (Basel). 2021 Feb 25;11(3). pii: ani11030599. doi: 10.3390/ani11030599.\n",
      "\n",
      "PMID- 33652915\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210510\n",
      "LR  - 20210510\n",
      "IS  - 2072-6643 (Electronic)\n",
      "IS  - 2072-6643 (Linking)\n",
      "VI  - 13\n",
      "IP  - 3\n",
      "DP  - 2021 Feb 26\n",
      "TI  - Nutrition and Rheumatoid Arthritis in the 'Omics' Era.\n",
      "LID - 763 [pii]\n",
      "LID - 10.3390/nu13030763 [doi]\n",
      "AB  - Modern high-throughput 'omics' science tools (including genomics,transcriptomics, proteomics, metabolomics and microbiomics) are currently beingapplied to nutritional sciences to unravel the fundamental processes of healtheffects ascribed to particular nutrients in humans and to contribute to moreprecise nutritional advice. Diet and food components are key environmentalfactors that interact with the genome, transcriptome, proteome, metabolome andthe microbiota, and this life-long interplay defines health and diseases state ofthe individual. Rheumatoid arthritis (RA) is a chronic autoimmune diseasefeatured by a systemic immune-inflammatory response, in genetically susceptibleindividuals exposed to environmental triggers, including diet. In recent yearsincreasing evidences suggested that nutritional factors and gut microbiome have acentral role in RA risk and progression. The aim of this review is to summarizethe main and most recent applications of 'omics' technologies in human nutrition and in RA research, examining the possible influences of some nutrients andnutritional patterns on RA pathogenesis, following a nutrigenomics approach. The opportunities and challenges of novel 'omics technologies' in the exploration of new avenues in RA and nutritional research to prevent and manage RA will be also discussed.\n",
      "FAU - Cassotta, Manuela\n",
      "AU  - Cassotta M\n",
      "AD  - Research Group on Foods, Nutritional Biochemistry and Health, Universidad Europeadel Atlantico, 39011 Santander, Spain.\n",
      "FAU - Forbes-Hernandez, Tamara Y\n",
      "AU  - Forbes-Hernandez TY\n",
      "AD  - Nutrition and Food Science Group, Department of Analytical and Food Chemistry,CITACA, CACTI, University of Vigo, 36310 Vigo, Spain.\n",
      "FAU - Cianciosi, Danila\n",
      "AU  - Cianciosi D\n",
      "AUID- ORCID: 0000-0002-8781-3535\n",
      "AD  - Department of Clinical Sciences, Faculty of Medicine, Polytechnic University ofMarche, 60131 Ancona, Italy.\n",
      "FAU - Elexpuru Zabaleta, Maria\n",
      "AU  - Elexpuru Zabaleta M\n",
      "AUID- ORCID: 0000-0003-2997-1049\n",
      "AD  - Research Group on Foods, Nutritional Biochemistry and Health, Universidad Europeadel Atlantico, 39011 Santander, Spain.\n",
      "FAU - Sumalla Cano, Sandra\n",
      "AU  - Sumalla Cano S\n",
      "AD  - Research Group on Foods, Nutritional Biochemistry and Health, Universidad Europeadel Atlantico, 39011 Santander, Spain.\n",
      "FAU - Dominguez, Irma\n",
      "AU  - Dominguez I\n",
      "AD  - Research Group on Foods, Nutritional Biochemistry and Health, Universidad Europeadel Atlantico, 39011 Santander, Spain.\n",
      "FAU - Bullon, Beatriz\n",
      "AU  - Bullon B\n",
      "AD  - Department of Periodontology, Dental School, University of Sevilla, 41004Sevilla, Spain.\n",
      "FAU - Regolo, Lucia\n",
      "AU  - Regolo L\n",
      "AUID- ORCID: 0000-0001-9432-0408\n",
      "AD  - Department of Clinical Sciences, Faculty of Medicine, Polytechnic University ofMarche, 60131 Ancona, Italy.\n",
      "FAU - Alvarez-Suarez, Jose Miguel\n",
      "AU  - Alvarez-Suarez JM\n",
      "AUID- ORCID: 0000-0001-8509-1498\n",
      "AD  - AgroScience & Food Research Group, Universidad de Las Americas, Quito 170125,Ecuador.\n",
      "AD  - King Fahd Medical Research Center, King Abdulaziz University, Jedda 21589, Saudi Arabia.\n",
      "FAU - Giampieri, Francesca\n",
      "AU  - Giampieri F\n",
      "AD  - Department of Clinical Sciences, Faculty of Medicine, Polytechnic University ofMarche, 60131 Ancona, Italy.\n",
      "AD  - Department of Biochemistry, Faculty of Sciences, King Abdulaziz University,Jeddah 21589, Saudi Arabia.\n",
      "FAU - Battino, Maurizio\n",
      "AU  - Battino M\n",
      "AD  - Department of Clinical Sciences, Faculty of Medicine, Polytechnic University ofMarche, 60131 Ancona, Italy.\n",
      "AD  - International Research Center for Food Nutrition and Safety, Jiangsu University, Zhenjiang 212013, China.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "DEP - 20210226\n",
      "PL  - Switzerland\n",
      "TA  - Nutrients\n",
      "JT  - Nutrients\n",
      "JID - 101521595\n",
      "SB  - IM\n",
      "MH  - Arthritis, Rheumatoid/*etiology\n",
      "MH  - Computational Biology/*trends\n",
      "MH  - Gene Expression Profiling/trends\n",
      "MH  - Genomics/trends\n",
      "MH  - Humans\n",
      "MH  - Metabolomics/trends\n",
      "MH  - Microbiota\n",
      "MH  - Nutrigenomics/*trends\n",
      "MH  - Proteomics/trends\n",
      "PMC - PMC7996781\n",
      "OTO - NOTNLM\n",
      "OT  - diet\n",
      "OT  - metabolomics\n",
      "OT  - microbiome\n",
      "OT  - microbiomics\n",
      "OT  - nutrigenomics\n",
      "OT  - proteomics\n",
      "OT  - rheumatoid arthritis\n",
      "EDAT- 2021/03/04 06:00\n",
      "MHDA- 2021/05/11 06:00\n",
      "CRDT- 2021/03/03 01:01\n",
      "PHST- 2021/01/07 00:00 [received]\n",
      "PHST- 2021/02/16 00:00 [revised]\n",
      "PHST- 2021/02/24 00:00 [accepted]\n",
      "PHST- 2021/03/03 01:01 [entrez]\n",
      "PHST- 2021/03/04 06:00 [pubmed]\n",
      "PHST- 2021/05/11 06:00 [medline]\n",
      "AID - nu13030763 [pii]\n",
      "AID - 10.3390/nu13030763 [doi]\n",
      "PST - epublish\n",
      "SO  - Nutrients. 2021 Feb 26;13(3). pii: nu13030763. doi: 10.3390/nu13030763.\n",
      "\n",
      "PMID- 33574059\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210721\n",
      "LR  - 20210818\n",
      "IS  - 1091-6490 (Electronic)\n",
      "IS  - 0027-8424 (Linking)\n",
      "VI  - 118\n",
      "IP  - 7\n",
      "DP  - 2021 Feb 16\n",
      "TI  - The genomics of ecological flexibility, large brains, and long lives in capuchin monkeys revealed with fecalFACS.\n",
      "LID - e2010632118 [pii]\n",
      "LID - 10.1073/pnas.2010632118 [doi]\n",
      "AB  - Ecological flexibility, extended lifespans, and large brains have long intrigued evolutionary biologists, and comparative genomics offers an efficient andeffective tool for generating new insights into the evolution of such traits.Studies of capuchin monkeys are particularly well situated to shed light on theselective pressures and genetic underpinnings of local adaptation to diversehabitats, longevity, and brain development. Distributed widely across Central andSouth America, they are inventive and extractive foragers, known for theirsensorimotor intelligence. Capuchins have among the largest relative brain sizeof any monkey and a lifespan that exceeds 50 y, despite their small (3 to 5 kg)body size. We assemble and annotate a de novo reference genome for Cebus imitatorThrough high-depth sequencing of DNA derived from blood, various tissues, andfeces via fluorescence-activated cell sorting (fecalFACS) to isolate monkeyepithelial cells, we compared genomes of capuchin populations from tropical dryforests and lowland rainforests and identified population divergence in genesinvolved in water balance, kidney function, and metabolism. Through a comparativegenomics approach spanning a wide diversity of mammals, we identified genes underpositive selection associated with longevity and brain development. Additionally,we provide a technological advancement in the use of noninvasive genomics forstudies of free-ranging mammals. Our intra- and interspecific comparative studyof capuchin genomics provides insights into processes underlying local adaptationto diverse and physiologically challenging environments, as well as the molecularbasis of brain evolution and longevity.\n",
      "CI  - Copyright (c) 2021 the Author(s). Published by PNAS.\n",
      "FAU - Orkin, Joseph D\n",
      "AU  - Orkin JD\n",
      "AUID- ORCID: 0000-0001-6922-2072\n",
      "AD  - Department of Anthropology and Archaeology, University of Calgary, Calgary, ABT2N 1N4, Canada; joseph.orkin@upf.edu amanda.melin@ucalgary.ca.\n",
      "AD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior deInvestigaciones Cientificas, 08003 Barcelona, Spain.\n",
      "AD  - Alberta Children's Hospital Research Institute, University of Calgary, Calgary,AB T38 6A8, Canada.\n",
      "FAU - Montague, Michael J\n",
      "AU  - Montague MJ\n",
      "AUID- ORCID: 0000-0003-0253-4404\n",
      "AD  - Department of Neuroscience, Perelman School of Medicine, University ofPennsylvania, Philadelphia, PA 19146.\n",
      "FAU - Tejada-Martinez, Daniela\n",
      "AU  - Tejada-Martinez D\n",
      "AUID- ORCID: 0000-0001-9531-5280\n",
      "AD  - Department of Biochemistry and Molecular Biology, Thomas Jefferson University,Philadelphia, PA 19107.\n",
      "AD  - Doctorado en Ciencias mencion Ecologia y Evolucion, Instituto de CienciasAmbientales y Evolutivas, Facultad de Ciencias, Universidad Austral de Chile,Valdivia 5090000, Chile.\n",
      "AD  - Integrative Genomics of Ageing Group, Institute of Ageing and Chronic Disease,University of Liverpool, Liverpool L7 8TX, United Kingdom.\n",
      "FAU - de Manuel, Marc\n",
      "AU  - de Manuel M\n",
      "AD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior deInvestigaciones Cientificas, 08003 Barcelona, Spain.\n",
      "FAU - Del Campo, Javier\n",
      "AU  - Del Campo J\n",
      "AUID- ORCID: 0000-0002-5292-1421\n",
      "AD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior deInvestigaciones Cientificas, 08003 Barcelona, Spain.\n",
      "FAU - Cheves Hernandez, Saul\n",
      "AU  - Cheves Hernandez S\n",
      "AD  - Area de Concervacion Guanacaste, 169-5000 Guanacaste, Costa Rica.\n",
      "FAU - Di Fiore, Anthony\n",
      "AU  - Di Fiore A\n",
      "AUID- ORCID: 0000-0001-8893-9052\n",
      "AD  - Department of Anthropology and Primate Molecular Ecology and EvolutionLaboratory, University of Texas at Austin, Austin, TX 78712.\n",
      "AD  - College of Biological and Environmental Sciences, Universidad San Francisco deQuito, 170901 Cumbaya, Ecuador.\n",
      "FAU - Fontsere, Claudia\n",
      "AU  - Fontsere C\n",
      "AUID- ORCID: 0000-0003-2233-6026\n",
      "AD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior deInvestigaciones Cientificas, 08003 Barcelona, Spain.\n",
      "FAU - Hodgson, Jason A\n",
      "AU  - Hodgson JA\n",
      "AUID- ORCID: 0000-0002-3849-2245\n",
      "AD  - Department of Anthropology, Pennsylvania State University, University Park, PA16802.\n",
      "AD  - Department of Zoology, University of Cambridge, Cambridge CB2 3EJ, UnitedKingdom.\n",
      "FAU - Janiak, Mareike C\n",
      "AU  - Janiak MC\n",
      "AUID- ORCID: 0000-0002-7759-2556\n",
      "AD  - Department of Anthropology and Archaeology, University of Calgary, Calgary, ABT2N 1N4, Canada.\n",
      "AD  - Alberta Children's Hospital Research Institute, University of Calgary, Calgary,AB T38 6A8, Canada.\n",
      "AD  - School of Science, Engineering and Environment, University of Salford, Salford M54WT, United Kingdom.\n",
      "FAU - Kuderna, Lukas F K\n",
      "AU  - Kuderna LFK\n",
      "AUID- ORCID: 0000-0002-9992-9295\n",
      "AD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior deInvestigaciones Cientificas, 08003 Barcelona, Spain.\n",
      "FAU - Lizano, Esther\n",
      "AU  - Lizano E\n",
      "AUID- ORCID: 0000-0003-3304-9807\n",
      "AD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior deInvestigaciones Cientificas, 08003 Barcelona, Spain.\n",
      "AD  - Institut Catala de Paleontologia Miquel Crusafont, Universitat Autonoma deBarcelona, 08193 Cerdanyola del Valles, Barcelona, Spain.\n",
      "FAU - Martin, Maria Pia\n",
      "AU  - Martin MP\n",
      "AD  - Kids Saving the Rainforest Wildlife Rescue Center, 60601 Quepos, Costa Rica.\n",
      "FAU - Niimura, Yoshihito\n",
      "AU  - Niimura Y\n",
      "AUID- ORCID: 0000-0001-7010-2537\n",
      "AD  - Department of Applied Biological Chemistry, Graduate School of Agricultural andLife Sciences, The University of Tokyo, Tokyo 113-8657, Japan.\n",
      "FAU - Perry, George H\n",
      "AU  - Perry GH\n",
      "AUID- ORCID: 0000-0003-4527-3806\n",
      "AD  - Department of Anthropology, Pennsylvania State University, University Park, PA16802.\n",
      "AD  - Department of Biology, Pennsylvania State University, University Park, PA 16802.\n",
      "FAU - Valverde, Carmen Soto\n",
      "AU  - Valverde CS\n",
      "AUID- ORCID: 0000-0001-9506-5429\n",
      "AD  - Kids Saving the Rainforest Wildlife Rescue Center, 60601 Quepos, Costa Rica.\n",
      "FAU - Tang, Jia\n",
      "AU  - Tang J\n",
      "AUID- ORCID: 0000-0002-6374-0036\n",
      "AD  - Department of Anthropology and Archaeology, University of Calgary, Calgary, ABT2N 1N4, Canada.\n",
      "FAU - Warren, Wesley C\n",
      "AU  - Warren WC\n",
      "AUID- ORCID: 0000-0003-2255-2730\n",
      "AD  - Division of Animal Sciences, School of Medicine, University of Missouri,Columbia, MO 65211.\n",
      "FAU - de Magalhaes, Joao Pedro\n",
      "AU  - de Magalhaes JP\n",
      "AUID- ORCID: 0000-0002-6363-2465\n",
      "AD  - Integrative Genomics of Ageing Group, Institute of Ageing and Chronic Disease,University of Liverpool, Liverpool L7 8TX, United Kingdom.\n",
      "FAU - Kawamura, Shoji\n",
      "AU  - Kawamura S\n",
      "AUID- ORCID: 0000-0003-0350-6050\n",
      "AD  - Department of Integrated Biosciences, Graduate School of Frontier Sciences, TheUniversity of Tokyo, 277-8562 Chiba, Japan.\n",
      "FAU - Marques-Bonet, Tomas\n",
      "AU  - Marques-Bonet T\n",
      "AUID- ORCID: 0000-0002-5597-3075\n",
      "AD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior deInvestigaciones Cientificas, 08003 Barcelona, Spain.\n",
      "AD  - Institut Catala de Paleontologia Miquel Crusafont, Universitat Autonoma deBarcelona, 08193 Cerdanyola del Valles, Barcelona, Spain.\n",
      "AD  - Catalan Institution of Research and Advanced Studies, 08010 Barcelona, Spain.\n",
      "AD  - Centro Nacional de Analisis Genomico-Centre for Genomic Regulation, BarcelonaInstitute of Science and Technology, 08028 Barcelona, Spain.\n",
      "FAU - Krawetz, Roman\n",
      "AU  - Krawetz R\n",
      "AUID- ORCID: 0000-0002-2576-4504\n",
      "AD  - Department of Cell Biology and Anatomy, Cumming School of Medicine, University ofCalgary, Calgary, AB T38 6A8, Canada.\n",
      "FAU - Melin, Amanda D\n",
      "AU  - Melin AD\n",
      "AUID- ORCID: 0000-0002-0612-2514\n",
      "AD  - Department of Anthropology and Archaeology, University of Calgary, Calgary, ABT2N 1N4, Canada; joseph.orkin@upf.edu amanda.melin@ucalgary.ca.\n",
      "AD  - Alberta Children's Hospital Research Institute, University of Calgary, Calgary,AB T38 6A8, Canada.\n",
      "AD  - Department of Medical Genetics, Cumming School of Medicine, University ofCalgary, Calgary, AB T38 6A8, Canada.\n",
      "LA  - eng\n",
      "GR  - BB/R014949/1/BB_/Biotechnology and Biological Sciences Research Council/UnitedKingdom\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PL  - United States\n",
      "TA  - Proc Natl Acad Sci U S A\n",
      "JT  - Proceedings of the National Academy of Sciences of the United States of America\n",
      "JID - 7505876\n",
      "SB  - IM\n",
      "EIN - Proc Natl Acad Sci U S A. 2021 Aug 24;118(34):. PMID: 34400510\n",
      "MH  - *Adaptation, Physiological\n",
      "MH  - Animals\n",
      "MH  - Brain/*growth & development\n",
      "MH  - Cebus/*genetics\n",
      "MH  - Evolution, Molecular\n",
      "MH  - Flow Cytometry/methods\n",
      "MH  - Forests\n",
      "MH  - *Genome\n",
      "MH  - Genomics/methods\n",
      "MH  - Longevity/*genetics\n",
      "PMC - PMC7896301\n",
      "OTO - NOTNLM\n",
      "OT  - *brain size\n",
      "OT  - *flow cytometry\n",
      "OT  - *longevity\n",
      "OT  - *noninvasive genomics\n",
      "OT  - *seasonality\n",
      "COIS- The authors declare no competing interest.\n",
      "EDAT- 2021/02/13 06:00\n",
      "MHDA- 2021/07/22 06:00\n",
      "CRDT- 2021/02/12 05:52\n",
      "PHST- 2021/02/12 05:52 [entrez]\n",
      "PHST- 2021/02/13 06:00 [pubmed]\n",
      "PHST- 2021/07/22 06:00 [medline]\n",
      "AID - 2010632118 [pii]\n",
      "AID - 10.1073/pnas.2010632118 [doi]\n",
      "PST - ppublish\n",
      "SO  - Proc Natl Acad Sci U S A. 2021 Feb 16;118(7). pii: 2010632118. doi:10.1073/pnas.2010632118.\n",
      "\n",
      "PMID- 33544737\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210701\n",
      "LR  - 20210701\n",
      "IS  - 1932-6203 (Electronic)\n",
      "IS  - 1932-6203 (Linking)\n",
      "VI  - 16\n",
      "IP  - 2\n",
      "DP  - 2021\n",
      "TI  - Genetic characterization of a mild isolate of papaya ringspot virus type-P(PRSV-P) and assessment of its cross-protection potential under greenhouse andfield conditions.\n",
      "PG  - e0241652\n",
      "LID - 10.1371/journal.pone.0241652 [doi]\n",
      "AB  - A mild isolate of Papaya ringspot virus type-P, abbreviated as PRSV-mild, fromEcuador was sequenced and characterized. The most distinguishing symptom induced by PRSV-mild was gray powder-like leaf patches radiating from secondary veins. Ingreenhouse experiments, PRSV-mild did not confer durable protection against asevere isolate of the virus (PRSV-sev), obtained from the same field.Furthermore, isolate specific detection in mixed-infected plants showed thatPRSV-sev becomes dominant in infections, rendering PRSV-mild undetectable at90-120 days post superinfection. Virus testing using isolate-specific primersdetected PRSV-mild in two out of five surveyed provinces, with 10% and 48% ofincidence in Santo Domingo and Los Rios, respectively. Comparative genomicsshowed that PRSV-mild lacks two amino acids from the coat protein region, whereasamino acid determinants for asymptomatic phenotypes were not identified.Recombination events were not predicted in the genomes of the Ecuadoreanisolates. Phylogenetic analyses placed both PRSV-mild and PRSV-sev in a cladethat includes an additional PRSV isolate from Ecuador and others from SouthAmerica.\n",
      "FAU - Medina-Salguero, Andres X\n",
      "AU  - Medina-Salguero AX\n",
      "AD  - Centro de Investigaciones Biotecnologicas del Ecuador, Escuela SuperiorPolitecnica del Litoral, ESPOL, Guayaquil, Guayas, Ecuador.\n",
      "FAU - Cornejo-Franco, Juan F\n",
      "AU  - Cornejo-Franco JF\n",
      "AD  - Centro de Investigaciones Biotecnologicas del Ecuador, Escuela SuperiorPolitecnica del Litoral, ESPOL, Guayaquil, Guayas, Ecuador.\n",
      "FAU - Grinstead, Sam\n",
      "AU  - Grinstead S\n",
      "AUID- ORCID: 0000-0002-1708-8235\n",
      "AD  - National Germplasm Resources Laboratory, USDA-ARS, Beltsville, MD, United States of America.\n",
      "FAU - Mowery, Joseph\n",
      "AU  - Mowery J\n",
      "AUID- ORCID: 0000-0001-7128-1397\n",
      "AD  - Electron and Confocal Microscopy Unit, USDA ARS, Beltsville, MD, United States ofAmerica.\n",
      "FAU - Mollov, Dimitre\n",
      "AU  - Mollov D\n",
      "AD  - National Germplasm Resources Laboratory, USDA-ARS, Beltsville, MD, United States of America.\n",
      "FAU - Quito-Avila, Diego F\n",
      "AU  - Quito-Avila DF\n",
      "AUID- ORCID: 0000-0002-6170-1113\n",
      "AD  - Centro de Investigaciones Biotecnologicas del Ecuador, Escuela SuperiorPolitecnica del Litoral, ESPOL, Guayaquil, Guayas, Ecuador.\n",
      "AD  - Facultad de Ciencias de la Vida, Escuela Superior Politecnica del Litoral, ESPOL,Guayaquil, Guayas, Ecuador.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210205\n",
      "PL  - United States\n",
      "TA  - PLoS One\n",
      "JT  - PloS one\n",
      "JID - 101285081\n",
      "RN  - Papaya ringspot virus\n",
      "SB  - IM\n",
      "MH  - Carica/*virology\n",
      "MH  - Genome, Viral\n",
      "MH  - Phylogeny\n",
      "MH  - Plant Diseases/*virology\n",
      "MH  - Potyvirus/*genetics/isolation & purification\n",
      "PMC - PMC7864462\n",
      "COIS- The authors have declared that no competing interests exist.\n",
      "EDAT- 2021/02/06 06:00\n",
      "MHDA- 2021/07/02 06:00\n",
      "CRDT- 2021/02/05 17:12\n",
      "PHST- 2020/10/15 00:00 [received]\n",
      "PHST- 2021/01/20 00:00 [accepted]\n",
      "PHST- 2021/02/05 17:12 [entrez]\n",
      "PHST- 2021/02/06 06:00 [pubmed]\n",
      "PHST- 2021/07/02 06:00 [medline]\n",
      "AID - 10.1371/journal.pone.0241652 [doi]\n",
      "AID - PONE-D-20-32443 [pii]\n",
      "PST - epublish\n",
      "SO  - PLoS One. 2021 Feb 5;16(2):e0241652. doi: 10.1371/journal.pone.0241652.eCollection 2021.\n",
      "\n",
      "PMID- 33503026\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210514\n",
      "LR  - 20210514\n",
      "IS  - 1553-7358 (Electronic)\n",
      "IS  - 1553-734X (Linking)\n",
      "VI  - 17\n",
      "IP  - 1\n",
      "DP  - 2021 Jan\n",
      "TI  - One is not enough: On the effects of reference genome for the mapping andsubsequent analyses of short-reads.\n",
      "PG  - e1008678\n",
      "LID - 10.1371/journal.pcbi.1008678 [doi]\n",
      "AB  - Mapping of high-throughput sequencing (HTS) reads to a single arbitrary referencegenome is a frequently used approach in microbial genomics. However, the choiceof a reference may represent a source of errors that may affect subsequentanalyses such as the detection of single nucleotide polymorphisms (SNPs) andphylogenetic inference. In this work, we evaluated the effect of reference choiceon short-read sequence data from five clinically and epidemiologically relevantbacteria (Klebsiella pneumoniae, Legionella pneumophila, Neisseria gonorrhoeae,Pseudomonas aeruginosa and Serratia marcescens). Publicly available whole-genome assemblies encompassing the genomic diversity of these species were selected asreference sequences, and read alignment statistics, SNP calling, recombinationrates, dN/dS ratios, and phylogenetic trees were evaluated depending on themapping reference. The choice of different reference genomes proved to have animpact on almost all the parameters considered in the five species. In addition, these biases had potential epidemiological implications such asincluding/excluding isolates of particular clades and the estimation of geneticdistances. These findings suggest that the single reference approach mightintroduce systematic errors during mapping that affect subsequent analyses,particularly for data sets with isolates from genetically diverse backgrounds. Inany case, exploring the effects of different references on the final conclusions is highly recommended.\n",
      "FAU - Valiente-Mullor, Carlos\n",
      "AU  - Valiente-Mullor C\n",
      "AUID- ORCID: 0000-0003-4009-6759\n",
      "AD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\n",
      "FAU - Beamud, Beatriz\n",
      "AU  - Beamud B\n",
      "AUID- ORCID: 0000-0003-0863-3390\n",
      "AD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\n",
      "FAU - Ansari, Ivan\n",
      "AU  - Ansari I\n",
      "AUID- ORCID: 0000-0003-0545-3837\n",
      "AD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\n",
      "FAU - Frances-Cuesta, Carlos\n",
      "AU  - Frances-Cuesta C\n",
      "AUID- ORCID: 0000-0003-4195-054X\n",
      "AD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\n",
      "FAU - Garcia-Gonzalez, Neris\n",
      "AU  - Garcia-Gonzalez N\n",
      "AUID- ORCID: 0000-0001-9089-4033\n",
      "AD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\n",
      "FAU - Mejia, Lorena\n",
      "AU  - Mejia L\n",
      "AUID- ORCID: 0000-0002-7600-6543\n",
      "AD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Ruiz-Hueso, Paula\n",
      "AU  - Ruiz-Hueso P\n",
      "AUID- ORCID: 0000-0003-0217-471X\n",
      "AD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\n",
      "FAU - Gonzalez-Candelas, Fernando\n",
      "AU  - Gonzalez-Candelas F\n",
      "AUID- ORCID: 0000-0002-0879-5798\n",
      "AD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\n",
      "AD  - CIBER in Epidemiology and Public Health, Valencia, Spain.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210127\n",
      "PL  - United States\n",
      "TA  - PLoS Comput Biol\n",
      "JT  - PLoS computational biology\n",
      "JID - 101238922\n",
      "SB  - IM\n",
      "MH  - Bacteria/classification/genetics\n",
      "MH  - Chromosome Mapping/*methods/*standards\n",
      "MH  - Genome, Bacterial/genetics\n",
      "MH  - Genomics/*methods\n",
      "MH  - High-Throughput Nucleotide Sequencing/*methods\n",
      "MH  - Phylogeny\n",
      "MH  - Polymorphism, Single Nucleotide/genetics\n",
      "MH  - Sequence Alignment\n",
      "PMC - PMC7870062\n",
      "COIS- The authors have declared that no competing interests exist.\n",
      "EDAT- 2021/01/28 06:00\n",
      "MHDA- 2021/05/15 06:00\n",
      "CRDT- 2021/01/27 17:12\n",
      "PHST- 2020/04/27 00:00 [received]\n",
      "PHST- 2021/01/05 00:00 [accepted]\n",
      "PHST- 2021/02/08 00:00 [revised]\n",
      "PHST- 2021/01/28 06:00 [pubmed]\n",
      "PHST- 2021/05/15 06:00 [medline]\n",
      "PHST- 2021/01/27 17:12 [entrez]\n",
      "AID - 10.1371/journal.pcbi.1008678 [doi]\n",
      "AID - PCOMPBIOL-D-20-00703 [pii]\n",
      "PST - epublish\n",
      "SO  - PLoS Comput Biol. 2021 Jan 27;17(1):e1008678. doi: 10.1371/journal.pcbi.1008678. eCollection 2021 Jan.\n",
      "\n",
      "PMID- 33419936\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210224\n",
      "LR  - 20220129\n",
      "IS  - 1095-9203 (Electronic)\n",
      "IS  - 0036-8075 (Linking)\n",
      "VI  - 371\n",
      "IP  - 6530\n",
      "DP  - 2021 Feb 12\n",
      "TI  - Establishment and lineage dynamics of the SARS-CoV-2 epidemic in the UK.\n",
      "PG  - 708-712\n",
      "LID - 10.1126/science.abf2946 [doi]\n",
      "AB  - The United Kingdom's COVID-19 epidemic during early 2020 was one of world'slargest and was unusually well represented by virus genomic sampling. Wedetermined the fine-scale genetic lineage structure of this epidemic throughanalysis of 50,887 severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2)genomes, including 26,181 from the UK sampled throughout the country's first waveof infection. Using large-scale phylogenetic analyses combined withepidemiological and travel data, we quantified the size, spatiotemporal origins, and persistence of genetically distinct UK transmission lineages. Rapidfluctuations in virus importation rates resulted in >1000 lineages; thoseintroduced prior to national lockdown tended to be larger and more dispersed.Lineage importation and regional lineage diversity declined after lockdown,whereas lineage elimination was size-dependent. We discuss the implications ofour genetic perspective on transmission dynamics for COVID-19 epidemiology andcontrol.\n",
      "CI  - Copyright (c) 2021 The Authors, some rights reserved; exclusive licensee AmericanAssociation for the Advancement of Science. No claim to original U.S. Government Works.\n",
      "FAU - du Plessis, Louis\n",
      "AU  - du Plessis L\n",
      "AUID- ORCID: https://orcid.org/0000-0003-0352-6289\n",
      "AD  - Department of Zoology, University of Oxford, Oxford, UK.\n",
      "FAU - McCrone, John T\n",
      "AU  - McCrone JT\n",
      "AUID- ORCID: https://orcid.org/0000-0002-9846-8917\n",
      "AD  - Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, UK.\n",
      "FAU - Zarebski, Alexander E\n",
      "AU  - Zarebski AE\n",
      "AUID- ORCID: https://orcid.org/0000-0003-1824-7653\n",
      "AD  - Department of Zoology, University of Oxford, Oxford, UK.\n",
      "FAU - Hill, Verity\n",
      "AU  - Hill V\n",
      "AUID- ORCID: https://orcid.org/0000-0002-3509-8146\n",
      "AD  - Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, UK.\n",
      "FAU - Ruis, Christopher\n",
      "AU  - Ruis C\n",
      "AUID- ORCID: https://orcid.org/0000-0003-0977-5534\n",
      "AD  - Molecular Immunity Unit, Department of Medicine, University of Cambridge,Cambridge, UK.\n",
      "AD  - Department of Veterinary Medicine, University of Cambridge, Cambridge, UK.\n",
      "FAU - Gutierrez, Bernardo\n",
      "AU  - Gutierrez B\n",
      "AUID- ORCID: https://orcid.org/0000-0002-9220-2739\n",
      "AD  - Department of Zoology, University of Oxford, Oxford, UK.\n",
      "AD  - School of Biological and Environmental Sciences, Universidad San Francisco deQuito, Quito, Ecuador.\n",
      "FAU - Raghwani, Jayna\n",
      "AU  - Raghwani J\n",
      "AUID- ORCID: https://orcid.org/0000-0001-8538-4995\n",
      "AD  - Department of Zoology, University of Oxford, Oxford, UK.\n",
      "FAU - Ashworth, Jordan\n",
      "AU  - Ashworth J\n",
      "AUID- ORCID: https://orcid.org/0000-0003-1740-7475\n",
      "AD  - Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, UK.\n",
      "FAU - Colquhoun, Rachel\n",
      "AU  - Colquhoun R\n",
      "AUID- ORCID: https://orcid.org/0000-0002-5577-9897\n",
      "AD  - Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, UK.\n",
      "FAU - Connor, Thomas R\n",
      "AU  - Connor TR\n",
      "AUID- ORCID: https://orcid.org/0000-0003-2394-6504\n",
      "AD  - School of Biosciences, Cardiff University, Cardiff, UK.\n",
      "AD  - Pathogen Genomics Unit, Public Health Wales NHS Trust, Cardiff, UK.\n",
      "FAU - Faria, Nuno R\n",
      "AU  - Faria NR\n",
      "AUID- ORCID: https://orcid.org/0000-0001-8839-2798\n",
      "AD  - Department of Zoology, University of Oxford, Oxford, UK.\n",
      "AD  - MRC Centre for Global Infectious Disease Analysis, J-IDEA, Imperial CollegeLondon, London, UK.\n",
      "FAU - Jackson, Ben\n",
      "AU  - Jackson B\n",
      "AUID- ORCID: https://orcid.org/0000-0002-9981-0649\n",
      "AD  - Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, UK.\n",
      "FAU - Loman, Nicholas J\n",
      "AU  - Loman NJ\n",
      "AUID- ORCID: https://orcid.org/0000-0002-9843-8988\n",
      "AD  - Institute of Microbiology and Infection, University of Birmingham, Birmingham,UK.\n",
      "FAU - O'Toole, Aine\n",
      "AU  - O'Toole A\n",
      "AUID- ORCID: https://orcid.org/0000-0001-8083-474X\n",
      "AD  - Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, UK.\n",
      "FAU - Nicholls, Samuel M\n",
      "AU  - Nicholls SM\n",
      "AUID- ORCID: https://orcid.org/0000-0003-4081-065X\n",
      "AD  - Institute of Microbiology and Infection, University of Birmingham, Birmingham,UK.\n",
      "FAU - Parag, Kris V\n",
      "AU  - Parag KV\n",
      "AUID- ORCID: https://orcid.org/0000-0002-7806-3605\n",
      "AD  - MRC Centre for Global Infectious Disease Analysis, J-IDEA, Imperial CollegeLondon, London, UK.\n",
      "FAU - Scher, Emily\n",
      "AU  - Scher E\n",
      "AUID- ORCID: https://orcid.org/0000-0002-5401-5879\n",
      "AD  - Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, UK.\n",
      "FAU - Vasylyeva, Tetyana I\n",
      "AU  - Vasylyeva TI\n",
      "AD  - Department of Zoology, University of Oxford, Oxford, UK.\n",
      "FAU - Volz, Erik M\n",
      "AU  - Volz EM\n",
      "AUID- ORCID: https://orcid.org/0000-0001-6268-8937\n",
      "AD  - MRC Centre for Global Infectious Disease Analysis, J-IDEA, Imperial CollegeLondon, London, UK.\n",
      "FAU - Watts, Alexander\n",
      "AU  - Watts A\n",
      "AUID- ORCID: https://orcid.org/0000-0002-2675-3460\n",
      "AD  - Li Ka Shing Knowledge Institute, St. Michael's Hospital, Toronto, Canada.\n",
      "AD  - BlueDot, Toronto, Canada.\n",
      "FAU - Bogoch, Isaac I\n",
      "AU  - Bogoch II\n",
      "AD  - Department of Medicine, University of Toronto, Toronto, Canada.\n",
      "AD  - Divisions of General Internal Medicine and Infectious Diseases, University HealthNetwork, Toronto, Canada.\n",
      "FAU - Khan, Kamran\n",
      "AU  - Khan K\n",
      "AUID- ORCID: https://orcid.org/0000-0003-4367-960X\n",
      "AD  - Li Ka Shing Knowledge Institute, St. Michael's Hospital, Toronto, Canada.\n",
      "AD  - BlueDot, Toronto, Canada.\n",
      "AD  - Department of Medicine, University of Toronto, Toronto, Canada.\n",
      "CN  - COVID-19 Genomics UK (COG-UK) Consortium\n",
      "FAU - Aanensen, David M\n",
      "AU  - Aanensen DM\n",
      "AUID- ORCID: https://orcid.org/0000-0001-6688-0854\n",
      "AD  - Centre for Genomic Pathogen Surveillance, Wellcome Genome Campus, Hinxton, UK.\n",
      "AD  - Big Data Institute, Li Ka Shing Centre for Health Information and Discovery,Nuffield Department of Medicine, University of Oxford, Oxford, UK.\n",
      "FAU - Kraemer, Moritz U G\n",
      "AU  - Kraemer MUG\n",
      "AUID- ORCID: https://orcid.org/0000-0001-8838-7147\n",
      "AD  - Department of Zoology, University of Oxford, Oxford, UK.\n",
      "FAU - Rambaut, Andrew\n",
      "AU  - Rambaut A\n",
      "AUID- ORCID: https://orcid.org/0000-0003-4337-3707\n",
      "AD  - Institute of Evolutionary Biology, University of Edinburgh, Edinburgh, UK.a.rambaut@ed.ac.uk oliver.pybus@zoo.ox.ac.uk.\n",
      "FAU - Pybus, Oliver G\n",
      "AU  - Pybus OG\n",
      "AUID- ORCID: https://orcid.org/0000-0002-8797-2667\n",
      "AD  - Department of Zoology, University of Oxford, Oxford, UK. a.rambaut@ed.ac.ukoliver.pybus@zoo.ox.ac.uk.\n",
      "AD  - Department of Pathobiology and Population Sciences, Royal Veterinary CollegeLondon, London, UK.\n",
      "LA  - eng\n",
      "GR  - MC_PC_19012/MRC_/Medical Research Council/United Kingdom\n",
      "GR  - MC_PC_19027/MRC_/Medical Research Council/United Kingdom\n",
      "GR  - Wellcome Trust/United Kingdom\n",
      "GR  - MR/J014370/1/MRC_/Medical Research Council/United Kingdom\n",
      "GR  - MR/T030062/1/MRC_/Medical Research Council/United Kingdom\n",
      "GR  - MR/L015080/1/MRC_/Medical Research Council/United Kingdom\n",
      "GR  - RP-PG-0514-20018/DH_/Department of Health/United Kingdom\n",
      "GR  - MR/M501621/1/MRC_/Medical Research Council/United Kingdom\n",
      "GR  - MR/R015600/1/MRC_/Medical Research Council/United Kingdom\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20210108\n",
      "PL  - United States\n",
      "TA  - Science\n",
      "JT  - Science (New York, N.Y.)\n",
      "JID - 0404511\n",
      "SB  - IM\n",
      "CIN - Science. 2021 Feb 12;371(6530):680-681. PMID: 33574202\n",
      "MH  - COVID-19/*epidemiology/prevention & control/transmission/*virology\n",
      "MH  - Chain of Infection\n",
      "MH  - Communicable Disease Control\n",
      "MH  - Communicable Diseases, Imported/epidemiology/virology\n",
      "MH  - Epidemics\n",
      "MH  - *Genome, Viral\n",
      "MH  - Humans\n",
      "MH  - Phylogeny\n",
      "MH  - SARS-CoV-2/*genetics\n",
      "MH  - Travel\n",
      "MH  - United Kingdom/epidemiology\n",
      "PMC - PMC7877493\n",
      "EDAT- 2021/01/10 06:00\n",
      "MHDA- 2021/02/25 06:00\n",
      "CRDT- 2021/01/09 05:31\n",
      "PHST- 2020/10/22 00:00 [received]\n",
      "PHST- 2020/12/18 00:00 [accepted]\n",
      "PHST- 2021/01/10 06:00 [pubmed]\n",
      "PHST- 2021/02/25 06:00 [medline]\n",
      "PHST- 2021/01/09 05:31 [entrez]\n",
      "AID - science.abf2946 [pii]\n",
      "AID - 10.1126/science.abf2946 [doi]\n",
      "PST - ppublish\n",
      "SO  - Science. 2021 Feb 12;371(6530):708-712. doi: 10.1126/science.abf2946. Epub 2021Jan 8.\n",
      "\n",
      "PMID- 33398025\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210702\n",
      "LR  - 20210702\n",
      "IS  - 2399-3642 (Electronic)\n",
      "IS  - 2399-3642 (Linking)\n",
      "VI  - 4\n",
      "IP  - 1\n",
      "DP  - 2021 Jan 4\n",
      "TI  - Host relatedness and landscape connectivity shape pathogen spread in the puma, a large secretive carnivore.\n",
      "PG  - 12\n",
      "LID - 10.1038/s42003-020-01548-2 [doi]\n",
      "AB  - Urban expansion can fundamentally alter wildlife movement and gene flow, but how urbanization alters pathogen spread is poorly understood. Here, we combine highresolution host and viral genomic data with landscape variables to examine thecontext of viral spread in puma (Puma concolor) from two contrasting regions: onebounded by the wildland urban interface (WUI) and one unbounded with minimalanthropogenic development (UB). We found landscape variables and host gene flowexplained significant amounts of variation of feline immunodeficiency virus (FIV)spread in the WUI, but not in the unbounded region. The most important predictorsof viral spread also differed; host spatial proximity, host relatedness, andmountain ranges played a role in FIV spread in the WUI, whereas roads might have facilitated viral spread in the unbounded region. Our research demonstrates howanthropogenic landscapes can alter pathogen spread, providing a more nuancedunderstanding of host-pathogen relationships to inform disease ecology infree-ranging species.\n",
      "FAU - Fountain-Jones, Nicholas M\n",
      "AU  - Fountain-Jones NM\n",
      "AUID- ORCID: 0000-0001-9248-8493\n",
      "AD  - Department of Veterinary Population Medicine, University of Minnesota, St Paul,MN, 55108, USA. Nick.FountainJones@utas.edu.au.\n",
      "AD  - School of Natural Sciences, University of Tasmania, Hobart, Australia, 7001.Nick.FountainJones@utas.edu.au.\n",
      "FAU - Kraberger, Simona\n",
      "AU  - Kraberger S\n",
      "AD  - Department of Microbiology, Immunology, and Pathology, Colorado State University,Fort Collins, CO, 80523, USA.\n",
      "FAU - Gagne, Roderick B\n",
      "AU  - Gagne RB\n",
      "AUID- ORCID: 0000-0002-4901-5081\n",
      "AD  - Department of Microbiology, Immunology, and Pathology, Colorado State University,Fort Collins, CO, 80523, USA.\n",
      "FAU - Trumbo, Daryl R\n",
      "AU  - Trumbo DR\n",
      "AD  - Department of Biology, Graduate Degree Program in Ecology, Colorado StateUniversity, Fort Collins, CO, 80523, USA.\n",
      "FAU - Salerno, Patricia E\n",
      "AU  - Salerno PE\n",
      "AD  - Department of Biology, Graduate Degree Program in Ecology, Colorado StateUniversity, Fort Collins, CO, 80523, USA.\n",
      "AD  - Universidad Regional Amazonica IKIAM, Km 7 via Muyuna, Tena, Ecuador.\n",
      "FAU - Chris Funk, W\n",
      "AU  - Chris Funk W\n",
      "AUID- ORCID: 0000-0002-6466-3618\n",
      "AD  - Department of Biology, Graduate Degree Program in Ecology, Colorado StateUniversity, Fort Collins, CO, 80523, USA.\n",
      "FAU - Crooks, Kevin\n",
      "AU  - Crooks K\n",
      "AD  - Department of Fish, Wildlife, and Conservation Biology, Colorado StateUniversity, Fort Collins, CO, 80523, USA.\n",
      "FAU - Biek, Roman\n",
      "AU  - Biek R\n",
      "AUID- ORCID: 0000-0003-3471-5357\n",
      "AD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of Glasgow, Glasgow, G12 8QQ, UK.\n",
      "FAU - Alldredge, Mathew\n",
      "AU  - Alldredge M\n",
      "AD  - Colorado Parks and Wildlife, Fort Collins, CO, 80526, USA.\n",
      "FAU - Logan, Ken\n",
      "AU  - Logan K\n",
      "AD  - Colorado Parks and Wildlife, Montrose, CO, 81401, USA.\n",
      "FAU - Baele, Guy\n",
      "AU  - Baele G\n",
      "AD  - Department of Microbiology, Immunology and Transplantation, Rega Institute, KULeuven, Herestraat 49, 3000, Leuven, Belgium.\n",
      "FAU - Dellicour, Simon\n",
      "AU  - Dellicour S\n",
      "AUID- ORCID: 0000-0001-9558-1052\n",
      "AD  - Department of Microbiology, Immunology and Transplantation, Rega Institute, KULeuven, Herestraat 49, 3000, Leuven, Belgium.\n",
      "AD  - Spatial Epidemiology Lab (SpELL), Universite Libre de Bruxelles, CP160/12 50, av.FD Roosevelt, 1050, Bruxelles, Belgium.\n",
      "FAU - Ernest, Holly B\n",
      "AU  - Ernest HB\n",
      "AD  - Wildlife Genomics and Disease Ecology Lab, Department of Veterinary Sciences,University of Wyoming, Laramie, WY, 82070, USA.\n",
      "FAU - VandeWoude, Sue\n",
      "AU  - VandeWoude S\n",
      "AUID- ORCID: 0000-0001-9227-1622\n",
      "AD  - Department of Microbiology, Immunology, and Pathology, Colorado State University,Fort Collins, CO, 80523, USA.\n",
      "FAU - Carver, Scott\n",
      "AU  - Carver S\n",
      "AUID- ORCID: 0000-0002-3579-7588\n",
      "AD  - School of Natural Sciences, University of Tasmania, Hobart, Australia, 7001.\n",
      "FAU - Craft, Meggan E\n",
      "AU  - Craft ME\n",
      "AUID- ORCID: 0000-0001-5333-8513\n",
      "AD  - Department of Veterinary Population Medicine, University of Minnesota, St Paul,MN, 55108, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20210104\n",
      "PL  - England\n",
      "TA  - Commun Biol\n",
      "JT  - Communications biology\n",
      "JID - 101719179\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Colorado/epidemiology\n",
      "MH  - Female\n",
      "MH  - Gene Flow\n",
      "MH  - Immunodeficiency Virus, Feline/*genetics\n",
      "MH  - Lentivirus Infections/epidemiology/transmission/*veterinary\n",
      "MH  - Male\n",
      "MH  - Phylogeography\n",
      "MH  - Puma/genetics/*microbiology\n",
      "MH  - *Urbanization\n",
      "PMC - PMC7782801\n",
      "EDAT- 2021/01/06 06:00\n",
      "MHDA- 2021/07/03 06:00\n",
      "CRDT- 2021/01/05 06:19\n",
      "PHST- 2019/11/12 00:00 [received]\n",
      "PHST- 2020/11/25 00:00 [accepted]\n",
      "PHST- 2021/01/05 06:19 [entrez]\n",
      "PHST- 2021/01/06 06:00 [pubmed]\n",
      "PHST- 2021/07/03 06:00 [medline]\n",
      "AID - 10.1038/s42003-020-01548-2 [doi]\n",
      "AID - 10.1038/s42003-020-01548-2 [pii]\n",
      "PST - epublish\n",
      "SO  - Commun Biol. 2021 Jan 4;4(1):12. doi: 10.1038/s42003-020-01548-2.\n",
      "\n",
      "PMID- 33326438\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210121\n",
      "LR  - 20210121\n",
      "IS  - 1553-7404 (Electronic)\n",
      "IS  - 1553-7390 (Linking)\n",
      "VI  - 16\n",
      "IP  - 12\n",
      "DP  - 2020 Dec\n",
      "TI  - Culture-free genome-wide locus sequence typing (GLST) provides new perspectiveson Trypanosoma cruzi dispersal and infection complexity.\n",
      "PG  - e1009170\n",
      "LID - 10.1371/journal.pgen.1009170 [doi]\n",
      "AB  - Analysis of genetic polymorphism is a powerful tool for epidemiologicalsurveillance and research. Powerful inference from pathogen genetic variation,however, is often restrained by limited access to representative target DNA,especially in the study of obligate parasitic species for which ex vivo cultureis resource-intensive or bias-prone. Modern sequence capture methods enablepathogen genetic variation to be analyzed directly from host/vector material but are often too complex and expensive for resource-poor settings where infectiousdiseases prevail. This study proposes a simple, cost-effective 'genome-wide locussequence typing' (GLST) tool based on massive parallel amplification ofinformation hotspots throughout the target pathogen genome. The multiplexedpolymerase chain reaction amplifies hundreds of different, user-defined genetictargets in a single reaction tube, and subsequent agarose gel-based clean-up and barcoding completes library preparation at under 4 USD per sample. Our studygenerates a flexible GLST primer panel design workflow for Trypanosoma cruzi, theparasitic agent of Chagas disease. We successfully apply our 203-target GLSTpanel to direct, culture-free metagenomic extracts from triatomine vectorscontaining a minimum of 3.69 pg/mul T. cruzi DNA and further elaborate on method performance by sequencing GLST libraries from T. cruzi reference clonesrepresenting discrete typing units (DTUs) TcI, TcIII, TcIV, TcV and TcVI. The 780SNP sites we identify in the sample set repeatably distinguish parasitesinfecting sympatric vectors and detect correlations between genetic andgeographic distances at regional (< 150 km) as well as continental scales. Themarkers also clearly separate TcI, TcIII, TcIV and TcV + TcVI and appear todistinguish multiclonal infections within TcI. We discuss the advantages,limitations and prospects of our method across a spectrum of epidemiologicalresearch.\n",
      "FAU - Schwabl, Philipp\n",
      "AU  - Schwabl P\n",
      "AD  - Institute of Biodiversity, Animal Health & Comparative Medicine, University ofGlasgow, Glasgow, United Kingdom.\n",
      "FAU - Maiguashca Sanchez, Jalil\n",
      "AU  - Maiguashca Sanchez J\n",
      "AUID- ORCID: 0000-0001-8513-4064\n",
      "AD  - Centro de Investigacion para la Salud en America Latina, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Costales, Jaime A\n",
      "AU  - Costales JA\n",
      "AUID- ORCID: 0000-0001-9418-6640\n",
      "AD  - Centro de Investigacion para la Salud en America Latina, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Ocana-Mayorga, Sofia\n",
      "AU  - Ocana-Mayorga S\n",
      "AUID- ORCID: 0000-0003-3861-9784\n",
      "AD  - Centro de Investigacion para la Salud en America Latina, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Segovia, Maikell\n",
      "AU  - Segovia M\n",
      "AD  - Laboratorio de Biologia Molecular de Protozoarios, Instituto de MedicinaTropical, Universidad Central de Venezuela, Caracas, Venezuela.\n",
      "FAU - Carrasco, Hernan J\n",
      "AU  - Carrasco HJ\n",
      "AUID- ORCID: 0000-0001-7500-0927\n",
      "AD  - Laboratorio de Biologia Molecular de Protozoarios, Instituto de MedicinaTropical, Universidad Central de Venezuela, Caracas, Venezuela.\n",
      "FAU - Hernandez, Carolina\n",
      "AU  - Hernandez C\n",
      "AD  - Grupo de Investigaciones Microbiologicas-UR (GIMUR), Departamento de Biologia,Facultad de Ciencias Naturales, Universidad del Rosario, Bogota, Colombia.\n",
      "FAU - Ramirez, Juan David\n",
      "AU  - Ramirez JD\n",
      "AUID- ORCID: 0000-0002-1344-9312\n",
      "AD  - Grupo de Investigaciones Microbiologicas-UR (GIMUR), Departamento de Biologia,Facultad de Ciencias Naturales, Universidad del Rosario, Bogota, Colombia.\n",
      "FAU - Lewis, Michael D\n",
      "AU  - Lewis MD\n",
      "AUID- ORCID: 0000-0001-9540-3280\n",
      "AD  - London School of Hygiene & Tropical Medicine, Keppel Street, London, UnitedKingdom.\n",
      "FAU - Grijalva, Mario J\n",
      "AU  - Grijalva MJ\n",
      "AUID- ORCID: 0000-0003-1964-1425\n",
      "AD  - Centro de Investigacion para la Salud en America Latina, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador.\n",
      "AD  - Infectious and Tropical Disease Institute, Biomedical Sciences Department,Heritage College of Osteopathic Medicine, Ohio University, Athens, OH, UnitedStates of America.\n",
      "FAU - Llewellyn, Martin S\n",
      "AU  - Llewellyn MS\n",
      "AUID- ORCID: 0000-0001-9856-1591\n",
      "AD  - Institute of Biodiversity, Animal Health & Comparative Medicine, University ofGlasgow, Glasgow, United Kingdom.\n",
      "LA  - eng\n",
      "GR  - 204820/Z/16/Z/WT_/Wellcome Trust/United Kingdom\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20201216\n",
      "PL  - United States\n",
      "TA  - PLoS Genet\n",
      "JT  - PLoS genetics\n",
      "JID - 101239074\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Costs and Cost Analysis\n",
      "MH  - DNA Barcoding, Taxonomic/economics/*methods/standards\n",
      "MH  - Disease Vectors\n",
      "MH  - *Genome, Protozoan\n",
      "MH  - Hemiptera/parasitology\n",
      "MH  - *Metagenome\n",
      "MH  - Metagenomics/economics/*methods/standards\n",
      "MH  - Polymorphism, Genetic\n",
      "MH  - Trypanosoma cruzi/*genetics/pathogenicity\n",
      "MH  - Virulence/genetics\n",
      "MH  - Whole Genome Sequencing/economics/*methods/standards\n",
      "PMC - PMC7743988\n",
      "COIS- The authors have declared that no competing interests exist.\n",
      "EDAT- 2020/12/17 06:00\n",
      "MHDA- 2021/01/22 06:00\n",
      "CRDT- 2020/12/16 17:14\n",
      "PHST- 2020/04/02 00:00 [received]\n",
      "PHST- 2020/10/02 00:00 [accepted]\n",
      "PHST- 2020/12/16 17:14 [entrez]\n",
      "PHST- 2020/12/17 06:00 [pubmed]\n",
      "PHST- 2021/01/22 06:00 [medline]\n",
      "AID - 10.1371/journal.pgen.1009170 [doi]\n",
      "AID - PGENETICS-D-20-00502 [pii]\n",
      "PST - epublish\n",
      "SO  - PLoS Genet. 2020 Dec 16;16(12):e1009170. doi: 10.1371/journal.pgen.1009170.eCollection 2020 Dec.\n",
      "\n",
      "PMID- 33321976\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20201229\n",
      "IS  - 2076-2615 (Print)\n",
      "IS  - 2076-2615 (Linking)\n",
      "VI  - 10\n",
      "IP  - 12\n",
      "DP  - 2020 Dec 10\n",
      "TI  - Porcine Digestible Peptides (PDP) in Weanling Diets Regulates the Expression ofGenes Involved in Gut Barrier Function, Immune Response and Nutrient Transport inNursery Pigs.\n",
      "LID - E2368 [pii]\n",
      "LID - 10.3390/ani10122368 [doi]\n",
      "AB  - This study was conducted to investigate the effects of dietary supplementation ofporcine digestible peptides (PDP), spray-dried plasma (SDP), or a combination of both, on growth performance and the expression of genes related to intestinalfunction of weaned pigs. A total of 180 piglets (trial 1) and 198 piglets (trial 2) were used to evaluate the partial substitution of soybean ingredients with 2% SDP or 2% PDP (trial 1), and with 3% SDP or the combination of 1% SDP and 2% PDP (SDP-PDP; trial 2) during the pre-starter period (0-14 days). The gene expressionof 56 genes was quantified in a qPCR platform in jejunum and ileum samplesobtained from piglets 14 d after weaning (trial 2). Piglets fed SDP, PDP andSDP-PDP had a higher body weight (BW), average daily gain (ADG) and feedefficiency (G:F) than the soybean control on day 14 (p < 0.05). In addition, the combination of SDP and PDP upregulated ten genes in jejunum samples (p < 0.05)related to intestinal function. More research is needed to confirm that geneexpression upregulation by PDP in combination with SDP has an impact onintestinal function and to elucidate its underlying mechanisms.\n",
      "FAU - Gonzalez-Sole, Francesc\n",
      "AU  - Gonzalez-Sole F\n",
      "AUID- ORCID: 0000-0002-3469-6240\n",
      "AD  - Department of Animal and Food Sciences, Animal Nutrition and Welfare Service,Autonomous University of Barcelona, 08193 Bellaterra, Spain.\n",
      "FAU - Criado-Mesas, Lourdes\n",
      "AU  - Criado-Mesas L\n",
      "AUID- ORCID: 0000-0002-1115-4131\n",
      "AD  - Plant and Animal Genomics, Centre for Research in Agricultural Genomics (CRAG),CSIC-IRTA-UAB-UB, 08193 Bellaterra, Spain.\n",
      "FAU - Villodre, Carmen\n",
      "AU  - Villodre C\n",
      "AD  - Department of Animal and Food Sciences, Animal Nutrition and Welfare Service,Autonomous University of Barcelona, 08193 Bellaterra, Spain.\n",
      "FAU - Garcia, Wellington C\n",
      "AU  - Garcia WC\n",
      "AUID- ORCID: 0000-0003-3182-2085\n",
      "AD  - Department of Animal and Food Sciences, Animal Nutrition and Welfare Service,Autonomous University of Barcelona, 08193 Bellaterra, Spain.\n",
      "AD  - Department of Animal Production, Agrarian University of Ecuador, Guayaquil090108, Ecuador.\n",
      "FAU - Farre, Merce\n",
      "AU  - Farre M\n",
      "AD  - Department of Mathematics, Area of Statistics and Operations Research, AutonomousUniversity of Barcelona, 08193 Bellaterra, Spain.\n",
      "FAU - Borda, Elisabet\n",
      "AU  - Borda E\n",
      "AUID- ORCID: 0000-0001-8028-7459\n",
      "AD  - R&D Animal Nutrition Department, Bioiberica S.A.U., 08389 Palafolls, Spain.\n",
      "FAU - Perez-Cano, Francisco J\n",
      "AU  - Perez-Cano FJ\n",
      "AUID- ORCID: 0000-0001-6243-9164\n",
      "AD  - Physiology Section, Department of Biochemistry and Physiology, Faculty ofPharmacy and Food Science, University of Barcelona (UB), 08028 Barcelona, Spain.\n",
      "AD  - Nutrition and Food Safety Research Institute (INSA-UB), 08921 Santa Coloma deGramenet, Spain.\n",
      "FAU - Folch, Josep M\n",
      "AU  - Folch JM\n",
      "AUID- ORCID: 0000-0003-3689-1303\n",
      "AD  - Department of Animal and Food Sciences, Animal Nutrition and Welfare Service,Autonomous University of Barcelona, 08193 Bellaterra, Spain.\n",
      "AD  - Plant and Animal Genomics, Centre for Research in Agricultural Genomics (CRAG),CSIC-IRTA-UAB-UB, 08193 Bellaterra, Spain.\n",
      "FAU - Sola-Oriol, David\n",
      "AU  - Sola-Oriol D\n",
      "AUID- ORCID: 0000-0001-8365-340X\n",
      "AD  - Department of Animal and Food Sciences, Animal Nutrition and Welfare Service,Autonomous University of Barcelona, 08193 Bellaterra, Spain.\n",
      "FAU - Perez, Jose F\n",
      "AU  - Perez JF\n",
      "AUID- ORCID: 0000-0001-8853-8945\n",
      "AD  - Department of Animal and Food Sciences, Animal Nutrition and Welfare Service,Autonomous University of Barcelona, 08193 Bellaterra, Spain.\n",
      "LA  - eng\n",
      "GR  - (AGL2016-75463-R)/Ministerio de Economia, Industria y Competitividad, Gobierno deEspana\n",
      "GR  - FPU grant (Formacion de Profesorado Universitario; FPU18/00401)/Ministerio deCiencia, Innovacion y Universidades, Gobierno de Espana\n",
      "PT  - Journal Article\n",
      "DEP - 20201210\n",
      "PL  - Switzerland\n",
      "TA  - Animals (Basel)\n",
      "JT  - Animals : an open access journal from MDPI\n",
      "JID - 101635614\n",
      "PMC - PMC7763307\n",
      "OTO - NOTNLM\n",
      "OT  - gene expression\n",
      "OT  - high throughput RT-qPCR\n",
      "OT  - intestinal function\n",
      "OT  - piglet\n",
      "OT  - porcine digestible peptides\n",
      "EDAT- 2020/12/17 06:00\n",
      "MHDA- 2020/12/17 06:01\n",
      "CRDT- 2020/12/16 01:02\n",
      "PHST- 2020/10/20 00:00 [received]\n",
      "PHST- 2020/12/05 00:00 [revised]\n",
      "PHST- 2020/12/08 00:00 [accepted]\n",
      "PHST- 2020/12/16 01:02 [entrez]\n",
      "PHST- 2020/12/17 06:00 [pubmed]\n",
      "PHST- 2020/12/17 06:01 [medline]\n",
      "AID - ani10122368 [pii]\n",
      "AID - 10.3390/ani10122368 [doi]\n",
      "PST - epublish\n",
      "SO  - Animals (Basel). 2020 Dec 10;10(12). pii: ani10122368. doi: 10.3390/ani10122368.\n",
      "\n",
      "PMID- 33294020\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20201210\n",
      "IS  - 1752-4571 (Print)\n",
      "IS  - 1752-4571 (Linking)\n",
      "VI  - 13\n",
      "IP  - 10\n",
      "DP  - 2020 Dec\n",
      "TI  - Host plants and Wolbachia shape the population genetics of sympatric herbivorepopulations.\n",
      "PG  - 2740-2753\n",
      "LID - 10.1111/eva.13079 [doi]\n",
      "AB  - Changing climate and land-use practices have the potential to bring previouslyisolated populations of pest insects into new sympatry. This heightens the needto better understand how differing patterns of host-plant association, and uniqueendosymbionts, serve to promote genetic isolation or integration. We addressedthese factors in populations of potato psyllid, Bactericera cockerelli (Sulc), a generalist herbivore that vectors a bacterial pathogen (Candidatus Liberibactersolanacearum, causal pathogen of zebra chip disease) of potato (Solanum tuberosumL.). Genome-wide SNP data revealed two major genetic clusters-psyllids collected from potato crops were genetically similar to psyllids found on a common weed,Lycium spp., but dissimilar from those found on another common non-crop host,Solanum dulcamara L. Most psyllids found on Lycium spp. and potato represented a single mitochondrial cytochrome oxidase I (COI) haplotype that has been suggestedto not be native to the region, and whose arrival may have been concurrent withzebra chip disease first emerging. The putatively introduced COI haplotypeusually co-occurred with endosymbiotic Wolbachia, while the putatively residentCOI haplotype generally did not. Genetic intermediates between the two geneticpopulations of insects were rare, consistent with recent sympatry or reproductiveisolation, although admixture patterns of apparent hybrids were consistent withintrogression of genes from introduced into resident populations. Our resultssuggest that both host-plant associations and endosymbionts are shaping thepopulation genetic structure of sympatric psyllid populations associated withdifferent non-crop hosts. It is of future interest to explicitly examinevectorial capacity of the two populations and their potential hybrids, aspopulation structure and hybridization might alter regional vector capacity anddisease outbreaks.\n",
      "CI  - (c) 2020 The Authors. Evolutionary Applications published by John Wiley & SonsLtd.\n",
      "FAU - Fu, Zhen\n",
      "AU  - Fu Z\n",
      "AUID- ORCID: https://orcid.org/0000-0002-8120-590X\n",
      "AD  - Department of Entomology Washington State University Pullman WA USA.\n",
      "AD  - Present address: Department of Entomology Texas A&M University College Station TXUSA.\n",
      "FAU - Meier, Amanda R\n",
      "AU  - Meier AR\n",
      "AUID- ORCID: https://orcid.org/0000-0003-1085-4715\n",
      "AD  - Department of Entomology University of Georgia Athens GA USA.\n",
      "FAU - Epstein, Brendan\n",
      "AU  - Epstein B\n",
      "AUID- ORCID: https://orcid.org/0000-0001-7083-1588\n",
      "AD  - Department of Plant and Microbial Biology University of Minnesota St. Paul MNUSA.\n",
      "FAU - Bergland, Alan O\n",
      "AU  - Bergland AO\n",
      "AUID- ORCID: https://orcid.org/0000-0001-7145-7575\n",
      "AD  - Department of Biology University of Virginia Charlottesville VA USA.\n",
      "FAU - Castillo Carrillo, Carmen I\n",
      "AU  - Castillo Carrillo CI\n",
      "AUID- ORCID: https://orcid.org/0000-0001-7893-5218\n",
      "AD  - Departamento de Proteccion Vegetal Estacion Experimental Santa Catalina InstitutoNacional de Investigaciones Agropecuarias (INIAP) Quito Ecuador.\n",
      "FAU - Cooper, William R\n",
      "AU  - Cooper WR\n",
      "AD  - Temperate Tree Fruit and Vegetable Research USDA-ARS Wapato WA USA.\n",
      "FAU - Cruzado, Regina K\n",
      "AU  - Cruzado RK\n",
      "AD  - Department of Entomology, Plant Pathology, and Nematology University of IdahoMoscow ID USA.\n",
      "FAU - Horton, David R\n",
      "AU  - Horton DR\n",
      "AD  - Temperate Tree Fruit and Vegetable Research USDA-ARS Wapato WA USA.\n",
      "FAU - Jensen, Andrew S\n",
      "AU  - Jensen AS\n",
      "AUID- ORCID: https://orcid.org/0000-0002-6675-4185\n",
      "AD  - Northwest Potato Research Consortium Lakeview OR USA.\n",
      "FAU - Kelley, Joanna L\n",
      "AU  - Kelley JL\n",
      "AUID- ORCID: https://orcid.org/0000-0002-7731-605X\n",
      "AD  - School of Biological Sciences Washington State University Pullman WA USA.\n",
      "FAU - Rashed, Arash\n",
      "AU  - Rashed A\n",
      "AUID- ORCID: https://orcid.org/0000-0001-6640-2457\n",
      "AD  - Department of Entomology, Plant Pathology, and Nematology University of IdahoMoscow ID USA.\n",
      "FAU - Reitz, Stuart R\n",
      "AU  - Reitz SR\n",
      "AD  - Malheur Experiment Station Oregon State University Ontario OR USA.\n",
      "FAU - Rondon, Silvia I\n",
      "AU  - Rondon SI\n",
      "AUID- ORCID: https://orcid.org/0000-0002-7819-6934\n",
      "AD  - Department of Crop and Soil Science Hermiston Agricultural Research and ExtensionCenter Hermiston OR USA.\n",
      "FAU - Thinakaran, Jenita\n",
      "AU  - Thinakaran J\n",
      "AD  - Temperate Tree Fruit and Vegetable Research USDA-ARS Wapato WA USA.\n",
      "FAU - Wenninger, Erik J\n",
      "AU  - Wenninger EJ\n",
      "AD  - Department of Entomology, Plant Pathology, and Nematology Kimberly Research andExtension Center University of Idaho Kimberly ID USA.\n",
      "FAU - Wohleb, Carrie H\n",
      "AU  - Wohleb CH\n",
      "AD  - Washington State University Extension Moses Lake WA USA.\n",
      "FAU - Crowder, David W\n",
      "AU  - Crowder DW\n",
      "AD  - Department of Entomology Washington State University Pullman WA USA.\n",
      "FAU - Snyder, William E\n",
      "AU  - Snyder WE\n",
      "AUID- ORCID: https://orcid.org/0000-0002-2747-3102\n",
      "AD  - Department of Entomology University of Georgia Athens GA USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20200912\n",
      "PL  - England\n",
      "TA  - Evol Appl\n",
      "JT  - Evolutionary applications\n",
      "JID - 101461828\n",
      "PMC - PMC7691456\n",
      "OTO - NOTNLM\n",
      "OT  - Wolbachia\n",
      "OT  - herbivore interactions\n",
      "OT  - interbreeding\n",
      "OT  - pest management\n",
      "OT  - plant\n",
      "OT  - population genomics\n",
      "EDAT- 2020/12/10 06:00\n",
      "MHDA- 2020/12/10 06:01\n",
      "CRDT- 2020/12/09 06:13\n",
      "PHST- 2020/01/16 00:00 [received]\n",
      "PHST- 2020/07/17 00:00 [revised]\n",
      "PHST- 2020/07/21 00:00 [accepted]\n",
      "PHST- 2020/12/09 06:13 [entrez]\n",
      "PHST- 2020/12/10 06:00 [pubmed]\n",
      "PHST- 2020/12/10 06:01 [medline]\n",
      "AID - 10.1111/eva.13079 [doi]\n",
      "AID - EVA13079 [pii]\n",
      "PST - epublish\n",
      "SO  - Evol Appl. 2020 Sep 12;13(10):2740-2753. doi: 10.1111/eva.13079. eCollection 2020Dec.\n",
      "\n",
      "PMID- 33254102\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210729\n",
      "LR  - 20211204\n",
      "IS  - 1878-0326 (Electronic)\n",
      "IS  - 1872-4973 (Linking)\n",
      "VI  - 51\n",
      "DP  - 2021 Mar\n",
      "TI  - A Y-chromosomal survey of Ecuador's multi-ethnic population reveals new insights into the tri-partite population structure and supports an early Holocene age ofthe rare Native American founder lineage C3-MPB373.\n",
      "PG  - 102427\n",
      "LID - S1872-4973(20)30199-X [pii]\n",
      "LID - 10.1016/j.fsigen.2020.102427 [doi]\n",
      "AB  - Ecuador is a multiethnic and pluricultural country with a complex history definedby migration and admixture processes. The present study aims to increase ourknowledge on the Ecuadorian Native Amerindian groups and the unique SouthAmerican Y-chromosome haplogroup C3-MPB373 through the analysis of up to 23Y-chromosome STRs (Y-STRs) and several Y-SNPs in a sample of 527 Ecuadorians from7 distinct populations and geographic areas, including Kichwa and non-KichwaNative Amerindians, Mestizos and Afro-Ecuadorians. Our results reveal thepresence of C3-MPB373 both in the Amazonian lowland Kichwa with frequencies up to28 % and, for the first time, in notable proportions in Kichwa populations fromthe Ecuadorian highlands. The substantially higher frequencies of C3-MPB373 inthe Amazonian lowlands found in Kichwa and Waorani individuals suggest a founder effect in that area. Notably, estimates for the time to the most recent commonancestor (TMRCA) in the range of 7.2-9.0 kya point to an ancient origin of thehaplogroup and suggest an early Holocene expansion of C3-MPB373 into SouthAmerica. Finally, the pairwise genetic distances (RST) separate the KichwaSalasaka from all the other Native Amerindian and Ecuadorian groups, indicating aso far hidden diversity among the Kichwa-speaking populations and suggesting amore southern origin of this population. In sum, our study provides a morein-depth knowledge of the male genetic structure of the multiethnic Ecuadorianpopulation, as well as a valuable reference dataset for forensic use.\n",
      "CI  - Copyright (c) 2020 Elsevier B.V. All rights reserved.\n",
      "FAU - Villaescusa, Patricia\n",
      "AU  - Villaescusa P\n",
      "AD  - BIOMICs Research Group, Lascaray Research Center, University of the BasqueCountry UPV/EHU, Vitoria-Gasteiz, Spain. Electronic address:patricia.villaescusa89@gmail.com.\n",
      "FAU - Seidel, Maria\n",
      "AU  - Seidel M\n",
      "AD  - Department of Forensic Genetics, Institute of Legal Medicine and ForensicSciences, Charite-Universitatsmedizin Berlin, Berlin, Germany.\n",
      "FAU - Nothnagel, Michael\n",
      "AU  - Nothnagel M\n",
      "AD  - Department of Statistical Genetics and Bioinformatics, Cologne Center forGenomics, University of Cologne, Cologne, Germany; University Hospital Cologne,Cologne, Germany.\n",
      "FAU - Pinotti, Thomaz\n",
      "AU  - Pinotti T\n",
      "AD  - Laboratorio de Biodiversidade e Evolucao Molecular (LBEM), Instituto de Ciencias Biologicas, Universidade Federal de Minas Gerais, Belo Horizonte, Brazil; Centre for GeoGenetics, Natural History Museum of Denmark, University of Copenhagen,Copenhagen, Denmark.\n",
      "FAU - Gonzalez-Andrade, Fabricio\n",
      "AU  - Gonzalez-Andrade F\n",
      "AD  - Central University of Ecuador, Faculty of Medical Sciences, Quito, Ecuador.\n",
      "FAU - Alvarez-Gila, Oscar\n",
      "AU  - Alvarez-Gila O\n",
      "AD  - Department of Medieval, Early Modern and American History, Faculty of Letters,University of the Basque Country UPV/EHU, Vitoria-Gasteiz, Spain.\n",
      "FAU - M de Pancorbo, Marian\n",
      "AU  - M de Pancorbo M\n",
      "AD  - BIOMICs Research Group, Lascaray Research Center, University of the BasqueCountry UPV/EHU, Vitoria-Gasteiz, Spain.\n",
      "FAU - Roewer, Lutz\n",
      "AU  - Roewer L\n",
      "AD  - Department of Forensic Genetics, Institute of Legal Medicine and ForensicSciences, Charite-Universitatsmedizin Berlin, Berlin, Germany.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20201116\n",
      "PL  - Netherlands\n",
      "TA  - Forensic Sci Int Genet\n",
      "JT  - Forensic science international. Genetics\n",
      "JID - 101317016\n",
      "SB  - IM\n",
      "MH  - *Chromosomes, Human, Y\n",
      "MH  - DNA Fingerprinting\n",
      "MH  - Ecuador\n",
      "MH  - Ethnicity/*genetics\n",
      "MH  - *Genetics, Population\n",
      "MH  - Haplotypes\n",
      "MH  - Humans\n",
      "MH  - Indians, South American/*genetics\n",
      "MH  - Male\n",
      "MH  - Microsatellite Repeats\n",
      "MH  - Polymorphism, Single Nucleotide\n",
      "OTO - NOTNLM\n",
      "OT  - *C3-MPB373\n",
      "OT  - *Ecuador\n",
      "OT  - *Kichwas\n",
      "OT  - *Y-SNPs\n",
      "OT  - *Y-STRs\n",
      "EDAT- 2020/12/01 06:00\n",
      "MHDA- 2021/07/30 06:00\n",
      "CRDT- 2020/11/30 20:15\n",
      "PHST- 2020/05/14 00:00 [received]\n",
      "PHST- 2020/11/03 00:00 [revised]\n",
      "PHST- 2020/11/10 00:00 [accepted]\n",
      "PHST- 2020/12/01 06:00 [pubmed]\n",
      "PHST- 2021/07/30 06:00 [medline]\n",
      "PHST- 2020/11/30 20:15 [entrez]\n",
      "AID - S1872-4973(20)30199-X [pii]\n",
      "AID - 10.1016/j.fsigen.2020.102427 [doi]\n",
      "PST - ppublish\n",
      "SO  - Forensic Sci Int Genet. 2021 Mar;51:102427. doi: 10.1016/j.fsigen.2020.102427.Epub 2020 Nov 16.\n",
      "\n",
      "PMID- 33184198\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20201117\n",
      "LR  - 20201118\n",
      "IS  - 1095-9203 (Electronic)\n",
      "IS  - 0036-8075 (Linking)\n",
      "VI  - 370\n",
      "IP  - 6518\n",
      "DP  - 2020 Nov 13\n",
      "TI  - Build international biorepository capacity.\n",
      "PG  - 773-774\n",
      "LID - 10.1126/science.abe4813 [doi]\n",
      "FAU - Colella, Jocelyn P\n",
      "AU  - Colella JP\n",
      "AD  - Biodiversity Institute, University of Kansas, Lawrence, KS 66045 USA.\n",
      "FAU - Agwanda, Bernard Risky\n",
      "AU  - Agwanda BR\n",
      "AD  - National Museums of Kenya, Nairobi, Kenya.\n",
      "FAU - Anwarali Khan, Faisal Ali\n",
      "AU  - Anwarali Khan FA\n",
      "AD  - Faculty of Resource Science and Technology, Universiti Malaysia Sarawak, Sarawak,Malaysia.\n",
      "FAU - Bates, John\n",
      "AU  - Bates J\n",
      "AD  - Field Museum, Chicago, IL 60605, USA.\n",
      "AD  - Natural Science Collections Alliance, Washington, DC 20005, USA.\n",
      "FAU - Carrion Bonilla, Carlos A\n",
      "AU  - Carrion Bonilla CA\n",
      "AD  - Museo de Zoologia, Escuela de Biologia, Pontificia Universidad Catolica delEcuador, Quito, Ecuador.\n",
      "AD  - Museum of Southwestern Biology and Biology Department, University of New Mexico, Albuquerque, NM 87131, USA.\n",
      "FAU - de la Sancha, Noe U\n",
      "AU  - de la Sancha NU\n",
      "AD  - Field Museum, Chicago, IL 60605, USA.\n",
      "AD  - Department of Biological Sciences, Chicago State University, Chicago, IL 60628,USA.\n",
      "FAU - Dunnum, Jonathan L\n",
      "AU  - Dunnum JL\n",
      "AD  - Museum of Southwestern Biology and Biology Department, University of New Mexico, Albuquerque, NM 87131, USA.\n",
      "FAU - Ferguson, Adam W\n",
      "AU  - Ferguson AW\n",
      "AD  - Field Museum, Chicago, IL 60605, USA.\n",
      "FAU - Greiman, Stephen E\n",
      "AU  - Greiman SE\n",
      "AD  - Department of Biology, Georgia Southern University, Statesboro, GA 30458, USA.\n",
      "FAU - Kiswele, Prince Kaleme\n",
      "AU  - Kiswele PK\n",
      "AD  - Centre de Recherche en Sciences Naturelles, Lwiro, Democratic Republic of Congo.\n",
      "FAU - Lessa, Enrique P\n",
      "AU  - Lessa EP\n",
      "AD  - Facultad de Ciencias, Universidad de la Republica, Montevideo, Uruguay.\n",
      "FAU - Soltis, Pamela\n",
      "AU  - Soltis P\n",
      "AD  - Florida Museum of Natural History and the University of Florida BiodiversityInstitute, University of Florida, Gainesville, FL 32611, USA.\n",
      "FAU - Thompson, Cody W\n",
      "AU  - Thompson CW\n",
      "AD  - Department of Ecology and Evolutionary Biology and the Museum of Zoology,University of Michigan, Ann Arbor, MI 48108, USA.\n",
      "FAU - Vanhove, Maarten P M\n",
      "AU  - Vanhove MPM\n",
      "AD  - Research Group Zoology: Biodiversity and Toxicology, Centre for EnvironmentalSciences, Hasselt University, Diepenbeek, Belgium.\n",
      "AD  - Department of Botany and Zoology, Faculty of Science, Masaryk University, Brno,Czech Republic.\n",
      "AD  - Laboratory of Biodiversity and Evolutionary Genomics, Department of Biology,University of Leuven, Leuven, Belgium.\n",
      "FAU - Webala, Paul W\n",
      "AU  - Webala PW\n",
      "AD  - Department of Forestry and Wildlife Management, Maasai Mara University, Narok,Kenya.\n",
      "FAU - Weksler, Marcelo\n",
      "AU  - Weksler M\n",
      "AD  - Departamento de Vertebrados, Museu Nacional, Universidade Federal do Rio deJaneiro, Rio de Janeiro, Brazil.\n",
      "FAU - Cook, Joseph A\n",
      "AU  - Cook JA\n",
      "AD  - Museum of Southwestern Biology and Biology Department, University of New Mexico, Albuquerque, NM 87131, USA. cookjose@unm.edu.\n",
      "LA  - eng\n",
      "PT  - Letter\n",
      "PT  - Comment\n",
      "PL  - United States\n",
      "TA  - Science\n",
      "JT  - Science (New York, N.Y.)\n",
      "JID - 0404511\n",
      "SB  - IM\n",
      "CON - Science. 2020 Jul 10;369(6500):145-147. PMID: 32646989\n",
      "CIN - Science. 2020 Nov 13;370(6518):774. PMID: 33184199\n",
      "MH  - *Animal Diseases\n",
      "MH  - Animals\n",
      "MH  - Animals, Wild\n",
      "MH  - *Epidemiological Monitoring/veterinary\n",
      "MH  - *Specimen Handling\n",
      "MH  - *Tissue Banks\n",
      "EDAT- 2020/11/14 06:00\n",
      "MHDA- 2020/11/18 06:00\n",
      "CRDT- 2020/11/13 05:45\n",
      "PHST- 2020/11/13 05:45 [entrez]\n",
      "PHST- 2020/11/14 06:00 [pubmed]\n",
      "PHST- 2020/11/18 06:00 [medline]\n",
      "AID - 370/6518/773-b [pii]\n",
      "AID - 10.1126/science.abe4813 [doi]\n",
      "PST - ppublish\n",
      "SO  - Science. 2020 Nov 13;370(6518):773-774. doi: 10.1126/science.abe4813.\n",
      "\n",
      "PMID- 33080926\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210716\n",
      "LR  - 20210716\n",
      "IS  - 2073-4425 (Electronic)\n",
      "IS  - 2073-4425 (Linking)\n",
      "VI  - 11\n",
      "IP  - 10\n",
      "DP  - 2020 Oct 17\n",
      "TI  - Landscape Genomics of a Widely Distributed Snake, Dolichophis caspius (Gmelin,1789) across Eastern Europe and Western Asia.\n",
      "LID - E1218 [pii]\n",
      "LID - 10.3390/genes11101218 [doi]\n",
      "AB  - Across the distribution of the Caspian whipsnake (Dolichophis caspius),populations have become increasingly disconnected due to habitat alteration. Tounderstand population dynamics and this widespread but locally endangered snake'sadaptive potential, we investigated population structure, admixture, andeffective migration patterns. We took a landscape-genomic approach to identifyselected genotypes associated with environmental variables relevant to D.caspius. With double-digest restriction-site associated DNA (ddRAD) sequencing of53 samples resulting in 17,518 single nucleotide polymorphisms (SNPs), weidentified 8 clusters within D. caspius reflecting complex evolutionary patterns of the species. Estimated Effective Migration Surfaces (EEMS) revealedhigher-than-average gene flow in most of the Balkan Peninsula andlower-than-average gene flow along the middle section of the Danube River.Landscape genomic analysis identified 751 selected genotypes correlated with 7climatic variables. Isothermality correlated with the highest number of selected genotypes (478) located in 41 genes, followed by annual range (127) and annualmean temperature (87). We conclude that environmental variables, especially theday-to-night temperature oscillation in comparison to the summer-to-winteroscillation, may have an important role in the distribution and adaptation of D. caspius.\n",
      "FAU - Mahtani-Williams, Sarita\n",
      "AU  - Mahtani-Williams S\n",
      "AD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,A-1160 Vienna, Austria.\n",
      "AD  - Cardiff School of Biosciences, Cardiff University, The Sir Martin Evans Building,Museum Ave, Cardiff CF103AX, UK.\n",
      "AD  - Fundacion Charles Darwin, Avenida Charles Darwin s/n, Casilla 200144, PuertoAyora EC-200350, Ecuador.\n",
      "FAU - Fulton, William\n",
      "AU  - Fulton W\n",
      "AUID- ORCID: 0000-0002-2803-5008\n",
      "AD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,A-1160 Vienna, Austria.\n",
      "AD  - Cardiff School of Biosciences, Cardiff University, The Sir Martin Evans Building,Museum Ave, Cardiff CF103AX, UK.\n",
      "FAU - Desvars-Larrive, Amelie\n",
      "AU  - Desvars-Larrive A\n",
      "AUID- ORCID: 0000-0001-7671-696X\n",
      "AD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,A-1160 Vienna, Austria.\n",
      "AD  - Institute of Food Safety, Food Technology and Veterinary Public Health, VetmeduniVienna, Veterinaerplatz 1, A-1210 Vienna, Austria.\n",
      "AD  - Complexity Science Hub Vienna, Josefstadter Strasse 39, A-1080 Vienna, Austria.\n",
      "FAU - Lado, Sara\n",
      "AU  - Lado S\n",
      "AD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,A-1160 Vienna, Austria.\n",
      "FAU - Elbers, Jean Pierre\n",
      "AU  - Elbers JP\n",
      "AD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,A-1160 Vienna, Austria.\n",
      "FAU - Halpern, Balint\n",
      "AU  - Halpern B\n",
      "AD  - MME Birdlife Hungary, Kolto utca 21., H-1121 Budapest, Hungary.\n",
      "FAU - Herczeg, David\n",
      "AU  - Herczeg D\n",
      "AUID- ORCID: 0000-0001-5098-961X\n",
      "AD  - Lendulet Evolutionary Ecology Research Group, Centre for Agricultural Research,Plant Protection Institute, Herman Otto ut 15., H-1022 Budapest, Hungary.\n",
      "FAU - Babocsay, Gergely\n",
      "AU  - Babocsay G\n",
      "AD  - MME Birdlife Hungary, Kolto utca 21., H-1121 Budapest, Hungary.\n",
      "AD  - Matra Museum of the Hungarian Natural History Museum, Kossuth Lajos utca 40.,H-3200 Gyongyos, Hungary.\n",
      "FAU - Laus, Boris\n",
      "AU  - Laus B\n",
      "AD  - Association HYLA, Lipocac I., No. 7, C-10000 Zagreb, Croatia.\n",
      "FAU - Nagy, Zoltan Tamas\n",
      "AU  - Nagy ZT\n",
      "AUID- ORCID: 0000-0002-8455-040X\n",
      "AD  - Independent Researcher, Hielscherstrasse 25, D-13158 Berlin, Germany.\n",
      "FAU - Jablonski, Daniel\n",
      "AU  - Jablonski D\n",
      "AUID- ORCID: 0000-0002-5394-0114\n",
      "AD  - Department of Zoology, Comenius University in Bratislava, Ilkovicova 6, MlynskaDolina, S-84215 Bratislava, Slovakia.\n",
      "FAU - Kukushkin, Oleg\n",
      "AU  - Kukushkin O\n",
      "AD  - Department of Biodiversity Studies and Ecological Monitoring, T. I. VyazemskyKaradag Scientific Station-Nature Reserve-Branch of Institute of Biology of theSouthern Seas of the Russian Academy of Sciences, Nauki Street 24, R-298188Theodosia, Crimea.\n",
      "AD  - Department of Herpetology, Zoological Institute of the Russian Academy ofSciences, Universitetskaya Embankment 1, R-199034 Saint Petersburg, Russia.\n",
      "FAU - Orozco-terWengel, Pablo\n",
      "AU  - Orozco-terWengel P\n",
      "AUID- ORCID: 0000-0002-7951-4148\n",
      "AD  - Cardiff School of Biosciences, Cardiff University, The Sir Martin Evans Building,Museum Ave, Cardiff CF103AX, UK.\n",
      "FAU - Voros, Judit\n",
      "AU  - Voros J\n",
      "AD  - Department of Zoology, Hungarian Natural History Museum, Baross u. 13., H-1088Budapest, Hungary.\n",
      "AD  - Molecular Taxonomy Laboratory, Hungarian Natural History Museum, Ludovika ter2-6., H-1083 Budapest, Hungary.\n",
      "FAU - Burger, Pamela Anna\n",
      "AU  - Burger PA\n",
      "AUID- ORCID: 0000-0002-6941-0257\n",
      "AD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,A-1160 Vienna, Austria.\n",
      "LA  - eng\n",
      "SI  - Dryad/10.5061/dryad.mgqnk98xm\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20201017\n",
      "PL  - Switzerland\n",
      "TA  - Genes (Basel)\n",
      "JT  - Genes\n",
      "JID - 101551097\n",
      "RN  - 0 (Reptilian Proteins)\n",
      "SB  - IM\n",
      "MH  - Adaptation, Physiological\n",
      "MH  - Animals\n",
      "MH  - Asia, Western\n",
      "MH  - Biological Evolution\n",
      "MH  - Europe, Eastern\n",
      "MH  - Gene Flow\n",
      "MH  - *Genetic Variation\n",
      "MH  - *Genetics, Population\n",
      "MH  - *Genome\n",
      "MH  - Genomics/*methods\n",
      "MH  - Genotype\n",
      "MH  - Reptilian Proteins/*genetics\n",
      "MH  - Snakes/*genetics\n",
      "PMC - PMC7603136\n",
      "OTO - NOTNLM\n",
      "OT  - *Caspian whipsnake\n",
      "OT  - *adaptive evolution\n",
      "OT  - *ddRAD\n",
      "OT  - *environmental correlates\n",
      "OT  - *genetic diversity\n",
      "EDAT- 2020/10/22 06:00\n",
      "MHDA- 2021/07/17 06:00\n",
      "CRDT- 2020/10/21 01:01\n",
      "PHST- 2020/08/31 00:00 [received]\n",
      "PHST- 2020/10/02 00:00 [revised]\n",
      "PHST- 2020/10/15 00:00 [accepted]\n",
      "PHST- 2020/10/21 01:01 [entrez]\n",
      "PHST- 2020/10/22 06:00 [pubmed]\n",
      "PHST- 2021/07/17 06:00 [medline]\n",
      "AID - genes11101218 [pii]\n",
      "AID - 10.3390/genes11101218 [doi]\n",
      "PST - epublish\n",
      "SO  - Genes (Basel). 2020 Oct 17;11(10). pii: genes11101218. doi:10.3390/genes11101218.\n",
      "\n",
      "PMID- 33062287\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20210728\n",
      "IS  - 2054-345X (Print)\n",
      "IS  - 2054-345X (Linking)\n",
      "VI  - 7\n",
      "DP  - 2020\n",
      "TI  - A toddler with phylloid-type pigmentary mosaicism and ambiguous genitaliaresulting from trisomy 14 induced by a der(Y)t(Y;14).\n",
      "PG  - 28\n",
      "LID - 10.1038/s41439-020-00113-x [doi]\n",
      "AB  - A 1-year-old baby with phylloid-type pigmentary mosaicism, hypotonia, ambiguousgenitalia, and a positive screening test for congenital adrenal hyperplasia wasreferred. Previous sonograph, cytogenetics, and metabolic profile wereinconclusive, therefore we performed an additional karyotype and a molecularcytogenetics studies. A mosaic karyotype 45,X/46,X,der(Y)t(Y;14) wascharacterized in peripheral blood. Congenital adrenal hyperplasia genes weresequenced and the results were negative. The ambiguous genitalia was the resultof the special gonosomal mosaicism. The low level of trisomy 14 led to minorphysical characteristics and mild mental retardation; also, Turner syndromefeatures can be expected rather than severe trisomy 14 stigmata.\n",
      "CI  - (c) The Author(s) 2020.\n",
      "FAU - Romero, V I\n",
      "AU  - Romero VI\n",
      "AD  - School of Medicine, Universidad San Francisco de Quito, Quito,Ecuador.grid.412251.10000 0000 9008 4711\n",
      "FAU - Pozo, J C\n",
      "AU  - Pozo JC\n",
      "AD  - School of Medicine, Universidad San Francisco de Quito, Quito,Ecuador.grid.412251.10000 0000 9008 4711\n",
      "FAU - Saenz, S\n",
      "AU  - Saenz S\n",
      "AD  - School of Medicine, Universidad San Francisco de Quito, Quito,Ecuador.grid.412251.10000 0000 9008 4711\n",
      "FAU - Llamos-Paneque, A\n",
      "AU  - Llamos-Paneque A\n",
      "AD  - Specialty Army Hospital No. 1. Medical Genetic Services, Sciences of LifeFaculty, School of Dentistry, International University of Ecuador, Quito,Ecuador.grid.442217.60000 0001 0435 9828\n",
      "FAU - Liehr, T\n",
      "AU  - Liehr T\n",
      "AD  - Institute of Human Genetics, Jena University Hospital, Friedrich SchillerUniversity, Am Klinikum 1, D-07747 Jena, Germany.grid.9613.d0000 0001 1939 2794\n",
      "FAU - Hosomichi, K\n",
      "AU  - Hosomichi K\n",
      "AUID- ORCID: 0000-0003-2712-1825\n",
      "AD  - Department of Bioinformatics and Genomics, Kanazawa University, Kanazawa,Japan.grid.9707.90000 0001 2308 3329\n",
      "FAU - Tajima, A\n",
      "AU  - Tajima A\n",
      "AUID- ORCID: 0000-0001-6808-5491\n",
      "AD  - Department of Bioinformatics and Genomics, Kanazawa University, Kanazawa,Japan.grid.9707.90000 0001 2308 3329\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20200925\n",
      "PL  - England\n",
      "TA  - Hum Genome Var\n",
      "JT  - Human genome variation\n",
      "JID - 101652445\n",
      "PMC - PMC7519037\n",
      "OTO - NOTNLM\n",
      "OT  - Genetic testing\n",
      "OT  - Genetics research\n",
      "OT  - Mosaicism\n",
      "OT  - Paediatrics\n",
      "COIS- Conflict of interestThe authors declare that they have no conflict of interest.\n",
      "EDAT- 2020/10/17 06:00\n",
      "MHDA- 2020/10/17 06:01\n",
      "CRDT- 2020/10/16 05:54\n",
      "PHST- 2020/06/10 00:00 [received]\n",
      "PHST- 2020/07/14 00:00 [revised]\n",
      "PHST- 2020/07/15 00:00 [accepted]\n",
      "PHST- 2020/10/16 05:54 [entrez]\n",
      "PHST- 2020/10/17 06:00 [pubmed]\n",
      "PHST- 2020/10/17 06:01 [medline]\n",
      "AID - 10.1038/s41439-020-00113-x [doi]\n",
      "AID - 113 [pii]\n",
      "PST - epublish\n",
      "SO  - Hum Genome Var. 2020 Sep 25;7:28. doi: 10.1038/s41439-020-00113-x. eCollection2020.\n",
      "\n",
      "PMID- 32970683\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20201026\n",
      "LR  - 20201026\n",
      "IS  - 1932-6203 (Electronic)\n",
      "IS  - 1932-6203 (Linking)\n",
      "VI  - 15\n",
      "IP  - 9\n",
      "DP  - 2020\n",
      "TI  - Does evolution of echolocation calls and morphology in Molossus result fromconvergence or stasis?\n",
      "PG  - e0238261\n",
      "LID - 10.1371/journal.pone.0238261 [doi]\n",
      "AB  - Although many processes of diversification have been described to explainvariation of morphological traits within clades that have obvious differentiationamong taxa, not much is known about these patterns in complexes of crypticspecies. Molossus is a genus of bats that is mainly Neotropical, occurring fromthe southeastern United States to southern Argentina, including the Caribbeanislands. Molossus comprises some groups of species that are morphologicallysimilar but phylogenetically divergent, and other groups of species that aregenetically similar but morphologically distinct. This contrast allowsinvestigation of unequal trait diversification and the evolution of morphologicaland behavioural characters. In this study, we assessed the role of phylogenetichistory in a genus of bat with three cryptic species complexes, and evaluated if morphology and behavior are evolving concertedly. The Genotype by Sequencegenomic approach was used to build a species-level phylogenetic tree for Molossusand to estimate the ancestral states of morphological and echolocation callcharacters. We measured the correlation of phylogenetic distances tomorphological and echolocation distances, and tested the relationship betweenmorphology and behavior when the effect of phylogeny is removed. Morphologyevolved via a mosaic of convergence and stasis, whereas call design wasinfluenced exclusively through local adaptation and convergent evolution.Furthermore, the frequency of echolocation calls is negatively correlated withthe size of the bat, but other characters do not seem to be evolving in concert. We hypothesize that slight variation in both morphology and behaviour amongspecies of the genus might result from niche specialization, and that traitsevolve to avoid competition for resources in similar environments.\n",
      "FAU - Loureiro, Livia O\n",
      "AU  - Loureiro LO\n",
      "AUID- ORCID: 0000-0003-0098-7901\n",
      "AD  - Hospital for Sick Children SickKids Learning Institute, The Centre for AppliedGenomics, Toronto, Ontario, Canada.\n",
      "AD  - Department of Natural History, Royal Ontario Museum, Toronto, Ontario, Canada.\n",
      "AD  - Department of Ecology and Evolutionary Biology, University of Toronto, Toronto,Ontario, Canada.\n",
      "FAU - Engstrom, Mark D\n",
      "AU  - Engstrom MD\n",
      "AD  - Department of Natural History, Royal Ontario Museum, Toronto, Ontario, Canada.\n",
      "AD  - Department of Ecology and Evolutionary Biology, University of Toronto, Toronto,Ontario, Canada.\n",
      "FAU - Lim, Burton K\n",
      "AU  - Lim BK\n",
      "AD  - Department of Natural History, Royal Ontario Museum, Toronto, Ontario, Canada.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20200924\n",
      "PL  - United States\n",
      "TA  - PLoS One\n",
      "JT  - PloS one\n",
      "JID - 101285081\n",
      "SB  - IM\n",
      "MH  - *Adaptation, Physiological\n",
      "MH  - Animals\n",
      "MH  - *Biological Evolution\n",
      "MH  - Chiroptera/*anatomy & histology/*physiology\n",
      "MH  - Echolocation/*physiology\n",
      "MH  - *Phylogeny\n",
      "MH  - Species Specificity\n",
      "PMC - PMC7514107\n",
      "COIS- Neotropical fieldwork has received additional financial support in Ecuador byEcuambiente Consulting Group. This does not alter our adherence to PLOS ONEpolicies on sharing data and materials.\n",
      "EDAT- 2020/09/25 06:00\n",
      "MHDA- 2020/10/27 06:00\n",
      "CRDT- 2020/09/24 17:14\n",
      "PHST- 2020/05/12 00:00 [received]\n",
      "PHST- 2020/08/12 00:00 [accepted]\n",
      "PHST- 2020/09/24 17:14 [entrez]\n",
      "PHST- 2020/09/25 06:00 [pubmed]\n",
      "PHST- 2020/10/27 06:00 [medline]\n",
      "AID - 10.1371/journal.pone.0238261 [doi]\n",
      "AID - PONE-D-20-14140 [pii]\n",
      "PST - epublish\n",
      "SO  - PLoS One. 2020 Sep 24;15(9):e0238261. doi: 10.1371/journal.pone.0238261.eCollection 2020.\n",
      "\n",
      "PMID- 32942623\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210302\n",
      "LR  - 20210302\n",
      "IS  - 1999-4915 (Electronic)\n",
      "IS  - 1999-4915 (Linking)\n",
      "VI  - 12\n",
      "IP  - 9\n",
      "DP  - 2020 Sep 15\n",
      "TI  - Transmission of the Bean-Associated Cytorhabdovirus by the Whitefly Bemisiatabaci MEAM1.\n",
      "LID - E1028 [pii]\n",
      "LID - 10.3390/v12091028 [doi]\n",
      "AB  - The knowledge of genomic data of new plant viruses is increasing exponentially;however, some aspects of their biology, such as vectors and host range, remainmostly unknown. This information is crucial for the understanding of virus-plant interactions, control strategies, and mechanisms to prevent outbreaks. Typically,rhabdoviruses infect monocot and dicot plants and are vectored in nature byhemipteran sap-sucking insects, including aphids, leafhoppers, and planthoppers. However, several strains of a potentially whitefly-transmitted virus, papayacytorhabdovirus, were recently described: (i) bean-associated cytorhabdovirus(BaCV) in Brazil, (ii) papaya virus E (PpVE) in Ecuador, and (iii)citrus-associated rhabdovirus (CiaRV) in China. Here, we examine the potential ofthe Bemisia tabaci Middle East-Asia Minor 1 (MEAM1) to transmit BaCV, itsmorphological and cytopathological characteristics, and assess the incidence ofBaCV across bean producing areas in Brazil. Our results show that BaCV isefficiently transmitted, in experimental conditions, by B. tabaci MEAM1 to beancultivars, and with lower efficiency to cowpea and soybean. Moreover, we detectedBaCV RNA in viruliferous whiteflies but we were unable to visualize viralparticles or viroplasm in the whitefly tissues. BaCV could not be singly isolatedfor pathogenicity tests, identification of the induced symptoms, and thetransmission assay. BaCV was detected in five out of the seven states in Brazilincluded in our study, suggesting that it is widely distributed throughout beanproducing areas in the country. This is the first report of awhitefly-transmitted rhabdovirus.\n",
      "FAU - Pinheiro-Lima, Bruna\n",
      "AU  - Pinheiro-Lima B\n",
      "AD  - Embrapa Recursos Geneticos e Biotecnologia, Brasilia DF 70770-017, Brazil.\n",
      "AD  - Departamento de Fitopatologia, Instituto de Biologia, Universidade de Brasilia,Brasilia DF 70275-970, Brazil.\n",
      "AD  - Departamento de Biologia Celular, Instituto de Biologia, Universidade deBrasilia, Brasilia DF 70275-970, Brazil.\n",
      "FAU - Pereira-Carvalho, Rita C\n",
      "AU  - Pereira-Carvalho RC\n",
      "AD  - Departamento de Fitopatologia, Instituto de Biologia, Universidade de Brasilia,Brasilia DF 70275-970, Brazil.\n",
      "FAU - Alves-Freitas, Dione M T\n",
      "AU  - Alves-Freitas DMT\n",
      "AUID- ORCID: 0000-0002-0443-7598\n",
      "AD  - Embrapa Recursos Geneticos e Biotecnologia, Brasilia DF 70770-017, Brazil.\n",
      "FAU - Kitajima, Elliot W\n",
      "AU  - Kitajima EW\n",
      "AD  - Departamento de Fitopatologia, Escola Superior de Agricultura Luiz de Queiroz,Piracicaba SP 13418-900, Brazil.\n",
      "FAU - Vidal, Andreza H\n",
      "AU  - Vidal AH\n",
      "AD  - Embrapa Recursos Geneticos e Biotecnologia, Brasilia DF 70770-017, Brazil.\n",
      "AD  - Departamento de Biologia Celular, Instituto de Biologia, Universidade deBrasilia, Brasilia DF 70275-970, Brazil.\n",
      "FAU - Lacorte, Cristiano\n",
      "AU  - Lacorte C\n",
      "AD  - Embrapa Recursos Geneticos e Biotecnologia, Brasilia DF 70770-017, Brazil.\n",
      "FAU - Godinho, Marcio T\n",
      "AU  - Godinho MT\n",
      "AD  - Embrapa Recursos Geneticos e Biotecnologia, Brasilia DF 70770-017, Brazil.\n",
      "FAU - Fontenele, Rafaela S\n",
      "AU  - Fontenele RS\n",
      "AD  - The Biodesign Center for Fundamental and Applied Microbiomics, Center forEvolution and Medicine School of Life Sciences, Arizona State University, Tempe, AZ 85287-5001, USA.\n",
      "FAU - Faria, Josias C\n",
      "AU  - Faria JC\n",
      "AUID- ORCID: 0000-0002-8125-7654\n",
      "AD  - Embrapa Arroz e Feijao, Goiania GO 75375-000, Brazil.\n",
      "FAU - Abreu, Emanuel F M\n",
      "AU  - Abreu EFM\n",
      "AD  - Embrapa Recursos Geneticos e Biotecnologia, Brasilia DF 70770-017, Brazil.\n",
      "FAU - Varsani, Arvind\n",
      "AU  - Varsani A\n",
      "AUID- ORCID: 0000-0003-4111-2415\n",
      "AD  - The Biodesign Center for Fundamental and Applied Microbiomics, Center forEvolution and Medicine School of Life Sciences, Arizona State University, Tempe, AZ 85287-5001, USA.\n",
      "AD  - Structural Biology Research Unit, Department of Integrative Biomedical Sciences, University of Cape Town, Observatory, Cape Town 7701, South Africa.\n",
      "FAU - Ribeiro, Simone G\n",
      "AU  - Ribeiro SG\n",
      "AUID- ORCID: 0000-0002-4965-1424\n",
      "AD  - Embrapa Recursos Geneticos e Biotecnologia, Brasilia DF 70770-017, Brazil.\n",
      "FAU - Melo, Fernando L\n",
      "AU  - Melo FL\n",
      "AUID- ORCID: 0000-0002-9591-0239\n",
      "AD  - Departamento de Fitopatologia, Instituto de Biologia, Universidade de Brasilia,Brasilia DF 70275-970, Brazil.\n",
      "AD  - Departamento de Biologia Celular, Instituto de Biologia, Universidade deBrasilia, Brasilia DF 70275-970, Brazil.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20200915\n",
      "PL  - Switzerland\n",
      "TA  - Viruses\n",
      "JT  - Viruses\n",
      "JID - 101509722\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Biological Evolution\n",
      "MH  - Brazil\n",
      "MH  - Carica/virology\n",
      "MH  - China\n",
      "MH  - Ecuador\n",
      "MH  - Genomics\n",
      "MH  - Hemiptera/*virology\n",
      "MH  - Middle East\n",
      "MH  - Plant Diseases/*virology\n",
      "MH  - Plant Leaves/virology\n",
      "MH  - Plant Viruses\n",
      "MH  - Plants/virology\n",
      "MH  - Rhabdoviridae/classification/genetics/*isolation & purification\n",
      "MH  - Rhabdoviridae Infections/*transmission/*virology\n",
      "MH  - Sequence Analysis\n",
      "PMC - PMC7551397\n",
      "OTO - NOTNLM\n",
      "OT  - *Bemisia tabaci\n",
      "OT  - *Phaseolus vulgaris\n",
      "OT  - *common bean\n",
      "OT  - *cytorhabdovirus\n",
      "OT  - *vector\n",
      "OT  - *virus evolution\n",
      "OT  - *virus transmission\n",
      "OT  - *whitefly\n",
      "EDAT- 2020/09/19 06:00\n",
      "MHDA- 2021/03/03 06:00\n",
      "CRDT- 2020/09/18 01:02\n",
      "PHST- 2020/08/04 00:00 [received]\n",
      "PHST- 2020/09/04 00:00 [revised]\n",
      "PHST- 2020/09/11 00:00 [accepted]\n",
      "PHST- 2020/09/18 01:02 [entrez]\n",
      "PHST- 2020/09/19 06:00 [pubmed]\n",
      "PHST- 2021/03/03 06:00 [medline]\n",
      "AID - v12091028 [pii]\n",
      "AID - 10.3390/v12091028 [doi]\n",
      "PST - epublish\n",
      "SO  - Viruses. 2020 Sep 15;12(9). pii: v12091028. doi: 10.3390/v12091028.\n",
      "\n",
      "PMID- 32931955\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20211104\n",
      "LR  - 20211104\n",
      "IS  - 1567-7257 (Electronic)\n",
      "IS  - 1567-1348 (Linking)\n",
      "VI  - 85\n",
      "DP  - 2020 Nov\n",
      "TI  - Subtractive proteomics and immunoinformatics approaches to explore Bartonellabacilliformis proteome (virulence factors) to design B and T cell multi-epitopesubunit vaccine.\n",
      "PG  - 104551\n",
      "LID - S1567-1348(20)30382-8 [pii]\n",
      "LID - 10.1016/j.meegid.2020.104551 [doi]\n",
      "AB  - Bartonella bacilliformis a gram-negative facultative aerobe responsible for theCarrion's disease widely distributed in Ecuador, Peru, and Colombia with a highmortality rate when no specific treatment is received. B bacilliformis istransmitted by Sand fly (Lutzomyia verrucarum) to healthy individuals.Immunoinformatic and subtractive proteomics approaches were employed in thisstudy to prioritize the best candidates for vaccine designing. These approachesresulted in five vaccine candidates, flagellar biosynthetic protein (Uniprot ID: A1UTU1), heme exporter protein C (UniProt ID: A1UU82), Cytochrome c-typebiogenesis protein (Uniprot ID: A1URZ7), Hemin ABC transporter (Uniprot ID:A1US20) and Phosphatidate cytidylyltransferase (Uniprot ID: A1USE3). Thementioned proteins are antigenic and essential for pathogen survival. A range of immune-informatics tools was applied for the prediction of B and T cell epitopes for the vaccine candidate proteins. In-silico vaccine was constructed usingcarefully evaluated epitopes and consequently modeled for docking with humanToll-like receptor 4. TLR-4 agonist 50S ribosomal protein L7/L12 (UniproKB ID;P9WHE3) was linked to the vaccine as an adjuvant to boost immune response towardsthe vaccine. For stability evaluation of the vaccine-TLR-4 docked complex, MDsimulations were performed. The final vaccine was back-translated and cloned inEschericia coli to attain the maximal expression of the vaccine protein. Themaximal expression was ensured, and the CAI score of 0.96 was reported. Thecurrent vaccine requires future experimental validation to confirm itseffectiveness. The vaccine developed will be helpful to protect against Bbacilliformis associated infections.\n",
      "CI  - Copyright (c) 2020 Elsevier B.V. All rights reserved.\n",
      "FAU - Gul, Hina\n",
      "AU  - Gul H\n",
      "AD  - Centre for Biotechnology and Microbiology, University of Swat, Swat, KhyberPakhtunkhwa, Pakistan.\n",
      "FAU - Ali, Syed Shujait\n",
      "AU  - Ali SS\n",
      "AD  - Centre for Biotechnology and Microbiology, University of Swat, Swat, KhyberPakhtunkhwa, Pakistan. Electronic address: shujaitswati@uswat.edu.pk.\n",
      "FAU - Saleem, Shoaib\n",
      "AU  - Saleem S\n",
      "AD  - National Center for Bioinformatics, Quaid-e-Azam University, Islamabad, Pakistan.\n",
      "FAU - Khan, Shahzeb\n",
      "AU  - Khan S\n",
      "AD  - Centre for Biotechnology and Microbiology, University of Swat, Swat, KhyberPakhtunkhwa, Pakistan.\n",
      "FAU - Khan, Jafar\n",
      "AU  - Khan J\n",
      "AD  - Centre for Biotechnology and Microbiology, University of Swat, Swat, KhyberPakhtunkhwa, Pakistan. Electronic address: Jafar@uswat.edu.pk.\n",
      "FAU - Wadood, Abdul\n",
      "AU  - Wadood A\n",
      "AD  - Department of Biochemistry, Abdul Wali Khan University, Mardan, KhyberPakhtunkhwa, Pakistan. Electronic address: awadood@awkum.edu.pk.\n",
      "FAU - Rehman, Ashfaq Ur\n",
      "AU  - Rehman AU\n",
      "AD  - Department of Bioinformatics and Biological statistics, School of Life Sciencesand Biotechnology, Shanghai Jiao Tong University, Shanghai 200240, PR China.Electronic address: raysjtu@sjtu.edu.cn.\n",
      "FAU - Ullah, Zia\n",
      "AU  - Ullah Z\n",
      "AD  - Centre for Biotechnology and Microbiology, University of Swat, Swat, KhyberPakhtunkhwa, Pakistan.\n",
      "FAU - Ali, Shahid\n",
      "AU  - Ali S\n",
      "AD  - Centre for Biotechnology and Microbiology, University of Swat, Swat, KhyberPakhtunkhwa, Pakistan. Electronic address: shahidali@uswat.edu.pk.\n",
      "FAU - Khan, Haji\n",
      "AU  - Khan H\n",
      "AD  - Centre for Biotechnology and Microbiology, University of Swat, Swat, KhyberPakhtunkhwa, Pakistan.\n",
      "FAU - Hussain, Zahid\n",
      "AU  - Hussain Z\n",
      "AD  - Centre for Biotechnology and Microbiology, University of Swat, Swat, KhyberPakhtunkhwa, Pakistan. Electronic address: zahidhussain@uswat.edu.pk.\n",
      "FAU - Akbar, Fazal\n",
      "AU  - Akbar F\n",
      "AD  - Centre for Biotechnology and Microbiology, University of Swat, Swat, KhyberPakhtunkhwa, Pakistan. Electronic address: fazalakbar@uswat.edu.pk.\n",
      "FAU - Khan, Abbas\n",
      "AU  - Khan A\n",
      "AD  - Department of Bioinformatics and Biological statistics, School of Life Sciencesand Biotechnology, Shanghai Jiao Tong University, Shanghai 200240, PR China.Electronic address: abbaskhan@sjtu.edu.cn.\n",
      "FAU - Wei, Dong-Qing\n",
      "AU  - Wei DQ\n",
      "AD  - Department of Bioinformatics and Biological statistics, School of Life Sciencesand Biotechnology, Shanghai Jiao Tong University, Shanghai 200240, PR China;State Key Laboratory of Microbial Metabolism, Shanghai-Islamabad-Belgrade JointInnovation Center on Antibacterial Resistances, Joint Laboratory of InternationalCooperation in Metabolic and Developmental Sciences, Ministry of Education andSchool of Life Sciences and Biotechnology, Shanghai Jiao Tong University,Shanghai 200030, PR China; Peng Cheng Laboratory, Vanke Cloud City Phase IBuilding 8, Xili Street, Nashan District, Shenzhen, Guangdong 518055, PR China.Electronic address: dqwei@sjtu.edu.cn.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20200912\n",
      "PL  - Netherlands\n",
      "TA  - Infect Genet Evol\n",
      "JT  - Infection, genetics and evolution : journal of molecular epidemiology andevolutionary genetics in infectious diseases\n",
      "JID - 101084138\n",
      "RN  - 0 (Bacterial Vaccines)\n",
      "RN  - 0 (Epitopes)\n",
      "RN  - 0 (Epitopes, B-Lymphocyte)\n",
      "RN  - 0 (Epitopes, T-Lymphocyte)\n",
      "RN  - 0 (Proteome)\n",
      "RN  - 0 (Virulence Factors)\n",
      "SB  - IM\n",
      "MH  - Bacterial Vaccines/genetics/*immunology\n",
      "MH  - Bartonella Infections/immunology/*microbiology/prevention & control\n",
      "MH  - Bartonella bacilliformis/immunology/*metabolism/pathogenicity\n",
      "MH  - *Computational Biology/methods\n",
      "MH  - *Epitopes/chemistry/immunology\n",
      "MH  - Epitopes, B-Lymphocyte/chemistry/immunology\n",
      "MH  - Epitopes, T-Lymphocyte/chemistry/immunology\n",
      "MH  - Host-Pathogen Interactions/immunology\n",
      "MH  - Humans\n",
      "MH  - Models, Molecular\n",
      "MH  - *Proteome\n",
      "MH  - *Proteomics\n",
      "MH  - Structure-Activity Relationship\n",
      "MH  - Vaccinology\n",
      "MH  - Virulence Factors/immunology\n",
      "OTO - NOTNLM\n",
      "OT  - *Bartonella bacilliformis\n",
      "OT  - *Docking\n",
      "OT  - *Molecular docking\n",
      "OT  - *Simulation\n",
      "OT  - *Vaccine\n",
      "EDAT- 2020/09/16 06:00\n",
      "MHDA- 2021/11/05 06:00\n",
      "CRDT- 2020/09/15 20:12\n",
      "PHST- 2020/04/10 00:00 [received]\n",
      "PHST- 2020/08/27 00:00 [revised]\n",
      "PHST- 2020/09/07 00:00 [accepted]\n",
      "PHST- 2020/09/16 06:00 [pubmed]\n",
      "PHST- 2021/11/05 06:00 [medline]\n",
      "PHST- 2020/09/15 20:12 [entrez]\n",
      "AID - S1567-1348(20)30382-8 [pii]\n",
      "AID - 10.1016/j.meegid.2020.104551 [doi]\n",
      "PST - ppublish\n",
      "SO  - Infect Genet Evol. 2020 Nov;85:104551. doi: 10.1016/j.meegid.2020.104551. Epub2020 Sep 12.\n",
      "\n",
      "PMID- 32912150\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210514\n",
      "LR  - 20210514\n",
      "IS  - 1471-2164 (Electronic)\n",
      "IS  - 1471-2164 (Linking)\n",
      "VI  - 21\n",
      "IP  - Suppl 7\n",
      "DP  - 2020 Sep 10\n",
      "TI  - Tracing the genetic history of the 'Canaris' from Ecuador and Peru usinguniparental DNA markers.\n",
      "PG  - 413\n",
      "LID - 10.1186/s12864-020-06834-1 [doi]\n",
      "AB  - BACKGROUND: According to history, in the pre-Hispanic period, during the conquestand Inka expansion in Ecuador, many Andean families of the Canar region wouldhave been displaced to several places of Tawantinsuyu, including Kanaris, aQuechua-speaking community located at the highlands of the Province of Ferrenafe,Lambayeque (Peru). Other families were probably taken from the Central Andes to aplace close to Kanaris, named Inkawasi. Evidence of this migration comes from thepresence near the Kanaris-Inkawasi communities of a village, a former Inka camp, which persists until the present day. This scenario could explain these toponyms,but it is still controversial. To clarify this historical question, the studypresented here focused on the inference of the genetic relationship between'Canaris' populations, particularly of Canar and Ferrenafe, compared to otherhighland populations. We analysed native patrilineal Y chromosome haplotypescomposed of 15 short tandem repeats, a set of SNPs, and maternal mitochondrialDNA haplotypes of control region sequences. RESULTS: After the geneticcomparisons of local populations-three from Ecuador and seven from Peru-, Ychromosome analyses (n = 376) indicated that individuals from the Canar region donot share Y haplotypes with the Kanaris, or even with those of the Inkawasi.However, some Y haplotypes of Ecuadorian 'Canaris' were associated withhaplotypes of the Peruvian populations of Cajamarca, Chivay (Arequipa), Cusco andLake Titicaca, an observation that is congruent with colonial records. Within theKanaris and Inkawasi communities there are at least five clans in which severalindividuals share haplotypes, indicating that they have recent common ancestors. Despite their relative isolation, most individuals of both communities arerelated to those of the Cajamarca and Chachapoyas in Peru, consistent with thespoken Quechua and their geographic proximity. With respect to mitochondrial DNA haplotypes (n = 379), with the exception of a shared haplotype of the D1 lineage between the Canar and Kanaris, there are no genetic affinities. CONCLUSION:Although there is no close genetic relationship between the Peruvian Kanaris(including Inkawasi) and Ecuadorian Canar populations, our results showed somecongruence with historical records.\n",
      "FAU - Sandoval, Jose R\n",
      "AU  - Sandoval JR\n",
      "AUID- ORCID: http://orcid.org/0000-0003-2673-4937\n",
      "AD  - Centro de Investigacion de Genetica y Biologia Molecular (CIGBM), Instituto deInvestigacion, Facultad de Medicina, Universidad de San Martin de Porres, Lima,Peru. jsandovals@usmp.pe.\n",
      "AD  - Laboratorio de Biodiversidade e Evolucao Molecular (LBEM), Instituto de Ciencias Biologicas, Universidade Federal de Minas Gerais, Belo Horizonte, Brazil.jsandovals@usmp.pe.\n",
      "FAU - Lacerda, Daniela R\n",
      "AU  - Lacerda DR\n",
      "AD  - Laboratorio de Biodiversidade e Evolucao Molecular (LBEM), Instituto de Ciencias Biologicas, Universidade Federal de Minas Gerais, Belo Horizonte, Brazil.\n",
      "FAU - Jota, Marilza M S\n",
      "AU  - Jota MMS\n",
      "AD  - Laboratorio de Biodiversidade e Evolucao Molecular (LBEM), Instituto de Ciencias Biologicas, Universidade Federal de Minas Gerais, Belo Horizonte, Brazil.\n",
      "FAU - Robles-Ruiz, Paulo\n",
      "AU  - Robles-Ruiz P\n",
      "AD  - Instituto de Investigaciones Biomedicas, Universidad de las Americas, Quito,Ecuador.\n",
      "FAU - Danos, Pierina\n",
      "AU  - Danos P\n",
      "AD  - Centro de Investigacion de Genetica y Biologia Molecular (CIGBM), Instituto deInvestigacion, Facultad de Medicina, Universidad de San Martin de Porres, Lima,Peru.\n",
      "FAU - Paz-Y-Mino, Cesar\n",
      "AU  - Paz-Y-Mino C\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Universidad Tecnologica Equinoccial,Quito, Ecuador.\n",
      "FAU - Wells, Spencer\n",
      "AU  - Wells S\n",
      "AD  - Department of Integrative Biology, University of Texas at Austin, Austin, TX,USA.\n",
      "FAU - Santos, Fabricio R\n",
      "AU  - Santos FR\n",
      "AD  - Laboratorio de Biodiversidade e Evolucao Molecular (LBEM), Instituto de Ciencias Biologicas, Universidade Federal de Minas Gerais, Belo Horizonte, Brazil.\n",
      "FAU - Fujita, Ricardo\n",
      "AU  - Fujita R\n",
      "AD  - Centro de Investigacion de Genetica y Biologia Molecular (CIGBM), Instituto deInvestigacion, Facultad de Medicina, Universidad de San Martin de Porres, Lima,Peru.\n",
      "LA  - eng\n",
      "GR  - The Genographic Project 2.0 grant, entitled Origins and descent of the IncaEmpire./National Geographic Society\n",
      "GR  - E10012017010/Universidad de San Martin de Porres Facultad de Medicina Humana\n",
      "PT  - Journal Article\n",
      "DEP - 20200910\n",
      "PL  - England\n",
      "TA  - BMC Genomics\n",
      "JT  - BMC genomics\n",
      "JID - 100965258\n",
      "RN  - 0 (DNA, Mitochondrial)\n",
      "RN  - 0 (Genetic Markers)\n",
      "SB  - IM\n",
      "MH  - *Chromosomes, Human, Y\n",
      "MH  - DNA, Mitochondrial/genetics\n",
      "MH  - Ecuador\n",
      "MH  - Genetic Markers\n",
      "MH  - Genetic Variation\n",
      "MH  - Genetics, Population\n",
      "MH  - Haplotypes\n",
      "MH  - Humans\n",
      "MH  - *Indians, South American/genetics\n",
      "MH  - Peru\n",
      "PMC - PMC7488242\n",
      "OTO - NOTNLM\n",
      "OT  - Canaris\n",
      "OT  - Mitochondrial DNA\n",
      "OT  - Population genetics\n",
      "OT  - SNP\n",
      "OT  - Short tandem repeats\n",
      "OT  - Y chromosome\n",
      "EDAT- 2020/09/12 06:00\n",
      "MHDA- 2021/05/15 06:00\n",
      "CRDT- 2020/09/11 05:30\n",
      "PHST- 2020/05/15 00:00 [received]\n",
      "PHST- 2020/06/16 00:00 [accepted]\n",
      "PHST- 2020/09/11 05:30 [entrez]\n",
      "PHST- 2020/09/12 06:00 [pubmed]\n",
      "PHST- 2021/05/15 06:00 [medline]\n",
      "AID - 10.1186/s12864-020-06834-1 [doi]\n",
      "AID - 10.1186/s12864-020-06834-1 [pii]\n",
      "PST - epublish\n",
      "SO  - BMC Genomics. 2020 Sep 10;21(Suppl 7):413. doi: 10.1186/s12864-020-06834-1.\n",
      "\n",
      "PMID- 32887672\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210621\n",
      "LR  - 20210621\n",
      "IS  - 2160-1836 (Electronic)\n",
      "IS  - 2160-1836 (Linking)\n",
      "VI  - 10\n",
      "IP  - 11\n",
      "DP  - 2020 Nov 5\n",
      "TI  - Genotypic Characterization of the U.S. Peanut Core Collection.\n",
      "PG  - 4013-4026\n",
      "LID - 10.1534/g3.120.401306 [doi]\n",
      "AB  - Cultivated peanut (Arachis hypogaea) is an important oil, food, and feed cropworldwide. The USDA peanut germplasm collection currently contains 8,982accessions. In the 1990s, 812 accessions were selected as a core collection onthe basis of phenotype and country of origin. The present study reportsgenotyping results for the entire available core collection. Each accession wasgenotyped with the Arachis_Axiom2 SNP array, yielding 14,430 high-quality,informative SNPs across the collection. Additionally, a subset of 253 accessions was replicated, using between two and five seeds per accession, to assessheterogeneity within these accessions. The genotypic diversity of the core ismostly captured in five genotypic clusters, which have some correspondence withbotanical variety and market type. There is little genetic clustering by country of origin, reflecting peanut's rapid global dispersion in the 18(th) and 19(th)centuries. A genetic cluster associated with the hypogaea/aequatoriana/peruviana varieties, with accessions coming primarily from Bolivia, Peru, and Ecuador, isconsistent with these having been the earliest landraces. The genetics,phenotypic characteristics, and biogeography are all consistent with previousreports of tetraploid peanut originating in Southeast Bolivia. Analysis of thegenotype data indicates an early genetic radiation, followed by regionaldistribution of major genetic classes through South America, and then a globaldissemination that retains much of the early genetic diversity in peanut.Comparison of the genotypic data relative to alleles from the diploid progenitorsalso indicates that subgenome exchanges, both large and small, have been majorcontributors to the genetic diversity in peanut.\n",
      "CI  - Copyright (c) 2020 Otyama et al.\n",
      "FAU - Otyama, Paul I\n",
      "AU  - Otyama PI\n",
      "AD  - Interdepartmental Genetics and Genomics, Iowa State University, Ames, IA.\n",
      "AD  - Agronomy Department, Iowa State University, Ames, IA.\n",
      "FAU - Kulkarni, Roshan\n",
      "AU  - Kulkarni R\n",
      "AD  - Interdepartmental Genetics and Genomics, Iowa State University, Ames, IA.\n",
      "AD  - Agronomy Department, Iowa State University, Ames, IA.\n",
      "AD  - ORISE Fellow, Corn Insects and Crop Genetics Research Unit, USDA-ARS, Ames, IA.\n",
      "FAU - Chamberlin, Kelly\n",
      "AU  - Chamberlin K\n",
      "AD  - USDA - Agricultural Research Service, Stillwater, OK.\n",
      "FAU - Ozias-Akins, Peggy\n",
      "AU  - Ozias-Akins P\n",
      "AD  - Institute of Plant Breeding, Genetics, and Genomics and Department ofHorticulture, University of Georgia, Tifton, GA.\n",
      "FAU - Chu, Ye\n",
      "AU  - Chu Y\n",
      "AD  - Institute of Plant Breeding, Genetics, and Genomics and Department ofHorticulture, University of Georgia, Tifton, GA.\n",
      "FAU - Lincoln, Lori M\n",
      "AU  - Lincoln LM\n",
      "AD  - USDA - Agricultural Research Service, Corn Insects and Crop Genetics ResearchUnit, Ames, IA.\n",
      "FAU - MacDonald, Gregory E\n",
      "AU  - MacDonald GE\n",
      "AD  - University of Florida, Gainesville, FL.\n",
      "FAU - Anglin, Noelle L\n",
      "AU  - Anglin NL\n",
      "AD  - International Potato Center, Lima, Peru.\n",
      "FAU - Dash, Sudhansu\n",
      "AU  - Dash S\n",
      "AD  - National Center for Genomic Resources, Santa Fe, NM.\n",
      "FAU - Bertioli, David J\n",
      "AU  - Bertioli DJ\n",
      "AD  - Institute of Plant Breeding, Genetics, and Genomics and Department ofHorticulture, University of Georgia, Tifton, GA.\n",
      "FAU - Fernandez-Baca, David\n",
      "AU  - Fernandez-Baca D\n",
      "AD  - Department of Computer Science, Iowa State University, Ames, IA.\n",
      "FAU - Graham, Michelle A\n",
      "AU  - Graham MA\n",
      "AD  - Agronomy Department, Iowa State University, Ames, IA.\n",
      "AD  - USDA - Agricultural Research Service, Corn Insects and Crop Genetics ResearchUnit, Ames, IA.\n",
      "FAU - Cannon, Steven B\n",
      "AU  - Cannon SB\n",
      "AD  - Agronomy Department, Iowa State University, Ames, IA.\n",
      "AD  - USDA - Agricultural Research Service, Corn Insects and Crop Genetics ResearchUnit, Ames, IA.\n",
      "FAU - Cannon, Ethalinda K S\n",
      "AU  - Cannon EKS\n",
      "AD  - USDA - Agricultural Research Service, Corn Insects and Crop Genetics ResearchUnit, Ames, IA ethy.cannon@ars.usda.gov.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20201105\n",
      "PL  - England\n",
      "TA  - G3 (Bethesda)\n",
      "JT  - G3 (Bethesda, Md.)\n",
      "JID - 101566598\n",
      "SB  - IM\n",
      "MH  - Alleles\n",
      "MH  - *Arachis/genetics\n",
      "MH  - *Genetic Variation\n",
      "MH  - Genotype\n",
      "MH  - Phylogeny\n",
      "PMC - PMC7642936\n",
      "OTO - NOTNLM\n",
      "OT  - *Arachis\n",
      "OT  - *genotype\n",
      "OT  - *germplasm core collection\n",
      "OT  - *peanut\n",
      "EDAT- 2020/09/06 06:00\n",
      "MHDA- 2021/06/22 06:00\n",
      "CRDT- 2020/09/05 05:15\n",
      "PHST- 2020/09/06 06:00 [pubmed]\n",
      "PHST- 2021/06/22 06:00 [medline]\n",
      "PHST- 2020/09/05 05:15 [entrez]\n",
      "AID - g3.120.401306 [pii]\n",
      "AID - 10.1534/g3.120.401306 [doi]\n",
      "PST - epublish\n",
      "SO  - G3 (Bethesda). 2020 Nov 5;10(11):4013-4026. doi: 10.1534/g3.120.401306.\n",
      "\n",
      "PMID- 32807182\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210510\n",
      "LR  - 20210510\n",
      "IS  - 1755-8794 (Electronic)\n",
      "IS  - 1755-8794 (Linking)\n",
      "VI  - 13\n",
      "IP  - 1\n",
      "DP  - 2020 Aug 17\n",
      "TI  - Clinical, genomics and networking analyses of a high-altitude native AmericanEcuadorian patient with congenital insensitivity to pain with anhidrosis: a case report.\n",
      "PG  - 113\n",
      "LID - 10.1186/s12920-020-00764-3 [doi]\n",
      "AB  - BACKGROUND: Congenital insensitivity to pain with anhidrosis (CIPA) is anextremely rare autosomal recessive disorder characterized by insensitivity topain, inability to sweat and intellectual disability. CIPA is caused by mutationsin the neurotrophic tyrosine kinase receptor type 1 gene (NTRK1) that encodes thehigh-affinity receptor of nerve growth factor (NGF). CASE PRESENTATION: Here, we present clinical and molecular findings in a 9-year-old girl with CIPA. Thehigh-altitude indigenous Ecuadorian patient presented several health problemssuch as anhidrosis, bone fractures, self-mutilation, osteochondroma, intellectualdisability and Riga-Fede disease. After the mutational analysis of NTRK1, thepatient showed a clearly autosomal recessive inheritance pattern with thepathogenic mutation rs763758904 (Arg602*) and the second missense mutationrs80356677 (Asp674Tyr). Additionally, the genomic analysis showed 69 pathogenicand/or likely pathogenic variants in 46 genes possibly related to phenotypicheterogeneity, including the rs324420 variant in the FAAH gene. The gene ontologyenrichment analysis showed 28 mutated genes involved in several biologicalprocesses. As a novel contribution, the protein-protein interaction networkanalysis showed that NTRK1, SPTBN2 and GRM6 interact with several proteins of thepain matrix involved in the response to stimulus and nervous system development. CONCLUSIONS: This is the first study that associates clinical, genomics andnetworking analyses in a Native American patient with consanguinity background inorder to better understand CIPA pathogenesis.\n",
      "FAU - Lopez-Cortes, Andres\n",
      "AU  - Lopez-Cortes A\n",
      "AUID- ORCID: 0000-0003-1503-1929\n",
      "AD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.aalc84@gmail.com.\n",
      "AD  - Latin American Network for Implementation and Validation of ClinicalPharmacogenomics Guidelines (RELIVAF-CYTED), Madrid, Spain. aalc84@gmail.com.\n",
      "FAU - Zambrano, Ana Karina\n",
      "AU  - Zambrano AK\n",
      "AD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\n",
      "FAU - Guevara-Ramirez, Patricia\n",
      "AU  - Guevara-Ramirez P\n",
      "AD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\n",
      "FAU - Echeverria, Byron Albuja\n",
      "AU  - Echeverria BA\n",
      "AD  - Hospital San Luis de Otavalo, Ministerio de Salud Publica, Antonio Jose de Sucre Avenue, 100201, Otavalo, Ecuador.\n",
      "FAU - Guerrero, Santiago\n",
      "AU  - Guerrero S\n",
      "AD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\n",
      "FAU - Cabascango, Eliana\n",
      "AU  - Cabascango E\n",
      "AD  - Sistemas Medicos (SIME), Universidad San Francisco de Quito, Interoceanica Avenueand Chimborazo, 170902, Cumbaya, Ecuador.\n",
      "FAU - Perez-Villa, Andy\n",
      "AU  - Perez-Villa A\n",
      "AD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\n",
      "FAU - Armendariz-Castillo, Isaac\n",
      "AU  - Armendariz-Castillo I\n",
      "AD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\n",
      "FAU - Garcia-Cardenas, Jennyfer M\n",
      "AU  - Garcia-Cardenas JM\n",
      "AD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\n",
      "FAU - Yumiceba, Veronica\n",
      "AU  - Yumiceba V\n",
      "AD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\n",
      "FAU - Perez-M, Gabriela\n",
      "AU  - Perez-M G\n",
      "AD  - Ministerio de Salud Publica, 100117, Ibarra, Ecuador.\n",
      "FAU - Leone, Paola E\n",
      "AU  - Leone PE\n",
      "AD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\n",
      "FAU - Paz-Y-Mino, Cesar\n",
      "AU  - Paz-Y-Mino C\n",
      "AD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.cesar.pazymino@ute.edu.ec.\n",
      "LA  - eng\n",
      "PT  - Case Reports\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20200817\n",
      "PL  - England\n",
      "TA  - BMC Med Genomics\n",
      "JT  - BMC medical genomics\n",
      "JID - 101319628\n",
      "RN  - 0 (Genetic Markers)\n",
      "SB  - IM\n",
      "MH  - *Altitude\n",
      "MH  - Child\n",
      "MH  - DNA Mutational Analysis\n",
      "MH  - Female\n",
      "MH  - *Genetic Markers\n",
      "MH  - Genomics\n",
      "MH  - Humans\n",
      "MH  - Hypohidrosis/genetics/metabolism/*pathology\n",
      "MH  - *Mutation\n",
      "MH  - Pain/genetics/metabolism/*pathology\n",
      "MH  - Pain Insensitivity, Congenital/genetics/metabolism/*pathology\n",
      "MH  - Protein Interaction Maps\n",
      "PMC - PMC7437939\n",
      "OTO - NOTNLM\n",
      "OT  - *CIPA\n",
      "OT  - *Ecuadorian\n",
      "OT  - *Genomics analysis\n",
      "OT  - *NTRK1\n",
      "OT  - *Native American\n",
      "EDAT- 2020/08/19 06:00\n",
      "MHDA- 2021/05/11 06:00\n",
      "CRDT- 2020/08/19 06:00\n",
      "PHST- 2020/01/02 00:00 [received]\n",
      "PHST- 2020/08/10 00:00 [accepted]\n",
      "PHST- 2020/08/19 06:00 [entrez]\n",
      "PHST- 2020/08/19 06:00 [pubmed]\n",
      "PHST- 2021/05/11 06:00 [medline]\n",
      "AID - 10.1186/s12920-020-00764-3 [doi]\n",
      "AID - 10.1186/s12920-020-00764-3 [pii]\n",
      "PST - epublish\n",
      "SO  - BMC Med Genomics. 2020 Aug 17;13(1):113. doi: 10.1186/s12920-020-00764-3.\n",
      "\n",
      "PMID- 32719690\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20200928\n",
      "IS  - 1664-462X (Print)\n",
      "IS  - 1664-462X (Linking)\n",
      "VI  - 11\n",
      "DP  - 2020\n",
      "TI  - Comparative Plastid Genomics of Neotropical Bulbophyllum (Orchidaceae;Epidendroideae).\n",
      "PG  - 799\n",
      "LID - 10.3389/fpls.2020.00799 [doi]\n",
      "AB  - Pantropical Bulbophyllum, with approximately 2,200 species, is one of the largestgenera in Orchidaceae. Although phylogenetics and taxonomy of the approximately60 American species in the genus are generally well understood, some speciescomplexes need more study to clearly delimit their component species and provide information about their evolutionary history. Previous research has suggestedthat the plastid genome includes phylogenetic markers capable of providingresolution at low taxonomic levels, and thus it could be an effective tool ifthese divergent regions can be identified. In this study, we sequenced thecomplete plastid genome of eight Bulbophyllum species, representing five of sixNeotropical taxonomic sections. All plastomes conserve the typical quadripartite structure, and, although the general structure of plastid genomes is conserved,differences in ndh-gene composition and total length were detected. Total length was determined by contraction and expansion of the small single-copy region, aresult of an independent loss of the seven ndh genes. Selection analysesindicated that protein-coding genes were generally well conserved, but in fourgenes, we identified 95 putative sites under positive selection. Furthermore, atotal of 54 polymorphic simple sequence repeats were identified, for which wedeveloped amplification primers. In addition, we propose 10 regions withpotential to improve phylogenetic analyses of Neotropical Bulbophyllum species.\n",
      "CI  - Copyright (c) 2020 Zavala-Paez, Vieira, Baura, Balsanelli, Souza, Cevallos, Chaseand Smidt.\n",
      "FAU - Zavala-Paez, Michelle\n",
      "AU  - Zavala-Paez M\n",
      "AD  - Departamento de Botanica, Universidade Federal do Parana, Curitiba, Brazil.\n",
      "FAU - Vieira, Leila do Nascimento\n",
      "AU  - Vieira LDN\n",
      "AD  - Departamento de Botanica, Universidade Federal do Parana, Curitiba, Brazil.\n",
      "FAU - de Baura, Valter Antonio\n",
      "AU  - de Baura VA\n",
      "AD  - Departamento de Bioquimica e Biologia Molecular, Universidade Federal do Parana, Curitiba, Brazil.\n",
      "FAU - Balsanelli, Eduardo\n",
      "AU  - Balsanelli E\n",
      "AD  - Departamento de Bioquimica e Biologia Molecular, Universidade Federal do Parana, Curitiba, Brazil.\n",
      "FAU - de Souza, Emanuel Maltempi\n",
      "AU  - de Souza EM\n",
      "AD  - Departamento de Bioquimica e Biologia Molecular, Universidade Federal do Parana, Curitiba, Brazil.\n",
      "FAU - Cevallos, Marco Cerna\n",
      "AU  - Cevallos MC\n",
      "AD  - Grupo de Investigacion Nunkui Wakan, Universidad Politecnica Salesiana, Quito,Ecuador.\n",
      "FAU - Chase, Mark W\n",
      "AU  - Chase MW\n",
      "AD  - Royal Botanic Gardens, Kew, Richmond, United Kingdom.\n",
      "AD  - Department of Environment and Agriculture, Curtin University, Perth, WA,Australia.\n",
      "FAU - Smidt, Eric de Camargo\n",
      "AU  - Smidt EC\n",
      "AD  - Departamento de Botanica, Universidade Federal do Parana, Curitiba, Brazil.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20200703\n",
      "PL  - Switzerland\n",
      "TA  - Front Plant Sci\n",
      "JT  - Frontiers in plant science\n",
      "JID - 101568200\n",
      "PMC - PMC7347972\n",
      "OTO - NOTNLM\n",
      "OT  - Neotropical orchids\n",
      "OT  - molecular evolution\n",
      "OT  - molecular markers\n",
      "OT  - next-generation sequencing\n",
      "OT  - plastid genome\n",
      "EDAT- 2020/07/29 06:00\n",
      "MHDA- 2020/07/29 06:01\n",
      "CRDT- 2020/07/29 06:00\n",
      "PHST- 2020/03/18 00:00 [received]\n",
      "PHST- 2020/05/19 00:00 [accepted]\n",
      "PHST- 2020/07/29 06:00 [entrez]\n",
      "PHST- 2020/07/29 06:00 [pubmed]\n",
      "PHST- 2020/07/29 06:01 [medline]\n",
      "AID - 10.3389/fpls.2020.00799 [doi]\n",
      "PST - epublish\n",
      "SO  - Front Plant Sci. 2020 Jul 3;11:799. doi: 10.3389/fpls.2020.00799. eCollection2020.\n",
      "\n",
      "PMID- 32661155\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200921\n",
      "LR  - 20210113\n",
      "IS  - 1091-6490 (Electronic)\n",
      "IS  - 0027-8424 (Linking)\n",
      "VI  - 117\n",
      "IP  - 31\n",
      "DP  - 2020 Aug 4\n",
      "TI  - Multiple origins of green coloration in frogs mediated by a novelbiliverdin-binding serpin.\n",
      "PG  - 18574-18581\n",
      "LID - 10.1073/pnas.2006771117 [doi]\n",
      "AB  - Many vertebrates have distinctive blue-green bones and other tissues due tounusually high biliverdin concentrations-a phenomenon called chlorosis. Despiteits prevalence, the biochemical basis, biology, and evolution of chlorosis arepoorly understood. In this study, we show that the occurrence of high biliverdin in anurans (frogs and toads) has evolved multiple times during their evolutionaryhistory, and relies on the same mechanism-the presence of a class of serpinfamily proteins that bind biliverdin. Using a diverse combination of techniques, we purified these serpins from several species of nonmodel treefrogs anddeveloped a pipeline that allowed us to assemble their complete amino acid andnucleotide sequences. The described proteins, hereafter named biliverdin-binding serpins (BBS), have absorption spectra that mimic those of phytochromes andbacteriophytochromes. Our models showed that physiological concentration of BBSs fine-tune the color of the animals, providing the physiological basis for crypsisin green foliage even under near-infrared light. Additionally, we found thatthese BBSs are most similar to human glycoprotein alpha-1-antitrypsin, but with aremarkable functional diversification. Our results present molecular andfunctional evidence of recurrent evolution of chlorosis, describe abiliverdin-binding protein in vertebrates, and introduce a function for a member of the serpin superfamily, the largest and most ubiquitous group of proteaseinhibitors.\n",
      "FAU - Taboada, Carlos\n",
      "AU  - Taboada C\n",
      "AUID- ORCID: 0000-0002-7074-1897\n",
      "AD  - Department of Biology, Duke University, Durham, NC 27708;carlostaboada84@gmail.com lchemes@iib.unsam.edu.ar bari@qi.fcen.uba.ar.\n",
      "AD  - Division Herpetologia, Museo Argentino de Ciencias Naturales \"BernardinoRivadavia,\" Consejo Nacional de Investigaciones Cientificas y Tecnicas (CONICET),Ciudad Autonoma de Buenos Aires C1405DJR, Argentina.\n",
      "AD  - Instituto de Quimica Fisica de los Materiales, Medio Ambiente y Energia, Consejo Nacional de Investigaciones Cientificas y Tecnicas (CONICET), Facultad deCiencias Exactas y Naturales, Universidad de Buenos Aires, Ciudad Autonoma deBuenos Aires C1428EHA, Argentina.\n",
      "AD  - Nucleo de Pesquisa em Produtos Naturais e Sinteticos (NPPNS), Departamento deCiencias BioMoleculares, Faculdade de Ciencias Farmaceuticas de Ribeirao Preto,Universidade de Sao Paulo, 14040903 Ribeirao Preto, Sao Paulo, Brazil.\n",
      "FAU - Brunetti, Andres E\n",
      "AU  - Brunetti AE\n",
      "AUID- ORCID: 0000-0001-7974-6099\n",
      "AD  - Nucleo de Pesquisa em Produtos Naturais e Sinteticos (NPPNS), Departamento deCiencias BioMoleculares, Faculdade de Ciencias Farmaceuticas de Ribeirao Preto,Universidade de Sao Paulo, 14040903 Ribeirao Preto, Sao Paulo, Brazil.\n",
      "AD  - Laboratorio de Genetica Evolutiva \"Claudio Juan Bidau,\" Instituto de BiologiaSubtropical (CONICET-UNaM), Facultad de Ciencias Exactas, Universidad Nacional deMisiones, 3300 Posadas, Misiones, Argentina.\n",
      "FAU - Lyra, Mariana L\n",
      "AU  - Lyra ML\n",
      "AUID- ORCID: 0000-0002-7863-4965\n",
      "AD  - Departamento de Biodiversidade e Centro de Aquicultura, Instituto de Biociencias,Universidade Estadual Paulista, 13506-900 Rio Claro, Sao Paulo, Brazil.\n",
      "FAU - Fitak, Robert R\n",
      "AU  - Fitak RR\n",
      "AUID- ORCID: 0000-0002-7398-6259\n",
      "AD  - Department of Biology, Duke University, Durham, NC 27708.\n",
      "AD  - Department of Biology, Genomics and Bioinformatics Cluster, University of CentralFlorida, Orlando, FL 32816.\n",
      "FAU - Faigon Soverna, Ana\n",
      "AU  - Faigon Soverna A\n",
      "AD  - Division Herpetologia, Museo Argentino de Ciencias Naturales \"BernardinoRivadavia,\" Consejo Nacional de Investigaciones Cientificas y Tecnicas (CONICET),Ciudad Autonoma de Buenos Aires C1405DJR, Argentina.\n",
      "FAU - Ron, Santiago R\n",
      "AU  - Ron SR\n",
      "AD  - Museo de Zoologia, Escuela de Biologia, Pontificia Universidad Catolica delEcuador, Aptdo. 17-01-2184, Quito, Ecuador.\n",
      "FAU - Lagorio, Maria G\n",
      "AU  - Lagorio MG\n",
      "AUID- ORCID: 0000-0003-3761-6733\n",
      "AD  - Instituto de Quimica Fisica de los Materiales, Medio Ambiente y Energia, Consejo Nacional de Investigaciones Cientificas y Tecnicas (CONICET), Facultad deCiencias Exactas y Naturales, Universidad de Buenos Aires, Ciudad Autonoma deBuenos Aires C1428EHA, Argentina.\n",
      "AD  - Departamento de Quimica Inorganica, Analitica y Quimica Fisica, Facultad deCiencias Exactas y Naturales, Universidad de Buenos Aires, Ciudad Autonoma deBuenos Aires C1428EHA, Argentina.\n",
      "FAU - Haddad, Celio F B\n",
      "AU  - Haddad CFB\n",
      "AUID- ORCID: 0000-0003-1057-5660\n",
      "AD  - Departamento de Biodiversidade e Centro de Aquicultura, Instituto de Biociencias,Universidade Estadual Paulista, 13506-900 Rio Claro, Sao Paulo, Brazil.\n",
      "FAU - Lopes, Norberto P\n",
      "AU  - Lopes NP\n",
      "AUID- ORCID: 0000-0002-8159-3658\n",
      "AD  - Nucleo de Pesquisa em Produtos Naturais e Sinteticos (NPPNS), Departamento deCiencias BioMoleculares, Faculdade de Ciencias Farmaceuticas de Ribeirao Preto,Universidade de Sao Paulo, 14040903 Ribeirao Preto, Sao Paulo, Brazil.\n",
      "FAU - Johnsen, Sonke\n",
      "AU  - Johnsen S\n",
      "AD  - Department of Biology, Duke University, Durham, NC 27708.\n",
      "FAU - Faivovich, Julian\n",
      "AU  - Faivovich J\n",
      "AUID- ORCID: 0000-0001-7157-8131\n",
      "AD  - Division Herpetologia, Museo Argentino de Ciencias Naturales \"BernardinoRivadavia,\" Consejo Nacional de Investigaciones Cientificas y Tecnicas (CONICET),Ciudad Autonoma de Buenos Aires C1405DJR, Argentina.\n",
      "AD  - Departamento de Biodiversidad y Biologia Experimental, Facultad de CienciasExactas y Naturales, Universidad de Buenos Aires, Ciudad Autonoma de Buenos AiresC1428EHA, Argentina.\n",
      "FAU - Chemes, Lucia B\n",
      "AU  - Chemes LB\n",
      "AUID- ORCID: 0000-0003-0192-9906\n",
      "AD  - Fundacion Instituto Leloir and Instituto de Investigaciones Bioquimicas de BuenosAires-Consejo Nacional de Investigaciones Cientificas y Tecnicas, Ciudad Autonomade Buenos Aires C1405BWE, Argentina; carlostaboada84@gmail.comlchemes@iib.unsam.edu.ar bari@qi.fcen.uba.ar.\n",
      "AD  - Instituto de Investigaciones Biotecnologicas, Universidad Nacional de San Martin,CP1650 San Martin, Buenos Aires, Argentina.\n",
      "FAU - Bari, Sara E\n",
      "AU  - Bari SE\n",
      "AUID- ORCID: 0000-0003-1604-6196\n",
      "AD  - Instituto de Quimica Fisica de los Materiales, Medio Ambiente y Energia, Consejo Nacional de Investigaciones Cientificas y Tecnicas (CONICET), Facultad deCiencias Exactas y Naturales, Universidad de Buenos Aires, Ciudad Autonoma deBuenos Aires C1428EHA, Argentina; carlostaboada84@gmail.comlchemes@iib.unsam.edu.ar bari@qi.fcen.uba.ar.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20200713\n",
      "PL  - United States\n",
      "TA  - Proc Natl Acad Sci U S A\n",
      "JT  - Proceedings of the National Academy of Sciences of the United States of America\n",
      "JID - 7505876\n",
      "RN  - 0 (Serpins)\n",
      "RN  - O9MIA842K9 (Biliverdine)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Anura/classification/genetics/*physiology\n",
      "MH  - Biliverdine/chemistry/*metabolism\n",
      "MH  - Biological Mimicry/physiology\n",
      "MH  - Serpins/chemistry/genetics/*metabolism\n",
      "MH  - Skin Pigmentation/genetics/*physiology\n",
      "PMC - PMC7414155\n",
      "COIS- The authors declare no competing interest.\n",
      "EDAT- 2020/07/15 06:00\n",
      "MHDA- 2020/09/22 06:00\n",
      "CRDT- 2020/07/15 06:00\n",
      "PHST- 2020/07/15 06:00 [pubmed]\n",
      "PHST- 2020/09/22 06:00 [medline]\n",
      "PHST- 2020/07/15 06:00 [entrez]\n",
      "AID - 2006771117 [pii]\n",
      "AID - 10.1073/pnas.2006771117 [doi]\n",
      "PST - ppublish\n",
      "SO  - Proc Natl Acad Sci U S A. 2020 Aug 4;117(31):18574-18581. doi:10.1073/pnas.2006771117. Epub 2020 Jul 13.\n",
      "\n",
      "PMID- 32648524\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210202\n",
      "LR  - 20210202\n",
      "IS  - 0031-949X (Print)\n",
      "IS  - 0031-949X (Linking)\n",
      "VI  - 111\n",
      "IP  - 1\n",
      "DP  - 2021 Jan\n",
      "TI  - Natural Defect of a Plant Rhabdovirus Glycoprotein Gene: A Case Study ofVirus-Plant Coevolution.\n",
      "PG  - 227-236\n",
      "LID - 10.1094/PHYTO-05-20-0191-FI [doi]\n",
      "AB  - Seven isolates of a putative cytorhabdovirus (family Rhabdoviridae, orderMononegavirales) designated as citrus-associated rhabdovirus (CiaRV) wereidentified in citrus, passion fruit, and paper bush from the same geographicalarea in China. CiaRV, bean-associated cytorhabdovirus (Brazil), and papaya virus E (Ecuador) should be taxonomically classified in the species Papayacytorhabdovirus. Due to natural mutations, the glycoprotein (G) and P4 genes wereimpaired in citrus-infecting isolates of CiaRV, resulting in an atypicalrhabdovirus genome organization of 3' leader-N-P-P3-M-L-5' trailer. The P3protein of CiaRV shared a common origin with begomoviral movement proteins(family Geminiviridae). Secondary structure analysis and trans-complementation ofmovement-deficient tomato mosaic virus and potato virus X mutants by CiaRV P3supported its function in viral cell-to-cell trafficking. The wide geographicaldispersal of CiaRV and related viruses suggests an efficient transmissionmechanism, as well as an underlying risk to global agriculture. Both the natural phenomenon and experimental analyses demonstrated presence of the \"degraded\" typeof CiaRV in citrus, in parallel to \"undegraded\" types in other host plantspecies. This case study shows a plant virus losing the function of an important but nonessential gene, likely due to host shift and adaption, which deepened our understanding of course of natural viral diversification.\n",
      "FAU - Zhang, Song\n",
      "AU  - Zhang S\n",
      "AD  - National Citrus Engineering and Technology Research Center, Citrus ResearchInstitute, Southwest University, Beibei, Chongqing 400712, China.\n",
      "AD  - Academy of Agricultural Sciences, Southwest University, Beibei, Chongqing 400715,China.\n",
      "FAU - Huang, Aijun\n",
      "AU  - Huang A\n",
      "AUID- ORCID: http://orcid.org/0000-0003-0733-2633\n",
      "AD  - National Navel Orange Research Center, College of Life Science, Gannan NormalUniversity, Ganzhou, China.\n",
      "FAU - Zhou, Xin\n",
      "AU  - Zhou X\n",
      "AD  - State Key Laboratory of Rice Biology, Institute of Biotechnology, ZhejiangUniversity, Hangzhou, Zhejiang, China.\n",
      "FAU - Li, Zhenghe\n",
      "AU  - Li Z\n",
      "AUID- ORCID: http://orcid.org/0000-0001-5497-2575\n",
      "AD  - State Key Laboratory of Rice Biology, Institute of Biotechnology, ZhejiangUniversity, Hangzhou, Zhejiang, China.\n",
      "FAU - Dietzgen, Ralf G\n",
      "AU  - Dietzgen RG\n",
      "AUID- ORCID: http://orcid.org/0000-0002-7772-2250\n",
      "AD  - Queensland Alliance for Agriculture and Food Innovation, The University ofQueensland, St. Lucia, Queensland 4072, Australia.\n",
      "FAU - Zhou, Changyong\n",
      "AU  - Zhou C\n",
      "AD  - National Citrus Engineering and Technology Research Center, Citrus ResearchInstitute, Southwest University, Beibei, Chongqing 400712, China.\n",
      "AD  - Academy of Agricultural Sciences, Southwest University, Beibei, Chongqing 400715,China.\n",
      "FAU - Cao, Mengji\n",
      "AU  - Cao M\n",
      "AUID- ORCID: http://orcid.org/0000-0003-0396-262X\n",
      "AD  - National Citrus Engineering and Technology Research Center, Citrus ResearchInstitute, Southwest University, Beibei, Chongqing 400712, China.\n",
      "AD  - Academy of Agricultural Sciences, Southwest University, Beibei, Chongqing 400715,China.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20201110\n",
      "PL  - United States\n",
      "TA  - Phytopathology\n",
      "JT  - Phytopathology\n",
      "JID - 9427222\n",
      "RN  - 0 (Glycoproteins)\n",
      "SB  - IM\n",
      "MH  - Brazil\n",
      "MH  - China\n",
      "MH  - Ecuador\n",
      "MH  - Genome, Viral\n",
      "MH  - Glycoproteins\n",
      "MH  - Open Reading Frames\n",
      "MH  - Phylogeny\n",
      "MH  - Plant Diseases\n",
      "MH  - *Plant Viruses/genetics\n",
      "MH  - *Rhabdoviridae/genetics\n",
      "OTO - NOTNLM\n",
      "OT  - citrus\n",
      "OT  - evolution\n",
      "OT  - gene defect\n",
      "OT  - genomics\n",
      "OT  - movement protein complementation\n",
      "OT  - plant rhabdovirus\n",
      "OT  - viral quasi-species\n",
      "OT  - virology\n",
      "EDAT- 2020/07/11 06:00\n",
      "MHDA- 2021/02/03 06:00\n",
      "CRDT- 2020/07/11 06:00\n",
      "PHST- 2020/07/11 06:00 [pubmed]\n",
      "PHST- 2021/02/03 06:00 [medline]\n",
      "PHST- 2020/07/11 06:00 [entrez]\n",
      "AID - 10.1094/PHYTO-05-20-0191-FI [doi]\n",
      "PST - ppublish\n",
      "SO  - Phytopathology. 2021 Jan;111(1):227-236. doi: 10.1094/PHYTO-05-20-0191-FI. Epub2020 Nov 10.\n",
      "\n",
      "PMID- 32645885\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210319\n",
      "LR  - 20210319\n",
      "IS  - 2073-4425 (Electronic)\n",
      "IS  - 2073-4425 (Linking)\n",
      "VI  - 11\n",
      "IP  - 7\n",
      "DP  - 2020 Jul 7\n",
      "TI  - ShadowCaster: Compositional Methods under the Shadow of Phylogenetic Models toDetect Horizontal Gene Transfers in Prokaryotes.\n",
      "LID - E756 [pii]\n",
      "LID - 10.3390/genes11070756 [doi]\n",
      "AB  - Horizontal gene transfer (HGT) plays an important role for evolutionaryinnovations within prokaryotic communities and is a crucial event for theirsurvival. Several computational approaches have arisen to identify HGT events in recipient genomes. However, this has been proven to be a complex task due to the generation of a great number of false positives and the prediction disagreementamong the existing methods. Phylogenetic reconstruction methods turned out to be the most reliable ones, but they are not extensible to all genes/species and are computationally demanding when dealing with large datasets. In contrast, theso-called surrogate methods that use heuristic solutions either based onnucleotide composition patterns or phyletic distribution of BLAST hits can beapplied easily to the genomic scale, but they fail in identifying common HGTevents. Here, we present ShadowCaster, a hybrid approach that sequentiallycombines nucleotide composition-based predictions by support vector machines(SVMs) under the shadow of phylogenetic models independent of treereconstruction, to improve the detection of HGT events in prokaryotes.ShadowCaster successfully predicted close and distant HGT events in bothartificial and bacterial genomes. ShadowCaster detected HGT related to heavymetal resistance in the genome of Rhodanobacter denitrificans with higheraccuracy than the most popular state-of-the-art computational approaches,encompassing most of the predicted cases made by other methods. ShadowCaster isreleased at the GitHub platform as an open-source software under the GPLv3license.\n",
      "FAU - Sanchez-Soto, Daniela\n",
      "AU  - Sanchez-Soto D\n",
      "AD  - Departamento de Ciencias Biologicas, Universidad Tecnica Particular de Loja, Loja110108, Ecuador.\n",
      "FAU - Aguero-Chapin, Guillermin\n",
      "AU  - Aguero-Chapin G\n",
      "AUID- ORCID: 0000-0002-9908-2418\n",
      "AD  - CIIMAR/CIMAR, Interdisciplinary Centre of Marine and Environmental Research,University of Porto, Terminal de Cruzeiros do Porto de Leixoes, Av. GeneralNorton de Matos s/n, 4450-208 Porto, Portugal.\n",
      "AD  - Department of Biology, Faculty of Sciences, University of Porto, Rua do CampoAlegre, 4169-007 Porto, Portugal.\n",
      "FAU - Armijos-Jaramillo, Vinicio\n",
      "AU  - Armijos-Jaramillo V\n",
      "AUID- ORCID: 0000-0003-2965-2515\n",
      "AD  - Grupo de Bio-Quimioinformatica & Carrera de Ingenieria en Biotecnologia, Facultadde Ingenieria y Ciencias Agropecuarias, Universidad de Las Americas, QuitoEC170125, Ecuador.\n",
      "FAU - Perez-Castillo, Yunierkis\n",
      "AU  - Perez-Castillo Y\n",
      "AUID- ORCID: 0000-0002-3710-0035\n",
      "AD  - Grupo de Bio-Quimioinformatica & Escuela de Ciencias Fisicas y Matematicas,Universidad de Las Americas, Quito EC170125, Ecuador.\n",
      "FAU - Tejera, Eduardo\n",
      "AU  - Tejera E\n",
      "AD  - Grupo de Bio-Quimioinformatica & Carrera de Ingenieria en Biotecnologia, Facultadde Ingenieria y Ciencias Agropecuarias, Universidad de Las Americas, QuitoEC170125, Ecuador.\n",
      "FAU - Antunes, Agostinho\n",
      "AU  - Antunes A\n",
      "AUID- ORCID: 0000-0002-1328-1732\n",
      "AD  - CIIMAR/CIMAR, Interdisciplinary Centre of Marine and Environmental Research,University of Porto, Terminal de Cruzeiros do Porto de Leixoes, Av. GeneralNorton de Matos s/n, 4450-208 Porto, Portugal.\n",
      "AD  - Department of Biology, Faculty of Sciences, University of Porto, Rua do CampoAlegre, 4169-007 Porto, Portugal.\n",
      "FAU - Sanchez-Rodriguez, Aminael\n",
      "AU  - Sanchez-Rodriguez A\n",
      "AD  - Departamento de Ciencias Biologicas, Universidad Tecnica Particular de Loja, Loja110108, Ecuador.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20200707\n",
      "PL  - Switzerland\n",
      "TA  - Genes (Basel)\n",
      "JT  - Genes\n",
      "JID - 101551097\n",
      "RN  - Rhodanobacter denitrificans\n",
      "SB  - IM\n",
      "MH  - Gammaproteobacteria/classification/genetics\n",
      "MH  - *Gene Transfer, Horizontal\n",
      "MH  - *Genome, Archaeal\n",
      "MH  - *Genome, Bacterial\n",
      "MH  - Genomics/*methods\n",
      "MH  - Phylogeny\n",
      "MH  - *Software\n",
      "MH  - Support Vector Machine\n",
      "PMC - PMC7397055\n",
      "OTO - NOTNLM\n",
      "OT  - *horizontal gene transfer\n",
      "OT  - *hybrid approach\n",
      "OT  - *implicit phylogenetic model\n",
      "OT  - *parametric method\n",
      "EDAT- 2020/07/11 06:00\n",
      "MHDA- 2021/03/20 06:00\n",
      "CRDT- 2020/07/11 06:00\n",
      "PHST- 2020/06/09 00:00 [received]\n",
      "PHST- 2020/06/27 00:00 [revised]\n",
      "PHST- 2020/06/30 00:00 [accepted]\n",
      "PHST- 2020/07/11 06:00 [entrez]\n",
      "PHST- 2020/07/11 06:00 [pubmed]\n",
      "PHST- 2021/03/20 06:00 [medline]\n",
      "AID - genes11070756 [pii]\n",
      "AID - 10.3390/genes11070756 [doi]\n",
      "PST - epublish\n",
      "SO  - Genes (Basel). 2020 Jul 7;11(7). pii: genes11070756. doi: 10.3390/genes11070756.\n",
      "\n",
      "PMID- 32598029\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20201210\n",
      "LR  - 20201214\n",
      "IS  - 1095-8649 (Electronic)\n",
      "IS  - 0022-1112 (Linking)\n",
      "VI  - 97\n",
      "IP  - 3\n",
      "DP  - 2020 Sep\n",
      "TI  - Genetic diversity and structure of circumtropical almaco jack, Seriola rivoliana:tool for conservation and management.\n",
      "PG  - 882-894\n",
      "LID - 10.1111/jfb.14450 [doi]\n",
      "AB  - The almaco jack, Seriola rivoliana, is a circumtropical pelagic fish ofimportance both in commercial fisheries and in aquaculture. To understand levels of genetic diversity within and among populations in the wild, population geneticstructure and the relative magnitude of migration were assessed using mtDNAsequence data and single nucleotide polymorphisms (SNPs) from individuals sampledfrom locations in the Pacific and Atlantic Oceans. A total of 25 variable sitesof cytochrome c oxidase subunit 1 and 3678 neutral SNPs were recovered. Threegenetic groups were identified, with both marker types distributed in differentoceanic regions: Pacific-1 in central Pacific, Pacific-2 in eastern Pacific andAtlantic in western Atlantic. Nonetheless, the analysis of SNP identified afourth population in the Pacific coast of Baja California Sur, Mexico(Pacific-3), whereas that of mtDNA did not. This mito-nuclear discordance islikely explained by a recently diverged Pacific-3 population. In addition, twomtDNA haplogroups were found within the western Atlantic, likely indicating that the species came into the Atlantic from the Indian Ocean with historical geneflow from the eastern Pacific. Relative gene flow among ocean basins was low withr m < 0.2, whereas in the eastern Pacific it was asymmetric and higher from southto north (r m > 0.79). The results reflect the importance of assessing geneticstructure and gene flow of natural populations for the purposes of sustainablemanagement.\n",
      "CI  - (c) 2020 Fisheries Society of the British Isles.\n",
      "FAU - Mendoza-Portillo, Veronica\n",
      "AU  - Mendoza-Portillo V\n",
      "AD  - Laboratorio de Genetica para la Conservacion, Centro de InvestigacionesBiologicas del Noroeste, La Paz, Mexico.\n",
      "FAU - Galvan-Tirado, Carolina\n",
      "AU  - Galvan-Tirado C\n",
      "AD  - CONACYT-Universidad Autonoma de Baja California Sur, La Paz, Mexico.\n",
      "FAU - Portnoy, David S\n",
      "AU  - Portnoy DS\n",
      "AD  - Marine Genomics Laboratory, Department of Life Sciences, Texas A&MUniversity-Corpus Christi, USA.\n",
      "FAU - Valenzuela-Quinonez, Fausto\n",
      "AU  - Valenzuela-Quinonez F\n",
      "AD  - CONACYT-Centro de Investigaciones Biologicas del Noroeste, La Paz, Mexico.\n",
      "FAU - Dominguez-Dominguez, Omar\n",
      "AU  - Dominguez-Dominguez O\n",
      "AD  - Laboratorio de Biologia Acuatica, Facultad de Biologia, Universidad Michoacana deSan Nicolas de Hidalgo, Morelia, Mexico.\n",
      "AD  - Instituto Nacional de Biodiversidad, Quito, Ecuador.\n",
      "FAU - Durand, Jean-Dominique\n",
      "AU  - Durand JD\n",
      "AD  - MARBEC, Universite Montpellier-IRD-Ifremer-CNRS, Montpellier Cedex, France.\n",
      "FAU - Perez-Urbiola, Juan Carlos\n",
      "AU  - Perez-Urbiola JC\n",
      "AD  - Centro de Investigaciones Biologicas del Noroeste, La Paz, Mexico.\n",
      "FAU - Garcia-De Leon, Francisco J\n",
      "AU  - Garcia-De Leon FJ\n",
      "AUID- ORCID: https://orcid.org/0000-0003-2323-2560\n",
      "AD  - Laboratorio de Genetica para la Conservacion, Centro de InvestigacionesBiologicas del Noroeste, La Paz, Mexico.\n",
      "LA  - eng\n",
      "GR  - CONACYT-CB-2014-240875/Consejo Nacional de Ciencia y Tecnologia\n",
      "GR  - 2505161165/Secretaria de Agricultura, Ganaderia, Desarrollo Rural, Pesca yAlimentacion\n",
      "PT  - Journal Article\n",
      "DEP - 20200727\n",
      "PL  - England\n",
      "TA  - J Fish Biol\n",
      "JT  - Journal of fish biology\n",
      "JID - 0214055\n",
      "RN  - 0 (DNA, Mitochondrial)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - *Conservation of Natural Resources\n",
      "MH  - DNA, Mitochondrial/genetics\n",
      "MH  - *Fisheries\n",
      "MH  - Gene Flow\n",
      "MH  - *Genetic Variation\n",
      "MH  - Genetics, Population\n",
      "MH  - Mexico\n",
      "MH  - Oceans and Seas\n",
      "MH  - Perciformes/classification/*genetics\n",
      "MH  - Tropical Climate\n",
      "OTO - NOTNLM\n",
      "OT  - Atlantic Ocean\n",
      "OT  - Pacific Ocean\n",
      "OT  - conservation\n",
      "OT  - gene flow\n",
      "OT  - genetic populations\n",
      "OT  - pelagic fish\n",
      "EDAT- 2020/07/01 06:00\n",
      "MHDA- 2020/12/15 06:00\n",
      "CRDT- 2020/06/30 06:00\n",
      "PHST- 2020/04/08 00:00 [received]\n",
      "PHST- 2020/06/13 00:00 [revised]\n",
      "PHST- 2020/06/26 00:00 [accepted]\n",
      "PHST- 2020/07/01 06:00 [pubmed]\n",
      "PHST- 2020/12/15 06:00 [medline]\n",
      "PHST- 2020/06/30 06:00 [entrez]\n",
      "AID - 10.1111/jfb.14450 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Fish Biol. 2020 Sep;97(3):882-894. doi: 10.1111/jfb.14450. Epub 2020 Jul 27.\n",
      "\n",
      "PMID- 32581039\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210614\n",
      "LR  - 20210614\n",
      "IS  - 1535-9484 (Electronic)\n",
      "IS  - 1535-9476 (Linking)\n",
      "VI  - 19\n",
      "IP  - 9\n",
      "DP  - 2020 Sep\n",
      "TI  - Proteomics of Galapagos Marine Iguanas Links Function of Femoral Gland Proteinsto the Immune System.\n",
      "PG  - 1523-1532\n",
      "LID - S1535-9476(20)35100-8 [pii]\n",
      "LID - 10.1074/mcp.RA120.001947 [doi]\n",
      "AB  - Communication between individuals via molecules, termed chemosignaling, iswidespread among animal and plant species. However, we lack knowledge on thespecific functions of the substances involved for most systems. The femoral glandis an organ that secretes a waxy substance involved in chemical communication in lizards. Although the lipids and volatile substances secreted by the femoralglands have been investigated in several biochemical studies, the proteincomposition and functions of secretions remain completely unknown. Applying aproteomic approach, we provide the first attempt to comprehensively characterize the protein composition of femoral gland secretions from the Galapagos marineiguana. Using samples from several organs, the marine iguana proteome wasassembled by next-generation sequencing and MS, resulting in 7513 proteins. Ofthese, 4305 proteins were present in the femoral gland, including keratins, smallserum proteins, and fatty acid-binding proteins. Surprisingly, no proteins withdiscernible roles in partner recognition or inter-species communication could be identified. However, we did find several proteins with direct associations to theinnate immune system, including lysozyme C, antileukoproteinase (ALP), pulmonary surfactant protein (SFTPD), and galectin (LGALS1) suggesting that the femoralglands function as an important barrier to infection. Furthermore, we reportseveral novel anti-microbial peptides from the femoral glands that show similaraction against Escherichia coli and Bacillus subtilis such as oncocin, a peptide known for its effectiveness against Gram-negative pathogens. This proteomics dataset is a valuable resource for future functional protein analysis anddemonstrates that femoral gland secretions also perform functions of the innateimmune system.\n",
      "CI  - (c) 2020 Tellkamp et al.\n",
      "FAU - Tellkamp, Frederik\n",
      "AU  - Tellkamp F\n",
      "AD  - CECAD Research Center Institute for Genetics, University of Cologne, Cologne,Germany.\n",
      "FAU - Lang, Franziska\n",
      "AU  - Lang F\n",
      "AD  - TRON - Translational Oncology at the University Medical Center of the JohannesGutenberg University Mainz, Mainz, Germany.\n",
      "FAU - Ibanez, Alejandro\n",
      "AU  - Ibanez A\n",
      "AD  - Department of Comparative Anatomy, Institute of Zoology and Biomedical Research, Jagiellonian University, Krakow, Poland.\n",
      "FAU - Abraham, Lena\n",
      "AU  - Abraham L\n",
      "AD  - CECAD Research Center Institute for Genetics, University of Cologne, Cologne,Germany.\n",
      "FAU - Quezada, Galo\n",
      "AU  - Quezada G\n",
      "AD  - Direccion Parque Nacional Galapagos, Puerto Ayora, Santa Cruz, Galapagos,Ecuador.\n",
      "FAU - Gunther, Stefan\n",
      "AU  - Gunther S\n",
      "AD  - Max Planck Institute for Heart and Lung Research, Bad Nauheim, Germany.\n",
      "FAU - Looso, Mario\n",
      "AU  - Looso M\n",
      "AD  - Max Planck Institute for Heart and Lung Research, Bad Nauheim, Germany.\n",
      "FAU - Tann, Fabian Jannik\n",
      "AU  - Tann FJ\n",
      "AD  - Institute for Biochemical Engineering and Analytics (IBVA), Giessen, Germany.\n",
      "FAU - Muller, Daniela\n",
      "AU  - Muller D\n",
      "AD  - Institute for Biochemical Engineering and Analytics (IBVA), Giessen, Germany.\n",
      "FAU - Cemic, Franz\n",
      "AU  - Cemic F\n",
      "AD  - Institute for Biochemical Engineering and Analytics (IBVA), Giessen, Germany.\n",
      "FAU - Hemberger, Jurgen\n",
      "AU  - Hemberger J\n",
      "AD  - Institute for Biochemical Engineering and Analytics (IBVA), Giessen, Germany.\n",
      "FAU - Steinfartz, Sebastian\n",
      "AU  - Steinfartz S\n",
      "AD  - Molecular Evolution and Systematics of Animals, University of Leipzig, Leipzig,Germany. Electronic address: steinfartz@uni-leipzig.de.\n",
      "FAU - Kruger, Marcus\n",
      "AU  - Kruger M\n",
      "AD  - CECAD Research Center Institute for Genetics, University of Cologne, Cologne,Germany; Center for Molecular Medicine Cologne (CMMC), University of Cologne,Cologne, Germany. Electronic address: marcus.krueger@uni-koeln.de.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20200624\n",
      "PL  - United States\n",
      "TA  - Mol Cell Proteomics\n",
      "JT  - Molecular & cellular proteomics : MCP\n",
      "JID - 101125647\n",
      "RN  - 0 (Anti-Infective Agents)\n",
      "RN  - 0 (Apoproteins)\n",
      "RN  - 0 (Chemotactic Factors)\n",
      "RN  - 0 (Galectins)\n",
      "RN  - 0 (Proteome)\n",
      "RN  - 0 (Pulmonary Surfactant-Associated Proteins)\n",
      "RN  - 0 (pulmonary surfactant apoprotein)\n",
      "RN  - EC 3.2.1.17 (Muramidase)\n",
      "RN  - EC 3.4.- (Endopeptidases)\n",
      "RN  - EC 3.4.99.- (Alp protease)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Anti-Infective Agents/*metabolism/*pharmacology\n",
      "MH  - Apoproteins/genetics/metabolism\n",
      "MH  - Bacillus subtilis/drug effects\n",
      "MH  - Brain/metabolism\n",
      "MH  - Chemotactic Factors/genetics/metabolism\n",
      "MH  - Ecuador\n",
      "MH  - Endopeptidases/genetics/metabolism\n",
      "MH  - Escherichia coli/drug effects\n",
      "MH  - Galectins/genetics/metabolism\n",
      "MH  - Heart/physiology\n",
      "MH  - High-Throughput Nucleotide Sequencing\n",
      "MH  - Humans\n",
      "MH  - Iguanas/genetics/immunology/*metabolism\n",
      "MH  - Immune System/*metabolism\n",
      "MH  - *Immunity, Innate/genetics\n",
      "MH  - Lung/metabolism\n",
      "MH  - Muramidase/genetics/metabolism\n",
      "MH  - Muscles/metabolism\n",
      "MH  - Myocardium/metabolism\n",
      "MH  - Organ Specificity\n",
      "MH  - Proteome/genetics/immunology/*metabolism\n",
      "MH  - Proteomics\n",
      "MH  - Pulmonary Surfactant-Associated Proteins/genetics/metabolism\n",
      "MH  - Skin/metabolism\n",
      "MH  - Tandem Mass Spectrometry\n",
      "MH  - *Transcriptome/genetics\n",
      "PMC - PMC8143647\n",
      "OTO - NOTNLM\n",
      "OT  - *Proteomics\n",
      "OT  - *animal models\n",
      "OT  - *database design\n",
      "OT  - *evolution\n",
      "OT  - *femoral glands\n",
      "OT  - *immune system\n",
      "OT  - *marine iguana\n",
      "OT  - *protease inhibitor\n",
      "OT  - *protease inhibitor protein identification\n",
      "OT  - *tissues\n",
      "COIS- Conflict of interest-Authors declare no competing interests.\n",
      "EDAT- 2020/06/26 06:00\n",
      "MHDA- 2021/06/16 06:00\n",
      "CRDT- 2020/06/26 06:00\n",
      "PHST- 2020/01/28 00:00 [received]\n",
      "PHST- 2020/05/11 00:00 [revised]\n",
      "PHST- 2020/06/26 06:00 [pubmed]\n",
      "PHST- 2021/06/16 06:00 [medline]\n",
      "PHST- 2020/06/26 06:00 [entrez]\n",
      "AID - S1535-9476(20)35100-8 [pii]\n",
      "AID - 10.1074/mcp.RA120.001947 [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Cell Proteomics. 2020 Sep;19(9):1523-1532. doi: 10.1074/mcp.RA120.001947.Epub 2020 Jun 24.\n",
      "\n",
      "PMID- 32512194\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20201116\n",
      "LR  - 20201116\n",
      "IS  - 1095-9513 (Electronic)\n",
      "IS  - 1055-7903 (Linking)\n",
      "VI  - 150\n",
      "DP  - 2020 Sep\n",
      "TI  - Diversification history of clown tree frogs in Neotropical rainforests (Anura,Hylidae, Dendropsophus leucophyllatus group).\n",
      "PG  - 106877\n",
      "LID - S1055-7903(20)30149-4 [pii]\n",
      "LID - 10.1016/j.ympev.2020.106877 [doi]\n",
      "AB  - General consensus emphasizes that no single biological process can explain thepatterns of species' distributions and diversification in the Neotropics.Instead, the interplay of several processes across space and time must be takeninto account. Here we investigated the phylogenetic relationships andbiogeographic history of tree frogs in the Dendropsophus leucophyllatus speciesgroup (Amphibia: Hylidae), which is distributed across Amazonia and the Atlantic rainforests. Using Next Generation Sequencing (NGS) and double digestrestriction-site associated DNA (ddRADseq), we inferred phylogeneticrelationships, species limits, and temporal and geographic patterns ofdiversification relative to the history of these biomes. Our results indicatethat the D. leucophyllatus species group includes at least 14 independentlineages, which are currently arranged into ten described species. Therefore, asignificant portion of species in the group are still unnamed. Differentprocesses were associated to the group diversification history. For instance, theAndes uplift likely caused allopatric speciation for Cis-Andean species, whereas it may also be responsible for changes in the Amazonian landscape triggeringparapatric speciation by local adaptation to ecological factors. Meanwhile,Atlantic Forest ancestors unable to cross the dry diagonal biomes afterrainforest's retraction, evolved in isolation into different species.Diversification in the group began in the early Miocene, when connections betweenAtlantic Forest and the Andes (Pacific Dominion) by way of a south corridor were possible. The historical scenario in Amazonia, characterized by severalspeciation events and habitat heterogeneity, helped promoting diversification,resulting in the highest species diversity for the group. This marked speciesdiversification did not happen in Atlantic Forest, where speciation is veryrecent (late Pliocene and Pleistocene), despite its remarkable climaticheterogeneity.\n",
      "CI  - Copyright (c) 2020 Elsevier Inc. All rights reserved.\n",
      "FAU - Pirani, Renata M\n",
      "AU  - Pirani RM\n",
      "AD  - Programa de Pos-Graduacao em Genetica, Conservacao e Biologia Evolutiva,Instituto Nacional de Pesquisas da Amazonia, Av. Ephigenio Sales 2239, 69060-000,Manaus, Amazonas, Brazil. Electronic address: renatampirani@gmail.com.\n",
      "FAU - Peloso, Pedro L V\n",
      "AU  - Peloso PLV\n",
      "AD  - Instituto de Ciencias Biologicas, Universidade Federal do Para, Rua AugustoCorrea, 01, Guama, 66075-110, Belem, Para, Brazil.\n",
      "FAU - Prado, Joyce R\n",
      "AU  - Prado JR\n",
      "AD  - Departamento de Ciencias Biologicas, Universidade de Sao Paulo, Piracicaba, SaoPaulo, Brazil.\n",
      "FAU - Polo, Erico M\n",
      "AU  - Polo EM\n",
      "AD  - Departamento de Genetica, Instituto de Ciencias Biologicas, Universidade Federal do Amazonas, Av. General Rodrigo Octavio, 6200, Manaus, Amazonas, Brazil.\n",
      "FAU - Knowles, L Lacey\n",
      "AU  - Knowles LL\n",
      "AD  - Department of Ecology and Evolutionary Biology, The University of Michigan, AnnArbor, MI 41809-1079, USA.\n",
      "FAU - Ron, Santiago R\n",
      "AU  - Ron SR\n",
      "AD  - Museo de Zoologia, Facultad de Ciencias Exactas y Naturales, PontificiaUniversidad Catolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Rodrigues, Miguel T\n",
      "AU  - Rodrigues MT\n",
      "AD  - Departamento de Zoologia, Instituto de Biociencias, Universidade de Sao Paulo,CEP 05508-090 Sao Paulo, Sao Paulo, Brazil.\n",
      "FAU - Sturaro, Marcelo J\n",
      "AU  - Sturaro MJ\n",
      "AD  - Departamento de Ecologia e Biologia Evolutiva, Universidade Federal de Sao Paulo,Av. Professor Artur Riedel, 275, Jardim Eldorado, Diadema, CEP 09972-270 SaoPaulo, Brazil.\n",
      "FAU - Werneck, Fernanda P\n",
      "AU  - Werneck FP\n",
      "AD  - Programa de Pos-Graduacao em Genetica, Conservacao e Biologia Evolutiva,Instituto Nacional de Pesquisas da Amazonia, Av. Ephigenio Sales 2239, 69060-000,Manaus, Amazonas, Brazil; Coordenacao de Biodiversidade, Programa de ColecoesCientificas Biologicas, Instituto Nacional de Pesquisas da Amazonia, Av. AndreAraujo 2936, CEP 69067-375 Manaus, Amazonas, Brazil.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20200606\n",
      "PL  - United States\n",
      "TA  - Mol Phylogenet Evol\n",
      "JT  - Molecular phylogenetics and evolution\n",
      "JID - 9304400\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Anura/*classification/genetics\n",
      "MH  - Biodiversity\n",
      "MH  - Databases, Genetic\n",
      "MH  - Phylogeny\n",
      "MH  - Rainforest\n",
      "MH  - South America\n",
      "MH  - Species Specificity\n",
      "OTO - NOTNLM\n",
      "OT  - *Biogeography\n",
      "OT  - *Cryptic species\n",
      "OT  - *Genomics\n",
      "OT  - *Parapatric speciation\n",
      "OT  - *South America\n",
      "OT  - *Species delimitation\n",
      "EDAT- 2020/06/09 06:00\n",
      "MHDA- 2020/11/18 06:00\n",
      "CRDT- 2020/06/09 06:00\n",
      "PHST- 2019/11/26 00:00 [received]\n",
      "PHST- 2020/05/29 00:00 [revised]\n",
      "PHST- 2020/06/01 00:00 [accepted]\n",
      "PHST- 2020/06/09 06:00 [pubmed]\n",
      "PHST- 2020/11/18 06:00 [medline]\n",
      "PHST- 2020/06/09 06:00 [entrez]\n",
      "AID - S1055-7903(20)30149-4 [pii]\n",
      "AID - 10.1016/j.ympev.2020.106877 [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Phylogenet Evol. 2020 Sep;150:106877. doi: 10.1016/j.ympev.2020.106877. Epub 2020 Jun 6.\n",
      "\n",
      "PMID- 32462583\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210311\n",
      "LR  - 20210311\n",
      "IS  - 1940-6029 (Electronic)\n",
      "IS  - 1064-3745 (Linking)\n",
      "VI  - 2139\n",
      "DP  - 2020\n",
      "TI  - Dimethyl Labeling-Based Quantitative Proteomics of Recalcitrant Cocoa Pod Tissue.\n",
      "PG  - 133-146\n",
      "LID - 10.1007/978-1-0716-0528-8_10 [doi]\n",
      "AB  - Dimethyl labeling is a type of stable-isotope labeling suitable for creatingisotopic variants of peptides and thus be utilized for quantitative proteomicsexperiments. Labeling is achieved through a reductive amination/alkylationreaction using the low-cost reagents formaldehyde and cyanoborohydride, resultingin dimethylation of free amine groups of Lys and N-termini. Availability ofisotopomeric forms of these reagents allows for the generation of up to sixdifferent isotopic variants. Here we describe the application of dimethylation tocreate two isotopic variants, light and heavy, differing in 4 Da, to label thetotal tryptic digest peptides of cocoa pod extracted from healthy pods fromcultivars susceptible and resistant to the fungal disease called \"frosty pod\"caused by Moniliophthora roreri.\n",
      "FAU - Esteve-Sanchez, Yoel\n",
      "AU  - Esteve-Sanchez Y\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry. Faculty of Sciences, University of Alicante, Alicante, Spain.\n",
      "FAU - Morante-Carriel, Jaime A\n",
      "AU  - Morante-Carriel JA\n",
      "AD  - Biotechnology and Molecular Biology Group, Quevedo State Technical University,Quevedo, Ecuador.\n",
      "FAU - Martinez-Marquez, Ascension\n",
      "AU  - Martinez-Marquez A\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry. Faculty of Sciences, University of Alicante, Alicante, Spain.\n",
      "FAU - Selles-Marchart, Susana\n",
      "AU  - Selles-Marchart S\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry. Faculty of Sciences, University of Alicante, Alicante, Spain.\n",
      "FAU - Bru-Martinez, Roque\n",
      "AU  - Bru-Martinez R\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry. Faculty of Sciences, University of Alicante, Alicante, Spain.roque.bru@ua.es.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PL  - United States\n",
      "TA  - Methods Mol Biol\n",
      "JT  - Methods in molecular biology (Clifton, N.J.)\n",
      "JID - 9214969\n",
      "RN  - 0 (Plant Extracts)\n",
      "RN  - 0 (Proteome)\n",
      "RN  - Moniliophthora roreri\n",
      "SB  - IM\n",
      "MH  - Agaricales/pathogenicity\n",
      "MH  - Amination/physiology\n",
      "MH  - Cacao/*metabolism/microbiology\n",
      "MH  - Isotope Labeling/methods\n",
      "MH  - Plant Diseases/microbiology\n",
      "MH  - Plant Extracts/metabolism\n",
      "MH  - Proteome/*metabolism\n",
      "MH  - Proteomics/methods\n",
      "OTO - NOTNLM\n",
      "OT  - *Cocoa pod\n",
      "OT  - *Dimethyl labeling\n",
      "OT  - *Fungal disease\n",
      "OT  - *Moniliophthora roreri\n",
      "OT  - *Plant proteomics\n",
      "OT  - *Quantitative proteomics\n",
      "OT  - *Stable-isotope labeling\n",
      "EDAT- 2020/05/29 06:00\n",
      "MHDA- 2021/03/12 06:00\n",
      "CRDT- 2020/05/29 06:00\n",
      "PHST- 2020/05/29 06:00 [entrez]\n",
      "PHST- 2020/05/29 06:00 [pubmed]\n",
      "PHST- 2021/03/12 06:00 [medline]\n",
      "AID - 10.1007/978-1-0716-0528-8_10 [doi]\n",
      "PST - ppublish\n",
      "SO  - Methods Mol Biol. 2020;2139:133-146. doi: 10.1007/978-1-0716-0528-8_10.\n",
      "\n",
      "PMID- 32447234\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210622\n",
      "LR  - 20210622\n",
      "IS  - 1873-5126 (Electronic)\n",
      "IS  - 1353-8020 (Linking)\n",
      "VI  - 75\n",
      "DP  - 2020 Jun\n",
      "TI  - Genetic characterization of Parkinson's disease patients in Ecuador and Colombia.\n",
      "PG  - 27-29\n",
      "LID - S1353-8020(20)30114-0 [pii]\n",
      "LID - 10.1016/j.parkreldis.2020.05.005 [doi]\n",
      "AB  - To help address the scarcity of studies on the genetics of Parkinson's disease(PD) in Latin America, we screened 426 Ecuadorians with PD and 80 Colombians (PD = 55, Control = 26) for mutations within several PD-related genes. AmongColombians, we identified several variants within PARKIN and PINK1 genes.\n",
      "CI  - Copyright (c) 2020 Elsevier Ltd. All rights reserved.\n",
      "FAU - Tipton, Philip W\n",
      "AU  - Tipton PW\n",
      "AD  - Department of Neurology, Mayo Clinic, Mangurian Building, Jacksonville, FL,32224, USA.\n",
      "FAU - Jaramillo-Koupermann, Gabriela\n",
      "AU  - Jaramillo-Koupermann G\n",
      "AD  - Department of Pathology, Hospital Eugenio Especjo, Quito, Ecuador.\n",
      "FAU - Soto-Beasley, Alexandra I\n",
      "AU  - Soto-Beasley AI\n",
      "AD  - Department of Neuroscience, Mayo Clinic, Jacksonville, FL, 32224, USA.\n",
      "FAU - Walton, Ronald L\n",
      "AU  - Walton RL\n",
      "AD  - Department of Neuroscience, Mayo Clinic, Jacksonville, FL, 32224, USA.\n",
      "FAU - Soler-Rangel, Silvia\n",
      "AU  - Soler-Rangel S\n",
      "AD  - Movement Disorders Clinic, Hospital Universitario San Ignacio, Bogota, Colombia; School of Medicine, Pontificia Universidad Javeriana, Bogota, Colombia.\n",
      "FAU - Romero-Osorio, Oscar\n",
      "AU  - Romero-Osorio O\n",
      "AD  - Movement Disorders Clinic, Hospital Universitario San Ignacio, Bogota, Colombia; School of Medicine, Pontificia Universidad Javeriana, Bogota, Colombia.\n",
      "FAU - Diaz, Cindy\n",
      "AU  - Diaz C\n",
      "AD  - Movement Disorders Clinic, Hospital Universitario San Ignacio, Bogota, Colombia; School of Medicine, Pontificia Universidad Javeriana, Bogota, Colombia.\n",
      "FAU - Moreno-Lopez, Claudia Lucia\n",
      "AU  - Moreno-Lopez CL\n",
      "AD  - Movement Disorders Clinic, Hospital Universitario San Ignacio, Bogota, Colombia; Fundacion Cardioinfantil, Bogota, Colombia.\n",
      "FAU - Ross, Owen A\n",
      "AU  - Ross OA\n",
      "AD  - Department of Neuroscience, Mayo Clinic, Jacksonville, FL, 32224, USA; Departmentof Clinical Genomics, Mayo Clinic, Jacksonville, FL, 32224, USA.\n",
      "FAU - Wszolek, Zbigniew K\n",
      "AU  - Wszolek ZK\n",
      "AD  - Department of Neurology, Mayo Clinic, Mangurian Building, Jacksonville, FL,32224, USA. Electronic address: Wszolek.Zbigniew@mayo.edu.\n",
      "FAU - Cerquera-Cleves, Catalina\n",
      "AU  - Cerquera-Cleves C\n",
      "AD  - Movement Disorders Clinic, Hospital Universitario San Ignacio, Bogota, Colombia; School of Medicine, Pontificia Universidad Javeriana, Bogota, Colombia; ClinicaUniversitaria Colombia, Bogota, Colombia.\n",
      "FAU - Alarcon, Fernando\n",
      "AU  - Alarcon F\n",
      "AD  - Department of Neurology, Movement Disorders Unit, Hospital Eugenio Especjo,Quito, Ecuador.\n",
      "LA  - eng\n",
      "GR  - R01 NS078086/NS/NINDS NIH HHS/United States\n",
      "GR  - P50 NS072187/NS/NINDS NIH HHS/United States\n",
      "GR  - U54 NS100693/NS/NINDS NIH HHS/United States\n",
      "GR  - U54 NS110435/NS/NINDS NIH HHS/United States\n",
      "PT  - Letter\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "DEP - 20200515\n",
      "PL  - England\n",
      "TA  - Parkinsonism Relat Disord\n",
      "JT  - Parkinsonism & related disorders\n",
      "JID - 9513583\n",
      "RN  - EC 2.3.2.27 (Ubiquitin-Protein Ligases)\n",
      "RN  - EC 2.3.2.27 (parkin protein)\n",
      "RN  - EC 2.7.- (Protein Kinases)\n",
      "RN  - EC 2.7.11.1 (PTEN-induced putative kinase)\n",
      "SB  - IM\n",
      "MH  - Aged\n",
      "MH  - Aged, 80 and over\n",
      "MH  - Colombia\n",
      "MH  - Ecuador\n",
      "MH  - Female\n",
      "MH  - Humans\n",
      "MH  - Male\n",
      "MH  - Middle Aged\n",
      "MH  - Parkinson Disease/*genetics\n",
      "MH  - Protein Kinases/genetics\n",
      "MH  - Ubiquitin-Protein Ligases/genetics\n",
      "OTO - NOTNLM\n",
      "OT  - *Genetics\n",
      "OT  - *Latin America\n",
      "OT  - *Parkinson's disease\n",
      "EDAT- 2020/05/25 06:00\n",
      "MHDA- 2021/06/23 06:00\n",
      "CRDT- 2020/05/25 06:00\n",
      "PHST- 2020/01/31 00:00 [received]\n",
      "PHST- 2020/04/10 00:00 [revised]\n",
      "PHST- 2020/05/04 00:00 [accepted]\n",
      "PHST- 2020/05/25 06:00 [pubmed]\n",
      "PHST- 2021/06/23 06:00 [medline]\n",
      "PHST- 2020/05/25 06:00 [entrez]\n",
      "AID - S1353-8020(20)30114-0 [pii]\n",
      "AID - 10.1016/j.parkreldis.2020.05.005 [doi]\n",
      "PST - ppublish\n",
      "SO  - Parkinsonism Relat Disord. 2020 Jun;75:27-29. doi:10.1016/j.parkreldis.2020.05.005. Epub 2020 May 15.\n",
      "\n",
      "PMID- 32419208\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210604\n",
      "LR  - 20210604\n",
      "IS  - 1365-294X (Electronic)\n",
      "IS  - 0962-1083 (Linking)\n",
      "VI  - 29\n",
      "IP  - 12\n",
      "DP  - 2020 Jun\n",
      "TI  - Regional differences in the abiotic environment contribute to genomic divergence within a wild tomato species.\n",
      "PG  - 2204-2217\n",
      "LID - 10.1111/mec.15477 [doi]\n",
      "AB  - The wild currant tomato Solanum pimpinellifolium inhabits a wide range of abiotichabitats across its native range of Ecuador and Peru. Although it has served as akey genetic resource for the improvement of domestic cultivars, little is knownabout the genetic basis of traits underlying local adaptation in this species,nor what abiotic variables are most important for driving differentiation. Herewe use redundancy analysis (RDA) and other multivariate statistical methods(structural equation modelling [SEM] and generalized dissimilarity modelling[GDM]) to quantify the relationship of genomic variation (6,830 single nucleotidepolymorphisms [SNPs]) with climate and geography, among 140 wild accessions. RDA,SEM and GDM each identified environment as explaining more genomic variation thangeography, suggesting that local adaptation to heterogeneous abiotic habitats maybe an important source of genetic diversity in this species. Environmentalfactors describing temporal variation in precipitation and evaporative demandexplained the most SNP variation among accessions, indicating that these forcesmay represent key selective agents. Lastly, by studying how SNP-environmentassociations vary throughout the genome (44,064 SNPs), we mapped the location andinvestigated the functions of loci putatively contributing to climaticadaptations. Together, our findings indicate an important role for selectionimposed by the abiotic environment in driving genomic differentiation betweenpopulations.\n",
      "CI  - (c) 2020 John Wiley & Sons Ltd.\n",
      "FAU - Gibson, Matthew J S\n",
      "AU  - Gibson MJS\n",
      "AUID- ORCID: 0000-0001-7855-1628\n",
      "AD  - Department of Biology, Indiana University, Bloomington, IN, USA.\n",
      "FAU - Moyle, Leonie C\n",
      "AU  - Moyle LC\n",
      "AUID- ORCID: 0000-0003-4960-8001\n",
      "AD  - Department of Biology, Indiana University, Bloomington, IN, USA.\n",
      "LA  - eng\n",
      "SI  - Dryad/10.5061/dryad.8gtht76k4\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20200609\n",
      "PL  - England\n",
      "TA  - Mol Ecol\n",
      "JT  - Molecular ecology\n",
      "JID - 9214478\n",
      "SB  - IM\n",
      "MH  - Ecuador\n",
      "MH  - *Environment\n",
      "MH  - *Genetics, Population\n",
      "MH  - Genomics\n",
      "MH  - Multivariate Analysis\n",
      "MH  - Peru\n",
      "MH  - Polymorphism, Single Nucleotide\n",
      "MH  - Solanum/*genetics\n",
      "OTO - NOTNLM\n",
      "OT  - *adaptation\n",
      "OT  - *eigenanalysis\n",
      "OT  - *landscape genetics\n",
      "OT  - *population structure\n",
      "OT  - *redundancy analysis\n",
      "EDAT- 2020/05/19 06:00\n",
      "MHDA- 2021/06/05 06:00\n",
      "CRDT- 2020/05/19 06:00\n",
      "PHST- 2019/09/02 00:00 [received]\n",
      "PHST- 2020/04/17 00:00 [revised]\n",
      "PHST- 2020/05/12 00:00 [accepted]\n",
      "PHST- 2020/05/19 06:00 [pubmed]\n",
      "PHST- 2021/06/05 06:00 [medline]\n",
      "PHST- 2020/05/19 06:00 [entrez]\n",
      "AID - 10.1111/mec.15477 [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Ecol. 2020 Jun;29(12):2204-2217. doi: 10.1111/mec.15477. Epub 2020 Jun 9.\n",
      "\n",
      "PMID- 32390256\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210301\n",
      "LR  - 20210301\n",
      "IS  - 1365-313X (Electronic)\n",
      "IS  - 0960-7412 (Linking)\n",
      "VI  - 103\n",
      "IP  - 3\n",
      "DP  - 2020 Aug\n",
      "TI  - Multiple ER-to-nucleus stress signaling pathways are activated during Plantagoasiatica mosaic virus and Turnip mosaic virus infection in Arabidopsis thaliana.\n",
      "PG  - 1233-1245\n",
      "LID - 10.1111/tpj.14798 [doi]\n",
      "AB  - Pathogens and other adverse environmental conditions can trigger endoplasmicreticulum (ER) stress. ER stress signaling increases the expression ofcytoprotective ER-chaperones. The inositol-requiring enzyme (IRE1) is one ERstress sensor that is activated to splice the bZIP60 mRNA that produces atruncated transcription factor that activates gene expression in the nucleus. TheIRE1/bZIP60 pathway is associated with restricting potyvirus and potexvirusinfection. This study shows that the Plantago asiatica mosaic virus (PlAMV)triple gene block 3 (TGB3) and the Turnip mosaic virus (TuMV) 6K2 proteinsactivate alternative transcription pathways involving the bZIP17, bZIP28, BAG7,NAC089 and NAC103 factors in Arabidopsis thaliana. Using the correspondingknockout mutant lines, we show that bZIP17, bZIP60, BAG7 and NAC089 are factorsin reducing PlAMV infection, whereas bZIP28 and bZIP60 are factors in reducingTuMV infection. We propose a model in which bZIP60 and bZIP17 synergisticallyinduce genes restricting PlAMV infection, while bZIP60 and bZIP28 independentlyinduce genes supporting PlAMV infection. Regarding TuMV-green fluorescent protein(GFP) infection, bZIP60 and bZIP28 serve to repress local and systemic infection.Finally, tauroursodeoxycholic acid treatments were used to demonstrate that theprotein folding capacity significantly influences PlAMV accumulation.\n",
      "CI  - (c) 2020 Society for Experimental Biology and John Wiley & Sons Ltd.\n",
      "FAU - Gayral, Mathieu\n",
      "AU  - Gayral M\n",
      "AD  - Texas A&M Agrilife Research and Extension Center in Dallas, 17360 Coit Rd,Dallas, TX, 75252, USA.\n",
      "FAU - Arias Gaguancela, Omar\n",
      "AU  - Arias Gaguancela O\n",
      "AD  - Texas A&M Agrilife Research and Extension Center in Dallas, 17360 Coit Rd,Dallas, TX, 75252, USA.\n",
      "FAU - Vasquez, Evelyn\n",
      "AU  - Vasquez E\n",
      "AD  - Departamento de Ciencias de la Vida y la Agricultura, Universidad de las Fuerzas Armadas-ESPE, Av. General Ruminahui s/n y Ambato, Sangolqui, 171103, Ecuador.\n",
      "FAU - Herath, Venura\n",
      "AU  - Herath V\n",
      "AD  - Texas A&M Agrilife Research and Extension Center in Dallas, 17360 Coit Rd,Dallas, TX, 75252, USA.\n",
      "AD  - Department of Plant Pathology and Microbiology, Institute for Plant Genomics and Biotechnology, Texas A&M University, 498 Olsen Blvd, College Station, TX, 77843, USA.\n",
      "AD  - Department of Agricultural Biology, Faculty of Agriculture, University ofPeradeniya, Peradeniya, 20400, Sri Lanka.\n",
      "FAU - Flores, Francisco J\n",
      "AU  - Flores FJ\n",
      "AD  - Departamento de Ciencias de la Vida y la Agricultura, Universidad de las Fuerzas Armadas-ESPE, Av. General Ruminahui s/n y Ambato, Sangolqui, 171103, Ecuador.\n",
      "AD  - Centro de Investigacion de Alimentos, CIAL, Facultad de Ciencias de la Ingenieriae Industrias, Universidad Tecnologica Equinoccial-UTE, Av. Mariscal Sucre yMariana de Jesus, Quito, Pichincha, 170129, Ecuador.\n",
      "FAU - Dickman, Martin B\n",
      "AU  - Dickman MB\n",
      "AD  - Department of Plant Pathology and Microbiology, Institute for Plant Genomics and Biotechnology, Texas A&M University, 498 Olsen Blvd, College Station, TX, 77843, USA.\n",
      "FAU - Verchot, Jeanmarie\n",
      "AU  - Verchot J\n",
      "AUID- ORCID: 0000-0001-8789-3144\n",
      "AD  - Texas A&M Agrilife Research and Extension Center in Dallas, 17360 Coit Rd,Dallas, TX, 75252, USA.\n",
      "AD  - Department of Plant Pathology and Microbiology, Institute for Plant Genomics and Biotechnology, Texas A&M University, 498 Olsen Blvd, College Station, TX, 77843, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20200625\n",
      "PL  - England\n",
      "TA  - Plant J\n",
      "JT  - The Plant journal : for cell and molecular biology\n",
      "JID - 9207397\n",
      "RN  - 0 (Arabidopsis Proteins)\n",
      "RN  - 0 (Basic-Leucine Zipper Transcription Factors)\n",
      "RN  - Plantago asiatica mosaic virus\n",
      "RN  - Turnip mosaic virus\n",
      "SB  - IM\n",
      "MH  - Arabidopsis/metabolism/*virology\n",
      "MH  - Arabidopsis Proteins/metabolism\n",
      "MH  - Basic-Leucine Zipper Transcription Factors/metabolism\n",
      "MH  - Cell Nucleus/*metabolism\n",
      "MH  - Endoplasmic Reticulum/*metabolism\n",
      "MH  - Gene Expression Regulation, Plant\n",
      "MH  - Gene Expression Regulation, Viral\n",
      "MH  - Plant Diseases/*virology\n",
      "MH  - Potexvirus/*metabolism\n",
      "MH  - Potyvirus/*metabolism\n",
      "MH  - *Signal Transduction\n",
      "MH  - Unfolded Protein Response\n",
      "OTO - NOTNLM\n",
      "OT  - * Plantago asiatica mosaic virus\n",
      "OT  - * Turnip mosaic virus\n",
      "OT  - *ER-to-nucleus signaling\n",
      "OT  - *protein folding capacity\n",
      "OT  - *unfolded protein response\n",
      "EDAT- 2020/05/12 06:00\n",
      "MHDA- 2021/03/02 06:00\n",
      "CRDT- 2020/05/12 06:00\n",
      "PHST- 2019/07/01 00:00 [received]\n",
      "PHST- 2020/04/02 00:00 [revised]\n",
      "PHST- 2020/04/23 00:00 [accepted]\n",
      "PHST- 2020/05/12 06:00 [pubmed]\n",
      "PHST- 2021/03/02 06:00 [medline]\n",
      "PHST- 2020/05/12 06:00 [entrez]\n",
      "AID - 10.1111/tpj.14798 [doi]\n",
      "PST - ppublish\n",
      "SO  - Plant J. 2020 Aug;103(3):1233-1245. doi: 10.1111/tpj.14798. Epub 2020 Jun 25.\n",
      "\n",
      "PMID- 32377357\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20211230\n",
      "IS  - 2662-6810 (Print)\n",
      "IS  - 2052-7276 (Linking)\n",
      "VI  - 7\n",
      "DP  - 2020\n",
      "TI  - Exploiting the diversity of tomato: the development of a phenotypically andgenetically detailed germplasm collection.\n",
      "PG  - 66\n",
      "LID - 10.1038/s41438-020-0291-7 [doi]\n",
      "AB  - A collection of 163 accessions, including Solanum pimpinellifolium, Solanumlycopersicum var. cerasiforme and Solanum lycopersicum var. lycopersicum, wasselected to represent the genetic and morphological variability of tomato at its centers of origin and domestication: Andean regions of Peru and Ecuador andMesoamerica. The collection is enriched with S. lycopersicum var. cerasiformefrom the Amazonian region that has not been analyzed previously nor usedextensively. The collection has been morphologically characterized showingdiversity for fruit, flower and vegetative traits. Their genomes were sequencedin the Varitome project and are publicly available(solgenomics.net/projects/varitome). The identified SNPs have been annotated withrespect to their impact and a total number of 37,974 out of 19,364,146 SNPs have been described as high impact by the SnpEeff analysis. GWAS has shownassociations for different traits, demonstrating the potential of this collectionfor this kind of analysis. We have not only identified known QTLs and genes, but also new regions associated with traits such as fruit color, number of flowersper inflorescence or inflorescence architecture. To speed up and facilitate theuse of this information, F2 populations were constructed by crossing the wholecollection with three different parents. This F2 collection is useful for testingSNPs identified by GWAs, selection sweeps or any other candidate gene. All datais available on Solanaceae Genomics Network and the accession and F2 seeds arefreely available at COMAV and at TGRC genebanks. All these resources togethermake this collection a good candidate for genetic studies.\n",
      "CI  - (c) The Author(s) 2020.\n",
      "FAU - Mata-Nicolas, Estefania\n",
      "AU  - Mata-Nicolas E\n",
      "AD  - 1Instituto Universitario de Conservacion y Mejora de la AgrodiversidadValenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\n",
      "FAU - Montero-Pau, Javier\n",
      "AU  - Montero-Pau J\n",
      "AUID- ORCID: 0000-0002-0864-8157\n",
      "AD  - 2Department of Biochemistry and Molecular Biology, Universitat de Valencia,Valencia, Spain.\n",
      "FAU - Gimeno-Paez, Esther\n",
      "AU  - Gimeno-Paez E\n",
      "AD  - 1Instituto Universitario de Conservacion y Mejora de la AgrodiversidadValenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\n",
      "FAU - Garcia-Carpintero, Victor\n",
      "AU  - Garcia-Carpintero V\n",
      "AD  - 1Instituto Universitario de Conservacion y Mejora de la AgrodiversidadValenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\n",
      "FAU - Ziarsolo, Peio\n",
      "AU  - Ziarsolo P\n",
      "AD  - 1Instituto Universitario de Conservacion y Mejora de la AgrodiversidadValenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\n",
      "FAU - Menda, Naama\n",
      "AU  - Menda N\n",
      "AD  - 3Boyce Thompson Institute, Ithaca, NY USA.\n",
      "FAU - Mueller, Lukas A\n",
      "AU  - Mueller LA\n",
      "AD  - 3Boyce Thompson Institute, Ithaca, NY USA.\n",
      "FAU - Blanca, Jose\n",
      "AU  - Blanca J\n",
      "AD  - 1Instituto Universitario de Conservacion y Mejora de la AgrodiversidadValenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\n",
      "FAU - Canizares, Joaquin\n",
      "AU  - Canizares J\n",
      "AD  - 1Instituto Universitario de Conservacion y Mejora de la AgrodiversidadValenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\n",
      "FAU - van der Knaap, Esther\n",
      "AU  - van der Knaap E\n",
      "AD  - 4Institute of Plant Breeding, Genetics and Genomics, University of Georgia,Georgia, GA USA.\n",
      "AD  - 5Department of Horticulture, University of Georgia, Georgia, GA USA.\n",
      "FAU - Diez, Maria Jose\n",
      "AU  - Diez MJ\n",
      "AD  - 1Instituto Universitario de Conservacion y Mejora de la AgrodiversidadValenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\n",
      "LA  - eng\n",
      "GR  - NSFIOS1564366/National Science Foundation (NSF)\n",
      "PT  - Journal Article\n",
      "DEP - 20200501\n",
      "PL  - England\n",
      "TA  - Hortic Res\n",
      "JT  - Horticulture research\n",
      "JID - 101655540\n",
      "PMC - PMC7192925\n",
      "OTO - NOTNLM\n",
      "OT  - *Agricultural genetics\n",
      "OT  - *Plant breeding\n",
      "COIS- Conflict of interestThe authors declare that they have no conflict of interest.\n",
      "EDAT- 2020/05/08 06:00\n",
      "MHDA- 2020/05/08 06:01\n",
      "CRDT- 2020/05/08 06:00\n",
      "PHST- 2019/12/28 00:00 [received]\n",
      "PHST- 2020/03/06 00:00 [revised]\n",
      "PHST- 2020/03/16 00:00 [accepted]\n",
      "PHST- 2020/05/08 06:00 [entrez]\n",
      "PHST- 2020/05/08 06:00 [pubmed]\n",
      "PHST- 2020/05/08 06:01 [medline]\n",
      "AID - 10.1038/s41438-020-0291-7 [doi]\n",
      "AID - 10.1038/s41438-020-0291-7 [pii]\n",
      "PST - epublish\n",
      "SO  - Hortic Res. 2020 May 1;7:66. doi: 10.1038/s41438-020-0291-7. eCollection 2020.\n",
      "\n",
      "PMID- 32359157\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20211118\n",
      "LR  - 20211118\n",
      "IS  - 1076-836X (Electronic)\n",
      "IS  - 1063-5157 (Linking)\n",
      "VI  - 70\n",
      "IP  - 1\n",
      "DP  - 2021 Jan 1\n",
      "TI  - Phylogenomics Reveals Ancient Gene Tree Discordance in the Amphibian Tree ofLife.\n",
      "PG  - 49-66\n",
      "LID - 10.1093/sysbio/syaa034 [doi]\n",
      "AB  - Molecular phylogenies have yielded strong support for many parts of the amphibianTree of Life, but poor support for the resolution of deeper nodes, includingrelationships among families and orders. To clarify these relationships, weprovide a phylogenomic perspective on amphibian relationships by developing ataxon-specific Anchored Hybrid Enrichment protocol targeting hundreds ofconserved exons which are effective across the class. After obtaining data from220 loci for 286 species (representing 94% of the families and 44% of thegenera), we estimate a phylogeny for extant amphibians and identify genetree-species tree conflict across the deepest branches of the amphibianphylogeny. We perform locus-by-locus genealogical interrogation of alternativetopological hypotheses for amphibian monophyly, focusing on interordinalrelationships. We find that phylogenetic signal deep in the amphibian phylogenyvaries greatly across loci in a manner that is consistent with incomplete lineagesorting in the ancestral lineage of extant amphibians. Our results overwhelminglysupport amphibian monophyly and a sister relationship between frogs andsalamanders, consistent with the Batrachia hypothesis. Species tree analysesconverge on a small set of topological hypotheses for the relationships amongextant amphibian families. These results clarify several contentious portions of the amphibian Tree of Life, which in conjunction with a set of vetted fossilcalibrations, support a surprisingly younger timescale for crown and ordinalamphibian diversification than previously reported. More broadly, our studyprovides insight into the sources, magnitudes, and heterogeneity of supportacross loci in phylogenomic data sets.[AIC; Amphibia; Batrachia; Phylogeny; gene tree-species tree discordance; genomics; information theory.].\n",
      "CI  - (c) The Author(s) 2020. Published by Oxford University Press, on behalf of theSociety of Systematic Biologists.\n",
      "FAU - Hime, Paul M\n",
      "AU  - Hime PM\n",
      "AD  - Biodiversity Institute, University of Kansas, Lawrence, KS 66045, USA.\n",
      "AD  - Department of Biology, University of Kentucky, Lexington, KY 40506, USA.\n",
      "FAU - Lemmon, Alan R\n",
      "AU  - Lemmon AR\n",
      "AD  - Department of Scientific Computing, Florida State University, Tallahassee, FL32306, USA.\n",
      "FAU - Lemmon, Emily C Moriarty\n",
      "AU  - Lemmon ECM\n",
      "AD  - Department of Biological Science, Florida State University, Tallahassee, FL32306, USA.\n",
      "FAU - Prendini, Elizabeth\n",
      "AU  - Prendini E\n",
      "AD  - Division of Vertebrate Zoology: Herpetology, American Museum of Natural History, New York, NY 10024, USA.\n",
      "FAU - Brown, Jeremy M\n",
      "AU  - Brown JM\n",
      "AD  - Department of Biological Sciences and Museum of Natural Science, Louisiana State University, Baton Rouge, LA 70803, USA.\n",
      "FAU - Thomson, Robert C\n",
      "AU  - Thomson RC\n",
      "AD  - School of Life Sciences, University of Hawai'i, Honolulu, HI 96822, USA.\n",
      "FAU - Kratovil, Justin D\n",
      "AU  - Kratovil JD\n",
      "AD  - Department of Biology, University of Kentucky, Lexington, KY 40506, USA.\n",
      "AD  - Department of Entomology, University of Kentucky, Lexington, KY 40546, USA.\n",
      "FAU - Noonan, Brice P\n",
      "AU  - Noonan BP\n",
      "AD  - Department of Biology, University of Mississippi, Oxford, MS 38677, USA.\n",
      "FAU - Pyron, R Alexander\n",
      "AU  - Pyron RA\n",
      "AD  - Department of Biological Sciences, The George Washington University, Washington, DC 20052, USA.\n",
      "FAU - Peloso, Pedro L V\n",
      "AU  - Peloso PLV\n",
      "AD  - Division of Vertebrate Zoology: Herpetology, American Museum of Natural History, New York, NY 10024, USA.\n",
      "AD  - Instituto de Ciencias Biologicas, Universidade Federal do Para, Belem, 66075-750,Brazil.\n",
      "FAU - Kortyna, Michelle L\n",
      "AU  - Kortyna ML\n",
      "AD  - Department of Biological Science, Florida State University, Tallahassee, FL32306, USA.\n",
      "FAU - Keogh, J Scott\n",
      "AU  - Keogh JS\n",
      "AD  - Division of Ecology and Evolution, Research School of Biology, The AustralianNational University, Canberra, 2601, Australia.\n",
      "FAU - Donnellan, Stephen C\n",
      "AU  - Donnellan SC\n",
      "AD  - South Australian Museum, North Terrace, Adelaide 5000, Australia.\n",
      "AD  - School of Biological Sciences, University of Adelaide, Adelaide 5005, Australia.\n",
      "FAU - Mueller, Rachel Lockridge\n",
      "AU  - Mueller RL\n",
      "AD  - Department of Biology, Colorado State University, Fort Collins, CO 80523, USA.\n",
      "FAU - Raxworthy, Christopher J\n",
      "AU  - Raxworthy CJ\n",
      "AD  - Division of Vertebrate Zoology: Herpetology, American Museum of Natural History, New York, NY 10024, USA.\n",
      "FAU - Kunte, Krushnamegh\n",
      "AU  - Kunte K\n",
      "AD  - National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bengaluru 560065, India.\n",
      "FAU - Ron, Santiago R\n",
      "AU  - Ron SR\n",
      "AD  - Museo de Zoologia, Escuela de Biologia, Pontificia Universidad Catolica delEcuador, Quito, Ecuador.\n",
      "FAU - Das, Sandeep\n",
      "AU  - Das S\n",
      "AD  - Forest Ecology and Biodiversity Conservation Division, Kerala Forest ResearchInstitute, Peechi, Kerala 680653, India.\n",
      "FAU - Gaitonde, Nikhil\n",
      "AU  - Gaitonde N\n",
      "AD  - National Centre for Biological Sciences, Tata Institute of Fundamental Research, Bengaluru 560065, India.\n",
      "FAU - Green, David M\n",
      "AU  - Green DM\n",
      "AD  - Redpath Museum, McGill University, Montreal, Quebec H3A 0C4, Canada.\n",
      "FAU - Labisko, Jim\n",
      "AU  - Labisko J\n",
      "AD  - The Durrell Institute of Conservation and Ecology, School of Anthropology andConservation, The University of Kent, Canterbury, Kent, CT2 7NR, UK.\n",
      "AD  - Island Biodiversity and Conservation Centre, University of Seychelles, PO Box1348, Anse Royale, Mahe, Seychelles.\n",
      "FAU - Che, Jing\n",
      "AU  - Che J\n",
      "AD  - State Key Laboratory of Genetic Resources and Evolution, Kunming Institute ofZoology, Kunming 650223, China.\n",
      "AD  - Center for Excellence in Animal Evolution and Genetics, Chinese Academy ofSciences, Kunming 650223, China.\n",
      "FAU - Weisrock, David W\n",
      "AU  - Weisrock DW\n",
      "AD  - Department of Biology, University of Kentucky, Lexington, KY 40506, USA.\n",
      "LA  - eng\n",
      "SI  - Dryad/10.5061/dryad.9kd51c5dc\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "PL  - England\n",
      "TA  - Syst Biol\n",
      "JT  - Systematic biology\n",
      "JID - 9302532\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Anura\n",
      "MH  - *Fossils\n",
      "MH  - *Genomics\n",
      "MH  - Humans\n",
      "MH  - Phylogeny\n",
      "PMC - PMC7823230\n",
      "EDAT- 2020/05/03 06:00\n",
      "MHDA- 2021/11/19 06:00\n",
      "CRDT- 2020/05/03 06:00\n",
      "PHST- 2019/04/26 00:00 [received]\n",
      "PHST- 2020/04/14 00:00 [revised]\n",
      "PHST- 2020/04/14 00:00 [accepted]\n",
      "PHST- 2020/05/03 06:00 [pubmed]\n",
      "PHST- 2021/11/19 06:00 [medline]\n",
      "PHST- 2020/05/03 06:00 [entrez]\n",
      "AID - 5828236 [pii]\n",
      "AID - 10.1093/sysbio/syaa034 [doi]\n",
      "PST - ppublish\n",
      "SO  - Syst Biol. 2021 Jan 1;70(1):49-66. doi: 10.1093/sysbio/syaa034.\n",
      "\n",
      "PMID- 32293439\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200817\n",
      "LR  - 20200817\n",
      "IS  - 1755-8794 (Electronic)\n",
      "IS  - 1755-8794 (Linking)\n",
      "VI  - 12\n",
      "IP  - 1\n",
      "DP  - 2019 Nov 21\n",
      "TI  - Clinical, cytogenetic, and molecular findings in a patient with ring chromosome4: case report and literature review.\n",
      "PG  - 167\n",
      "LID - 10.1186/s12920-019-0614-4 [doi]\n",
      "AB  - BACKGROUND: Since 1969, 49 cases have been presented on ring chromosome 4. All ofthese cases have been characterized for the loss of genetic material. The geneslocated in these chromosomal regions are related to the phenotype. CASEPRESENTATION: A 10-year-old Ecuadorian Mestizo girl with ring chromosome 4 wasclinically, cytogenetically and molecularly analysed. Clinical examinationrevealed congenital anomalies, including microcephaly, prominent nose,micrognathia, low set ears, bilateral clinodactyly of the fifth finger, smallsacrococcygeal dimple, short stature and mental retardation. Cytogenetic studies showed a mosaic karyotype, mos 46,XX,r(4)(p16.3q35.2)/46,XX, with a ringchromosome 4 from 75 to 79% in three studies conducted over ten years. Theseresults were confirmed by fluorescence in situ hybridization (FISH). Loss of 1.7 Mb and gain of 342 kb in 4p16.3 and loss of 3 Mb in 4q35.2 were identified byhigh-resolution mapping array. CONCLUSION: Most cases with ring chromosome 4 havedeletion of genetic material in terminal regions; however, our case has inv dupdel rearrangement in the ring chromosome formation. Heterogeneous clinicalfeatures in all cases reviewed are related to the amount of genetic material lostor gained. The application of several techniques can increase our knowledge ofring chromosome 4 and its deviations from typical \"ring syndrome.\"\n",
      "FAU - Paz-Y-Mino, Cesar\n",
      "AU  - Paz-Y-Mino C\n",
      "AUID- ORCID: 0000-0002-6693-7344\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE. Av. Mariscal Sucre y Av. Mariana de Jesus, Sede Occidental, Bloque I, 2 floor, 170129, Quito, Ecuador. cesar.pazymino@ute.edu.ec.\n",
      "FAU - Proano, Ana\n",
      "AU  - Proano A\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE. Av. Mariscal Sucre y Av. Mariana de Jesus, Sede Occidental, Bloque I, 2 floor, 170129, Quito, Ecuador.\n",
      "FAU - Verdezoto, Stella D\n",
      "AU  - Verdezoto SD\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE. Av. Mariscal Sucre y Av. Mariana de Jesus, Sede Occidental, Bloque I, 2 floor, 170129, Quito, Ecuador.\n",
      "FAU - Garcia, Juan Luis\n",
      "AU  - Garcia JL\n",
      "AD  - Institute of Molecular and Cellular Biology of Cancer (IBMCC), University ofSalamanca-SACYL-CSIC, Salamanca, Spain.\n",
      "AD  - Molecular Medicine Unit, Department of Medicine, Biomedical Research Institute ofSalamanca (IBSAL), Salamanca, Spain.\n",
      "FAU - Hernandez-Rivas, Jesus Maria\n",
      "AU  - Hernandez-Rivas JM\n",
      "AD  - Molecular Medicine Unit, Department of Medicine, Biomedical Research Institute ofSalamanca (IBSAL), Salamanca, Spain.\n",
      "AD  - Servicio de Hematologia, Hospital Universitario de Salamanca, Universidad deSalamanca, Salamanca, Spain.\n",
      "FAU - Leone, Paola E\n",
      "AU  - Leone PE\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE. Av. Mariscal Sucre y Av. Mariana de Jesus, Sede Occidental, Bloque I, 2 floor, 170129, Quito, Ecuador. paola.leone@ute.edu.ec.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20191121\n",
      "PL  - England\n",
      "TA  - BMC Med Genomics\n",
      "JT  - BMC medical genomics\n",
      "JID - 101319628\n",
      "RN  - Ring chromosome 4 syndrome\n",
      "SB  - IM\n",
      "MH  - Child\n",
      "MH  - Chromosome Disorders/*genetics/*pathology\n",
      "MH  - Chromosomes, Human, Pair 4/genetics\n",
      "MH  - Cytogenetic Analysis\n",
      "MH  - Female\n",
      "MH  - Humans\n",
      "MH  - In Situ Hybridization, Fluorescence\n",
      "MH  - Karyotyping\n",
      "MH  - Ring Chromosomes\n",
      "PMC - PMC7087353\n",
      "OTO - NOTNLM\n",
      "OT  - *46,XX,r(4)(p16.3q35.2)\n",
      "OT  - *FISH\n",
      "OT  - *Mapping array\n",
      "OT  - *Mosaic\n",
      "OT  - *Ring chromosome 4\n",
      "OT  - *inv dup del rearrangement\n",
      "EDAT- 2020/04/16 06:00\n",
      "MHDA- 2020/08/18 06:00\n",
      "CRDT- 2020/04/16 06:00\n",
      "PHST- 2019/02/08 00:00 [received]\n",
      "PHST- 2019/11/06 00:00 [accepted]\n",
      "PHST- 2020/04/16 06:00 [entrez]\n",
      "PHST- 2020/04/16 06:00 [pubmed]\n",
      "PHST- 2020/08/18 06:00 [medline]\n",
      "AID - 10.1186/s12920-019-0614-4 [doi]\n",
      "AID - 10.1186/s12920-019-0614-4 [pii]\n",
      "PST - epublish\n",
      "SO  - BMC Med Genomics. 2019 Nov 21;12(1):167. doi: 10.1186/s12920-019-0614-4.\n",
      "\n",
      "PMID- 32292644\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20200928\n",
      "IS  - 2167-8359 (Print)\n",
      "IS  - 2167-8359 (Linking)\n",
      "VI  - 8\n",
      "DP  - 2020\n",
      "TI  - ECuADOR-Easy Curation of Angiosperm Duplicated Organellar Regions, a tool forcleaning and curating plastomes assembled from next generation sequencingpipelines.\n",
      "PG  - e8699\n",
      "LID - 10.7717/peerj.8699 [doi]\n",
      "AB  - Background: With the rapid increase in availability of genomic resources offered by Next-Generation Sequencing (NGS) and the availability of free online genomicdatabases, efficient and standardized metadata curation approaches have becomeincreasingly critical for the post-processing stages of biological data.Especially in organelle-based studies using circular chloroplast genome datasets,the assembly of the main structural regions in random order and orientationrepresents a major limitation in our ability to easily generate \"ready-to-align\" datasets for phylogenetic reconstruction, at both small and large taxonomicscales. In addition, current practices discard the most variable regions of thegenomes to facilitate the alignment of the remaining coding regions.Nevertheless, no software is currently available to perform curation to such adegree, through simple detection, organization and positioning of the mainplastome regions, making it a time-consuming and error-prone process. Here weintroduce a fast and user friendly software ECuADOR, a Perl script specificallydesigned to automate the detection and reorganization of newly assembledplastomes obtained from any source available (NGS, sanger sequencing or assembleroutput). Methods: ECuADOR uses a sliding-window approach to detect long repeated sequences in draft sequences, which then identifies the inverted repeat regions(IRs), even in case of artifactual breaks or sequencing errors and automates the rearrangement of the sequence to the widely used LSC-Irb-SSC-IRa order. Thisfacilitates rapid post-editing steps such as creation of genome alignments,detection of variable regions, SNP detection and phylogenomic analyses. Results: ECuADOR was successfully tested on plant families throughout the angiospermphylogeny by curating 161 chloroplast datasets. ECuADOR first identified andreordered the central regions (LSC-Irb-SSC-IRa) for each dataset and thenproduced a new annotation for the chloroplast sequences. The process took lessthan 20 min with a maximum memory requirement of 150 MB and an accuracy of over99%. Conclusions: ECuADOR is the sole de novo one-step recognition andre-ordination tool that provides facilitation in the post-processing analysis of the extra nuclear genomes from NGS data. The program is available athttps://github.com/BiodivGenomic/ECuADOR/.\n",
      "CI  - (c) 2020 Armijos Carrion et al.\n",
      "FAU - Armijos Carrion, Angelo D\n",
      "AU  - Armijos Carrion AD\n",
      "AD  - Biodiversity Genomics Team, Plant Ecophysiology & Evolution Group, Guangxi KeyLaboratory of Forest Ecology and Conservation, College of Forestry, GuangxiUniversity, Nanning, Guangxi, PR China.\n",
      "FAU - Hinsinger, Damien D\n",
      "AU  - Hinsinger DD\n",
      "AUID- ORCID: 0000-0001-7459-7610\n",
      "AD  - Biodiversity Genomics Team, Plant Ecophysiology & Evolution Group, Guangxi KeyLaboratory of Forest Ecology and Conservation, College of Forestry, GuangxiUniversity, Nanning, Guangxi, PR China.\n",
      "AD  - Alliance for Conservation Tree Genomics, Pha Tad Ke Botanical Garden, LuangPrabang, Laos.\n",
      "FAU - Strijk, Joeri S\n",
      "AU  - Strijk JS\n",
      "AUID- ORCID: 0000-0003-1109-7015\n",
      "AD  - Biodiversity Genomics Team, Plant Ecophysiology & Evolution Group, Guangxi KeyLaboratory of Forest Ecology and Conservation, College of Forestry, GuangxiUniversity, Nanning, Guangxi, PR China.\n",
      "AD  - Alliance for Conservation Tree Genomics, Pha Tad Ke Botanical Garden, LuangPrabang, Laos.\n",
      "AD  - State Key Laboratory for Conservation and Utilization of SubtropicalAgro-Bioresources, Guangxi University, Nanning, Guangxi, PR China.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20200407\n",
      "PL  - United States\n",
      "TA  - PeerJ\n",
      "JT  - PeerJ\n",
      "JID - 101603425\n",
      "PMC - PMC7147433\n",
      "OTO - NOTNLM\n",
      "OT  - Automated workflow\n",
      "OT  - Bioinformatics\n",
      "OT  - NGS\n",
      "OT  - Phylogenomics\n",
      "OT  - Plastome\n",
      "OT  - Sliding window\n",
      "COIS- The authors declare that they have no competing interests.\n",
      "EDAT- 2020/04/16 06:00\n",
      "MHDA- 2020/04/16 06:01\n",
      "CRDT- 2020/04/16 06:00\n",
      "PHST- 2019/10/28 00:00 [received]\n",
      "PHST- 2020/02/06 00:00 [accepted]\n",
      "PHST- 2020/04/16 06:00 [entrez]\n",
      "PHST- 2020/04/16 06:00 [pubmed]\n",
      "PHST- 2020/04/16 06:01 [medline]\n",
      "AID - 10.7717/peerj.8699 [doi]\n",
      "AID - 8699 [pii]\n",
      "PST - epublish\n",
      "SO  - PeerJ. 2020 Apr 7;8:e8699. doi: 10.7717/peerj.8699. eCollection 2020.\n",
      "\n",
      "PMID- 32247171\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210618\n",
      "LR  - 20210618\n",
      "IS  - 1876-7737 (Electronic)\n",
      "IS  - 1874-3919 (Linking)\n",
      "VI  - 220\n",
      "DP  - 2020 May 30\n",
      "TI  - Snake venomics, experimental toxic activities and clinical characteristics ofhuman envenomation by Bothrocophias myersi (Serpentes: Viperidae) from Colombia.\n",
      "PG  - 103758\n",
      "LID - S1874-3919(20)30126-3 [pii]\n",
      "LID - 10.1016/j.jprot.2020.103758 [doi]\n",
      "AB  - Venoms of the viperid genus Bothrocophias, restricted to Colombia and Ecuador,are poorly known. Only a proteomic analysis of B. campbelli venom has beendescribed. In this work we present a proteomic study of B. myersi venom, itsbiological activities, and describe the clinical characteristics of a patientbitten by this species. B. myersi venom mainly consists of phospholipases A2(54.0%) and metalloproteinases (21.5%), among proteins of twelve differentfamilies. This venom exhibited proteolytic, phospholipase A2, myotoxic,edema-forming, and lethal activities. Enzymatic activities did not showstatistically significant differences in comparison to Bothrops asper venom, but B. myersi venom displayed weaker hemorrhagic and coagulant activities. PolyvalentViperidae antivenoms produced in Costa Rica and Colombia cross-recognized B.myersi venom by ELISA, however only the latter neutralized its lethal activity inmice when tested at a ratio of 3mg venom/mL antivenom, suggesting it should beuseful to treat envenomings inflicted by this species. A patient bitten by B.myersi developed edema and myotoxicity, evidenced by an increased creatine kinaseactivity in plasma. A good correlation was found between experimental biological activities of Bothrocophias myersi venom and the clinical features of anenvenoming provoked by this species. SIGNIFICANCE: The proteomiccharacterization, toxicity, immunorecognition and neutralization of Bothrocophiasmyersi venom have been determined for the first time. The distribution of thispit viper is restricted to Colombia and Ecuador, and its venom contains a highproportion of phospholipases A2 and metalloproteinases. The polyvalent antivenom produced in Colombia neutralized the lethal activity of this venom in vivo, andtherefore should be effective in the treatment of envenomings by this snake.\n",
      "CI  - Copyright (c) 2020 Elsevier B.V. All rights reserved.\n",
      "FAU - Pereanez, Jaime Andres\n",
      "AU  - Pereanez JA\n",
      "AD  - Programa de Ofidismo/Escorpionismo-Toxinologia, alternativas terapeuticas yalimentarias, Departamento de Farmacia, Facultad de Ciencias Farmaceuticas yAlimentarias, Universidad de Antioquia UdeA, Calle 70 No. 52-21, Medellin,Colombia. Electronic address: jaime.pereanez@udea.edu.co.\n",
      "FAU - Preciado, Lina Maria\n",
      "AU  - Preciado LM\n",
      "AD  - Programa de Ofidismo/Escorpionismo-Toxinologia, alternativas terapeuticas yalimentarias, Departamento de Farmacia, Facultad de Ciencias Farmaceuticas yAlimentarias, Universidad de Antioquia UdeA, Calle 70 No. 52-21, Medellin,Colombia.\n",
      "FAU - Fernandez, Julian\n",
      "AU  - Fernandez J\n",
      "AD  - Instituto Clodomiro Picado, Facultad de Microbiologia, Universidad de Costa Rica,San Jose 11501, Costa Rica.\n",
      "FAU - Camacho, Erika\n",
      "AU  - Camacho E\n",
      "AD  - Instituto Clodomiro Picado, Facultad de Microbiologia, Universidad de Costa Rica,San Jose 11501, Costa Rica.\n",
      "FAU - Lomonte, Bruno\n",
      "AU  - Lomonte B\n",
      "AD  - Instituto Clodomiro Picado, Facultad de Microbiologia, Universidad de Costa Rica,San Jose 11501, Costa Rica.\n",
      "FAU - Castro, Fernando\n",
      "AU  - Castro F\n",
      "AD  - Laboratorio de Herpetologia y Toxinologia, Facultad de Salud, Universidad delValle, Cali, Colombia.\n",
      "FAU - Canas, Carlos Alberto\n",
      "AU  - Canas CA\n",
      "AD  - Unit of Rheumatology, Fundacion Valle del Lili & Universidad Icesi, Cali,Colombia.\n",
      "FAU - Galvis, Carlos\n",
      "AU  - Galvis C\n",
      "AD  - Fundacion Zoologico de Cali, Valle del Cauca, Cali, Colombia.\n",
      "FAU - Castano, Santiago\n",
      "AU  - Castano S\n",
      "AD  - Laboratorio de Herpetologia y Toxinologia, Facultad de Salud, Universidad delValle, Cali, Colombia.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20200401\n",
      "PL  - Netherlands\n",
      "TA  - J Proteomics\n",
      "JT  - Journal of proteomics\n",
      "JID - 101475056\n",
      "RN  - 0 (Antivenins)\n",
      "RN  - 0 (Crotalid Venoms)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Antivenins\n",
      "MH  - Colombia\n",
      "MH  - *Crotalid Venoms\n",
      "MH  - *Crotalinae\n",
      "MH  - Humans\n",
      "MH  - Mice\n",
      "MH  - Proteomics\n",
      "MH  - *Viperidae\n",
      "OTO - NOTNLM\n",
      "OT  - *Bothrocophias\n",
      "OT  - *Colombia\n",
      "OT  - *Proteomics\n",
      "OT  - *Snake venom\n",
      "OT  - *Snakebite\n",
      "COIS- Declaration of Competing Interest None.\n",
      "EDAT- 2020/04/05 06:00\n",
      "MHDA- 2021/06/22 06:00\n",
      "CRDT- 2020/04/05 06:00\n",
      "PHST- 2020/01/25 00:00 [received]\n",
      "PHST- 2020/02/21 00:00 [revised]\n",
      "PHST- 2020/03/23 00:00 [accepted]\n",
      "PHST- 2020/04/05 06:00 [pubmed]\n",
      "PHST- 2021/06/22 06:00 [medline]\n",
      "PHST- 2020/04/05 06:00 [entrez]\n",
      "AID - S1874-3919(20)30126-3 [pii]\n",
      "AID - 10.1016/j.jprot.2020.103758 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Proteomics. 2020 May 30;220:103758. doi: 10.1016/j.jprot.2020.103758. Epub 2020Apr 1.\n",
      "\n",
      "PMID- 32221920\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210201\n",
      "LR  - 20210201\n",
      "IS  - 1940-6029 (Electronic)\n",
      "IS  - 1064-3745 (Linking)\n",
      "VI  - 2116\n",
      "DP  - 2020\n",
      "TI  - Genome-Wide Proteomics and Phosphoproteomics Analysis of Trypanosoma cruzi DuringDifferentiation.\n",
      "PG  - 139-159\n",
      "LID - 10.1007/978-1-0716-0294-2_11 [doi]\n",
      "AB  - Trypanosoma cruzi is a pathogenic protozoan that still has an impact on publichealth, despite the decrease in the number of infection cases along the years. T.cruzi possesses an heteroxenic life cycle in which it differentiates in at least four forms. Among the differentiation processes, metacyclogenesis has beenexploited in different views by researchers. An intriguing question that rises ishow metacyclogenesis is triggered and controlled by cell signaling and which are the differentially expressed proteins and posttranslational modificationsinvolved in this process. An important cell signaling pathway is the proteinphosphorylation, and it is reinforced in T. cruzi in which the gene expressioncontrol occurs almost exclusively posttranscriptionally. Additionally, the numberof protein kinases in T. cruzi is relatively high compared to other organisms. A way to approach these questions is evaluating the cells through phosphoproteomicsand proteomics. In this chapter, we will describe the steps from the cell proteinextraction, digestion and fractionation, phosphopeptide enrichment, to LC-MS/MSanalysis as well as a brief overview on peptide identification. In addition, apublished method for in vitro metacyclogenesis will be detailed.\n",
      "FAU - Batista, Michel\n",
      "AU  - Batista M\n",
      "AD  - Carlos Chagas Institute, Fiocruz Parana, Curitiba, Parana, Brazil.\n",
      "FAU - Amorim, Juliana Carolina\n",
      "AU  - Amorim JC\n",
      "AD  - Faculty of Medicine, Catholic University of Cuenca, Cuenca, Azuay, Ecuador.\n",
      "FAU - Lucena, Aline Castro Rodrigues\n",
      "AU  - Lucena ACR\n",
      "AD  - Carlos Chagas Institute, Fiocruz Parana, Curitiba, Parana, Brazil.\n",
      "FAU - Kugeratski, Fernanda Grande\n",
      "AU  - Kugeratski FG\n",
      "AD  - Department of Cancer Biology, Metastasis Research Center, University of Texas MD Anderson Cancer Center, Houston, TX, USA.\n",
      "FAU - de Paula Lima, Carla Vanessa\n",
      "AU  - de Paula Lima CV\n",
      "AD  - Molecular Biology Institute of Parana, Curitiba, Parana, Brazil.\n",
      "FAU - Marchini, Fabricio Klerynton\n",
      "AU  - Marchini FK\n",
      "AD  - Carlos Chagas Institute, Fiocruz Parana, Curitiba, Parana, Brazil.fabricio.marchini@fiocruz.br.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - United States\n",
      "TA  - Methods Mol Biol\n",
      "JT  - Methods in molecular biology (Clifton, N.J.)\n",
      "JID - 9214969\n",
      "RN  - 0 (Phosphoproteins)\n",
      "RN  - 0 (Protozoan Proteins)\n",
      "SB  - IM\n",
      "MH  - Chromatography, Liquid/methods\n",
      "MH  - Gene Expression Regulation, Developmental\n",
      "MH  - Life Cycle Stages/genetics\n",
      "MH  - Parasitology/methods\n",
      "MH  - Phosphoproteins/*analysis/metabolism\n",
      "MH  - Phosphorylation/physiology\n",
      "MH  - Proteomics/*methods\n",
      "MH  - Protozoan Proteins/*analysis/metabolism\n",
      "MH  - Tandem Mass Spectrometry/methods\n",
      "MH  - Trypanosoma cruzi/*physiology\n",
      "OTO - NOTNLM\n",
      "OT  - *LC-MS/MS\n",
      "OT  - *Mass spectrometry\n",
      "OT  - *Metacyclogenesis\n",
      "OT  - *Phosphoproteomics\n",
      "OT  - *Proteomics\n",
      "OT  - *Trypanosoma cruzi\n",
      "EDAT- 2020/03/30 06:00\n",
      "MHDA- 2021/02/02 06:00\n",
      "CRDT- 2020/03/30 06:00\n",
      "PHST- 2020/03/30 06:00 [entrez]\n",
      "PHST- 2020/03/30 06:00 [pubmed]\n",
      "PHST- 2021/02/02 06:00 [medline]\n",
      "AID - 10.1007/978-1-0716-0294-2_11 [doi]\n",
      "PST - ppublish\n",
      "SO  - Methods Mol Biol. 2020;2116:139-159. doi: 10.1007/978-1-0716-0294-2_11.\n",
      "\n",
      "PMID- 32088760\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20201223\n",
      "LR  - 20210716\n",
      "IS  - 1432-0614 (Electronic)\n",
      "IS  - 0175-7598 (Linking)\n",
      "VI  - 104\n",
      "IP  - 8\n",
      "DP  - 2020 Apr\n",
      "TI  - Functional genomic analysis of bacterial lignin degraders: diversity inmechanisms of lignin oxidation and metabolism.\n",
      "PG  - 3305-3320\n",
      "LID - 10.1007/s00253-019-10318-y [doi]\n",
      "AB  - Although several bacterial lignin-oxidising enzymes have been discovered inrecent years, it is not yet clear whether different lignin-degrading bacteria usesimilar mechanisms for lignin oxidation and degradation of lignin fragments.Genome sequences of 13 bacterial lignin-oxidising bacteria, including new genome sequences for Microbacterium phyllosphaerae and Agrobacterium sp., were analysed for the presence of lignin-oxidising enzymes and aromatic degradation geneclusters that could be used to metabolise the products of lignin degradation. Tenbacterial genomes contain DyP-type peroxidases, and ten bacterial strains containputative multi-copper oxidases (MCOs), both known to have activity for ligninoxidation. Only one strain lacks both MCOs and DyP-type peroxidase genes. Eleven bacterial genomes contain aromatic degradation gene clusters, of which tencontain the central beta-ketoadipate pathway, with variable numbers and types of degradation clusters for other aromatic substrates. Hence, there appear to bediverse metabolic strategies used for lignin oxidation in bacteria, while thebeta-ketoadipate pathway appears to be the most common route for aromaticmetabolism in lignin-degrading bacteria.\n",
      "FAU - Granja-Travez, Rommel Santiago\n",
      "AU  - Granja-Travez RS\n",
      "AD  - Department of Chemistry, University of Warwick, Coventry, CV4 7AL, UK.\n",
      "AD  - Facultad de Ciencias de la Ingenieria e Industrias, Universidad UTE, Quito,Ecuador.\n",
      "FAU - Persinoti, Gabriela Felix\n",
      "AU  - Persinoti GF\n",
      "AD  - Laboratorio Nacional de Biorrenovaveis (LNBR/CNPEM), Campinas, Brazil.\n",
      "FAU - Squina, Fabio M\n",
      "AU  - Squina FM\n",
      "AD  - Programa de Processos Tecnologicos e Ambientais, Universidade de Sorocaba,Sorocaba, Brazil.\n",
      "FAU - Bugg, Timothy D H\n",
      "AU  - Bugg TDH\n",
      "AUID- ORCID: http://orcid.org/0000-0003-3964-4498\n",
      "AD  - Department of Chemistry, University of Warwick, Coventry, CV4 7AL, UK.T.D.Bugg@warwick.ac.uk.\n",
      "LA  - eng\n",
      "GR  - BB/P01738X/1/BB_/Biotechnology and Biological Sciences Research Council/UnitedKingdom\n",
      "GR  - 2015/50590-4/FAPESP\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "DEP - 20200222\n",
      "PL  - Germany\n",
      "TA  - Appl Microbiol Biotechnol\n",
      "JT  - Applied microbiology and biotechnology\n",
      "JID - 8406612\n",
      "RN  - 0 (Bacterial Proteins)\n",
      "RN  - 9005-53-2 (Lignin)\n",
      "RN  - EC 1.- (Oxidoreductases)\n",
      "RN  - EC 1.11.1.- (Peroxidases)\n",
      "RN  - EC 1.16.- (copper oxidase)\n",
      "RN  - Microbacterium phyllosphaerae\n",
      "SB  - IM\n",
      "MH  - Agrobacterium/enzymology/genetics\n",
      "MH  - Bacteria/*enzymology/*genetics\n",
      "MH  - Bacterial Proteins/metabolism\n",
      "MH  - Biochemical Phenomena\n",
      "MH  - *Genome, Bacterial\n",
      "MH  - Genomics\n",
      "MH  - Lignin/*metabolism\n",
      "MH  - Metabolic Engineering\n",
      "MH  - Microbacterium/enzymology/genetics\n",
      "MH  - Oxidation-Reduction\n",
      "MH  - Oxidoreductases/metabolism\n",
      "MH  - Peroxidases/metabolism\n",
      "OTO - NOTNLM\n",
      "OT  - Aromatic degradation pathways\n",
      "OT  - Bacterial lignin degradation\n",
      "OT  - DyP-type peroxidase\n",
      "OT  - Genome sequences\n",
      "OT  - Multi-copper oxidase\n",
      "EDAT- 2020/02/24 06:00\n",
      "MHDA- 2020/12/29 06:00\n",
      "CRDT- 2020/02/24 06:00\n",
      "PHST- 2019/09/13 00:00 [received]\n",
      "PHST- 2019/12/13 00:00 [accepted]\n",
      "PHST- 2019/12/06 00:00 [revised]\n",
      "PHST- 2020/02/24 06:00 [pubmed]\n",
      "PHST- 2020/12/29 06:00 [medline]\n",
      "PHST- 2020/02/24 06:00 [entrez]\n",
      "AID - 10.1007/s00253-019-10318-y [doi]\n",
      "AID - 10.1007/s00253-019-10318-y [pii]\n",
      "PST - ppublish\n",
      "SO  - Appl Microbiol Biotechnol. 2020 Apr;104(8):3305-3320. doi:10.1007/s00253-019-10318-y. Epub 2020 Feb 22.\n",
      "\n",
      "PMID- 31947590\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200929\n",
      "LR  - 20200929\n",
      "IS  - 2073-4425 (Electronic)\n",
      "IS  - 2073-4425 (Linking)\n",
      "VI  - 11\n",
      "IP  - 1\n",
      "DP  - 2020 Jan 14\n",
      "TI  - The Complete Genome of an Endogenous Nimavirus (Nimav-1_LVa) From the PacificWhiteleg Shrimp Penaeus (Litopenaeus) Vannamei.\n",
      "LID - E94 [pii]\n",
      "LID - 10.3390/genes11010094 [doi]\n",
      "AB  - White spot syndrome virus (WSSV), the lone virus of the genus Whispovirus underthe family Nimaviridae, is one of the most devastating viruses affecting theshrimp farming industry. Knowledge about this virus, in particular, its evolutionhistory, has been limited, partly due to its large genome and the lack of otherclosely related free-living viruses for comparative studies. In this study, wereconstructed a full-length endogenous nimavirus consensus genome, Nimav-1_LVa(279,905 bp), in the genome sequence of Penaeus (Litopenaeus) vannamei breedKehai No. 1 (ASM378908v1). This endogenous virus seemed to insert exclusivelyinto the telomeric pentanucleotide microsatellite (TAACC/GGTTA)n. It encoded 117 putative genes, with some containing introns, such as g012 (inhibitor ofapoptosis, IAP), g046 (crustacean hyperglycemic hormone, CHH), g155 (innexin),g158 (Bax inhibitor 1 like). More than a dozen Nimav-1_LVa genes are involved in the pathogen-host interactions. We hypothesized that g046, g155, g158, and g227(semaphorin 1A like) were recruited host genes for their roles in immuneregulation. Sequence analysis indicated that a total of 43 WSSV genes belonged tothe ancestral/core nimavirus gene set, including four genes reported in thisstudy: wsv112 (dUTPase), wsv206, wsv226, and wsv308 (nucleocapsid protein). Theavailability of the Nimav-1_LVa sequence would help understand the geneticdiversity, epidemiology, evolution, and virulence of WSSV.\n",
      "FAU - Bao, Weidong\n",
      "AU  - Bao W\n",
      "AD  - Genetic Information Research Institute, 20380 Town Center Lane, Suite 240,Cupertino, CA 95014, USA.\n",
      "FAU - Tang, Kathy F J\n",
      "AU  - Tang KFJ\n",
      "AD  - Yellow Sea Fisheries Research Institute, Chinese Academy of Fishery Sciences, 106Nanjing Road, Qingdao 266071, China.\n",
      "FAU - Alcivar-Warren, Acacia\n",
      "AU  - Alcivar-Warren A\n",
      "AD  - Fundacion para la Conservation de la Biodiversidad Acuatica y Terrestre (FUCOBI),Quito EC1701, Ecuador.\n",
      "AD  - Environmental Genomics Inc., ONE HEALTH Epigenomics Educational Initiative, P. O.Box 196, Southborough, MA 01772, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20200114\n",
      "PL  - Switzerland\n",
      "TA  - Genes (Basel)\n",
      "JT  - Genes\n",
      "JID - 101551097\n",
      "RN  - 0 (Viral Proteins)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - *Genome, Viral\n",
      "MH  - Host-Pathogen Interactions/*genetics\n",
      "MH  - Nimaviridae/*physiology\n",
      "MH  - Penaeidae/*virology\n",
      "MH  - Viral Proteins/*genetics\n",
      "PMC - PMC7016691\n",
      "OTO - NOTNLM\n",
      "OT  - *DNAV-1_LVa\n",
      "OT  - *Nimav-1_LVa\n",
      "OT  - *Nimaviridae\n",
      "OT  - *Penaeus (Litopenaeus) vannamei\n",
      "OT  - *WSSV\n",
      "OT  - *white spot syndrome virus\n",
      "COIS- The authors declare no conflict of interest.\n",
      "EDAT- 2020/01/18 06:00\n",
      "MHDA- 2020/09/30 06:00\n",
      "CRDT- 2020/01/18 06:00\n",
      "PHST- 2019/12/17 00:00 [received]\n",
      "PHST- 2020/01/08 00:00 [revised]\n",
      "PHST- 2020/01/09 00:00 [accepted]\n",
      "PHST- 2020/01/18 06:00 [entrez]\n",
      "PHST- 2020/01/18 06:00 [pubmed]\n",
      "PHST- 2020/09/30 06:00 [medline]\n",
      "AID - genes11010094 [pii]\n",
      "AID - 10.3390/genes11010094 [doi]\n",
      "PST - epublish\n",
      "SO  - Genes (Basel). 2020 Jan 14;11(1). pii: genes11010094. doi: 10.3390/genes11010094.\n",
      "\n",
      "PMID- 31911190\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200406\n",
      "LR  - 20200408\n",
      "IS  - 1532-0456 (Print)\n",
      "IS  - 1532-0456 (Linking)\n",
      "VI  - 230\n",
      "DP  - 2020 Apr\n",
      "TI  - Assessing the stability of historical and desiccated snake venoms from amedically important Ecuadorian collection.\n",
      "PG  - 108702\n",
      "LID - S1532-0456(20)30002-8 [pii]\n",
      "LID - 10.1016/j.cbpc.2020.108702 [doi]\n",
      "AB  - Bothrops asper and Bothrops atrox are important venomous snakes from Ecuadorresponsible for the most of ophidic accidents, which in the past were treatedwith a national polyvant antivenom. For years, the venom pools were collected andstored at room temperature in a laboratory. Taking into account the controversialability of desiccated samples to retain their biological effects and enzymaticactivities, we investigated the biochemical and toxicological properties ofvenoms after years of storage. The proteomic profiles of historical venomsanalyzed by high-performance liquid chromatography and electrophoresis are verysimilar. The fresh batches of venom were more lethal than those stored for years,just as the initial and current LD50 values of these samples changed. Significantdifferences were showed in the myotoxic and hemorrhagic activity of some venompools, while no significant statistical differences were found for the edemaactivity. The enzymatic assays revealed a variation in proteolytic activity onazocasein and phospholipase A2 activity, and low differences were reported forthrombin-like serine protease activity. The maintenance of the proteomic profile and certain toxicological activities convert this venom library in a valuablesource for research purposes. Nonetheless, the significative reduction oftoxicological activities, such as hemorrhagic activity not feasible using thesesamples for the antivenom production.\n",
      "CI  - Copyright (c) 2020 Elsevier Inc. All rights reserved.\n",
      "FAU - Almeida, Jose R\n",
      "AU  - Almeida JR\n",
      "AD  - Biomolecules Discovery Group, Universidad Regional Amazonica Ikiam, Km 7 ViaMuyuna, Tena, Napo, Ecuador. Electronic address: rafael.dealmeida@ikiam.edu.ec.\n",
      "FAU - Mendes, Bruno\n",
      "AU  - Mendes B\n",
      "AD  - Departamento de Biologia Animal, Instituto de Biologia, Universidade Estadual de Campinas (UNICAMP), Campinas, Sao Paulo, Brazil.\n",
      "FAU - Patino, Ricardo S P\n",
      "AU  - Patino RSP\n",
      "AD  - Biomolecules Discovery Group, Universidad Regional Amazonica Ikiam, Km 7 ViaMuyuna, Tena, Napo, Ecuador.\n",
      "FAU - Pico, Jose\n",
      "AU  - Pico J\n",
      "AD  - Instituto Nacional de Investigacion en Salud Publica (INSPI), Guayaquil,Guayaquil, Ecuador.\n",
      "FAU - Laines, Johanna\n",
      "AU  - Laines J\n",
      "AD  - Instituto Nacional de Investigacion en Salud Publica (INSPI), Guayaquil,Guayaquil, Ecuador.\n",
      "FAU - Teran, Maria\n",
      "AU  - Teran M\n",
      "AD  - Instituto Nacional de Investigacion en Salud Publica (INSPI), Guayaquil,Guayaquil, Ecuador.\n",
      "FAU - Mogollon, Noroska G S\n",
      "AU  - Mogollon NGS\n",
      "AD  - Biomolecules Discovery Group, Universidad Regional Amazonica Ikiam, Km 7 ViaMuyuna, Tena, Napo, Ecuador.\n",
      "FAU - Zaruma-Torres, Fausto\n",
      "AU  - Zaruma-Torres F\n",
      "AD  - Facultad de Ciencias Quimica, Universidad de Cuenca, Cuenca, Azuay, Ecuador;Centro de Innovacion de la Salud - EUS/EP, Cuenca, Azuay, Ecuador.\n",
      "FAU - Caldeira, Cleopatra A da S\n",
      "AU  - Caldeira CADS\n",
      "AD  - Centro de Estudos de Biomoleculas Aplicadas a Saude (CEBio), Fundacao OswaldoCruz de Rondonia (Fiocruz Rondonia), Porto Velho, RO, Brazil; Programa dePos-graduacao em Biologia Experimental (PGBIOEXP), Universidade Federal deRondonia (UNIR).\n",
      "FAU - da Silva, Saulo L\n",
      "AU  - da Silva SL\n",
      "AD  - Facultad de Ciencias Quimica, Universidad de Cuenca, Cuenca, Azuay, Ecuador;Centro de Innovacion de la Salud - EUS/EP, Cuenca, Azuay, Ecuador.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20200103\n",
      "PL  - United States\n",
      "TA  - Comp Biochem Physiol C Toxicol Pharmacol\n",
      "JT  - Comparative biochemistry and physiology. Toxicology & pharmacology : CBP\n",
      "JID - 100959500\n",
      "RN  - 0 (Crotalid Venoms)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Bothrops/metabolism\n",
      "MH  - Crotalid Venoms/*chemistry\n",
      "MH  - Desiccation\n",
      "MH  - Ecuador\n",
      "MH  - Enzyme Stability\n",
      "MH  - Lethal Dose 50\n",
      "MH  - Male\n",
      "MH  - Mice\n",
      "MH  - Proteomics\n",
      "MH  - Specimen Handling\n",
      "OTO - NOTNLM\n",
      "OT  - Desiccated venom\n",
      "OT  - Proteomic\n",
      "OT  - Snake venom\n",
      "OT  - Stability\n",
      "OT  - Toxicological\n",
      "COIS- Declaration of competing interest The authors declare no relevant conflicts ofinterest concerning the present manuscript.\n",
      "EDAT- 2020/01/09 06:00\n",
      "MHDA- 2020/04/09 06:00\n",
      "CRDT- 2020/01/09 06:00\n",
      "PHST- 2019/11/30 00:00 [received]\n",
      "PHST- 2019/12/30 00:00 [revised]\n",
      "PHST- 2020/01/02 00:00 [accepted]\n",
      "PHST- 2020/01/09 06:00 [pubmed]\n",
      "PHST- 2020/04/09 06:00 [medline]\n",
      "PHST- 2020/01/09 06:00 [entrez]\n",
      "AID - S1532-0456(20)30002-8 [pii]\n",
      "AID - 10.1016/j.cbpc.2020.108702 [doi]\n",
      "PST - ppublish\n",
      "SO  - Comp Biochem Physiol C Toxicol Pharmacol. 2020 Apr;230:108702. doi:10.1016/j.cbpc.2020.108702. Epub 2020 Jan 3.\n",
      "\n",
      "PMID- 31902732\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20201222\n",
      "LR  - 20201222\n",
      "IS  - 1879-0445 (Electronic)\n",
      "IS  - 0960-9822 (Linking)\n",
      "VI  - 30\n",
      "IP  - 3\n",
      "DP  - 2020 Feb 3\n",
      "TI  - Genomic and Fitness Consequences of Genetic Rescue in Wild Populations.\n",
      "PG  - 517-522.e5\n",
      "LID - S0960-9822(19)31532-5 [pii]\n",
      "LID - 10.1016/j.cub.2019.11.062 [doi]\n",
      "AB  - Gene flow is an enigmatic evolutionary force because it can limit adaptation but may also rescue small populations from inbreeding depression [1-3]. Severaliconic examples of genetic rescue-increased population growth caused by gene flow[4, 5]-have reversed population declines [6, 7]. However, concerns aboutoutbreeding depression and maladaptive gene flow limit the use of human-mediated gene flow in conservation [8, 9]. Rescue effects of immigration throughdemographic and/or genetic mechanisms have received theoretical and empiricalsupport, but studies that monitor initial and long-term effects of gene flow onindividuals and populations in the wild are lacking. Here, we usedindividual-based mark-recapture, multigenerational pedigrees, and genomics totest the demographic and evolutionary consequences of manipulating gene flow intwo isolated, wild Trinidadian guppy populations. Recipient and sourcepopulations originated from environments with different predation, flow, andresource regimes [10]. We documented 10-fold increases in population sizefollowing gene flow and found that, on average, hybrids lived longer andreproduced more than residents and immigrants. Despite overall genomichomogenization, alleles potentially associated with local adaptation were notentirely swamped by gene flow. Our results suggest that genetic rescue was causednot just by increasing individual genetic diversity, rather new genomic variationfrom immigrants combined with alleles from the recipient population resulted inhighly fit hybrids and subsequent increases in population size. Contrary to theclassic view of maladaptive gene flow, our study reveals conditions under whichimmigration can produce long-term fitness benefits in small populations withoutentirely swamping adaptive variation.\n",
      "CI  - Copyright (c) 2019 The Authors. Published by Elsevier Ltd.. All rights reserved.\n",
      "FAU - Fitzpatrick, Sarah W\n",
      "AU  - Fitzpatrick SW\n",
      "AD  - W.K. Kellogg Biological Station, Michigan State University, 3700 E. Gull LakeDrive, Hickory Corners, MI 49060, USA; Department of Integrative Biology,Michigan State University, 288 Farm Lane, East Lansing, MI 48824, USA; Ecology,Evolutionary Biology, and Behavior Program, Michigan State University, EastLansing, MI 48824, USA. Electronic address: sfitz@msu.edu.\n",
      "FAU - Bradburd, Gideon S\n",
      "AU  - Bradburd GS\n",
      "AD  - Department of Integrative Biology, Michigan State University, 288 Farm Lane, EastLansing, MI 48824, USA; Ecology, Evolutionary Biology, and Behavior Program,Michigan State University, East Lansing, MI 48824, USA.\n",
      "FAU - Kremer, Colin T\n",
      "AU  - Kremer CT\n",
      "AD  - W.K. Kellogg Biological Station, Michigan State University, 3700 E. Gull LakeDrive, Hickory Corners, MI 49060, USA.\n",
      "FAU - Salerno, Patricia E\n",
      "AU  - Salerno PE\n",
      "AD  - Department of Biology, Colorado State University, 1878 Campus Delivery, FortCollins, CO 80523, USA; Universidad Regional Amazonica Ikiam, Km 7 Via Muyuna,Tena, Napo, Ecuador.\n",
      "FAU - Angeloni, Lisa M\n",
      "AU  - Angeloni LM\n",
      "AD  - Department of Biology, Colorado State University, 1878 Campus Delivery, FortCollins, CO 80523, USA; Graduate Degree Program in Ecology, Colorado StateUniversity, Fort Collins, CO 80523, USA.\n",
      "FAU - Funk, W Chris\n",
      "AU  - Funk WC\n",
      "AD  - Department of Biology, Colorado State University, 1878 Campus Delivery, FortCollins, CO 80523, USA; Graduate Degree Program in Ecology, Colorado StateUniversity, Fort Collins, CO 80523, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20200102\n",
      "PL  - England\n",
      "TA  - Curr Biol\n",
      "JT  - Current biology : CB\n",
      "JID - 9107782\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Animals, Wild/genetics/physiology\n",
      "MH  - Female\n",
      "MH  - *Gene Flow\n",
      "MH  - *Genetic Fitness\n",
      "MH  - *Hybridization, Genetic\n",
      "MH  - Male\n",
      "MH  - Pedigree\n",
      "MH  - Poecilia/genetics/*physiology\n",
      "MH  - Population Growth\n",
      "OTO - NOTNLM\n",
      "OT  - *Trinidadian guppies\n",
      "OT  - *admixture\n",
      "OT  - *demography\n",
      "OT  - *eco-evolutionary dynamics\n",
      "OT  - *fitness\n",
      "OT  - *gene flow\n",
      "OT  - *genetic rescue\n",
      "OT  - *genomics\n",
      "OT  - *inbreeding\n",
      "OT  - *pedigree\n",
      "COIS- Declaration of Interests The authors declare no competing interests.\n",
      "EDAT- 2020/01/07 06:00\n",
      "MHDA- 2020/12/23 06:00\n",
      "CRDT- 2020/01/07 06:00\n",
      "PHST- 2019/08/02 00:00 [received]\n",
      "PHST- 2019/10/11 00:00 [revised]\n",
      "PHST- 2019/11/20 00:00 [accepted]\n",
      "PHST- 2020/01/07 06:00 [pubmed]\n",
      "PHST- 2020/12/23 06:00 [medline]\n",
      "PHST- 2020/01/07 06:00 [entrez]\n",
      "AID - S0960-9822(19)31532-5 [pii]\n",
      "AID - 10.1016/j.cub.2019.11.062 [doi]\n",
      "PST - ppublish\n",
      "SO  - Curr Biol. 2020 Feb 3;30(3):517-522.e5. doi: 10.1016/j.cub.2019.11.062. Epub 2020Jan 2.\n",
      "\n",
      "PMID- 31891233\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200108\n",
      "LR  - 20210110\n",
      "IS  - 1365-2486 (Electronic)\n",
      "IS  - 1354-1013 (Linking)\n",
      "VI  - 26\n",
      "IP  - 1\n",
      "DP  - 2020 Jan\n",
      "TI  - TRY plant trait database - enhanced coverage and open access.\n",
      "PG  - 119-188\n",
      "LID - 10.1111/gcb.14904 [doi]\n",
      "AB  - Plant traits-the morphological, anatomical, physiological, biochemical andphenological characteristics of plants-determine how plants respond toenvironmental factors, affect other trophic levels, and influence ecosystemproperties and their benefits and detriments to people. Plant trait data thusrepresent the basis for a vast area of research spanning from evolutionarybiology, community and functional ecology, to biodiversity conservation,ecosystem and landscape management, restoration, biogeography and earth systemmodelling. Since its foundation in 2007, the TRY database of plant traits hasgrown continuously. It now provides unprecedented data coverage under an openaccess data policy and is the main plant trait database used by the researchcommunity worldwide. Increasingly, the TRY database also supports new frontiersof trait-based plant research, including the identification of data gaps and the subsequent mobilization or measurement of new data. To support this development, in this article we evaluate the extent of the trait data compiled in TRY andanalyse emerging patterns of data coverage and representativeness. Best speciescoverage is achieved for categorical traits-almost complete coverage for 'plantgrowth form'. However, most traits relevant for ecology and vegetation modelling are characterized by continuous intraspecific variation and trait-environmentalrelationships. These traits have to be measured on individual plants in theirrespective environment. Despite unprecedented data coverage, we observe ahumbling lack of completeness and representativeness of these continuous traitsin many aspects. We, therefore, conclude that reducing data gaps and biases inthe TRY database remains a key challenge and requires a coordinated approach todata mobilization and trait measurements. This can only be achieved incollaboration with other initiatives.\n",
      "CI  - (c) 2019 The Authors. Global Change Biology published by John Wiley & Sons Ltd.\n",
      "FAU - Kattge, Jens\n",
      "AU  - Kattge J\n",
      "AUID- ORCID: 0000-0002-1022-8469\n",
      "AD  - Max Planck Institute for Biogeochemistry, Jena, Germany.\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "FAU - Bonisch, Gerhard\n",
      "AU  - Bonisch G\n",
      "AD  - Max Planck Institute for Biogeochemistry, Jena, Germany.\n",
      "FAU - Diaz, Sandra\n",
      "AU  - Diaz S\n",
      "AUID- ORCID: 0000-0003-0012-4612\n",
      "AD  - Consejo Nacional de Investigaciones Cientificas y Tecnicas, InstitutoMultidisciplinario de Biologia Vegetal (IMBIV), and Factulad de Ciencias Exactas,Fisicas y Naturales, Universidad Nacional de Cordoba, Cordoba, Argentina.\n",
      "FAU - Lavorel, Sandra\n",
      "AU  - Lavorel S\n",
      "AD  - Univ. Grenoble Alpes, CNRS, Univ. Savoie Mont Blanc, LECA, Grenoble, France.\n",
      "FAU - Prentice, Iain Colin\n",
      "AU  - Prentice IC\n",
      "AD  - Imperial College, London, UK.\n",
      "FAU - Leadley, Paul\n",
      "AU  - Leadley P\n",
      "AD  - Ecologie Systematique Evolution, CNRS, AgroParisTech, University of Paris-Sud,Universite Paris-Saclay, Orsay, France.\n",
      "FAU - Tautenhahn, Susanne\n",
      "AU  - Tautenhahn S\n",
      "AUID- ORCID: 0000-0002-2753-3443\n",
      "AD  - Max Planck Institute for Biogeochemistry, Jena, Germany.\n",
      "FAU - Werner, Gijsbert D A\n",
      "AU  - Werner GDA\n",
      "AUID- ORCID: 0000-0002-5426-2562\n",
      "AD  - Department of Zoology, University of Oxford, Oxford, UK.\n",
      "AD  - Balliol College, University of Oxford, Oxford, UK.\n",
      "FAU - Aakala, Tuomas\n",
      "AU  - Aakala T\n",
      "AUID- ORCID: 0000-0003-0160-6410\n",
      "AD  - University of Helsinki, Helsinki, Finland.\n",
      "FAU - Abedi, Mehdi\n",
      "AU  - Abedi M\n",
      "AUID- ORCID: 0000-0002-1499-0119\n",
      "AD  - Department of Range Management, Faculty of Natural Resources and Marine Sciences,Tarbiat Modares University, Noor, Iran.\n",
      "FAU - Acosta, Alicia T R\n",
      "AU  - Acosta ATR\n",
      "AUID- ORCID: 0000-0001-6572-3187\n",
      "AD  - University of Roma Tre, Rome, Italy.\n",
      "FAU - Adamidis, George C\n",
      "AU  - Adamidis GC\n",
      "AUID- ORCID: 0000-0001-8704-6623\n",
      "AD  - Biodiversity Conservation Laboratory, Department of Environment, University ofthe Aegean, Mytilene, Greece.\n",
      "AD  - Institute of Ecology and Evolution, University of Bern, Bern, Switzerland.\n",
      "FAU - Adamson, Kairi\n",
      "AU  - Adamson K\n",
      "AD  - Tartu Observatory, University of Tartu, Tartumaa, Estonia.\n",
      "FAU - Aiba, Masahiro\n",
      "AU  - Aiba M\n",
      "AUID- ORCID: 0000-0002-5966-1562\n",
      "AD  - Graduate School of Life Sciences, Tohoku University, Sendai, Japan.\n",
      "FAU - Albert, Cecile H\n",
      "AU  - Albert CH\n",
      "AUID- ORCID: 0000-0002-0991-1068\n",
      "AD  - Aix Marseille Univ, Univ Avignon, CNRS, IRD, IMBE, Marseille, France.\n",
      "FAU - Alcantara, Julio M\n",
      "AU  - Alcantara JM\n",
      "AUID- ORCID: 0000-0002-8003-7844\n",
      "AD  - Universidad de Jaen, Jaen, Spain.\n",
      "FAU - Alcazar C, Carolina\n",
      "AU  - Alcazar C C\n",
      "AUID- ORCID: 0000-0002-9366-8098\n",
      "AD  - Instituto Alexander Von Humboldt, Bogota, Colombia.\n",
      "FAU - Aleixo, Izabela\n",
      "AU  - Aleixo I\n",
      "AUID- ORCID: 0000-0001-9220-8965\n",
      "AD  - National Institute of Amazonian Research (INPA), Manaus, Brazil.\n",
      "FAU - Ali, Hamada\n",
      "AU  - Ali H\n",
      "AUID- ORCID: 0000-0002-7062-9344\n",
      "AD  - Botany Department, Faculty of Science, Suez Canal University, Ismailia, Egypt.\n",
      "FAU - Amiaud, Bernard\n",
      "AU  - Amiaud B\n",
      "AD  - Universite de Lorraine, Lorraine, France.\n",
      "FAU - Ammer, Christian\n",
      "AU  - Ammer C\n",
      "AUID- ORCID: 0000-0002-4235-0135\n",
      "AD  - Forest Sciences, University of Gottingen, Gottingen, Germany.\n",
      "AD  - Centre for Biodiversity and Sustainable Land-use, University of Gottingen,Gottingen, Germany.\n",
      "FAU - Amoroso, Mariano M\n",
      "AU  - Amoroso MM\n",
      "AD  - Instituto de Investigaciones en Recursos Naturales, Agroecologia y DesarrolloRural (IRNAD), Universidad Nacional de Rio Negro, El Bolson, Argentina.\n",
      "AD  - Conicet-Consejo Nacional de Investigaciones Cientificas y Tecnicas, Buenos Aires,Argentina.\n",
      "FAU - Anand, Madhur\n",
      "AU  - Anand M\n",
      "AD  - University of Guelph, Guelph, ON, Canada.\n",
      "FAU - Anderson, Carolyn\n",
      "AU  - Anderson C\n",
      "AUID- ORCID: 0000-0003-4211-5765\n",
      "AD  - Pacific Northwest National Laboratory, Richland, WA, USA.\n",
      "AD  - University of Massachusetts Amherst, Amherst, MA, USA.\n",
      "FAU - Anten, Niels\n",
      "AU  - Anten N\n",
      "AD  - Centre for Crop Systems Analysis, Wageningen University, Wageningen, TheNetherlands.\n",
      "FAU - Antos, Joseph\n",
      "AU  - Antos J\n",
      "AD  - University of Victoria, Victoria, BC, Canada.\n",
      "FAU - Apgaua, Deborah Mattos Guimaraes\n",
      "AU  - Apgaua DMG\n",
      "AUID- ORCID: 0000-0002-6303-6989\n",
      "AD  - College of Science & Engineering, James Cook University, Smithfield, Qld,Australia.\n",
      "FAU - Ashman, Tia-Lynn\n",
      "AU  - Ashman TL\n",
      "AUID- ORCID: 0000-0002-9884-5954\n",
      "AD  - University of Pittsburgh, Pittsburgh, PA, USA.\n",
      "FAU - Asmara, Degi Harja\n",
      "AU  - Asmara DH\n",
      "AD  - Centre for Forest Research, Institute for Integrative Systems Biology, UniversiteLaval, Quebec, QC, Canada.\n",
      "FAU - Asner, Gregory P\n",
      "AU  - Asner GP\n",
      "AUID- ORCID: 0000-0001-7893-6421\n",
      "AD  - Arizona State University, Tempe, AZ, USA.\n",
      "FAU - Aspinwall, Michael\n",
      "AU  - Aspinwall M\n",
      "AUID- ORCID: 0000-0003-0199-2972\n",
      "AD  - Department of Biology, University of North Florida, Jacksonville, FL, USA.\n",
      "FAU - Atkin, Owen\n",
      "AU  - Atkin O\n",
      "AUID- ORCID: 0000-0003-1041-5202\n",
      "AD  - ARC Centre for Excellence in Plant Energy Biology, Australian NationalUniversity, Acton, ACT, Australia.\n",
      "FAU - Aubin, Isabelle\n",
      "AU  - Aubin I\n",
      "AUID- ORCID: 0000-0002-5953-1012\n",
      "AD  - Great Lakes Forestry Centre, Canadian Forest Service, Natural Resources Canada,Sault Ste. Marie, ON, Canada.\n",
      "FAU - Baastrup-Spohr, Lars\n",
      "AU  - Baastrup-Spohr L\n",
      "AUID- ORCID: 0000-0001-8382-984X\n",
      "AD  - Department of Biology, University of Copenhagen, Copenhagen, Denmark.\n",
      "FAU - Bahalkeh, Khadijeh\n",
      "AU  - Bahalkeh K\n",
      "AUID- ORCID: 0000-0003-1485-0316\n",
      "AD  - Department of Range Management, Faculty of Natural Resources and Marine Sciences,Tarbiat Modares University, Noor, Iran.\n",
      "FAU - Bahn, Michael\n",
      "AU  - Bahn M\n",
      "AUID- ORCID: 0000-0001-7482-9776\n",
      "AD  - Department of Ecology, University of Innsbruck, Innsbruck, Austria.\n",
      "FAU - Baker, Timothy\n",
      "AU  - Baker T\n",
      "AD  - School of Geography, University of Leeds, Leeds, UK.\n",
      "FAU - Baker, William J\n",
      "AU  - Baker WJ\n",
      "AUID- ORCID: 0000-0001-6727-1831\n",
      "AD  - Royal Botanic Gardens Kew, Richmond, UK.\n",
      "FAU - Bakker, Jan P\n",
      "AU  - Bakker JP\n",
      "AUID- ORCID: 0000-0001-7475-5906\n",
      "AD  - Conservation Ecology, Groningen Institute for Evolutionary Life Sciences(GELIFES), University of Groningen, Groningen, The Netherlands.\n",
      "FAU - Baldocchi, Dennis\n",
      "AU  - Baldocchi D\n",
      "AUID- ORCID: 0000-0003-3496-4919\n",
      "AD  - Department of Environmental Science, Policy and Management, University ofCalifornia Berkeley, Berkeley, CA, USA.\n",
      "FAU - Baltzer, Jennifer\n",
      "AU  - Baltzer J\n",
      "AUID- ORCID: 0000-0001-7476-5928\n",
      "AD  - Biology Department, Wilfrid Laurier University, Waterloo, ON, Canada.\n",
      "FAU - Banerjee, Arindam\n",
      "AU  - Banerjee A\n",
      "AD  - Department of Forest Resources, University of Minnesota, St. Paul, MN, USA.\n",
      "FAU - Baranger, Anne\n",
      "AU  - Baranger A\n",
      "AD  - AgroParisTech, Paris, France.\n",
      "FAU - Barlow, Jos\n",
      "AU  - Barlow J\n",
      "AUID- ORCID: 0000-0003-4992-2594\n",
      "AD  - Lancaster Environment Centre, Lancaster University, Lancaster, UK.\n",
      "FAU - Barneche, Diego R\n",
      "AU  - Barneche DR\n",
      "AUID- ORCID: 0000-0002-4568-2362\n",
      "AD  - College of Life and Environmental Sciences, University of Exeter, Penryn, UK.\n",
      "FAU - Baruch, Zdravko\n",
      "AU  - Baruch Z\n",
      "AUID- ORCID: 0000-0002-7264-4812\n",
      "AD  - School of Biological Sciences, The University of Adelaide, Adelaide, SA,Australia.\n",
      "FAU - Bastianelli, Denis\n",
      "AU  - Bastianelli D\n",
      "AUID- ORCID: 0000-0002-6394-5920\n",
      "AD  - CIRAD, UMR SELMET, Montpellier, France.\n",
      "AD  - SELMET, CIRAD, INRA, Univ Montpellier, Montpellier SupAgro, France.\n",
      "FAU - Battles, John\n",
      "AU  - Battles J\n",
      "AUID- ORCID: 0000-0001-7124-7893\n",
      "AD  - University of California at Berkeley, Berkeley, CA, USA.\n",
      "FAU - Bauerle, William\n",
      "AU  - Bauerle W\n",
      "AUID- ORCID: 0000-0003-3090-234X\n",
      "AD  - Department of Horticulture and Landscape Architecture, Colorado State University,Fort Collins, CO, USA.\n",
      "FAU - Bauters, Marijn\n",
      "AU  - Bauters M\n",
      "AUID- ORCID: 0000-0003-0978-6639\n",
      "AD  - Department of Green Chemistry and Technology, Ghent University, Gent, Belgium.\n",
      "AD  - Department of Environment, Ghent University, Gent, Belgium.\n",
      "FAU - Bazzato, Erika\n",
      "AU  - Bazzato E\n",
      "AD  - Department of Life and Environmental Sciences, Botany Division, University ofCagliari, Cagliari, Italy.\n",
      "FAU - Beckmann, Michael\n",
      "AU  - Beckmann M\n",
      "AUID- ORCID: 0000-0002-5678-265X\n",
      "AD  - Helmholtz Centre for Environmental Research - UFZ, Leipzig, Germany.\n",
      "FAU - Beeckman, Hans\n",
      "AU  - Beeckman H\n",
      "AUID- ORCID: 0000-0001-8954-6277\n",
      "AD  - Royal Museum for Central Africa, Tervuren, Belgium.\n",
      "FAU - Beierkuhnlein, Carl\n",
      "AU  - Beierkuhnlein C\n",
      "AUID- ORCID: 0000-0002-6456-4628\n",
      "AD  - University of Bayreuth, Bayreuth, Germany.\n",
      "FAU - Bekker, Renee\n",
      "AU  - Bekker R\n",
      "AD  - Groningen Institute of Archaeology (GIA), University of Groningen, Groningen, TheNetherlands.\n",
      "FAU - Belfry, Gavin\n",
      "AU  - Belfry G\n",
      "AUID- ORCID: 0000-0003-3405-5950\n",
      "AD  - Department of Biological Sciences, University of Tennessee, Knoxville, TN, USA.\n",
      "AD  - Rocky Mountain Biological Laboratory, Crested Butte, CO, USA.\n",
      "FAU - Belluau, Michael\n",
      "AU  - Belluau M\n",
      "AUID- ORCID: 0000-0001-6707-546X\n",
      "AD  - Departement des Science, Universite du Quebec A Montreal, Montreal, QC, Canada.\n",
      "FAU - Beloiu, Mirela\n",
      "AU  - Beloiu M\n",
      "AUID- ORCID: 0000-0002-3592-8170\n",
      "AD  - Department of Biogeography, University of Bayreuth, Bayreuth, Germany.\n",
      "FAU - Benavides, Raquel\n",
      "AU  - Benavides R\n",
      "AUID- ORCID: 0000-0003-2328-5371\n",
      "AD  - Museo Nacional de Ciencias Naturales-CSIC, Madrid, Spain.\n",
      "FAU - Benomar, Lahcen\n",
      "AU  - Benomar L\n",
      "AUID- ORCID: 0000-0001-9301-5655\n",
      "AD  - Universite Laval, Quebec, QC, Canada.\n",
      "FAU - Berdugo-Lattke, Mary Lee\n",
      "AU  - Berdugo-Lattke ML\n",
      "AUID- ORCID: 0000-0002-6662-6458\n",
      "AD  - Instituto de Ciencias Naturales, Universidad Nacional de Colombia, Bogota,Colombia.\n",
      "AD  - Fundacion Natura, Bogota, Colombia.\n",
      "FAU - Berenguer, Erika\n",
      "AU  - Berenguer E\n",
      "AUID- ORCID: 0000-0001-7357-8805\n",
      "AD  - Environmental Change Institute, University of Oxford, Oxford, UK.\n",
      "FAU - Bergamin, Rodrigo\n",
      "AU  - Bergamin R\n",
      "AUID- ORCID: 0000-0002-2405-9977\n",
      "AD  - Laboratorio de Estudos em Vegetacao Campestre (LEVCamp), Programa dePos-Graduacao em Botanica, Universidade Federal do Rio Grande do Sul, PortoAlegre, Brazil.\n",
      "FAU - Bergmann, Joana\n",
      "AU  - Bergmann J\n",
      "AUID- ORCID: 0000-0002-2008-4198\n",
      "AD  - Institut fur Biologie, Freie Universitat Berlin, Berlin, Germany.\n",
      "AD  - Berlin-Brandenburg Institute of Advanced Biodiversity Research (BBIB), Berlin,Germany.\n",
      "FAU - Bergmann Carlucci, Marcos\n",
      "AU  - Bergmann Carlucci M\n",
      "AUID- ORCID: 0000-0002-5868-7090\n",
      "AD  - Laboratorio de Ecologia Funcional de Comunidades (LABEF), Departamento deBotanica, Universidade Federal do Parana, Curitiba, Brazil.\n",
      "FAU - Berner, Logan\n",
      "AU  - Berner L\n",
      "AUID- ORCID: 0000-0001-8947-0479\n",
      "AD  - School of Informatics, Computing, and Cyber Systems, Northern Arizona University,Flagstaff, AZ, USA.\n",
      "FAU - Bernhardt-Romermann, Markus\n",
      "AU  - Bernhardt-Romermann M\n",
      "AUID- ORCID: 0000-0002-2740-2304\n",
      "AD  - Institute of Ecology and Evolution, Friedrich Schiller University Jena, Jena,Germany.\n",
      "FAU - Bigler, Christof\n",
      "AU  - Bigler C\n",
      "AUID- ORCID: 0000-0003-3757-6356\n",
      "AD  - ETH Zurich, Zurich, Switzerland.\n",
      "FAU - Bjorkman, Anne D\n",
      "AU  - Bjorkman AD\n",
      "AUID- ORCID: 0000-0003-2174-7800\n",
      "AD  - Department of Biological and Environmental Sciences, University of Gothenburg,Gothenburg, Sweden.\n",
      "FAU - Blackman, Chris\n",
      "AU  - Blackman C\n",
      "AD  - PIAF, INRA, Universite Clermont-Auvergne, Clermont-Ferrand, France.\n",
      "FAU - Blanco, Carolina\n",
      "AU  - Blanco C\n",
      "AUID- ORCID: 0000-0002-8959-2633\n",
      "AD  - Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil.\n",
      "FAU - Blonder, Benjamin\n",
      "AU  - Blonder B\n",
      "AUID- ORCID: 0000-0002-5061-2385\n",
      "AD  - Rocky Mountain Biological Laboratory, Crested Butte, CO, USA.\n",
      "AD  - School of Life Sciences, Arizona State University, Tempe, AZ, USA.\n",
      "FAU - Blumenthal, Dana\n",
      "AU  - Blumenthal D\n",
      "AUID- ORCID: 0000-0001-7496-0766\n",
      "AD  - USDA-ARS Rangeland Resources & Systems Research Unit, Fort Collins, CO, USA.\n",
      "FAU - Bocanegra-Gonzalez, Kelly T\n",
      "AU  - Bocanegra-Gonzalez KT\n",
      "AUID- ORCID: 0000-0001-7177-5856\n",
      "AD  - Grupo de Investigacion en Biodiversidad y Dinamica de Ecosistemas Tropicales -Universidad del Tolima, Ibague, Colombia.\n",
      "FAU - Boeckx, Pascal\n",
      "AU  - Boeckx P\n",
      "AUID- ORCID: 0000-0003-3998-0010\n",
      "AD  - Isotope Bioscience Laboratory - ISOFYS, Ghent University, Gent, Belgium.\n",
      "FAU - Bohlman, Stephanie\n",
      "AU  - Bohlman S\n",
      "AD  - School of Forest Resources and Conservation, University of Florida, Gainesville, FL, USA.\n",
      "FAU - Bohning-Gaese, Katrin\n",
      "AU  - Bohning-Gaese K\n",
      "AUID- ORCID: 0000-0003-0477-5586\n",
      "AD  - Senckenberg Biodiversity and Climate Research Centre, Frankfurt am Main, Germany.\n",
      "AD  - Department of Biological Sciences, Goethe Universitat Frankfurt, Frankfurt amMain, Germany.\n",
      "FAU - Boisvert-Marsh, Laura\n",
      "AU  - Boisvert-Marsh L\n",
      "AUID- ORCID: 0000-0002-0939-8196\n",
      "AD  - Great Lakes Forestry Centre, Canadian Forest Service, Natural Resources Canada,Sault Ste. Marie, ON, Canada.\n",
      "FAU - Bond, William\n",
      "AU  - Bond W\n",
      "AUID- ORCID: 0000-0002-3441-2084\n",
      "AD  - Department of Biological Sciences, University of Cape Town, Cape Town, SouthAfrica.\n",
      "AD  - SAEON Fynbos Node, Claremont, South Africa.\n",
      "FAU - Bond-Lamberty, Ben\n",
      "AU  - Bond-Lamberty B\n",
      "AUID- ORCID: 0000-0001-9525-4633\n",
      "AD  - Pacific Northwest National Laboratory, College Park, MD, USA.\n",
      "FAU - Boom, Arnoud\n",
      "AU  - Boom A\n",
      "AUID- ORCID: 0000-0003-1299-691X\n",
      "AD  - School of Geography, Geology and Environment, University of Leicester, Leicester,UK.\n",
      "FAU - Boonman, Coline C F\n",
      "AU  - Boonman CCF\n",
      "AUID- ORCID: 0000-0003-2417-1579\n",
      "AD  - Department of Environmental Science, Institute for Water and Wetland Research,Radboud University, Nijmegen, The Netherlands.\n",
      "FAU - Bordin, Kauane\n",
      "AU  - Bordin K\n",
      "AUID- ORCID: 0000-0003-3871-6293\n",
      "AD  - Laboratorio de Ecologia Vegetal (LEVEG), Programa de Pos-Graduacao em Ecologia,Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil.\n",
      "FAU - Boughton, Elizabeth H\n",
      "AU  - Boughton EH\n",
      "AUID- ORCID: 0000-0003-0932-280X\n",
      "AD  - Archbold Biological Station's Buck Island Ranch, FL, Lake Placid, USA.\n",
      "FAU - Boukili, Vanessa\n",
      "AU  - Boukili V\n",
      "AUID- ORCID: 0000-0002-5950-2123\n",
      "AD  - Department of Ecology and Evolutionary Biology, University of Connecticut,Storrs, CT, USA.\n",
      "FAU - Bowman, David M J S\n",
      "AU  - Bowman DMJS\n",
      "AD  - University of Tasmania, Hobart, Tas., Australia.\n",
      "FAU - Bravo, Sandra\n",
      "AU  - Bravo S\n",
      "AUID- ORCID: 0000-0001-6098-7255\n",
      "AD  - Facultad de Ciencias Forestales, Universidad Nacional de Santiago del Estero,Santiago del Estero, Argentina.\n",
      "FAU - Brendel, Marco Richard\n",
      "AU  - Brendel MR\n",
      "AUID- ORCID: 0000-0003-4356-4597\n",
      "AD  - Institute of Landscape and Plant Ecology, University of Hohenheim, Stuttgart,Germany.\n",
      "FAU - Broadley, Martin R\n",
      "AU  - Broadley MR\n",
      "AUID- ORCID: 0000-0003-3964-7226\n",
      "AD  - School of Geography, University of Nottingham, Nottingham, UK.\n",
      "FAU - Brown, Kerry A\n",
      "AU  - Brown KA\n",
      "AUID- ORCID: 0000-0002-6342-8966\n",
      "AD  - Department of Geography and Geology, Kingston University, Kingston upon Thames,UK.\n",
      "FAU - Bruelheide, Helge\n",
      "AU  - Bruelheide H\n",
      "AUID- ORCID: 0000-0003-3135-0356\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "AD  - Institute of Biology/Geobotany and Botanical Garden, Martin Luther UniversityHalle-Wittenberg, Halle, Germany.\n",
      "FAU - Brumnich, Federico\n",
      "AU  - Brumnich F\n",
      "AUID- ORCID: 0000-0002-0502-6694\n",
      "AD  - Conicet-Consejo Nacional de Investigaciones Cientificas y Tecnicas, Buenos Aires,Argentina.\n",
      "AD  - Facultad de Ingenieria y Ciencias Hidricas, Universidad Nacional del Litoral(FICH-UNL), Santa Fe, Argentina.\n",
      "FAU - Bruun, Hans Henrik\n",
      "AU  - Bruun HH\n",
      "AUID- ORCID: 0000-0003-0674-2577\n",
      "AD  - Department of Biology, University of Copenhagen, Copenhagen, Denmark.\n",
      "FAU - Bruy, David\n",
      "AU  - Bruy D\n",
      "AUID- ORCID: 0000-0002-7339-0375\n",
      "AD  - AMAP, CIRAD, CNRS, IRD, INRA, Universite de Montpellier, Montpellier, France.\n",
      "AD  - AMAP, IRD, Herbier de Nouvelle-Caledonie, Noumea, New Caledonia.\n",
      "FAU - Buchanan, Serra W\n",
      "AU  - Buchanan SW\n",
      "AD  - University of Toronto Scarborough, Scarborough, ON, Canada.\n",
      "FAU - Bucher, Solveig Franziska\n",
      "AU  - Bucher SF\n",
      "AUID- ORCID: 0000-0002-2303-4583\n",
      "AD  - Friedrich-Schiller-Universitat Jena, Jena, Germany.\n",
      "FAU - Buchmann, Nina\n",
      "AU  - Buchmann N\n",
      "AUID- ORCID: 0000-0003-0826-2980\n",
      "AD  - ETH Zurich, Zurich, Switzerland.\n",
      "FAU - Buitenwerf, Robert\n",
      "AU  - Buitenwerf R\n",
      "AUID- ORCID: 0000-0003-3356-2301\n",
      "AD  - Section for Ecoinformatics and Biodiversity, Department of Bioscience, AarhusUniversity, Aarhus, Denmark.\n",
      "AD  - Center for Biodiversity Dynamics in a Changing World (BIOCHANGE), Department ofBioscience, Aarhus University, Aarhus, Denmark.\n",
      "FAU - Bunker, Daniel E\n",
      "AU  - Bunker DE\n",
      "AUID- ORCID: 0000-0002-7234-3403\n",
      "AD  - New Jersey Institute of Technology, Newark, NJ, USA.\n",
      "FAU - Burger, Jana\n",
      "AU  - Burger J\n",
      "AD  - Faculty of Agricultural and Environmental Sciences, University of Rostock,Rostock, Germany.\n",
      "FAU - Burrascano, Sabina\n",
      "AU  - Burrascano S\n",
      "AUID- ORCID: 0000-0002-6537-3313\n",
      "AD  - Sapienza University of Rome, Rome, Italy.\n",
      "FAU - Burslem, David F R P\n",
      "AU  - Burslem DFRP\n",
      "AUID- ORCID: 0000-0001-6033-0990\n",
      "AD  - School of Biological Sciences, University of Aberdeen, Aberdeen, UK.\n",
      "FAU - Butterfield, Bradley J\n",
      "AU  - Butterfield BJ\n",
      "AUID- ORCID: 0000-0003-0974-9811\n",
      "AD  - Center for Ecosystem Science and Society, Northern Arizona University, Flagstaff,AZ, USA.\n",
      "FAU - Byun, Chaeho\n",
      "AU  - Byun C\n",
      "AUID- ORCID: 0000-0003-3209-3275\n",
      "AD  - School of Civil and Environmental Engineering, Yonsei University, Seoul, Korea.\n",
      "FAU - Marques, Marcia\n",
      "AU  - Marques M\n",
      "AUID- ORCID: 0000-0002-1003-9596\n",
      "AD  - Departamento de Botanica, SCB, UFPR - Federal University of Parana, Curitiba,Brazil.\n",
      "FAU - Scalon, Marina C\n",
      "AU  - Scalon MC\n",
      "AUID- ORCID: 0000-0003-2069-8226\n",
      "AD  - Centro Politecnico, Universidade Federal do Parana, Curitiba, Brazil.\n",
      "FAU - Caccianiga, Marco\n",
      "AU  - Caccianiga M\n",
      "AUID- ORCID: 0000-0001-9715-1830\n",
      "AD  - Dipartimento di Bioscienze, Universita degli Studi di Milano, Milano, Italy.\n",
      "FAU - Cadotte, Marc\n",
      "AU  - Cadotte M\n",
      "AUID- ORCID: 0000-0002-5816-7693\n",
      "AD  - University of Toronto Scarborough, Scarborough, ON, Canada.\n",
      "FAU - Cailleret, Maxime\n",
      "AU  - Cailleret M\n",
      "AUID- ORCID: 0000-0001-6561-1943\n",
      "AD  - IRSTEA Aix-en-Provence, UMR RECOVER, Aix-Marseille University, Aix-en-Provence,France.\n",
      "AD  - Department of Environmental Systems Science, ETH Zurich, Zurich, Switzerland.\n",
      "AD  - Swiss Federal Institute for Forest, Snow and Landscape Research WSL, Birmensdorf,Switzerland.\n",
      "FAU - Camac, James\n",
      "AU  - Camac J\n",
      "AUID- ORCID: 0000-0003-4785-0742\n",
      "AD  - Centre of Excellence for Bioscurity Risk Analysis, The University of Melbourne,Melbourne, Vic., Australia.\n",
      "FAU - Camarero, Jesus Julio\n",
      "AU  - Camarero JJ\n",
      "AUID- ORCID: 0000-0003-2436-2922\n",
      "AD  - Instituto Pirenaico de Ecologia (IPE-CSIC), Zaragoza, Spain.\n",
      "FAU - Campany, Courtney\n",
      "AU  - Campany C\n",
      "AD  - Colgate University, Hamilton, NY, USA.\n",
      "FAU - Campetella, Giandiego\n",
      "AU  - Campetella G\n",
      "AUID- ORCID: 0000-0001-6126-522X\n",
      "AD  - School of Biosciences and Veterinary Medicine, Plant Diversity and EcosystemsManagement Unit, University of Camerino, Camerino, Italy.\n",
      "FAU - Campos, Juan Antonio\n",
      "AU  - Campos JA\n",
      "AUID- ORCID: 0000-0001-5992-2753\n",
      "AD  - Department of Plant Biology and Ecology, University of the Basque CountryUPV/EHU, Bilbao, Spain.\n",
      "FAU - Cano-Arboleda, Laura\n",
      "AU  - Cano-Arboleda L\n",
      "AD  - Fundacion Natura, Bogota, Colombia.\n",
      "AD  - Departamento de Geociencias y Medio Ambiente, Universidad Nacional de Colombia,Medellin, Colombia.\n",
      "FAU - Canullo, Roberto\n",
      "AU  - Canullo R\n",
      "AUID- ORCID: 0000-0002-9913-6981\n",
      "AD  - School of Biosciences and Veterinary Medicine, Plant Diversity and EcosystemsManagement Unit, University of Camerino, Camerino, Italy.\n",
      "FAU - Carbognani, Michele\n",
      "AU  - Carbognani M\n",
      "AUID- ORCID: 0000-0001-7701-9859\n",
      "AD  - Department of Chemistry, Life Sciences and Environmental Sustainability,University of Parma, Parma, Italy.\n",
      "FAU - Carvalho, Fabio\n",
      "AU  - Carvalho F\n",
      "AUID- ORCID: 0000-0002-6305-5602\n",
      "AD  - Lancaster Environment Centre, Lancaster University, Lancaster, UK.\n",
      "FAU - Casanoves, Fernando\n",
      "AU  - Casanoves F\n",
      "AUID- ORCID: 0000-0001-8765-9382\n",
      "AD  - CATIE-Centro Agronomico Tropical de Investigacion y Ensenanza, Turrialba, CostaRica.\n",
      "FAU - Castagneyrol, Bastien\n",
      "AU  - Castagneyrol B\n",
      "AUID- ORCID: 0000-0001-8795-7806\n",
      "AD  - Univ. Bordeaux, INRAE, BIOGECO, Cestas, France.\n",
      "FAU - Catford, Jane A\n",
      "AU  - Catford JA\n",
      "AUID- ORCID: 0000-0003-0582-5960\n",
      "AD  - Department of Geography, King's College London, London, UK.\n",
      "FAU - Cavender-Bares, Jeannine\n",
      "AU  - Cavender-Bares J\n",
      "AUID- ORCID: 0000-0003-3375-9630\n",
      "AD  - Department of Ecology, Evolution, and Behavior, University of Minnesota, St.Paul, MN, USA.\n",
      "FAU - Cerabolini, Bruno E L\n",
      "AU  - Cerabolini BEL\n",
      "AUID- ORCID: 0000-0002-3793-0733\n",
      "AD  - Department of Biotechnology and Life Sciences, University of Insubria, Varese,Italy.\n",
      "FAU - Cervellini, Marco\n",
      "AU  - Cervellini M\n",
      "AUID- ORCID: 0000-0002-0853-2330\n",
      "AD  - School of Biosciences and Veterinary Medicine, Plant Diversity and EcosystemsManagement Unit, University of Camerino, Camerino, Italy.\n",
      "AD  - BIGEA, Department of Biological, Geological and Environmental Sciences, AlmaMater Studiorum - University of Bologna, Bologna, Italy.\n",
      "FAU - Chacon-Madrigal, Eduardo\n",
      "AU  - Chacon-Madrigal E\n",
      "AUID- ORCID: 0000-0002-8328-5456\n",
      "AD  - Escuela de Biologia, Universidad de Costa Rica, San Jose, Costa Rica.\n",
      "FAU - Chapin, Kenneth\n",
      "AU  - Chapin K\n",
      "AUID- ORCID: 0000-0002-8382-4050\n",
      "AD  - The University of Arizona, Tucson, AZ, USA.\n",
      "FAU - Chapin, F Stuart\n",
      "AU  - Chapin FS\n",
      "AUID- ORCID: 0000-0002-2558-9910\n",
      "AD  - Institute of Arctic Biology, University of Alaska Fairbanks, Fairbanks, AK, USA.\n",
      "FAU - Chelli, Stefano\n",
      "AU  - Chelli S\n",
      "AUID- ORCID: 0000-0001-7184-8242\n",
      "AD  - School of Biosciences and Veterinary Medicine, Plant Diversity and EcosystemsManagement Unit, University of Camerino, Camerino, Italy.\n",
      "FAU - Chen, Si-Chong\n",
      "AU  - Chen SC\n",
      "AUID- ORCID: 0000-0002-6855-2595\n",
      "AD  - Royal Botanic Gardens, Kew, West Sussex, UK.\n",
      "FAU - Chen, Anping\n",
      "AU  - Chen A\n",
      "AUID- ORCID: 0000-0003-2085-3863\n",
      "AD  - Department of Biology, Colorado State University, Fort Collins, CO, USA.\n",
      "FAU - Cherubini, Paolo\n",
      "AU  - Cherubini P\n",
      "AUID- ORCID: 0000-0002-9809-250X\n",
      "AD  - WSL Swiss Federal Research Institute, Birmensdorf, Switzerland.\n",
      "AD  - Faculty of Forestry, University of British Columbia, Vancouver, BC, Canada.\n",
      "FAU - Chianucci, Francesco\n",
      "AU  - Chianucci F\n",
      "AUID- ORCID: 0000-0002-5688-2060\n",
      "AD  - CREA - Research Centre for Forestry and Wood, Arezzo, Italy.\n",
      "FAU - Choat, Brendan\n",
      "AU  - Choat B\n",
      "AUID- ORCID: 0000-0002-9105-640X\n",
      "AD  - Hawkesbury Institute for the Environment, Western Sydney University, Sydney, NSW,Australia.\n",
      "FAU - Chung, Kyong-Sook\n",
      "AU  - Chung KS\n",
      "AUID- ORCID: 0000-0002-4464-4698\n",
      "AD  - Jungwon University, Goesan, Chungbuk, Korea.\n",
      "FAU - Chytry, Milan\n",
      "AU  - Chytry M\n",
      "AUID- ORCID: 0000-0002-8122-3075\n",
      "AD  - Department of Botany and Zoology, Masaryk University, Brno, Czech Republic.\n",
      "FAU - Ciccarelli, Daniela\n",
      "AU  - Ciccarelli D\n",
      "AUID- ORCID: 0000-0001-9715-9779\n",
      "AD  - Department of Biology, University of Pisa, Pisa, Italy.\n",
      "FAU - Coll, Lluis\n",
      "AU  - Coll L\n",
      "AD  - Department of Agriculture and Forest Engineering (EAGROF), University of Lleida, Lleida, Spain.\n",
      "AD  - Joint Research Unit CTFC - AGROTECNIO, Solsona, Spain.\n",
      "FAU - Collins, Courtney G\n",
      "AU  - Collins CG\n",
      "AD  - University of California Riverside, Riverside, CA, USA.\n",
      "FAU - Conti, Luisa\n",
      "AU  - Conti L\n",
      "AUID- ORCID: 0000-0001-8047-1467\n",
      "AD  - Faculty of Environmental Sciences, University of Life Sciences Prague,Praha-Suchdol, Czech Republic.\n",
      "AD  - Institute of Botany, Czech Academy of Sciences, Trebon, Czech Republic.\n",
      "FAU - Coomes, David\n",
      "AU  - Coomes D\n",
      "AUID- ORCID: 0000-0002-8261-2582\n",
      "AD  - Department of Plant Sciences, University of Cambridge, Cambridge, UK.\n",
      "FAU - Cornelissen, Johannes H C\n",
      "AU  - Cornelissen JHC\n",
      "AUID- ORCID: 0000-0002-2346-1585\n",
      "AD  - Systems Ecology, Department of Ecological Science, Vrije Universiteit, Amsterdam,The Netherlands.\n",
      "FAU - Cornwell, William K\n",
      "AU  - Cornwell WK\n",
      "AUID- ORCID: 0000-0003-4080-4073\n",
      "AD  - School of Biological, Earth and Environmental Sciences, UNSW Sydney, Sydney, NSW,Australia.\n",
      "FAU - Corona, Piermaria\n",
      "AU  - Corona P\n",
      "AUID- ORCID: 0000-0002-8105-0792\n",
      "AD  - CREA - Research Centre for Forestry and Wood, Arezzo, Italy.\n",
      "FAU - Coyea, Marie\n",
      "AU  - Coyea M\n",
      "AD  - Faculte de foresterie, de geographie et de geomatique, Universite Laval, Quebec, QC, Canada.\n",
      "FAU - Craine, Joseph\n",
      "AU  - Craine J\n",
      "AUID- ORCID: 0000-0001-6561-3244\n",
      "AD  - Jonah Ventures, Boulder, CO, USA.\n",
      "FAU - Craven, Dylan\n",
      "AU  - Craven D\n",
      "AUID- ORCID: 0000-0003-3940-833X\n",
      "AD  - Centro de Modelacion y Monitoreo de Ecosistemas, Universidad Mayor, Santiago,Chile.\n",
      "FAU - Cromsigt, Joris P G M\n",
      "AU  - Cromsigt JPGM\n",
      "AD  - Department of Wildlife, Fish and Environmental Studies, Swedish University ofAgricultural Sciences, Umea, Sweden.\n",
      "AD  - Centre for African Conservation Ecology, Department of Zoology, Nelson MandelaUniversity, Port Elizabeth, South Africa.\n",
      "FAU - Csecserits, Aniko\n",
      "AU  - Csecserits A\n",
      "AUID- ORCID: 0000-0002-0538-4520\n",
      "AD  - MTA Centre for Ecological Research, Tihany, Hungary.\n",
      "FAU - Cufar, Katarina\n",
      "AU  - Cufar K\n",
      "AUID- ORCID: 0000-0002-7403-3994\n",
      "AD  - Biotechnical Faculty, University of Ljubljana, Ljubljana, Slovenia.\n",
      "FAU - Cuntz, Matthias\n",
      "AU  - Cuntz M\n",
      "AUID- ORCID: 0000-0002-5966-1829\n",
      "AD  - Universite de Lorraine, AgroParisTech, INRAE, UMR Silva, Nancy, France.\n",
      "FAU - da Silva, Ana Carolina\n",
      "AU  - da Silva AC\n",
      "AUID- ORCID: 0000-0002-1285-640X\n",
      "AD  - Santa Catarina State University, Lages, SC, Brazil.\n",
      "FAU - Dahlin, Kyla M\n",
      "AU  - Dahlin KM\n",
      "AUID- ORCID: 0000-0002-6016-2605\n",
      "AD  - Department of Geography, Environment, and Spatial Sciences, Michigan StateUniversity, East Lansing, MI, USA.\n",
      "FAU - Dainese, Matteo\n",
      "AU  - Dainese M\n",
      "AUID- ORCID: 0000-0001-7052-5572\n",
      "AD  - Eurac Research, Institute for Alpine Environment, Bozen-Bolzano, Italy.\n",
      "FAU - Dalke, Igor\n",
      "AU  - Dalke I\n",
      "AUID- ORCID: 0000-0001-5711-9916\n",
      "AD  - Institute of Biology of Komi Science Centre of the Ural Branch of the RussianAcademy of Sciences, Syktyvkar, Komi Republic, Russia.\n",
      "FAU - Dalle Fratte, Michele\n",
      "AU  - Dalle Fratte M\n",
      "AUID- ORCID: 0000-0002-7907-1586\n",
      "AD  - Department of Biotechnology and Life Sciences, University of Insubria, Varese,Italy.\n",
      "FAU - Dang-Le, Anh Tuan\n",
      "AU  - Dang-Le AT\n",
      "AUID- ORCID: 0000-0002-9794-0669\n",
      "AD  - University of Science - Vietnam National University Ho Chi Minh City, Ho Chi MinhCity, Vietnam.\n",
      "FAU - Danihelka, Jiri\n",
      "AU  - Danihelka J\n",
      "AUID- ORCID: 0000-0002-2640-7867\n",
      "AD  - Department of Botany and Zoology, Masaryk University, Brno, Czech Republic.\n",
      "AD  - Institute of Botany, Czech Academy of Sciences, Trebon, Czech Republic.\n",
      "FAU - Dannoura, Masako\n",
      "AU  - Dannoura M\n",
      "AUID- ORCID: 0000-0003-0389-871X\n",
      "AD  - Graduate School of Agriculture, Kyoto University, Kyoto, Japan.\n",
      "AD  - Graduate School of Global Environmental Studies, Kyoto University, Kyoto, Japan.\n",
      "FAU - Dawson, Samantha\n",
      "AU  - Dawson S\n",
      "AUID- ORCID: 0000-0001-5165-9591\n",
      "AD  - Swedish Species Information Centre, Swedish University of Agricultural Sciences, Uppsala, Sweden.\n",
      "FAU - de Beer, Arend Jacobus\n",
      "AU  - de Beer AJ\n",
      "AD  - Department of Plant and Soil Sciences, University of Pretoria, Pretoria, SouthAfrica.\n",
      "FAU - De Frutos, Angel\n",
      "AU  - De Frutos A\n",
      "AUID- ORCID: 0000-0002-5250-9488\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "AD  - Helmholtz Centre for Environmental Research - UFZ, Leipzig, Germany.\n",
      "FAU - De Long, Jonathan R\n",
      "AU  - De Long JR\n",
      "AUID- ORCID: 0000-0002-7951-4818\n",
      "AD  - Department of Terrestrial Ecology, Netherlands Institute of Ecology, Wageningen, The Netherlands.\n",
      "FAU - Dechant, Benjamin\n",
      "AU  - Dechant B\n",
      "AUID- ORCID: 0000-0001-5171-2364\n",
      "AD  - Department Computational Landscape Ecology, UFZ - Helmholtz Centre forEnvironmental Research, Leipzig, Germany.\n",
      "AD  - Department Computational Hydrosystems, UFZ - Helmholtz Centre for EnvironmentalResearch, Leipzig, Germany.\n",
      "AD  - Department of Landscape Architecture and Rural Systems Engineering, SeoulNational University, Seoul, Republic of Korea.\n",
      "FAU - Delagrange, Sylvain\n",
      "AU  - Delagrange S\n",
      "AD  - Institute of Temperate Forest Sciences (ISFORT), Ripon, QC, Canada.\n",
      "AD  - UQO, Department of Natural Sciences, Ripon, QC, Canada.\n",
      "FAU - Delpierre, Nicolas\n",
      "AU  - Delpierre N\n",
      "AUID- ORCID: 0000-0003-0906-9402\n",
      "AD  - Ecologie Systematique Evolution, CNRS, AgroParisTech, University of Paris-Sud,Universite Paris-Saclay, Orsay, France.\n",
      "FAU - Derroire, Geraldine\n",
      "AU  - Derroire G\n",
      "AUID- ORCID: 0000-0001-7239-2881\n",
      "AD  - Cirad, UMR EcoFoG (Agroparistech, CNRS, INRA, Universite des Antilles, Universitede la Guyane), Kourou, French Guiana, France.\n",
      "FAU - Dias, Arildo S\n",
      "AU  - Dias AS\n",
      "AUID- ORCID: 0000-0002-5495-3435\n",
      "AD  - Institut fur Physische Geographie, Biogeography and Biodiversity Lab,Goethe-Universitat Frankfurt, Frankfurt am Main, Germany.\n",
      "FAU - Diaz-Toribio, Milton Hugo\n",
      "AU  - Diaz-Toribio MH\n",
      "AUID- ORCID: 0000-0003-1675-2699\n",
      "AD  - Department of Biology, University of Florida, Gainesville, FL, USA.\n",
      "FAU - Dimitrakopoulos, Panayiotis G\n",
      "AU  - Dimitrakopoulos PG\n",
      "AUID- ORCID: 0000-0002-8374-4392\n",
      "AD  - Biodiversity Conservation Laboratory, Department of Environment, University ofthe Aegean, Mytilene, Greece.\n",
      "FAU - Dobrowolski, Mark\n",
      "AU  - Dobrowolski M\n",
      "AUID- ORCID: 0000-0001-5586-4023\n",
      "AD  - Iluka Resources, Perth, WA, Australia.\n",
      "AD  - School of Biological Sciences, The University of Western Australia, Perth, WA,Australia.\n",
      "FAU - Doktor, Daniel\n",
      "AU  - Doktor D\n",
      "AD  - Helmholtz Centre for Environmental Research - UFZ, Leipzig, Germany.\n",
      "FAU - Drevojan, Pavel\n",
      "AU  - Drevojan P\n",
      "AUID- ORCID: 0000-0003-0802-3509\n",
      "AD  - Department of Botany and Zoology, Masaryk University, Brno, Czech Republic.\n",
      "FAU - Dong, Ning\n",
      "AU  - Dong N\n",
      "AUID- ORCID: 0000-0003-0793-8854\n",
      "AD  - Department of Biological Sciences, Macquarie University, Sydney, NSW, Australia.\n",
      "FAU - Dransfield, John\n",
      "AU  - Dransfield J\n",
      "AD  - Royal Botanic Gardens Kew, Richmond, UK.\n",
      "FAU - Dressler, Stefan\n",
      "AU  - Dressler S\n",
      "AD  - Department of Botany and Molecular Evolution, Senckenberg Research Institute and Natural History Museum, Frankfurt am Main, Germany.\n",
      "FAU - Duarte, Leandro\n",
      "AU  - Duarte L\n",
      "AUID- ORCID: 0000-0003-1771-0407\n",
      "AD  - Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil.\n",
      "FAU - Ducouret, Emilie\n",
      "AU  - Ducouret E\n",
      "AD  - Cirad, UMR EcoFoG (Agroparistech, CNRS, INRA, Universite des Antilles, Universitede la Guyane), Kourou, French Guiana, France.\n",
      "FAU - Dullinger, Stefan\n",
      "AU  - Dullinger S\n",
      "AUID- ORCID: 0000-0003-3919-0887\n",
      "AD  - Department of Botany and Biodiversity Research, University of Vienna, Vienna,Austria.\n",
      "FAU - Durka, Walter\n",
      "AU  - Durka W\n",
      "AUID- ORCID: 0000-0002-6611-2246\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "AD  - Helmholtz Centre for Environmental Research - UFZ, Halle, Germany.\n",
      "FAU - Duursma, Remko\n",
      "AU  - Duursma R\n",
      "AD  - Hawkesbury Institute for the Environment, Western Sydney University, Sydney, NSW,Australia.\n",
      "FAU - Dymova, Olga\n",
      "AU  - Dymova O\n",
      "AUID- ORCID: 0000-0003-2008-6350\n",
      "AD  - Institute of Biology of Komi Science Centre of the Ural Branch of the RussianAcademy of Sciences, Syktyvkar, Komi Republic, Russia.\n",
      "FAU - E-Vojtko, Anna\n",
      "AU  - E-Vojtko A\n",
      "AUID- ORCID: 0000-0001-6370-680X\n",
      "AD  - Institute of Botany, Czech Academy of Sciences, Trebon, Czech Republic.\n",
      "AD  - Department of Botany, Faculty of Sciences, University of South Bohemia, CeskeBudejovice, Czech Republic.\n",
      "FAU - Eckstein, Rolf Lutz\n",
      "AU  - Eckstein RL\n",
      "AUID- ORCID: 0000-0002-6953-3855\n",
      "AD  - Department of Environmental and Life Sciences - Biology, Karlstad University,Karlstad, Sweden.\n",
      "FAU - Ejtehadi, Hamid\n",
      "AU  - Ejtehadi H\n",
      "AUID- ORCID: 0000-0002-6128-2481\n",
      "AD  - Quantitative Plant Ecology and Biodiversity Research Laboratory, Department ofBiology, Faculty of Science, Ferdowsi University of Mashhad, Mashhad, Iran.\n",
      "FAU - Elser, James\n",
      "AU  - Elser J\n",
      "AUID- ORCID: 0000-0002-1460-2155\n",
      "AD  - Flathead Lake Biological Station, University of Montana, Polson, MT, USA.\n",
      "AD  - School of Sustainability, Arizona State University, Tempe, AZ, USA.\n",
      "FAU - Emilio, Thaise\n",
      "AU  - Emilio T\n",
      "AUID- ORCID: 0000-0001-5415-1822\n",
      "AD  - Programa Nacional de Pos-Doutorado (PNPD), Programa de Pos Graduacao em Ecologia,Institute of Biology, University of Campinas UNICAMP, Brazil.\n",
      "FAU - Engemann, Kristine\n",
      "AU  - Engemann K\n",
      "AUID- ORCID: 0000-0003-1431-1726\n",
      "AD  - Section for Ecoinformatics and Biodiversity, Department of Bioscience, AarhusUniversity, Aarhus, Denmark.\n",
      "FAU - Erfanian, Mohammad Bagher\n",
      "AU  - Erfanian MB\n",
      "AUID- ORCID: 0000-0002-8671-9037\n",
      "AD  - Quantitative Plant Ecology and Biodiversity Research Laboratory, Department ofBiology, Faculty of Science, Ferdowsi University of Mashhad, Mashhad, Iran.\n",
      "FAU - Erfmeier, Alexandra\n",
      "AU  - Erfmeier A\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "AD  - Institute for Ecosystem Research/Geobotany, Kiel University, Kiel, Germany.\n",
      "FAU - Esquivel-Muelbert, Adriane\n",
      "AU  - Esquivel-Muelbert A\n",
      "AUID- ORCID: 0000-0001-5335-1259\n",
      "AD  - School of Geography, University of Leeds, Leeds, UK.\n",
      "AD  - School of Geography, Earth and Environmental Sciences - University of Birmingham,Birmingham, UK.\n",
      "FAU - Esser, Gerd\n",
      "AU  - Esser G\n",
      "AD  - Institute for Plant Ecology, Justus Liebig University, Giessen, Germany.\n",
      "FAU - Estiarte, Marc\n",
      "AU  - Estiarte M\n",
      "AUID- ORCID: 0000-0003-1176-8480\n",
      "AD  - Spanish National Research Council - CSIC, Catalonia, Spain.\n",
      "AD  - CREAF, Catalonia, Spain.\n",
      "FAU - Domingues, Tomas F\n",
      "AU  - Domingues TF\n",
      "AUID- ORCID: 0000-0003-2857-9838\n",
      "AD  - Department of Biology - FFCLRP/USP, Ribeirao Preto, Brazil.\n",
      "FAU - Fagan, William F\n",
      "AU  - Fagan WF\n",
      "AUID- ORCID: 0000-0003-2433-9052\n",
      "AD  - University of Maryland, College Park, MD, USA.\n",
      "FAU - Fagundez, Jaime\n",
      "AU  - Fagundez J\n",
      "AUID- ORCID: 0000-0001-6605-7278\n",
      "AD  - Campus da Zapateira, University of A Coruna, A Coruna, Spain.\n",
      "FAU - Falster, Daniel S\n",
      "AU  - Falster DS\n",
      "AUID- ORCID: 0000-0002-9814-092X\n",
      "AD  - Evolution & Ecology Research Centre, and School of Biological, Earth andEnvironmental Sciences, UNSW Sydney, Sydney, NSW, Australia.\n",
      "FAU - Fan, Ying\n",
      "AU  - Fan Y\n",
      "AUID- ORCID: 0000-0002-0024-7965\n",
      "AD  - Rutgers University, Piscataway, NJ, USA.\n",
      "FAU - Fang, Jingyun\n",
      "AU  - Fang J\n",
      "AD  - Peking University, Beijing, China.\n",
      "FAU - Farris, Emmanuele\n",
      "AU  - Farris E\n",
      "AUID- ORCID: 0000-0002-9843-5998\n",
      "AD  - Department of Chemistry and Pharmacy, University of Sassari, Sassari, Italy.\n",
      "FAU - Fazlioglu, Fatih\n",
      "AU  - Fazlioglu F\n",
      "AUID- ORCID: 0000-0002-4723-3640\n",
      "AD  - Faculty of Arts and Sciences, Molecular Biology and Genetics, Ordu University,Ordu, Turkey.\n",
      "FAU - Feng, Yanhao\n",
      "AU  - Feng Y\n",
      "AUID- ORCID: 0000-0003-0460-4883\n",
      "AD  - State Key Laboratory of Grassland Agro-ecosystems, College of PastoralAgriculture Science and Technology, Lanzhou University, Lanzhou, China.\n",
      "FAU - Fernandez-Mendez, Fernando\n",
      "AU  - Fernandez-Mendez F\n",
      "AUID- ORCID: 0000-0001-8693-790X\n",
      "AD  - Grupo de Investigacion en Biodiversidad y Dinamica de Ecosistemas Tropicales -Universidad del Tolima, Ibague, Colombia.\n",
      "AD  - Centro Forestal Tropical Bajo Calima, Universidad del Tolima, Buenaventura,Colombia.\n",
      "FAU - Ferrara, Carlotta\n",
      "AU  - Ferrara C\n",
      "AD  - CREA - Research Centre for Forestry and Wood, Arezzo, Italy.\n",
      "FAU - Ferreira, Joice\n",
      "AU  - Ferreira J\n",
      "AUID- ORCID: 0000-0002-4008-2341\n",
      "AD  - Embrapa Amazonia Oriental, Belem, Brazil.\n",
      "FAU - Fidelis, Alessandra\n",
      "AU  - Fidelis A\n",
      "AUID- ORCID: 0000-0001-9545-2285\n",
      "AD  - Instituto de Biociencias, Laboratory of Vegetation Ecology, Universidade EstadualPaulista (UNESP), Rio Claro, Brazil.\n",
      "FAU - Finegan, Bryan\n",
      "AU  - Finegan B\n",
      "AD  - CATIE-Centro Agronomico Tropical de Investigacion y Ensenanza, Turrialba, CostaRica.\n",
      "FAU - Firn, Jennifer\n",
      "AU  - Firn J\n",
      "AUID- ORCID: 0000-0001-6026-8912\n",
      "AD  - Queensland University of Technology (QUT), Brisbane, Australia.\n",
      "FAU - Flowers, Timothy J\n",
      "AU  - Flowers TJ\n",
      "AUID- ORCID: 0000-0002-2712-9504\n",
      "AD  - School of Life Sciences, University of Sussex, Brighton, UK.\n",
      "FAU - Flynn, Dan F B\n",
      "AU  - Flynn DFB\n",
      "AUID- ORCID: 0000-0002-2978-5257\n",
      "AD  - Arnold Arboretum of Harvard University, Boston, MA, USA.\n",
      "FAU - Fontana, Veronika\n",
      "AU  - Fontana V\n",
      "AD  - Eurac Research, Institute for Alpine Environment, Bozen-Bolzano, Italy.\n",
      "FAU - Forey, Estelle\n",
      "AU  - Forey E\n",
      "AUID- ORCID: 0000-0001-6082-3023\n",
      "AD  - Laboratoire ECODIV URA, IRSTEA/EA 1293, Normandie Universite, UFR ST, Universite de Rouen, Mont Saint-Aignan, France.\n",
      "FAU - Forgiarini, Cristiane\n",
      "AU  - Forgiarini C\n",
      "AUID- ORCID: 0000-0002-0319-7430\n",
      "AD  - Department of Botany, Biosciences Institute, Federal University of Rio Grande do Sul, Porto Alegre, Brazil.\n",
      "FAU - Francois, Louis\n",
      "AU  - Francois L\n",
      "AUID- ORCID: 0000-0001-8292-8360\n",
      "AD  - Unit of Research SPHERES, University of Liege, Liege, Belgium.\n",
      "FAU - Frangipani, Marcelo\n",
      "AU  - Frangipani M\n",
      "AUID- ORCID: 0000-0001-8050-8282\n",
      "AD  - University of Guelph, Guelph, ON, Canada.\n",
      "AD  - Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil.\n",
      "FAU - Frank, Dorothea\n",
      "AU  - Frank D\n",
      "AD  - Max Planck Institute for Biogeochemistry, Jena, Germany.\n",
      "FAU - Frenette-Dussault, Cedric\n",
      "AU  - Frenette-Dussault C\n",
      "AD  - Geopole de l'Universite de Sherbrooke, Quebec, QC, Canada.\n",
      "FAU - Freschet, Gregoire T\n",
      "AU  - Freschet GT\n",
      "AUID- ORCID: 0000-0002-8830-3860\n",
      "AD  - Theoretical and Experimental Ecology Station, CNRS, Paul Sabatier UniversityToulouse, Moulis, France.\n",
      "FAU - Fry, Ellen L\n",
      "AU  - Fry EL\n",
      "AUID- ORCID: 0000-0001-7513-2006\n",
      "AD  - School of Earth and Environment Science, University of Manchester, Manchester,UK.\n",
      "FAU - Fyllas, Nikolaos M\n",
      "AU  - Fyllas NM\n",
      "AUID- ORCID: 0000-0002-5651-5578\n",
      "AD  - Biodiversity Conservation Laboratory, Department of Environment, University ofthe Aegean, Mytilene, Greece.\n",
      "FAU - Mazzochini, Guilherme G\n",
      "AU  - Mazzochini GG\n",
      "AUID- ORCID: 0000-0002-6932-8544\n",
      "AD  - Department of Plant Biology, Institute of Biology, University of Campinas,Campinas, Brazil.\n",
      "FAU - Gachet, Sophie\n",
      "AU  - Gachet S\n",
      "AD  - Aix Marseille Univ, Univ Avignon, CNRS, IRD, IMBE, Marseille, France.\n",
      "FAU - Gallagher, Rachael\n",
      "AU  - Gallagher R\n",
      "AUID- ORCID: 0000-0002-4680-8115\n",
      "AD  - Department of Biological Sciences, Macquarie University, Sydney, NSW, Australia.\n",
      "FAU - Ganade, Gislene\n",
      "AU  - Ganade G\n",
      "AD  - Universidade Federal do Rio Grande do Norte - UFRN, Natal, RN, Brazil.\n",
      "FAU - Ganga, Francesca\n",
      "AU  - Ganga F\n",
      "AD  - Department of Life and Environmental Sciences, Botany Division, University ofCagliari, Cagliari, Italy.\n",
      "FAU - Garcia-Palacios, Pablo\n",
      "AU  - Garcia-Palacios P\n",
      "AUID- ORCID: 0000-0002-6367-4761\n",
      "AD  - Departamento de Biologia y Geologia, Fisica y Quimica Inorganica y Analitica,Universidad Rey Juan Carlos, Mostoles, Spain.\n",
      "FAU - Gargaglione, Veronica\n",
      "AU  - Gargaglione V\n",
      "AUID- ORCID: 0000-0002-0797-1868\n",
      "AD  - Instituto Nacional de Tecnologia Agropecuaria, Consejo Nacional de InvetigacionesCientificas y Tecnicas, Universidad Nacional de La Patagonia Austral, RioGallegos, Argentina.\n",
      "FAU - Garnier, Eric\n",
      "AU  - Garnier E\n",
      "AUID- ORCID: 0000-0002-9392-5154\n",
      "AD  - UMR 5175 CEFE, Univ. Montpellier, CNRS, EPHE, IRD, Univ. Paul Valery,Montpellier, France.\n",
      "FAU - Garrido, Jose Luis\n",
      "AU  - Garrido JL\n",
      "AUID- ORCID: 0000-0002-6859-4234\n",
      "AD  - Estacion Experimental del Zaidin, Consejo Superior de InvestigacionesCientificas, Granada, Spain.\n",
      "AD  - Estacion Biologica de Donana, Consejo Superior de Investigaciones Cientificas,Sevilla, Spain.\n",
      "FAU - de Gasper, Andre Luis\n",
      "AU  - de Gasper AL\n",
      "AUID- ORCID: 0000-0002-1940-9581\n",
      "AD  - Universidade Regional de Blumenau, Blumenau, SC, Brazil.\n",
      "FAU - Gea-Izquierdo, Guillermo\n",
      "AU  - Gea-Izquierdo G\n",
      "AUID- ORCID: 0000-0003-0148-3721\n",
      "AD  - INIA-CIFOR, Madrid, Spain.\n",
      "FAU - Gibson, David\n",
      "AU  - Gibson D\n",
      "AUID- ORCID: 0000-0002-0308-7506\n",
      "AD  - School of Biological Sciences, Southern Illinois University Carbondale,Carbondale, IL, USA.\n",
      "FAU - Gillison, Andrew N\n",
      "AU  - Gillison AN\n",
      "AUID- ORCID: 0000-0003-2781-1500\n",
      "AD  - Center for Biodiversity Management, Yungaburra, Qld, Australia.\n",
      "FAU - Giroldo, Aelton\n",
      "AU  - Giroldo A\n",
      "AUID- ORCID: 0000-0001-7850-0011\n",
      "AD  - Instituto Federal de Educacao Ciencia e Tecnologia do Ceara, Crateus, Brazil.\n",
      "FAU - Glasenhardt, Mary-Claire\n",
      "AU  - Glasenhardt MC\n",
      "AD  - The Morton Arboretum, Lisle, IL, USA.\n",
      "FAU - Gleason, Sean\n",
      "AU  - Gleason S\n",
      "AUID- ORCID: 0000-0002-5607-4741\n",
      "AD  - Water Management and Systems Research Unit, United States Department ofAgriculture, Agricultural Research Service, Fort Collins, CO, USA.\n",
      "FAU - Gliesch, Mariana\n",
      "AU  - Gliesch M\n",
      "AUID- ORCID: 0000-0003-3226-6762\n",
      "AD  - Institute of Integrative Biology, ETH Zurich (Swiss Federal Institute ofTechnology), Zurich, Switzerland.\n",
      "FAU - Goldberg, Emma\n",
      "AU  - Goldberg E\n",
      "AD  - Department of Ecology, Evolution & Behavior, University of Minnesota,Minneapolis, MN, USA.\n",
      "FAU - Goldel, Bastian\n",
      "AU  - Goldel B\n",
      "AD  - Section for Ecoinformatics and Biodiversity, Department of Bioscience, AarhusUniversity, Aarhus, Denmark.\n",
      "FAU - Gonzalez-Akre, Erika\n",
      "AU  - Gonzalez-Akre E\n",
      "AUID- ORCID: 0000-0001-8305-6672\n",
      "AD  - Smithsonian Conservation Biology Institute, Front Royal, VA, USA.\n",
      "FAU - Gonzalez-Andujar, Jose L\n",
      "AU  - Gonzalez-Andujar JL\n",
      "AUID- ORCID: 0000-0003-2356-4098\n",
      "AD  - CSIC - Institute for Sustainable Agriculture (IAS), Cordoba, Spain.\n",
      "FAU - Gonzalez-Melo, Andres\n",
      "AU  - Gonzalez-Melo A\n",
      "AD  - Facultad de Ciencias Naturales y Matematicas, Universidad del Rosario, Bogota,Colombia.\n",
      "FAU - Gonzalez-Robles, Ana\n",
      "AU  - Gonzalez-Robles A\n",
      "AUID- ORCID: 0000-0003-3135-8905\n",
      "AD  - Departamento de Biologia Animal, Biologia Vegetal y Ecologia, Universidad deJaen, Jaen, Spain.\n",
      "FAU - Graae, Bente Jessen\n",
      "AU  - Graae BJ\n",
      "AUID- ORCID: 0000-0002-5568-4759\n",
      "AD  - Norwegian University of Science and Technology NTNU, Trondheim, Norway.\n",
      "FAU - Granda, Elena\n",
      "AU  - Granda E\n",
      "AUID- ORCID: 0000-0002-9559-4213\n",
      "AD  - Department of Life Sciences, University of Alcala, Alcala de Henares, Spain.\n",
      "FAU - Graves, Sarah\n",
      "AU  - Graves S\n",
      "AUID- ORCID: 0000-0003-3805-4242\n",
      "AD  - University of Florida, Gainesville, FL, USA.\n",
      "FAU - Green, Walton A\n",
      "AU  - Green WA\n",
      "AD  - Department of Organismic and Evolutionary Biology, Harvard University, Cambridge,MA, USA.\n",
      "FAU - Gregor, Thomas\n",
      "AU  - Gregor T\n",
      "AUID- ORCID: 0000-0001-7692-8348\n",
      "AD  - Department of Botany and Molecular Evolution, Senckenberg Research Institute and Natural History Museum, Frankfurt am Main, Germany.\n",
      "FAU - Gross, Nicolas\n",
      "AU  - Gross N\n",
      "AUID- ORCID: 0000-0001-9730-3240\n",
      "AD  - UCA, INRA, VetAgro Sup, UMR Ecosysteme Prairial, Clermont-Ferrand, France.\n",
      "AD  - Departamento de Biologia y Geologia, Fisica y Quimica Inorganica, EscuelaSuperior de Ciencias Experimentales y Tecnologia, Universidad Rey Juan Carlos,Mostoles, Spain.\n",
      "FAU - Guerin, Greg R\n",
      "AU  - Guerin GR\n",
      "AUID- ORCID: 0000-0002-2104-6695\n",
      "AD  - School of Biological Sciences, The University of Adelaide, Adelaide, SA,Australia.\n",
      "FAU - Gunther, Angela\n",
      "AU  - Gunther A\n",
      "AD  - Max Planck Institute for Biogeochemistry, Jena, Germany.\n",
      "FAU - Gutierrez, Alvaro G\n",
      "AU  - Gutierrez AG\n",
      "AUID- ORCID: 0000-0001-8928-3198\n",
      "AD  - Departamento de Ciencias Ambientales y Recursos Naturales Renovables, Facultad deCiencias Agronomicas, Universidad de Chile, Santiago, Chile.\n",
      "FAU - Haddock, Lillie\n",
      "AU  - Haddock L\n",
      "AUID- ORCID: 0000-0003-3432-6191\n",
      "AD  - Pacific Northwest National Laboratory, Joint Global Change Research Institute,College Park, MD, USA.\n",
      "FAU - Haines, Anna\n",
      "AU  - Haines A\n",
      "AD  - The University of Manchester, Manchester, UK.\n",
      "FAU - Hall, Jefferson\n",
      "AU  - Hall J\n",
      "AUID- ORCID: 0000-0003-4761-9268\n",
      "AD  - Smithsonian Tropical Research Institute, Balboa, Ancon, Republic of Panama.\n",
      "FAU - Hambuckers, Alain\n",
      "AU  - Hambuckers A\n",
      "AD  - Unit of research SPHERES, University of Liege, Liege, Belgium.\n",
      "FAU - Han, Wenxuan\n",
      "AU  - Han W\n",
      "AUID- ORCID: 0000-0001-8678-7147\n",
      "AD  - College of Resources and Environmental Sciences, China Agricultural University,Beijing, China.\n",
      "AD  - Xinjiang Institute of Ecology and Geography, Chinese Academy of Sciences, Urumqi,China.\n",
      "AD  - Research Center for Ecology and Environment of Central Asia, Chinese Academy ofSciences, Urumqi, China.\n",
      "FAU - Harrison, Sandy P\n",
      "AU  - Harrison SP\n",
      "AUID- ORCID: 0000-0001-5687-1903\n",
      "AD  - University of Reading, Reading, UK.\n",
      "FAU - Hattingh, Wesley\n",
      "AU  - Hattingh W\n",
      "AD  - School of Animal, Plant and Environmental Sciences, University of theWitwatersrand, Johannesburg, South Africa.\n",
      "FAU - Hawes, Joseph E\n",
      "AU  - Hawes JE\n",
      "AUID- ORCID: 0000-0003-0053-2018\n",
      "AD  - Applied Ecology Research Group, School of Life Sciences, Anglia RuskinUniversity, Cambridge, UK.\n",
      "AD  - Faculty of Environmental Sciences and Natural Resource Management, NorwegianUniversity of Life Sciences, As, Norway.\n",
      "FAU - He, Tianhua\n",
      "AU  - He T\n",
      "AD  - School of Molecular and Life Sciences, Curtin University, Perth, WA, Australia.\n",
      "AD  - College of Science, Health, Engineering and Education, Murdoch University,Murdoch, WA, Australia.\n",
      "FAU - He, Pengcheng\n",
      "AU  - He P\n",
      "AUID- ORCID: 0000-0001-9531-531X\n",
      "AD  - South China Botanical Garden, Chinese Academy of Sciences, Guangzhou, China.\n",
      "FAU - Heberling, Jacob Mason\n",
      "AU  - Heberling JM\n",
      "AUID- ORCID: 0000-0003-0756-5090\n",
      "AD  - Carnegie Museum of Natural History, Pittsburgh, PA, USA.\n",
      "FAU - Helm, Aveliina\n",
      "AU  - Helm A\n",
      "AUID- ORCID: 0000-0003-2338-4564\n",
      "AD  - Institute of Ecology and Earth Sciences, University of Tartu, Tartu, Estonia.\n",
      "FAU - Hempel, Stefan\n",
      "AU  - Hempel S\n",
      "AD  - Institut fur Biologie, Freie Universitat Berlin, Berlin, Germany.\n",
      "AD  - Berlin-Brandenburg Institute of Advanced Biodiversity Research (BBIB), Berlin,Germany.\n",
      "FAU - Hentschel, Jorn\n",
      "AU  - Hentschel J\n",
      "AD  - Herbarium Haussknecht, Friedrich-Schiller-Universitat Jena, Jena, Germany.\n",
      "FAU - Herault, Bruno\n",
      "AU  - Herault B\n",
      "AUID- ORCID: 0000-0002-6950-7286\n",
      "AD  - Cirad, Universite de Montpellier, Montpellier, France.\n",
      "AD  - Institut National Polytehcnique Felix Houphouet-Boigny, INP-HB, Yamoussoukro,Ivory Coast.\n",
      "FAU - Heres, Ana-Maria\n",
      "AU  - Heres AM\n",
      "AUID- ORCID: 0000-0002-1839-1770\n",
      "AD  - Department of Forest Sciences, Transilvania University of Brasov, Brasov,Romania.\n",
      "AD  - BC3 - Basque Centre for Climate Change, Scientific Campus of the University ofthe Basque Country, Leioa, Spain.\n",
      "FAU - Herz, Katharina\n",
      "AU  - Herz K\n",
      "AUID- ORCID: 0000-0001-5676-3646\n",
      "AD  - Institute of Biology/Geobotany and Botanical Garden, Martin Luther UniversityHalle-Wittenberg, Halle, Germany.\n",
      "FAU - Heuertz, Myriam\n",
      "AU  - Heuertz M\n",
      "AUID- ORCID: 0000-0002-6322-3645\n",
      "AD  - Univ. Bordeaux, INRAE, BIOGECO, Cestas, France.\n",
      "FAU - Hickler, Thomas\n",
      "AU  - Hickler T\n",
      "AD  - Senckenberg Biodiversity and Climate Research Centre, Frankfurt am Main, Germany.\n",
      "AD  - Department of Physical Geography, Goethe University, Frankfurt am Main, Germany.\n",
      "FAU - Hietz, Peter\n",
      "AU  - Hietz P\n",
      "AUID- ORCID: 0000-0002-0458-6593\n",
      "AD  - Institute of Botany, University of Natural Resources and Life Sciences, Vienna,Austria.\n",
      "FAU - Higuchi, Pedro\n",
      "AU  - Higuchi P\n",
      "AUID- ORCID: 0000-0002-3855-555X\n",
      "AD  - Santa Catarina State University, Lages, SC, Brazil.\n",
      "FAU - Hipp, Andrew L\n",
      "AU  - Hipp AL\n",
      "AUID- ORCID: 0000-0002-1241-9904\n",
      "AD  - The Morton Arboretum, Lisle, IL, USA.\n",
      "AD  - The Field Museum, Chicago, IL, USA.\n",
      "FAU - Hirons, Andrew\n",
      "AU  - Hirons A\n",
      "AD  - University Centre Myerscough, Preston, UK.\n",
      "FAU - Hock, Maria\n",
      "AU  - Hock M\n",
      "AUID- ORCID: 0000-0001-8401-7309\n",
      "AD  - Institute for Ecosystem Research/Geobotany, Kiel University, Kiel, Germany.\n",
      "FAU - Hogan, James Aaron\n",
      "AU  - Hogan JA\n",
      "AUID- ORCID: 0000-0001-9806-3074\n",
      "AD  - Department of Biological Sciences, Florida International University, Miami, FL,USA.\n",
      "AD  - Oak Ridge National Laboratory, US Department of Energy, Oak Ridge, TN.\n",
      "FAU - Holl, Karen\n",
      "AU  - Holl K\n",
      "AUID- ORCID: 0000-0003-2893-6161\n",
      "AD  - University of California, Santa Cruz, Santa Cruz, CA, USA.\n",
      "FAU - Honnay, Olivier\n",
      "AU  - Honnay O\n",
      "AUID- ORCID: 0000-0002-4287-8511\n",
      "AD  - Plant Conservation and Population Biology, Department of Biology, KU Leuven,Leuven, Belgium.\n",
      "AD  - Division of Ecology, Evolution and Biodiversity Conservation, Heverlee, Belgium.\n",
      "FAU - Hornstein, Daniel\n",
      "AU  - Hornstein D\n",
      "AD  - University of Bayreuth, Bayreuth, Germany.\n",
      "FAU - Hou, Enqing\n",
      "AU  - Hou E\n",
      "AUID- ORCID: 0000-0003-4864-2347\n",
      "AD  - South China Botanical Garden, Chinese Academy of Sciences, Guangzhou, China.\n",
      "FAU - Hough-Snee, Nate\n",
      "AU  - Hough-Snee N\n",
      "AUID- ORCID: 0000-0003-4581-0931\n",
      "AD  - Four Peaks Environmental Science and Data Solutions, Wenatchee, WA, USA.\n",
      "FAU - Hovstad, Knut Anders\n",
      "AU  - Hovstad KA\n",
      "AUID- ORCID: 0000-0002-7108-0787\n",
      "AD  - Department of Landscape and Biodiversity, Norwegian Institute of BioeconomyResearch (NIBIO), As, Norway.\n",
      "FAU - Ichie, Tomoaki\n",
      "AU  - Ichie T\n",
      "AUID- ORCID: 0000-0002-1979-5806\n",
      "AD  - Kochi University, Nankoku, Japan.\n",
      "FAU - Igic, Boris\n",
      "AU  - Igic B\n",
      "AUID- ORCID: 0000-0002-6302-8030\n",
      "AD  - University of Illinois at Chicago, Chicago, IL, USA.\n",
      "FAU - Illa, Estela\n",
      "AU  - Illa E\n",
      "AUID- ORCID: 0000-0001-7136-6518\n",
      "AD  - Department of Evolutionary Biology, Ecology and Environmental Sciences,Biodiversity Research Institute (IRBio), Universitat de Barcelona, Barcelona,Spain.\n",
      "FAU - Isaac, Marney\n",
      "AU  - Isaac M\n",
      "AUID- ORCID: 0000-0002-5975-833X\n",
      "AD  - University of Toronto, Toronto, ON, Canada.\n",
      "FAU - Ishihara, Masae\n",
      "AU  - Ishihara M\n",
      "AUID- ORCID: 0000-0002-8365-7766\n",
      "AD  - Ashiu Forest Research Station, Field Science Education and Research Center, KyotoUniversity, Kyoto, Japan.\n",
      "FAU - Ivanov, Leonid\n",
      "AU  - Ivanov L\n",
      "AUID- ORCID: 0000-0001-6900-5086\n",
      "AD  - Institute Botanic Garden, Ural Branch of the Russian Academy of Sciences,Ekaterinburg, Russia.\n",
      "AD  - Tyumen State University, Tyumen, Russia.\n",
      "FAU - Ivanova, Larissa\n",
      "AU  - Ivanova L\n",
      "AUID- ORCID: 0000-0003-2363-9619\n",
      "AD  - Institute Botanic Garden, Ural Branch of the Russian Academy of Sciences,Ekaterinburg, Russia.\n",
      "AD  - Tyumen State University, Tyumen, Russia.\n",
      "FAU - Iversen, Colleen M\n",
      "AU  - Iversen CM\n",
      "AUID- ORCID: 0000-0001-8293-3450\n",
      "AD  - Oak Ridge National Laboratory, Oak Ridge, TN, USA.\n",
      "FAU - Izquierdo, Jordi\n",
      "AU  - Izquierdo J\n",
      "AUID- ORCID: 0000-0002-3949-1295\n",
      "AD  - Barcelona School of Agricultural Engineering, Universitat Politecnica deCatalunya, Catalonia, Spain.\n",
      "FAU - Jackson, Robert B\n",
      "AU  - Jackson RB\n",
      "AUID- ORCID: 0000-0001-8846-7147\n",
      "AD  - Earth System Science Department, Stanford University, Stanford, CA, USA.\n",
      "FAU - Jackson, Benjamin\n",
      "AU  - Jackson B\n",
      "AUID- ORCID: 0000-0001-6405-8111\n",
      "AD  - Global Academy of Agriculture and Food Security, University of Edinburgh,Midlothian, Scotland.\n",
      "FAU - Jactel, Herve\n",
      "AU  - Jactel H\n",
      "AUID- ORCID: 0000-0002-8106-5310\n",
      "AD  - Univ. Bordeaux, INRAE, BIOGECO, Cestas, France.\n",
      "FAU - Jagodzinski, Andrzej M\n",
      "AU  - Jagodzinski AM\n",
      "AUID- ORCID: 0000-0001-6899-0985\n",
      "AD  - Institute of Dendrology, Polish Academy of Sciences, Kornik, Poland.\n",
      "AD  - Department of Game Management and Forest Protection, Faculty of Forestry, Poznan University of Life Sciences, Poznan, Poland.\n",
      "FAU - Jandt, Ute\n",
      "AU  - Jandt U\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "AD  - Geobotany and Botanical Garden, Martin Luther University Halle-Wittenberg, Halle,Germany.\n",
      "FAU - Jansen, Steven\n",
      "AU  - Jansen S\n",
      "AUID- ORCID: 0000-0002-4476-5334\n",
      "AD  - Institute of Systematic Botany and Ecology, Ulm University, Ulm, Germany.\n",
      "FAU - Jenkins, Thomas\n",
      "AU  - Jenkins T\n",
      "AUID- ORCID: 0000-0002-3036-971X\n",
      "AD  - Department of Zoology, University of Oxford, Oxford, UK.\n",
      "AD  - Rocky Mountain Biological Laboratory, Crested Butte, CO, USA.\n",
      "FAU - Jentsch, Anke\n",
      "AU  - Jentsch A\n",
      "AUID- ORCID: 0000-0002-2345-8300\n",
      "AD  - BayCEER, Department of Disturbance Ecology, University of Bayreuth, Bayreuth,Germany.\n",
      "FAU - Jespersen, Jens Rasmus Plantener\n",
      "AU  - Jespersen JRP\n",
      "AD  - Department of Biology, University of Copenhagen, Frederiksberg C, Denmark.\n",
      "FAU - Jiang, Guo-Feng\n",
      "AU  - Jiang GF\n",
      "AUID- ORCID: 0000-0002-3221-8608\n",
      "AD  - Plant Ecophysiology & Evolution Group, Guangxi Key Laboratory of Forest Ecologyand Conservation, College of Forestry, Guangxi University, Nanning, Guangxi, PRChina.\n",
      "AD  - State Key Laboratory for Conservation and Utilization of SubtropicalAgro-bioresources, Guangxi University, Nanning, PR China.\n",
      "FAU - Johansen, Jesper Liengaard\n",
      "AU  - Johansen JL\n",
      "AUID- ORCID: 0000-0001-7602-316X\n",
      "AD  - Terrestrial Ecology Section, Department of Biology, University of Copenhagen,Denmark.\n",
      "FAU - Johnson, David\n",
      "AU  - Johnson D\n",
      "AUID- ORCID: 0000-0003-2299-2525\n",
      "AD  - The University of Manchester, Manchester, UK.\n",
      "FAU - Jokela, Eric J\n",
      "AU  - Jokela EJ\n",
      "AUID- ORCID: 0000-0002-1819-1285\n",
      "AD  - School of Forest Resources and Conservation, University of Florida, Gainesville, FL, USA.\n",
      "FAU - Joly, Carlos Alfredo\n",
      "AU  - Joly CA\n",
      "AUID- ORCID: 0000-0002-7945-2805\n",
      "AD  - State University of Campinas/UNICAMP, Campinas, SP, Brazil.\n",
      "FAU - Jordan, Gregory J\n",
      "AU  - Jordan GJ\n",
      "AUID- ORCID: 0000-0002-6033-2766\n",
      "AD  - Biological Sciences, University of Tasmania, Hobart, Australia.\n",
      "FAU - Joseph, Grant Stuart\n",
      "AU  - Joseph GS\n",
      "AUID- ORCID: 0000-0002-8731-9828\n",
      "AD  - Department of Zoology, School of Mathematical and Natural Science, University of Venda, Thohoyandou, South Africa.\n",
      "AD  - Department of Biological Sciences, DST/NRF Centre of Excellence, PercyFitzPatrick Institute of African Ornithology, University of Cape Town,Rondebosch, South Africa.\n",
      "FAU - Junaedi, Decky\n",
      "AU  - Junaedi D\n",
      "AUID- ORCID: 0000-0001-5846-6135\n",
      "AD  - Cibodas Botanical Garden - Indonesian Institute of Sciences (LIPI), Jl. KebunRaya Cibodas, Cipanas, Indonesia.\n",
      "AD  - Centre of Excellence for Biosecurity Risk Analysis (CEBRA), School OfBiosciences, University of Melbourne, Parkville, Vic., Australia.\n",
      "FAU - Junker, Robert R\n",
      "AU  - Junker RR\n",
      "AUID- ORCID: 0000-0002-7919-9678\n",
      "AD  - Evolutionary Ecology of Plants, Department Biology, Philipps-University Marburg, Marburg, Germany.\n",
      "AD  - Department of Bioscience, University Salzburg, Salzburg, Austria.\n",
      "FAU - Justes, Eric\n",
      "AU  - Justes E\n",
      "AUID- ORCID: 0000-0001-7390-7058\n",
      "AD  - PERSYST Department, CIRAD, Montpellier Cedex 5, France.\n",
      "FAU - Kabzems, Richard\n",
      "AU  - Kabzems R\n",
      "AD  - BC Ministry Forest, Lands, Natural Resource Operations and Rural Development,Dawson Creek, BC, Canada.\n",
      "FAU - Kane, Jeffrey\n",
      "AU  - Kane J\n",
      "AUID- ORCID: 0000-0003-1453-9608\n",
      "AD  - Humboldt State University, Arcata, CA, USA.\n",
      "FAU - Kaplan, Zdenek\n",
      "AU  - Kaplan Z\n",
      "AUID- ORCID: 0000-0003-1707-7461\n",
      "AD  - Institute of Botany, The Czech Academy of Sciences, Pruhonice, Czech Republic.\n",
      "AD  - Department of Botany, Faculty of Science, Charles University, Prague, CzechRepublic.\n",
      "FAU - Kattenborn, Teja\n",
      "AU  - Kattenborn T\n",
      "AUID- ORCID: 0000-0001-7381-3828\n",
      "AD  - Institute of Geography and Geoecology, Karlsruhe Institute of Technology,Karlsruhe, Germany.\n",
      "FAU - Kavelenova, Lyudmila\n",
      "AU  - Kavelenova L\n",
      "AD  - Samara National Research University, Samara, Russia.\n",
      "FAU - Kearsley, Elizabeth\n",
      "AU  - Kearsley E\n",
      "AUID- ORCID: 0000-0003-0046-3606\n",
      "AD  - CAVElab - Computational and Applied Vegetation Ecology, Ghent University, Ghent, Belgium.\n",
      "FAU - Kempel, Anne\n",
      "AU  - Kempel A\n",
      "AUID- ORCID: 0000-0002-0563-901X\n",
      "AD  - Institute of Plant Sciences, Bern, Switzerland.\n",
      "FAU - Kenzo, Tanaka\n",
      "AU  - Kenzo T\n",
      "AUID- ORCID: 0000-0002-4856-7204\n",
      "AD  - Forestry and Forest Products Research Institute, Tsukuba, Japan.\n",
      "FAU - Kerkhoff, Andrew\n",
      "AU  - Kerkhoff A\n",
      "AD  - Kenyon College, Gambier, OH, USA.\n",
      "FAU - Khalil, Mohammed I\n",
      "AU  - Khalil MI\n",
      "AUID- ORCID: 0000-0002-9310-2780\n",
      "AD  - Department of Biology, University of Garmian, Kalar, Iraq.\n",
      "AD  - School of Biological Sciences and Center for Ecology, Southern IllinoisUniversity Carbondale, Carbondale, IL, USA.\n",
      "FAU - Kinlock, Nicole L\n",
      "AU  - Kinlock NL\n",
      "AUID- ORCID: 0000-0002-2917-5133\n",
      "AD  - Department of Ecology and Evolution, State University of New York at Stony Brook,Stony Brook, NY, USA.\n",
      "FAU - Kissling, Wilm Daniel\n",
      "AU  - Kissling WD\n",
      "AUID- ORCID: 0000-0002-7274-6755\n",
      "AD  - Institute for Biodiversity and Ecosystem Dynamics (IBED), University ofAmsterdam, Amsterdam, The Netherlands.\n",
      "FAU - Kitajima, Kaoru\n",
      "AU  - Kitajima K\n",
      "AUID- ORCID: 0000-0001-6822-8536\n",
      "AD  - Graduate School of Agriculture, Kyoto University, Kyoto, Japan.\n",
      "AD  - University of Florida, Gainesville, FL, USA.\n",
      "AD  - School of Molecular and Life Sciences, Curtin University, Perth, WA, Australia.\n",
      "FAU - Kitzberger, Thomas\n",
      "AU  - Kitzberger T\n",
      "AUID- ORCID: 0000-0002-9754-4121\n",
      "AD  - Instituto de Investigaciones en Biodiversidad y Medioambiente (INIBIOMA),CONICET, Bariloche, Argentina.\n",
      "AD  - Departamento de Ecologia, Universidad Nacional del Comahue, Bariloche, Argentina.\n",
      "FAU - Kjoller, Rasmus\n",
      "AU  - Kjoller R\n",
      "AUID- ORCID: 0000-0002-2027-4119\n",
      "AD  - Department of Biology, University of Copenhagen, Copenhagen, Denmark.\n",
      "FAU - Klein, Tamir\n",
      "AU  - Klein T\n",
      "AUID- ORCID: 0000-0002-3882-8845\n",
      "AD  - Department of Plant and Environmental Sciences, Weizmann Institute of Science,Rehovot, Israel.\n",
      "FAU - Kleyer, Michael\n",
      "AU  - Kleyer M\n",
      "AUID- ORCID: 0000-0002-0824-2974\n",
      "AD  - Landscape Ecology Group, Institute of Biology and Environmental Sciences,University of Oldenburg, Oldenburg, Germany.\n",
      "FAU - Klimesova, Jitka\n",
      "AU  - Klimesova J\n",
      "AUID- ORCID: 0000-0003-0123-3263\n",
      "AD  - Institute of Botany, Czech Academy of Sciences, Trebon, Czech Republic.\n",
      "AD  - Faculty of Sciences, Charles University, Praha, Czech Republic.\n",
      "FAU - Klipel, Joice\n",
      "AU  - Klipel J\n",
      "AUID- ORCID: 0000-0003-3936-9692\n",
      "AD  - Laboratorio de Ecologia Vegetal (LEVEG), Programa de Pos-Graduacao em Ecologia,Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil.\n",
      "FAU - Kloeppel, Brian\n",
      "AU  - Kloeppel B\n",
      "AUID- ORCID: 0000-0003-0215-4671\n",
      "AD  - Department of Geosciences and Natural Resources, Western Carolina University,Cullowhee, NC, USA.\n",
      "FAU - Klotz, Stefan\n",
      "AU  - Klotz S\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "AD  - Department of Community Ecology, Helmholtz Centre for Environmental Research-UFZ,Halle (Saale), Germany.\n",
      "FAU - Knops, Johannes M H\n",
      "AU  - Knops JMH\n",
      "AD  - Health and Environmental Sciences, Xi'an Jiaotong Liverpool University, Suzhou,Jiangsu, China.\n",
      "FAU - Kohyama, Takashi\n",
      "AU  - Kohyama T\n",
      "AUID- ORCID: 0000-0001-7186-8585\n",
      "AD  - Hokkaido University, Sapporo, Japan.\n",
      "FAU - Koike, Fumito\n",
      "AU  - Koike F\n",
      "AUID- ORCID: 0000-0002-6588-6485\n",
      "AD  - Graduate School of Environment and Information Sciences, Yokohama NationalUniversity, Yokohama, Japan.\n",
      "FAU - Kollmann, Johannes\n",
      "AU  - Kollmann J\n",
      "AUID- ORCID: 0000-0002-4990-3636\n",
      "AD  - Technical University of Munich, Freising, Germany.\n",
      "FAU - Komac, Benjamin\n",
      "AU  - Komac B\n",
      "AD  - Institut d Estudis Andorrans, Andorra.\n",
      "FAU - Komatsu, Kimberly\n",
      "AU  - Komatsu K\n",
      "AUID- ORCID: 0000-0001-7056-4547\n",
      "AD  - Smithsonian Environmental Research Center, Edgewater, MD, USA.\n",
      "FAU - Konig, Christian\n",
      "AU  - Konig C\n",
      "AUID- ORCID: 0000-0003-0585-5983\n",
      "AD  - Department of Geography, Humboldt University of Berlin, Berlin, Germany.\n",
      "AD  - Biodiversity, Macroecology and Biogeography, University of Goettingen, Gottingen,Germany.\n",
      "FAU - Kraft, Nathan J B\n",
      "AU  - Kraft NJB\n",
      "AUID- ORCID: 0000-0001-8867-7806\n",
      "AD  - Department of Ecology and Evolutionary Biology, University of California, LosAngeles, CA, USA.\n",
      "FAU - Kramer, Koen\n",
      "AU  - Kramer K\n",
      "AUID- ORCID: 0000-0002-1402-2775\n",
      "AD  - Wageningen University & Research, Wageningen, The Netherlands.\n",
      "AD  - Land Life Company, Amsterdam, The Netherlands.\n",
      "FAU - Kreft, Holger\n",
      "AU  - Kreft H\n",
      "AUID- ORCID: 0000-0003-4471-8236\n",
      "AD  - Biodiversity, Macroecology and Biogeography, University of Goettingen, Gottingen,Germany.\n",
      "AD  - Centre of Biodiversity and Sustainable Land Use (CBL), University of Goettingen, Gottingen, Germany.\n",
      "FAU - Kuhn, Ingolf\n",
      "AU  - Kuhn I\n",
      "AUID- ORCID: 0000-0003-1691-8249\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "AD  - Helmholtz Centre for Environmental Research - UFZ, Halle, Germany.\n",
      "AD  - Martin Luther University Halle-Wittenberg, Halle, Germany.\n",
      "FAU - Kumarathunge, Dushan\n",
      "AU  - Kumarathunge D\n",
      "AUID- ORCID: 0000-0003-1309-4731\n",
      "AD  - Hawkesbury Institute for the Environment, Western Sydney University, Sydney, NSW,Australia.\n",
      "AD  - Plant Physiology Division, Coconut Research Institute of Sri Lanka, Lunuwila, SriLanka.\n",
      "FAU - Kuppler, Jonas\n",
      "AU  - Kuppler J\n",
      "AUID- ORCID: 0000-0003-4409-9367\n",
      "AD  - Institute of Evolutionary Ecology and Conservation Genomics, Ulm University, Ulm,Germany.\n",
      "FAU - Kurokawa, Hiroko\n",
      "AU  - Kurokawa H\n",
      "AUID- ORCID: 0000-0001-8778-8045\n",
      "AD  - Forestry and Forest Products Research Institute, Tsukuba, Japan.\n",
      "FAU - Kurosawa, Yoko\n",
      "AU  - Kurosawa Y\n",
      "AUID- ORCID: 0000-0002-1419-7575\n",
      "AD  - Yamagata University, Yamagata, Japan.\n",
      "FAU - Kuyah, Shem\n",
      "AU  - Kuyah S\n",
      "AUID- ORCID: 0000-0001-7692-585X\n",
      "AD  - Jomo Kenyatta University of Agriculture and Technology (JKUAT), Nairobi, Kenya.\n",
      "FAU - Laclau, Jean-Paul\n",
      "AU  - Laclau JP\n",
      "AUID- ORCID: 0000-0002-2506-214X\n",
      "AD  - CIRAD, UMR Eco&Sols, Montpellier, France.\n",
      "AD  - Eco&Sols, CIRAD, INRA, IRD, SupAgro, University of Montpellier, Montpellier,France.\n",
      "FAU - Lafleur, Benoit\n",
      "AU  - Lafleur B\n",
      "AD  - Institut de recherche sur les forets, Universite du Quebec enAbitibi-Temiscamingue, Rouyn-Noranda, QC, Canada.\n",
      "FAU - Lallai, Erik\n",
      "AU  - Lallai E\n",
      "AD  - Department of Life and Environmental Sciences, Botany Division, University ofCagliari, Cagliari, Italy.\n",
      "FAU - Lamb, Eric\n",
      "AU  - Lamb E\n",
      "AUID- ORCID: 0000-0001-5201-4541\n",
      "AD  - Department of Plant Sciences, University of Saskatchewan, Saskatoon, SK, Canada.\n",
      "FAU - Lamprecht, Andrea\n",
      "AU  - Lamprecht A\n",
      "AUID- ORCID: 0000-0002-8719-026X\n",
      "AD  - GLORIA-Coordination, Institute for Interdisciplinary Mountain Research, Austrian Academy of Sciences & Department of Integrative Biology and BiodiversityResearch, University of Natural Resources and Life Sciences Vienna, Vienna,Austria.\n",
      "FAU - Larkin, Daniel J\n",
      "AU  - Larkin DJ\n",
      "AUID- ORCID: 0000-0001-6378-0495\n",
      "AD  - Department of Fisheries, Wildlife and Conservation Biology, University ofMinnesota, St. Paul, MN, USA.\n",
      "FAU - Laughlin, Daniel\n",
      "AU  - Laughlin D\n",
      "AUID- ORCID: 0000-0002-9651-5732\n",
      "AD  - Botany Department, University of Wyoming, Laramie, WY, USA.\n",
      "FAU - Le Bagousse-Pinguet, Yoann\n",
      "AU  - Le Bagousse-Pinguet Y\n",
      "AUID- ORCID: 0000-0002-5615-5541\n",
      "AD  - Aix Marseille Univ, Univ Avignon, CNRS, IRD, IMBE, Marseille, France.\n",
      "FAU - le Maire, Guerric\n",
      "AU  - le Maire G\n",
      "AUID- ORCID: 0000-0002-5227-958X\n",
      "AD  - CIRAD, UMR Eco&Sols, Montpellier, France.\n",
      "AD  - Eco&Sols, CIRAD, INRA, IRD, SupAgro, University of Montpellier, Montpellier,France.\n",
      "FAU - le Roux, Peter C\n",
      "AU  - le Roux PC\n",
      "AUID- ORCID: 0000-0002-7941-7444\n",
      "AD  - Department of Plant and Soil Sciences, University of Pretoria, Pretoria, SouthAfrica.\n",
      "FAU - le Roux, Elizabeth\n",
      "AU  - le Roux E\n",
      "AUID- ORCID: 0000-0001-8468-8284\n",
      "AD  - Nelson Mandela University, Port Elizabeth, South Africa.\n",
      "FAU - Lee, Tali\n",
      "AU  - Lee T\n",
      "AUID- ORCID: 0000-0002-3933-6607\n",
      "AD  - University of Wisconsin Eau Claire, Eau Claire, WI, USA.\n",
      "FAU - Lens, Frederic\n",
      "AU  - Lens F\n",
      "AUID- ORCID: 0000-0002-5001-0149\n",
      "AD  - Naturalis Biodiversity Center, Leiden, The Netherlands.\n",
      "FAU - Lewis, Simon L\n",
      "AU  - Lewis SL\n",
      "AUID- ORCID: 0000-0002-8066-6851\n",
      "AD  - School of Geography, University of Leeds, Leeds, UK.\n",
      "AD  - Department of Geography, University College London, London, UK.\n",
      "FAU - Lhotsky, Barbara\n",
      "AU  - Lhotsky B\n",
      "AD  - Jonah Ventures, Boulder, CO, USA.\n",
      "FAU - Li, Yuanzhi\n",
      "AU  - Li Y\n",
      "AUID- ORCID: 0000-0002-5258-877X\n",
      "AD  - Sun Yat-sen University, Guangzhou, China.\n",
      "FAU - Li, Xine\n",
      "AU  - Li X\n",
      "AD  - Yangzhou University, Yangzhou, Jiangsu, China.\n",
      "FAU - Lichstein, Jeremy W\n",
      "AU  - Lichstein JW\n",
      "AUID- ORCID: 0000-0001-5553-6142\n",
      "AD  - Department of Biology, University of Florida, Gainesville, FL, USA.\n",
      "FAU - Liebergesell, Mario\n",
      "AU  - Liebergesell M\n",
      "AD  - University of Leipzig, Leipzig, Germany.\n",
      "FAU - Lim, Jun Ying\n",
      "AU  - Lim JY\n",
      "AUID- ORCID: 0000-0001-7493-2159\n",
      "AD  - Institute for Biodiversity and Ecosystem Dynamics (IBED), University ofAmsterdam, Amsterdam, The Netherlands.\n",
      "FAU - Lin, Yan-Shih\n",
      "AU  - Lin YS\n",
      "AD  - Macquarie University, North Ryde, NSW, Australia.\n",
      "FAU - Linares, Juan Carlos\n",
      "AU  - Linares JC\n",
      "AUID- ORCID: 0000-0001-8375-6353\n",
      "AD  - University Pablo de Olavide, Seville, Spain.\n",
      "FAU - Liu, Chunjiang\n",
      "AU  - Liu C\n",
      "AD  - School of Agriculture and Biology, Shanghai Jiao Tong University, Shanghai, P. R.China.\n",
      "AD  - Shanghai Urban Forest Ecosystem Research Station, National Forestry and GrasslandAdministration, Shanghai, P.R. China.\n",
      "FAU - Liu, Daijun\n",
      "AU  - Liu D\n",
      "AUID- ORCID: 0000-0002-0993-0832\n",
      "AD  - School of Geography, Earth and Environmental Sciences - University of Birmingham,Birmingham, UK.\n",
      "FAU - Liu, Udayangani\n",
      "AU  - Liu U\n",
      "AUID- ORCID: 0000-0002-6192-1319\n",
      "AD  - Royal Botanic Gardens, Kew, West Sussex, UK.\n",
      "FAU - Livingstone, Stuart\n",
      "AU  - Livingstone S\n",
      "AD  - University of Toronto Scarborough, Scarborough, ON, Canada.\n",
      "FAU - Llusia, Joan\n",
      "AU  - Llusia J\n",
      "AD  - Universitat Autonoma de Barcelona, Barcelona, Spain.\n",
      "FAU - Lohbeck, Madelon\n",
      "AU  - Lohbeck M\n",
      "AUID- ORCID: 0000-0002-3959-1800\n",
      "AD  - Forest Ecology and Forest Management Group, Wageningen University and Research,Wageningen, The Netherlands.\n",
      "AD  - World Agroforestry (ICRAF), Nairobi, Kenya.\n",
      "FAU - Lopez-Garcia, Alvaro\n",
      "AU  - Lopez-Garcia A\n",
      "AUID- ORCID: 0000-0001-8267-3572\n",
      "AD  - Department of Biology, University of Copenhagen, Copenhagen, Denmark.\n",
      "AD  - Dept. Animal Biology, Plant Biology and Ecology, University of Jaen, Jaen, Spain.\n",
      "FAU - Lopez-Gonzalez, Gabriela\n",
      "AU  - Lopez-Gonzalez G\n",
      "AUID- ORCID: 0000-0002-0814-8830\n",
      "AD  - water@leeds, School of Geography, University of Leeds, Leeds, UK.\n",
      "FAU - Lososova, Zdenka\n",
      "AU  - Lososova Z\n",
      "AUID- ORCID: 0000-0001-9152-7462\n",
      "AD  - Department of Botany and Zoology, Masaryk University, Brno, Czech Republic.\n",
      "FAU - Louault, Frederique\n",
      "AU  - Louault F\n",
      "AUID- ORCID: 0000-0002-9430-4261\n",
      "AD  - UCA, INRA, VetAgro Sup, UMR Ecosysteme Prairial, Clermont-Ferrand, France.\n",
      "FAU - Lukacs, Balazs A\n",
      "AU  - Lukacs BA\n",
      "AUID- ORCID: 0000-0002-3163-1281\n",
      "AD  - Department for Tisza River Research, MTA Centre for Ecological Research, DRI,Debrecen, Hungary.\n",
      "FAU - Lukes, Petr\n",
      "AU  - Lukes P\n",
      "AD  - Global Change Research Institute AS CR, Brno, Czech Republic.\n",
      "FAU - Luo, Yunjian\n",
      "AU  - Luo Y\n",
      "AD  - Department of Ecology, School of Horticulture and Plant Protection, YangzhouUniversity, Yangzhou, China.\n",
      "AD  - State Key Laboratory of Urban and Regional Ecology, Research Centre forEco-Environmental Sciences, Chinese Academy of Sciences, Beijing, China.\n",
      "FAU - Lussu, Michele\n",
      "AU  - Lussu M\n",
      "AUID- ORCID: 0000-0002-1313-4732\n",
      "AD  - Department of Life and Environmental Sciences, Botany Division, University ofCagliari, Cagliari, Italy.\n",
      "FAU - Ma, Siyan\n",
      "AU  - Ma S\n",
      "AD  - University of California at Berkeley, Berkeley, CA, USA.\n",
      "FAU - Maciel Rabelo Pereira, Camilla\n",
      "AU  - Maciel Rabelo Pereira C\n",
      "AUID- ORCID: 0000-0002-7027-7430\n",
      "AD  - University of Copenhagen, Copenhagen, Denmark.\n",
      "FAU - Mack, Michelle\n",
      "AU  - Mack M\n",
      "AD  - Center for Ecosystem Science and Society, Northern Arizona University, Flagstaff,AZ, USA.\n",
      "FAU - Maire, Vincent\n",
      "AU  - Maire V\n",
      "AUID- ORCID: 0000-0002-3245-2568\n",
      "AD  - Universite du Quebec a Trois-Rivieres, Trois-Rivieres, Canada.\n",
      "FAU - Makela, Annikki\n",
      "AU  - Makela A\n",
      "AUID- ORCID: 0000-0001-9633-7350\n",
      "AD  - Institute of Atmospheric and Earth System Research (INAR), University ofHelsinki, Helsinki, Finland.\n",
      "FAU - Makinen, Harri\n",
      "AU  - Makinen H\n",
      "AUID- ORCID: 0000-0002-1820-6264\n",
      "AD  - Natural Resources Institute Finland, Espoo, Finland.\n",
      "FAU - Malhado, Ana Claudia Mendes\n",
      "AU  - Malhado ACM\n",
      "AUID- ORCID: 0000-0003-3621-779X\n",
      "AD  - Federal University of Alagoas, Maceio, AL, Brazil.\n",
      "FAU - Mallik, Azim\n",
      "AU  - Mallik A\n",
      "AUID- ORCID: 0000-0002-6275-6255\n",
      "AD  - Lakehead University, Thunder Bay, ON, Canada.\n",
      "FAU - Manning, Peter\n",
      "AU  - Manning P\n",
      "AUID- ORCID: 0000-0002-7940-2023\n",
      "AD  - Senckenberg Biodiversity and Climate Research Centre, Frankfurt am Main, Germany.\n",
      "FAU - Manzoni, Stefano\n",
      "AU  - Manzoni S\n",
      "AUID- ORCID: 0000-0002-5960-5712\n",
      "AD  - Department of Physical Geography, Stockholm University, Stockholm, Sweden.\n",
      "AD  - Bolin Centre for Climate Research, Stockholm, Sweden.\n",
      "FAU - Marchetti, Zuleica\n",
      "AU  - Marchetti Z\n",
      "AUID- ORCID: 0000-0002-9241-9217\n",
      "AD  - Conicet-Consejo Nacional de Investigaciones Cientificas y Tecnicas, Buenos Aires,Argentina.\n",
      "AD  - Facultad de Ingenieria y Ciencias Hidricas, Universidad Nacional del Litoral(FICH-UNL), Santa Fe, Argentina.\n",
      "FAU - Marchino, Luca\n",
      "AU  - Marchino L\n",
      "AD  - CREA - Research Centre for Forestry and Wood, Arezzo, Italy.\n",
      "FAU - Marcilio-Silva, Vinicius\n",
      "AU  - Marcilio-Silva V\n",
      "AUID- ORCID: 0000-0002-5276-2838\n",
      "AD  - Universidade Federal do Parana, Curitiba, PR, Brazil.\n",
      "FAU - Marcon, Eric\n",
      "AU  - Marcon E\n",
      "AUID- ORCID: 0000-0002-5249-321X\n",
      "AD  - Cirad, UMR EcoFoG (Agroparistech, CNRS, INRA, Universite des Antilles, Universitede la Guyane), Kourou, French Guiana, France.\n",
      "FAU - Marignani, Michela\n",
      "AU  - Marignani M\n",
      "AUID- ORCID: 0000-0002-8420-5454\n",
      "AD  - Department of Life and Environmental Sciences, Botany Division, University ofCagliari, Cagliari, Italy.\n",
      "FAU - Markesteijn, Lars\n",
      "AU  - Markesteijn L\n",
      "AUID- ORCID: 0000-0003-3046-3121\n",
      "AD  - School of Natural Sciences, Bangor University, Bangor, UK.\n",
      "FAU - Martin, Adam\n",
      "AU  - Martin A\n",
      "AUID- ORCID: 0000-0002-7207-4004\n",
      "AD  - Department of Physical and Environmental Sciences, University of TorontoScarborough, Toronto, ON, Canada.\n",
      "FAU - Martinez-Garza, Cristina\n",
      "AU  - Martinez-Garza C\n",
      "AUID- ORCID: 0000-0002-9310-564X\n",
      "AD  - Centro de Investigacion en Biodiversidad y Conservacion, Universidad Autonoma delEstado de Morelos, Morelos, Mexico.\n",
      "FAU - Martinez-Vilalta, Jordi\n",
      "AU  - Martinez-Vilalta J\n",
      "AUID- ORCID: 0000-0002-2332-7298\n",
      "AD  - CREAF, Catalonia, Spain.\n",
      "AD  - Universitat Autonoma de Barcelona, Catalonia, Spain.\n",
      "FAU - Maskova, Tereza\n",
      "AU  - Maskova T\n",
      "AUID- ORCID: 0000-0001-8386-5146\n",
      "AD  - Faculty of Sciences, Charles University, Praha, Czech Republic.\n",
      "FAU - Mason, Kelly\n",
      "AU  - Mason K\n",
      "AUID- ORCID: 0000-0003-3426-3178\n",
      "AD  - Centre for Ecology & Hydrology, Lancaster Environment Centre, Lancaster, UK.\n",
      "FAU - Mason, Norman\n",
      "AU  - Mason N\n",
      "AD  - Manaaki Whenua - Landcare Research, Hamilton, New Zealand.\n",
      "FAU - Massad, Tara Joy\n",
      "AU  - Massad TJ\n",
      "AD  - Department of Scientific Services, Gorongosa National Park, Beira, SofalaProvince, Mozambique.\n",
      "FAU - Masse, Jacynthe\n",
      "AU  - Masse J\n",
      "AD  - Institut de recherche en biologie vegetale, Montreal, QC, Canada.\n",
      "AD  - Universite de Montreal, Montreal, QC, Canada.\n",
      "FAU - Mayrose, Itay\n",
      "AU  - Mayrose I\n",
      "AUID- ORCID: 0000-0002-8460-1502\n",
      "AD  - School of Plant Sciences and Food Security, George S. Wise Faculty of LifeSciences, Tel Aviv University, Tel Aviv, Israel.\n",
      "FAU - McCarthy, James\n",
      "AU  - McCarthy J\n",
      "AUID- ORCID: 0000-0003-3060-1678\n",
      "AD  - School of Biological Sciences, The University of Queensland, Brisbane, Qld,Australia.\n",
      "AD  - CSIRO, Canberra, ACT, Australia.\n",
      "AD  - Manaaki Whenua - Landcare Research, Lincoln, New Zealand.\n",
      "FAU - McCormack, M Luke\n",
      "AU  - McCormack ML\n",
      "AUID- ORCID: 0000-0002-8300-5215\n",
      "AD  - Center For Tree Science, The Morton Arboretum, Lisle, IL, USA.\n",
      "FAU - McCulloh, Katherine\n",
      "AU  - McCulloh K\n",
      "AUID- ORCID: 0000-0003-0801-3968\n",
      "AD  - Department of Botany, University of Wisconsin-Madison, Madison, WI, USA.\n",
      "FAU - McFadden, Ian R\n",
      "AU  - McFadden IR\n",
      "AUID- ORCID: 0000-0002-4508-7272\n",
      "AD  - Department of Environmental Systems Science, ETH Zurich, Zurich, Switzerland.\n",
      "AD  - Swiss Federal Institute for Forest, Snow and Landscape Research WSL, Birmensdorf,Switzerland.\n",
      "AD  - Department of Ecology and Evolutionary Biology, University of California, LosAngeles, CA, USA.\n",
      "FAU - McGill, Brian J\n",
      "AU  - McGill BJ\n",
      "AD  - University of Maine, Orono, ME, USA.\n",
      "FAU - McPartland, Mara Y\n",
      "AU  - McPartland MY\n",
      "AUID- ORCID: 0000-0002-1242-0438\n",
      "AD  - Department of Geography, Environment & Society, University of Minnesota,Minneapolis, MN, USA.\n",
      "FAU - Medeiros, Juliana S\n",
      "AU  - Medeiros JS\n",
      "AD  - Holden Arboretum, Kirtland, Ohio, USA.\n",
      "FAU - Medlyn, Belinda\n",
      "AU  - Medlyn B\n",
      "AUID- ORCID: 0000-0001-5728-9827\n",
      "AD  - Hawkesbury Institute for the Environment, Western Sydney University, Sydney, NSW,Australia.\n",
      "FAU - Meerts, Pierre\n",
      "AU  - Meerts P\n",
      "AD  - Universite Libre de Bruxelles, Bruxelles, Belgium.\n",
      "FAU - Mehrabi, Zia\n",
      "AU  - Mehrabi Z\n",
      "AUID- ORCID: 0000-0001-9574-0420\n",
      "AD  - Institute for Resources Environment and Sustainability, University of BritishColumbia, Vancouver, BC, Canada.\n",
      "FAU - Meir, Patrick\n",
      "AU  - Meir P\n",
      "AUID- ORCID: 0000-0002-2362-0398\n",
      "AD  - Research School of Biology, The Australian National University, Canberra, ACT,Australia.\n",
      "AD  - School of Geosciences, The University of Edinburgh, Edinburgh, UK.\n",
      "FAU - Melo, Felipe P L\n",
      "AU  - Melo FPL\n",
      "AUID- ORCID: 0000-0002-1271-3214\n",
      "AD  - Universidade Federal de Pernambuco, Recife, PE, Brazil.\n",
      "FAU - Mencuccini, Maurizio\n",
      "AU  - Mencuccini M\n",
      "AUID- ORCID: 0000-0003-0840-1477\n",
      "AD  - ICREA, Barcelona, Spain.\n",
      "AD  - CREAF, Barcelona, Spain.\n",
      "FAU - Meredieu, Celine\n",
      "AU  - Meredieu C\n",
      "AUID- ORCID: 0000-0002-2950-2253\n",
      "AD  - INRA, UEFP, Cestas, France.\n",
      "FAU - Messier, Julie\n",
      "AU  - Messier J\n",
      "AUID- ORCID: 0000-0003-1637-6793\n",
      "AD  - Department of Biology, University of Waterloo, Waterloo, ON, Canada.\n",
      "FAU - Meszaros, Ilona\n",
      "AU  - Meszaros I\n",
      "AD  - Department of Botany, Faculty of Science and Technology, University of Debrecen, Debrecen, Hungary.\n",
      "FAU - Metsaranta, Juha\n",
      "AU  - Metsaranta J\n",
      "AD  - Natural Resources Canada, Canadian Forest Service, Northern Forestry Centre,Edmonton, AB, Canada.\n",
      "FAU - Michaletz, Sean T\n",
      "AU  - Michaletz ST\n",
      "AUID- ORCID: 0000-0003-2158-6525\n",
      "AD  - Department of Botany and Biodiversity Research Centre, University of BritishColumbia, Vancouver, BC, Canada.\n",
      "FAU - Michelaki, Chrysanthi\n",
      "AU  - Michelaki C\n",
      "AUID- ORCID: 0000-0001-9217-0063\n",
      "AD  - Biodiversity Conservation Laboratory, Department of Environment, University ofthe Aegean, Mytilene, Greece.\n",
      "FAU - Migalina, Svetlana\n",
      "AU  - Migalina S\n",
      "AD  - Institute Botanic Garden, Ural Branch of the Russian Academy of Sciences,Ekaterinburg, Russia.\n",
      "AD  - Tyumen State University, Tyumen, Russia.\n",
      "FAU - Milla, Ruben\n",
      "AU  - Milla R\n",
      "AUID- ORCID: 0000-0001-8912-4373\n",
      "AD  - Tulipan s/n, Mostoles-Madrid, Spain.\n",
      "FAU - Miller, Jesse E D\n",
      "AU  - Miller JED\n",
      "AUID- ORCID: 0000-0002-2316-779X\n",
      "AD  - Department of Biology, Stanford University, Stanford, CA, USA.\n",
      "FAU - Minden, Vanessa\n",
      "AU  - Minden V\n",
      "AUID- ORCID: 0000-0002-4933-5931\n",
      "AD  - Institute of Biology and Environmental Sciences, University of Oldenburg,Oldenburg, Germany.\n",
      "AD  - Department of Biology, Vrije Universiteit Brussel, Brussels, Belgium.\n",
      "FAU - Ming, Ray\n",
      "AU  - Ming R\n",
      "AUID- ORCID: 0000-0002-9417-5789\n",
      "AD  - University of Illinois at Urbana-Champaign, Urbana-Champaign, IL, USA.\n",
      "FAU - Mokany, Karel\n",
      "AU  - Mokany K\n",
      "AUID- ORCID: 0000-0003-4199-3697\n",
      "AD  - CSIRO, Canberra, ACT, Australia.\n",
      "FAU - Moles, Angela T\n",
      "AU  - Moles AT\n",
      "AUID- ORCID: 0000-0003-2041-7762\n",
      "AD  - Evolution & Ecology Research Centre, and School of Biological, Earth andEnvironmental Sciences, UNSW Sydney, Sydney, NSW, Australia.\n",
      "FAU - Molnar, Attila 5th\n",
      "AU  - Molnar A 5th\n",
      "AUID- ORCID: 0000-0001-7096-9579\n",
      "AD  - Department of Botany, University of Debrecen, Hungary.\n",
      "FAU - Molofsky, Jane\n",
      "AU  - Molofsky J\n",
      "AUID- ORCID: 0000-0001-7927-516X\n",
      "AD  - University of Vermont, Burlington, VT, USA.\n",
      "FAU - Molz, Martin\n",
      "AU  - Molz M\n",
      "AD  - Fundacao Zoobotanica do Rio Grande do Sul, Porto Allegre, Brazil.\n",
      "FAU - Montgomery, Rebecca A\n",
      "AU  - Montgomery RA\n",
      "AD  - University of Minnesota, St. Paul, MN, USA.\n",
      "FAU - Monty, Arnaud\n",
      "AU  - Monty A\n",
      "AUID- ORCID: 0000-0003-1910-6748\n",
      "AD  - Gembloux Agro-Bio Tech, Biodiversity and landscape, University of Liege, Belgium.\n",
      "FAU - Moravcova, Lenka\n",
      "AU  - Moravcova L\n",
      "AD  - Department of Invasion Ecology, Institute of Botany, Czech Academy of Sciences,Pruhonice, Czech Republic.\n",
      "FAU - Moreno-Martinez, Alvaro\n",
      "AU  - Moreno-Martinez A\n",
      "AUID- ORCID: 0000-0003-2990-7768\n",
      "AD  - Numerical Terradynamic Simulation Group (NTSG), College of Forestry andConservation, University of Montana, Missoula, USA.\n",
      "FAU - Moretti, Marco\n",
      "AU  - Moretti M\n",
      "AUID- ORCID: 0000-0002-5845-3198\n",
      "AD  - Swiss Federal Institute for Forest, Snow and Landscape Research WSL, Birmensdorf,Switzerland.\n",
      "FAU - Mori, Akira S\n",
      "AU  - Mori AS\n",
      "AUID- ORCID: 0000-0002-8422-1198\n",
      "AD  - Graduate School of Environment and Information Sciences, Yokohama NationalUniversity, Yokohama, Japan.\n",
      "FAU - Mori, Shigeta\n",
      "AU  - Mori S\n",
      "AUID- ORCID: 0000-0001-5604-2443\n",
      "AD  - Yamagata University, Yamagata, Japan.\n",
      "FAU - Morris, Dave\n",
      "AU  - Morris D\n",
      "AUID- ORCID: 0000-0002-5739-0594\n",
      "AD  - Ontario Ministry of Natural Resources and Forestry, Centre for Northern ForestEcosystem Research, Thunder Bay, ON, Canada.\n",
      "FAU - Morrison, Jane\n",
      "AU  - Morrison J\n",
      "AUID- ORCID: 0000-0002-4055-4679\n",
      "AD  - Universitat Politecnica de Catalunya, Castelldefels, Spain.\n",
      "FAU - Mucina, Ladislav\n",
      "AU  - Mucina L\n",
      "AUID- ORCID: 0000-0003-0317-8886\n",
      "AD  - Harry Butler Institute, Murdoch University, Perth, WA, Australia.\n",
      "AD  - Dept of Geography & Environmental Studies, Stellenbosch University, Matieland,Stellenbosch, South Africa.\n",
      "FAU - Mueller, Sandra\n",
      "AU  - Mueller S\n",
      "AD  - Geobotany, Faculty of Biology, University of Freiburg, Freiburg im Breisgau,Germany.\n",
      "FAU - Muir, Christopher D\n",
      "AU  - Muir CD\n",
      "AUID- ORCID: 0000-0003-2555-3878\n",
      "AD  - Department of Botany, University of Hawai'i, Honolulu, HI, USA.\n",
      "FAU - Muller, Sandra Cristina\n",
      "AU  - Muller SC\n",
      "AUID- ORCID: 0000-0002-6316-2897\n",
      "AD  - Laboratorio de Ecologia Vegetal, Departamento de Ecologia, Programa dePos-Graduacao em Ecologia, Universidade Federal do Rio Grande do Sul, PortoAlegre, RS, Brazil.\n",
      "FAU - Munoz, Francois\n",
      "AU  - Munoz F\n",
      "AUID- ORCID: 0000-0001-8776-4705\n",
      "AD  - Laboratoire d'Ecologie Alpine, Universite Grenoble-Alpes, Grenoble Cedex 9,France.\n",
      "AD  - French Institute of Pondicherry, Puducherry, India.\n",
      "FAU - Myers-Smith, Isla H\n",
      "AU  - Myers-Smith IH\n",
      "AUID- ORCID: 0000-0002-8417-6112\n",
      "AD  - School of Geosciences, The University of Edinburgh, Edinburgh, UK.\n",
      "FAU - Myster, Randall W\n",
      "AU  - Myster RW\n",
      "AD  - Biology Department, Oklahoma State University, Oklahoma, OK, USA.\n",
      "FAU - Nagano, Masahiro\n",
      "AU  - Nagano M\n",
      "AD  - Osaka City University, Osaka, Japan.\n",
      "FAU - Naidu, Shawna\n",
      "AU  - Naidu S\n",
      "AD  - University of Illinois at Urbana-Champaign, Urbana-Champaign, IL, USA.\n",
      "FAU - Narayanan, Ayyappan\n",
      "AU  - Narayanan A\n",
      "AD  - Department of Ecology, French Institute of Pondicherry, Pondicherry, India.\n",
      "FAU - Natesan, Balachandran\n",
      "AU  - Natesan B\n",
      "AUID- ORCID: 0000-0001-6396-3865\n",
      "AD  - Department of Ecology, French Institute of Pondicherry, Pondicherry, India.\n",
      "FAU - Negoita, Luka\n",
      "AU  - Negoita L\n",
      "AUID- ORCID: 0000-0003-3225-7575\n",
      "AD  - Galapagos Verde 2050, Charles Darwin Foundation, Charles Darwin Research Station,Galapagos, Ecuador.\n",
      "FAU - Nelson, Andrew S\n",
      "AU  - Nelson AS\n",
      "AUID- ORCID: 0000-0001-8715-7825\n",
      "AD  - Forest, Rangeland, and Fire Sciences Department, University of Idaho, Moscow, ID,USA.\n",
      "FAU - Neuschulz, Eike Lena\n",
      "AU  - Neuschulz EL\n",
      "AUID- ORCID: 0000-0001-7526-2580\n",
      "AD  - Senckenberg Biodiversity and Climate Research Centre, Frankfurt am Main, Germany.\n",
      "FAU - Ni, Jian\n",
      "AU  - Ni J\n",
      "AUID- ORCID: 0000-0001-5411-7050\n",
      "AD  - College of Chemistry and Life Sciences, Zhejiang Normal University, Jinhua,China.\n",
      "FAU - Niedrist, Georg\n",
      "AU  - Niedrist G\n",
      "AUID- ORCID: 0000-0002-7511-6273\n",
      "AD  - Eurac Research, Institute for Alpine Environment, Bozen-Bolzano, Italy.\n",
      "FAU - Nieto, Jhon\n",
      "AU  - Nieto J\n",
      "AD  - Instituto de Investigacion de Recursos Biologicos Alexander von Humboldt, Bogota,Colombia.\n",
      "AD  - Universidad Distrital Francisco Jose de Caldas, Bogota, Colombia.\n",
      "FAU - Niinemets, Ulo\n",
      "AU  - Niinemets U\n",
      "AUID- ORCID: 0000-0002-3078-2192\n",
      "AD  - Estonian University of Life Sciences, Tartu, Estonia.\n",
      "FAU - Nolan, Rachael\n",
      "AU  - Nolan R\n",
      "AUID- ORCID: 0000-0001-9277-5142\n",
      "AD  - Hawkesbury Institute for the Environment, Western Sydney University, Sydney, NSW,Australia.\n",
      "FAU - Nottebrock, Henning\n",
      "AU  - Nottebrock H\n",
      "AUID- ORCID: 0000-0003-2238-2700\n",
      "AD  - Plant Ecology, University of Bayreuth, Bayreuth, Germany.\n",
      "FAU - Nouvellon, Yann\n",
      "AU  - Nouvellon Y\n",
      "AUID- ORCID: 0000-0003-1920-3847\n",
      "AD  - CIRAD, UMR Eco&Sols, Montpellier, France.\n",
      "AD  - Eco&Sols, CIRAD, INRA, IRD, SupAgro, University of Montpellier, Montpellier,France.\n",
      "FAU - Novakovskiy, Alexander\n",
      "AU  - Novakovskiy A\n",
      "AUID- ORCID: 0000-0003-4105-7436\n",
      "AD  - Institute of Biology of Komi Science Centre of the Ural Branch of the RussianAcademy of Sciences, Syktyvkar, Komi Republic, Russia.\n",
      "CN  - Nutrient Network\n",
      "FAU - Nystuen, Kristin Odden\n",
      "AU  - Nystuen KO\n",
      "AUID- ORCID: 0000-0003-1027-469X\n",
      "AD  - Faculty of Biosciences and Aquaculture, NORD University, Steinkjer, Norway.\n",
      "AD  - Department of Biology, Norwegian University of Science and Technology, NTNU,Trondheim, Norway.\n",
      "FAU - O'Grady, Anthony\n",
      "AU  - O'Grady A\n",
      "AD  - CSIRO Land and Water, Hobart Tas, Tas., Australia.\n",
      "FAU - O'Hara, Kevin\n",
      "AU  - O'Hara K\n",
      "AUID- ORCID: 0000-0002-3800-9188\n",
      "AD  - University of California at Berkeley, Berkeley, CA, USA.\n",
      "FAU - O'Reilly-Nugent, Andrew\n",
      "AU  - O'Reilly-Nugent A\n",
      "AUID- ORCID: 0000-0003-2071-6279\n",
      "AD  - Institute of Applied Ecology, University of Canberra, Canberra, ACT, Australia.\n",
      "FAU - Oakley, Simon\n",
      "AU  - Oakley S\n",
      "AUID- ORCID: 0000-0002-5757-7420\n",
      "AD  - Centre for Ecology & Hydrology, Lancaster Environment Centre, Lancaster, UK.\n",
      "FAU - Oberhuber, Walter\n",
      "AU  - Oberhuber W\n",
      "AUID- ORCID: 0000-0002-5197-7044\n",
      "AD  - University of Innsbruck, Innsbruck, Austria.\n",
      "FAU - Ohtsuka, Toshiyuki\n",
      "AU  - Ohtsuka T\n",
      "AUID- ORCID: 0000-0002-9032-7933\n",
      "AD  - River Basin Research Center, Gifu University, Gifu, Gifu, Japan.\n",
      "FAU - Oliveira, Ricardo\n",
      "AU  - Oliveira R\n",
      "AUID- ORCID: 0000-0001-8044-3458\n",
      "AD  - Departamento de Botanica, Universidade Federal do Parana, Curitiba, Brazil.\n",
      "FAU - Ollerer, Kinga\n",
      "AU  - Ollerer K\n",
      "AUID- ORCID: 0000-0003-3142-0000\n",
      "AD  - Institute of Biology Bucharest, Romanian Academy, Bucharest, Romania.\n",
      "AD  - Institute of Ecology and Botany, MTA Centre for Ecological Research, Vacratot,Hungary.\n",
      "FAU - Olson, Mark E\n",
      "AU  - Olson ME\n",
      "AUID- ORCID: 0000-0003-3715-4567\n",
      "AD  - Instituto de Biologia, Tercer Circuito s/n de Ciudad Universitaria, Mexico City, Mexico.\n",
      "AD  - Universidad Nacional Autonoma de Mexico, Coyoacan, Mexico.\n",
      "FAU - Onipchenko, Vladimir\n",
      "AU  - Onipchenko V\n",
      "AUID- ORCID: 0000-0002-1626-1171\n",
      "AD  - Department of Ecology and Plant Geography, Faculty of Biology, Moscow LomonosovState University, Moscow, Russia.\n",
      "FAU - Onoda, Yusuke\n",
      "AU  - Onoda Y\n",
      "AUID- ORCID: 0000-0001-6245-2342\n",
      "AD  - Kyoto University, Kyoto, Japan.\n",
      "FAU - Onstein, Renske E\n",
      "AU  - Onstein RE\n",
      "AUID- ORCID: 0000-0002-2295-3510\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "FAU - Ordonez, Jenny C\n",
      "AU  - Ordonez JC\n",
      "AD  - Facultad de Ingenieria Agroindustrial, Universidad de las Americas, Quito,Equador.\n",
      "FAU - Osada, Noriyuki\n",
      "AU  - Osada N\n",
      "AD  - Meijo University, Nagoya, Aichi, Japan.\n",
      "FAU - Ostonen, Ivika\n",
      "AU  - Ostonen I\n",
      "AUID- ORCID: 0000-0001-9043-6083\n",
      "AD  - Institute of Ecology and Earth Sciences, University of Tartu, Tartu, Estonia.\n",
      "FAU - Ottaviani, Gianluigi\n",
      "AU  - Ottaviani G\n",
      "AUID- ORCID: 0000-0003-3027-4638\n",
      "AD  - Institute of Botany, Czech Academy of Sciences, Trebon, Czech Republic.\n",
      "FAU - Otto, Sarah\n",
      "AU  - Otto S\n",
      "AUID- ORCID: 0000-0003-3042-0818\n",
      "AD  - Department of Zoology, University of British Columbia, Vancouver, BC, Canada.\n",
      "FAU - Overbeck, Gerhard E\n",
      "AU  - Overbeck GE\n",
      "AUID- ORCID: 0000-0002-8716-5136\n",
      "AD  - Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil.\n",
      "FAU - Ozinga, Wim A\n",
      "AU  - Ozinga WA\n",
      "AUID- ORCID: 0000-0002-6369-7859\n",
      "AD  - Wageningen Environmental Research, Wageningen, The Netherlands.\n",
      "FAU - Pahl, Anna T\n",
      "AU  - Pahl AT\n",
      "AD  - Restoration Ecology, Technische Universitat Munchen, Munchen, Germany.\n",
      "FAU - Paine, C E Timothy\n",
      "AU  - Paine CET\n",
      "AUID- ORCID: 0000-0001-8705-3719\n",
      "AD  - University of New England, Armidale, NSW, Australia.\n",
      "FAU - Pakeman, Robin J\n",
      "AU  - Pakeman RJ\n",
      "AUID- ORCID: 0000-0001-6248-4133\n",
      "AD  - The James Hutton Institute, Aberdeen, UK.\n",
      "FAU - Papageorgiou, Aristotelis C\n",
      "AU  - Papageorgiou AC\n",
      "AUID- ORCID: 0000-0001-6657-7820\n",
      "AD  - Department for Molecular Biology and Genetics, Democritus University of Thrace,Alexandroupolis, Greece.\n",
      "FAU - Parfionova, Evgeniya\n",
      "AU  - Parfionova E\n",
      "AD  - Samara National Research University, Samara, Russia.\n",
      "FAU - Partel, Meelis\n",
      "AU  - Partel M\n",
      "AUID- ORCID: 0000-0002-5874-0138\n",
      "AD  - University of Tartu, Tartu, Estonia.\n",
      "FAU - Patacca, Marco\n",
      "AU  - Patacca M\n",
      "AD  - Wageningen University & Research, Wageningen, The Netherlands.\n",
      "FAU - Paula, Susana\n",
      "AU  - Paula S\n",
      "AUID- ORCID: 0000-0001-5405-6155\n",
      "AD  - Instituto de Ciencias Ambientales y Evolutivas, Universidad Austral de Chile,Valdivia, Chile.\n",
      "FAU - Paule, Juraj\n",
      "AU  - Paule J\n",
      "AUID- ORCID: 0000-0001-5375-7689\n",
      "AD  - Department of Botany and Molecular Evolution, Senckenberg Research Institute and Natural History Museum, Frankfurt am Main, Germany.\n",
      "FAU - Pauli, Harald\n",
      "AU  - Pauli H\n",
      "AUID- ORCID: 0000-0002-9842-9934\n",
      "AD  - GLORIA-Coordination, Institute for Interdisciplinary Mountain Research, Austrian Academy of Sciences & Department of Integrative Biology and BiodiversityResearch, University of Natural Resources and Life Sciences Vienna, Vienna,Austria.\n",
      "FAU - Pausas, Juli G\n",
      "AU  - Pausas JG\n",
      "AUID- ORCID: 0000-0003-3533-5786\n",
      "AD  - Desertification Research Center (CIDE-CSIC), Valencia, Spain.\n",
      "FAU - Peco, Begona\n",
      "AU  - Peco B\n",
      "AUID- ORCID: 0000-0003-2149-1438\n",
      "AD  - Departamento de Ecologia, Centro de Investigacion en Biodiversidad y CambioGlobal (CIBC), Universidad Autonoma de Madrid, Madrid, Spain.\n",
      "FAU - Penuelas, Josep\n",
      "AU  - Penuelas J\n",
      "AUID- ORCID: 0000-0002-7215-0150\n",
      "AD  - CREAF, Catalonia, Spain.\n",
      "AD  - Global Ecology Unit CREAF-CSIC, Universitat Autonoma de Barcelona, Barcelona,Spain.\n",
      "FAU - Perea, Antonio\n",
      "AU  - Perea A\n",
      "AUID- ORCID: 0000-0001-8351-9358\n",
      "AD  - University of Jaen, Jaen, Spain.\n",
      "FAU - Peri, Pablo Luis\n",
      "AU  - Peri PL\n",
      "AUID- ORCID: 0000-0002-5398-4408\n",
      "AD  - Instituto Nacional de Tecnologia Agropecuaria (INTA), Rio Gallegos, Santa Cruz,Argentina.\n",
      "AD  - Universidad Nacional de la Patagonia Austral (UNPA), CONICET, Rio Gallegos,Argentina.\n",
      "FAU - Petisco-Souza, Ana Carolina\n",
      "AU  - Petisco-Souza AC\n",
      "AUID- ORCID: 0000-0002-2393-6210\n",
      "AD  - Pos-graduacao em Ecologia e Conservacao, Universidade Federal do Parana,Curitiba, PR, Brazil.\n",
      "FAU - Petraglia, Alessandro\n",
      "AU  - Petraglia A\n",
      "AUID- ORCID: 0000-0003-4632-2251\n",
      "AD  - Department of Chemistry, Life Sciences and Environmental Sustainability,University of Parma, Parma, Italy.\n",
      "FAU - Petritan, Any Mary\n",
      "AU  - Petritan AM\n",
      "AUID- ORCID: 0000-0003-3683-1108\n",
      "AD  - National Institute for Research-Development in Forestry, Voluntari, Romania.\n",
      "FAU - Phillips, Oliver L\n",
      "AU  - Phillips OL\n",
      "AUID- ORCID: 0000-0002-8993-6168\n",
      "AD  - School of Geography, University of Leeds, Leeds, UK.\n",
      "FAU - Pierce, Simon\n",
      "AU  - Pierce S\n",
      "AUID- ORCID: 0000-0003-1182-987X\n",
      "AD  - Department of Agricultural and Environmental Sciences (DiSAA), University ofMilan, Milano, Italy.\n",
      "FAU - Pillar, Valerio D\n",
      "AU  - Pillar VD\n",
      "AUID- ORCID: 0000-0001-6408-2891\n",
      "AD  - Department of Ecology, Universidade Federal do Rio Grande do Sul, Porto Alegre,RS, Brazil.\n",
      "FAU - Pisek, Jan\n",
      "AU  - Pisek J\n",
      "AUID- ORCID: 0000-0003-0396-2072\n",
      "AD  - Tartu Observatory, University of Tartu, Tartumaa, Estonia.\n",
      "FAU - Pomogaybin, Alexandr\n",
      "AU  - Pomogaybin A\n",
      "AD  - Botanical Garden of the Samara University, Samara, Russia.\n",
      "FAU - Poorter, Hendrik\n",
      "AU  - Poorter H\n",
      "AUID- ORCID: 0000-0001-9900-2433\n",
      "AD  - Department of Biological Sciences, Macquarie University, Sydney, NSW, Australia.\n",
      "AD  - Plant Sciences (IBG2), Forschungszentrum Julich GmbH, Julich, Germany.\n",
      "FAU - Portsmuth, Angelika\n",
      "AU  - Portsmuth A\n",
      "AUID- ORCID: 0000-0002-5100-2888\n",
      "AD  - Institute of Ecology, Tallinn University, Tallinn, Estonia.\n",
      "FAU - Poschlod, Peter\n",
      "AU  - Poschlod P\n",
      "AUID- ORCID: 0000-0003-4473-7656\n",
      "AD  - Ecology and Conservation Biology, Institute of Plant Sciences, University ofRegensburg, Regensburg, Germany.\n",
      "FAU - Potvin, Catherine\n",
      "AU  - Potvin C\n",
      "AD  - McGill University, Montreal, QC, Canada.\n",
      "FAU - Pounds, Devon\n",
      "AU  - Pounds D\n",
      "AD  - Morton Arboretum, Lisle, IL, USA.\n",
      "FAU - Powell, A Shafer\n",
      "AU  - Powell AS\n",
      "AUID- ORCID: 0000-0002-9622-0061\n",
      "AD  - Environmental Sciences Division & Climate Change Science Institute, Oak RidgeNational Laboratory, Oak Ridge, TN, USA.\n",
      "FAU - Power, Sally A\n",
      "AU  - Power SA\n",
      "AUID- ORCID: 0000-0002-2723-8671\n",
      "AD  - Hawkesbury Institute for the Environment, Western Sydney University, Sydney, NSW,Australia.\n",
      "FAU - Prinzing, Andreas\n",
      "AU  - Prinzing A\n",
      "AD  - Research Unit ECOBIO - Ecosystemes, Biodiversite, Evolution, Universite Rennes1/CNRS, Rennes, France.\n",
      "FAU - Puglielli, Giacomo\n",
      "AU  - Puglielli G\n",
      "AUID- ORCID: 0000-0003-0085-4535\n",
      "AD  - Estonian University of Life Sciences, Tartu, Estonia.\n",
      "FAU - Pysek, Petr\n",
      "AU  - Pysek P\n",
      "AUID- ORCID: 0000-0001-8500-442X\n",
      "AD  - Department of Invasion Ecology, Institute of Botany, Czech Academy of Sciences,Pruhonice, Czech Republic.\n",
      "AD  - Department of Ecology, Faculty of Science, Charles University, Prague, CzechRepublic.\n",
      "FAU - Raevel, Valerie\n",
      "AU  - Raevel V\n",
      "AUID- ORCID: 0000-0002-9631-114X\n",
      "AD  - AMAP, CIRAD, CNRS, IRD, INRA, Universite de Montpellier, Montpellier, France.\n",
      "AD  - French Institute of Pondicherry, Puducherry, India.\n",
      "AD  - CEFE, CNRS, EPHE, Universite de Montpellier, Universite Paul Valery, Montpellier,France.\n",
      "FAU - Rammig, Anja\n",
      "AU  - Rammig A\n",
      "AD  - TUM School of Life Sciences Weihenstephan, Technical University of Munich,Freising, Germany.\n",
      "FAU - Ransijn, Johannes\n",
      "AU  - Ransijn J\n",
      "AD  - University of Copenhagen, Copenhagen, Denmark.\n",
      "FAU - Ray, Courtenay A\n",
      "AU  - Ray CA\n",
      "AUID- ORCID: 0000-0002-2276-5915\n",
      "AD  - Rocky Mountain Biological Laboratory, Crested Butte, CO, USA.\n",
      "AD  - School of Life Sciences, Arizona State University, Tempe, AZ, USA.\n",
      "FAU - Reich, Peter B\n",
      "AU  - Reich PB\n",
      "AUID- ORCID: 0000-0003-4424-662X\n",
      "AD  - Department of Forest Resources, University of Minnesota, St. Paul, MN, USA.\n",
      "AD  - Hawkesbury Institute for the Environment, Western Sydney University, Sydney, NSW,Australia.\n",
      "FAU - Reichstein, Markus\n",
      "AU  - Reichstein M\n",
      "AD  - Max Planck Institute for Biogeochemistry, Jena, Germany.\n",
      "FAU - Reid, Douglas E B\n",
      "AU  - Reid DEB\n",
      "AUID- ORCID: 0000-0002-0967-5122\n",
      "AD  - Ontario Ministry of Natural Resources and Forestry, Centre for Northern ForestEcosystem Research, Thunder Bay, ON, Canada.\n",
      "FAU - Rejou-Mechain, Maxime\n",
      "AU  - Rejou-Mechain M\n",
      "AUID- ORCID: 0000-0003-2824-267X\n",
      "AD  - AMAP, CIRAD, CNRS, IRD, INRA, Universite de Montpellier, Montpellier, France.\n",
      "FAU - de Dios, Victor Resco\n",
      "AU  - de Dios VR\n",
      "AUID- ORCID: 0000-0002-5721-1656\n",
      "AD  - School of Life Science and Engineering, Southwest University of Science andTechnology, Mianyang, China.\n",
      "AD  - Department of Crop and Forest Sciences & Agrotecnio Center, Universitat deLleida, Lleida, Spain.\n",
      "FAU - Ribeiro, Sabina\n",
      "AU  - Ribeiro S\n",
      "AUID- ORCID: 0000-0002-4504-3050\n",
      "AD  - Universidade Federal do Acre, Rio Branco, AC, Brazil.\n",
      "FAU - Richardson, Sarah\n",
      "AU  - Richardson S\n",
      "AUID- ORCID: 0000-0002-4097-0381\n",
      "AD  - Manaaki Whenua-Landcare Research, Lincoln, New Zealand.\n",
      "FAU - Riibak, Kersti\n",
      "AU  - Riibak K\n",
      "AD  - University of Tartu, Tartu, Estonia.\n",
      "FAU - Rillig, Matthias C\n",
      "AU  - Rillig MC\n",
      "AUID- ORCID: 0000-0003-3541-7853\n",
      "AD  - Berlin-Brandenburg Institute of Advanced Biodiversity Research (BBIB), Berlin,Germany.\n",
      "AD  - Freie Universitat Berlin, Berlin, Germany.\n",
      "FAU - Riviera, Fiamma\n",
      "AU  - Riviera F\n",
      "AUID- ORCID: 0000-0003-2505-8180\n",
      "AD  - The University of Western Australia, Crawley, WA, Australia.\n",
      "FAU - Robert, Elisabeth M R\n",
      "AU  - Robert EMR\n",
      "AUID- ORCID: 0000-0002-3611-7265\n",
      "AD  - Centre for Ecological Research and Forestry Applications (CREAF), Cerdanyola del Valles, Spain.\n",
      "AD  - Ecology and Biodiversity, Vrije Universiteit Brussel, Brussels, Belgium.\n",
      "AD  - Laboratory of Wood Biology and Xylarium, Royal Museum for Central-Africa (RMCA), Tervuren, Belgium.\n",
      "FAU - Roberts, Scott\n",
      "AU  - Roberts S\n",
      "AD  - Department of Forestry, Mississippi State University, Starkville, MS, USA.\n",
      "FAU - Robroek, Bjorn\n",
      "AU  - Robroek B\n",
      "AUID- ORCID: 0000-0002-6714-0652\n",
      "AD  - School of Biological Sciences, University of Southampton, Southampton, UK.\n",
      "AD  - Aquatic Ecology and Environmental Biology, Radboud University Nijmegen, Nijmegen,The Netherlands.\n",
      "FAU - Roddy, Adam\n",
      "AU  - Roddy A\n",
      "AUID- ORCID: 0000-0002-4423-8729\n",
      "AD  - School of Forestry & Environmental Studies, Yale University, New Haven, CT, USA.\n",
      "FAU - Rodrigues, Arthur Vinicius\n",
      "AU  - Rodrigues AV\n",
      "AUID- ORCID: 0000-0003-2656-558X\n",
      "AD  - Programa de pos-graduacao em Ecologia, Universidade Federal do Rio Grande do Sul,Porto Alegre, RS, Brasil.\n",
      "FAU - Rogers, Alistair\n",
      "AU  - Rogers A\n",
      "AUID- ORCID: 0000-0001-9262-7430\n",
      "AD  - Environmental and Climate Sciences Department, Brookhaven National Laboratory,NY, Upton, USA.\n",
      "FAU - Rollinson, Emily\n",
      "AU  - Rollinson E\n",
      "AUID- ORCID: 0000-0003-3428-2352\n",
      "AD  - Department of Biological Sciences, East Stroudsburg University, East Stroudsburg,PA, USA.\n",
      "FAU - Rolo, Victor\n",
      "AU  - Rolo V\n",
      "AUID- ORCID: 0000-0001-5854-9512\n",
      "AD  - Forest Research Group, INDEHESA, University of Extremadura, Plasencia, Spain.\n",
      "FAU - Romermann, Christine\n",
      "AU  - Romermann C\n",
      "AUID- ORCID: 0000-0003-3471-0951\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "AD  - Institute of Ecology and Evolution, Friedrich Schiller University Jena, Jena,Germany.\n",
      "FAU - Ronzhina, Dina\n",
      "AU  - Ronzhina D\n",
      "AUID- ORCID: 0000-0003-0854-0223\n",
      "AD  - Institute Botanic Garden, Ural Branch of the Russian Academy of Sciences,Ekaterinburg, Russia.\n",
      "AD  - Tyumen State University, Tyumen, Russia.\n",
      "FAU - Roscher, Christiane\n",
      "AU  - Roscher C\n",
      "AUID- ORCID: 0000-0001-9301-7909\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "AD  - Physiological Diversity, Helmholtz Centre for Environmental Research (UFZ),Leipzig, Germany.\n",
      "FAU - Rosell, Julieta A\n",
      "AU  - Rosell JA\n",
      "AUID- ORCID: 0000-0001-5741-8027\n",
      "AD  - Laboratorio Nacional de Ciencias de la Sostenibilidad, Instituto de Ecologia,Universidad Nacional Autonoma de Mexico, Ciudad Universitaria, Mexico City,Mexico.\n",
      "FAU - Rosenfield, Milena Fermina\n",
      "AU  - Rosenfield MF\n",
      "AUID- ORCID: 0000-0002-1799-2822\n",
      "AD  - School of Environmental Sciences, University of Guelph, Guelph, ON, Canada.\n",
      "FAU - Rossi, Christian\n",
      "AU  - Rossi C\n",
      "AD  - Remote Sensing Laboratories, Department of Geography, University of Zurich,Zurich, Switzerland.\n",
      "AD  - Research Unit Community Ecology, Swiss Federal Institute for Forest, Snow andLandscape Research WSL, Birmensdorf, Switzerland.\n",
      "AD  - Department of Research and Geoinformation, Swiss National Park, ChastePlanta-Wildenberg, Zernez, Switzerland.\n",
      "FAU - Roy, David B\n",
      "AU  - Roy DB\n",
      "AUID- ORCID: 0000-0002-5147-0331\n",
      "AD  - Centre for Ecology & Hydrology (CEH), Wallingford, Oxfordshire, UK.\n",
      "FAU - Royer-Tardif, Samuel\n",
      "AU  - Royer-Tardif S\n",
      "AUID- ORCID: 0000-0001-7809-6485\n",
      "AD  - Natural Resources Canada, Canadian Forest Service, Quebec, QC, Canada.\n",
      "FAU - Ruger, Nadja\n",
      "AU  - Ruger N\n",
      "AUID- ORCID: 0000-0003-2371-4172\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "AD  - Smithsonian Tropical Research Institute, Balboa, Ancon, Republic of Panama.\n",
      "FAU - Ruiz-Peinado, Ricardo\n",
      "AU  - Ruiz-Peinado R\n",
      "AUID- ORCID: 0000-0003-0126-1651\n",
      "AD  - Departamento de Dinamica y Gestion Forestal, INIA-CIFOR, Madrid, Spain.\n",
      "AD  - Sustainable Forest Management Research Institute, University of Valladolid-INIA, Madrid, Spain.\n",
      "FAU - Rumpf, Sabine B\n",
      "AU  - Rumpf SB\n",
      "AUID- ORCID: 0000-0001-5909-9568\n",
      "AD  - Department of Botany and Biodiversity Research, University of Vienna, Vienna,Austria.\n",
      "AD  - Department of Ecology and Evolution, University of Lausanne, Lausanne,Switzerland.\n",
      "FAU - Rusch, Graciela M\n",
      "AU  - Rusch GM\n",
      "AD  - Norwegian Institute for Nature Research, Trondheim, Norway.\n",
      "FAU - Ryo, Masahiro\n",
      "AU  - Ryo M\n",
      "AUID- ORCID: 0000-0002-5271-3446\n",
      "AD  - Berlin-Brandenburg Institute of Advanced Biodiversity Research (BBIB), Berlin,Germany.\n",
      "AD  - Freie Universitat Berlin, Berlin, Germany.\n",
      "FAU - Sack, Lawren\n",
      "AU  - Sack L\n",
      "AUID- ORCID: 0000-0002-7009-7202\n",
      "AD  - Department of Ecology and Evolutionary Biology, University of California, LosAngeles, CA, USA.\n",
      "FAU - Saldana, Angela\n",
      "AU  - Saldana A\n",
      "AD  - Universidad Nacional Autonoma de Mexico, Coyoacan, Mexico.\n",
      "FAU - Salgado-Negret, Beatriz\n",
      "AU  - Salgado-Negret B\n",
      "AUID- ORCID: 0000-0002-3103-9878\n",
      "AD  - Departamento de Biologia, Universidad Nacional de Colombia, Bogota, Colombia.\n",
      "FAU - Salguero-Gomez, Roberto\n",
      "AU  - Salguero-Gomez R\n",
      "AUID- ORCID: 0000-0002-6085-4433\n",
      "AD  - Oxford University, Oxford, UK.\n",
      "FAU - Santa-Regina, Ignacio\n",
      "AU  - Santa-Regina I\n",
      "AUID- ORCID: 0000-0002-7312-3605\n",
      "AD  - Instituto de Recursos Naturales y Agrobiologia de Salamanca (IRNASA-CSIC),Salamanca, Spain.\n",
      "FAU - Santacruz-Garcia, Ana Carolina\n",
      "AU  - Santacruz-Garcia AC\n",
      "AUID- ORCID: 0000-0002-3340-4859\n",
      "AD  - Conicet-Consejo Nacional de Investigaciones Cientificas y Tecnicas, Buenos Aires,Argentina.\n",
      "AD  - Facultad de Ciencias Forestales, Universidad Nacional de Santiago del Estero,Santiago del Estero, Argentina.\n",
      "FAU - Santos, Joaquim\n",
      "AU  - Santos J\n",
      "AUID- ORCID: 0000-0002-2160-4968\n",
      "AD  - Centre for Functional Ecology, Departamento de Ciencias da Vida, Universidade de Coimbra, Coimbra, Portugal.\n",
      "FAU - Sardans, Jordi\n",
      "AU  - Sardans J\n",
      "AUID- ORCID: 0000-0003-2478-0219\n",
      "AD  - Global Ecology Unit CREAF-CSIC, Universitat Autonoma de Barcelona, Barcelona,Spain.\n",
      "FAU - Schamp, Brandon\n",
      "AU  - Schamp B\n",
      "AUID- ORCID: 0000-0002-6885-2029\n",
      "AD  - Algoma University, Sault Ste. Marie, ON, Canada.\n",
      "FAU - Scherer-Lorenzen, Michael\n",
      "AU  - Scherer-Lorenzen M\n",
      "AUID- ORCID: 0000-0001-9566-590X\n",
      "AD  - Geobotany, Faculty of Biology, University of Freiburg, Freiburg im Breisgau,Germany.\n",
      "FAU - Schleuning, Matthias\n",
      "AU  - Schleuning M\n",
      "AUID- ORCID: 0000-0001-9426-045X\n",
      "AD  - Senckenberg Biodiversity and Climate Research Centre, Frankfurt am Main, Germany.\n",
      "FAU - Schmid, Bernhard\n",
      "AU  - Schmid B\n",
      "AUID- ORCID: 0000-0002-8430-3214\n",
      "AD  - Department of Geography, University of Zurich, Zurich, Switzerland.\n",
      "FAU - Schmidt, Marco\n",
      "AU  - Schmidt M\n",
      "AUID- ORCID: 0000-0001-6087-6117\n",
      "AD  - Senckenberg Biodiversitat und Klima Forschungszentrum (SBiK-F), Frankfurt,Germany.\n",
      "AD  - Palmengarten der Stadt Frankfurt am Main, Frankfurt, Germany.\n",
      "FAU - Schmitt, Sylvain\n",
      "AU  - Schmitt S\n",
      "AUID- ORCID: 0000-0001-7759-7106\n",
      "AD  - Univ. Bordeaux, INRAE, BIOGECO, Cestas, France.\n",
      "FAU - Schneider, Julio V\n",
      "AU  - Schneider JV\n",
      "AUID- ORCID: 0000-0002-9823-6569\n",
      "AD  - Department of Botany and Molecular Evolution, Senckenberg Research Institute and Natural History Museum, Frankfurt am Main, Germany.\n",
      "AD  - Entomology III, Senckenberg Research Institute and Natural History Museum,Frankfurt am Main, Germany.\n",
      "FAU - Schowanek, Simon D\n",
      "AU  - Schowanek SD\n",
      "AUID- ORCID: 0000-0001-6036-7507\n",
      "AD  - Section for Ecoinformatics and Biodiversity, Department of Bioscience, AarhusUniversity, Aarhus, Denmark.\n",
      "AD  - Center for Biodiversity Dynamics in a Changing World (BIOCHANGE), Department ofBioscience, Aarhus University, Aarhus, Denmark.\n",
      "FAU - Schrader, Julian\n",
      "AU  - Schrader J\n",
      "AUID- ORCID: 0000-0002-8392-211X\n",
      "AD  - Biodiversity, Macroecology and Biogeography, University of Goettingen, Gottingen,Germany.\n",
      "FAU - Schrodt, Franziska\n",
      "AU  - Schrodt F\n",
      "AUID- ORCID: 0000-0001-9053-8872\n",
      "AD  - School of Geography, University of Nottingham, Nottingham, UK.\n",
      "FAU - Schuldt, Bernhard\n",
      "AU  - Schuldt B\n",
      "AD  - Julius-von-Sachs-Institute for Biological Sciences, Chair of Ecophysiology andVegetation Ecology, University of Wuerzburg, Wuerzburg, Germany.\n",
      "FAU - Schurr, Frank\n",
      "AU  - Schurr F\n",
      "AUID- ORCID: 0000-0002-4518-0602\n",
      "AD  - Institute of Landscape and Plant Ecology, University of Hohenheim, Stuttgart,Germany.\n",
      "FAU - Selaya Garvizu, Galia\n",
      "AU  - Selaya Garvizu G\n",
      "AUID- ORCID: 0000-0003-1920-7450\n",
      "AD  - Herencia, Santa Cruz, Bolivia.\n",
      "FAU - Semchenko, Marina\n",
      "AU  - Semchenko M\n",
      "AUID- ORCID: 0000-0001-6196-3562\n",
      "AD  - Department of Earth and Environmental Sciences, University of Manchester,Manchester, UK.\n",
      "FAU - Seymour, Colleen\n",
      "AU  - Seymour C\n",
      "AD  - South African National Biodiversity Institute, Pretoria, South Africa.\n",
      "FAU - Sfair, Julia C\n",
      "AU  - Sfair JC\n",
      "AUID- ORCID: 0000-0003-3823-7233\n",
      "AD  - Federal University of Pernambuco, Recife, PE, Brazil.\n",
      "FAU - Sharpe, Joanne M\n",
      "AU  - Sharpe JM\n",
      "AD  - Sharplex Services, Edgecomb, ME, USA.\n",
      "FAU - Sheppard, Christine S\n",
      "AU  - Sheppard CS\n",
      "AUID- ORCID: 0000-0002-7847-2310\n",
      "AD  - Institute of Landscape and Plant Ecology, University of Hohenheim, Stuttgart,Germany.\n",
      "FAU - Sheremetiev, Serge\n",
      "AU  - Sheremetiev S\n",
      "AUID- ORCID: 0000-0002-0318-6766\n",
      "AD  - Komarov Botanical Institute RAS, St. Petersburg, Russia.\n",
      "FAU - Shiodera, Satomi\n",
      "AU  - Shiodera S\n",
      "AUID- ORCID: 0000-0002-6849-8568\n",
      "AD  - Research Institute for Humanity and Nature, Kyoto, Japan.\n",
      "AD  - Center for Southeast Asian Studies, Kyoto University, Kyoto, Japan.\n",
      "FAU - Shipley, Bill\n",
      "AU  - Shipley B\n",
      "AUID- ORCID: 0000-0002-7026-3880\n",
      "AD  - Universite de Sherbrooke, Sherbrooke, QC, Canada.\n",
      "FAU - Shovon, Tanvir Ahmed\n",
      "AU  - Shovon TA\n",
      "AUID- ORCID: 0000-0003-3311-1797\n",
      "AD  - Department of Biology, University of Regina, Regina, SK, Canada.\n",
      "FAU - Siebenkas, Alrun\n",
      "AU  - Siebenkas A\n",
      "AUID- ORCID: 0000-0001-9676-7094\n",
      "AD  - Technische Universitat Ilmenau, Ilmenau, Germany.\n",
      "FAU - Sierra, Carlos\n",
      "AU  - Sierra C\n",
      "AUID- ORCID: 0000-0003-0009-4169\n",
      "AD  - Max Planck Institute for Biogeochemistry, Jena, Germany.\n",
      "FAU - Silva, Vasco\n",
      "AU  - Silva V\n",
      "AUID- ORCID: 0000-0003-2729-1824\n",
      "AD  - Centre for Applied Ecology \"Professor Baeta Neves\" (CEABN), School ofAgriculture, University of Lisbon, Lisbon, Portugal.\n",
      "FAU - Silva, Mateus\n",
      "AU  - Silva M\n",
      "AUID- ORCID: 0000-0002-4281-3400\n",
      "AD  - Department of Biology, Federal University of Lavras, Lavras, MG, Brazil.\n",
      "FAU - Sitzia, Tommaso\n",
      "AU  - Sitzia T\n",
      "AUID- ORCID: 0000-0001-6221-4256\n",
      "AD  - Department Land, Environment, Agriculture and Forestry, Universita degli Studi diPadova, Padua, Italy.\n",
      "FAU - Sjoman, Henrik\n",
      "AU  - Sjoman H\n",
      "AUID- ORCID: 0000-0002-5526-6303\n",
      "AD  - Department of Landscape Architecture, Planning and Management, Swedish Universityof Agricultural Sciences, Alnarp, Sweden.\n",
      "AD  - Gothenburg Botanical Garden, Gothenburg, Sweden.\n",
      "AD  - Gothenburg Global Biodiversity Centre, Gothenburg, Sweden.\n",
      "FAU - Slot, Martijn\n",
      "AU  - Slot M\n",
      "AUID- ORCID: 0000-0002-5558-1792\n",
      "AD  - Smithsonian Tropical Research Institute, Balboa, Ancon, Republic of Panama.\n",
      "FAU - Smith, Nicholas G\n",
      "AU  - Smith NG\n",
      "AUID- ORCID: 0000-0001-7048-4387\n",
      "AD  - Texas Tech University, Lubbock, TX, USA.\n",
      "FAU - Sodhi, Darwin\n",
      "AU  - Sodhi D\n",
      "AUID- ORCID: 0000-0003-2119-2681\n",
      "AD  - Forest Sciences Centre, Faculty of Forestry and Conservation Science, University of British Columbia, Vancouver, BC, Canada.\n",
      "FAU - Soltis, Pamela\n",
      "AU  - Soltis P\n",
      "AUID- ORCID: 0000-0001-9310-8659\n",
      "AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL, USA.\n",
      "FAU - Soltis, Douglas\n",
      "AU  - Soltis D\n",
      "AUID- ORCID: 0000-0001-8638-4137\n",
      "AD  - Florida Museum of Natural History, University of Florida, Gainesville, FL, USA.\n",
      "FAU - Somers, Ben\n",
      "AU  - Somers B\n",
      "AUID- ORCID: 0000-0002-7875-107X\n",
      "AD  - Department of Earth and Environmental Sciences, KU Leuven, Leuven, Belgium.\n",
      "FAU - Sonnier, Gregory\n",
      "AU  - Sonnier G\n",
      "AUID- ORCID: 0000-0002-0614-7033\n",
      "AD  - Archbold Biological Station, Venus, FL, USA.\n",
      "FAU - Sorensen, Mia Vedel\n",
      "AU  - Sorensen MV\n",
      "AUID- ORCID: 0000-0002-2225-6117\n",
      "AD  - Norwegian University of Science and Technology NTNU, Trondheim, Norway.\n",
      "FAU - Sosinski, Enio Egon Jr\n",
      "AU  - Sosinski EE Jr\n",
      "AUID- ORCID: 0000-0001-6310-9474\n",
      "AD  - Embrapa Recursos Geneticos e Biotecnologia, Brasilia, DF, Brazil.\n",
      "FAU - Soudzilovskaia, Nadejda A\n",
      "AU  - Soudzilovskaia NA\n",
      "AUID- ORCID: 0000-0002-9584-2109\n",
      "AD  - Institute of Environmental Sciences, Leiden University, Leiden, The Netherlands.\n",
      "FAU - Souza, Alexandre F\n",
      "AU  - Souza AF\n",
      "AUID- ORCID: 0000-0001-7468-3631\n",
      "AD  - Departamento de Ecologia, Universidade Federal do Rio Grande do Norte, Natal, RN,Brazil.\n",
      "FAU - Spasojevic, Marko\n",
      "AU  - Spasojevic M\n",
      "AUID- ORCID: 0000-0003-1808-0048\n",
      "AD  - Department of Evolution, Ecology, and Organismal Biology, University ofCalifornia Riverside, Riverside, CA, USA.\n",
      "FAU - Sperandii, Marta Gaia\n",
      "AU  - Sperandii MG\n",
      "AUID- ORCID: 0000-0002-2507-5928\n",
      "AD  - University of Roma Tre, Rome, Italy.\n",
      "FAU - Stan, Amanda B\n",
      "AU  - Stan AB\n",
      "AUID- ORCID: 0000-0001-6111-0578\n",
      "AD  - Department of Geography, Planning and Recreation, Northern Arizona University,Flagstaff, AZ, USA.\n",
      "FAU - Stegen, James\n",
      "AU  - Stegen J\n",
      "AUID- ORCID: 0000-0001-9135-7424\n",
      "AD  - Pacific Northwest National Laboratory, Richland, WA, USA.\n",
      "FAU - Steinbauer, Klaus\n",
      "AU  - Steinbauer K\n",
      "AUID- ORCID: 0000-0002-3730-9920\n",
      "AD  - GLORIA-Coordination, Institute for Interdisciplinary Mountain Research, Austrian Academy of Sciences & Department of Integrative Biology and BiodiversityResearch, University of Natural Resources and Life Sciences Vienna, Vienna,Austria.\n",
      "FAU - Stephan, Jorg G\n",
      "AU  - Stephan JG\n",
      "AUID- ORCID: 0000-0001-6195-7867\n",
      "AD  - Swedish Species Information Centre, Swedish University of Agricultural Sciences, Uppsala, Sweden.\n",
      "AD  - Department of Ecology, Swedish University of Agricultural Sciences, Uppsala,Sweden.\n",
      "FAU - Sterck, Frank\n",
      "AU  - Sterck F\n",
      "AD  - Forest Ecology and Forest Management Group, Wageningen University, TheNetherlands.\n",
      "FAU - Stojanovic, Dejan B\n",
      "AU  - Stojanovic DB\n",
      "AUID- ORCID: 0000-0003-2967-2049\n",
      "AD  - Institute of Lowland Forestry and Environment, University of Novi Sad, Novi Sad, Serbia.\n",
      "FAU - Strydom, Tanya\n",
      "AU  - Strydom T\n",
      "AD  - Stockholm University, Stockholm, Sweden.\n",
      "FAU - Suarez, Maria Laura\n",
      "AU  - Suarez ML\n",
      "AD  - Instituto de Investigaciones en Biodiversidad y Medioambiente-CONICET,Universidad Nacional del Comahue, Bariloche, Argentina.\n",
      "FAU - Svenning, Jens-Christian\n",
      "AU  - Svenning JC\n",
      "AUID- ORCID: 0000-0002-3415-0862\n",
      "AD  - Section for Ecoinformatics and Biodiversity, Department of Bioscience, AarhusUniversity, Aarhus, Denmark.\n",
      "AD  - Center for Biodiversity Dynamics in a Changing World (BIOCHANGE), Department ofBioscience, Aarhus University, Aarhus, Denmark.\n",
      "FAU - Svitkova, Ivana\n",
      "AU  - Svitkova I\n",
      "AUID- ORCID: 0000-0002-5671-0330\n",
      "AD  - Institute of Botany, Plant Science and Biodiversity Center, Slovak Academy ofSciences, Bratislava, Slovakia.\n",
      "FAU - Svitok, Marek\n",
      "AU  - Svitok M\n",
      "AUID- ORCID: 0000-0003-2710-8102\n",
      "AD  - Department of Ecology and General Biology, Faculty of Ecology and EnvironmentalSciences, Technical University in Zvolen, Zvolen, Slovakia.\n",
      "AD  - Department of Ecosystem Biology, Faculty of Science, University of South Bohemia,Ceske Budejovice, Czech Republic.\n",
      "FAU - Svoboda, Miroslav\n",
      "AU  - Svoboda M\n",
      "AUID- ORCID: 0000-0003-4050-3422\n",
      "AD  - Faculty of Forestry and Wood Sciences, Czech University of Life Sciences, Prague,Czech Republic.\n",
      "FAU - Swaine, Emily\n",
      "AU  - Swaine E\n",
      "AD  - School of Biological Sciences, University of Aberdeen, Aberdeen, UK.\n",
      "FAU - Swenson, Nathan\n",
      "AU  - Swenson N\n",
      "AUID- ORCID: 0000-0003-3819-9767\n",
      "AD  - Department of Biology, University of Maryland, College Park, MD, USA.\n",
      "FAU - Tabarelli, Marcelo\n",
      "AU  - Tabarelli M\n",
      "AUID- ORCID: 0000-0001-7573-7216\n",
      "AD  - Departamento de Botanica, Universidade Federal de Pernambuco, Recife, PE, Brazil.\n",
      "FAU - Takagi, Kentaro\n",
      "AU  - Takagi K\n",
      "AUID- ORCID: 0000-0002-1321-2841\n",
      "AD  - Teshio Experimental Forest, Hokkaido University, Horonobe, Japan.\n",
      "FAU - Tappeiner, Ulrike\n",
      "AU  - Tappeiner U\n",
      "AUID- ORCID: 0000-0002-0195-7459\n",
      "AD  - Department of Ecology, University of Innsbruck, Innsbruck, Austria.\n",
      "AD  - Eurac Research, Institute for Alpine Environment, Bozen-Bolzano, Italy.\n",
      "FAU - Tarifa, Ruben\n",
      "AU  - Tarifa R\n",
      "AUID- ORCID: 0000-0002-0288-1978\n",
      "AD  - Departamento de Ecologia Funcional y Evolutiva, Estacion Experimental de ZonasAridas (CSIC), La Canada de San Urbano, Spain.\n",
      "FAU - Tauugourdeau, Simon\n",
      "AU  - Tauugourdeau S\n",
      "AUID- ORCID: 0000-0001-6561-3228\n",
      "AD  - SELMET, CIRAD, INRA, Univ Montpellier, Montpellier SupAgro, France.\n",
      "AD  - CIRAD-UMR SELMET-PZZS, Dakar, Senegal.\n",
      "FAU - Tavsanoglu, Cagatay\n",
      "AU  - Tavsanoglu C\n",
      "AUID- ORCID: 0000-0003-4447-6492\n",
      "AD  - Department of Biology, Hacettepe University, Ankara, Turkey.\n",
      "FAU - Te Beest, Mariska\n",
      "AU  - Te Beest M\n",
      "AUID- ORCID: 0000-0003-3673-4105\n",
      "AD  - Environmental Sciences, Copernicus Institute of Sustainable Development, Utrecht University, Utrecht, The Netherlands.\n",
      "AD  - Centre for African Conservation Ecology, Nelson Mandela University, PortElizabeth, South Africa.\n",
      "FAU - Tedersoo, Leho\n",
      "AU  - Tedersoo L\n",
      "AUID- ORCID: 0000-0002-1635-1249\n",
      "AD  - Institute of Ecology and Earth Sciences, University of Tartu, Tartu, Estonia.\n",
      "FAU - Thiffault, Nelson\n",
      "AU  - Thiffault N\n",
      "AUID- ORCID: 0000-0003-2017-6890\n",
      "AD  - Natural Resources Canada, Canadian Wood Fibre Centre, Quebec, QC, Canada.\n",
      "FAU - Thom, Dominik\n",
      "AU  - Thom D\n",
      "AUID- ORCID: 0000-0001-8091-6075\n",
      "AD  - Rubenstein School of Environment and Natural Resources, University of Vermont,Burlington, VT, USA.\n",
      "FAU - Thomas, Evert\n",
      "AU  - Thomas E\n",
      "AUID- ORCID: 0000-0002-7838-6228\n",
      "AD  - Bioversity International, Lima, Peru.\n",
      "FAU - Thompson, Ken\n",
      "AU  - Thompson K\n",
      "AD  - Department of Animal and Plant Sciences, University of Sheffield, Sheffield, UK.\n",
      "FAU - Thornton, Peter E\n",
      "AU  - Thornton PE\n",
      "AUID- ORCID: 0000-0002-4759-5158\n",
      "AD  - Oak Ridge National Laboratory, Oak Ridge, TN, USA.\n",
      "FAU - Thuiller, Wilfried\n",
      "AU  - Thuiller W\n",
      "AUID- ORCID: 0000-0002-5388-5274\n",
      "AD  - Univ. Grenoble Alpes, CNRS, Univ. Savoie Mont Blanc, LECA, Grenoble, France.\n",
      "FAU - Tichy, Lubomir\n",
      "AU  - Tichy L\n",
      "AUID- ORCID: 0000-0001-8400-7741\n",
      "AD  - Department of Botany and Zoology, Masaryk University, Brno, Czech Republic.\n",
      "FAU - Tissue, David\n",
      "AU  - Tissue D\n",
      "AUID- ORCID: 0000-0002-8497-2047\n",
      "AD  - Hawkesbury Institute for the Environment, Western Sydney University, Sydney, NSW,Australia.\n",
      "FAU - Tjoelker, Mark G\n",
      "AU  - Tjoelker MG\n",
      "AUID- ORCID: 0000-0003-4607-5238\n",
      "AD  - Hawkesbury Institute for the Environment, Western Sydney University, Sydney, NSW,Australia.\n",
      "FAU - Tng, David Yue Phin\n",
      "AU  - Tng DYP\n",
      "AUID- ORCID: 0000-0001-5135-0922\n",
      "AD  - Centre for Rainforest Studies, The School for Field Studies, Yungaburra, Qld,Australia.\n",
      "FAU - Tobias, Joseph\n",
      "AU  - Tobias J\n",
      "AUID- ORCID: 0000-0003-2429-6179\n",
      "AD  - Department of Life Sciences, Imperial College London, Silwood Park, Ascot, UK.\n",
      "FAU - Torok, Peter\n",
      "AU  - Torok P\n",
      "AUID- ORCID: 0000-0002-4428-3327\n",
      "AD  - MTA-DE Lendulet Functional and Restoration Ecology Research Group, Debrecen,Hungary.\n",
      "AD  - Department of Ecology, University of Debrecen, Debrecen, Hungary.\n",
      "FAU - Tarin, Tonantzin\n",
      "AU  - Tarin T\n",
      "AUID- ORCID: 0000-0001-5125-579X\n",
      "AD  - Department of Soil and Plant Sciences, University of Delaware, Newark, DE, USA.\n",
      "FAU - Torres-Ruiz, Jose M\n",
      "AU  - Torres-Ruiz JM\n",
      "AUID- ORCID: 0000-0003-1367-7056\n",
      "AD  - INRA - Universite Clermont-Auvergne, UMR PIAF, Clermont-Ferrand, France.\n",
      "FAU - Tothmeresz, Bela\n",
      "AU  - Tothmeresz B\n",
      "AUID- ORCID: 0000-0002-4766-7668\n",
      "AD  - MTA-TKI Biodiversity and Ecosystem Services Research Group, Debrecen, Hungary.\n",
      "FAU - Treurnicht, Martina\n",
      "AU  - Treurnicht M\n",
      "AUID- ORCID: 0000-0002-0853-0247\n",
      "AD  - SAEON Fynbos Node, Claremont, South Africa.\n",
      "AD  - Department of Conservation Ecology and Entomology, Stellenbosch University,Matieland, South Africa.\n",
      "FAU - Trivellone, Valeria\n",
      "AU  - Trivellone V\n",
      "AD  - Illinois Natural History Survey, Prairie Research Institute, University ofIllinois, Champaign, IL, USA.\n",
      "FAU - Trolliet, Franck\n",
      "AU  - Trolliet F\n",
      "AUID- ORCID: 0000-0003-1472-2592\n",
      "AD  - Unit for Modelling of Climate and Biogeochemical Cycles, UR-SPHERES, Universityof Liege, Liege, Belgique.\n",
      "FAU - Trotsiuk, Volodymyr\n",
      "AU  - Trotsiuk V\n",
      "AUID- ORCID: 0000-0002-8363-656X\n",
      "AD  - Swiss Federal Institute for Forest, Snow and Landscape Research WSL, Birmensdorf,Switzerland.\n",
      "AD  - Faculty of Forestry and Wood Sciences, Czech University of Life Sciences, Prague,Czech Republic.\n",
      "AD  - Institute of Agricultural Sciences, Department of Environmental Systems Science, ETH Zurich, Zurich, Switzerland.\n",
      "FAU - Tsakalos, James L\n",
      "AU  - Tsakalos JL\n",
      "AUID- ORCID: 0000-0001-5067-196X\n",
      "AD  - School of Biological Sciences, The University of Western Australia, Crawley, WA, Australia.\n",
      "FAU - Tsiripidis, Ioannis\n",
      "AU  - Tsiripidis I\n",
      "AUID- ORCID: 0000-0001-9373-676X\n",
      "AD  - School of Biology, Department of Botany, Aristotle University of Thessaloniki,Greece.\n",
      "FAU - Tysklind, Niklas\n",
      "AU  - Tysklind N\n",
      "AD  - CIRAD, UMR EcoFoG (Agroparistech, CNRS, INRA, Universite des Antilles, Universitede la Guyane), Kourou, France.\n",
      "FAU - Umehara, Toru\n",
      "AU  - Umehara T\n",
      "AD  - Osaka Natural History Center, Osaka, Japan.\n",
      "FAU - Usoltsev, Vladimir\n",
      "AU  - Usoltsev V\n",
      "AUID- ORCID: 0000-0003-4587-8952\n",
      "AD  - Ural State Forest Engineering University, Ekaterinburg, Russia.\n",
      "AD  - Botanical Garden of Ural Branch of Russian Academy of Sciences, Ekaterinburg,Russia.\n",
      "FAU - Vadeboncoeur, Matthew\n",
      "AU  - Vadeboncoeur M\n",
      "AUID- ORCID: 0000-0002-8269-0708\n",
      "AD  - University of New Hampshire, Durham, NH, USA.\n",
      "FAU - Vaezi, Jamil\n",
      "AU  - Vaezi J\n",
      "AD  - Department of Biology, Faculty of Science, Ferdowsi University of Mashhad,Mashhad, Iran.\n",
      "FAU - Valladares, Fernando\n",
      "AU  - Valladares F\n",
      "AUID- ORCID: 0000-0002-5374-4682\n",
      "AD  - Museo Nacional de Ciencias Naturales-CSIC, Madrid, Spain.\n",
      "FAU - Vamosi, Jana\n",
      "AU  - Vamosi J\n",
      "AUID- ORCID: 0000-0002-2376-0729\n",
      "AD  - Department of Biological Sciences, University of Calgary, Calgary, AB, Canada.\n",
      "FAU - van Bodegom, Peter M\n",
      "AU  - van Bodegom PM\n",
      "AUID- ORCID: 0000-0003-0771-4500\n",
      "AD  - Institute of Environmental Sciences, Leiden University, Leiden, The Netherlands.\n",
      "FAU - van Breugel, Michiel\n",
      "AU  - van Breugel M\n",
      "AUID- ORCID: 0000-0003-2778-7803\n",
      "AD  - College Environmental Studies, Yale University, New Haven, CT, USA.\n",
      "AD  - Department of Biological Sciences, National University of Singapore, Singapore,Singapore.\n",
      "AD  - Smithsonian Tropical Research Institute, Panama City, Panama.\n",
      "FAU - Van Cleemput, Elisa\n",
      "AU  - Van Cleemput E\n",
      "AUID- ORCID: 0000-0002-5305-9749\n",
      "AD  - Department of Earth and Environmental Sciences, KU Leuven, Leuven, Belgium.\n",
      "FAU - van de Weg, Martine\n",
      "AU  - van de Weg M\n",
      "AD  - School of Geosciences, Edinburgh University, Edinburgh, UK.\n",
      "FAU - van der Merwe, Stephni\n",
      "AU  - van der Merwe S\n",
      "AUID- ORCID: 0000-0002-6385-8536\n",
      "AD  - Department of Biological Sciences, University of Cape Town, Cape Town, SouthAfrica.\n",
      "FAU - van der Plas, Fons\n",
      "AU  - van der Plas F\n",
      "AD  - Systematic Botany and Functional Biodiversity, Institute of Biology, LeipzigUniversity, Leipzig, Germany.\n",
      "FAU - van der Sande, Masha T\n",
      "AU  - van der Sande MT\n",
      "AUID- ORCID: 0000-0002-6845-2308\n",
      "AD  - Institute for Biodiversity and Ecosystem Dynamics (IBED), University ofAmsterdam, Amsterdam, The Netherlands.\n",
      "AD  - Forest Ecology and Forest Management Group, Wageningen University and Research,Wageningen, The Netherlands.\n",
      "AD  - Institute for Global Ecology, Florida Institute of Technology, Melbourne, FL,USA.\n",
      "FAU - van Kleunen, Mark\n",
      "AU  - van Kleunen M\n",
      "AUID- ORCID: 0000-0002-2861-3701\n",
      "AD  - Department of Biology, University of Konstanz, Konstanz, Germany.\n",
      "AD  - Zhejiang Provincial Key Laboratory of Plant Evolutionary Ecology andConservation, Taizhou University, Taizhou, China.\n",
      "FAU - Van Meerbeek, Koenraad\n",
      "AU  - Van Meerbeek K\n",
      "AUID- ORCID: 0000-0002-9260-3815\n",
      "AD  - Department of Earth and Environmental Sciences, KU Leuven, Leuven, Belgium.\n",
      "FAU - Vanderwel, Mark\n",
      "AU  - Vanderwel M\n",
      "AD  - Department of Biology, University of Regina, Regina, SK, Canada.\n",
      "FAU - Vanselow, Kim Andre\n",
      "AU  - Vanselow KA\n",
      "AUID- ORCID: 0000-0003-3299-6220\n",
      "AD  - Institute of Geography, University of Erlangen-Nuremberg, Erlangen, Germany.\n",
      "FAU - Varhammar, Angelica\n",
      "AU  - Varhammar A\n",
      "AD  - Hawkesbury Institute for the Environment, Western Sydney University, Sydney, NSW,Australia.\n",
      "FAU - Varone, Laura\n",
      "AU  - Varone L\n",
      "AUID- ORCID: 0000-0002-9989-5283\n",
      "AD  - Department of Environmental Biology, Sapienza University of Rome, Rome, Italy.\n",
      "FAU - Vasquez Valderrama, Maribel Yesenia\n",
      "AU  - Vasquez Valderrama MY\n",
      "AUID- ORCID: 0000-0001-7839-7229\n",
      "AD  - Universidad Distrital Francisco Jose de Caldas, Bogota, Colombia.\n",
      "AD  - Laboratorio de invasiones Biologicas, Universidad de Concepcion, Concepcion,Chile.\n",
      "FAU - Vassilev, Kiril\n",
      "AU  - Vassilev K\n",
      "AUID- ORCID: 0000-0003-4376-5575\n",
      "AD  - Institute of Biodiversity and Ecosystem Research, Bulgarian Academy of Sciences, Sofia, Bulgaria.\n",
      "FAU - Vellend, Mark\n",
      "AU  - Vellend M\n",
      "AUID- ORCID: 0000-0002-2491-956X\n",
      "AD  - Universite de Sherbrooke, Sherbrooke, QC, Canada.\n",
      "FAU - Veneklaas, Erik J\n",
      "AU  - Veneklaas EJ\n",
      "AUID- ORCID: 0000-0002-7030-4056\n",
      "AD  - School of Biological Sciences and School of Agriculture and Environment, TheUniversity of Western Australia, Crawley, WA, Australia.\n",
      "FAU - Verbeeck, Hans\n",
      "AU  - Verbeeck H\n",
      "AUID- ORCID: 0000-0003-1490-0168\n",
      "AD  - CAVElab - Computational and Applied Vegetation Ecology, Ghent University, Ghent, Belgium.\n",
      "FAU - Verheyen, Kris\n",
      "AU  - Verheyen K\n",
      "AUID- ORCID: 0000-0002-2067-9108\n",
      "AD  - Department of Environment, Forest & Nature Lab, Ghent University, Gontrode-Melle,Belgium.\n",
      "FAU - Vibrans, Alexander\n",
      "AU  - Vibrans A\n",
      "AUID- ORCID: 0000-0002-8789-5833\n",
      "AD  - Universidade Regional de Blumenau, Blumenau, SC, Brazil.\n",
      "FAU - Vieira, Ima\n",
      "AU  - Vieira I\n",
      "AUID- ORCID: 0000-0003-1233-318X\n",
      "AD  - Museu Paraense Emilio Goeldi, Belem, PA, Brazil.\n",
      "FAU - Villacis, Jaime\n",
      "AU  - Villacis J\n",
      "AUID- ORCID: 0000-0001-7752-8506\n",
      "AD  - Departamento de Ciencias de la Vida, Universidad de las Fuerzas Armadas (ESPE),Sangolqui, Ecuador.\n",
      "FAU - Violle, Cyrille\n",
      "AU  - Violle C\n",
      "AUID- ORCID: 0000-0002-2471-9226\n",
      "AD  - UMR 5175 CEFE, Univ. Montpellier, CNRS, EPHE, IRD, Univ. Paul Valery,Montpellier, France.\n",
      "FAU - Vivek, Pandi\n",
      "AU  - Vivek P\n",
      "AD  - Department of Botany, Goa University, Goa, India.\n",
      "AD  - Department of Ecology and Environmental Sciences, Pondicherry University,Puducherry, India.\n",
      "FAU - Wagner, Katrin\n",
      "AU  - Wagner K\n",
      "AUID- ORCID: 0000-0003-4326-1086\n",
      "AD  - Carl von Ossietzky University of Oldenburg, Oldenburg, Germany.\n",
      "FAU - Waldram, Matthew\n",
      "AU  - Waldram M\n",
      "AD  - School of Geography, Geology and Environment, University of Leicester, Leicester,UK.\n",
      "FAU - Waldron, Anthony\n",
      "AU  - Waldron A\n",
      "AD  - Zoology Department, Edward Grey Institute, Oxford University, Oxford, UK.\n",
      "AD  - Department of Zoology, Cambridge University, Cambridge Conservation Initiative,Cambridge, UK.\n",
      "FAU - Walker, Anthony P\n",
      "AU  - Walker AP\n",
      "AUID- ORCID: 0000-0003-0557-5594\n",
      "AD  - Environmental Sciences Division & Climate Change Science Institute, Oak RidgeNational Laboratory, Oak Ridge, TN, USA.\n",
      "FAU - Waller, Martyn\n",
      "AU  - Waller M\n",
      "AUID- ORCID: 0000-0003-3876-2492\n",
      "AD  - Department of Geography and Geology, Kingston University, Kingston upon Thames,UK.\n",
      "FAU - Walther, Gabriel\n",
      "AU  - Walther G\n",
      "AD  - Institute of Ecology and Evolution, Friedrich Schiller University Jena, Jena,Germany.\n",
      "FAU - Wang, Han\n",
      "AU  - Wang H\n",
      "AUID- ORCID: 0000-0003-2482-1818\n",
      "AD  - Ministry of Education Key Laboratory for Earth System Modeling, Department ofEarth System Science, Tsinghua University, Beijing, China.\n",
      "FAU - Wang, Feng\n",
      "AU  - Wang F\n",
      "AUID- ORCID: 0000-0003-3595-515X\n",
      "AD  - Institute of Desertification Studies, Chinese Academy of Forestry, Beijing,China.\n",
      "FAU - Wang, Weiqi\n",
      "AU  - Wang W\n",
      "AUID- ORCID: 0000-0001-8503-0978\n",
      "AD  - Institute of Geography, Fujian Normal University, Fuzhou, China.\n",
      "FAU - Watkins, Harry\n",
      "AU  - Watkins H\n",
      "AUID- ORCID: 0000-0002-4038-7145\n",
      "AD  - Department of Landscape Architecture, University of Sheffield, Sheffield, UK.\n",
      "FAU - Watkins, James\n",
      "AU  - Watkins J\n",
      "AD  - Department of Biology, Colgate University, Hamilton, NY, USA.\n",
      "FAU - Weber, Ulrich\n",
      "AU  - Weber U\n",
      "AD  - Max Planck Institute for Biogeochemistry, Jena, Germany.\n",
      "FAU - Weedon, James T\n",
      "AU  - Weedon JT\n",
      "AUID- ORCID: 0000-0003-0491-8719\n",
      "AD  - Ecological Sciences, Vrije Universiteit Amsterdam, Amsterdam, The Netherlands.\n",
      "FAU - Wei, Liping\n",
      "AU  - Wei L\n",
      "AUID- ORCID: 0000-0002-4296-9851\n",
      "AD  - Institut de recherche sur les forets, Universite du Quebec enAbitibi-Temiscamingue, Rouyn-Noranda, QC, Canada.\n",
      "FAU - Weigelt, Patrick\n",
      "AU  - Weigelt P\n",
      "AUID- ORCID: 0000-0002-2485-3708\n",
      "AD  - Biodiversity, Macroecology and Biogeography, University of Goettingen, Gottingen,Germany.\n",
      "FAU - Weiher, Evan\n",
      "AU  - Weiher E\n",
      "AUID- ORCID: 0000-0002-5375-9964\n",
      "AD  - University of Wisconsin Eau Claire, Eau Claire, WI, USA.\n",
      "FAU - Wells, Aidan W\n",
      "AU  - Wells AW\n",
      "AUID- ORCID: 0000-0002-2209-3282\n",
      "AD  - Rocky Mountain Biological Laboratory, Crested Butte, CO, USA.\n",
      "AD  - Maritime and Science Technology Academy, Miami, FL, USA.\n",
      "FAU - Wellstein, Camilla\n",
      "AU  - Wellstein C\n",
      "AUID- ORCID: 0000-0001-6994-274X\n",
      "AD  - Free University of Bozen-Bolzano, Bolzano, Italy.\n",
      "FAU - Wenk, Elizabeth\n",
      "AU  - Wenk E\n",
      "AD  - Evolution & Ecology Research Centre, and School of Biological, Earth andEnvironmental Sciences, UNSW Sydney, Sydney, NSW, Australia.\n",
      "FAU - Westoby, Mark\n",
      "AU  - Westoby M\n",
      "AUID- ORCID: 0000-0001-7690-4530\n",
      "AD  - Department of Biological Sciences, Macquarie University, Sydney, NSW, Australia.\n",
      "FAU - Westwood, Alana\n",
      "AU  - Westwood A\n",
      "AD  - University of Winnipeg, Winnipeg, Manitoba, Canada.\n",
      "FAU - White, Philip John\n",
      "AU  - White PJ\n",
      "AUID- ORCID: 0000-0003-0827-288X\n",
      "AD  - The James Hutton Institute, Dundee, UK.\n",
      "AD  - King Saud University, Riyadh, Saudi Arabia.\n",
      "FAU - Whitten, Mark\n",
      "AU  - Whitten M\n",
      "AD  - University of Florida, Gainesville, FL, USA.\n",
      "FAU - Williams, Mathew\n",
      "AU  - Williams M\n",
      "AUID- ORCID: 0000-0001-6117-5208\n",
      "AD  - School of Geosciences, The University of Edinburgh, Edinburgh, UK.\n",
      "FAU - Winkler, Daniel E\n",
      "AU  - Winkler DE\n",
      "AUID- ORCID: 0000-0003-4825-9073\n",
      "AD  - University of California - Irvine, Irvine, CA, USA.\n",
      "AD  - Southwest Biological Science Center, U. S. Geological Survey, Moab, UT, USA.\n",
      "FAU - Winter, Klaus\n",
      "AU  - Winter K\n",
      "AD  - Smithsonian Tropical Research Institute, Balboa, Ancon, Republic of Panama.\n",
      "FAU - Womack, Chevonne\n",
      "AU  - Womack C\n",
      "AD  - Department of Plant and Soil Sciences, University of Pretoria, Pretoria, SouthAfrica.\n",
      "FAU - Wright, Ian J\n",
      "AU  - Wright IJ\n",
      "AUID- ORCID: 0000-0001-8338-9143\n",
      "AD  - Department of Biological Sciences, Macquarie University, Sydney, NSW, Australia.\n",
      "FAU - Wright, S Joseph\n",
      "AU  - Wright SJ\n",
      "AUID- ORCID: 0000-0003-4260-5676\n",
      "AD  - Smithsonian Tropical Research Institute, Balboa, Ancon, Republic of Panama.\n",
      "FAU - Wright, Justin\n",
      "AU  - Wright J\n",
      "AUID- ORCID: 0000-0002-9102-5347\n",
      "AD  - Department of Biology, Duke University, Durham, NC, USA.\n",
      "FAU - Pinho, Bruno X\n",
      "AU  - Pinho BX\n",
      "AUID- ORCID: 0000-0002-6588-3575\n",
      "AD  - Departamento de Botanica, Universidade Federal de Pernambuco, Recife, PE, Brazil.\n",
      "FAU - Ximenes, Fabiano\n",
      "AU  - Ximenes F\n",
      "AUID- ORCID: 0000-0003-0516-9414\n",
      "AD  - NSW Department of Primary Industries, Parramatta, NSW, Australia.\n",
      "FAU - Yamada, Toshihiro\n",
      "AU  - Yamada T\n",
      "AUID- ORCID: 0000-0001-9307-6255\n",
      "AD  - Hiroshima University, Higashi-Hiroshima, Japan.\n",
      "FAU - Yamaji, Keiko\n",
      "AU  - Yamaji K\n",
      "AUID- ORCID: 0000-0003-2875-5619\n",
      "AD  - Graduate School of Life and Environmental Sciences, University of Tsukuba,Tsukuba, Japan.\n",
      "FAU - Yanai, Ruth\n",
      "AU  - Yanai R\n",
      "AUID- ORCID: 0000-0001-6987-2489\n",
      "AD  - SUNY-College of Environmental Science and Forestry, Syracuse, NY, USA.\n",
      "FAU - Yankov, Nikolay\n",
      "AU  - Yankov N\n",
      "AD  - Botanical Garden of the Samara University, Samara, Russia.\n",
      "FAU - Yguel, Benjamin\n",
      "AU  - Yguel B\n",
      "AD  - Centre d'Ecologie et des Sciences de la Conservation (CESCO), Museum Nationald'Histoire Naturelle, Centre National de la Recherche Scientifique,Sorbonne-Universite, Paris, France.\n",
      "FAU - Zanini, Katia Janaina\n",
      "AU  - Zanini KJ\n",
      "AD  - Laboratorio de Ecologia Vegetal (LEVEG), Porto Alegre, RS, Brazil.\n",
      "FAU - Zanne, Amy E\n",
      "AU  - Zanne AE\n",
      "AUID- ORCID: 0000-0001-6379-9452\n",
      "AD  - Biological Sciences, George Washington University, Washington, DC, USA.\n",
      "FAU - Zeleny, David\n",
      "AU  - Zeleny D\n",
      "AUID- ORCID: 0000-0001-5157-044X\n",
      "AD  - National Taiwan University, Taipei, Taiwan.\n",
      "FAU - Zhao, Yun-Peng\n",
      "AU  - Zhao YP\n",
      "AD  - College of Life Sciences, Zhejiang University, Hangzhou, China.\n",
      "FAU - Zheng, Jingming\n",
      "AU  - Zheng J\n",
      "AUID- ORCID: 0000-0002-8517-9999\n",
      "AD  - Forestry College, Beijing Forestry University, Beijing, China.\n",
      "FAU - Zheng, Ji\n",
      "AU  - Zheng J\n",
      "AUID- ORCID: 0000-0003-4936-1168\n",
      "AD  - School of Agriculture and Biology, Shanghai Jiao Tong University, Shanghai, P. R.China.\n",
      "AD  - Shanghai Urban Forest Ecosystem Research Station, National Forestry and GrasslandAdministration, Shanghai, P.R. China.\n",
      "FAU - Zieminska, Kasia\n",
      "AU  - Zieminska K\n",
      "AUID- ORCID: 0000-0001-7750-4758\n",
      "AD  - Imperial College, London, UK.\n",
      "FAU - Zirbel, Chad R\n",
      "AU  - Zirbel CR\n",
      "AUID- ORCID: 0000-0002-9289-1722\n",
      "AD  - Department of Ecology, Evolution, and Behavior, University of Minnesota, St.Paul, MN, USA.\n",
      "FAU - Zizka, Georg\n",
      "AU  - Zizka G\n",
      "AD  - Department of Biological Sciences, Goethe Universitat Frankfurt, Frankfurt amMain, Germany.\n",
      "AD  - Department of Botany and Molecular Evolution, Senckenberg Research Institute and Natural History Museum, Frankfurt am Main, Germany.\n",
      "FAU - Zo-Bi, Irie Casimir\n",
      "AU  - Zo-Bi IC\n",
      "AUID- ORCID: 0000-0003-0982-8579\n",
      "AD  - Institut National Polytechnique Felix Houphouet-Boigny (INP-HB), Yamoussoukro,Cote d'Ivoire.\n",
      "FAU - Zotz, Gerhard\n",
      "AU  - Zotz G\n",
      "AUID- ORCID: 0000-0002-6823-2268\n",
      "AD  - Smithsonian Tropical Research Institute, Balboa, Ancon, Republic of Panama.\n",
      "AD  - Institute for Biology and Environmental Sciences, University Oldenburg,Oldenburg, Germany.\n",
      "FAU - Wirth, Christian\n",
      "AU  - Wirth C\n",
      "AD  - Max Planck Institute for Biogeochemistry, Jena, Germany.\n",
      "AD  - German Center for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig,Leipzig, Germany.\n",
      "AD  - University of Leipzig, Leipzig, Germany.\n",
      "LA  - eng\n",
      "GR  - International Geosphere-Biosphere Programme (IGBP)/International\n",
      "GR  - GIS 'Climat, Environnement et Societe' France/International\n",
      "GR  - German Centre for Integrative Biodiversity Research (iDiv)Halle-Jena-Leipzig/International\n",
      "GR  - AXA Research Fund/International\n",
      "GR  - Max Planck Society/International\n",
      "GR  - UK Natural Environment Research Council (NERC)/International\n",
      "GR  - French Foundation for Biodiversity Research (FRB)/International\n",
      "GR  - Max Planck Institute for Biogeochemistry/International\n",
      "GR  - International Programme of Biodiversity Science (DIVERSITAS)/International\n",
      "GR  - P 29142/FWF_/Austrian Science Fund FWF/Austria\n",
      "GR  - Future Earth/International\n",
      "PT  - Journal Article\n",
      "DEP - 20191231\n",
      "PL  - England\n",
      "TA  - Glob Chang Biol\n",
      "JT  - Global change biology\n",
      "JID - 9888746\n",
      "SB  - IM\n",
      "CIN - Glob Chang Biol. 2020 Jan;26(1):189-190. PMID: 31891658\n",
      "MH  - *Access to Information\n",
      "MH  - Biodiversity\n",
      "MH  - Ecology\n",
      "MH  - *Ecosystem\n",
      "MH  - Plants\n",
      "OTO - NOTNLM\n",
      "OT  - *TRY plant trait database\n",
      "OT  - *data coverage\n",
      "OT  - *data integration\n",
      "OT  - *data representativeness\n",
      "OT  - *functional diversity\n",
      "OT  - *plant traits\n",
      "EDAT- 2020/01/01 06:00\n",
      "MHDA- 2020/01/09 06:00\n",
      "CRDT- 2020/01/01 06:00\n",
      "PHST- 2019/08/15 00:00 [received]\n",
      "PHST- 2019/09/12 00:00 [accepted]\n",
      "PHST- 2020/01/01 06:00 [pubmed]\n",
      "PHST- 2020/01/09 06:00 [medline]\n",
      "PHST- 2020/01/01 06:00 [entrez]\n",
      "AID - 10.1111/gcb.14904 [doi]\n",
      "PST - ppublish\n",
      "SO  - Glob Chang Biol. 2020 Jan;26(1):119-188. doi: 10.1111/gcb.14904. Epub 2019 Dec31.\n",
      "\n",
      "PMID- 31822297\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200518\n",
      "LR  - 20200518\n",
      "IS  - 1479-7364 (Electronic)\n",
      "IS  - 1473-9542 (Linking)\n",
      "VI  - 13\n",
      "IP  - 1\n",
      "DP  - 2019 Dec 10\n",
      "TI  - Human genetics and genomics research in Ecuador: historical survey, currentstate, and future directions.\n",
      "PG  - 64\n",
      "LID - 10.1186/s40246-019-0249-8 [doi]\n",
      "AB  - BACKGROUND: In South America, the history of human genetics is extensive and its beginnings go back to the onset of the twentieth century. In Ecuador, thehistorical record of human genetics and genomics research is limited. In thiscontext, our work analyzes the current status and historical panorama of thesefields, based on bibliographic searches in Scopus, Google Scholar, PubMed, andWeb of Science. RESULTS: Our results determined that the oldest paper in humangenetics coauthored by an Ecuadorian institution originates from the CentralUniversity of Ecuador in 1978. From a historical standpoint, the number ofarticles has increased since the 1990s. This growth has intensified and it isreflected in 137 manuscripts recorded from 2010 to 2019. Areas such as humanpopulation genetics, phylogeography, and forensic sciences are the core ofgenetics and genomics-associated research in Ecuador. Important advances havebeen made in the understanding of the bases of cancer, some genetic diseases, andcongenital disorders. Fields such as pharmacogenetics and pharmacogenomics havebegun to be explored during the last years. CONCLUSIONS: This work paints acomprehensive picture and provides additional insights into the future panoramaof human genetic and genomic research in Ecuador as an example of an emerging,resource-limited country with interesting phylogeographic characteristics andpublic health implications.\n",
      "FAU - Zambrano-Mila, Marlon S\n",
      "AU  - Zambrano-Mila MS\n",
      "AUID- ORCID: 0000-0001-5942-2986\n",
      "AD  - School of Biological Sciences and Engineering, Yachay Tech University, San Miguelde Urcuqui, Ecuador. marlonz1996@hotmail.com.\n",
      "FAU - Agathos, Spiros N\n",
      "AU  - Agathos SN\n",
      "AD  - School of Biological Sciences and Engineering, Yachay Tech University, San Miguelde Urcuqui, Ecuador.\n",
      "AD  - Bioengineering Lab, Earth & Life Institute, Universite Catholique de Louvain,Louvain-la-Neuve, Belgium.\n",
      "FAU - Reichardt, Juergen K V\n",
      "AU  - Reichardt JKV\n",
      "AD  - Australian Institute of Tropical Health and Medicine (AITHM), James CookUniversity, Smithfield, QLD, Australia.\n",
      "LA  - eng\n",
      "GR  - Internal grant 39-2017/Yachay Tech/International\n",
      "PT  - Historical Article\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20191210\n",
      "PL  - England\n",
      "TA  - Hum Genomics\n",
      "JT  - Human genomics\n",
      "JID - 101202210\n",
      "SB  - IM\n",
      "MH  - Ecuador\n",
      "MH  - Genomics/*trends\n",
      "MH  - History, 20th Century\n",
      "MH  - History, 21st Century\n",
      "MH  - Human Genetics\n",
      "MH  - Humans\n",
      "MH  - Publications\n",
      "MH  - Research/*history/*trends\n",
      "MH  - *Surveys and Questionnaires\n",
      "PMC - PMC6902328\n",
      "OTO - NOTNLM\n",
      "OT  - *Database search\n",
      "OT  - *Ecuador\n",
      "OT  - *Google Scholar\n",
      "OT  - *Human genetics\n",
      "OT  - *Human genomics\n",
      "OT  - *PubMed\n",
      "OT  - *Scopus\n",
      "OT  - *South America\n",
      "OT  - *Web of Science\n",
      "EDAT- 2019/12/12 06:00\n",
      "MHDA- 2020/05/19 06:00\n",
      "CRDT- 2019/12/12 06:00\n",
      "PHST- 2019/09/30 00:00 [received]\n",
      "PHST- 2019/11/14 00:00 [accepted]\n",
      "PHST- 2019/12/12 06:00 [entrez]\n",
      "PHST- 2019/12/12 06:00 [pubmed]\n",
      "PHST- 2020/05/19 06:00 [medline]\n",
      "AID - 10.1186/s40246-019-0249-8 [doi]\n",
      "AID - 10.1186/s40246-019-0249-8 [pii]\n",
      "PST - epublish\n",
      "SO  - Hum Genomics. 2019 Dec 10;13(1):64. doi: 10.1186/s40246-019-0249-8.\n",
      "\n",
      "PMID- 31773318\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200914\n",
      "LR  - 20210110\n",
      "IS  - 1437-1596 (Electronic)\n",
      "IS  - 0937-9827 (Linking)\n",
      "VI  - 134\n",
      "IP  - 1\n",
      "DP  - 2020 Jan\n",
      "TI  - Targeted exon sequencing in deceased schizophrenia patients in Denmark.\n",
      "PG  - 135-147\n",
      "LID - 10.1007/s00414-019-02212-z [doi]\n",
      "AB  - Schizophrenia patients have higher mortality rates and lower life expectancy thanthe general population. However, forensic investigations of their deaths oftenfail to determine the cause of death, hindering prevention. As schizophrenia is ahighly heritable condition and given recent advances in our understanding of the genetics of schizophrenia, it is now possible to investigate how genetic factors may contribute to mortality. We made use of findings from genome-wide associationstudies (GWAS) to design a targeted panel (PsychPlex) for sequencing of exons of 451 genes near index single nucleotide polymorphisms (SNPs) identified with GWAS.We sequenced the DNA of 95 deceased schizophrenia patients included in SURVIVE, aprospective, autopsy-based study of mentally ill persons in Denmark. We compared the allele frequencies of 1039 SNPs in these cases with the frequencies of 2000Danes without psychiatric diseases and calculated their deleteriousness (CADD)scores. For 81 SNPs highly associated with schizophrenia and CADD scores above15, expression profiles in the Genotype-Tissue Expression (GTEx) Projectindicated that these variants were in exons, whose expressions are increased inseveral types of brain tissues, particularly in the cerebellum. Molecular pathwayanalysis indicated the involvement of 163 different pathways. As for rare SNPvariants, most variants were scored as either benign or likely benign with anaverage of 17 variants of unknown significance per individual and no pathogenicvariant. Our results highlight the potential of DNA sequencing of an exon panelto discover genetic factors that may be involved in the development ofschizophrenia.\n",
      "FAU - Espregueira Themudo, Goncalo\n",
      "AU  - Espregueira Themudo G\n",
      "AUID- ORCID: https://orcid.org/0000-0002-5019-9439\n",
      "AD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.gethemudo@gmail.com.\n",
      "AD  - CIIMAR - Interdisciplinary Centre of Marine and Environmental Research of theUniversity of Porto, Terminal de Cruzeiros do Porto de Leixoes, Avenida GeneralNorton de Matos, S/N, 4450-208, Matosinhos, Portugal. gethemudo@gmail.com.\n",
      "FAU - Leerschool, Anna-Roos\n",
      "AU  - Leerschool AR\n",
      "AD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\n",
      "AD  - Department of Complex Genetics, Maastricht University, PO Box 616 6200, MD,Maastricht, The Netherlands.\n",
      "FAU - Rodriguez-Proano, Carla\n",
      "AU  - Rodriguez-Proano C\n",
      "AD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\n",
      "AD  - Clinical Laboratory, Ambulatory Clinical Surgical Center and Day Hospital \"ElBatan\", Quito, Ecuador.\n",
      "FAU - Christiansen, Sofie Lindgren\n",
      "AU  - Christiansen SL\n",
      "AUID- ORCID: https://orcid.org/0000-0003-3810-4123\n",
      "AD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\n",
      "FAU - Andersen, Jeppe Dyrberg\n",
      "AU  - Andersen JD\n",
      "AUID- ORCID: https://orcid.org/0000-0002-4395-5232\n",
      "AD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\n",
      "FAU - Busch, Johannes Rodbro\n",
      "AU  - Busch JR\n",
      "AD  - Section of Forensic Pathology, Department of Forensic Medicine, Faculty of Healthand Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\n",
      "FAU - Christensen, Martin Roest\n",
      "AU  - Christensen MR\n",
      "AD  - Section of Forensic Pathology, Department of Forensic Medicine, Faculty of Healthand Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\n",
      "FAU - Banner, Jytte\n",
      "AU  - Banner J\n",
      "AUID- ORCID: https://orcid.org/0000-0002-7559-1285\n",
      "AD  - Section of Forensic Pathology, Department of Forensic Medicine, Faculty of Healthand Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\n",
      "FAU - Morling, Niels\n",
      "AU  - Morling N\n",
      "AUID- ORCID: https://orcid.org/0000-0002-9463-5087\n",
      "AD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20191126\n",
      "PL  - Germany\n",
      "TA  - Int J Legal Med\n",
      "JT  - International journal of legal medicine\n",
      "JID - 9101456\n",
      "SB  - IM\n",
      "MH  - Adult\n",
      "MH  - Aged\n",
      "MH  - Cause of Death\n",
      "MH  - Denmark\n",
      "MH  - *Exons\n",
      "MH  - Female\n",
      "MH  - Gene Expression Profiling\n",
      "MH  - *Gene Frequency\n",
      "MH  - *Genetic Variation\n",
      "MH  - Genome-Wide Association Study\n",
      "MH  - Humans\n",
      "MH  - Male\n",
      "MH  - Middle Aged\n",
      "MH  - *Polymorphism, Single Nucleotide\n",
      "MH  - Prospective Studies\n",
      "MH  - Schizophrenia/*genetics/mortality\n",
      "MH  - Sequence Analysis, DNA/*methods\n",
      "OTO - NOTNLM\n",
      "OT  - Genomics\n",
      "OT  - Haloplex\n",
      "OT  - Massively parallel sequencing\n",
      "OT  - Molecular autopsy\n",
      "OT  - Schizophrenia\n",
      "EDAT- 2019/11/28 06:00\n",
      "MHDA- 2020/09/15 06:00\n",
      "CRDT- 2019/11/28 06:00\n",
      "PHST- 2019/08/29 00:00 [received]\n",
      "PHST- 2019/11/13 00:00 [accepted]\n",
      "PHST- 2019/11/28 06:00 [pubmed]\n",
      "PHST- 2020/09/15 06:00 [medline]\n",
      "PHST- 2019/11/28 06:00 [entrez]\n",
      "AID - 10.1007/s00414-019-02212-z [doi]\n",
      "AID - 10.1007/s00414-019-02212-z [pii]\n",
      "PST - ppublish\n",
      "SO  - Int J Legal Med. 2020 Jan;134(1):135-147. doi: 10.1007/s00414-019-02212-z. Epub2019 Nov 26.\n",
      "\n",
      "PMID- 31746150\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210120\n",
      "LR  - 20210120\n",
      "IS  - 2045-8827 (Electronic)\n",
      "IS  - 2045-8827 (Linking)\n",
      "VI  - 9\n",
      "IP  - 2\n",
      "DP  - 2020 Feb\n",
      "TI  - Evolutionary changes of an intestinal Lactobacillus reuteri during probioticmanufacture.\n",
      "PG  - e972\n",
      "LID - 10.1002/mbo3.972 [doi]\n",
      "AB  - Probiotic bacteria are frequently used to treat intestinal diseases or to improvehealth; however, little is known about the evolutionary changes of these bacteriaduring probiotic manufacture and the bacterial ability to colonize the intestine.It has been observed that when bacteria adapt to a new environment, they losesome traits required to thrive in the original niche. In this study, a strain of Lactobacillus reuteri was isolated from mouse duodenum and subjected to 150serial passes in milk to simulate the industrial propagation of probioticbacteria. The strains adapted to milk outperformed their ancestor when grown inmilk; we also showed evidence of reduced intestinal colonization of milk-adapted strains. Whole-genome sequencing showed that bacterial adaptation to milk selectsmutants with altered metabolic functions.\n",
      "CI  - (c) 2019 The Authors. MicrobiologyOpen published by John Wiley & Sons Ltd.\n",
      "FAU - Lopez, Lazaro\n",
      "AU  - Lopez L\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Calderon, Diana\n",
      "AU  - Calderon D\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Cardenas, Paul\n",
      "AU  - Cardenas P\n",
      "AUID- ORCID: 0000-0001-9626-4489\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Prado, Maria B\n",
      "AU  - Prado MB\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Valle, Carlos\n",
      "AU  - Valle C\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Trueba, Gabriel\n",
      "AU  - Trueba G\n",
      "AUID- ORCID: 0000-0003-2617-9021\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "LA  - eng\n",
      "SI  - GENBANK/NC_009513.1\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20191119\n",
      "PL  - England\n",
      "TA  - Microbiologyopen\n",
      "JT  - MicrobiologyOpen\n",
      "JID - 101588314\n",
      "RN  - 0 (Bacterial Proteins)\n",
      "RN  - VJT6J7R4TR (Rifampin)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Bacterial Proteins/genetics\n",
      "MH  - *Biological Evolution\n",
      "MH  - Drug Resistance, Bacterial\n",
      "MH  - Female\n",
      "MH  - *Gastrointestinal Microbiome\n",
      "MH  - Genome, Bacterial\n",
      "MH  - Genomics/methods\n",
      "MH  - Lactobacillus reuteri/drug effects/*physiology\n",
      "MH  - Mice\n",
      "MH  - Milk/microbiology\n",
      "MH  - Mutation\n",
      "MH  - *Probiotics\n",
      "MH  - Rifampin/pharmacology\n",
      "PMC - PMC7002098\n",
      "OTO - NOTNLM\n",
      "OT  - * Lactobacillus reuteri\n",
      "OT  - *antagonistic pleiotropy\n",
      "OT  - *experimental evolution\n",
      "OT  - *probiotics\n",
      "EDAT- 2019/11/21 06:00\n",
      "MHDA- 2021/01/21 06:00\n",
      "CRDT- 2019/11/21 06:00\n",
      "PHST- 2019/09/06 00:00 [received]\n",
      "PHST- 2019/10/30 00:00 [revised]\n",
      "PHST- 2019/10/31 00:00 [accepted]\n",
      "PHST- 2019/11/21 06:00 [pubmed]\n",
      "PHST- 2021/01/21 06:00 [medline]\n",
      "PHST- 2019/11/21 06:00 [entrez]\n",
      "AID - 10.1002/mbo3.972 [doi]\n",
      "PST - ppublish\n",
      "SO  - Microbiologyopen. 2020 Feb;9(2):e972. doi: 10.1002/mbo3.972. Epub 2019 Nov 19.\n",
      "\n",
      "PMID- 31698729\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20200928\n",
      "IS  - 2075-4450 (Print)\n",
      "IS  - 2075-4450 (Linking)\n",
      "VI  - 10\n",
      "IP  - 11\n",
      "DP  - 2019 Nov 6\n",
      "TI  - Do Host Plant and Associated Ant Species Affect Microbial Communities inMyrmecophytes?\n",
      "LID - E391 [pii]\n",
      "LID - 10.3390/insects10110391 [doi]\n",
      "AB  - Ant-associated microorganisms can play crucial and often overlooked roles, andgiven the diversity of interactions that ants have developed, the study of theassociated microbiomes is of interest. We focused here on specialist plant-antspecies of the genus Allomerus that grow a fungus to build galleries on theirhost-plant stems. Allomerus-inhabited domatia, thus, might be a rich arena formicrobes associated with the ants, the plant, and the fungus. We investigated themicrobial communities present in domatia colonised by four arboreal ants:Allomerus decemarticulatus, A. octoarticulatus, A. octoarticulatus var.demerarae, and the non-fungus growing plant-ant Azteca sp. cf. depilis,inhabiting Hirtella physophora or Cordia nodosa in French Guiana. We hypothesizedthat the microbial community will differ among these species. We isolatedmicroorganisms from five colonies of each species, sequenced the 16S rRNA orInternal TranscribedSpacer (ITS) regions, and described both the alpha and betadiversities. We identified 69 microbial taxa, which belong to five bacterial and two fungal phyla. The most diverse phyla were Proteobacteria and Actinobacteria. The microbial community of Azteca cf. depilis and Allomerus spp. differed incomposition and richness. Geographical distance affected microbial communitiesand richness but plant species did not. Actinobacteria were only associated with Allomerus spp.\n",
      "FAU - Ruiz-Gonzalez, Mario X\n",
      "AU  - Ruiz-Gonzalez MX\n",
      "AUID- ORCID: 0000-0003-3654-773X\n",
      "AD  - Departamento de Ciencias Biologicas, Universidad Tecnica Particular de Loja, San Cayetano Alto s/n, Loja 1101608, Ecuador.\n",
      "FAU - Leroy, Celine\n",
      "AU  - Leroy C\n",
      "AD  - AMAP, IRD, CIRAD, CNRS, INRA, Universite de Montpellier, 34000 Montpellier,France.\n",
      "AD  - CNRS, UMR EcoFoG, Agroparistech, CIRAD, INRA, Universite de Guyane, Universitedes Antilles, Campus Agronomique, 97379 Kourou, France.\n",
      "FAU - Dejean, Alain\n",
      "AU  - Dejean A\n",
      "AD  - CNRS, UMR EcoFoG, Agroparistech, CIRAD, INRA, Universite de Guyane, Universitedes Antilles, Campus Agronomique, 97379 Kourou, France.\n",
      "AD  - Ecolab, Universite de Toulouse, CNRS, INPT, UPS, 31062 Toulouse, France.\n",
      "FAU - Gryta, Herve\n",
      "AU  - Gryta H\n",
      "AD  - Laboratoire Evolution & Diversite Biologique (EDB UMR 5174), CNRS, IRD,Universite de Toulouse, 31062 Toulouse, France.\n",
      "FAU - Jargeat, Patricia\n",
      "AU  - Jargeat P\n",
      "AD  - Laboratoire Evolution & Diversite Biologique (EDB UMR 5174), CNRS, IRD,Universite de Toulouse, 31062 Toulouse, France.\n",
      "FAU - Armijos Carrion, Angelo D\n",
      "AU  - Armijos Carrion AD\n",
      "AD  - Biodiversity Genomics Team, Plant Ecophysiology & Evolution Group, Guangxi KeyLaboratory of Forest Ecology and Conservation, College of Forestry, Daxuedonglu100, Nanning 530005, Guangxi, China.\n",
      "FAU - Orivel, Jerome\n",
      "AU  - Orivel J\n",
      "AD  - CNRS, UMR EcoFoG, Agroparistech, CIRAD, INRA, Universite de Guyane, Universitedes Antilles, Campus Agronomique, 97379 Kourou, France.\n",
      "LA  - eng\n",
      "GR  - CEBA, ANR-10-LABX-25-01/Agence Nationale de la Recherche\n",
      "GR  - BING, GY0007194/PO-FEDER 2014-2020, Region Guyane\n",
      "PT  - Journal Article\n",
      "DEP - 20191106\n",
      "PL  - Switzerland\n",
      "TA  - Insects\n",
      "JT  - Insects\n",
      "JID - 101574235\n",
      "PMC - PMC6920819\n",
      "OTO - NOTNLM\n",
      "OT  - Allomerus decemarticulatus\n",
      "OT  - Allomerus octoarticulatus\n",
      "OT  - Azteca sp. cf. depilis\n",
      "OT  - Cordia nodosa\n",
      "OT  - Hirtella physophora\n",
      "OT  - domatia\n",
      "OT  - microbial diversity\n",
      "EDAT- 2019/11/09 06:00\n",
      "MHDA- 2019/11/09 06:01\n",
      "CRDT- 2019/11/09 06:00\n",
      "PHST- 2019/09/24 00:00 [received]\n",
      "PHST- 2019/10/31 00:00 [revised]\n",
      "PHST- 2019/11/02 00:00 [accepted]\n",
      "PHST- 2019/11/09 06:00 [entrez]\n",
      "PHST- 2019/11/09 06:00 [pubmed]\n",
      "PHST- 2019/11/09 06:01 [medline]\n",
      "AID - insects10110391 [pii]\n",
      "AID - 10.3390/insects10110391 [doi]\n",
      "PST - epublish\n",
      "SO  - Insects. 2019 Nov 6;10(11). pii: insects10110391. doi: 10.3390/insects10110391.\n",
      "\n",
      "PMID- 31671555\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200902\n",
      "LR  - 20200902\n",
      "IS  - 2218-273X (Electronic)\n",
      "IS  - 2218-273X (Linking)\n",
      "VI  - 9\n",
      "IP  - 11\n",
      "DP  - 2019 Oct 30\n",
      "TI  - Unravelling the Skin Secretion Peptides of the Gliding Leaf Frog, Agalychnisspurrelli (Hylidae).\n",
      "LID - E667 [pii]\n",
      "LID - 10.3390/biom9110667 [doi]\n",
      "AB  - Frog skin secretions contain medically-valuable molecules, which are useful forthe discovery of new biopharmaceuticals. The peptide profile of the skinsecretion of Agalychnis spurrelli has not been investigated; therefore, thestructural and biological characterization of its compounds signify aninestimable opportunity to acquire new biologically-active chemical scaffolds. Inthis work, skin secretion from this amphibian was analysed by molecular cloningand tandem mass spectrometry. Although the extent of this work was notexhaustive, eleven skin secretion peptides belonging to five peptide familieswere identified. Among these, we report the occurrence of two phyllokinins, andone medusin-SP which were previously reported in other related species. Inaddition, eight novel peptides were identified, including four dermaseptins,DRS-SP2 to DRS-SP5, one phylloseptin-SP1, and three orphan peptides.Phylloseptin-SP1 and dermaseptins-SP2 were identified in HPLC fractions based on their molecular masses determined by MALDI-TOF MS. Among the antimicrobialpeptides, dermaseptin-SP2 was the most potent, inhibiting Escherichia coli,Staphylococcus aureus, and ORSA with a minimum inhibitory concentration (MIC) of 2.68 muM, and Candida albicans with an MIC of 10.71 muM, without haemolyticeffects. The peptides described in this study represent but a superficial glance at the considerable structural diversity of bioactive peptides produced in theskin secretion of A. spurrelli.\n",
      "FAU - Proano-Bolanos, Carolina\n",
      "AU  - Proano-Bolanos C\n",
      "AUID- ORCID: 0000-0001-9279-1038\n",
      "AD  - Natural Drug Discovery Group, School of Pharmacy, Queen's University, Belfast 97 Lisburn Road, Belfast BT9 7BL, Northern Ireland, UK. cproanobolanos01@qub.ac.uk.\n",
      "AD  - Biomolecules Discovery Group, Laboratory of Molecular Biology and Biochemistry,Universidad Regional Amazonica Ikiam, km 7 (1/2) via Muyuna, Tena 150150,Ecuador. cproanobolanos01@qub.ac.uk.\n",
      "FAU - Blasco-Zuniga, Ailin\n",
      "AU  - Blasco-Zuniga A\n",
      "AD  - Laboratorio de Investigacion en Citogenetica y Biomoleculas de Anfibios (LICBA), Centro de Investigacion para la Salud en America Latina (CISeAL), PontificiaUniversidad Catolica del Ecuador, Av 12 de Octubre 1076 y Roca, Quito 170150,Ecuador. ailin.blasco.z@gmail.com.\n",
      "FAU - Almeida, Jose Rafael\n",
      "AU  - Almeida JR\n",
      "AUID- ORCID: 0000-0002-4637-4468\n",
      "AD  - Biomolecules Discovery Group, Laboratory of Molecular Biology and Biochemistry,Universidad Regional Amazonica Ikiam, km 7 (1/2) via Muyuna, Tena 150150,Ecuador. rafael.dealmeida@ikiam.edu.ec.\n",
      "FAU - Wang, Lei\n",
      "AU  - Wang L\n",
      "AUID- ORCID: 0000-0002-8811-8489\n",
      "AD  - Natural Drug Discovery Group, School of Pharmacy, Queen's University, Belfast 97 Lisburn Road, Belfast BT9 7BL, Northern Ireland, UK. l.wang@qub.ac.uk.\n",
      "FAU - Llumiquinga, Miguel Angel\n",
      "AU  - Llumiquinga MA\n",
      "AD  - Laboratorio de Investigacion en Citogenetica y Biomoleculas de Anfibios (LICBA), Centro de Investigacion para la Salud en America Latina (CISeAL), PontificiaUniversidad Catolica del Ecuador, Av 12 de Octubre 1076 y Roca, Quito 170150,Ecuador. mallumiquinga@puce.edu.ec.\n",
      "FAU - Rivera, Miryan\n",
      "AU  - Rivera M\n",
      "AD  - Laboratorio de Investigacion en Citogenetica y Biomoleculas de Anfibios (LICBA), Centro de Investigacion para la Salud en America Latina (CISeAL), PontificiaUniversidad Catolica del Ecuador, Av 12 de Octubre 1076 y Roca, Quito 170150,Ecuador. miryanrivera@gmail.com.\n",
      "FAU - Zhou, Mei\n",
      "AU  - Zhou M\n",
      "AUID- ORCID: 0000-0003-2576-0252\n",
      "AD  - Natural Drug Discovery Group, School of Pharmacy, Queen's University, Belfast 97 Lisburn Road, Belfast BT9 7BL, Northern Ireland, UK. m.zhou@qub.ac.uk.\n",
      "FAU - Chen, Tianbao\n",
      "AU  - Chen T\n",
      "AUID- ORCID: 0000-0001-8011-9260\n",
      "AD  - Natural Drug Discovery Group, School of Pharmacy, Queen's University, Belfast 97 Lisburn Road, Belfast BT9 7BL, Northern Ireland, UK. t.chen@qub.ac.uk.\n",
      "FAU - Shaw, Chris\n",
      "AU  - Shaw C\n",
      "AD  - Natural Drug Discovery Group, School of Pharmacy, Queen's University, Belfast 97 Lisburn Road, Belfast BT9 7BL, Northern Ireland, UK. chris.shaw@qub.ac.uk.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20191030\n",
      "PL  - Switzerland\n",
      "TA  - Biomolecules\n",
      "JT  - Biomolecules\n",
      "JID - 101596414\n",
      "RN  - 0 (Anti-Infective Agents)\n",
      "RN  - 0 (Peptides)\n",
      "SB  - IM\n",
      "MH  - Amino Acid Sequence\n",
      "MH  - Animals\n",
      "MH  - Anti-Infective Agents/chemistry/*metabolism/*pharmacology\n",
      "MH  - Anura/*metabolism\n",
      "MH  - Peptides/chemistry/*metabolism/*pharmacology\n",
      "MH  - Proteomics\n",
      "MH  - Skin/*metabolism\n",
      "PMC - PMC6920962\n",
      "OTO - NOTNLM\n",
      "OT  - *Agalychnis spurrelli\n",
      "OT  - *dermaseptins\n",
      "OT  - *frog skin secretion\n",
      "OT  - *peptidomics\n",
      "OT  - *phylloseptins\n",
      "OT  - *tandem mass spectrometry\n",
      "EDAT- 2019/11/02 06:00\n",
      "MHDA- 2020/09/04 06:00\n",
      "CRDT- 2019/11/02 06:00\n",
      "PHST- 2019/09/18 00:00 [received]\n",
      "PHST- 2019/10/14 00:00 [revised]\n",
      "PHST- 2019/10/16 00:00 [accepted]\n",
      "PHST- 2019/11/02 06:00 [entrez]\n",
      "PHST- 2019/11/02 06:00 [pubmed]\n",
      "PHST- 2020/09/04 06:00 [medline]\n",
      "AID - biom9110667 [pii]\n",
      "AID - 10.3390/biom9110667 [doi]\n",
      "PST - epublish\n",
      "SO  - Biomolecules. 2019 Oct 30;9(11). pii: biom9110667. doi: 10.3390/biom9110667.\n",
      "\n",
      "PMID- 31628908\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200414\n",
      "LR  - 20200414\n",
      "IS  - 1090-2392 (Electronic)\n",
      "IS  - 0011-2240 (Linking)\n",
      "VI  - 91\n",
      "DP  - 2019 Dec\n",
      "TI  - Two-step accelerating freezing protocol yields a better motility, membranes andDNA integrities of thawed ram sperm than three-steps freezing protocols.\n",
      "PG  - 84-89\n",
      "LID - S0011-2240(19)30197-X [pii]\n",
      "LID - 10.1016/j.cryobiol.2019.10.007 [doi]\n",
      "AB  - The present study compares a protocol that mimics freezing of ram semen in staticnitrogen vapor with two protocols with an initial low cooling rate in the firststep, followed by higher cooling rates where ice nucleation occurs. Semenejaculates, obtained from twelve adults rams, were diluted with TEST-basedextender and frozen with either Protocol 1 (three-step decelerating cooling):from +5 degrees C to -35 degrees C (40 degrees C/min), from -35 degrees C to -65 degrees C (17 degrees C/min), and then from -65 degrees C to -85 degrees C (3degrees C/min); or Protocol 2 (three-step accelerating cooling): from +5 degrees C to -5 degrees C (4 degrees C/min), from -5 degrees C to -110 degrees C (25degrees C/min), and then from -110 degrees C to -140 degrees C (35 degreesC/min); or Protocol 3 (two-step accelerating cooling), from +5 degrees C to -10degrees C (5 degrees C/min), and then from -10 degrees C to -130 degrees C (60degrees C/min). Post-thaw sperm quality was reduced for all protocols (p < .05)compared with fresh semen. Post-thaw percentages of sperm motilitycharacteristics and sperm with intact plasma membrane, intact acrosome, andintact mitochondrial membrane were greater using Protocol 3 than Protocol 2 (p < .05) and Protocol 1 (p < .01). In addition, the post-thaw percentage of spermwith fragmented DNA was lower (p < .05) using Protocol 3 compared with Protocol1. The present results indicate that a cooling rate of 60 degrees C/min aroundand after the time point of ice nucleation provided better post thaw survival andfunction of ram sperm than lower (and/or decelerating) cooling rates.\n",
      "CI  - Copyright (c) 2019 The Authors. Published by Elsevier Inc. All rights reserved.\n",
      "FAU - Galarza, Diego A\n",
      "AU  - Galarza DA\n",
      "AD  - Departamento de Reproduccion Animal, INIA, 28040, Madrid, Spain; Facultad deCiencias Agropecuarias, Universidad de Cuenca, EC010205, Cuenca, Ecuador.Electronic address: dgalarza@ucm.es.\n",
      "FAU - Lopez-Sebastian, Antonio\n",
      "AU  - Lopez-Sebastian A\n",
      "AD  - Departamento de Reproduccion Animal, INIA, 28040, Madrid, Spain.\n",
      "FAU - Woelders, Henri\n",
      "AU  - Woelders H\n",
      "AD  - Wageningen University & Research, Animal Breeding and Genomics, P.O. BOX 338,6700 AH, Wageningen, the Netherlands.\n",
      "FAU - Blesbois, Elizabeth\n",
      "AU  - Blesbois E\n",
      "AD  - UMR-PRC INRA-CNRS-IFCE- Francois Rabelais University, 37380, Nouzilly, France.\n",
      "FAU - Santiago-Moreno, Julian\n",
      "AU  - Santiago-Moreno J\n",
      "AD  - Departamento de Reproduccion Animal, INIA, 28040, Madrid, Spain.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20191016\n",
      "PL  - Netherlands\n",
      "TA  - Cryobiology\n",
      "JT  - Cryobiology\n",
      "JID - 0006252\n",
      "RN  - 9007-49-2 (DNA)\n",
      "SB  - IM\n",
      "MH  - Acrosome/*physiology\n",
      "MH  - Animals\n",
      "MH  - Cell Membrane\n",
      "MH  - Cryopreservation/*methods\n",
      "MH  - DNA\n",
      "MH  - DNA Fragmentation/*drug effects\n",
      "MH  - Freezing\n",
      "MH  - Humans\n",
      "MH  - Male\n",
      "MH  - Semen/*physiology\n",
      "MH  - Semen Analysis\n",
      "MH  - Semen Preservation/*methods\n",
      "MH  - Sheep\n",
      "MH  - Sperm Motility/physiology\n",
      "OTO - NOTNLM\n",
      "OT  - *Cooling\n",
      "OT  - *DNA integrity\n",
      "OT  - *Ram\n",
      "OT  - *Semen cryopreservation\n",
      "OT  - *Semen quality\n",
      "EDAT- 2019/10/20 06:00\n",
      "MHDA- 2020/04/15 06:00\n",
      "CRDT- 2019/10/20 06:00\n",
      "PHST- 2019/07/01 00:00 [received]\n",
      "PHST- 2019/10/04 00:00 [revised]\n",
      "PHST- 2019/10/14 00:00 [accepted]\n",
      "PHST- 2019/10/20 06:00 [pubmed]\n",
      "PHST- 2020/04/15 06:00 [medline]\n",
      "PHST- 2019/10/20 06:00 [entrez]\n",
      "AID - S0011-2240(19)30197-X [pii]\n",
      "AID - 10.1016/j.cryobiol.2019.10.007 [doi]\n",
      "PST - ppublish\n",
      "SO  - Cryobiology. 2019 Dec;91:84-89. doi: 10.1016/j.cryobiol.2019.10.007. Epub 2019Oct 16.\n",
      "\n",
      "PMID- 31616044\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20210104\n",
      "LR  - 20210104\n",
      "IS  - 1473-1150 (Electronic)\n",
      "IS  - 1470-269X (Linking)\n",
      "VI  - 20\n",
      "IP  - 1\n",
      "DP  - 2020 Feb\n",
      "TI  - Pharmacogenomics, biomarker network, and allele frequencies in colorectal cancer.\n",
      "PG  - 136-158\n",
      "LID - 10.1038/s41397-019-0102-4 [doi]\n",
      "AB  - Colorectal cancer is one of the leading causes of cancer death worldwide. Overthe last decades, several studies have shown that tumor-related genomicalterations predict tumor prognosis, drug response, and toxicity. Theseobservations have led to the development of several therapies based on individualgenomic profiles. As part of these approaches, pharmacogenomics analyses genomic alterations which may predict an efficient therapeutic response. Studying thesemutations as biomarkers for predicting drug response is of a great interest toimprove precision medicine. We conduct a comprehensive review of the mainpharmacogenomics biomarkers and genomic alterations affecting enzyme activity,transporter capacity, channels, and receptors; and therefore the new advances in CRC precision medicine to select the best therapeutic strategy in populationsworldwide, with a focus on Latin America.\n",
      "FAU - Lopez-Cortes, Andres\n",
      "AU  - Lopez-Cortes A\n",
      "AUID- ORCID: http://orcid.org/0000-0003-1503-1929\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, 170527, Quito, Ecuador. aalc84@gmail.com.\n",
      "AD  - RNASA-IMEDIR, Computer Science Faculty, University of Coruna, 15071, Coruna,Spain. aalc84@gmail.com.\n",
      "AD  - Red Latinoamericana de Implementacion y Validacion de Guias ClinicasFarmacogenomicas (RELIVAF-CYTED), Santiago, Chile. aalc84@gmail.com.\n",
      "FAU - Paz-Y-Mino, Cesar\n",
      "AU  - Paz-Y-Mino C\n",
      "AUID- ORCID: http://orcid.org/0000-0002-6693-7344\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, 170527, Quito, Ecuador.cesar.pazymino@ute.edu.ec.\n",
      "FAU - Guerrero, Santiago\n",
      "AU  - Guerrero S\n",
      "AUID- ORCID: http://orcid.org/0000-0003-3473-7214\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, 170527, Quito, Ecuador.\n",
      "AD  - Gene Regulation, Stem Cells and Cancer Programme, Centre for Genomics Regulation (CRG), The Barcelona Institute for Science and Technology (BIST), UniversitatPompeu Fabra (UPF), 08003, Barcelona, Spain.\n",
      "FAU - Jaramillo-Koupermann, Gabriela\n",
      "AU  - Jaramillo-Koupermann G\n",
      "AUID- ORCID: http://orcid.org/0000-0001-8068-1763\n",
      "AD  - Laboratorio de Biologia Molecular, Subproceso de Anatomia Patologica, Hospital deEspecialidades Eugenio Espejo, 170403, Quito, Ecuador.\n",
      "FAU - Leon Caceres, Angela\n",
      "AU  - Leon Caceres A\n",
      "AD  - Global Health Institute, Faculty of Medicine, Heidelberg University, 69120,Heidelberg, Germany.\n",
      "FAU - Intriago-Baldeon, Damaris P\n",
      "AU  - Intriago-Baldeon DP\n",
      "AUID- ORCID: http://orcid.org/0000-0001-9157-3979\n",
      "AD  - Division of Molecular Genome Analysis, German Cancer Research Center (DKFZ),69120, Heidelberg, Germany.\n",
      "AD  - Faculty of Biosciences, Heidelberg University, 69120, Heidelberg, Germany.\n",
      "FAU - Garcia-Cardenas, Jennyfer M\n",
      "AU  - Garcia-Cardenas JM\n",
      "AUID- ORCID: http://orcid.org/0000-0001-9035-7668\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, 170527, Quito, Ecuador.\n",
      "FAU - Guevara-Ramirez, Patricia\n",
      "AU  - Guevara-Ramirez P\n",
      "AUID- ORCID: http://orcid.org/0000-0002-4829-3653\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, 170527, Quito, Ecuador.\n",
      "FAU - Armendariz-Castillo, Isaac\n",
      "AU  - Armendariz-Castillo I\n",
      "AUID- ORCID: http://orcid.org/0000-0003-2636-2707\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, 170527, Quito, Ecuador.\n",
      "FAU - Leone, Paola E\n",
      "AU  - Leone PE\n",
      "AUID- ORCID: http://orcid.org/0000-0003-3351-2275\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, 170527, Quito, Ecuador.\n",
      "FAU - Quinones, Luis Abel\n",
      "AU  - Quinones LA\n",
      "AD  - Red Latinoamericana de Implementacion y Validacion de Guias ClinicasFarmacogenomicas (RELIVAF-CYTED), Santiago, Chile.\n",
      "AD  - Laboratory of Chemical Carcinogenesis and Pharmacogenetics, Department ofBasic-Clinical Oncology, Faculty of Medicine, University of Chile, 70111,Santiago, Chile.\n",
      "FAU - Cayun, Juan Pablo\n",
      "AU  - Cayun JP\n",
      "AD  - Red Latinoamericana de Implementacion y Validacion de Guias ClinicasFarmacogenomicas (RELIVAF-CYTED), Santiago, Chile.\n",
      "AD  - Laboratory of Chemical Carcinogenesis and Pharmacogenetics, Department ofBasic-Clinical Oncology, Faculty of Medicine, University of Chile, 70111,Santiago, Chile.\n",
      "FAU - Soria, Nestor W\n",
      "AU  - Soria NW\n",
      "AD  - Catedra de Biotecnologia, Facultad de Ciencias Quimicas, Universidad Catolica de Cordoba, Unidad Asociada al CONICET, 5000, Cordoba, Argentina.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Review\n",
      "DEP - 20191015\n",
      "PL  - United States\n",
      "TA  - Pharmacogenomics J\n",
      "JT  - The pharmacogenomics journal\n",
      "JID - 101083949\n",
      "RN  - 0 (Antineoplastic Agents)\n",
      "RN  - 0 (Biomarkers, Tumor)\n",
      "SB  - IM\n",
      "MH  - Antineoplastic Agents/administration & dosage/adverse effects/metabolism\n",
      "MH  - Biomarkers, Tumor/*genetics/metabolism\n",
      "MH  - Colorectal Neoplasms/drug therapy/*genetics\n",
      "MH  - Gene Frequency/drug effects/*genetics\n",
      "MH  - Gene Regulatory Networks/drug effects/*genetics\n",
      "MH  - Humans\n",
      "MH  - Pharmacogenetics/*methods\n",
      "EDAT- 2019/10/17 06:00\n",
      "MHDA- 2021/01/05 06:00\n",
      "CRDT- 2019/10/17 06:00\n",
      "PHST- 2018/10/09 00:00 [received]\n",
      "PHST- 2019/10/02 00:00 [accepted]\n",
      "PHST- 2019/09/09 00:00 [revised]\n",
      "PHST- 2019/10/17 06:00 [pubmed]\n",
      "PHST- 2021/01/05 06:00 [medline]\n",
      "PHST- 2019/10/17 06:00 [entrez]\n",
      "AID - 10.1038/s41397-019-0102-4 [doi]\n",
      "AID - 10.1038/s41397-019-0102-4 [pii]\n",
      "PST - ppublish\n",
      "SO  - Pharmacogenomics J. 2020 Feb;20(1):136-158. doi: 10.1038/s41397-019-0102-4. Epub 2019 Oct 15.\n",
      "\n",
      "PMID- 31611888\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20200930\n",
      "IS  - 1664-462X (Print)\n",
      "IS  - 1664-462X (Linking)\n",
      "VI  - 10\n",
      "DP  - 2019\n",
      "TI  - Alternative Oxidase (AOX) Senses Stress Levels to Coordinate Auxin-InducedReprogramming From Seed Germination to Somatic Embryogenesis-A Role Relevant for Seed Vigor Prediction and Plant Robustness.\n",
      "PG  - 1134\n",
      "LID - 10.3389/fpls.2019.01134 [doi]\n",
      "AB  - Somatic embryogenesis (SE) is the most striking and prominent example of plantplasticity upon severe stress. Inducing immature carrot seeds perform SE assubstitute to germination by auxin treatment can be seen as switch between stresslevels associated to morphophysiological plasticity. This experimental system is highly powerful to explore stress response factors that mediate the metabolicswitch between cell and tissue identities. Developmental plasticity per se is an emerging trait for in vitro systems and crop improvement. It is supposed tounderlie multi-stress tolerance. High plasticity can protect plants throughoutlife cycles against variable abiotic and biotic conditions. We provide proof ofconcepts for the existing hypothesis that alternative oxidase (AOX) can berelevant for developmental plasticity and be associated to yield stability. Ourperspective on AOX as relevant coordinator of cell reprogramming is supported by real-time polymerase chain reaction (PCR) analyses and gross metabolism data fromcalorespirometry complemented by SHAM-inhibitor studies on primed, elevatedpartial pressure of oxygen (EPPO)-stressed, and endophyte-treated seeds. Insilico studies on public experimental data from diverse species strengthengenerality of our insights. Finally, we highlight ready-to-use concepts for plantselection and optimizing in vivo and in vitro propagation that do not requirefurther details on molecular physiology and metabolism. This is demonstrated byapplying our research & technology concepts to pea genotypes with differentialyield performance in multilocation fields and chickpea types known fordifferential robustness in the field. By using these concepts and toolsappropriately, also other marker candidates than AOX and complex genomics datacan be efficiently validated for prebreeding and seed vigor prediction.\n",
      "CI  - Copyright (c) 2019 Mohanapriya, Bharadwaj, Noceda, Costa, Kumar, Sathishkumar,Thiers, Santos Macedo, Silva, Annicchiarico, Groot, Kodde, Kumari, Gupta andArnholdt-Schmitt.\n",
      "FAU - Mohanapriya, Gunasekaran\n",
      "AU  - Mohanapriya G\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "AD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University ofEvora, Evora, Portugal.\n",
      "FAU - Bharadwaj, Revuru\n",
      "AU  - Bharadwaj R\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "AD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University ofEvora, Evora, Portugal.\n",
      "FAU - Noceda, Carlos\n",
      "AU  - Noceda C\n",
      "AD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University ofEvora, Evora, Portugal.\n",
      "AD  - Cell and Molecular Biology of Plants (BPOCEMP)/Industrial Biotechnology andBioproducts, Department of Sciences of the Vidaydela Agriculture, University ofthe Armed Forces-ESPE, Milagro, Ecuador.\n",
      "AD  - Faculty of Engineering, State University of Milagro (UNEMI), Milagro, Ecuador.\n",
      "FAU - Costa, Jose Helio\n",
      "AU  - Costa JH\n",
      "AD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University ofEvora, Evora, Portugal.\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "FAU - Kumar, Sarma Rajeev\n",
      "AU  - Kumar SR\n",
      "AD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University ofEvora, Evora, Portugal.\n",
      "FAU - Sathishkumar, Ramalingam\n",
      "AU  - Sathishkumar R\n",
      "AD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, BharathiarUniversity, Coimbatore, India.\n",
      "AD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University ofEvora, Evora, Portugal.\n",
      "FAU - Thiers, Karine Leitao Lima\n",
      "AU  - Thiers KLL\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "FAU - Santos Macedo, Elisete\n",
      "AU  - Santos Macedo E\n",
      "AD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University ofEvora, Evora, Portugal.\n",
      "FAU - Silva, Sofia\n",
      "AU  - Silva S\n",
      "AD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University ofEvora, Evora, Portugal.\n",
      "FAU - Annicchiarico, Paolo\n",
      "AU  - Annicchiarico P\n",
      "AD  - Council for Agricultural Research and Economics (CREA), Research Centre forAnimal Production and Aquaculture, Lodi, Italy.\n",
      "FAU - Groot, Steven P C\n",
      "AU  - Groot SPC\n",
      "AD  - Wageningen Plant Research, Wageningen University & Research, Wageningen,Netherlands.\n",
      "FAU - Kodde, Jan\n",
      "AU  - Kodde J\n",
      "AD  - Wageningen Plant Research, Wageningen University & Research, Wageningen,Netherlands.\n",
      "FAU - Kumari, Aprajita\n",
      "AU  - Kumari A\n",
      "AD  - National Institute of Plant Genome Research, New Delhi, India.\n",
      "FAU - Gupta, Kapuganti Jagadis\n",
      "AU  - Gupta KJ\n",
      "AD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University ofEvora, Evora, Portugal.\n",
      "AD  - National Institute of Plant Genome Research, New Delhi, India.\n",
      "FAU - Arnholdt-Schmitt, Birgit\n",
      "AU  - Arnholdt-Schmitt B\n",
      "AD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University ofEvora, Evora, Portugal.\n",
      "AD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry andMolecular Biology, Federal University of Ceara, Fortaleza, Brazil.\n",
      "AD  - CERNAS-Research Center for Natural Resources, Environment and Society, Departmentof Environment, Escola Superior Agraria de Coimbra, Coimbra, Portugal.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20190920\n",
      "PL  - Switzerland\n",
      "TA  - Front Plant Sci\n",
      "JT  - Frontiers in plant science\n",
      "JID - 101568200\n",
      "PMC - PMC6776121\n",
      "OTO - NOTNLM\n",
      "OT  - developmental plasticity\n",
      "OT  - endophytes\n",
      "OT  - environmental stress\n",
      "OT  - metabolic biomarker\n",
      "OT  - plant performance prediction\n",
      "OT  - seed technology\n",
      "EDAT- 2019/10/16 06:00\n",
      "MHDA- 2019/10/16 06:01\n",
      "CRDT- 2019/10/16 06:00\n",
      "PHST- 2019/04/06 00:00 [received]\n",
      "PHST- 2019/08/16 00:00 [accepted]\n",
      "PHST- 2019/10/16 06:00 [entrez]\n",
      "PHST- 2019/10/16 06:00 [pubmed]\n",
      "PHST- 2019/10/16 06:01 [medline]\n",
      "AID - 10.3389/fpls.2019.01134 [doi]\n",
      "PST - epublish\n",
      "SO  - Front Plant Sci. 2019 Sep 20;10:1134. doi: 10.3389/fpls.2019.01134. eCollection2019.\n",
      "\n",
      "PMID- 31464148\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20201119\n",
      "LR  - 20201119\n",
      "IS  - 1473-0766 (Electronic)\n",
      "IS  - 0951-3590 (Linking)\n",
      "VI  - 36\n",
      "IP  - 1\n",
      "DP  - 2020 Jan\n",
      "TI  - A report of congenital adrenal hyperplasia due to 17alpha-hydroxylase deficiency in two 46,XX sisters.\n",
      "PG  - 24-29\n",
      "LID - 10.1080/09513590.2019.1650342 [doi]\n",
      "AB  - Congenital adrenal hyperplasia (CAH) is a group of rare orphan disorders causedby mutations in seven different enzymes that impair cortisol biosynthesis. The17alpha-hydroxylase deficiency (17OHD) is one of the less common forms of CAH,corresponding to approximately 1% of the cases, with an estimated annualincidence of 1 in 50,000 newborns. Cases description - two phenotypically female Ecuadorian sisters, both with primary amenorrhea, absence of secondary sexualcharacteristics, and osteoporosis. High blood pressure was present in the oldersister. Hypergonadotropic hypogonadism profile was observed: decreased cortisoland dehydroepiandrosterone sulfate (DHEAS), increased adrenocorticotropic hormone(ACTH) and normal levels of 17-hydroxyprogesterone, extremely highdeoxycorticosterone (DOC) levels, and a tomography showed bilateral adrenalhyperplasia in both sisters. Consanguinity was evident in their ancestors.Furthermore, in the exon 7, the variant c.1216T > C, p.Trp406Arg was detected in homozygosis in the CYP17A1 gene of both sisters. We report a homozygous missense mutation in the CYP17A1 gene causing 17OHD in two sisters from Loja, Ecuador.According to the authors, this is the first time such deficiency and mutation aredescribed in two members of the same family in Ecuador.\n",
      "FAU - Espinosa-Herrera, Fernando\n",
      "AU  - Espinosa-Herrera F\n",
      "AUID- ORCID: http://orcid.org/0000-0002-7762-5688\n",
      "AD  - Escuela de Medicina, Facultad de Ciencias de la Salud, Universidad de lasAmericas (UDLA), Quito, Ecuador.\n",
      "AD  - Sociedad Ecuatoriana de Medicina Familiar (SEMF), Hospital Vozandes Quito, Quito,Ecuador.\n",
      "FAU - Espin, Estefania\n",
      "AU  - Espin E\n",
      "AD  - Escuela de Medicina, Facultad de Ciencias de la Salud, Universidad de lasAmericas (UDLA), Quito, Ecuador.\n",
      "FAU - Tito-Alvarez, Ana M\n",
      "AU  - Tito-Alvarez AM\n",
      "AD  - Escuela de Enfermeria, Facultad de Ciencias de la Salud, Universidad de lasAmericas (UDLA), Quito, Ecuador.\n",
      "FAU - Beltran, Leonardo-J\n",
      "AU  - Beltran LJ\n",
      "AD  - Escuela de Medicina, Facultad de Ciencias de la Salud, Universidad de lasAmericas (UDLA), Quito, Ecuador.\n",
      "FAU - Gomez-Correa, Diego\n",
      "AU  - Gomez-Correa D\n",
      "AD  - Escuela de Medicina, Facultad de Ciencias de la Salud, Universidad de lasAmericas (UDLA), Quito, Ecuador.\n",
      "FAU - Burgos, German\n",
      "AU  - Burgos G\n",
      "AD  - Escuela de Medicina, Facultad de Ciencias de la Salud, Universidad de lasAmericas (UDLA), Quito, Ecuador.\n",
      "FAU - Llamos, Arianne\n",
      "AU  - Llamos A\n",
      "AD  - Facultad de Ciencias Medicas, de la Salud y de La Vida, Escuela de Odontologia,Universidad Internacional del Ecuador (UIDE), Quito, Ecuador.\n",
      "FAU - Zurita, Camilo\n",
      "AU  - Zurita C\n",
      "AD  - Unidad de Investigaciones en Biomedicina, Zurita & Zurita Laboratorios, Catedrade Inmunologia, Facultad de Medicina, Universidad Central del Ecuador (UCE),Quito, Ecuador.\n",
      "FAU - Rojas, Samantha\n",
      "AU  - Rojas S\n",
      "AD  - Hospital Isidro Ayora de Loja, Ministerio de Salud Publica del Ecuador, Quito,Ecuador.\n",
      "FAU - Duenas-Espin, Ivan\n",
      "AU  - Duenas-Espin I\n",
      "AD  - Instituto de Salud Publica, Facultad de Medicina, Pontificia Universidad Catolicadel Ecuador, Quito, Ecuador.\n",
      "FAU - Cueva-Ludena, Kenny\n",
      "AU  - Cueva-Ludena K\n",
      "AD  - Hospital General Docente de Calderon, Ministerio de Salud Publica del Ecuador,Quito, Ecuador.\n",
      "FAU - Salazar-Vega, Jorge\n",
      "AU  - Salazar-Vega J\n",
      "AD  - Escuela de Medicina, Facultad de Ciencias de la Salud, Universidad de lasAmericas (UDLA), Quito, Ecuador.\n",
      "AD  - Hospital Eugenio Espejo, Ministerio de Salud Publica del Ecuador, Quito, Ecuador.\n",
      "FAU - Pinto-Basto, Jorge\n",
      "AU  - Pinto-Basto J\n",
      "AD  - Molecular Diagnostics and Clinical Genomics Laboratories, CGC Genetics, Porto,Portugal.\n",
      "LA  - eng\n",
      "PT  - Case Reports\n",
      "PT  - Journal Article\n",
      "DEP - 20190829\n",
      "PL  - England\n",
      "TA  - Gynecol Endocrinol\n",
      "JT  - Gynecological endocrinology : the official journal of the International Societyof Gynecological Endocrinology\n",
      "JID - 8807913\n",
      "RN  - 40GP35YQ49 (Desoxycorticosterone)\n",
      "RN  - 57B09Q7FJR (Dehydroepiandrosterone Sulfate)\n",
      "RN  - 68-96-2 (17-alpha-Hydroxyprogesterone)\n",
      "RN  - 9002-60-2 (Adrenocorticotropic Hormone)\n",
      "RN  - EC 1.14.14.19 (CYP17A1 protein, human)\n",
      "RN  - EC 1.14.14.19 (Steroid 17-alpha-Hydroxylase)\n",
      "RN  - WI4X0X7BPJ (Hydrocortisone)\n",
      "SB  - IM\n",
      "MH  - 17-alpha-Hydroxyprogesterone/metabolism\n",
      "MH  - Adrenal Hyperplasia, Congenital/complications/*genetics/metabolism\n",
      "MH  - Adrenocorticotropic Hormone/metabolism\n",
      "MH  - Amenorrhea/etiology\n",
      "MH  - Consanguinity\n",
      "MH  - Dehydroepiandrosterone Sulfate/metabolism\n",
      "MH  - Desoxycorticosterone/metabolism\n",
      "MH  - Diagnostic Errors\n",
      "MH  - Ecuador\n",
      "MH  - Female\n",
      "MH  - Homozygote\n",
      "MH  - Humans\n",
      "MH  - Hydrocortisone/metabolism\n",
      "MH  - Hypertension/etiology\n",
      "MH  - Hypogonadism/etiology/metabolism\n",
      "MH  - Hypokalemia/etiology\n",
      "MH  - Mosaicism\n",
      "MH  - Osteoporosis/etiology\n",
      "MH  - *Siblings\n",
      "MH  - Steroid 17-alpha-Hydroxylase/*genetics\n",
      "MH  - Turner Syndrome/diagnosis\n",
      "MH  - Young Adult\n",
      "OTO - NOTNLM\n",
      "OT  - 17 alpha hydroxylase deficiency\n",
      "OT  - Congenital adrenal hyperplasia\n",
      "OT  - consanguineous family\n",
      "OT  - sisters\n",
      "EDAT- 2019/08/30 06:00\n",
      "MHDA- 2020/11/20 06:00\n",
      "CRDT- 2019/08/30 06:00\n",
      "PHST- 2019/08/30 06:00 [pubmed]\n",
      "PHST- 2020/11/20 06:00 [medline]\n",
      "PHST- 2019/08/30 06:00 [entrez]\n",
      "AID - 10.1080/09513590.2019.1650342 [doi]\n",
      "PST - ppublish\n",
      "SO  - Gynecol Endocrinol. 2020 Jan;36(1):24-29. doi: 10.1080/09513590.2019.1650342.Epub 2019 Aug 29.\n",
      "\n",
      "PMID- 31417618\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20201001\n",
      "IS  - 1752-4571 (Print)\n",
      "IS  - 1752-4571 (Linking)\n",
      "VI  - 12\n",
      "IP  - 7\n",
      "DP  - 2019 Aug\n",
      "TI  - Urbanization erodes niche segregation in Darwin's finches.\n",
      "PG  - 1329-1343\n",
      "LID - 10.1111/eva.12721 [doi]\n",
      "AB  - Urbanization is influencing patterns of biological evolution in ways that areonly beginning to be explored. One potential effect of urbanization is inmodifying ecological resource distributions that underlie niche differences andthat thus promote and maintain species diversification. Few studies have assessedsuch modifications, or their potential evolutionary consequences, in the context of ongoing adaptive radiation. We study this effect in Darwin's finches on theGalapagos Islands, by quantifying feeding preferences and diet niche partitioningacross sites with different degrees of urbanization. We found higher finchdensity in urban sites and that feeding preferences and diets at urban sites skewheavily toward human food items. Furthermore, we show that finches at urban sitesappear to be accustomed to the presence of people, compared with birds at siteswith few people. In addition, we found that human behavior via the tendency tofeed birds at non-urban but tourist sites is likely an important driver of finch preferences for human foods. Site differences in diet and feeding behavior haveresulted in larger niche breadth within finch species and wider niche overlapbetween species at the urban sites. Both factors effectively minimize nichedifferences that would otherwise facilitate interspecies coexistence. Thesefindings suggest that both human behavior and ongoing urbanization in Galapagosare starting to erode ecological differences that promote and maintain adaptiveradiation in Darwin's finches. Smoothing of adaptive landscapes underlyingdiversification represents a potentially important yet underappreciatedconsequence of urbanization. Overall, our findings accentuate the fragility ofthe initial stages of adaptive radiation in Darwin's finches and raise concernsabout the fate of the Galapagos ecosystems in the face of increasingurbanization.\n",
      "FAU - De Leon, Luis F\n",
      "AU  - De Leon LF\n",
      "AUID- ORCID: 0000-0001-9317-420X\n",
      "AD  - Department of Biology University of Massachusetts Boston Boston Massachusetts.\n",
      "AD  - Centro de Biodiversidad y Descubrimiento de Drogas Instituto de InvestigacionesCientificas y Servicios de Alta Tecnologia (INDICASAT-AIP) Panama City Panama.\n",
      "FAU - Sharpe, Diana M T\n",
      "AU  - Sharpe DMT\n",
      "AD  - Redpath Museum, Department of Biology McGill University Montreal Quebec Canada.\n",
      "FAU - Gotanda, Kiyoko M\n",
      "AU  - Gotanda KM\n",
      "AD  - Redpath Museum, Department of Biology McGill University Montreal Quebec Canada.\n",
      "AD  - Department of Zoology University of Cambridge Cambridge UK.\n",
      "FAU - Raeymaekers, Joost A M\n",
      "AU  - Raeymaekers JAM\n",
      "AD  - Centre for Biodiversity Dynamics, Department of Biology Norwegian University ofScience and Technology Trondheim Norway.\n",
      "AD  - Laboratory of Biodiversity and Evolutionary Genomics University of Leuven Leuven Belgium.\n",
      "FAU - Chaves, Jaime A\n",
      "AU  - Chaves JA\n",
      "AD  - Colegio de Ciencias Biologicas y Ambientales Universidad San Francisco de Quito, Diego de Robles y Pampite Quito Ecuador.\n",
      "AD  - Galapagos Science Center Puerto Baquerizo Moreno Galapagos Ecuador.\n",
      "FAU - Hendry, Andrew P\n",
      "AU  - Hendry AP\n",
      "AD  - Redpath Museum, Department of Biology McGill University Montreal Quebec Canada.\n",
      "FAU - Podos, Jeffrey\n",
      "AU  - Podos J\n",
      "AD  - Department of Biology University of Massachusetts Amherst Amherst Massachusetts.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20181218\n",
      "PL  - England\n",
      "TA  - Evol Appl\n",
      "JT  - Evolutionary applications\n",
      "JID - 101461828\n",
      "PMC - PMC6691225\n",
      "OTO - NOTNLM\n",
      "OT  - Galapagos\n",
      "OT  - adaptive divergence\n",
      "OT  - human disturbances\n",
      "OT  - maladaptation\n",
      "OT  - niche partitioning\n",
      "OT  - resource distribution\n",
      "OT  - urban ecology\n",
      "OT  - urbanization\n",
      "COIS- None declared.\n",
      "EDAT- 2019/08/17 06:00\n",
      "MHDA- 2019/08/17 06:01\n",
      "CRDT- 2019/08/17 06:00\n",
      "PHST- 2018/05/15 00:00 [received]\n",
      "PHST- 2018/10/01 00:00 [revised]\n",
      "PHST- 2018/10/02 00:00 [accepted]\n",
      "PHST- 2019/08/17 06:00 [entrez]\n",
      "PHST- 2019/08/17 06:00 [pubmed]\n",
      "PHST- 2019/08/17 06:01 [medline]\n",
      "AID - 10.1111/eva.12721 [doi]\n",
      "AID - EVA12721 [pii]\n",
      "PST - epublish\n",
      "SO  - Evol Appl. 2018 Dec 18;12(7):1329-1343. doi: 10.1111/eva.12721. eCollection 2019 Aug.\n",
      "\n",
      "PMID- 31376146\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200727\n",
      "LR  - 20211204\n",
      "IS  - 1532-6535 (Electronic)\n",
      "IS  - 0009-9236 (Linking)\n",
      "VI  - 107\n",
      "IP  - 1\n",
      "DP  - 2020 Jan\n",
      "TI  - Genomic Ancestry, CYP2D6, CYP2C9, and CYP2C19 Among Latin Americans.\n",
      "PG  - 257-268\n",
      "LID - 10.1002/cpt.1598 [doi]\n",
      "AB  - We present the distribution of CYP2D6, CYP2C9, and CYP2C19 variants and predictedphenotypes in 33 native and admixed populations from Ibero-America (n > 6,000) inthe context of genetic ancestry (n = 3,387). Continental ancestries are the majordeterminants of frequencies of the increased-activity allele CYP2C19*17 andCYP2C19 gUMs (negatively associated with Native American ancestry),decreased-activity alleles CYP2D6*41 and CYP2C9*2 (positively associated withEuropean ancestry), and decreased-activity alleles CYP2D6*17 and CYP2D6*29(positively associated with African ancestry). For the rare alleles, CYP2C9*2 andCYPC19*17, European admixture accounts for their presence in Native Americanpopulations, but rare alleles CYP2D6*5 (null-activity), CYP2D6-multiplicationalleles (increased activity), and CYP2C9*3 (decreased-activity) were present inthe pre-Columbian Americas. The study of a broad spectrum of Native Americanpopulations from different ethno-linguistic groups show how autochthonousdiversity shaped the distribution of pharmaco-alleles and give insights on theprevalence of clinically relevant phenotypes associated with drugs, such asparoxetine, tamoxifen, warfarin, and clopidogrel.\n",
      "CI  - (c) 2019 The Authors Clinical Pharmacology & Therapeutics (c) 2019 AmericanSociety for Clinical Pharmacology and Therapeutics.\n",
      "FAU - Rodrigues-Soares, Fernanda\n",
      "AU  - Rodrigues-Soares F\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Departamento de Genetica, Ecologia e Evolucao, Universidade Federal de MinasGerais, Belo Horizonte, Minas Gerais, Brazil.\n",
      "AD  - Departamento de Patologia, Genetica e Evolucao, Universidade Federal do TrianguloMineiro, Uberaba, Minas Gerais, Brazil.\n",
      "FAU - Penas-Lledo, Eva M\n",
      "AU  - Penas-Lledo EM\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Instituto de Investigacion Biosanitaria de Extremadura, Universidad deExtremadura, SES, Badajoz, Extremadura, Spain.\n",
      "FAU - Tarazona-Santos, Eduardo\n",
      "AU  - Tarazona-Santos E\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Departamento de Genetica, Ecologia e Evolucao, Universidade Federal de MinasGerais, Belo Horizonte, Minas Gerais, Brazil.\n",
      "AD  - Universidad Peruana Cayetano Heredia, Lima, Peru.\n",
      "AD  - Instituto de Estudos Avancados Transdisciplinares, Universidade Federal de Minas Gerais, Belo Horizonte, Minas Gerais, Brazil.\n",
      "FAU - Sosa-Macias, Martha\n",
      "AU  - Sosa-Macias M\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Instituto Politecnico Nacional, Durango, Mexico.\n",
      "FAU - Teran, Enrique\n",
      "AU  - Teran E\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Colegio de Ciencias de la Salud, Universidad San Francisco de Quito, Quito,Ecuador.\n",
      "FAU - Lopez-Lopez, Marisol\n",
      "AU  - Lopez-Lopez M\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Universidad Autonoma Metropolitana, Ciudad de Mexico, Mexico.\n",
      "FAU - Rodeiro, Idania\n",
      "AU  - Rodeiro I\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Instituto de Ciencias del Mar, La Habana, Cuba.\n",
      "FAU - Moya, Graciela E\n",
      "AU  - Moya GE\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Universidad Catolica de Argentina, Buenos Aires, Argentina.\n",
      "FAU - Calzadilla, Luis R\n",
      "AU  - Calzadilla LR\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Centro Comunitario de Salud Mental, La Habana Vieja, La Habana, Cuba.\n",
      "FAU - Ramirez-Roa, Ronald\n",
      "AU  - Ramirez-Roa R\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Universidad Nacional Autonoma de Nicaragua, Leon, Nicaragua.\n",
      "FAU - Grazina, Manuela\n",
      "AU  - Grazina M\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Faculdade de Medicina & Centro de Neurociencias e Biologia Celular, University ofCoimbra, Coimbra, Portugal.\n",
      "FAU - Estevez-Carrizo, Francisco E\n",
      "AU  - Estevez-Carrizo FE\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Universidad de Montevideo, Montevideo, Uruguay.\n",
      "FAU - Barrantes, Ramiro\n",
      "AU  - Barrantes R\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Universidad de Costa Rica, San Jose, Costa Rica.\n",
      "FAU - LLerena, Adrian\n",
      "AU  - LLerena A\n",
      "AD  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics, Badajoz,Extremadura, Spain.\n",
      "AD  - Instituto de Investigacion Biosanitaria de Extremadura, Universidad deExtremadura, SES, Badajoz, Extremadura, Spain.\n",
      "CN  - RIBEF Ibero-American Network of Pharmacogenetics and Pharmacogenomics\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20191007\n",
      "PL  - United States\n",
      "TA  - Clin Pharmacol Ther\n",
      "JT  - Clinical pharmacology and therapeutics\n",
      "JID - 0372741\n",
      "RN  - EC 1.14.13.- (CYP2C9 protein, human)\n",
      "RN  - EC 1.14.13.- (Cytochrome P-450 CYP2C9)\n",
      "RN  - EC 1.14.14.1 (CYP2C19 protein, human)\n",
      "RN  - EC 1.14.14.1 (Cytochrome P-450 CYP2C19)\n",
      "RN  - EC 1.14.14.1 (Cytochrome P-450 CYP2D6)\n",
      "SB  - IM\n",
      "MH  - Alleles\n",
      "MH  - Blacks/genetics\n",
      "MH  - Cytochrome P-450 CYP2C19/*genetics\n",
      "MH  - Cytochrome P-450 CYP2C9/*genetics\n",
      "MH  - Cytochrome P-450 CYP2D6/*genetics\n",
      "MH  - Genetic Variation\n",
      "MH  - Genomics\n",
      "MH  - Humans\n",
      "MH  - Indians, North American/genetics\n",
      "MH  - Latin America\n",
      "MH  - Phenotype\n",
      "MH  - Racial Groups/*genetics\n",
      "MH  - Whites/genetics\n",
      "IR  - Tinoco CA\n",
      "FIR - Tinoco, Catalina Altamirano\n",
      "IR  - Alvarez M\n",
      "FIR - Alvarez, Mayra\n",
      "IR  - Borbon A\n",
      "FIR - Borbon, Angelica\n",
      "IR  - Cespedes-Garro C\n",
      "FIR - Cespedes-Garro, Carolina\n",
      "IR  - Cobaleda J\n",
      "FIR - Cobaleda, Jesus\n",
      "IR  - de Andres F\n",
      "FIR - de Andres, Fernando\n",
      "IR  - Delgado R\n",
      "FIR - Delgado, Rene\n",
      "IR  - Dorado P\n",
      "FIR - Dorado, Pedro\n",
      "IR  - Farinas H\n",
      "FIR - Farinas, Humberto\n",
      "IR  - Ferreiro V\n",
      "FIR - Ferreiro, Veronica\n",
      "IR  - Fricke-Galindo I\n",
      "FIR - Fricke-Galindo, Ingrid\n",
      "IR  - Galaviz-Hernandez C\n",
      "FIR - Galaviz-Hernandez, Carlos\n",
      "IR  - Garza-Ocanas L\n",
      "FIR - Garza-Ocanas, Lourdes\n",
      "IR  - Gilman RH\n",
      "FIR - Gilman, Robert H\n",
      "IR  - Hernandez F\n",
      "FIR - Hernandez, Francisco\n",
      "IR  - Jimenez-Arce G\n",
      "FIR - Jimenez-Arce, Gerardo\n",
      "IR  - Jung-Cook H\n",
      "FIR - Jung-Cook, Helgi\n",
      "IR  - Lares-Aseff I\n",
      "FIR - Lares-Aseff, Ismael\n",
      "IR  - Lazalde-Ramos BP\n",
      "FIR - Lazalde-Ramos, Blanca P\n",
      "IR  - Michelin L\n",
      "FIR - Michelin, Lucas\n",
      "IR  - Monroy-Jaramillo N\n",
      "FIR - Monroy-Jaramillo, Nancy\n",
      "IR  - Naranjo MEG\n",
      "FIR - Naranjo, Maria Eugenia G\n",
      "IR  - Ortega-Vazquez A\n",
      "FIR - Ortega-Vazquez, Alberto\n",
      "IR  - Ortiz-Lopez R\n",
      "FIR - Ortiz-Lopez, Rocio\n",
      "IR  - Perez B\n",
      "FIR - Perez, Barbaro\n",
      "IR  - Perez-Paramo YX\n",
      "FIR - Perez-Paramo, Yadira X\n",
      "IR  - Remirez D\n",
      "FIR - Remirez, Diadelis\n",
      "IR  - Rojas-Martinez A\n",
      "FIR - Rojas-Martinez, Augusto\n",
      "IR  - Sarmiento AP\n",
      "FIR - Sarmiento, Alba P\n",
      "IR  - Scliar M\n",
      "FIR - Scliar, Marilia\n",
      "IR  - Teran S\n",
      "FIR - Teran, Santiago\n",
      "IR  - Zamudio R\n",
      "FIR - Zamudio, Roxana\n",
      "EDAT- 2019/08/04 06:00\n",
      "MHDA- 2020/07/28 06:00\n",
      "CRDT- 2019/08/04 06:00\n",
      "PHST- 2019/05/10 00:00 [received]\n",
      "PHST- 2019/07/25 00:00 [accepted]\n",
      "PHST- 2019/08/04 06:00 [pubmed]\n",
      "PHST- 2020/07/28 06:00 [medline]\n",
      "PHST- 2019/08/04 06:00 [entrez]\n",
      "AID - 10.1002/cpt.1598 [doi]\n",
      "PST - ppublish\n",
      "SO  - Clin Pharmacol Ther. 2020 Jan;107(1):257-268. doi: 10.1002/cpt.1598. Epub 2019Oct 7.\n",
      "\n",
      "PMID- 31350885\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200217\n",
      "LR  - 20200217\n",
      "IS  - 1537-1719 (Electronic)\n",
      "IS  - 0737-4038 (Linking)\n",
      "VI  - 36\n",
      "IP  - 12\n",
      "DP  - 2019 Dec 1\n",
      "TI  - The Current Genomic Landscape of Western South America: Andes, Amazonia, andPacific Coast.\n",
      "PG  - 2698-2713\n",
      "LID - 10.1093/molbev/msz174 [doi]\n",
      "AB  - Studies of Native South American genetic diversity have helped to shed light onthe peopling and differentiation of the continent, but available data are sparse for the major ecogeographic domains. These include the Pacific Coast, a potentialearly migration route; the Andes, home to the most expansive complex societiesand to one of the most widely spoken indigenous language families of thecontinent (Quechua); and Amazonia, with its understudied population structure andrich cultural diversity. Here, we explore the genetic structure of 176individuals from these three domains, genotyped with the Affymetrix Human Originsarray. We infer multiple sources of ancestry within the Native American ancestry component; one with clear predominance on the Coast and in the Andes, and atleast two distinct substrates in neighboring Amazonia, including a previouslyundetected ancestry characteristic of northern Ecuador and Colombia. Amazonianpopulations are also involved in recent gene-flow with each other and acrossecogeographic domains, which does not accord with the traditional view of small, isolated groups. Long-distance genetic connections between speakers of the samelanguage family suggest that indigenous languages here were spread not bycultural contact alone. Finally, Native American populations admixed withpost-Columbian European and African sources at different times, with few cases ofprolonged isolation. With our results we emphasize the importance of includingunderstudied regions of the continent in high-resolution genetic studies, and we illustrate the potential of SNP chip arrays for informative regional-scaleanalysis.\n",
      "CI  - (c) The Author(s) 2019. Published by Oxford University Press on behalf of theSociety for Molecular Biology and Evolution.\n",
      "FAU - Barbieri, Chiara\n",
      "AU  - Barbieri C\n",
      "AD  - Department of Linguistic and Cultural Evolution, Max Planck Institute for theScience of Human History, Jena, Germany.\n",
      "AD  - Department of Evolutionary Biology and Environmental Studies, University ofZurich, Zurich, Switzerland.\n",
      "FAU - Barquera, Rodrigo\n",
      "AU  - Barquera R\n",
      "AD  - Department of Archaeogenetics, Max Planck Institute for the Science of HumanHistory, Jena, Germany.\n",
      "FAU - Arias, Leonardo\n",
      "AU  - Arias L\n",
      "AD  - Department of Evolutionary Genetics, Max Planck Institute for EvolutionaryAnthropology, Leipzig, Germany.\n",
      "FAU - Sandoval, Jose R\n",
      "AU  - Sandoval JR\n",
      "AD  - Centro de Investigacion de Genetica y Biologia Molecular (CIGBM), Universidad de San Martin de Porres, Lima, Peru.\n",
      "FAU - Acosta, Oscar\n",
      "AU  - Acosta O\n",
      "AD  - Centro de Investigacion de Genetica y Biologia Molecular (CIGBM), Universidad de San Martin de Porres, Lima, Peru.\n",
      "FAU - Zurita, Camilo\n",
      "AU  - Zurita C\n",
      "AD  - Catedra de Inmunologia, Facultad de Medicina, Universidad Central del Ecuador,Quito, Ecuador.\n",
      "AD  - Zurita & Zurita Laboratorios, Unidad de Investigaciones en Biomedicina, Quito,Ecuador.\n",
      "FAU - Aguilar-Campos, Abraham\n",
      "AU  - Aguilar-Campos A\n",
      "AD  - Clinical Laboratory, Unidad Medica de Alta Especialidad (UMAE) # 2, InstitutoMexicano del Seguro Social (IMSS), Ciudad Obregon, Sonora, Mexico.\n",
      "FAU - Tito-Alvarez, Ana M\n",
      "AU  - Tito-Alvarez AM\n",
      "AD  - Carrera de Enfermeria, Facultad de Ciencias de la Salud, Universidad de LasAmericas, Quito, Ecuador.\n",
      "FAU - Serrano-Osuna, Ricardo\n",
      "AU  - Serrano-Osuna R\n",
      "AD  - Clinical Laboratory, Unidad Medica de Alta Especialidad (UMAE) # 2, InstitutoMexicano del Seguro Social (IMSS), Ciudad Obregon, Sonora, Mexico.\n",
      "FAU - Gray, Russell D\n",
      "AU  - Gray RD\n",
      "AD  - Department of Linguistic and Cultural Evolution, Max Planck Institute for theScience of Human History, Jena, Germany.\n",
      "FAU - Mafessoni, Fabrizio\n",
      "AU  - Mafessoni F\n",
      "AD  - Department of Evolutionary Genetics, Max Planck Institute for EvolutionaryAnthropology, Leipzig, Germany.\n",
      "FAU - Heggarty, Paul\n",
      "AU  - Heggarty P\n",
      "AD  - Department of Linguistic and Cultural Evolution, Max Planck Institute for theScience of Human History, Jena, Germany.\n",
      "FAU - Shimizu, Kentaro K\n",
      "AU  - Shimizu KK\n",
      "AD  - Department of Evolutionary Biology and Environmental Studies, University ofZurich, Zurich, Switzerland.\n",
      "FAU - Fujita, Ricardo\n",
      "AU  - Fujita R\n",
      "AD  - Centro de Investigacion de Genetica y Biologia Molecular (CIGBM), Universidad de San Martin de Porres, Lima, Peru.\n",
      "FAU - Stoneking, Mark\n",
      "AU  - Stoneking M\n",
      "AD  - Department of Evolutionary Genetics, Max Planck Institute for EvolutionaryAnthropology, Leipzig, Germany.\n",
      "FAU - Pugach, Irina\n",
      "AU  - Pugach I\n",
      "AD  - Department of Evolutionary Genetics, Max Planck Institute for EvolutionaryAnthropology, Leipzig, Germany.\n",
      "FAU - Fehren-Schmitz, Lars\n",
      "AU  - Fehren-Schmitz L\n",
      "AD  - UCSC Paleogenomics, Department of Anthropology, University of California, SantaCruz, CA.\n",
      "AD  - Genomics Institute, University of California, Santa Cruz, CA.\n",
      "LA  - eng\n",
      "PT  - Historical Article\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "PL  - United States\n",
      "TA  - Mol Biol Evol\n",
      "JT  - Molecular biology and evolution\n",
      "JID - 8501455\n",
      "SB  - IM\n",
      "MH  - *Genome, Human\n",
      "MH  - History, Ancient\n",
      "MH  - Human Migration/*history\n",
      "MH  - Humans\n",
      "MH  - Language\n",
      "MH  - Peru\n",
      "MH  - Phylogeography\n",
      "PMC - PMC6878948\n",
      "OTO - NOTNLM\n",
      "OT  - *Native American population genetics\n",
      "OT  - *South American prehistory\n",
      "OT  - *admixture\n",
      "OT  - *human migration\n",
      "OT  - *runs of homozygosity\n",
      "EDAT- 2019/07/28 06:00\n",
      "MHDA- 2020/02/18 06:00\n",
      "CRDT- 2019/07/28 06:00\n",
      "PHST- 2019/07/28 06:00 [pubmed]\n",
      "PHST- 2020/02/18 06:00 [medline]\n",
      "PHST- 2019/07/28 06:00 [entrez]\n",
      "AID - 5539872 [pii]\n",
      "AID - 10.1093/molbev/msz174 [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Biol Evol. 2019 Dec 1;36(12):2698-2713. doi: 10.1093/molbev/msz174.\n",
      "\n",
      "PMID- 31284667\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20191217\n",
      "LR  - 20200225\n",
      "IS  - 1420-3049 (Electronic)\n",
      "IS  - 1420-3049 (Linking)\n",
      "VI  - 24\n",
      "IP  - 13\n",
      "DP  - 2019 Jul 7\n",
      "TI  - Evaluation of Glycosyl-Hydrolases, Phosphatases, Esterases and Proteases asPotential Biomarker for NaCl-Stress Tolerance in Solanum lycopersicum L.Varieties.\n",
      "LID - E2488 [pii]\n",
      "LID - 10.3390/molecules24132488 [doi]\n",
      "AB  - Salinity stress limited the production in over 30% of irrigated crops and 7% ofdryland agriculture worldwide. The objective was to evaluate the effects ofNaCl-stress on the enzymatic activity in tomato. Two experiments were carried outin germination and early vegetative growth stages. The activity of proline andperoxidase of eight varieties (Missouri, Yaqui, Vita, Feroz, Rio Grande, Tropic, Ace, and Floradade) submitted to NaCl concentrations (0, 50, 100, 150 and 200 mM de NaCl) and the semi-quantitative activity of 19 enzymes APY ZYM((R)) weremeasured under a completely randomized design with four replications. Data wereanalyzed using univariate-multivariate analysis of variance, Tukey's HSD (p =0.05), canonical discriminant and cluster analysis. The results showedsignificant differences between varieties and NaCl in proline content. Prolineincreased as the NaCl concentration increased. Peroxidase did no show significantdifferences. Eight enzymes were included within the model to properly classifythe varieties and NaCl. In shoots, varieties and NaCl showed that enzymaticactivity was higher in the order of alkaline-phosphatase > leucine arylamidase > acid phosphatase > naphthol-AS-BI-phosphohydrolase >n-acetyl-beta-glucosaminidase > beta-galactosidase, while in roots was higher in the order of alkaline-phosphatase > naphthol-AS-BI-phosphohydrolase > acidphosphatase > n-acetyl-beta-glucosaminidase. Acid and alkali phosphatase, lipase,esterase, beta-galactosidase, and trypsin can be a potential biomarker forNaCl-stress tolerance in tomato.\n",
      "FAU - Reyes-Perez, Juan Jose\n",
      "AU  - Reyes-Perez JJ\n",
      "AUID- ORCID: 0000-0001-5372-2523\n",
      "AD  - Universidad Tecnica Estatal de Quevedo, Quevedo, Los Rios 120501, Ecuador.\n",
      "FAU - Ruiz-Espinoza, Francisco Higinio\n",
      "AU  - Ruiz-Espinoza FH\n",
      "AD  - Universidad Autonoma de Baja California Sur, La Paz, Baja California Sur 23080,Mexico.\n",
      "FAU - Hernandez-Montiel, Luis Guillermo\n",
      "AU  - Hernandez-Montiel LG\n",
      "AUID- ORCID: 0000-0002-8236-1074\n",
      "AD  - Centro de Investigaciones Biologicas del Noroeste, S.C. La Paz, Baja CaliforniaSur 23096, Mexico; lhernandezcibnor.mx.\n",
      "FAU - de Lucia, Barbara\n",
      "AU  - de Lucia B\n",
      "AUID- ORCID: 0000-0001-7661-1818\n",
      "AD  - University of Bari Aldo Moro, Bari 70121, Italy.\n",
      "FAU - Cristiano, Giuseppe\n",
      "AU  - Cristiano G\n",
      "AUID- ORCID: 0000-0003-4646-3310\n",
      "AD  - University of Bari Aldo Moro, Bari 70121, Italy.\n",
      "FAU - Murillo-Amador, Bernardo\n",
      "AU  - Murillo-Amador B\n",
      "AUID- ORCID: 0000-0002-9489-4054\n",
      "AD  - Centro de Investigaciones Biologicas del Noroeste, S.C. La Paz, Baja CaliforniaSur 23096, Mexico; lhernandezcibnor.mx. bmurillo04@cibnor.mx.\n",
      "LA  - eng\n",
      "GR  - 4631/CONSEJO NACIONA DE CIENCIA Y TECNOLOGIA\n",
      "GR  - PAZA/CENTRO DE INVESTIGACIONES BIOLOGICAS DEL NOROESTE, S.C.\n",
      "GR  - 806/SATREPS-JICA\n",
      "PT  - Journal Article\n",
      "DEP - 20190707\n",
      "PL  - Switzerland\n",
      "TA  - Molecules\n",
      "JT  - Molecules (Basel, Switzerland)\n",
      "JID - 100964009\n",
      "RN  - 0 (Biomarkers)\n",
      "RN  - 0 (Proteome)\n",
      "RN  - 451W47IQ8X (Sodium Chloride)\n",
      "RN  - 9DLQ4CIU6V (Proline)\n",
      "RN  - EC 3.1.- (Esterases)\n",
      "RN  - EC 3.1.3.2 (Phosphoric Monoester Hydrolases)\n",
      "RN  - EC 3.2.2.- (N-Glycosyl Hydrolases)\n",
      "RN  - EC 3.4.- (Peptide Hydrolases)\n",
      "SB  - IM\n",
      "MH  - Biomarkers\n",
      "MH  - Cluster Analysis\n",
      "MH  - Enzyme Activation\n",
      "MH  - Esterases/*metabolism\n",
      "MH  - Lycopersicon esculentum/*drug effects/*physiology\n",
      "MH  - N-Glycosyl Hydrolases/*metabolism\n",
      "MH  - Peptide Hydrolases/*metabolism\n",
      "MH  - Phosphoric Monoester Hydrolases/*metabolism\n",
      "MH  - Plant Shoots/physiology\n",
      "MH  - Proline/metabolism\n",
      "MH  - Proteome\n",
      "MH  - Proteomics\n",
      "MH  - *Salt Tolerance\n",
      "MH  - Seedlings/physiology\n",
      "MH  - Sodium Chloride/*pharmacology\n",
      "PMC - PMC6651099\n",
      "OTO - NOTNLM\n",
      "OT  - biochemical indicators\n",
      "OT  - peroxidase\n",
      "OT  - proline\n",
      "OT  - salinity-stress tolerance\n",
      "OT  - semi-quantitative enzymes\n",
      "OT  - tomato\n",
      "EDAT- 2019/07/10 06:00\n",
      "MHDA- 2019/12/18 06:00\n",
      "CRDT- 2019/07/10 06:00\n",
      "PHST- 2019/05/27 00:00 [received]\n",
      "PHST- 2019/06/30 00:00 [revised]\n",
      "PHST- 2019/07/05 00:00 [accepted]\n",
      "PHST- 2019/07/10 06:00 [entrez]\n",
      "PHST- 2019/07/10 06:00 [pubmed]\n",
      "PHST- 2019/12/18 06:00 [medline]\n",
      "AID - molecules24132488 [pii]\n",
      "AID - 10.3390/molecules24132488 [doi]\n",
      "PST - epublish\n",
      "SO  - Molecules. 2019 Jul 7;24(13). pii: molecules24132488. doi:10.3390/molecules24132488.\n",
      "\n",
      "PMID- 31254307\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200330\n",
      "LR  - 20200330\n",
      "IS  - 1365-294X (Electronic)\n",
      "IS  - 0962-1083 (Linking)\n",
      "VI  - 28\n",
      "IP  - 13\n",
      "DP  - 2019 Jul\n",
      "TI  - Phylogeography of a cryptic speciation continuum in Eurasian spadefoot toads(Pelobates).\n",
      "PG  - 3257-3270\n",
      "LID - 10.1111/mec.15133 [doi]\n",
      "AB  - Cryptic phylogeographic diversifications provide unique models to examine therole of phylogenetic divergence on the evolution of reproductive isolation,without extrinsic factors such as ecological and behavioural differentiation.Yet, to date very few comparative studies have been attempted within suchradiations. Here, we characterize a new speciation continuum in a group ofwidespread Eurasian amphibians, the Pelobates spadefoot toads, by conductingmultilocus (restriction site associated DNA sequencing and mitochondrial DNA)phylogenetic, phylogeographic and hybrid zone analyses. Within the P. syriacuscomplex, we discovered species-level cryptic divergences (>5 million years ago[My]) between populations distributed in the Near-East (hereafter P. syriacussensu stricto [s.s.]) and southeastern Europe (hereafter P. balcanicus), eachfeaturing deep intraspecific lineages. Altogether, we could scale hybridizabilityto divergence time along six different stages, spanning from sympatry withoutgene flow (P. fuscus and P. balcanicus, >10 My), parapatry with highly restrictedhybridization (P. balcanicus and P. syriacus s.s., >5 My), narrow hybrid zones(~15 km) consistent with partial reproductive isolation (P. fuscus and P.vespertinus, ~3 My), to extensive admixture between Pleistocene and refugiallineages (</=2 My). This full spectrum empirically supports a gradual build up ofreproductive barriers through time, reversible up until a threshold that weestimate at ~3 My. Hence, cryptic phylogeographic lineages may fade away orbecome reproductively isolated species simply depending on the time they persist in allopatry, and without definite ecomorphological divergence.\n",
      "CI  - (c) 2019 John Wiley & Sons Ltd.\n",
      "FAU - Dufresnes, Christophe\n",
      "AU  - Dufresnes C\n",
      "AUID- ORCID: 0000-0002-8497-8908\n",
      "AD  - Department of Animal and Plant Sciences, University of Sheffield, Sheffield, UK.\n",
      "AD  - Hintermann & Weber SA, Montreux, Switzerland.\n",
      "AD  - Laboratory for Conservation Biology, University of Lausanne, Lausanne,Switzerland.\n",
      "FAU - Strachinis, Ilias\n",
      "AU  - Strachinis I\n",
      "AUID- ORCID: 0000-0001-9944-4522\n",
      "AD  - School of Biology, Aristotle University of Thessaloniki, Thessaloniki, Greece.\n",
      "FAU - Suriadna, Nataliia\n",
      "AU  - Suriadna N\n",
      "AUID- ORCID: 0000-0002-0681-4465\n",
      "AD  - Melitopol Insitute of Ecology and Social Technologies of University \"Ukraine\",Melitopol, Zaporizhia, Ukraine.\n",
      "FAU - Mykytynets, Galyna\n",
      "AU  - Mykytynets G\n",
      "AUID- ORCID: 0000-0001-9623-8088\n",
      "AD  - Pryazovsky National Nature Park, Melitopol, Zaporizhia, Ukraine.\n",
      "FAU - Cogalniceanu, Dan\n",
      "AU  - Cogalniceanu D\n",
      "AUID- ORCID: 0000-0003-2959-014X\n",
      "AD  - Faculty of Natural Sciences and Agricultural Sciences, University OvidiusConstanta, Constanta, Romania.\n",
      "FAU - Szekely, Paul\n",
      "AU  - Szekely P\n",
      "AUID- ORCID: 0000-0002-9252-1596\n",
      "AD  - Departamento de Ciencias Biologicas, EcoSs Lab, Universidad Tecnica Particular deLoja, Loja, Ecuador.\n",
      "FAU - Vukov, Tanja\n",
      "AU  - Vukov T\n",
      "AUID- ORCID: 0000-0002-2365-8679\n",
      "AD  - Department of Evolutionary Biology, Institute for Biological Research \"SinisaStankovic\", National Institute of Republic of Serbia, University of Belgrade,Belgrade, Serbia.\n",
      "FAU - Arntzen, Jan W\n",
      "AU  - Arntzen JW\n",
      "AUID- ORCID: 0000-0003-3229-5993\n",
      "AD  - Naturalis Biodiversity Center, Leiden, The Netherlands.\n",
      "FAU - Wielstra, Ben\n",
      "AU  - Wielstra B\n",
      "AUID- ORCID: 0000-0002-7112-5965\n",
      "AD  - Naturalis Biodiversity Center, Leiden, The Netherlands.\n",
      "AD  - Institute of Biology Leiden, Leiden University, Leiden, The Netherlands.\n",
      "FAU - Lymberakis, Petros\n",
      "AU  - Lymberakis P\n",
      "AUID- ORCID: 0000-0002-5067-9600\n",
      "AD  - Natural History Museum of Crete, University of Crete, Irakleio, Crete, Greece.\n",
      "FAU - Geffen, Eli\n",
      "AU  - Geffen E\n",
      "AUID- ORCID: 0000-0002-3028-0045\n",
      "AD  - School of Zoology, Tel Aviv University, Tel Aviv, Israel.\n",
      "FAU - Gafny, Sarig\n",
      "AU  - Gafny S\n",
      "AD  - School of Marine Sciences, Ruppin Academic Center, Michmoret, Israel.\n",
      "FAU - Kumlutas, Yusuf\n",
      "AU  - Kumlutas Y\n",
      "AUID- ORCID: 0000-0003-1154-6757\n",
      "AD  - Department of Biology, Faculty of Science, Dokuz Eylul University, Buca, Izmir,Turkey.\n",
      "AD  - Research and Application Center for Fauna and Flora, Dokuz Eylul University,Buca, Izmir, Turkey.\n",
      "FAU - Ilgaz, Cetin\n",
      "AU  - Ilgaz C\n",
      "AUID- ORCID: 0000-0001-7862-9106\n",
      "AD  - Department of Biology, Faculty of Science, Dokuz Eylul University, Buca, Izmir,Turkey.\n",
      "AD  - Research and Application Center for Fauna and Flora, Dokuz Eylul University,Buca, Izmir, Turkey.\n",
      "FAU - Candan, Kamil\n",
      "AU  - Candan K\n",
      "AUID- ORCID: 0000-0002-6934-3971\n",
      "AD  - Department of Biology, Faculty of Science, Dokuz Eylul University, Buca, Izmir,Turkey.\n",
      "AD  - Research and Application Center for Fauna and Flora, Dokuz Eylul University,Buca, Izmir, Turkey.\n",
      "FAU - Mizsei, Edvard\n",
      "AU  - Mizsei E\n",
      "AUID- ORCID: 0000-0002-8162-5293\n",
      "AD  - Department of Tisza River Research, Danube Research Institute, Centre forEcological Research, Hungarian Academy of Sciences, Debrecen, Hungary.\n",
      "FAU - Szabolcs, Marton\n",
      "AU  - Szabolcs M\n",
      "AUID- ORCID: 0000-0001-9375-9937\n",
      "AD  - Department of Tisza River Research, Danube Research Institute, Centre forEcological Research, Hungarian Academy of Sciences, Debrecen, Hungary.\n",
      "FAU - Kolenda, Krzysztof\n",
      "AU  - Kolenda K\n",
      "AUID- ORCID: 0000-0001-8683-9867\n",
      "AD  - Department of Evolutionary Biology and Conservation of Vertebrates, Institute of Environmental Biology, University of Wroclaw, Wroclaw, Poland.\n",
      "FAU - Smirnov, Nazar\n",
      "AU  - Smirnov N\n",
      "AUID- ORCID: 0000-0001-9731-6404\n",
      "AD  - Department of Nature, Chernivtsi Regional Museum, Chernivtsi, Ukraine.\n",
      "FAU - Geniez, Philippe\n",
      "AU  - Geniez P\n",
      "AD  - CEFE, EPHE-PSL, CNRS, University of Montpellier, University Paul ValeryMontpellier 3, IRD, Montpellier, France.\n",
      "FAU - Lukanov, Simeon\n",
      "AU  - Lukanov S\n",
      "AUID- ORCID: 0000-0002-2546-6590\n",
      "AD  - Institute of Biodiversity and Ecosystem Research, Bulgarian Academy of Sciences, Sofia, Bulgaria.\n",
      "FAU - Crochet, Pierre-Andre\n",
      "AU  - Crochet PA\n",
      "AUID- ORCID: 0000-0002-0422-3960\n",
      "AD  - CEFE, CNRS, University of Montpellier, University Paul Valery Montpellier 3,EPHE, IRD, Montpellier, France.\n",
      "FAU - Dubey, Sylvain\n",
      "AU  - Dubey S\n",
      "AD  - Hintermann & Weber SA, Montreux, Switzerland.\n",
      "AD  - Department of Ecology and Evolution, University of Lausanne, Lausanne,Switzerland.\n",
      "AD  - Agrosustain SA, Nyon, Switzerland.\n",
      "FAU - Perrin, Nicolas\n",
      "AU  - Perrin N\n",
      "AUID- ORCID: 0000-0002-7756-6323\n",
      "AD  - Department of Ecology and Evolution, University of Lausanne, Lausanne,Switzerland.\n",
      "FAU - Litvinchuk, Spartak N\n",
      "AU  - Litvinchuk SN\n",
      "AD  - Institute of Cytology, Russian Academy of Sciences, St. Petersburg, Russia.\n",
      "AD  - Department of Zoology and Physiology, Dagestan State University, Makhachkala,Russia.\n",
      "FAU - Denoel, Mathieu\n",
      "AU  - Denoel M\n",
      "AUID- ORCID: 0000-0002-3586-8323\n",
      "AD  - Laboratory of Fish and Amphibian Ethology, Behavioural Biology Group, Freshwater and OCeanic science Unit of reSearch (FOCUS), University of Liege, Liege,Belgium.\n",
      "LA  - eng\n",
      "SI  - GENBANK/MK902544\n",
      "SI  - GENBANK/MK902597\n",
      "SI  - GENBANK/MK902523\n",
      "SI  - GENBANK/MK902543\n",
      "SI  - GENBANK/PRJNA542138\n",
      "SI  - GENBANK/SAMN11612144\n",
      "SI  - GENBANK/SAMN11612336\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20190628\n",
      "PL  - England\n",
      "TA  - Mol Ecol\n",
      "JT  - Molecular ecology\n",
      "JID - 9214478\n",
      "RN  - 0 (DNA, Mitochondrial)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Anura/*classification\n",
      "MH  - Bayes Theorem\n",
      "MH  - DNA, Mitochondrial/genetics\n",
      "MH  - Europe\n",
      "MH  - Gene Flow\n",
      "MH  - *Genetic Speciation\n",
      "MH  - *Genetics, Population\n",
      "MH  - Hybridization, Genetic\n",
      "MH  - Middle East\n",
      "MH  - Phylogeny\n",
      "MH  - Phylogeography\n",
      "MH  - *Reproductive Isolation\n",
      "MH  - Sequence Analysis, DNA\n",
      "MH  - Sympatry\n",
      "OTO - NOTNLM\n",
      "OT  - *amphibia\n",
      "OT  - *anuran\n",
      "OT  - *hybrid zones\n",
      "OT  - *population genomics\n",
      "OT  - *reproductive isolation\n",
      "OT  - *speciation continuum\n",
      "EDAT- 2019/06/30 06:00\n",
      "MHDA- 2020/03/31 06:00\n",
      "CRDT- 2019/06/30 06:00\n",
      "PHST- 2019/02/02 00:00 [received]\n",
      "PHST- 2019/03/30 00:00 [revised]\n",
      "PHST- 2019/05/17 00:00 [accepted]\n",
      "PHST- 2019/06/30 06:00 [pubmed]\n",
      "PHST- 2020/03/31 06:00 [medline]\n",
      "PHST- 2019/06/30 06:00 [entrez]\n",
      "AID - 10.1111/mec.15133 [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Ecol. 2019 Jul;28(13):3257-3270. doi: 10.1111/mec.15133. Epub 2019 Jun 28.\n",
      "\n",
      "PMID- 31155034\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200108\n",
      "LR  - 20200108\n",
      "IS  - 1879-3231 (Electronic)\n",
      "IS  - 0093-691X (Linking)\n",
      "VI  - 133\n",
      "DP  - 2019 Jul 15\n",
      "TI  - Sperm collection and storage for the sustainable management of amphibianbiodiversity.\n",
      "PG  - 187-200\n",
      "LID - S0093-691X(19)30085-8 [pii]\n",
      "LID - 10.1016/j.theriogenology.2019.03.035 [doi]\n",
      "AB  - Current rates of biodiversity loss pose an unprecedented challenge to theconservation community, particularly with amphibians and freshwater fish as themost threatened vertebrates. An increasing number of environmental challenges,including habitat loss, pathogens, and global warming, demand a global responsetoward the sustainable management of ecosystems and their biodiversity.Conservation Breeding Programs (CBPs) are needed for the sustainable managementof amphibian species threatened with extinction. CBPs support species survivalwhile increasing public awareness and political influence. Current CBPs onlycater for 10% of the almost 500 amphibian species in need. However, the use ofsperm storage to increase efficiency and reliability, along with an increasednumber of CBPs, offer the potential to significantly reduce species loss. Theestablishment and refinement of techniques over the last two decades, for thecollection and storage of amphibian spermatozoa, gives confidence for their usein CBPs and other biotechnical applications. Cryopreserved spermatozoa hasproduced breeding pairs of frogs and salamanders and the stage is set forLifecycle Proof of Concept Programs that use cryopreserved sperm in CBPs alongwith repopulation, supplementation, and translocation programs. The applicationof cryopreserved sperm in CBPs, is complimentary to but separate from archivalgene banking and general cell and tissue storage. However, where appropriateamphibian sperm banking should be integrated into other global biobankingprojects, especially those for fish, and those that include the use ofcryopreserved material for genomics and other research. Research over a broaderrange of amphibian species, and more uniformity in experimental methodology, isneeded to inform both theory and application. Genomics is revolutionising ourunderstanding of biological processes and increasingly guiding speciesconservation through the identification of evolutionary significant units as the conservation focus, and through revealing the intimate relationship betweenevolutionary history and sperm physiology that ultimately affects the amenabilityof sperm to refrigerated or frozen storage. In the present review we provide anascent phylogenetic framework for integration with other research lines tofurther the potential of amphibian sperm banking.\n",
      "CI  - Copyright (c) 2019 Elsevier Inc. All rights reserved.\n",
      "FAU - Browne, Robert K\n",
      "AU  - Browne RK\n",
      "AD  - Sustainability America, Sarteneja, Corozal District, Belize. Electronic address: robert.browne@gmail.com.\n",
      "FAU - Silla, Aimee J\n",
      "AU  - Silla AJ\n",
      "AD  - School of Earth, Atmospheric and Life Sciences, University of Wollongong, NSW,2522, Australia.\n",
      "FAU - Upton, Rose\n",
      "AU  - Upton R\n",
      "AD  - School of Environmental and Life Sciences, University of Newcastle, CallaghanDrive, Callaghan, NSW, 2308, Australia.\n",
      "FAU - Della-Togna, Gina\n",
      "AU  - Della-Togna G\n",
      "AD  - Smithsonian Tropical Research Institute, Panama Amphibian Rescue and ConservationProject, Panama City, Panama; Universidad Interamericana de Panama, Direccion de Investigacion, Sede Central, Panama.\n",
      "FAU - Marcec-Greaves, Ruth\n",
      "AU  - Marcec-Greaves R\n",
      "AD  - National Amphibian Conservation Center Detroit Zoological Society, Detroit, USA.\n",
      "FAU - Shishova, Natalia V\n",
      "AU  - Shishova NV\n",
      "AD  - Institute of Cell Biophysics, Russian Academy of Sciences, Pushchino, MoscowRegion, 142290, Russia.\n",
      "FAU - Uteshev, Victor K\n",
      "AU  - Uteshev VK\n",
      "AD  - Institute of Cell Biophysics, Russian Academy of Sciences, Pushchino, MoscowRegion, 142290, Russia.\n",
      "FAU - Proano, Belin\n",
      "AU  - Proano B\n",
      "AD  - Escuela de Ciencias Biologicas, Pontificia Universidad Catolica Del Ecuador,Ecuador.\n",
      "FAU - Perez, Oscar D\n",
      "AU  - Perez OD\n",
      "AD  - Escuela de Ciencias Biologicas, Pontificia Universidad Catolica Del Ecuador,Ecuador.\n",
      "FAU - Mansour, Nabil\n",
      "AU  - Mansour N\n",
      "AD  - Faculty of Veterinary Medicine, Kafrelsheikh University, Egypt.\n",
      "FAU - Kaurova, Svetlana A\n",
      "AU  - Kaurova SA\n",
      "AD  - Institute of Cell Biophysics, Russian Academy of Sciences, Pushchino, MoscowRegion, 142290, Russia.\n",
      "FAU - Gakhova, Edith N\n",
      "AU  - Gakhova EN\n",
      "AD  - Institute of Cell Biophysics, Russian Academy of Sciences, Pushchino, MoscowRegion, 142290, Russia.\n",
      "FAU - Cosson, Jacky\n",
      "AU  - Cosson J\n",
      "AD  - University of South Bohemia in Ceske Budejovice, Faculty of Fisheries andProtection of Waters, South Bohemian Research Center of Aquaculture andBiodiversity of Hydrocenoses, 38925, Vodnany, Czech Republic.\n",
      "FAU - Dyzuba, Borys\n",
      "AU  - Dyzuba B\n",
      "AD  - University of South Bohemia in Ceske Budejovice, Faculty of Fisheries andProtection of Waters, South Bohemian Research Center of Aquaculture andBiodiversity of Hydrocenoses, 38925, Vodnany, Czech Republic.\n",
      "FAU - Kramarova, Ludmila I\n",
      "AU  - Kramarova LI\n",
      "AD  - Institute of Theoretical and Experimental Biophysics, Russian Academy ofSciences, Pushchino, Moscow Region, 142290, Russia.\n",
      "FAU - McGinnity, Dale\n",
      "AU  - McGinnity D\n",
      "AD  - Nashville Zoo at Grassmere, Nashville, TN, 37211, USA.\n",
      "FAU - Gonzalez, Manuel\n",
      "AU  - Gonzalez M\n",
      "AD  - Departamento de Produccion Animal, Universidad de Guadalajara, Guadalajara,Jalisco, Mexico.\n",
      "FAU - Clulow, John\n",
      "AU  - Clulow J\n",
      "AD  - School of Environmental and Life Sciences, University of Newcastle, CallaghanDrive, Callaghan, NSW, 2308, Australia.\n",
      "FAU - Clulow, Simon\n",
      "AU  - Clulow S\n",
      "AD  - School of Environmental and Life Sciences, University of Newcastle, CallaghanDrive, Callaghan, NSW, 2308, Australia; Department of Biological Sciences,Macquarie University, Sydney, NSW, 2109, Australia.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - United States\n",
      "TA  - Theriogenology\n",
      "JT  - Theriogenology\n",
      "JID - 0421510\n",
      "SB  - IM\n",
      "MH  - *Amphibians\n",
      "MH  - Animals\n",
      "MH  - *Biodiversity\n",
      "MH  - Breeding\n",
      "MH  - Cryopreservation/veterinary\n",
      "MH  - DNA Fragmentation\n",
      "MH  - Phylogeny\n",
      "MH  - Reproduction\n",
      "MH  - Semen Preservation/methods/veterinary\n",
      "MH  - Specimen Handling\n",
      "MH  - Sperm Retrieval/*veterinary\n",
      "OTO - NOTNLM\n",
      "OT  - Amphibian\n",
      "OT  - Conservation breeding programs\n",
      "OT  - Cryopreservation\n",
      "OT  - Sperm\n",
      "OT  - Storage\n",
      "EDAT- 2019/06/04 06:00\n",
      "MHDA- 2020/01/09 06:00\n",
      "CRDT- 2019/06/04 06:00\n",
      "PHST- 2019/03/25 00:00 [received]\n",
      "PHST- 2019/03/30 00:00 [accepted]\n",
      "PHST- 2019/06/04 06:00 [entrez]\n",
      "PHST- 2019/06/04 06:00 [pubmed]\n",
      "PHST- 2020/01/09 06:00 [medline]\n",
      "AID - S0093-691X(19)30085-8 [pii]\n",
      "AID - 10.1016/j.theriogenology.2019.03.035 [doi]\n",
      "PST - ppublish\n",
      "SO  - Theriogenology. 2019 Jul 15;133:187-200. doi:10.1016/j.theriogenology.2019.03.035.\n",
      "\n",
      "PMID- 31154984\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200427\n",
      "LR  - 20200722\n",
      "IS  - 1471-2970 (Electronic)\n",
      "IS  - 0962-8436 (Linking)\n",
      "VI  - 374\n",
      "IP  - 1777\n",
      "DP  - 2019 Jul 22\n",
      "TI  - Is there convergence of gut microbes in blood-feeding vertebrates?\n",
      "PG  - 20180249\n",
      "LID - 10.1098/rstb.2018.0249 [doi]\n",
      "AB  - Animal microbiomes play an important role in dietary adaptation, yet the extentto which microbiome changes exhibit parallel evolution is unclear. Of particular interest is an adaptation to extreme diets, such as blood, which poses specialchallenges in its content of proteins and lack of essential nutrients. In thisstudy, we assessed taxonomic signatures (by 16S rRNA amplicon profiling) andpotential functional signatures (inferred by Phylogenetic Investigation ofCommunities by Reconstruction of Unobserved States (PICRUSt)) of haematophagy in birds and bats. Our goal was to test three alternative hypotheses: no convergenceof microbiomes, convergence in taxonomy and convergence in function. We find astatistically significant effect of haematophagy in terms of microbial taxonomic convergence across the blood-feeding bats and birds, although this effect issmall compared to the differences found between haematophagous andnon-haematophagous species within the two host clades. We also find some evidenceof convergence at the predicted functional level, although it is possible thatthe lack of metagenomic data and the poor representation of microbial lineagesadapted to haematophagy in genome databases limit the power of this approach. Theresults provide a paradigm for exploring convergent microbiome evolutionreplicated with independent contrasts in different host lineages. This article ispart of the theme issue 'Convergent evolution in the genomics era: new insightsand directions'.\n",
      "FAU - Song, Se Jin\n",
      "AU  - Song SJ\n",
      "AD  - 1 Department of Pediatrics, University of California San Diego , La Jolla, CA92093 , USA.\n",
      "FAU - Sanders, Jon G\n",
      "AU  - Sanders JG\n",
      "AD  - 1 Department of Pediatrics, University of California San Diego , La Jolla, CA92093 , USA.\n",
      "AD  - 2 Cornell Institute for Host-Microbe Interactions and Disease, Cornell University, Ithaca, NY 14853 , USA.\n",
      "FAU - Baldassarre, Daniel T\n",
      "AU  - Baldassarre DT\n",
      "AD  - 3 Department of Biology, University of Miami , Coral Gables, FL 33146 , USA.\n",
      "FAU - Chaves, Jaime A\n",
      "AU  - Chaves JA\n",
      "AD  - 4 Colegio de Ciencias Biologicas y Ambientales, Universidad San Francisco deQuito, Quito, Ecuador.\n",
      "AD  - 5 Galapagos Science Center, Puerto Baquerizo Moreno, Galapagos, Ecuador.\n",
      "FAU - Johnson, Nicholas S\n",
      "AU  - Johnson NS\n",
      "AD  - 6 United States Geological Survey, Great Lakes Science Center , Hammond BayBiological Station, Millersburg, MI 49759 , USA.\n",
      "FAU - Piaggio, Antoinette J\n",
      "AU  - Piaggio AJ\n",
      "AD  - 7 National Wildlife Research Center, U.S. Department of Agriculture , FortCollins, CO 80521 , USA.\n",
      "FAU - Stuckey, Matthew J\n",
      "AU  - Stuckey MJ\n",
      "AD  - 8 Department of Population Health and Reproduction, School of VeterinaryMedicine, University of California Davis , Davis CA 95616 , USA.\n",
      "FAU - Novakova, Eva\n",
      "AU  - Novakova E\n",
      "AD  - 9 Faculty of Science, University of South Bohemia , Ceske Budejovice , CzechRepublic.\n",
      "AD  - 10 Biology Centre of ASCR, Institute of Parasitology , Ceske Budejovice , CzechRepublic.\n",
      "FAU - Metcalf, Jessica L\n",
      "AU  - Metcalf JL\n",
      "AD  - 11 Department of Animal Sciences, Colorado State University , Fort Collins CO80523 , USA.\n",
      "FAU - Chomel, Bruno B\n",
      "AU  - Chomel BB\n",
      "AD  - 8 Department of Population Health and Reproduction, School of VeterinaryMedicine, University of California Davis , Davis CA 95616 , USA.\n",
      "FAU - Aguilar-Setien, Alvaro\n",
      "AU  - Aguilar-Setien A\n",
      "AD  - 12 Unidad de Investigacion Medica en Inmunologia, Coordinacion de Investigacion, Instituto Mexicano del Seguro Social, Centro Medico Nacional Siglo XXI, Hospital de Pediatria 3er piso , Mexico City , Mexico.\n",
      "FAU - Knight, Rob\n",
      "AU  - Knight R\n",
      "AD  - 1 Department of Pediatrics, University of California San Diego , La Jolla, CA92093 , USA.\n",
      "AD  - 13 Center for Microbiome Research, University of California San Diego , SanDiego, CA 92093 , USA.\n",
      "AD  - 14 Department of Computer Science and Engineering, University of California SanDiego , San Diego, CA 92093 , USA.\n",
      "FAU - McKenzie, Valerie J\n",
      "AU  - McKenzie VJ\n",
      "AD  - 15 Department of Ecology and Evolutionary Biology, University of Colorado Boulder, Boulder, CO 80309 , USA.\n",
      "LA  - eng\n",
      "SI  - figshare/10.6084/m9.figshare.c.4467776\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20190603\n",
      "PL  - England\n",
      "TA  - Philos Trans R Soc Lond B Biol Sci\n",
      "JT  - Philosophical transactions of the Royal Society of London. Series B, Biologicalsciences\n",
      "JID - 7503623\n",
      "RN  - 0 (DNA, Bacterial)\n",
      "RN  - 0 (RNA, Ribosomal, 16S)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Bacteria/classification/*genetics/isolation & purification\n",
      "MH  - Biological Evolution\n",
      "MH  - Birds/*genetics/microbiology/physiology\n",
      "MH  - Chiroptera/*genetics/microbiology/physiology\n",
      "MH  - DNA, Bacterial/genetics\n",
      "MH  - Feeding Behavior\n",
      "MH  - *Gastrointestinal Microbiome\n",
      "MH  - Phylogeny\n",
      "MH  - RNA, Ribosomal, 16S/genetics\n",
      "PMC - PMC6560276\n",
      "OTO - NOTNLM\n",
      "OT  - *convergence\n",
      "OT  - *haematophagy\n",
      "OT  - *microbiome\n",
      "EDAT- 2019/06/04 06:00\n",
      "MHDA- 2020/04/28 06:00\n",
      "CRDT- 2019/06/04 06:00\n",
      "PHST- 2019/06/04 06:00 [entrez]\n",
      "PHST- 2019/06/04 06:00 [pubmed]\n",
      "PHST- 2020/04/28 06:00 [medline]\n",
      "AID - 10.1098/rstb.2018.0249 [doi]\n",
      "PST - ppublish\n",
      "SO  - Philos Trans R Soc Lond B Biol Sci. 2019 Jul 22;374(1777):20180249. doi:10.1098/rstb.2018.0249. Epub 2019 Jun 3.\n",
      "\n",
      "PMID- 31138640\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200706\n",
      "LR  - 20200706\n",
      "IS  - 1477-9145 (Electronic)\n",
      "IS  - 0022-0949 (Linking)\n",
      "VI  - 222\n",
      "IP  - Pt 12\n",
      "DP  - 2019 Jun 20\n",
      "TI  - Molecular physiology of chemical defenses in a poison frog.\n",
      "LID - jeb204149 [pii]\n",
      "LID - 10.1242/jeb.204149 [doi]\n",
      "AB  - Poison frogs sequester small molecule lipophilic alkaloids from their diet ofleaf litter arthropods for use as chemical defenses against predation. Althoughthe dietary acquisition of chemical defenses in poison frogs is well documented, the physiological mechanisms of alkaloid sequestration has not been investigated.Here, we used RNA sequencing and proteomics to determine how alkaloids impactmRNA or protein abundance in the little devil frog (Oophaga sylvatica), andcompared wild-caught chemically defended frogs with laboratory frogs raised on analkaloid-free diet. To understand how poison frogs move alkaloids from their dietto their skin granular glands, we focused on measuring gene expression in theintestines, skin and liver. Across these tissues, we found many differentiallyexpressed transcripts involved in small molecule transport and metabolism, aswell as sodium channels and other ion pumps. We then used proteomic approaches toquantify plasma proteins, where we found several protein abundance differencesbetween wild and laboratory frogs, including the amphibian neurotoxin bindingprotein saxiphilin. Finally, because many blood proteins are synthesized in theliver, we used thermal proteome profiling as an untargeted screen for solubleproteins that bind the alkaloid decahydroquinoline. Using this approach, weidentified several candidate proteins that interact with this alkaloid, includingsaxiphilin. These transcript and protein abundance patterns suggest that thepresence of alkaloids influences frog physiology and that small moleculetransport proteins may be involved in toxin bioaccumulation in dendrobatid poisonfrogs.\n",
      "CI  - (c) 2019. Published by The Company of Biologists Ltd.\n",
      "FAU - Caty, Stephanie N\n",
      "AU  - Caty SN\n",
      "AD  - Department of Biology, Stanford University, Stanford, CA 94305, USA.\n",
      "FAU - Alvarez-Buylla, Aurora\n",
      "AU  - Alvarez-Buylla A\n",
      "AUID- ORCID: 0000-0001-6256-0300\n",
      "AD  - Department of Biology, Stanford University, Stanford, CA 94305, USA.\n",
      "FAU - Byrd, Gary D\n",
      "AU  - Byrd GD\n",
      "AUID- ORCID: 0000-0002-5437-9293\n",
      "AD  - Small Molecule Mass Spectrometry Facility, Harvard University, Cambridge, MA02138, USA.\n",
      "FAU - Vidoudez, Charles\n",
      "AU  - Vidoudez C\n",
      "AUID- ORCID: 0000-0003-0953-7800\n",
      "AD  - Small Molecule Mass Spectrometry Facility, Harvard University, Cambridge, MA02138, USA.\n",
      "FAU - Roland, Alexandre B\n",
      "AU  - Roland AB\n",
      "AUID- ORCID: 0000-0002-9463-9838\n",
      "AD  - Center for Systems Biology, Harvard University, Cambridge, MA 02138, USA.\n",
      "FAU - Tapia, Elicio E\n",
      "AU  - Tapia EE\n",
      "AUID- ORCID: 0000-0001-9005-5328\n",
      "AD  - Centro Jambatu de Investigacion y Conservacion de Anfibios, Fundacion Otonga, SanRafael, Quito, Ecuador.\n",
      "FAU - Budnik, Bogdan\n",
      "AU  - Budnik B\n",
      "AD  - Proteomics Mass Spectrometry Facility, Harvard University, Cambridge, MA 02138,USA.\n",
      "FAU - Trauger, Sunia A\n",
      "AU  - Trauger SA\n",
      "AUID- ORCID: 0000-0002-1678-8715\n",
      "AD  - Small Molecule Mass Spectrometry Facility, Harvard University, Cambridge, MA02138, USA.\n",
      "FAU - Coloma, Luis A\n",
      "AU  - Coloma LA\n",
      "AUID- ORCID: 0000-0003-0158-2455\n",
      "AD  - Centro Jambatu de Investigacion y Conservacion de Anfibios, Fundacion Otonga, SanRafael, Quito, Ecuador.\n",
      "FAU - O'Connell, Lauren A\n",
      "AU  - O'Connell LA\n",
      "AUID- ORCID: 0000-0002-2706-4077\n",
      "AD  - Department of Biology, Stanford University, Stanford, CA 94305, USAloconnel@stanford.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20190620\n",
      "PL  - England\n",
      "TA  - J Exp Biol\n",
      "JT  - The Journal of experimental biology\n",
      "JID - 0243705\n",
      "RN  - 0 (Alkaloids)\n",
      "RN  - 0 (Blood Proteins)\n",
      "RN  - 0 (Toxins, Biological)\n",
      "SB  - IM\n",
      "MH  - Alkaloids/administration & dosage/*metabolism\n",
      "MH  - Animals\n",
      "MH  - Anura/blood/genetics/*physiology\n",
      "MH  - Blood Proteins/*metabolism\n",
      "MH  - Diet\n",
      "MH  - Female\n",
      "MH  - *Gene Expression\n",
      "MH  - Intestines\n",
      "MH  - Liver/metabolism\n",
      "MH  - Male\n",
      "MH  - Proteomics\n",
      "MH  - Skin/metabolism\n",
      "MH  - Toxins, Biological/biosynthesis/*physiology\n",
      "OTO - NOTNLM\n",
      "OT  - *Alkaloids\n",
      "OT  - *Dendrobatidae\n",
      "OT  - *Oophaga sylvatica\n",
      "OT  - *Plasma\n",
      "OT  - *Proteomics\n",
      "OT  - *RNA sequencing\n",
      "OT  - *Saxiphilin\n",
      "OT  - *Sequestration\n",
      "COIS- Competing interestsThe authors declare no competing or financial interests.\n",
      "EDAT- 2019/05/30 06:00\n",
      "MHDA- 2020/07/07 06:00\n",
      "CRDT- 2019/05/30 06:00\n",
      "PHST- 2019/03/28 00:00 [received]\n",
      "PHST- 2019/05/16 00:00 [accepted]\n",
      "PHST- 2019/05/30 06:00 [pubmed]\n",
      "PHST- 2020/07/07 06:00 [medline]\n",
      "PHST- 2019/05/30 06:00 [entrez]\n",
      "AID - jeb.204149 [pii]\n",
      "AID - 10.1242/jeb.204149 [doi]\n",
      "PST - epublish\n",
      "SO  - J Exp Biol. 2019 Jun 20;222(Pt 12). pii: jeb.204149. doi: 10.1242/jeb.204149.\n",
      "\n",
      "PMID- 31067326\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200603\n",
      "LR  - 20200603\n",
      "IS  - 1465-7333 (Electronic)\n",
      "IS  - 0022-1503 (Linking)\n",
      "VI  - 110\n",
      "IP  - 3\n",
      "DP  - 2019 May 7\n",
      "TI  - The Expectations and Challenges of Wildlife Disease Research in the Era ofGenomics: Forecasting with a Horizon Scan-like Exercise.\n",
      "PG  - 261-274\n",
      "LID - 10.1093/jhered/esz001 [doi]\n",
      "AB  - The outbreak and transmission of disease-causing pathogens are contributing tothe unprecedented rate of biodiversity decline. Recent advances in genomics have coalesced into powerful tools to monitor, detect, and reconstruct the role ofpathogens impacting wildlife populations. Wildlife researchers are thus uniquely positioned to merge ecological and evolutionary studies with genomic technologiesto exploit unprecedented \"Big Data\" tools in disease research; however, manyresearchers lack the training and expertise required to use these computationallyintensive methodologies. To address this disparity, the inaugural \"Genomics ofDisease in Wildlife\" workshop assembled early to mid-career professionals withexpertise across scientific disciplines (e.g., genomics, wildlife biology,veterinary sciences, and conservation management) for training in the applicationof genomic tools to wildlife disease research. A horizon scanning-like exercise, an activity to identify forthcoming trends and challenges, performed by theworkshop participants identified and discussed 5 themes considered to be the mostpressing to the application of genomics in wildlife disease research: 1)\"Improving communication,\" 2) \"Methodological and analytical advancements,\" 3)\"Translation into practice,\" 4) \"Integrating landscape ecology and genomics,\" and5) \"Emerging new questions.\" Wide-ranging solutions from the horizon scan wereinternational in scope, itemized both deficiencies and strengths in wildlifegenomic initiatives, promoted the use of genomic technologies to unite wildlifeand human disease research, and advocated best practices for optimal use ofgenomic tools in wildlife disease projects. The results offer a glimpse of thepotential revolution in human and wildlife disease research possible throughmulti-disciplinary collaborations at local, regional, and global scales.\n",
      "CI  - (c) The American Genetic Association 2019. All rights reserved. For permissions, please e-mail: journals.permissions@oup.com.\n",
      "FAU - Fitak, Robert R\n",
      "AU  - Fitak RR\n",
      "AD  - Department of Biology, Duke University, Durham, NC, USA.\n",
      "FAU - Antonides, Jennifer D\n",
      "AU  - Antonides JD\n",
      "AD  - Department of Forestry & Natural Resources, Purdue University, West Lafayette,IN.\n",
      "FAU - Baitchman, Eric J\n",
      "AU  - Baitchman EJ\n",
      "AD  - The Zoo New England Division of Animal Health and Conservation, Boston, MA.\n",
      "FAU - Bonaccorso, Elisa\n",
      "AU  - Bonaccorso E\n",
      "AD  - The Instituto BIOSFERA and Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, via Interoceanica y Diego de Robles, Quito,Ecuador.\n",
      "FAU - Braun, Josephine\n",
      "AU  - Braun J\n",
      "AD  - The Institute for Conservation Research, San Diego Zoo Global, Escondido, CA.\n",
      "FAU - Kubiski, Steven\n",
      "AU  - Kubiski S\n",
      "AD  - The Institute for Conservation Research, San Diego Zoo Global, Escondido, CA.\n",
      "FAU - Chiu, Elliott\n",
      "AU  - Chiu E\n",
      "AD  - Department of Microbiology, Immunology and Pathology, Colorado State University, Fort Collins, CO.\n",
      "FAU - Fagre, Anna C\n",
      "AU  - Fagre AC\n",
      "AD  - Department of Microbiology, Immunology and Pathology, Colorado State University, Fort Collins, CO.\n",
      "FAU - Gagne, Roderick B\n",
      "AU  - Gagne RB\n",
      "AD  - Department of Microbiology, Immunology and Pathology, Colorado State University, Fort Collins, CO.\n",
      "FAU - Lee, Justin S\n",
      "AU  - Lee JS\n",
      "AD  - Department of Microbiology, Immunology and Pathology, Colorado State University, Fort Collins, CO.\n",
      "FAU - Malmberg, Jennifer L\n",
      "AU  - Malmberg JL\n",
      "AD  - Department of Microbiology, Immunology and Pathology, Colorado State University, Fort Collins, CO.\n",
      "FAU - Stenglein, Mark D\n",
      "AU  - Stenglein MD\n",
      "AD  - Department of Microbiology, Immunology and Pathology, Colorado State University, Fort Collins, CO.\n",
      "FAU - Dusek, Robert J\n",
      "AU  - Dusek RJ\n",
      "AD  - The U. S. Geological Survey, National Wildlife Health Center, Madison, WI.\n",
      "FAU - Forgacs, David\n",
      "AU  - Forgacs D\n",
      "AD  - The Interdisciplinary Graduate Program of Genetics, Texas A&M University, CollegeStation, TX.\n",
      "FAU - Fountain-Jones, Nicholas M\n",
      "AU  - Fountain-Jones NM\n",
      "AD  - The Department of Veterinary Population Medicine, University of Minnesota, St.Paul, MN.\n",
      "FAU - Gilbertson, Marie L J\n",
      "AU  - Gilbertson MLJ\n",
      "AD  - The Department of Veterinary Population Medicine, University of Minnesota, St.Paul, MN.\n",
      "FAU - Worsley-Tonks, Katherine E L\n",
      "AU  - Worsley-Tonks KEL\n",
      "AD  - The Department of Veterinary Population Medicine, University of Minnesota, St.Paul, MN.\n",
      "FAU - Funk, W Chris\n",
      "AU  - Funk WC\n",
      "AD  - The Department of Biology, Colorado State University, Fort Collins, CO.\n",
      "FAU - Trumbo, Daryl R\n",
      "AU  - Trumbo DR\n",
      "AD  - The Department of Biology, Colorado State University, Fort Collins, CO.\n",
      "FAU - Ghersi, Bruno M\n",
      "AU  - Ghersi BM\n",
      "AD  - The University of Tennessee, Knoxville, TN.\n",
      "FAU - Grimaldi, Wray\n",
      "AU  - Grimaldi W\n",
      "AD  - The Encinitas, CA.\n",
      "FAU - Heisel, Sara E\n",
      "AU  - Heisel SE\n",
      "AD  - The Odum School of Ecology, University of Georgia, Athens, GA.\n",
      "FAU - Jardine, Claire M\n",
      "AU  - Jardine CM\n",
      "AD  - The Department of Pathobiology, Canadian Wildlife Health Cooperative, University of Guelph, Guelph, Ontario, Canada.\n",
      "FAU - Kamath, Pauline L\n",
      "AU  - Kamath PL\n",
      "AD  - The School of Food and Agriculture, University of Maine, Orono, ME.\n",
      "FAU - Karmacharya, Dibesh\n",
      "AU  - Karmacharya D\n",
      "AD  - The Center for Molecular Dynamics Nepal, Kathmandu, Nepal.\n",
      "FAU - Kozakiewicz, Christopher P\n",
      "AU  - Kozakiewicz CP\n",
      "AD  - The School of Biological Sciences, University of Tasmania, Hobart, Tasmania,Australia.\n",
      "FAU - Kraberger, Simona\n",
      "AU  - Kraberger S\n",
      "AD  - The Biodesign Center for Fundamental and Applied Microbiomics, Center forEvolution and Medicine, School of Life Sciences, Arizona State University, Tempe,AZ.\n",
      "FAU - Loisel, Dagan A\n",
      "AU  - Loisel DA\n",
      "AD  - The Department of Biology, Saint Michael's College, Colchester, VT.\n",
      "FAU - McDonald, Cait\n",
      "AU  - McDonald C\n",
      "AD  - The Department of Ecology & Evolutionary Biology, Cornell University, Ithaca, NY (McDonald).\n",
      "FAU - Miller, Steven\n",
      "AU  - Miller S\n",
      "AD  - The Department of Biology, Drexel University, Philadelphia, PA.\n",
      "FAU - O'Rourke, Devon\n",
      "AU  - O'Rourke D\n",
      "AD  - The University of New Hampshire, Durham NH.\n",
      "FAU - Ott-Conn, Caitlin N\n",
      "AU  - Ott-Conn CN\n",
      "AD  - The Michigan Department of Natural Resources, Wildlife Disease Laboratory,Lansing, MI.\n",
      "FAU - Paez-Vacas, Monica\n",
      "AU  - Paez-Vacas M\n",
      "AD  - The Centro de Investigacion de la Biodiversidad y Cambio Climatico (BioCamb),Facultad de Ciencias de Medio Ambiente, Universidad Tecnologica Indoamerica,Machala y Sabanilla, Quito, Ecuador.\n",
      "FAU - Peel, Alison J\n",
      "AU  - Peel AJ\n",
      "AD  - The Environmental Futures Research Institute, Griffith University, Nathan,Queensland, Australia.\n",
      "FAU - Turner, Wendy C\n",
      "AU  - Turner WC\n",
      "AD  - The Department of Biological Sciences, University at Albany, State University of New York, Albany, NY.\n",
      "FAU - VanAcker, Meredith C\n",
      "AU  - VanAcker MC\n",
      "AD  - The Department of Ecology, Evolution, and Environmental Biology, ColumbiaUniversity, New York, NY.\n",
      "FAU - VandeWoude, Sue\n",
      "AU  - VandeWoude S\n",
      "AD  - The College of Veterinary Medicine and Biomedical Sciences, Colorado StateUniversity, Fort Collins, CO.\n",
      "FAU - Pecon-Slattery, Jill\n",
      "AU  - Pecon-Slattery J\n",
      "AD  - The Center for Species Survival, Smithsonian Conservation BiologyInstitute-National Zoological Park, Front Royal, VA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "PL  - United States\n",
      "TA  - J Hered\n",
      "JT  - The Journal of heredity\n",
      "JID - 0375373\n",
      "SB  - IM\n",
      "EIN - J Hered. 2019 Aug 16;110(5):639. PMID: 31301132\n",
      "MH  - Animal Diseases/epidemiology/*etiology/transmission\n",
      "MH  - Animals\n",
      "MH  - *Animals, Wild\n",
      "MH  - Biodiversity\n",
      "MH  - Biological Evolution\n",
      "MH  - Computational Biology/methods\n",
      "MH  - Disease Susceptibility\n",
      "MH  - Ecology\n",
      "MH  - Environment\n",
      "MH  - Genome\n",
      "MH  - *Genomics/methods\n",
      "MH  - Host-Pathogen Interactions/genetics\n",
      "MH  - Humans\n",
      "MH  - *Research\n",
      "OTO - NOTNLM\n",
      "OT  - *biodiversity\n",
      "OT  - *bioinformatics\n",
      "OT  - *comparative genomics\n",
      "OT  - *host\n",
      "OT  - *next generation DNA sequencing\n",
      "OT  - *pathogen\n",
      "EDAT- 2019/05/09 06:00\n",
      "MHDA- 2020/06/04 06:00\n",
      "CRDT- 2019/05/09 06:00\n",
      "PHST- 2018/05/02 00:00 [received]\n",
      "PHST- 2019/01/08 00:00 [accepted]\n",
      "PHST- 2019/05/09 06:00 [entrez]\n",
      "PHST- 2019/05/09 06:00 [pubmed]\n",
      "PHST- 2020/06/04 06:00 [medline]\n",
      "AID - 5288600 [pii]\n",
      "AID - 10.1093/jhered/esz001 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Hered. 2019 May 7;110(3):261-274. doi: 10.1093/jhered/esz001.\n",
      "\n",
      "PMID- 30981810\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20191209\n",
      "LR  - 20191217\n",
      "IS  - 1095-9513 (Electronic)\n",
      "IS  - 1055-7903 (Linking)\n",
      "VI  - 136\n",
      "DP  - 2019 Jul\n",
      "TI  - Introgressive hybridisation between two widespread sharks in the east Pacificregion.\n",
      "PG  - 119-127\n",
      "LID - S1055-7903(18)30410-X [pii]\n",
      "LID - 10.1016/j.ympev.2019.04.013 [doi]\n",
      "AB  - With just a handful of documented cases of hybridisation in cartilaginous fishes,shark hybridisation remains poorly investigated. Small amounts of admixture have been detected between Galapagos (Carcharhinus galapagensis) and dusky(Carcharhinus obscurus) sharks previously, generating a hypothesis of ongoinghybridisation. We sampled a large number of individuals from areas where thespecies co-occur (contact zones) across the Pacific Ocean and used bothmitochondrial and nuclear-encoded SNPs to examine genetic admixture andintrogression between the two species. Using empirical analytical approaches and simulations, we first developed a set of 1873 highly informative SNPs for thesetwo species to evaluate the degree of admixture between them. Overall, resultsindicate a high discriminatory power of nuclear SNPs (FST=0.47, p<0.05) betweenthe two species, unlike mitochondrial DNA (PhiST=0.00 p>0.05), which failed todifferentiate these species. We identified four hybrid individuals (approximately 1%) and detected bi-directional introgression between C.galapagensis and C. obscurus in the Gulf of California along the east Pacificcoast of the Americas. We emphasize the importance of including a combination of mtDNA and diagnostic nuclear markers to properly assess species identification,detect patterns of hybridisation, and better inform management and conservationof these sharks, especially given the morphological similarities within the genusCarcharhinus.\n",
      "CI  - Crown Copyright (c) 2019. Published by Elsevier Inc. All rights reserved.\n",
      "FAU - Pazmino, Diana A\n",
      "AU  - Pazmino DA\n",
      "AD  - Galapagos Science Center, Universidad San Francisco de Quito, Isla San Cristobal,Galapagos 200150, Ecuador; Centre for Sustainable Tropical Fisheries andAquaculture, College of Science and Engineering, James Cook University,Townsville, 4811 QLD, Australia; Comparative Genomics Centre, College of Science and Engineering, James Cook University, Townsville, 4811 QLD, Australia.Electronic address: dapazmino@usfq.edu.ec.\n",
      "FAU - van Herderden, Lynne\n",
      "AU  - van Herderden L\n",
      "AD  - Centre for Sustainable Tropical Fisheries and Aquaculture, College of Science andEngineering, James Cook University, Townsville, 4811 QLD, Australia; Comparative Genomics Centre, College of Science and Engineering, James Cook University,Townsville, 4811 QLD, Australia.\n",
      "FAU - Simpfendorfer, Colin A\n",
      "AU  - Simpfendorfer CA\n",
      "AD  - Centre for Sustainable Tropical Fisheries and Aquaculture, College of Science andEngineering, James Cook University, Townsville, 4811 QLD, Australia.\n",
      "FAU - Junge, Claudia\n",
      "AU  - Junge C\n",
      "AD  - School of Biological Sciences and Environment Institute, The University ofAdelaide, SA 5005, Australia; Havforskningsinstituttet (Institute of MarineResearch, IMR), Tromso, Norway.\n",
      "FAU - Donnellan, Stephen C\n",
      "AU  - Donnellan SC\n",
      "AD  - School of Biological Sciences and Environment Institute, The University ofAdelaide, SA 5005, Australia; South Australian Museum, North Terrace, Adelaide5000, Australia.\n",
      "FAU - Hoyos-Padilla, E Mauricio\n",
      "AU  - Hoyos-Padilla EM\n",
      "AD  - Pelagios-Kakunja A.C., Sinaloa 1540, La Paz, Baja California Sur CP 23020,Mexico.\n",
      "FAU - Duffy, Clinton A J\n",
      "AU  - Duffy CAJ\n",
      "AD  - Auckland War Memorial Museum, The Domain, Auckland, New Zealand; Department ofConservation, Private Bag 68908, Newton, Auckland 1145, New Zealand.\n",
      "FAU - Huveneers, Charlie\n",
      "AU  - Huveneers C\n",
      "AD  - College of Science and Engineering, Flinders University, Adelaide, 5042 SA,Australia.\n",
      "FAU - Gillanders, Bronwyn M\n",
      "AU  - Gillanders BM\n",
      "AD  - School of Biological Sciences and Environment Institute, The University ofAdelaide, SA 5005, Australia.\n",
      "FAU - Butcher, Paul A\n",
      "AU  - Butcher PA\n",
      "AD  - New South Wales Department of Primary Industries, National Marine Science Centre,Coffs Harbour, 2450 NSW, Australia.\n",
      "FAU - Maes, Gregory E\n",
      "AU  - Maes GE\n",
      "AD  - Centre for Sustainable Tropical Fisheries and Aquaculture, College of Science andEngineering, James Cook University, Townsville, 4811 QLD, Australia; Comparative Genomics Centre, College of Science and Engineering, James Cook University,Townsville, 4811 QLD, Australia; Laboratory of Biodiversity and EvolutionaryGenomics, University of Leuven, B-3000 Leuven, Belgium; Laboratory forCytogenetics and Genome Research, Center for Human Genetics, Genomics Core, KULeuven, 3000 Leuven, Belgium.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20190411\n",
      "PL  - United States\n",
      "TA  - Mol Phylogenet Evol\n",
      "JT  - Molecular phylogenetics and evolution\n",
      "JID - 9304400\n",
      "RN  - 0 (DNA, Mitochondrial)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Bayes Theorem\n",
      "MH  - California\n",
      "MH  - Computer Simulation\n",
      "MH  - DNA, Mitochondrial/genetics\n",
      "MH  - Geography\n",
      "MH  - *Hybridization, Genetic\n",
      "MH  - Mitochondria/genetics\n",
      "MH  - Pacific Ocean\n",
      "MH  - Phylogeny\n",
      "MH  - Polymorphism, Single Nucleotide/genetics\n",
      "MH  - Sample Size\n",
      "MH  - Sharks/*genetics\n",
      "OTO - NOTNLM\n",
      "OT  - *Elasmobranchs\n",
      "OT  - *Hybridisation\n",
      "OT  - *Introgression\n",
      "OT  - *Misidentification\n",
      "OT  - *Single nucleotide polymorphisms\n",
      "EDAT- 2019/04/15 06:00\n",
      "MHDA- 2019/12/18 06:00\n",
      "CRDT- 2019/04/15 06:00\n",
      "PHST- 2018/06/30 00:00 [received]\n",
      "PHST- 2019/03/27 00:00 [revised]\n",
      "PHST- 2019/04/10 00:00 [accepted]\n",
      "PHST- 2019/04/15 06:00 [pubmed]\n",
      "PHST- 2019/12/18 06:00 [medline]\n",
      "PHST- 2019/04/15 06:00 [entrez]\n",
      "AID - S1055-7903(18)30410-X [pii]\n",
      "AID - 10.1016/j.ympev.2019.04.013 [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Phylogenet Evol. 2019 Jul;136:119-127. doi: 10.1016/j.ympev.2019.04.013. Epub2019 Apr 11.\n",
      "\n",
      "PMID- 30880832\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20200225\n",
      "IS  - 0018-8158 (Print)\n",
      "IS  - 0018-8158 (Linking)\n",
      "VI  - 832\n",
      "IP  - 1\n",
      "DP  - 2019\n",
      "TI  - An exploration of the links between parasites, trophic ecology, morphology, andimmunogenetics in the Lake Tanganyika cichlid radiation.\n",
      "PG  - 215-233\n",
      "LID - 10.1007/s10750-018-3798-2 [doi]\n",
      "AB  - Differences in habitat and diet between species are often associated withmorphological differences. Habitat and trophic adaptation have therefore beenproposed as important drivers of speciation and adaptive radiation. Importantly, habitat and diet shifts likely impose changes in exposure to different parasites and infection risk. As strong selective agents influencing survival and matechoice, parasites might play an important role in host diversification. Weexplore this possibility for the adaptive radiation of Lake Tanganyika (LT)cichlids. We first compare metazoan macroparasites infection levels betweencichlid tribes. We then describe the cichlids' genetic diversity at the majorhistocompatibility complex (MHC), which plays a key role in vertebrate immunity. Finally, we evaluate to what extent trophic ecology and morphology explainvariation in infection levels and MHC, accounting for phylogenetic relationships.We show that different cichlid tribes in LT feature partially non-overlappingparasite communities and partially non-overlapping MHC diversity. Whilemorphology explained 15% of the variation in mean parasite abundance, trophicecology accounted for 16% and 22% of the MHC variation at the nucleotide and atthe amino acid level, respectively. Parasitism and immunogenetic adaptation maythus add additional dimensions to the LT cichlid radiation.\n",
      "FAU - Meyer, Britta S\n",
      "AU  - Meyer BS\n",
      "AUID- ORCID: 0000-0002-2549-1825\n",
      "AD  - 1Zoological Institute, University of Basel, Vesalgasse 1, 4051 Basel,Switzerland.0000 0004 1937 0642grid.6612.3\n",
      "AD  - 2Evolutionary Ecology of Marine Fishes, Helmholtz Centre for Ocean Research Kiel,GEOMAR, Dusternbrooker Weg 20, 24105 Kiel, Germany.0000 0000 90569663grid.15649.3f\n",
      "AD  - 3Present Address: Max Planck Institute for Evolutionary Biology, Max PlanckResearch Group Behavioural Genomics, August-Thienemann-Str. 2, 24306 Plon,Germany.0000 0001 2105 1091grid.4372.2\n",
      "FAU - Hablutzel, Pascal I\n",
      "AU  - Hablutzel PI\n",
      "AUID- ORCID: 0000-0002-6739-4994\n",
      "AD  - 4Laboratory of Biodiversity and Evolutionary Genomics, University of Leuven, Ch. Deberiotstraat 32, 3000 Louvain, Belgium.0000 0001 0668 7884grid.5596.f\n",
      "AD  - 5Present Address: Flanders Marine Institute, Wandelaarkaai 7, 8400 Ostend,Belgium.0000 0001 2230 9672grid.426539.f\n",
      "FAU - Roose, Anna K\n",
      "AU  - Roose AK\n",
      "AD  - 4Laboratory of Biodiversity and Evolutionary Genomics, University of Leuven, Ch. Deberiotstraat 32, 3000 Louvain, Belgium.0000 0001 0668 7884grid.5596.f\n",
      "FAU - Hofmann, Melinda J\n",
      "AU  - Hofmann MJ\n",
      "AUID- ORCID: 0000-0001-7224-7334\n",
      "AD  - 6Department of Biodiversity and Evolutionary Biology, Museo Nacional de Ciencias Naturales, CSIC, Calle Jose Gutierrez Abascal 2, 28006 Madrid, Spain.0000 00041768 463Xgrid.420025.1\n",
      "AD  - 7Present Address: Museo de Zoologia, Pontificia Universidad Catolica del Ecuador,Av. 12 de Octubre 1076, Quito, Ecuador.0000 0001 1941 7306grid.412527.7\n",
      "FAU - Salzburger, Walter\n",
      "AU  - Salzburger W\n",
      "AUID- ORCID: 0000-0002-9988-1674\n",
      "AD  - 1Zoological Institute, University of Basel, Vesalgasse 1, 4051 Basel,Switzerland.0000 0004 1937 0642grid.6612.3\n",
      "FAU - Raeymaekers, Joost A M\n",
      "AU  - Raeymaekers JAM\n",
      "AUID- ORCID: 0000-0003-2732-7495\n",
      "AD  - 1Zoological Institute, University of Basel, Vesalgasse 1, 4051 Basel,Switzerland.0000 0004 1937 0642grid.6612.3\n",
      "AD  - 4Laboratory of Biodiversity and Evolutionary Genomics, University of Leuven, Ch. Deberiotstraat 32, 3000 Louvain, Belgium.0000 0001 0668 7884grid.5596.f\n",
      "AD  - 8Present Address: Faculty of Biosciences and Aquaculture, Nord University, 8049Bodo, Norway.grid.465487.c\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20181026\n",
      "PL  - Netherlands\n",
      "TA  - Hydrobiologia\n",
      "JT  - Hydrobiologia\n",
      "JID - 9879627\n",
      "PMC - PMC6394741\n",
      "OTO - NOTNLM\n",
      "OT  - Immunogenetics\n",
      "OT  - Macroparasites\n",
      "OT  - Morphology\n",
      "OT  - Parasites\n",
      "OT  - Trophic ecology\n",
      "EDAT- 2019/03/19 06:00\n",
      "MHDA- 2019/03/19 06:01\n",
      "CRDT- 2019/03/19 06:00\n",
      "PHST- 2018/02/15 00:00 [received]\n",
      "PHST- 2018/10/02 00:00 [revised]\n",
      "PHST- 2018/10/12 00:00 [accepted]\n",
      "PHST- 2019/03/19 06:00 [entrez]\n",
      "PHST- 2019/03/19 06:00 [pubmed]\n",
      "PHST- 2019/03/19 06:01 [medline]\n",
      "AID - 10.1007/s10750-018-3798-2 [doi]\n",
      "AID - 3798 [pii]\n",
      "PST - ppublish\n",
      "SO  - Hydrobiologia. 2019;832(1):215-233. doi: 10.1007/s10750-018-3798-2. Epub 2018 Oct26.\n",
      "\n",
      "PMID- 30581024\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200123\n",
      "LR  - 20200123\n",
      "IS  - 1879-0445 (Electronic)\n",
      "IS  - 0960-9822 (Linking)\n",
      "VI  - 29\n",
      "IP  - 1\n",
      "DP  - 2019 Jan 7\n",
      "TI  - Y Chromosome Sequences Reveal a Short Beringian Standstill, Rapid Expansion, and early Population structure of Native American Founders.\n",
      "PG  - 149-157.e3\n",
      "LID - S0960-9822(18)31495-7 [pii]\n",
      "LID - 10.1016/j.cub.2018.11.029 [doi]\n",
      "AB  - The Americas were the last inhabitable continents to be occupied by humans, with a growing multidisciplinary consensus for entry 15-25 thousand years ago (kya)from northeast Asia via the former Beringia land bridge [1-4]. Autosomal DNAanalyses have dated the separation of Native American ancestors from the Asiangene pool to 23 kya or later [5, 6] and mtDNA analyses to approximately 25 kya[7], followed by isolation (\"Beringian Standstill\" [8, 9]) for 2.4-9 ky and then a rapid expansion throughout the Americas. Here, we present a calibratedsequence-based analysis of 222 Native American and relevant Eurasian Ychromosomes (24 new) from haplogroups Q and C [10], with four major conclusions. First, we identify three to four independent lineages as autochthonous and likelyfounders: the major Q-M3 and rarer Q-CTS1780 present throughout the Americas, thevery rare C3-MPB373 in South America, and possibly the C3-P39/Z30536 in NorthAmerica. Second, from the divergence times and Eurasian/American distribution of lineages, we estimate a Beringian Standstill duration of 2.7 ky or 4.6 ky,according to alternative models, and entry south of the ice sheet after 19.5 kya.Third, we describe the star-like expansion of Q-M848 (within Q-M3) starting at 15kya [11] in the Americas, followed by establishment of substantial spatialstructure in South America by 12 kya. Fourth, the deep branches of the Q-CTS1780 lineage present at low frequencies throughout the Americas today [12] may reflecta separate out-of-Beringia dispersal after the melting of the glaciers at the endof the Pleistocene.\n",
      "CI  - Copyright (c) 2018 The Authors. Published by Elsevier Ltd.. All rights reserved.\n",
      "FAU - Pinotti, Thomaz\n",
      "AU  - Pinotti T\n",
      "AD  - Laboratorio de Biodiversidade e Evolucao Molecular (LBEM), Instituto de Ciencias Biologicas, Universidade Federal de Minas Gerais, Av. Antonio Carlos 6627,31270-010 Belo Horizonte, Brazil; The Wellcome Sanger Institute, Wellcome Genome Campus, Hinxton, Cambridgeshire CB10 1SA, UK.\n",
      "FAU - Bergstrom, Anders\n",
      "AU  - Bergstrom A\n",
      "AD  - The Wellcome Sanger Institute, Wellcome Genome Campus, Hinxton, CambridgeshireCB10 1SA, UK.\n",
      "FAU - Geppert, Maria\n",
      "AU  - Geppert M\n",
      "AD  - Institute of Legal Medicine and Forensic Sciences, Department of ForensicGenetics, Charite-Universitatsmedizin Berlin, Berlin, Germany.\n",
      "FAU - Bawn, Matt\n",
      "AU  - Bawn M\n",
      "AD  - Centro de Genetica y Biologia Molecular (CGBM), Instituto de Investigacion,Facultad de Medicina Humana, Universidad de San Martin de Porres, 15009 Lima,Peru; The Earlham Institute, NR4 7UG Norwich, UK.\n",
      "FAU - Ohasi, Dominique\n",
      "AU  - Ohasi D\n",
      "AD  - Laboratorio de Biodiversidade e Evolucao Molecular (LBEM), Instituto de Ciencias Biologicas, Universidade Federal de Minas Gerais, Av. Antonio Carlos 6627,31270-010 Belo Horizonte, Brazil.\n",
      "FAU - Shi, Wentao\n",
      "AU  - Shi W\n",
      "AD  - The Wellcome Sanger Institute, Wellcome Genome Campus, Hinxton, CambridgeshireCB10 1SA, UK; Department of Genetics, School of Basic Medical Sciences, TianjinMedical University, 300070 Tianjin, China.\n",
      "FAU - Lacerda, Daniela R\n",
      "AU  - Lacerda DR\n",
      "AD  - Laboratorio de Biodiversidade e Evolucao Molecular (LBEM), Instituto de Ciencias Biologicas, Universidade Federal de Minas Gerais, Av. Antonio Carlos 6627,31270-010 Belo Horizonte, Brazil.\n",
      "FAU - Solli, Arne\n",
      "AU  - Solli A\n",
      "AD  - Q Nordic Independent Researchers; Department of Archaeology, History, CulturalStudies and Religion (AHKR), University of Bergen, Norway.\n",
      "FAU - Norstedt, Jakob\n",
      "AU  - Norstedt J\n",
      "AD  - Q Nordic Independent Researchers.\n",
      "FAU - Reed, Kate\n",
      "AU  - Reed K\n",
      "AD  - Q Nordic Independent Researchers.\n",
      "FAU - Dawtry, Kim\n",
      "AU  - Dawtry K\n",
      "AD  - Q Nordic Independent Researchers.\n",
      "FAU - Gonzalez-Andrade, Fabricio\n",
      "AU  - Gonzalez-Andrade F\n",
      "AD  - Translational Medicine Unit, Central University of Ecuador, Faculty of MedicalSciences, Iquique N14-121 y Sodiro-Itchimbia, Sector El Dorado, 170403 Quito,Ecuador.\n",
      "FAU - Paz-Y-Mino, Cesar\n",
      "AU  - Paz-Y-Mino C\n",
      "AD  - Universidad de las Americas, Av. de los Granados E12-41, 170513 Quito, Ecuador.\n",
      "FAU - Revollo, Susana\n",
      "AU  - Revollo S\n",
      "AD  - Universidad Mayor de San Andres, Av. Villazon 1995, 2008 La Paz, Bolivia.\n",
      "FAU - Cuellar, Cinthia\n",
      "AU  - Cuellar C\n",
      "AD  - Universidad Mayor de San Andres, Av. Villazon 1995, 2008 La Paz, Bolivia.\n",
      "FAU - Jota, Marilza S\n",
      "AU  - Jota MS\n",
      "AD  - Laboratorio de Biodiversidade e Evolucao Molecular (LBEM), Instituto de Ciencias Biologicas, Universidade Federal de Minas Gerais, Av. Antonio Carlos 6627,31270-010 Belo Horizonte, Brazil.\n",
      "FAU - Santos, Jose E Jr\n",
      "AU  - Santos JE Jr\n",
      "AD  - Laboratorio de Biodiversidade e Evolucao Molecular (LBEM), Instituto de Ciencias Biologicas, Universidade Federal de Minas Gerais, Av. Antonio Carlos 6627,31270-010 Belo Horizonte, Brazil.\n",
      "FAU - Ayub, Qasim\n",
      "AU  - Ayub Q\n",
      "AD  - The Wellcome Sanger Institute, Wellcome Genome Campus, Hinxton, CambridgeshireCB10 1SA, UK; Monash University Malaysia Genomics Facility, Tropical Medicine andBiology Multidisciplinary Platform, 47500 Bandar Sunway, Selangor Darul Ehsan,Malaysia; School of Science, Monash University Malaysia, 47500 Bandar Sunway,Selangor Darul Ehsan, Malaysia.\n",
      "FAU - Kivisild, Toomas\n",
      "AU  - Kivisild T\n",
      "AD  - Department of Archaeology and Anthropology, University of Cambridge, CB2 1QHCambridge, UK; Estonian Biocentre, 51010 Tartu, Estonia.\n",
      "FAU - Sandoval, Jose R\n",
      "AU  - Sandoval JR\n",
      "AD  - Centro de Genetica y Biologia Molecular (CGBM), Instituto de Investigacion,Facultad de Medicina Humana, Universidad de San Martin de Porres, 15009 Lima,Peru.\n",
      "FAU - Fujita, Ricardo\n",
      "AU  - Fujita R\n",
      "AD  - Centro de Genetica y Biologia Molecular (CGBM), Instituto de Investigacion,Facultad de Medicina Humana, Universidad de San Martin de Porres, 15009 Lima,Peru.\n",
      "FAU - Xue, Yali\n",
      "AU  - Xue Y\n",
      "AD  - The Wellcome Sanger Institute, Wellcome Genome Campus, Hinxton, CambridgeshireCB10 1SA, UK.\n",
      "FAU - Roewer, Lutz\n",
      "AU  - Roewer L\n",
      "AD  - Institute of Legal Medicine and Forensic Sciences, Department of ForensicGenetics, Charite-Universitatsmedizin Berlin, Berlin, Germany.\n",
      "FAU - Santos, Fabricio R\n",
      "AU  - Santos FR\n",
      "AD  - Laboratorio de Biodiversidade e Evolucao Molecular (LBEM), Instituto de Ciencias Biologicas, Universidade Federal de Minas Gerais, Av. Antonio Carlos 6627,31270-010 Belo Horizonte, Brazil. Electronic address: fsantos@icb.ufmg.br.\n",
      "FAU - Tyler-Smith, Chris\n",
      "AU  - Tyler-Smith C\n",
      "AD  - The Wellcome Sanger Institute, Wellcome Genome Campus, Hinxton, CambridgeshireCB10 1SA, UK. Electronic address: cts@sanger.ac.uk.\n",
      "LA  - eng\n",
      "SI  - Dryad/10.5061/dryad.h38853n\n",
      "GR  - 098051/WT_/Wellcome Trust/United Kingdom\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20181220\n",
      "PL  - England\n",
      "TA  - Curr Biol\n",
      "JT  - Current biology : CB\n",
      "JID - 9107782\n",
      "RN  - 0 (DNA, Ancient)\n",
      "RN  - 0 (DNA, Mitochondrial)\n",
      "SB  - IM\n",
      "MH  - American Native Continental Ancestry Group/*genetics\n",
      "MH  - Archaeology\n",
      "MH  - Chromosomes, Human, Y/*genetics\n",
      "MH  - DNA, Ancient/*analysis\n",
      "MH  - DNA, Mitochondrial/genetics\n",
      "MH  - Female\n",
      "MH  - Genome, Human/genetics\n",
      "MH  - *Genotype\n",
      "MH  - *Human Migration\n",
      "MH  - Humans\n",
      "MH  - Male\n",
      "OTO - NOTNLM\n",
      "OT  - *Beringia\n",
      "OT  - *Native Americans\n",
      "OT  - *Y chromosome lineages\n",
      "OT  - *phylogeography\n",
      "OT  - *pre-Columbian settlement of Americas\n",
      "EDAT- 2018/12/26 06:00\n",
      "MHDA- 2020/01/24 06:00\n",
      "CRDT- 2018/12/25 06:00\n",
      "PHST- 2018/05/03 00:00 [received]\n",
      "PHST- 2018/09/03 00:00 [revised]\n",
      "PHST- 2018/11/09 00:00 [accepted]\n",
      "PHST- 2018/12/26 06:00 [pubmed]\n",
      "PHST- 2020/01/24 06:00 [medline]\n",
      "PHST- 2018/12/25 06:00 [entrez]\n",
      "AID - S0960-9822(18)31495-7 [pii]\n",
      "AID - 10.1016/j.cub.2018.11.029 [doi]\n",
      "PST - ppublish\n",
      "SO  - Curr Biol. 2019 Jan 7;29(1):149-157.e3. doi: 10.1016/j.cub.2018.11.029. Epub 2018Dec 20.\n",
      "\n",
      "PMID- 30541084\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200429\n",
      "LR  - 20200429\n",
      "IS  - 1465-7333 (Electronic)\n",
      "IS  - 0022-1503 (Linking)\n",
      "VI  - 110\n",
      "IP  - 2\n",
      "DP  - 2019 Mar 5\n",
      "TI  - Galapagos Islands Endemic Vertebrates: A Population Genetics Perspective.\n",
      "PG  - 137-157\n",
      "LID - 10.1093/jhered/esy066 [doi]\n",
      "AB  - The organisms of the Galapagos Islands played a central role in the developmentof the theory of evolution by Charles Darwin. Examination of the populationgenetics factors of many of these organisms with modern molecular methods hasexpanded our understanding of their evolution. Here, I provide a perspective onhow selection, gene flow, genetic drift, mutation, and inbreeding havecontributed to the evolution of 6 iconic Galapagos species: flightless cormorant,pink iguana, marine iguana, Galapagos hawk, giant tortoises, and Darwin'sfinches. Because of the inherent biological differences among these species that have colonized the Galapagos, different population genetic factors appear to bemore or less important in these different species. For example, the Galapagosprovided novel environments in which strong selection took place and the Darwin'sfinches diversified to produce new species and the cormorant adapted to thenutrient-rich western shores of the Galapagos by losing its ability to fly andgenomic data have now identified candidate genes. In both the pink iguana, which exists in one small population, and the Galapagos hawk, which has smallpopulation sizes, genetic drift has been potentially quite important. Thereappears to be very limited interisland gene flow in the flightless cormorant and the Galapagos hawk. On the other hand, both the marine iguana and some of theDarwin's finches appear to have significant interisland gene flow. Hybridization between species and subspecies has also introduced new adaptive variation, and insome cases, hybridization might have resulted in despeciation. Overall, newpopulation genetics and genomics research has provided additional insight intothe evolution of vertebrate species in the Galapagos.\n",
      "CI  - (c) The American Genetic Association 2018. All rights reserved. For permissions, please e-mail: journals.permissions@oup.com.\n",
      "FAU - Hedrick, Philip W\n",
      "AU  - Hedrick PW\n",
      "AD  - School of Life Sciences, Arizona State University, Tempe, AZ.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - United States\n",
      "TA  - J Hered\n",
      "JT  - The Journal of heredity\n",
      "JID - 0375373\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Biodiversity\n",
      "MH  - *Biological Evolution\n",
      "MH  - Ecuador\n",
      "MH  - Gene Flow\n",
      "MH  - Genetic Drift\n",
      "MH  - Genetic Variation\n",
      "MH  - *Genetics, Population\n",
      "MH  - Geography\n",
      "MH  - Inbreeding\n",
      "MH  - Selection, Genetic\n",
      "MH  - Vertebrates/*genetics\n",
      "OTO - NOTNLM\n",
      "OT  - *gene flow\n",
      "OT  - *genetic drift\n",
      "OT  - *hybridization\n",
      "OT  - *inbreeding\n",
      "OT  - *selection\n",
      "EDAT- 2018/12/13 06:00\n",
      "MHDA- 2020/04/30 06:00\n",
      "CRDT- 2018/12/13 06:00\n",
      "PHST- 2018/12/10 00:00 [accepted]\n",
      "PHST- 2018/12/13 06:00 [pubmed]\n",
      "PHST- 2020/04/30 06:00 [medline]\n",
      "PHST- 2018/12/13 06:00 [entrez]\n",
      "AID - 5239640 [pii]\n",
      "AID - 10.1093/jhered/esy066 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Hered. 2019 Mar 5;110(2):137-157. doi: 10.1093/jhered/esy066.\n",
      "\n",
      "PMID- 30514945\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20190509\n",
      "LR  - 20190509\n",
      "IS  - 1476-4687 (Electronic)\n",
      "IS  - 0028-0836 (Linking)\n",
      "VI  - 564\n",
      "IP  - 7734\n",
      "DP  - 2018 Dec\n",
      "TI  - The secrets of Lonesome George.\n",
      "PG  - 5-6\n",
      "LID - 10.1038/d41586-018-07630-4 [doi]\n",
      "LA  - eng\n",
      "PT  - News\n",
      "PT  - Comment\n",
      "PL  - England\n",
      "TA  - Nature\n",
      "JT  - Nature\n",
      "JID - 0410462\n",
      "SB  - IM\n",
      "CON - Nature. 2015 Jan 15;517(7534):271. PMID: 25592525\n",
      "MH  - Animals\n",
      "MH  - Ecuador\n",
      "MH  - Genomics\n",
      "MH  - *Turtles\n",
      "OTO - NOTNLM\n",
      "OT  - *Ageing\n",
      "OT  - *Evolution\n",
      "OT  - *Genomics\n",
      "OT  - *Immunology\n",
      "EDAT- 2018/12/06 06:00\n",
      "MHDA- 2019/05/10 06:00\n",
      "CRDT- 2018/12/06 06:00\n",
      "PHST- 2018/12/06 06:00 [entrez]\n",
      "PHST- 2018/12/06 06:00 [pubmed]\n",
      "PHST- 2019/05/10 06:00 [medline]\n",
      "AID - 10.1038/d41586-018-07630-4 [doi]\n",
      "AID - 10.1038/d41586-018-07630-4 [pii]\n",
      "PST - ppublish\n",
      "SO  - Nature. 2018 Dec;564(7734):5-6. doi: 10.1038/d41586-018-07630-4.\n",
      "\n",
      "PMID- 30510174\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20190523\n",
      "LR  - 20210109\n",
      "IS  - 2397-334X (Electronic)\n",
      "IS  - 2397-334X (Linking)\n",
      "VI  - 3\n",
      "IP  - 1\n",
      "DP  - 2019 Jan\n",
      "TI  - Giant tortoise genomes provide insights into longevity and age-related disease.\n",
      "PG  - 87-95\n",
      "LID - 10.1038/s41559-018-0733-x [doi]\n",
      "AB  - Giant tortoises are among the longest-lived vertebrate animals and, as such,provide an excellent model to study traits like longevity and age-relateddiseases. However, genomic and molecular evolutionary information on gianttortoises is scarce. Here, we describe a global analysis of the genomes ofLonesome George-the iconic last member of Chelonoidis abingdonii-and the Aldabra giant tortoise (Aldabrachelys gigantea). Comparison of these genomes with thoseof related species, using both unsupervised and supervised analyses, led us todetect lineage-specific variants affecting DNA repair genes, inflammatorymediators and genes related to cancer development. Our study also hints atspecific evolutionary strategies linked to increased lifespan, and expands ourunderstanding of the genomic determinants of ageing. These new genome sequencesalso provide important resources to help the efforts for restoration of gianttortoise populations.\n",
      "FAU - Quesada, Victor\n",
      "AU  - Quesada V\n",
      "AUID- ORCID: http://orcid.org/0000-0002-8398-3457\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Freitas-Rodriguez, Sandra\n",
      "AU  - Freitas-Rodriguez S\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Miller, Joshua\n",
      "AU  - Miller J\n",
      "AUID- ORCID: http://orcid.org/0000-0002-4019-7675\n",
      "AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT,USA.\n",
      "FAU - Perez-Silva, Jose G\n",
      "AU  - Perez-Silva JG\n",
      "AUID- ORCID: http://orcid.org/0000-0003-4421-3676\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Jiang, Zi-Feng\n",
      "AU  - Jiang ZF\n",
      "AD  - Institute for Genomics and Systems Biology, The University of Chicago, Chicago,IL, USA.\n",
      "FAU - Tapia, Washington\n",
      "AU  - Tapia W\n",
      "AD  - Galapagos National Park Directorate, Galapagos Islands, Ecuador.\n",
      "AD  - Galapagos Conservancy, Fairfax, VA, USA.\n",
      "FAU - Santiago-Fernandez, Olaya\n",
      "AU  - Santiago-Fernandez O\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Campos-Iglesias, Diana\n",
      "AU  - Campos-Iglesias D\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Kuderna, Lukas F K\n",
      "AU  - Kuderna LFK\n",
      "AUID- ORCID: http://orcid.org/0000-0002-9992-9295\n",
      "AD  - Institute of Evolutionary Biology (UPF-CSIC), Barcelona, Spain.\n",
      "AD  - CNAG-CRG, Centre for Genomic Regulation, Barcelona Institute of Science andTechnology , Barcelona, Spain.\n",
      "FAU - Quinzin, Maud\n",
      "AU  - Quinzin M\n",
      "AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT,USA.\n",
      "FAU - Alvarez, Miguel G\n",
      "AU  - Alvarez MG\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Carrero, Dido\n",
      "AU  - Carrero D\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Beheregaray, Luciano B\n",
      "AU  - Beheregaray LB\n",
      "AD  - College of Science and Engineering, Flinders University, Adelaide, SouthAustralia, Australia.\n",
      "FAU - Gibbs, James P\n",
      "AU  - Gibbs JP\n",
      "AD  - College of Environmental Science and Forestry, State University of New York,Syracuse, NY, USA.\n",
      "FAU - Chiari, Ylenia\n",
      "AU  - Chiari Y\n",
      "AUID- ORCID: http://orcid.org/0000-0003-2338-8602\n",
      "AD  - Department of Biology, University of South Alabama, Mobile, AL, USA.\n",
      "FAU - Glaberman, Scott\n",
      "AU  - Glaberman S\n",
      "AUID- ORCID: http://orcid.org/0000-0003-0594-4732\n",
      "AD  - Department of Biology, University of South Alabama, Mobile, AL, USA.\n",
      "FAU - Ciofi, Claudio\n",
      "AU  - Ciofi C\n",
      "AUID- ORCID: http://orcid.org/0000-0001-8537-8659\n",
      "AD  - Department of Biology, University of Florence, Florence, Italy.\n",
      "FAU - Araujo-Voces, Miguel\n",
      "AU  - Araujo-Voces M\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Mayoral, Pablo\n",
      "AU  - Mayoral P\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Arango, Javier R\n",
      "AU  - Arango JR\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Tamargo-Gomez, Isaac\n",
      "AU  - Tamargo-Gomez I\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Roiz-Valle, David\n",
      "AU  - Roiz-Valle D\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Pascual-Torner, Maria\n",
      "AU  - Pascual-Torner M\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Evans, Benjamin R\n",
      "AU  - Evans BR\n",
      "AUID- ORCID: http://orcid.org/0000-0003-2069-4938\n",
      "AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT,USA.\n",
      "FAU - Edwards, Danielle L\n",
      "AU  - Edwards DL\n",
      "AD  - School of Natural Sciences, University of California, Merced, CA, USA.\n",
      "FAU - Garrick, Ryan C\n",
      "AU  - Garrick RC\n",
      "AD  - Department of Biology, University of Mississippi, Oxford, MS, USA.\n",
      "FAU - Russello, Michael A\n",
      "AU  - Russello MA\n",
      "AUID- ORCID: http://orcid.org/0000-0002-9297-2485\n",
      "AD  - Department of Biology, The University of British Columbia, Kelowna, BritishColumbia, Canada.\n",
      "FAU - Poulakakis, Nikos\n",
      "AU  - Poulakakis N\n",
      "AD  - Department of Biology, School of Sciences and Engineering, University of Crete,Heraklion, Greece.\n",
      "AD  - Natural History Museum of Crete, Heraklion, Greece.\n",
      "FAU - Gaughran, Stephen J\n",
      "AU  - Gaughran SJ\n",
      "AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT,USA.\n",
      "FAU - Rueda, Danny O\n",
      "AU  - Rueda DO\n",
      "AD  - Galapagos National Park Directorate, Galapagos Islands, Ecuador.\n",
      "FAU - Bretones, Gabriel\n",
      "AU  - Bretones G\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain.\n",
      "FAU - Marques-Bonet, Tomas\n",
      "AU  - Marques-Bonet T\n",
      "AUID- ORCID: http://orcid.org/0000-0002-5597-3075\n",
      "AD  - Institute of Evolutionary Biology (UPF-CSIC), Barcelona, Spain.\n",
      "AD  - CNAG-CRG, Centre for Genomic Regulation, Barcelona Institute of Science andTechnology , Barcelona, Spain.\n",
      "AD  - Catalan Institution of Research and Advanced Studies, Barcelona, Spain.\n",
      "AD  - Institut Catala de Paleontologia Miquel Crusafont, Universitat Autonoma deBarcelona, Barcelona, Spain.\n",
      "FAU - White, Kevin P\n",
      "AU  - White KP\n",
      "AD  - Institute for Genomics and Systems Biology, The University of Chicago, Chicago,IL, USA.\n",
      "FAU - Caccone, Adalgisa\n",
      "AU  - Caccone A\n",
      "AUID- ORCID: http://orcid.org/0000-0002-8949-9260\n",
      "AD  - Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT,USA. adalgisa.caccone@yale.edu.\n",
      "FAU - Lopez-Otin, Carlos\n",
      "AU  - Lopez-Otin C\n",
      "AUID- ORCID: http://orcid.org/0000-0001-6964-1904\n",
      "AD  - Departamento de Bioquimica y Biologia Molecular, Instituto Universitario deOncologia del Principado de Asturias, CIBERONC, Universidad de Oviedo, Oviedo,Spain. clo@uniovi.es.\n",
      "LA  - eng\n",
      "GR  - U01 MH106874/MH/NIMH NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20181203\n",
      "PL  - England\n",
      "TA  - Nat Ecol Evol\n",
      "JT  - Nature ecology & evolution\n",
      "JID - 101698577\n",
      "RN  - 0 (Inflammation Mediators)\n",
      "SB  - IM\n",
      "MH  - Aging/*genetics\n",
      "MH  - Animals\n",
      "MH  - DNA Repair/genetics\n",
      "MH  - Evolution, Molecular\n",
      "MH  - *Genome\n",
      "MH  - HEK293 Cells\n",
      "MH  - Humans\n",
      "MH  - Inflammation Mediators\n",
      "MH  - Male\n",
      "MH  - Neoplasms/genetics\n",
      "MH  - Phylogeny\n",
      "MH  - Population Density\n",
      "MH  - Turtles/*genetics\n",
      "PMC - PMC6314442\n",
      "MID - EMS80309\n",
      "EDAT- 2018/12/05 06:00\n",
      "MHDA- 2019/05/24 06:00\n",
      "CRDT- 2018/12/05 06:00\n",
      "PHST- 2018/01/24 00:00 [received]\n",
      "PHST- 2018/10/25 00:00 [accepted]\n",
      "PHST- 2018/12/05 06:00 [pubmed]\n",
      "PHST- 2019/05/24 06:00 [medline]\n",
      "PHST- 2018/12/05 06:00 [entrez]\n",
      "AID - 10.1038/s41559-018-0733-x [doi]\n",
      "AID - 10.1038/s41559-018-0733-x [pii]\n",
      "PST - ppublish\n",
      "SO  - Nat Ecol Evol. 2019 Jan;3(1):87-95. doi: 10.1038/s41559-018-0733-x. Epub 2018 Dec3.\n",
      "\n",
      "PMID- 30459831\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20201001\n",
      "IS  - 1752-4571 (Print)\n",
      "IS  - 1752-4571 (Linking)\n",
      "VI  - 11\n",
      "IP  - 10\n",
      "DP  - 2018 Dec\n",
      "TI  - Population genomics through time provides insights into the consequences ofdecline and rapid demographic recovery through head-starting in a Galapagos gianttortoise.\n",
      "PG  - 1811-1821\n",
      "LID - 10.1111/eva.12682 [doi]\n",
      "AB  - Population genetic theory related to the consequences of rapid population declineis well-developed, but there are very few empirical studies where sampling wasconducted before and after a known bottleneck event. Such knowledge is ofparticular importance for species restoration, given links between geneticdiversity and the probability of long-term persistence. To directly evaluate the relationship between current genetic diversity and past demographic events, wecollected genome-wide single nucleotide polymorphism data from prebottleneckhistorical (c.1906) and postbottleneck contemporary (c.2014) samples of Pinzongiant tortoises (Chelonoidis duncanensis; n = 25 and 149 individuals,respectively) endemic to a single island in the Galapagos. Pinzon giant tortoiseshad a historically large population size that was reduced to just 150-200individuals in the mid 20th century. Since then, Pinzon's tortoise population hasrecovered through an ex situ head-start programme in which eggs or pre-emergentindividuals were collected from natural nests on the island, reared ex situ incaptivity until they were 4-5 years old and subsequently repatriated. We foundthat the extent and distribution of genetic variation in the historical andcontemporary samples were very similar, with the latter group not exhibiting the characteristic genetic patterns of recent population decline. No populationstructure was detected either spatially or temporally. We estimated an effective population size (N e) of 58 (95% CI = 50-69) for the postbottleneck population;no prebottleneck N e point estimate was attainable (95% CI = 39-infinity) likely due to the sample size being lower than the true N e. Overall, the historicalsample provided a valuable benchmark for evaluating the head-start captivebreeding programme, revealing high retention of genetic variation and no skew in representation despite the documented bottleneck event. Moreover, this workdemonstrates the effectiveness of head-starting in rescuing the Pinzon gianttortoise from almost certain extinction.\n",
      "FAU - Jensen, Evelyn L\n",
      "AU  - Jensen EL\n",
      "AUID- ORCID: 0000-0002-1819-3117\n",
      "AD  - Department of Biology University of British Columbia Okanagan Kelowna BritishColumbia Canada.\n",
      "AD  - Present address: Department of Biology Queen's University Kingston OntarioCanada.\n",
      "FAU - Edwards, Danielle L\n",
      "AU  - Edwards DL\n",
      "AD  - Life and Environmental Sciences University of California Merced California.\n",
      "FAU - Garrick, Ryan C\n",
      "AU  - Garrick RC\n",
      "AD  - Department of Biology University of Mississippi Oxford Mississippi.\n",
      "FAU - Miller, Joshua M\n",
      "AU  - Miller JM\n",
      "AUID- ORCID: 0000-0002-4019-7675\n",
      "AD  - Department of Ecology and Evolutionary Biology Yale University New HavenConnecticut.\n",
      "FAU - Gibbs, James P\n",
      "AU  - Gibbs JP\n",
      "AD  - College of Environmental Science and Forestry State University of New YorkSyracuse New York.\n",
      "FAU - Cayot, Linda J\n",
      "AU  - Cayot LJ\n",
      "AD  - Galapagos Conservancy Fairfax Virginia.\n",
      "FAU - Tapia, Washington\n",
      "AU  - Tapia W\n",
      "AD  - Department of Applied Research Galapagos National Park Service Puerto AyoraEcuador.\n",
      "AD  - Galapagos Conservancy Santa Cruz Ecuador.\n",
      "FAU - Caccone, Adalgisa\n",
      "AU  - Caccone A\n",
      "AD  - Department of Ecology and Evolutionary Biology Yale University New HavenConnecticut.\n",
      "FAU - Russello, Michael A\n",
      "AU  - Russello MA\n",
      "AUID- ORCID: 0000-0002-9297-2485\n",
      "AD  - Department of Biology University of British Columbia Okanagan Kelowna BritishColumbia Canada.\n",
      "LA  - eng\n",
      "SI  - Dryad/10.5061/dryad.7tp3sg0\n",
      "PT  - Journal Article\n",
      "DEP - 20180813\n",
      "PL  - England\n",
      "TA  - Evol Appl\n",
      "JT  - Evolutionary applications\n",
      "JID - 101461828\n",
      "PMC - PMC6231475\n",
      "OTO - NOTNLM\n",
      "OT  - Chelonoidis\n",
      "OT  - RAD-seq\n",
      "OT  - bottleneck\n",
      "OT  - effective population size\n",
      "OT  - historical DNA\n",
      "OT  - hybrid capture\n",
      "OT  - museum specimen\n",
      "OT  - population genetics\n",
      "EDAT- 2018/11/22 06:00\n",
      "MHDA- 2018/11/22 06:01\n",
      "CRDT- 2018/11/22 06:00\n",
      "PHST- 2018/04/27 00:00 [received]\n",
      "PHST- 2018/07/09 00:00 [revised]\n",
      "PHST- 2018/07/16 00:00 [accepted]\n",
      "PHST- 2018/11/22 06:00 [entrez]\n",
      "PHST- 2018/11/22 06:00 [pubmed]\n",
      "PHST- 2018/11/22 06:01 [medline]\n",
      "AID - 10.1111/eva.12682 [doi]\n",
      "AID - EVA12682 [pii]\n",
      "PST - epublish\n",
      "SO  - Evol Appl. 2018 Aug 13;11(10):1811-1821. doi: 10.1111/eva.12682. eCollection 2018Dec.\n",
      "\n",
      "PMID- 30371872\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20190408\n",
      "LR  - 20190408\n",
      "IS  - 1076-836X (Electronic)\n",
      "IS  - 1063-5157 (Linking)\n",
      "VI  - 68\n",
      "IP  - 2\n",
      "DP  - 2019 Mar 1\n",
      "TI  - Species Limits and Phylogenomic Relationships of Darwin's Finches RemainUnresolved: Potential Consequences of a Volatile Ecological Setting.\n",
      "PG  - 347-357\n",
      "LID - 10.1093/sysbio/syy073 [doi]\n",
      "AB  - Island biotas have become paradigms for illustrating many evolutionary processes.The fauna of the Galapagos Islands includes several taxa that have been focalpoints for evolutionary studies. Perhaps their most famous inhabitants, Darwin's finches, represent a go-to icon when thinking about how species originate andadapt to the environment. However, unlike other adaptive radiations, pastmorphological and molecular studies of Darwin's finches have yielded inconsistenthypotheses of species limits and phylogenetic relationships. Expecting thatidiosyncrasies of prior data and analytic methods explained different proposedclassifications, we were surprised to observe that three new phylogenetichypotheses derived mostly from the same genomics data were topologicallyinconsistent. We found that the differences between some of these genomics trees were as great as one would expect between two random trees with the same numberof taxa. Thus, the phylogeny of Darwin's finches remains unresolved, as it hasfor more than a century. A component of phylogenetic uncertainty comes fromunclear species limits, under any species concept, in the ground finches(Geospiza) and tree finches (Camarhynchus). We suggest that past authors shouldhave tested the species limits of Lack, rather than uncritically accepting them. In fact, the impressive amount of genomics data do not provide unambiguoushypotheses of the number of species of Geospiza or Camarhynchus, although theyimply greater species diversity than Lack's taxonomy. We suggest thatinsufficient sampling of species populations across islands (35.6% formorphometrics and 20.4% for genomics) prevents accurate diagnoses of specieslimits. However, it is unknown whether samples from a greater number of islandsmight result in bridging differences between species, or reveal many new ones. Weconclude that attempts to interpret patterns of variation among the finches understandard evolutionary paradigms have obscured some major messages, mostspecifically the ongoing reciprocal interactions between geographic isolation andlineage divergence, and dispersal and gene flow caused by the volatile ecologicalconditions in the islands. Although the finches provide textbook examples ofnatural selection, better understanding of species limits and a robustphylogenetic hypothesis are required to corroborate past hypotheses of speciationand adaptive radiation in the finches of the Galapagos.\n",
      "FAU - Zink, Robert M\n",
      "AU  - Zink RM\n",
      "AD  - School of Natural Resources.\n",
      "AD  - School of Biological Sciences.\n",
      "AD  - Nebraska State Museum, University of Nebraska-Lincoln, Lincoln, NE 68583, USA.\n",
      "FAU - Vazquez-Miranda, Hernan\n",
      "AU  - Vazquez-Miranda H\n",
      "AD  - School of Natural Resources.\n",
      "AD  - Nebraska State Museum, University of Nebraska-Lincoln, Lincoln, NE 68583, USA.\n",
      "AD  - Coleccion Nacional de Aves (CNAV), Departamento de Zoologia, Instituto deBiologia, Universidad Nacional Autonoma de Mexico, Ciudad de Mexico, C.P. 04500, Mexico.\n",
      "LA  - eng\n",
      "SI  - Dryad/10.5061/dryad.dc82gp3\n",
      "PT  - Journal Article\n",
      "PL  - England\n",
      "TA  - Syst Biol\n",
      "JT  - Systematic biology\n",
      "JID - 9302532\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - *Ecosystem\n",
      "MH  - Ecuador\n",
      "MH  - Finches/*classification/genetics\n",
      "MH  - Genetic Variation\n",
      "MH  - Genome/genetics\n",
      "MH  - *Phylogeny\n",
      "MH  - Species Specificity\n",
      "EDAT- 2018/10/30 06:00\n",
      "MHDA- 2019/04/09 06:00\n",
      "CRDT- 2018/10/30 06:00\n",
      "PHST- 2018/05/26 00:00 [received]\n",
      "PHST- 2018/10/25 00:00 [accepted]\n",
      "PHST- 2018/10/30 06:00 [pubmed]\n",
      "PHST- 2019/04/09 06:00 [medline]\n",
      "PHST- 2018/10/30 06:00 [entrez]\n",
      "AID - 5146315 [pii]\n",
      "AID - 10.1093/sysbio/syy073 [doi]\n",
      "PST - ppublish\n",
      "SO  - Syst Biol. 2019 Mar 1;68(2):347-357. doi: 10.1093/sysbio/syy073.\n",
      "\n",
      "PMID- 30248857\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20181221\n",
      "LR  - 20181221\n",
      "IS  - 1879-1026 (Electronic)\n",
      "IS  - 0048-9697 (Linking)\n",
      "VI  - 645\n",
      "DP  - 2018 Dec 15\n",
      "TI  - Quito's virome: Metagenomic analysis of viral diversity in urban streams ofEcuador's capital city.\n",
      "PG  - 1334-1343\n",
      "LID - S0048-9697(18)32705-0 [pii]\n",
      "LID - 10.1016/j.scitotenv.2018.07.213 [doi]\n",
      "AB  - In Quito, the microbiological contamination of surface water represents a public health problem, mainly due to the lack of sewage treatment from urban wastewater.Contaminated water contributes to the transmission of many enteric pathogensthrough direct consumption, agricultural and recreational use. Among thedifferent pathogens present in urban discharges, viruses play an important roleon disease, being causes of gastroenteritis, hepatitis, meningitis, respiratoryinfections, among others. This study analyzes the presence of viruses in highlyimpacted surface waters of urban rivers using next-generation sequencingtechniques. Three representative locations of urban rivers, receiving the maindischarges from Quito sewerage system, were selected. Water samples of 500mL wereconcentrated by skimmed-milk flocculation method and the viral nucleic acid wasextracted and processed for high throughput sequencing using Illumina MiSeq. The results yielded very relevant data of circulating viruses in the capital ofEcuador. A total of 29 viral families were obtained, of which 26 species wereassociated with infections in humans. Among the 26 species identified, severalwere related to gastroenteritis: Human Mastadenovirus F, Bufavirus, Sapporovirus,Norwalk virus and Mamastrovirus 1. Also detected were: Gammapapillomavirusassociated with skin infections, Polyomavirus 1 related to cases of kidneydamage, Parechovirus A described as cause of neonatal sepsis with neurologicalaffectations and Hepatovirus A, the etiologic agent of Hepatitis A. Otheremergent viruses identified, of which its pathogenicity remains to be fullyclarified, were: Bocavirus, Circovirus, Aichi Virus and Cosavirus. The widediversity of species detected through metagenomics gives us key information aboutthe public health risks present in the urban rivers of Quito. In addition, thisstudy describes for the first time the presence of important infectious agentsnot previously reported in Ecuador and with very little reports in Latin America.\n",
      "CI  - Copyright (c) 2018 Elsevier B.V. All rights reserved.\n",
      "FAU - Guerrero-Latorre, Laura\n",
      "AU  - Guerrero-Latorre L\n",
      "AD  - Grupo de investigacion Biodiversidad, Medio Ambiente y Salud (BIOMAS), Facultadde Ingenierias y Ciencias Aplicadas (FICA), Ingenieria en Biotecnologia,Universidad de las Americas, Quito, Ecuador. Electronic address:laura.guerrero.latorre@gmail.com.\n",
      "FAU - Romero, Brigette\n",
      "AU  - Romero B\n",
      "AD  - Grupo de investigacion Biodiversidad, Medio Ambiente y Salud (BIOMAS), Facultadde Ingenierias y Ciencias Aplicadas (FICA), Ingenieria en Biotecnologia,Universidad de las Americas, Quito, Ecuador. Electronic address:bmromero@udlanet.ec.\n",
      "FAU - Bonifaz, Edison\n",
      "AU  - Bonifaz E\n",
      "AD  - Grupo de investigacion Biodiversidad, Medio Ambiente y Salud (BIOMAS), Facultadde Ingenierias y Ciencias Aplicadas (FICA), Ingenieria en Biotecnologia,Universidad de las Americas, Quito, Ecuador. Electronic address:ebonifaz@udlanet.ec.\n",
      "FAU - Timoneda, Natalia\n",
      "AU  - Timoneda N\n",
      "AD  - Laboratory of Virus Contaminants of Water and Food, Department of Genetics,Microbiology and Statistics, University of Barcelona, Barcelona, Catalonia,Spain.\n",
      "FAU - Rusinol, Marta\n",
      "AU  - Rusinol M\n",
      "AD  - Laboratory of Virus Contaminants of Water and Food, Department of Genetics,Microbiology and Statistics, University of Barcelona, Barcelona, Catalonia,Spain. Electronic address: mrusinol@ub.edu.\n",
      "FAU - Girones, Rosina\n",
      "AU  - Girones R\n",
      "AD  - Laboratory of Virus Contaminants of Water and Food, Department of Genetics,Microbiology and Statistics, University of Barcelona, Barcelona, Catalonia,Spain. Electronic address: rgirones@ub.edu.\n",
      "FAU - Rios-Touma, Blanca\n",
      "AU  - Rios-Touma B\n",
      "AD  - Grupo de investigacion Biodiversidad, Medio Ambiente y Salud (BIOMAS), Facultadde Ingenierias y Ciencias Aplicadas (FICA), Ingenieria Ambiental, Universidad de las Americas, Quito, Ecuador. Electronic address: blanca.rios@udla.edu.ec.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20180723\n",
      "PL  - Netherlands\n",
      "TA  - Sci Total Environ\n",
      "JT  - The Science of the total environment\n",
      "JID - 0330500\n",
      "SB  - IM\n",
      "MH  - Cities\n",
      "MH  - Ecuador\n",
      "MH  - *Environmental Monitoring\n",
      "MH  - Humans\n",
      "MH  - Metagenomics\n",
      "MH  - Rivers/*virology\n",
      "MH  - Water Pollution/*analysis/statistics & numerical data\n",
      "OTO - NOTNLM\n",
      "OT  - Human viruses\n",
      "OT  - Public health\n",
      "OT  - Surface water quality\n",
      "OT  - Urban river\n",
      "OT  - Virome\n",
      "EDAT- 2018/09/27 06:00\n",
      "MHDA- 2018/12/24 06:00\n",
      "CRDT- 2018/09/26 06:00\n",
      "PHST- 2018/05/25 00:00 [received]\n",
      "PHST- 2018/07/16 00:00 [revised]\n",
      "PHST- 2018/07/16 00:00 [accepted]\n",
      "PHST- 2018/09/26 06:00 [entrez]\n",
      "PHST- 2018/09/27 06:00 [pubmed]\n",
      "PHST- 2018/12/24 06:00 [medline]\n",
      "AID - S0048-9697(18)32705-0 [pii]\n",
      "AID - 10.1016/j.scitotenv.2018.07.213 [doi]\n",
      "PST - ppublish\n",
      "SO  - Sci Total Environ. 2018 Dec 15;645:1334-1343. doi:10.1016/j.scitotenv.2018.07.213. Epub 2018 Jul 23.\n",
      "\n",
      "PMID- 30228363\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20191114\n",
      "LR  - 20211204\n",
      "IS  - 2045-2322 (Electronic)\n",
      "IS  - 2045-2322 (Linking)\n",
      "VI  - 8\n",
      "IP  - 1\n",
      "DP  - 2018 Sep 18\n",
      "TI  - Analysis of Racial/Ethnic Representation in Select Basic and Applied CancerResearch Studies.\n",
      "PG  - 13978\n",
      "LID - 10.1038/s41598-018-32264-x [doi]\n",
      "AB  - Over the past decades, consistent studies have shown that race/ethnicity have agreat impact on cancer incidence, survival, drug response, molecular pathways andepigenetics. Despite the influence of race/ethnicity in cancer outcomes and itsimpact in health care quality, a comprehensive understanding of racial/ethnicinclusion in oncological research has never been addressed. We therefore exploredthe racial/ethnic composition of samples/individuals included in fundamental(patient-derived oncological models, biobanks and genomics) and applied cancerresearch studies (clinical trials). Regarding patient-derived oncological models (n = 794), 48.3% have no records on their donor's race/ethnicity, the rest wereisolated from White (37.5%), Asian (10%), African American (3.8%) and Hispanic(0.4%) donors. Biobanks (n = 8,293) hold specimens from unknown (24.56%), White(59.03%), African American (11.05%), Asian (4.12%) and other individuals (1.24%).Genomic projects (n = 6,765,447) include samples from unknown (0.6%), White(91.1%), Asian (5.6%), African American (1.7%), Hispanic (0.5%) and otherpopulations (0.5%). Concerning clinical trials (n = 89,212), no racial/ethnicregistries were found in 66.95% of participants, and records were mainly obtainedfrom Whites (25.94%), Asians (4.97%), African Americans (1.08%), Hispanics(0.16%) and other minorities (0.9%). Thus, two tendencies were observed acrossoncological studies: lack of racial/ethnic information and overrepresentation of Caucasian/White samples/individuals. These results clearly indicate a need todiversify oncological studies to other populations along with novel strategies toenhanced race/ethnicity data recording and reporting.\n",
      "FAU - Guerrero, Santiago\n",
      "AU  - Guerrero S\n",
      "AUID- ORCID: 0000-0003-3473-7214\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, BlockI, 2nd floor, 170129, Quito, Ecuador. santiago.guerrero@ute.edu.ec.\n",
      "FAU - Lopez-Cortes, Andres\n",
      "AU  - Lopez-Cortes A\n",
      "AUID- ORCID: 0000-0003-1503-1929\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, BlockI, 2nd floor, 170129, Quito, Ecuador.\n",
      "FAU - Indacochea, Alberto\n",
      "AU  - Indacochea A\n",
      "AD  - Gene Regulation, Stem Cells and Cancer Programme, Centre for Genomic Regulation(CRG), The Barcelona Institute of Science and Technology, Barcelona, Spain.\n",
      "AD  - Oncology and Molecular Pathology Research Group-VHIR- Vall d' Hebron Institut de Recerca-Vall d' Hebron Hospital, P/de la Vall d'Hebron, Barcelona, Spain.\n",
      "FAU - Garcia-Cardenas, Jennyfer M\n",
      "AU  - Garcia-Cardenas JM\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, BlockI, 2nd floor, 170129, Quito, Ecuador.\n",
      "FAU - Zambrano, Ana Karina\n",
      "AU  - Zambrano AK\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, BlockI, 2nd floor, 170129, Quito, Ecuador.\n",
      "FAU - Cabrera-Andrade, Alejandro\n",
      "AU  - Cabrera-Andrade A\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, BlockI, 2nd floor, 170129, Quito, Ecuador.\n",
      "AD  - Carrera de Enfermeria, Facultad de Ciencias de la Salud, Universidad de lasAmericas, Avenue de los Granados, Quito, 170125, Ecuador.\n",
      "AD  - Grupo de Bio-Quimioinformatica, Universidad de las Americas, Avenue de losGranados, Quito, 170125, Ecuador.\n",
      "FAU - Guevara-Ramirez, Patricia\n",
      "AU  - Guevara-Ramirez P\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, BlockI, 2nd floor, 170129, Quito, Ecuador.\n",
      "FAU - Gonzalez, Diana Abigail\n",
      "AU  - Gonzalez DA\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, BlockI, 2nd floor, 170129, Quito, Ecuador.\n",
      "FAU - Leone, Paola E\n",
      "AU  - Leone PE\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, BlockI, 2nd floor, 170129, Quito, Ecuador.\n",
      "FAU - Paz-Y-Mino, Cesar\n",
      "AU  - Paz-Y-Mino C\n",
      "AD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la SaludEugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, BlockI, 2nd floor, 170129, Quito, Ecuador. cesar.pazymino@ute.edu.ec.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20180918\n",
      "PL  - England\n",
      "TA  - Sci Rep\n",
      "JT  - Scientific reports\n",
      "JID - 101563288\n",
      "SB  - IM\n",
      "MH  - Biological Specimen Banks/*statistics & numerical data\n",
      "MH  - *Biomedical Research\n",
      "MH  - Clinical Trials as Topic/*statistics & numerical data\n",
      "MH  - Ethnicity/*statistics & numerical data\n",
      "MH  - Female\n",
      "MH  - Genomics/*methods\n",
      "MH  - Humans\n",
      "MH  - Male\n",
      "MH  - Neoplasms/diagnosis/*epidemiology/therapy\n",
      "MH  - Patient Participation\n",
      "MH  - Racial Groups/*statistics & numerical data\n",
      "MH  - Registries/*statistics & numerical data\n",
      "MH  - United States/epidemiology\n",
      "PMC - PMC6143551\n",
      "EDAT- 2018/09/20 06:00\n",
      "MHDA- 2019/11/15 06:00\n",
      "CRDT- 2018/09/20 06:00\n",
      "PHST- 2018/02/23 00:00 [received]\n",
      "PHST- 2018/07/26 00:00 [accepted]\n",
      "PHST- 2018/09/20 06:00 [entrez]\n",
      "PHST- 2018/09/20 06:00 [pubmed]\n",
      "PHST- 2019/11/15 06:00 [medline]\n",
      "AID - 10.1038/s41598-018-32264-x [doi]\n",
      "AID - 10.1038/s41598-018-32264-x [pii]\n",
      "PST - epublish\n",
      "SO  - Sci Rep. 2018 Sep 18;8(1):13978. doi: 10.1038/s41598-018-32264-x.\n",
      "\n",
      "PMID- 30096036\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20200207\n",
      "LR  - 20200207\n",
      "IS  - 1943-3700 (Electronic)\n",
      "IS  - 0090-3558 (Linking)\n",
      "VI  - 55\n",
      "IP  - 1\n",
      "DP  - 2019 Jan\n",
      "TI  - Fibropapillomatosis in a Green Sea Turtle ( Chelonia mydas) from the SoutheasternPacific.\n",
      "PG  - 169-173\n",
      "LID - 10.7589/2017-12-295 [doi]\n",
      "AB  - Fibropapillomatosis is a neoplastic disease that afflicts sea turtles. Althoughit is disseminated worldwide, cases of the disease have not been reported in the southeastern Pacific region. We describe a case of fibropapillomatosis in a greensea turtle ( Chelonia mydas) during its rehabilitation at the Machalilla NationalPark Rehabilitation Center, Ecuador. Viral presence was confirmed by PCR,targeting fragments of the chelonid alphaherpesvirus 5 (ChHV5) unique long (UL)genes, UL27, UL28, and UL30. The amplicons were sequenced and included in aglobal phylogenetic analysis of the virus with other reported sequences fromGenBank. Results showed that the available viral sequences segregated into fivephylogeographic groups: western Atlantic and eastern Caribbean, central Pacific, western Pacific, Atlantic, and eastern Pacific groups. The concatenated ChHV5sequences from Ecuador clustered with the eastern Pacific sequences.\n",
      "FAU - Cardenas, Diana M\n",
      "AU  - Cardenas DM\n",
      "AD  - 1 Laboratorio para Investigaciones Biomedicas, Facultad de Ciencias de la Vida,Escuela Superior Politecnica del Litoral, 090920, Guayaquil, Guayas, Ecuador.\n",
      "FAU - Cucalon, Roberto V\n",
      "AU  - Cucalon RV\n",
      "AD  - 1 Laboratorio para Investigaciones Biomedicas, Facultad de Ciencias de la Vida,Escuela Superior Politecnica del Litoral, 090920, Guayaquil, Guayas, Ecuador.\n",
      "FAU - Medina-Magues, Lex G\n",
      "AU  - Medina-Magues LG\n",
      "AD  - 1 Laboratorio para Investigaciones Biomedicas, Facultad de Ciencias de la Vida,Escuela Superior Politecnica del Litoral, 090920, Guayaquil, Guayas, Ecuador.\n",
      "FAU - Jones, Karina\n",
      "AU  - Jones K\n",
      "AD  - 2 College of Public Health, Medical and Veterinary Sciences, James CookUniversity, 4811, Townsville, Queensland, Australia.\n",
      "FAU - Aleman, Ruben A\n",
      "AU  - Aleman RA\n",
      "AD  - 3 Parque Nacional Machalilla-Ministerio del Ambiente, Garcia Moreno y EloyAlfaro, 130315, Puerto Lopez, Manabi, Ecuador.\n",
      "FAU - Alfaro-Nunez, Alonzo\n",
      "AU  - Alfaro-Nunez A\n",
      "AD  - 4 Section for Evolutionary Genomics, Centre for GeoGenetics, Natural HistoryMuseum of Denmark, 1353 Copenhagen K, Denmark.\n",
      "AD  - 5 School of Biological Science and Engineering, Yachay Tech University, 100115,Hacienda San Jose, Urcuqui-Imbabura, Ecuador.\n",
      "FAU - Cardenas, Washington B\n",
      "AU  - Cardenas WB\n",
      "AD  - 1 Laboratorio para Investigaciones Biomedicas, Facultad de Ciencias de la Vida,Escuela Superior Politecnica del Litoral, 090920, Guayaquil, Guayas, Ecuador.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20180810\n",
      "PL  - United States\n",
      "TA  - J Wildl Dis\n",
      "JT  - Journal of wildlife diseases\n",
      "JID - 0244160\n",
      "SB  - IM\n",
      "MH  - Alphaherpesvirinae/*genetics\n",
      "MH  - Animals\n",
      "MH  - Herpesviridae Infections/epidemiology/*veterinary/virology\n",
      "MH  - Pacific Ocean/epidemiology\n",
      "MH  - Phylogeny\n",
      "MH  - Skin Neoplasms/*veterinary/virology\n",
      "MH  - Turtles/*virology\n",
      "OTO - NOTNLM\n",
      "OT  - *\n",
      "OT  - *Ecuador\n",
      "OT  - *chelonid alphaherpesvirus 5\n",
      "OT  - *fibropapillomatosis\n",
      "OT  - *phylogenetics\n",
      "OT  - *sea turtles\n",
      "OT  - *southeastern Pacific\n",
      "EDAT- 2018/08/11 06:00\n",
      "MHDA- 2020/02/08 06:00\n",
      "CRDT- 2018/08/11 06:00\n",
      "PHST- 2018/08/11 06:00 [pubmed]\n",
      "PHST- 2020/02/08 06:00 [medline]\n",
      "PHST- 2018/08/11 06:00 [entrez]\n",
      "AID - 10.7589/2017-12-295 [pii]\n",
      "AID - 10.7589/2017-12-295 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Wildl Dis. 2019 Jan;55(1):169-173. doi: 10.7589/2017-12-295. Epub 2018 Aug 10.\n",
      "\n",
      "PMID- 30056980\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20181120\n",
      "LR  - 20181202\n",
      "IS  - 1873-2550 (Electronic)\n",
      "IS  - 0304-4017 (Linking)\n",
      "VI  - 259\n",
      "DP  - 2018 Aug 15\n",
      "TI  - The somatic proteins of Toxocara canis larvae and excretory-secretory productsrevealed by proteomics.\n",
      "PG  - 25-34\n",
      "LID - S0304-4017(18)30255-3 [pii]\n",
      "LID - 10.1016/j.vetpar.2018.06.015 [doi]\n",
      "AB  - Toxocariasis is a widespread helminth infection of dogs and cats, caused byToxocara canis and Toxocara cati larvae, respectively. Toxocara spp. can causezoonotic infections in humans by invading tissues and organs causing pathology.Toxocara spp. larvae release excretory-secretory molecules (TES) into the body oftheir host that are fundamental to the host-parasite interaction and could beused as targets for novel diagnostics and vaccines. In the present study, weidentified 646 T. canis proteins from TES and larval extract using 1D-SDS PAGEfollowed by mass spectrometry. A wide range of proteins was identified that mayplay a role both in the induction of the host immune response and host pathology,and in parasite metabolism and survival. Among these proteins there are potentialcandidates for novel diagnostics and vaccines for dogs and cats toxocariases.\n",
      "CI  - Copyright (c) 2018 Elsevier B.V. All rights reserved.\n",
      "FAU - da Silva, Marcia B\n",
      "AU  - da Silva MB\n",
      "AD  - Instituto de Ciencias da Saude, Universidade Federal da Bahia, Salvador, Bahia,Brazil. Electronic address: marciasilva25j@yahoo.com.br.\n",
      "FAU - Urrego A, Juan R\n",
      "AU  - Urrego A JR\n",
      "AD  - Instituto de Ciencias da Saude, Universidade Federal da Bahia, Salvador, Bahia,Brazil; Department of Pharmaceutical Sciences, University of Cartagena,Cartagena, Colombia. Electronic address: ricardo_urrego@hotmail.com.\n",
      "FAU - Oviedo, Yisela\n",
      "AU  - Oviedo Y\n",
      "AD  - Instituto de Ciencias da Saude, Universidade Federal da Bahia, Salvador, Bahia,Brazil. Electronic address: goviv1@yahoo.com.\n",
      "FAU - Cooper, Philip J\n",
      "AU  - Cooper PJ\n",
      "AD  - Facultad de Ciencias Medicas, de la Salud y la Vida, Universidad Internacionaldel Ecuador, Quito, Ecuador; Insitute of Infection and Immunity, St George'sUniversity of London, London, United Kingdom. Electronic address:pcooper@sgul.ac.uk.\n",
      "FAU - Pacheco, Luis G C\n",
      "AU  - Pacheco LGC\n",
      "AD  - Instituto de Ciencias da Saude, Universidade Federal da Bahia, Salvador, Bahia,Brazil. Electronic address: lgcpacheco@gmail.com.\n",
      "FAU - Pinheiro, Carina S\n",
      "AU  - Pinheiro CS\n",
      "AD  - Instituto de Ciencias da Saude, Universidade Federal da Bahia, Salvador, Bahia,Brazil. Electronic address: carinasilvapinheiro@gmail.com.\n",
      "FAU - Ferreira, Fatima\n",
      "AU  - Ferreira F\n",
      "AD  - Department of Biosciences, University of Salzburg, Salzburg, Austria. Electronic address: fatima.ferreira-briza@sbg.ac.at.\n",
      "FAU - Briza, Peter\n",
      "AU  - Briza P\n",
      "AD  - Department of Biosciences, University of Salzburg, Salzburg, Austria. Electronic address: peter.briza@sbg.ac.at.\n",
      "FAU - Alcantara-Neves, Neuza M\n",
      "AU  - Alcantara-Neves NM\n",
      "AD  - Instituto de Ciencias da Saude, Universidade Federal da Bahia, Salvador, Bahia,Brazil. Electronic address: neuza@ufba.br.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20180626\n",
      "PL  - Netherlands\n",
      "TA  - Vet Parasitol\n",
      "JT  - Veterinary parasitology\n",
      "JID - 7602745\n",
      "RN  - 0 (Antigens, Helminth)\n",
      "RN  - 0 (Helminth Proteins)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Antigens, Helminth/immunology\n",
      "MH  - Databases, Genetic\n",
      "MH  - Dogs\n",
      "MH  - Electrophoresis, Polyacrylamide Gel\n",
      "MH  - Female\n",
      "MH  - Helminth Proteins/*analysis/genetics/isolation & purification/*metabolism\n",
      "MH  - Humans\n",
      "MH  - Larva/*chemistry\n",
      "MH  - *Proteomics\n",
      "MH  - Toxocara canis/*chemistry/pathogenicity\n",
      "MH  - Toxocariasis/parasitology\n",
      "MH  - Zoonoses/parasitology/transmission\n",
      "OTO - NOTNLM\n",
      "OT  - Excretory/secretory proteins\n",
      "OT  - Larvae\n",
      "OT  - Proteomics\n",
      "OT  - Toxocara canis\n",
      "OT  - Toxocariasis\n",
      "EDAT- 2018/07/31 06:00\n",
      "MHDA- 2018/11/21 06:00\n",
      "CRDT- 2018/07/31 06:00\n",
      "PHST- 2018/03/15 00:00 [received]\n",
      "PHST- 2018/06/02 00:00 [revised]\n",
      "PHST- 2018/06/23 00:00 [accepted]\n",
      "PHST- 2018/07/31 06:00 [entrez]\n",
      "PHST- 2018/07/31 06:00 [pubmed]\n",
      "PHST- 2018/11/21 06:00 [medline]\n",
      "AID - S0304-4017(18)30255-3 [pii]\n",
      "AID - 10.1016/j.vetpar.2018.06.015 [doi]\n",
      "PST - ppublish\n",
      "SO  - Vet Parasitol. 2018 Aug 15;259:25-34. doi: 10.1016/j.vetpar.2018.06.015. Epub2018 Jun 26.\n",
      "\n",
      "PMID- 29969982\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20181218\n",
      "LR  - 20181218\n",
      "IS  - 1471-2164 (Electronic)\n",
      "IS  - 1471-2164 (Linking)\n",
      "VI  - 19\n",
      "IP  - 1\n",
      "DP  - 2018 Jul 3\n",
      "TI  - Genome sequence and effectorome of Moniliophthora perniciosa and Moniliophthoraroreri subpopulations.\n",
      "PG  - 509\n",
      "LID - 10.1186/s12864-018-4875-7 [doi]\n",
      "AB  - BACKGROUND: The hemibiotrophic pathogens Moniliophthora perniciosa (witches'broom disease) and Moniliophthora roreri (frosty pod rot disease) are among themost important pathogens of cacao. Moniliophthora perniciosa has a broad hostrange and infects a variety of meristematic tissues in cacao plants, whereas M.roreri infects only pods of Theobroma and Herrania genera. Comparativepathogenomics of these fungi is essential to understand Moniliophthora infection strategies, therefore the detection and in silico functional characterization of effector candidates are important steps to gain insight on their pathogenicity.RESULTS: Candidate secreted effector proteins repertoire were predicted using thegenomes of five representative isolates of M. perniciosa subpopulations (threefrom cacao and two from solanaceous hosts), and one representative isolate of M. roreri from Peru. Many putative effectors candidates were identified in M.perniciosa: 157 and 134 in cacao isolates from Bahia, Brazil; 109 in cacaoisolate from Ecuador, 92 and 80 in wild solanaceous isolates from Minas Gerais(Lobeira) and Bahia (Caicara), Brazil; respectively. Moniliophthora roreri showedthe highest number of effector candidates, a total of 243. A set of eight coreeffectors were shared among all Moniliophthora isolates, while others were sharedeither between the wild solanaceous isolates or among cacao isolates. Mostly,candidate effectors of M. perniciosa were shared among the isolates, whereas inM. roreri nearly 50% were exclusive to the specie. In addition, a large number ofcell wall-degrading enzymes characteristic of hemibiotrophic fungi were found.From these, we highlighted the proteins involved in cell wall modification, anenzymatic arsenal that allows the plant pathogens to inhabit environments withoxidative stress, which promotes degradation of plant compounds and facilitatesinfection. CONCLUSIONS: The present work reports six genomes and provides adatabase of the putative effectorome of Moniliophthora, a first step towards the understanding of the functional basis of fungal pathogenicity.\n",
      "FAU - Barbosa, Ceslaine Santos\n",
      "AU  - Barbosa CS\n",
      "AD  - Departamento de Ciencias Biologicas (DCB), Centro de Biotecnologia e Genetica(CBG), Universidade Estadual de Santa Cruz (UESC), Rodovia Ilheus-Itabuna, km 16,Ilheus, 45662-900, Bahia, Brazil.\n",
      "AD  - Comissao Executiva do Plano da Lavoura Cacaueira (CEPLAC), Centro de Pesquisas doCacau (CEPEC), Secao de Fitossanidade (SEFIT), Laboratorio de FitopatologiaMolecular (FITOMOL), km 22 Rod. Ilheus Itabuna, Ilheus, 45600-970, Bahia, Brazil.\n",
      "FAU - Fonseca, Rute R da\n",
      "AU  - Fonseca RRD\n",
      "AD  - The Bioinformatics Centre, Department of Biology, University of Copenhagen,Copenhagen, Denmark.\n",
      "AD  - CIMAR/CIIMAR, Centro Interdisciplinar de Investigacao Marinha e Ambiental,Universidade do Porto, Porto, Portugal.\n",
      "FAU - Batista, Thiago Mafra\n",
      "AU  - Batista TM\n",
      "AD  - Departamento de Bioquimica e Imunologia, Universidade Federal de MinasGerais/Belo Horizonte, Minas Gerais, Brazil.\n",
      "FAU - Barreto, Mariana Araujo\n",
      "AU  - Barreto MA\n",
      "AD  - Departamento de Ciencias Biologicas (DCB), Centro de Biotecnologia e Genetica(CBG), Universidade Estadual de Santa Cruz (UESC), Rodovia Ilheus-Itabuna, km 16,Ilheus, 45662-900, Bahia, Brazil.\n",
      "AD  - Comissao Executiva do Plano da Lavoura Cacaueira (CEPLAC), Centro de Pesquisas doCacau (CEPEC), Secao de Fitossanidade (SEFIT), Laboratorio de FitopatologiaMolecular (FITOMOL), km 22 Rod. Ilheus Itabuna, Ilheus, 45600-970, Bahia, Brazil.\n",
      "FAU - Argolo, Caio Suzart\n",
      "AU  - Argolo CS\n",
      "AD  - Departamento de Ciencias Biologicas (DCB), Centro de Biotecnologia e Genetica(CBG), Universidade Estadual de Santa Cruz (UESC), Rodovia Ilheus-Itabuna, km 16,Ilheus, 45662-900, Bahia, Brazil.\n",
      "FAU - Carvalho, Mariana Rocha de\n",
      "AU  - Carvalho MR\n",
      "AD  - Departamento de Ciencias Biologicas (DCB), Centro de Biotecnologia e Genetica(CBG), Universidade Estadual de Santa Cruz (UESC), Rodovia Ilheus-Itabuna, km 16,Ilheus, 45662-900, Bahia, Brazil.\n",
      "AD  - Comissao Executiva do Plano da Lavoura Cacaueira (CEPLAC), Centro de Pesquisas doCacau (CEPEC), Secao de Fitossanidade (SEFIT), Laboratorio de FitopatologiaMolecular (FITOMOL), km 22 Rod. Ilheus Itabuna, Ilheus, 45600-970, Bahia, Brazil.\n",
      "FAU - Amaral, Daniel Oliveira Jordao do\n",
      "AU  - Amaral DOJD\n",
      "AD  - Departamento de Ciencias Biologicas (DCB), Centro de Biotecnologia e Genetica(CBG), Universidade Estadual de Santa Cruz (UESC), Rodovia Ilheus-Itabuna, km 16,Ilheus, 45662-900, Bahia, Brazil.\n",
      "FAU - Silva, Edson Mario de Andrade\n",
      "AU  - Silva EMA\n",
      "AD  - Departamento de Ciencias Biologicas (DCB), Centro de Biotecnologia e Genetica(CBG), Universidade Estadual de Santa Cruz (UESC), Rodovia Ilheus-Itabuna, km 16,Ilheus, 45662-900, Bahia, Brazil.\n",
      "FAU - Arevalo-Gardini, Enrique\n",
      "AU  - Arevalo-Gardini E\n",
      "AD  - Instituto de Cultivos Tropicales -ICT, Tarapoto, Peru.\n",
      "FAU - Hidalgo, Karina Solis\n",
      "AU  - Hidalgo KS\n",
      "AD  - Instituto Nacional de Investigaciones Agropecuarias - INIAP, Departamento deProteccion Vegetal, Quito, Ecuador.\n",
      "FAU - Franco, Gloria Regina\n",
      "AU  - Franco GR\n",
      "AD  - Departamento de Bioquimica e Imunologia, Universidade Federal de MinasGerais/Belo Horizonte, Minas Gerais, Brazil.\n",
      "FAU - Pirovani, Carlos Priminho\n",
      "AU  - Pirovani CP\n",
      "AD  - Departamento de Ciencias Biologicas (DCB), Centro de Biotecnologia e Genetica(CBG), Universidade Estadual de Santa Cruz (UESC), Rodovia Ilheus-Itabuna, km 16,Ilheus, 45662-900, Bahia, Brazil.\n",
      "FAU - Micheli, Fabienne\n",
      "AU  - Micheli F\n",
      "AD  - Departamento de Ciencias Biologicas (DCB), Centro de Biotecnologia e Genetica(CBG), Universidade Estadual de Santa Cruz (UESC), Rodovia Ilheus-Itabuna, km 16,Ilheus, 45662-900, Bahia, Brazil.\n",
      "AD  - CIRAD, UMR AGAP, F-34398, Montpellier, France.\n",
      "FAU - Gramacho, Karina Peres\n",
      "AU  - Gramacho KP\n",
      "AUID- ORCID: http://orcid.org/0000-0003-4037-1554\n",
      "AD  - Departamento de Ciencias Biologicas (DCB), Centro de Biotecnologia e Genetica(CBG), Universidade Estadual de Santa Cruz (UESC), Rodovia Ilheus-Itabuna, km 16,Ilheus, 45662-900, Bahia, Brazil. gramachokp@hotmail.com.\n",
      "AD  - Comissao Executiva do Plano da Lavoura Cacaueira (CEPLAC), Centro de Pesquisas doCacau (CEPEC), Secao de Fitossanidade (SEFIT), Laboratorio de FitopatologiaMolecular (FITOMOL), km 22 Rod. Ilheus Itabuna, Ilheus, 45600-970, Bahia, Brazil.gramachokp@hotmail.com.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20180703\n",
      "PL  - England\n",
      "TA  - BMC Genomics\n",
      "JT  - BMC genomics\n",
      "JID - 100965258\n",
      "RN  - 0 (DNA, Fungal)\n",
      "RN  - 0 (Fungal Proteins)\n",
      "SB  - IM\n",
      "MH  - Agaricales/classification/*genetics/isolation & purification\n",
      "MH  - Brazil\n",
      "MH  - Cacao/microbiology\n",
      "MH  - DNA, Fungal/chemistry/isolation & purification/metabolism\n",
      "MH  - Fungal Proteins/genetics\n",
      "MH  - *Genome, Fungal\n",
      "MH  - Phylogeny\n",
      "MH  - Plant Diseases/*microbiology\n",
      "MH  - Whole Genome Sequencing\n",
      "PMC - PMC6029071\n",
      "OTO - NOTNLM\n",
      "OT  - Frosty pod rot\n",
      "OT  - Pathogenicity factors\n",
      "OT  - Plant pathogens\n",
      "OT  - Theobroma cacao\n",
      "OT  - Witches' broom\n",
      "EDAT- 2018/07/05 06:00\n",
      "MHDA- 2018/12/19 06:00\n",
      "CRDT- 2018/07/05 06:00\n",
      "PHST- 2018/03/02 00:00 [received]\n",
      "PHST- 2018/06/18 00:00 [accepted]\n",
      "PHST- 2018/07/05 06:00 [entrez]\n",
      "PHST- 2018/07/05 06:00 [pubmed]\n",
      "PHST- 2018/12/19 06:00 [medline]\n",
      "AID - 10.1186/s12864-018-4875-7 [doi]\n",
      "AID - 10.1186/s12864-018-4875-7 [pii]\n",
      "PST - epublish\n",
      "SO  - BMC Genomics. 2018 Jul 3;19(1):509. doi: 10.1186/s12864-018-4875-7.\n",
      "\n",
      "PMID- 29714581\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20190416\n",
      "LR  - 20200103\n",
      "IS  - 2168-4804 (Electronic)\n",
      "IS  - 2168-4790 (Linking)\n",
      "VI  - 52\n",
      "IP  - 6\n",
      "DP  - 2018 Nov\n",
      "TI  - CRISPR-Cas9: A Precise Approach to Genome Engineering.\n",
      "PG  - 701-707\n",
      "LID - 10.1177/2168479018762798 [doi]\n",
      "AB  - In the last few decades, genomic manipulation has made significant progress as a result of the development of recombinant DNA technologies; however, more oftenthan not, these techniques have been costly and labor intensive. In contrast,recently developed next-generation sequencing (NGS) technologies have provided a cheaper, faster, and easier process to study genomics. In particular, an NGStechnique emerged from bacterial CRISPR-associated protein-9 nuclease (Cas9) as arevolutionary method to modify, regulate, or mark specific genomic sequences onvirtually any organism. A later adaptation of this bacterial defense mechanismthat successfully and permanently edits dysfunctional genes and corrects missing proteins has resulted in a new era for disease genetic engineering. Clinicaltrials using this technique are already being performed, and the applicability ofCRISPR-Cas9 techniques is actively being investigated using in vivo studies.However, the concept of genome correction poses great concerns from a regulatory perspective, especially in terms of security, so principles for the regulation ofthese methodologies are being established. We delved into CRISPR-Cas9 from itsnatural and ortholog origins to its engineered variants and behaviors to present its notable and diverse applications in the fields of biotechnology and humantherapeutics.\n",
      "FAU - Simon, Jorge E\n",
      "AU  - Simon JE\n",
      "AD  - 1 School of Biological Sciences and Engineering, Yachay Tech University, Urcuqui,Ecuador.\n",
      "FAU - Rodriguez, Angel S\n",
      "AU  - Rodriguez AS\n",
      "AD  - 1 School of Biological Sciences and Engineering, Yachay Tech University, Urcuqui,Ecuador.\n",
      "FAU - Santiago Vispo, Nelson\n",
      "AU  - Santiago Vispo N\n",
      "AUID- ORCID: 0000-0002-4081-3984\n",
      "AD  - 1 School of Biological Sciences and Engineering, Yachay Tech University, Urcuqui,Ecuador.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "DEP - 20180402\n",
      "PL  - Switzerland\n",
      "TA  - Ther Innov Regul Sci\n",
      "JT  - Therapeutic innovation & regulatory science\n",
      "JID - 101597411\n",
      "RN  - 0 (Bacterial Proteins)\n",
      "RN  - EC 3.1.- (CRISPR-Associated Protein 9)\n",
      "SB  - IM\n",
      "MH  - Bacterial Proteins/metabolism\n",
      "MH  - CRISPR-Associated Protein 9/*metabolism\n",
      "MH  - CRISPR-Cas Systems\n",
      "MH  - Clinical Trials as Topic/legislation & jurisprudence\n",
      "MH  - Genetic Engineering/*legislation & jurisprudence/methods\n",
      "MH  - Genome, Human\n",
      "MH  - High-Throughput Nucleotide Sequencing\n",
      "MH  - Humans\n",
      "MH  - Mutation\n",
      "MH  - Sequence Analysis, DNA\n",
      "OTO - NOTNLM\n",
      "OT  - *CRISPR-Cas9\n",
      "OT  - *NGS\n",
      "OT  - *bacterial adaptive defense system\n",
      "OT  - *clinical trials\n",
      "OT  - *genetic engineering\n",
      "EDAT- 2018/05/02 06:00\n",
      "MHDA- 2019/04/17 06:00\n",
      "CRDT- 2018/05/02 06:00\n",
      "PHST- 2018/05/02 06:00 [pubmed]\n",
      "PHST- 2019/04/17 06:00 [medline]\n",
      "PHST- 2018/05/02 06:00 [entrez]\n",
      "AID - 10.1177/2168479018762798 [doi]\n",
      "PST - ppublish\n",
      "SO  - Ther Innov Regul Sci. 2018 Nov;52(6):701-707. doi: 10.1177/2168479018762798. Epub2018 Apr 2.\n",
      "\n",
      "PMID- 29632024\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20190702\n",
      "LR  - 20210202\n",
      "IS  - 1528-0020 (Electronic)\n",
      "IS  - 0006-4971 (Linking)\n",
      "VI  - 132\n",
      "IP  - 1\n",
      "DP  - 2018 Jul 5\n",
      "TI  - Genetic and mechanistic diversity in pediatric hemophagocyticlymphohistiocytosis.\n",
      "PG  - 89-100\n",
      "LID - 10.1182/blood-2017-11-814244 [doi]\n",
      "AB  - The HLH-2004 criteria are used to diagnose hemophagocytic lymphohistiocytosis(HLH), yet concern exists for their misapplication, resulting in suboptimaltreatment of some patients. We sought to define the genomic spectrum andassociated outcomes of a diverse cohort of children who met the HLH-2004criteria. Genetic testing was performed clinically or through research-basedwhole-exome sequencing. Clinical metrics were analyzed with respect to genomicresults. Of 122 subjects enrolled over the course of 17 years, 101 subjectsreceived genetic testing. Biallelic familial HLH (fHLH) gene defects wereidentified in only 19 (19%) and correlated with presentation at younger than 1year of age (P < .0001). Digenic fHLH variants were observed but lackedstatistical support for disease association. In 28 (58%) of 48 subjects, researchwhole-exome sequencing analyses successfully identified likely molecularexplanations, including underlying primary immunodeficiency diseases,dysregulated immune activation and proliferation disorders, and potentially novelgenetic conditions. Two-thirds of patients identified by the HLH-2004 criteriahad underlying etiologies for HLH, including genetic defects, autoimmunity, andmalignancy. Overall survival was 45%, and increased mortality correlated with HLHtriggered by infection or malignancy (P < .05). Differences in survival did notcorrelate with genetic profile or extent of therapy. HLH should be conceptualizedas a phenotype of critical illness characterized by toxic activation of immunecells from different underlying mechanisms. In most patients with HLH, targetedsequencing of fHLH genes remains insufficient for identifying pathogenicmechanisms. Whole-exome sequencing, however, may identify specific therapeuticopportunities and affect hematopoietic stem cell transplantation options forthese patients.\n",
      "FAU - Chinn, Ivan K\n",
      "AU  - Chinn IK\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Immunology/Allergy/Rheumatology and.\n",
      "FAU - Eckstein, Olive S\n",
      "AU  - Eckstein OS\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Hematology/Oncology, Texas Children's Hospital CancerCenter, Houston, TX.\n",
      "FAU - Peckham-Gregory, Erin C\n",
      "AU  - Peckham-Gregory EC\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Division of Pediatric Hematology/Oncology, Texas Children's Hospital CancerCenter, Houston, TX.\n",
      "FAU - Goldberg, Baruch R\n",
      "AU  - Goldberg BR\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Immunology/Allergy/Rheumatology and.\n",
      "FAU - Forbes, Lisa R\n",
      "AU  - Forbes LR\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Immunology/Allergy/Rheumatology and.\n",
      "FAU - Nicholas, Sarah K\n",
      "AU  - Nicholas SK\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Immunology/Allergy/Rheumatology and.\n",
      "FAU - Mace, Emily M\n",
      "AU  - Mace EM\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Immunology/Allergy/Rheumatology and.\n",
      "FAU - Vogel, Tiphanie P\n",
      "AU  - Vogel TP\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Immunology/Allergy/Rheumatology and.\n",
      "FAU - Abhyankar, Harshal A\n",
      "AU  - Abhyankar HA\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Hematology/Oncology, Texas Children's Hospital CancerCenter, Houston, TX.\n",
      "FAU - Diaz, Maria I\n",
      "AU  - Diaz MI\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Hematology/Oncology, Texas Children's Hospital CancerCenter, Houston, TX.\n",
      "FAU - Heslop, Helen E\n",
      "AU  - Heslop HE\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Hematology/Oncology, Texas Children's Hospital CancerCenter, Houston, TX.\n",
      "AD  - Center for Cell and Gene Therapy, Baylor College of Medicine, Houston, TX.\n",
      "FAU - Krance, Robert A\n",
      "AU  - Krance RA\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Hematology/Oncology, Texas Children's Hospital CancerCenter, Houston, TX.\n",
      "AD  - Center for Cell and Gene Therapy, Baylor College of Medicine, Houston, TX.\n",
      "FAU - Martinez, Caridad A\n",
      "AU  - Martinez CA\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Hematology/Oncology, Texas Children's Hospital CancerCenter, Houston, TX.\n",
      "AD  - Center for Cell and Gene Therapy, Baylor College of Medicine, Houston, TX.\n",
      "FAU - Nguyen, Trung C\n",
      "AU  - Nguyen TC\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Critical Care Medicine, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Center for Translational Research on Inflammatory Diseases, Michael E. DeBakeyVeteran Affairs Medical Center, Houston, TX.\n",
      "FAU - Bashir, Dalia A\n",
      "AU  - Bashir DA\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Critical Care Medicine, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Center for Translational Research on Inflammatory Diseases, Michael E. DeBakeyVeteran Affairs Medical Center, Houston, TX.\n",
      "FAU - Goldman, Jordana R\n",
      "AU  - Goldman JR\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Critical Care Medicine, Texas Children's Hospital, Houston, TX.\n",
      "FAU - Stray-Pedersen, Asbjorg\n",
      "AU  - Stray-Pedersen A\n",
      "AD  - Norwegian National Unit for Newborn Screening, Department of Pediatric andAdolescent Medicine, Oslo University Hospital, Oslo, Norway.\n",
      "AD  - Institute of Clinical Medicine, University of Oslo, Oslo, Norway.\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics, Houston, TX.\n",
      "FAU - Pedroza, Luis A\n",
      "AU  - Pedroza LA\n",
      "AD  - Universidad San Francisco de Quito, Colegio de Ciencias de la Salud-Hospital delos Valles, Quito, Ecuador.\n",
      "FAU - Poli, M Cecilia\n",
      "AU  - Poli MC\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Universidad del Desarrollo, Clinica Alemana de Santiago, Santiago, Chile.\n",
      "FAU - Aldave-Becerra, Juan C\n",
      "AU  - Aldave-Becerra JC\n",
      "AD  - Division of Allergy and Immunology, Hospital Nacional Edgardo Rebagliati Martins,Lima, Peru.\n",
      "FAU - McGhee, Sean A\n",
      "AU  - McGhee SA\n",
      "AD  - Division of Immunology and Allergy, Department of Pediatrics, Stanford UniversitySchool of Medicine, Palo Alto, CA.\n",
      "FAU - Al-Herz, Waleed\n",
      "AU  - Al-Herz W\n",
      "AD  - Department of Pediatrics, Kuwait University, Kuwait City, Kuwait.\n",
      "FAU - Chamdin, Aghiad\n",
      "AU  - Chamdin A\n",
      "AD  - Department of Pediatrics and Human Development, Michigan State University,Lansing, MI; and.\n",
      "FAU - Coban-Akdemir, Zeynep H\n",
      "AU  - Coban-Akdemir ZH\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics, Houston, TX.\n",
      "AD  - Department of Molecular and Human Genetics and.\n",
      "FAU - Jhangiani, Shalini N\n",
      "AU  - Jhangiani SN\n",
      "AD  - Department of Molecular and Human Genetics and.\n",
      "AD  - Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX.\n",
      "FAU - Muzny, Donna M\n",
      "AU  - Muzny DM\n",
      "AD  - Department of Molecular and Human Genetics and.\n",
      "AD  - Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX.\n",
      "FAU - Cao, Tram N\n",
      "AU  - Cao TN\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Immunology/Allergy/Rheumatology and.\n",
      "FAU - Hong, Diana N\n",
      "AU  - Hong DN\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Immunology/Allergy/Rheumatology and.\n",
      "FAU - Gibbs, Richard A\n",
      "AU  - Gibbs RA\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics, Houston, TX.\n",
      "AD  - Department of Molecular and Human Genetics and.\n",
      "AD  - Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX.\n",
      "FAU - Lupski, James R\n",
      "AU  - Lupski JR\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics, Houston, TX.\n",
      "AD  - Department of Molecular and Human Genetics and.\n",
      "AD  - Human Genome Sequencing Center, Baylor College of Medicine, Houston, TX.\n",
      "FAU - Orange, Jordan S\n",
      "AU  - Orange JS\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Immunology/Allergy/Rheumatology and.\n",
      "FAU - McClain, Kenneth L\n",
      "AU  - McClain KL\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Hematology/Oncology, Texas Children's Hospital CancerCenter, Houston, TX.\n",
      "FAU - Allen, Carl E\n",
      "AU  - Allen CE\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n",
      "AD  - Department of Pediatrics, Texas Children's Hospital, Houston, TX.\n",
      "AD  - Division of Pediatric Hematology/Oncology, Texas Children's Hospital CancerCenter, Houston, TX.\n",
      "LA  - eng\n",
      "GR  - R01 AI120989/AI/NIAID NIH HHS/United States\n",
      "GR  - R25 CA160078/CA/NCI NIH HHS/United States\n",
      "GR  - UM1 HG006542/HG/NHGRI NIH HHS/United States\n",
      "GR  - R01 AI067946/AI/NIAID NIH HHS/United States\n",
      "GR  - P50 CA126752/CA/NCI NIH HHS/United States\n",
      "GR  - R01 NS058529/NS/NINDS NIH HHS/United States\n",
      "PT  - Clinical Trial\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20180409\n",
      "PL  - United States\n",
      "TA  - Blood\n",
      "JT  - Blood\n",
      "JID - 7603509\n",
      "SB  - IM\n",
      "CIN - Blood. 2018 Jul 5;132(1):5-7. PMID: 29976779\n",
      "MH  - Adolescent\n",
      "MH  - Child\n",
      "MH  - Child, Preschool\n",
      "MH  - Cohort Studies\n",
      "MH  - Female\n",
      "MH  - *Genetic Testing\n",
      "MH  - *Genome, Human\n",
      "MH  - Genome-Wide Association Study\n",
      "MH  - *High-Throughput Nucleotide Sequencing\n",
      "MH  - Humans\n",
      "MH  - Infant\n",
      "MH  - Infant, Newborn\n",
      "MH  - Lymphohistiocytosis, Hemophagocytic/*genetics/pathology/therapy\n",
      "MH  - Male\n",
      "MH  - Multifactorial Inheritance\n",
      "PMC - PMC6034641\n",
      "EDAT- 2018/04/11 06:00\n",
      "MHDA- 2019/07/03 06:00\n",
      "CRDT- 2018/04/11 06:00\n",
      "PHST- 2017/11/06 00:00 [received]\n",
      "PHST- 2018/04/03 00:00 [accepted]\n",
      "PHST- 2018/04/11 06:00 [pubmed]\n",
      "PHST- 2019/07/03 06:00 [medline]\n",
      "PHST- 2018/04/11 06:00 [entrez]\n",
      "AID - S0006-4971(20)32106-6 [pii]\n",
      "AID - 10.1182/blood-2017-11-814244 [doi]\n",
      "PST - ppublish\n",
      "SO  - Blood. 2018 Jul 5;132(1):89-100. doi: 10.1182/blood-2017-11-814244. Epub 2018 Apr9.\n",
      "\n",
      "PMID- 29617771\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20181211\n",
      "LR  - 20210109\n",
      "IS  - 2047-217X (Electronic)\n",
      "IS  - 2047-217X (Linking)\n",
      "VI  - 7\n",
      "IP  - 4\n",
      "DP  - 2018 Apr 1\n",
      "TI  - Real-time DNA barcoding in a rainforest using nanopore sequencing: opportunities for rapid biodiversity assessments and local capacity building.\n",
      "LID - 10.1093/gigascience/giy033 [doi]\n",
      "AB  - Background: Advancements in portable scientific instruments provide promisingavenues to expedite field work in order to understand the diverse array oforganisms that inhabit our planet. Here, we tested the feasibility for in situmolecular analyses of endemic fauna using a portable laboratory fitting within a single backpack in one of the world's most imperiled biodiversity hotspots, theEcuadorian Choco rainforest. We used portable equipment, including the MinIONnanopore sequencer (Oxford Nanopore Technologies) and the miniPCR (miniPCR), toperform DNA extraction, polymerase chain reaction amplification, and real-timeDNA barcoding of reptile specimens in the field. Findings: We demonstrate thatnanopore sequencing can be implemented in a remote tropical forest to quickly andaccurately identify species using DNA barcoding, as we generated consensussequences for species resolution with an accuracy of >99% in less than 24 hoursafter collecting specimens. The flexibility of our mobile laboratory furtherallowed us to generate sequence information at the Universidad TecnologicaIndoamerica in Quito for rare, endangered, and undescribed species. This includesthe recently rediscovered Jambato toad, which was thought to be extinct for 28years. Sequences generated on the MinION required as few as 30 reads to achievehigh accuracy relative to Sanger sequencing, and with further multiplexing ofsamples, nanopore sequencing can become a cost-effective approach for rapid andportable DNA barcoding. Conclusions: Overall, we establish how mobilelaboratories and nanopore sequencing can help to accelerate speciesidentification in remote areas to aid in conservation efforts and be applied toresearch facilities in developing countries. This opens up possibilities forbiodiversity studies by promoting local research capacity building, teachingnonspecialists and students about the environment, tackling wildlife crime, andpromoting conservation via research-focused ecotourism.\n",
      "FAU - Pomerantz, Aaron\n",
      "AU  - Pomerantz A\n",
      "AD  - Department of Integrative Biology, University of California, Berkeley, CA, USA.\n",
      "FAU - Penafiel, Nicolas\n",
      "AU  - Penafiel N\n",
      "AD  - Centro de Investigacion de la Biodiversidad y Cambio Climatico (BioCamb) eIngenieria en Biodiversidad y Recursos Geneticos, Facultad de Ciencias de MedioAmbiente, Universidad Tecnologica Indoamerica, Machala y Sabanilla, Quito,Ecuador.\n",
      "FAU - Arteaga, Alejandro\n",
      "AU  - Arteaga A\n",
      "AD  - Richard Gilder Graduate School, American Museum of Natural History, New York,USA.\n",
      "AD  - Department of Herpetology, American Museum of Natural History, New York, USA.\n",
      "AD  - Tropical Herping, Quito, Ecuador.\n",
      "FAU - Bustamante, Lucas\n",
      "AU  - Bustamante L\n",
      "AD  - Tropical Herping, Quito, Ecuador.\n",
      "FAU - Pichardo, Frank\n",
      "AU  - Pichardo F\n",
      "AD  - Tropical Herping, Quito, Ecuador.\n",
      "FAU - Coloma, Luis A\n",
      "AU  - Coloma LA\n",
      "AD  - Centro Jambatu de Investigacion y Conservacion de Anfibios, Fundacion Otonga,Quito, Ecuador.\n",
      "FAU - Barrio-Amoros, Cesar L\n",
      "AU  - Barrio-Amoros CL\n",
      "AD  - Doc Frog Expeditions, Uvita, Costa Rica.\n",
      "FAU - Salazar-Valenzuela, David\n",
      "AU  - Salazar-Valenzuela D\n",
      "AD  - Centro de Investigacion de la Biodiversidad y Cambio Climatico (BioCamb) eIngenieria en Biodiversidad y Recursos Geneticos, Facultad de Ciencias de MedioAmbiente, Universidad Tecnologica Indoamerica, Machala y Sabanilla, Quito,Ecuador.\n",
      "FAU - Prost, Stefan\n",
      "AU  - Prost S\n",
      "AD  - Department of Integrative Biology, University of California, Berkeley, CA, USA.\n",
      "AD  - Program for Conservation Genomics, Department of Biology, Stanford University,Stanford, CA, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PL  - United States\n",
      "TA  - Gigascience\n",
      "JT  - GigaScience\n",
      "JID - 101596872\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Biodiversity\n",
      "MH  - Ecuador\n",
      "MH  - Nanopores\n",
      "MH  - Rainforest\n",
      "MH  - Reptiles/*genetics\n",
      "MH  - Sequence Analysis, DNA/*methods\n",
      "PMC - PMC5905381\n",
      "EDAT- 2018/04/05 06:00\n",
      "MHDA- 2018/12/12 06:00\n",
      "CRDT- 2018/04/05 06:00\n",
      "PHST- 2017/12/17 00:00 [received]\n",
      "PHST- 2018/03/22 00:00 [accepted]\n",
      "PHST- 2018/04/05 06:00 [pubmed]\n",
      "PHST- 2018/12/12 06:00 [medline]\n",
      "PHST- 2018/04/05 06:00 [entrez]\n",
      "AID - 4958980 [pii]\n",
      "AID - 10.1093/gigascience/giy033 [doi]\n",
      "PST - ppublish\n",
      "SO  - Gigascience. 2018 Apr 1;7(4). pii: 4958980. doi: 10.1093/gigascience/giy033.\n",
      "\n",
      "PMID- 29580753\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180920\n",
      "LR  - 20180920\n",
      "IS  - 1873-2232 (Electronic)\n",
      "IS  - 0378-4320 (Linking)\n",
      "VI  - 192\n",
      "DP  - 2018 May\n",
      "TI  - Sephadex filtration as successful alternative to density-gradient centrifugation procedures for ram sperm selection with improved kinetics.\n",
      "PG  - 261-270\n",
      "LID - S0378-4320(18)30136-2 [pii]\n",
      "LID - 10.1016/j.anireprosci.2018.03.022 [doi]\n",
      "AB  - Density-gradients centrifugation (DGC) and filtration columns (FC) are used toseparate deformed or dead sperm, debris, and other cells that may negativelyaffect the fertilizing capacity of sperm in fresh, chilled and frozen/thawedsemen. The present study was conducted to evaluate the suitability of DGC(BoviPure(R), Percoll(R) and Accudenz(R)) and FC (Sephadex G-15(R)) spermselection procedures for fresh-extended and cold-stored ram semen by assessmentof post-treatment sperm quality variables. Twenty normospermic ejaculates fromten adult Merino rams were used. Sperm concentration of recovered cells wasgreater (P<0.001) after BoviPure treatment than other procedures in both freshand cold semen. With the Sephadex method, there were more desirable values thanwith use of DGC procedures in several sperm motility variables measured by using the CASA system. In non-refrigerated semen samples, the percentage of progressivesperm motility (%PSM) after Sephadex filtration was greater (P<0.05) than afterBoviPure treatment; the straightline velocity (VSL) value after Sephadexfiltration was greater (P<0.01) than after Accudenz treatment; the amplitude oflateral head displacement (ALH) after Sephadex and Accudenz treatment was lessthan non-filtered semen (P<0.001) and after Percoll (P<0.01) and BoviPure(P<0.05) treatments. In cold-stored semen samples, the %PSM after Sephadexfiltration was greater than non-filtered (P<0.05) semen and after BoviPure(P<0.05), Percoll (P<0.05) and Accudenz (P<0.001) treatments. It is concludedthat Sephadex column filtration can be used to select ram sperm innon-refrigerated and cooled semen, because percentage progressively motile sperm and some other sperm motility characteristics are greater with use of thistechniques as compared with use of DGC methods.\n",
      "CI  - Copyright (c) 2018 Elsevier B.V. All rights reserved.\n",
      "FAU - Galarza, D A\n",
      "AU  - Galarza DA\n",
      "AD  - Departamento de Reproduccion Animal, INIA, 28040, Madrid, Spain; Facultad deCiencias Agropecuarias, Universidad de Cuenca, EC010220, Cuenca, Ecuador.Electronic address: dgalarza@ucm.es.\n",
      "FAU - Lopez-Sebastian, A\n",
      "AU  - Lopez-Sebastian A\n",
      "AD  - Departamento de Reproduccion Animal, INIA, 28040, Madrid, Spain.\n",
      "FAU - Woelders, H\n",
      "AU  - Woelders H\n",
      "AD  - Wageningen University & Research, Animal Breeding and Genomics, P.O. BOX 338,6700 AH, Wageningen, The Netherlands.\n",
      "FAU - Blesbois, E\n",
      "AU  - Blesbois E\n",
      "AD  - UMR-PRC INRA-CNRS-IFCE- Francois Rabelais University, 37380, Nouzilly, France.\n",
      "FAU - Santiago-Moreno, J\n",
      "AU  - Santiago-Moreno J\n",
      "AD  - Departamento de Reproduccion Animal, INIA, 28040, Madrid, Spain.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20180320\n",
      "PL  - Netherlands\n",
      "TA  - Anim Reprod Sci\n",
      "JT  - Animal reproduction science\n",
      "JID - 7807205\n",
      "RN  - 0 (Dextrans)\n",
      "RN  - 9014-76-0 (sephadex)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Centrifugation, Density Gradient/*veterinary\n",
      "MH  - Cold Temperature\n",
      "MH  - *Dextrans\n",
      "MH  - Male\n",
      "MH  - Sheep/*physiology\n",
      "MH  - Specimen Handling\n",
      "MH  - Sperm Motility/*physiology\n",
      "MH  - Spermatozoa/*physiology\n",
      "OTO - NOTNLM\n",
      "OT  - Centrifugation\n",
      "OT  - Filtration\n",
      "OT  - Gradients\n",
      "OT  - Selection\n",
      "OT  - Sperm\n",
      "EDAT- 2018/03/28 06:00\n",
      "MHDA- 2018/09/21 06:00\n",
      "CRDT- 2018/03/28 06:00\n",
      "PHST- 2018/02/07 00:00 [received]\n",
      "PHST- 2018/03/12 00:00 [revised]\n",
      "PHST- 2018/03/19 00:00 [accepted]\n",
      "PHST- 2018/03/28 06:00 [pubmed]\n",
      "PHST- 2018/09/21 06:00 [medline]\n",
      "PHST- 2018/03/28 06:00 [entrez]\n",
      "AID - S0378-4320(18)30136-2 [pii]\n",
      "AID - 10.1016/j.anireprosci.2018.03.022 [doi]\n",
      "PST - ppublish\n",
      "SO  - Anim Reprod Sci. 2018 May;192:261-270. doi: 10.1016/j.anireprosci.2018.03.022.Epub 2018 Mar 20.\n",
      "\n",
      "PMID- 29477599\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180829\n",
      "LR  - 20180829\n",
      "IS  - 1876-4347 (Electronic)\n",
      "IS  - 1871-6784 (Linking)\n",
      "VI  - 42\n",
      "DP  - 2018 May 25\n",
      "TI  - Rosa hybrida orcinol O-methyl transferase-mediated production of pterostilbene inmetabolically engineered grapevine cell cultures.\n",
      "PG  - 62-70\n",
      "LID - S1871-6784(17)30675-1 [pii]\n",
      "LID - 10.1016/j.nbt.2018.02.011 [doi]\n",
      "AB  - Stilbenes are naturally scarce high-added-value plant compounds withchemopreventive, pharmacological and cosmetic properties. Bioproductionstrategies include engineering the metabolisms of bacterial, fungal and plantcell systems. Strikingly, one of the most effective strategies consists in theelicitation of wild grapevine cell cultures, which leads to vast stilbeneresveratrol accumulation in the extracellular medium. The combination of bothcell culture elicitation and metabolic engineering strategies to produceresveratrol analogs proved more efficient for the hydroxylated derivativepiceatannol than for the dimethylated derivative pterostilbene, for which humanhydroxylase HsCYP1B1- and grapevine O-methyltransferase VvROMT-transformed cellcultures were respectively used. Rose orcinol O-methyltransferase (OOMT) displaysenzymatic properties, which makes it an appealing candidate to substitute VvROMT in the combined strategy to enhance the pterostilbene production level byengineered grapevine cells upon elicitation. Here we cloned a Rosa hybrida OOMTgene, and created a genetic construction suitable for Agrobacterium-mediatedplant transformation. OOMT's ability to catalyze the conversion of resveratrolinto pterostilbene was first assessed in vitro using protein extracts ofagroinfiltrated N. benthamiana leaves and transformed grapevine callus. Thegrapevine cell cultures transformed with RhOOMT produced about 16mg/L culture of pterostilbene and reached an extracellular distribution of up to 34% of totalproduction at the best, which is by far the highest production reported to datein a plant system. A bonus large resveratrol production of ca. 1500-3000mg/L was simultaneously obtained. Our results demonstrate a viable successful metabolicengineering strategy to produce pterostilbene, a resveratrol analog with enhancedpharmacological properties.\n",
      "CI  - Copyright (c) 2018 Elsevier B.V. All rights reserved.\n",
      "FAU - Martinez-Marquez, Ascension\n",
      "AU  - Martinez-Marquez A\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science and Multidisciplinary Institut for EnvironmentStudies \"Ramon Margalef\", University of Alicante, Alicante, Spain.\n",
      "FAU - Morante-Carriel, Jaime A\n",
      "AU  - Morante-Carriel JA\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science and Multidisciplinary Institut for EnvironmentStudies \"Ramon Margalef\", University of Alicante, Alicante, Spain; Biotechnology and Molecular Biology Group, Quevedo State Technical University, Quevedo,Ecuador.\n",
      "FAU - Palazon, Javier\n",
      "AU  - Palazon J\n",
      "AD  - Laboratory of Plant Physiology, Faculty of Pharmacy, University of Barcelona,Av.Joan XXIII sn, E-08028 Barcelona, Spain.\n",
      "FAU - Bru-Martinez, Roque\n",
      "AU  - Bru-Martinez R\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science and Multidisciplinary Institut for EnvironmentStudies \"Ramon Margalef\", University of Alicante, Alicante, Spain; Institute for Healthcare and Biomedical Research of Alicante ISABIAL-FISABIO, E-03010 Alicante,Spain. Electronic address: roque.bru@ua.es.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - Netherlands\n",
      "TA  - N Biotechnol\n",
      "JT  - New biotechnology\n",
      "JID - 101465345\n",
      "RN  - 0 (Plant Proteins)\n",
      "RN  - 0 (Stilbenes)\n",
      "RN  - 26R60S6A5I (pterostilbene)\n",
      "RN  - EC 2.1.1.- (Methyltransferases)\n",
      "SB  - IM\n",
      "MH  - *Metabolic Engineering\n",
      "MH  - *Methyltransferases/genetics/metabolism\n",
      "MH  - Plant Cells/*enzymology\n",
      "MH  - *Plant Proteins/genetics/metabolism\n",
      "MH  - Rosa/enzymology/*genetics\n",
      "MH  - Stilbenes/*metabolism\n",
      "MH  - Vitis/*cytology/enzymology/genetics\n",
      "OTO - NOTNLM\n",
      "OT  - Grapevine cell culture\n",
      "OT  - Metabolic engineering\n",
      "OT  - Orcinol O-methyltransferase\n",
      "OT  - Pterostilbene\n",
      "OT  - Resveratrol\n",
      "OT  - Vitis vinifera\n",
      "EDAT- 2018/02/27 06:00\n",
      "MHDA- 2018/08/30 06:00\n",
      "CRDT- 2018/02/26 06:00\n",
      "PHST- 2017/12/04 00:00 [received]\n",
      "PHST- 2018/02/19 00:00 [revised]\n",
      "PHST- 2018/02/20 00:00 [accepted]\n",
      "PHST- 2018/02/27 06:00 [pubmed]\n",
      "PHST- 2018/08/30 06:00 [medline]\n",
      "PHST- 2018/02/26 06:00 [entrez]\n",
      "AID - S1871-6784(17)30675-1 [pii]\n",
      "AID - 10.1016/j.nbt.2018.02.011 [doi]\n",
      "PST - ppublish\n",
      "SO  - N Biotechnol. 2018 May 25;42:62-70. doi: 10.1016/j.nbt.2018.02.011.\n",
      "\n",
      "PMID- 29475888\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20191120\n",
      "IS  - 1098-6596 (Electronic)\n",
      "IS  - 0066-4804 (Linking)\n",
      "VI  - 62\n",
      "IP  - 3\n",
      "DP  - 2018 Mar\n",
      "TI  - Erratum for Arias et al., \"A Prospective Cohort Multicenter Study of MolecularEpidemiology and Phylogenomics of Staphylococcus aureus Bacteremia in Nine Latin American Countries\".\n",
      "LID - e00095-18 [pii]\n",
      "LID - 10.1128/AAC.00095-18 [doi]\n",
      "FAU - Arias, Cesar A\n",
      "AU  - Arias CA\n",
      "AD  - Center for Antimicrobial Resistance and Microbial Genomics and Division ofInfectious Diseases, University of Texas McGovern Medical School at Houston,Houston, Texas, USA.\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Reyes, Jinnethe\n",
      "AU  - Reyes J\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Carvajal, Lina Paola\n",
      "AU  - Carvajal LP\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Rincon, Sandra\n",
      "AU  - Rincon S\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Diaz, Lorena\n",
      "AU  - Diaz L\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Panesso, Diana\n",
      "AU  - Panesso D\n",
      "AD  - Center for Antimicrobial Resistance and Microbial Genomics and Division ofInfectious Diseases, University of Texas McGovern Medical School at Houston,Houston, Texas, USA.\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Ibarra, Gabriel\n",
      "AU  - Ibarra G\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Rios, Rafael\n",
      "AU  - Rios R\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Munita, Jose M\n",
      "AU  - Munita JM\n",
      "AD  - Grupo de Genomica Microbiana, Instituto de Ciencias e Innovacion en Medicina,Clinica Alemana, Universidad del Desarrollo, Santiago, Chile.\n",
      "FAU - Salles, Mauro J\n",
      "AU  - Salles MJ\n",
      "AD  - Division of Infectious Diseases, Department of Internal Medicine, Santa Casa deSao Paulo School of Medicine, Sao Paulo, Brazil.\n",
      "FAU - Alvarez-Moreno, Carlos\n",
      "AU  - Alvarez-Moreno C\n",
      "AD  - Unidad Infectologia, Departamento de Medicina Interna, Facultad de Medicina,Universidad Nacional de Colombia, Clinica Universitaria Colombia, Colsanitas,Bogota, Colombia.\n",
      "FAU - Labarca, Jaime\n",
      "AU  - Labarca J\n",
      "AD  - Department of Infectious Diseases, School of Medicine, Pontificia UniversidadCatolica de Chile, Santiago, Chile.\n",
      "FAU - Garcia, Coralith\n",
      "AU  - Garcia C\n",
      "AD  - Hospital Cayetano Heredia, Instituto de Medicina Tropical Alexander von Humboldt,Universidad Peruana Cayetano Heredia, Lima, Peru.\n",
      "FAU - Luna, Carlos M\n",
      "AU  - Luna CM\n",
      "AD  - Pulmonary Division, Department of Medicine, Jose de San Martin Hospital,University of Buenos Aires, Buenos Aires, Argentina.\n",
      "FAU - Mejia-Villatoro, Carlos\n",
      "AU  - Mejia-Villatoro C\n",
      "AD  - Clinica de Enfermedades Infecciosas, Hospital Roosevelt, Guatemala City,Guatemala.\n",
      "FAU - Zurita, Jeannete\n",
      "AU  - Zurita J\n",
      "AD  - Hospital Vozandes, Facultad de Medicina, Pontificia Universidad Catolica delEcuador, Quito, Ecuador.\n",
      "FAU - Guzman-Blanco, Manuel\n",
      "AU  - Guzman-Blanco M\n",
      "AD  - Centro Medico de Caracas, Caracas, Venezuela.\n",
      "FAU - Rodriguez-Noriega, Eduardo\n",
      "AU  - Rodriguez-Noriega E\n",
      "AD  - Hospital Civil de Guadalajara, Fray Antonio Alcalde, and Instituto de PatologiaInfecciosa y Experimental, Centro Universitario de Ciencias de la Salud,Universidad de Guadalajara, Guadalajara, Mexico.\n",
      "FAU - Narechania, Apurva\n",
      "AU  - Narechania A\n",
      "AD  - Sackler Institute for Comparative Genomics, American Museum of Natural History,New York, New York, USA.\n",
      "FAU - Rojas, Laura J\n",
      "AU  - Rojas LJ\n",
      "AD  - Department of Molecular Biology and Microbiology, Case Western ReserveUniversity, and Research Service, Louis Stokes Cleveland Department of VeteransAffairs, Cleveland, Ohio, USA.\n",
      "FAU - Planet, Paul J\n",
      "AU  - Planet PJ\n",
      "AD  - Sackler Institute for Comparative Genomics, American Museum of Natural History,New York, New York, USA.\n",
      "AD  - Department of Pediatrics, Division of Pediatric Infectious Diseases, ColumbiaUniversity, College of Physicians and Surgeons, New York, New York, USA.\n",
      "AD  - Department of Pediatrics, Division of Infectious Diseases, The Children'sHospital of Philadelphia, Philadelphia, Pennsylvania, USA.\n",
      "FAU - Weinstock, George M\n",
      "AU  - Weinstock GM\n",
      "AD  - The Jackson Laboratory for Genomic Medicine, Farmington, Connecticut, USA.\n",
      "FAU - Gotuzzo, Eduardo\n",
      "AU  - Gotuzzo E\n",
      "AD  - Hospital Cayetano Heredia, Instituto de Medicina Tropical Alexander von Humboldt,Universidad Peruana Cayetano Heredia, Lima, Peru.\n",
      "FAU - Seas, Carlos\n",
      "AU  - Seas C\n",
      "AD  - Hospital Cayetano Heredia, Instituto de Medicina Tropical Alexander von Humboldt,Universidad Peruana Cayetano Heredia, Lima, Peru.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Published Erratum\n",
      "DEP - 20180223\n",
      "PL  - United States\n",
      "TA  - Antimicrob Agents Chemother\n",
      "JT  - Antimicrobial agents and chemotherapy\n",
      "JID - 0315061\n",
      "EFR - Antimicrob Agents Chemother. 2017 Sep 22;61(10 ):. PMID: 28760895\n",
      "PMC - PMC5826103\n",
      "EDAT- 2018/02/25 06:00\n",
      "MHDA- 2018/02/25 06:01\n",
      "CRDT- 2018/02/25 06:00\n",
      "PHST- 2018/02/25 06:00 [entrez]\n",
      "PHST- 2018/02/25 06:00 [pubmed]\n",
      "PHST- 2018/02/25 06:01 [medline]\n",
      "AID - 62/3/e00095-18 [pii]\n",
      "AID - 10.1128/AAC.00095-18 [doi]\n",
      "PST - epublish\n",
      "SO  - Antimicrob Agents Chemother. 2018 Feb 23;62(3). pii: 62/3/e00095-18. doi:10.1128/AAC.00095-18. Print 2018 Mar.\n",
      "\n",
      "PMID- 29321624\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20190603\n",
      "LR  - 20190603\n",
      "IS  - 1365-2540 (Electronic)\n",
      "IS  - 0018-067X (Linking)\n",
      "VI  - 120\n",
      "IP  - 5\n",
      "DP  - 2018 May\n",
      "TI  - Strong trans-Pacific break and local conservation units in the Galapagos shark(Carcharhinus galapagensis) revealed by genome-wide cytonuclear markers.\n",
      "PG  - 407-421\n",
      "LID - 10.1038/s41437-017-0025-2 [doi]\n",
      "AB  - The application of genome-wide cytonuclear molecular data to identify management and adaptive units at various spatio-temporal levels is particularly importantfor overharvested large predatory organisms, often characterized by smaller,localized populations. Despite being \"near threatened\", current understanding of habitat use and population structure of Carcharhinus galapagensis is limited tospecific areas within its distribution. We evaluated population structure andconnectivity across the Pacific Ocean using genome-wide single-nucleotidepolymorphisms (~7200 SNPs) and mitochondrial control region sequences (945 bp)for 229 individuals. Neutral SNPs defined at least two genetically discretegeographic groups: an East Tropical Pacific (Mexico, east and west GalapagosIslands), and another central-west Pacific (Lord Howe Island, Middleton Reef,Norfolk Island, Elizabeth Reef, Kermadec, Hawaii and Southern Africa). Morefine-grade population structure was suggested using outlier SNPs: west Pacific,Hawaii, Mexico, and Galapagos. Consistently, mtDNA pairwise PhiST defined threeregional stocks: east, central and west Pacific. Compared to neutral SNPs (FST = 0.023-0.035), mtDNA exhibited more divergence (PhiST = 0.258-0.539) and highoverall genetic diversity (h = 0.794 +/- 0.014; pi = 0.004 +/- 0.000), consistentwith the longstanding eastern Pacific barrier between the east and central-westPacific. Hawaiian and Southern African populations group within the west Pacific cluster. Effective population sizes were moderate/high for east/west populations (738 and 3421, respectively). Insights into the biology, connectivity, geneticdiversity, and population demographics informs for improved conservation of this species, by delineating three to four conservation units across their Pacificdistribution. Implementing such conservation management may be challenging, butis necessary to achieve long-term population resilience at basin and regionalscales.\n",
      "FAU - Pazmino, Diana A\n",
      "AU  - Pazmino DA\n",
      "AUID- ORCID: http://orcid.org/0000-0001-7082-3254\n",
      "AD  - Centre for Sustainable Tropical Fisheries and Aquaculture, College of Science andEngineering, James Cook University, Townsville, QLD, Australia.diana.pazminojaramillo@my.jcu.edu.au.\n",
      "AD  - Comparative Genomics Centre, College of Science and Engineering, James CookUniversity, Townsville, QLD, Australia. diana.pazminojaramillo@my.jcu.edu.au.\n",
      "AD  - Universidad San Francisco de Quito - Galapagos Science Center, Quito, Ecuador.diana.pazminojaramillo@my.jcu.edu.au.\n",
      "FAU - Maes, Gregory E\n",
      "AU  - Maes GE\n",
      "AD  - Centre for Sustainable Tropical Fisheries and Aquaculture, College of Science andEngineering, James Cook University, Townsville, QLD, Australia.\n",
      "AD  - Comparative Genomics Centre, College of Science and Engineering, James CookUniversity, Townsville, QLD, Australia.\n",
      "AD  - Laboratory of Biodiversity and Evolutionary Genomics, University of Leuven,Leuven, Belgium.\n",
      "AD  - Laboratory for Cytogenetics and Genome Research, Center for Human Genetics,Genomics Core, KU Leuven, Leuven, Belgium.\n",
      "FAU - Green, Madeline E\n",
      "AU  - Green ME\n",
      "AD  - Institute for Marine and Antarctic Studies, University of Tasmania, Private Bag49, Hobart, TAS, Australia.\n",
      "AD  - CSIRO Oceans & Atmosphere, Castray Esplanade, Battery Point, Hobart, TAS,Australia.\n",
      "FAU - Simpfendorfer, Colin A\n",
      "AU  - Simpfendorfer CA\n",
      "AD  - Centre for Sustainable Tropical Fisheries and Aquaculture, College of Science andEngineering, James Cook University, Townsville, QLD, Australia.\n",
      "FAU - Hoyos-Padilla, E Mauricio\n",
      "AU  - Hoyos-Padilla EM\n",
      "AD  - Pelagios-Kakunja A.C. Sinaloa 1540, La Paz, Baja California Sur, Mexico.\n",
      "FAU - Duffy, Clinton J A\n",
      "AU  - Duffy CJA\n",
      "AD  - Auckland War Memorial Museum, The Domain, Auckland, New Zealand.\n",
      "AD  - Department of Conservation, Private Bag 68908, Newton, Auckland, New Zealand.\n",
      "FAU - Meyer, Carl G\n",
      "AU  - Meyer CG\n",
      "AD  - Hawaii Institute of Marine Biology, University of Hawaii at Manoa, CoconutIsland, Kaneohe, HI, USA.\n",
      "FAU - Kerwath, Sven E\n",
      "AU  - Kerwath SE\n",
      "AD  - Department of Biological Sciences, University of Cape Town, Private Bag x 3,Rondebosch, South Africa.\n",
      "AD  - Department of Agriculture, Forestry and Fisheries: Fisheries Branch, Private Bag x 2, Vlaeberg, Cape Town, South Africa.\n",
      "FAU - Salinas-de-Leon, Pelayo\n",
      "AU  - Salinas-de-Leon P\n",
      "AD  - Department of Marine Sciences, Charles Darwin Research Station. Av Charles Darwins/n, Puerto Ayora, Galapagos Islands, Santa Cruz, Ecuador.\n",
      "AD  - Pristine Seas, National Geographic Society, Washington, D. C., USA.\n",
      "FAU - van Herwerden, Lynne\n",
      "AU  - van Herwerden L\n",
      "AD  - Centre for Sustainable Tropical Fisheries and Aquaculture, College of Science andEngineering, James Cook University, Townsville, QLD, Australia.\n",
      "AD  - Comparative Genomics Centre, College of Science and Engineering, James CookUniversity, Townsville, QLD, Australia.\n",
      "LA  - eng\n",
      "SI  - Dryad/10.5061/dryad.h2kh3\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20180111\n",
      "PL  - England\n",
      "TA  - Heredity (Edinb)\n",
      "JT  - Heredity\n",
      "JID - 0373007\n",
      "RN  - 0 (DNA, Mitochondrial)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Conservation of Natural Resources\n",
      "MH  - DNA, Mitochondrial/genetics\n",
      "MH  - Ecuador\n",
      "MH  - Female\n",
      "MH  - *Genetic Variation\n",
      "MH  - *Genetics, Population\n",
      "MH  - Genotype\n",
      "MH  - Male\n",
      "MH  - Pacific Ocean\n",
      "MH  - Phylogeny\n",
      "MH  - Polymorphism, Single Nucleotide/genetics\n",
      "MH  - Sharks/*genetics\n",
      "PMC - PMC5889387\n",
      "EDAT- 2018/01/13 06:00\n",
      "MHDA- 2019/06/04 06:00\n",
      "CRDT- 2018/01/12 06:00\n",
      "PHST- 2017/06/19 00:00 [received]\n",
      "PHST- 2017/10/17 00:00 [accepted]\n",
      "PHST- 2017/10/07 00:00 [revised]\n",
      "PHST- 2018/01/13 06:00 [pubmed]\n",
      "PHST- 2019/06/04 06:00 [medline]\n",
      "PHST- 2018/01/12 06:00 [entrez]\n",
      "AID - 10.1038/s41437-017-0025-2 [doi]\n",
      "AID - 10.1038/s41437-017-0025-2 [pii]\n",
      "PST - ppublish\n",
      "SO  - Heredity (Edinb). 2018 May;120(5):407-421. doi: 10.1038/s41437-017-0025-2. Epub2018 Jan 11.\n",
      "\n",
      "PMID- 29188006\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20210109\n",
      "IS  - 2045-7758 (Print)\n",
      "IS  - 2045-7758 (Linking)\n",
      "VI  - 7\n",
      "IP  - 22\n",
      "DP  - 2017 Nov\n",
      "TI  - Radiation of the polymorphic Little Devil poison frog (Oophaga sylvatica) inEcuador.\n",
      "PG  - 9750-9762\n",
      "LID - 10.1002/ece3.3503 [doi]\n",
      "AB  - Some South American poison frogs (Dendrobatidae) are chemically defended and use bright aposematic colors to warn potential predators of their unpalatability.Aposematic signals are often frequency-dependent where individuals deviating froma local model are at a higher risk of predation. However, extreme diversity inthe aposematic signal has been documented in poison frogs, especially in Oophaga.Here, we explore the phylogeographic pattern among color-divergent populations ofthe Little Devil poison frog Oophaga sylvatica by analyzing population structure and genetic differentiation to evaluate which processes could account for colordiversity within and among populations. With a combination of PCR amplicons(three mitochondrial and three nuclear markers) and genome-wide markers from adouble-digested RAD (ddRAD) approach, we characterized the phylogenetic andgenetic structure of 199 individuals from 13 populations (12 monomorphic and 1polymorphic) across the O. sylvatica distribution. Individuals segregated intotwo main lineages by their northern or southern latitudinal distribution. A high level of genetic and phenotypic polymorphism within the northern lineage suggestsongoing gene flow. In contrast, low levels of genetic differentiation weredetected among the southern lineage populations and support recent rangeexpansions from populations in the northern lineage. We propose that acombination of climatic gradients and structured landscapes might be promotinggene flow and phylogenetic diversification. Alternatively, we cannot rule outthat the observed phenotypic and genomic variations are the result of geneticdrift on near or neutral alleles in a small number of genes.\n",
      "FAU - Roland, Alexandre B\n",
      "AU  - Roland AB\n",
      "AUID- ORCID: 0000-0002-9463-9838\n",
      "AD  - FAS Center for Systems Biology Harvard University Cambridge MA USA.\n",
      "FAU - Santos, Juan C\n",
      "AU  - Santos JC\n",
      "AUID- ORCID: 0000-0002-7777-8094\n",
      "AD  - Department of Biological Sciences St. John's University Queens NY USA.\n",
      "FAU - Carriker, Bella C\n",
      "AU  - Carriker BC\n",
      "AD  - Lakeside High School Seattle WA USA.\n",
      "FAU - Caty, Stephanie N\n",
      "AU  - Caty SN\n",
      "AD  - FAS Center for Systems Biology Harvard University Cambridge MA USA.\n",
      "FAU - Tapia, Elicio E\n",
      "AU  - Tapia EE\n",
      "AD  - Centro Jambatu de Investigacion y Conservacion de Anfibios Fundacion Otonga QuitoEcuador.\n",
      "FAU - Coloma, Luis A\n",
      "AU  - Coloma LA\n",
      "AD  - Centro Jambatu de Investigacion y Conservacion de Anfibios Fundacion Otonga QuitoEcuador.\n",
      "FAU - O'Connell, Lauren A\n",
      "AU  - O'Connell LA\n",
      "AUID- ORCID: 0000-0002-2706-4077\n",
      "AD  - FAS Center for Systems Biology Harvard University Cambridge MA USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20171018\n",
      "PL  - England\n",
      "TA  - Ecol Evol\n",
      "JT  - Ecology and evolution\n",
      "JID - 101566408\n",
      "PMC - PMC5696431\n",
      "OTO - NOTNLM\n",
      "OT  - *Dendrobatidae\n",
      "OT  - *Ecuador\n",
      "OT  - *Oophaga sylvatica\n",
      "OT  - *amphibian\n",
      "OT  - *aposematism\n",
      "OT  - *ddRAD\n",
      "OT  - *gene flow\n",
      "OT  - *phenotypic variation\n",
      "OT  - *population genomics\n",
      "EDAT- 2017/12/01 06:00\n",
      "MHDA- 2017/12/01 06:01\n",
      "CRDT- 2017/12/01 06:00\n",
      "PHST- 2017/05/17 00:00 [received]\n",
      "PHST- 2017/09/03 00:00 [revised]\n",
      "PHST- 2017/09/13 00:00 [accepted]\n",
      "PHST- 2017/12/01 06:00 [entrez]\n",
      "PHST- 2017/12/01 06:00 [pubmed]\n",
      "PHST- 2017/12/01 06:01 [medline]\n",
      "AID - 10.1002/ece3.3503 [doi]\n",
      "AID - ECE33503 [pii]\n",
      "PST - epublish\n",
      "SO  - Ecol Evol. 2017 Oct 18;7(22):9750-9762. doi: 10.1002/ece3.3503. eCollection 2017 Nov.\n",
      "\n",
      "PMID- 29140767\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180717\n",
      "LR  - 20211204\n",
      "IS  - 1557-8100 (Electronic)\n",
      "IS  - 1536-2310 (Linking)\n",
      "VI  - 21\n",
      "IP  - 11\n",
      "DP  - 2017 Nov\n",
      "TI  - Genomic Medicine Without Borders: Which Strategies Should Developing CountriesEmploy to Invest in Precision Medicine? A New \"Fast-Second Winner\" Strategy.\n",
      "PG  - 647-657\n",
      "LID - 10.1089/omi.2017.0141 [doi]\n",
      "AB  - Genomic medicine has greatly matured in terms of its technical capabilities, but the diffusion of genomic innovations worldwide faces significant barriers beyond mere access to technology. New global development strategies are sorely neededfor biotechnologies such as genomics and their applications toward precisionmedicine without borders. Moreover, diffusion of genomic medicine globally cannotadhere to a \"one-size-fits-all-countries\" development strategy, in the same waythat drug treatments should be customized. This begs a timely, difficult butcrucial question: How should developing countries, and the resource-limitedregions of developed countries, invest in genomic medicine? Although a full-scaleinvestment in infrastructure from discovery to the translational implementationof genomic science is ideal, this may not always be feasible in all countries at all times. A simple \"transplantation of genomics\" from developed to developingcountries is unlikely to be feasible. Nor should developing countries be seen as simple recipients and beneficiaries of genomic medicine developed elsewherebecause important advances in genomic medicine have materialized in developingcountries as well. There are several noteworthy examples of genomic medicinesuccess stories involving resource-limited settings that are contextualized anddescribed in this global genomic medicine innovation analysis. In addition, weoutline here a new long-term development strategy for global genomic medicine in a way that recognizes the individual country's pressing public health priorities and disease burdens. We term this approach the \"Fast-Second Winner\" model ofinnovation that supports innovation commencing not only \"upstream\" of discoveryscience but also \"mid-stream,\" building on emerging highly promising biomarkerand diagnostic candidates from the global science discovery pipeline, based onthe unique needs of each country. A mid-stream entry into innovation can enhance collective learning from other innovators' mistakes upstream in discovery scienceand boost the probability of success for translation and implementation whenresources are limited. This a la carte model of global innovation and developmentstrategy offers multiple entry points into the global genomics innovationecosystem for developing countries, whether or not extensive and expensivediscovery infrastructures are already in place. Ultimately, broadening ourthinking beyond the linear model of innovation will help us to enable the vision and practice of genomics without borders in both developed and resource-limitedsettings.\n",
      "FAU - Mitropoulos, Konstantinos\n",
      "AU  - Mitropoulos K\n",
      "AD  - 1 School of Medicine, University of Athens , Athens, Greece .\n",
      "FAU - Cooper, David N\n",
      "AU  - Cooper DN\n",
      "AD  - 2 Institute of Medical Genetics, School of Medicine, Cardiff University ,Cardiff, United Kingdom .\n",
      "FAU - Mitropoulou, Christina\n",
      "AU  - Mitropoulou C\n",
      "AD  - 3 The Golden Helix Foundation , London, United Kingdom .\n",
      "FAU - Agathos, Spiros\n",
      "AU  - Agathos S\n",
      "AD  - 4 Yachay Tech University , San Miguel de Urcuqui, Ecuador .\n",
      "FAU - Reichardt, Jurgen K V\n",
      "AU  - Reichardt JKV\n",
      "AD  - 4 Yachay Tech University , San Miguel de Urcuqui, Ecuador .\n",
      "FAU - Al-Maskari, Fatima\n",
      "AU  - Al-Maskari F\n",
      "AD  - 5 Department of Pathology, College of Medicine and Health Sciences, United ArabEmirates University , Al-Ain, United Arab Emirates .\n",
      "AD  - 6 Zayed Bin Sultan Center for Health Sciences, United Arab Emirates University , Al-Ain, United Arab Emirates .\n",
      "FAU - Chantratita, Wasun\n",
      "AU  - Chantratita W\n",
      "AD  - 7 Department of Pathology, Medical Genomic Center, Ramathibodi Hospital, Faculty of Medicine, Mahidol University , Bangkok, Thailand .\n",
      "FAU - Wonkam, Ambroise\n",
      "AU  - Wonkam A\n",
      "AD  - 8 Division of Human Genetics, Department of Medicine and Institute of Infectious Disease and Molecular Medicine, University of Cape Town , Cape Town, South Africa.\n",
      "FAU - Dandara, Collet\n",
      "AU  - Dandara C\n",
      "AD  - 8 Division of Human Genetics, Department of Medicine and Institute of Infectious Disease and Molecular Medicine, University of Cape Town , Cape Town, South Africa.\n",
      "FAU - Katsila, Theodora\n",
      "AU  - Katsila T\n",
      "AD  - 9 Department of Pharmacy, School of Health Sciences, University of Patras ,Patras, Greece .\n",
      "FAU - Lopez-Correa, Catalina\n",
      "AU  - Lopez-Correa C\n",
      "AD  - 10 Genome British Columbia , Vancouver, Canada .\n",
      "FAU - Ali, Bassam R\n",
      "AU  - Ali BR\n",
      "AD  - 5 Department of Pathology, College of Medicine and Health Sciences, United ArabEmirates University , Al-Ain, United Arab Emirates .\n",
      "AD  - 6 Zayed Bin Sultan Center for Health Sciences, United Arab Emirates University , Al-Ain, United Arab Emirates .\n",
      "FAU - Patrinos, George P\n",
      "AU  - Patrinos GP\n",
      "AD  - 5 Department of Pathology, College of Medicine and Health Sciences, United ArabEmirates University , Al-Ain, United Arab Emirates .\n",
      "AD  - 6 Zayed Bin Sultan Center for Health Sciences, United Arab Emirates University , Al-Ain, United Arab Emirates .\n",
      "AD  - 9 Department of Pharmacy, School of Health Sciences, University of Patras ,Patras, Greece .\n",
      "LA  - eng\n",
      "GR  - U24 HL135600/HL/NHLBI NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PL  - United States\n",
      "TA  - OMICS\n",
      "JT  - Omics : a journal of integrative biology\n",
      "JID - 101131135\n",
      "SB  - IM\n",
      "MH  - Biotechnology/economics/*trends\n",
      "MH  - Cost of Illness\n",
      "MH  - Developing Countries/*economics\n",
      "MH  - Ecological Parameter Monitoring\n",
      "MH  - Ethnicity\n",
      "MH  - Genomics/economics/*trends\n",
      "MH  - Health Personnel\n",
      "MH  - Humans\n",
      "MH  - Linear Models\n",
      "MH  - Pharmacogenetics\n",
      "MH  - Precision Medicine/economics/*trends\n",
      "MH  - Public Health\n",
      "PMC - PMC6436026\n",
      "OTO - NOTNLM\n",
      "OT  - *Fast-Second Winner\n",
      "OT  - *Strategy for global biotechnology development\n",
      "OT  - *developing countries\n",
      "OT  - *precision medicine\n",
      "OT  - *public health genomic\n",
      "EDAT- 2017/11/16 06:00\n",
      "MHDA- 2018/07/18 06:00\n",
      "CRDT- 2017/11/16 06:00\n",
      "PHST- 2017/11/16 06:00 [entrez]\n",
      "PHST- 2017/11/16 06:00 [pubmed]\n",
      "PHST- 2018/07/18 06:00 [medline]\n",
      "AID - 10.1089/omi.2017.0141 [doi]\n",
      "PST - ppublish\n",
      "SO  - OMICS. 2017 Nov;21(11):647-657. doi: 10.1089/omi.2017.0141.\n",
      "\n",
      "PMID- 29086402\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180612\n",
      "LR  - 20180905\n",
      "IS  - 1940-6029 (Electronic)\n",
      "IS  - 1064-3745 (Linking)\n",
      "VI  - 1696\n",
      "DP  - 2018\n",
      "TI  - Targeted Quantification of Isoforms of a Thylakoid-Bound Protein: MRM MethodDevelopment.\n",
      "PG  - 147-162\n",
      "LID - 10.1007/978-1-4939-7411-5_10 [doi]\n",
      "AB  - Targeted mass spectrometric methods such as selected/multiple reaction monitoring(SRM/MRM) have found intense application in protein detection and quantification which competes with classical immunoaffinity techniques. It provides a universal procedure to develop a fast, highly specific, sensitive, accurate, and cheapmethodology for targeted detection and quantification of proteins based on thedirect analysis of their surrogate peptides typically generated by trypticdigestion. This methodology can be advantageously applied in the field of plantproteomics and particularly for non-model species since immunoreagents arescarcely available. Here, we describe the issues to take into consideration inorder to develop a MRM method to detect and quantify isoforms of thethylakoid-bound protein polyphenol oxidase from the non-model and databaseunderrepresented species Eriobotrya japonica Lindl.\n",
      "FAU - Bru-Martinez, Roque\n",
      "AU  - Bru-Martinez R\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science and IMEM Ramon Margalef, University of Alicante,Ctra. San Vicente del Raspeig s/n, 03080, Alicante, Spain. Roque.Bru@ua.es.\n",
      "FAU - Martinez-Marquez, Ascension\n",
      "AU  - Martinez-Marquez A\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science and IMEM Ramon Margalef, University of Alicante,Ctra. San Vicente del Raspeig s/n, 03080, Alicante, Spain.\n",
      "FAU - Morante-Carriel, Jaime\n",
      "AU  - Morante-Carriel J\n",
      "AD  - Biotechnology and Molecular Biology Group, Quevedo State Technical University,Quevedo, 120501, Ecuador.\n",
      "FAU - Selles-Marchart, Susana\n",
      "AU  - Selles-Marchart S\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science and IMEM Ramon Margalef, University of Alicante,Ctra. San Vicente del Raspeig s/n, 03080, Alicante, Spain.\n",
      "AD  - Research Technical Facility, Proteomics and Genomics Division, University ofAlicante, 03080, Alicante, Spain.\n",
      "FAU - Martinez-Esteso, Maria Jose\n",
      "AU  - Martinez-Esteso MJ\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science and IMEM Ramon Margalef, University of Alicante,Ctra. San Vicente del Raspeig s/n, 03080, Alicante, Spain.\n",
      "FAU - Pineda-Lucas, Jose Luis\n",
      "AU  - Pineda-Lucas JL\n",
      "AD  - Laboratorio Quimico-Microbiologico, S.A., Poligono Industrial Oeste (ClPrincipal), 21-PAR 21/1, 30169, Murcia, Spain.\n",
      "FAU - Luque, Ignacio\n",
      "AU  - Luque I\n",
      "AD  - Institute of Plant Biochemistry and Photosynthesis, University of Seville-CSIC,Av. A. Vespucio 49, 41092, Seville, Spain.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PL  - United States\n",
      "TA  - Methods Mol Biol\n",
      "JT  - Methods in molecular biology (Clifton, N.J.)\n",
      "JID - 9214969\n",
      "RN  - 0 (Chloroplast Proteins)\n",
      "RN  - 0 (Membrane Proteins)\n",
      "RN  - 0 (Protein Isoforms)\n",
      "SB  - IM\n",
      "MH  - Chloroplast Proteins/genetics/isolation & purification\n",
      "MH  - Chromatography, High Pressure Liquid\n",
      "MH  - Eriobotrya/*cytology/genetics/metabolism\n",
      "MH  - Mass Spectrometry\n",
      "MH  - Membrane Proteins/genetics/isolation & purification\n",
      "MH  - Protein Isoforms/genetics/*isolation & purification\n",
      "MH  - Proteomics/*methods\n",
      "MH  - Thylakoids/*metabolism\n",
      "OTO - NOTNLM\n",
      "OT  - *Loquat\n",
      "OT  - *MRM\n",
      "OT  - *Polyphenol oxidase\n",
      "OT  - *Protein isoform\n",
      "OT  - *Thylakoid-bound\n",
      "EDAT- 2017/11/01 06:00\n",
      "MHDA- 2018/06/13 06:00\n",
      "CRDT- 2017/11/01 06:00\n",
      "PHST- 2017/11/01 06:00 [entrez]\n",
      "PHST- 2017/11/01 06:00 [pubmed]\n",
      "PHST- 2018/06/13 06:00 [medline]\n",
      "AID - 10.1007/978-1-4939-7411-5_10 [doi]\n",
      "PST - ppublish\n",
      "SO  - Methods Mol Biol. 2018;1696:147-162. doi: 10.1007/978-1-4939-7411-5_10.\n",
      "\n",
      "PMID- 29048566\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180514\n",
      "LR  - 20181113\n",
      "IS  - 1460-2431 (Electronic)\n",
      "IS  - 0022-0957 (Linking)\n",
      "VI  - 68\n",
      "IP  - 17\n",
      "DP  - 2017 Oct 13\n",
      "TI  - Deciphering the Theobroma cacao self-incompatibility system: from genomics todiagnostic markers for self-compatibility.\n",
      "PG  - 4775-4790\n",
      "LID - 10.1093/jxb/erx293 [doi]\n",
      "AB  - Cocoa self-compatibility is an important yield factor and has been described asbeing controlled by a late gameto-sporophytic system expressed only at the level of the embryo sac. It results in gametic non-fusion and involves several loci. Inthis work, we identified two loci, located on chromosomes 1 and 4 (CH1 and CH4), involved in cocoa self-incompatibility by two different processes. Both loci are responsible for gametic selection, but only one (the CH4 locus) is involved inthe main fruit drop. The CH1 locus acts prior to the gamete fusion step andindependently of the CH4 locus. Using fine-mapping and genome-wide associationstudies, we focused analyses on restricted regions and identified candidategenes. Some of them showed a differential expression between incompatible andcompatible reactions. Immunolocalization experiments provided evidence of CH1candidate genes expressed in ovule and style tissues. Highly polymorphic simplesequence repeat (SSR) diagnostic markers were designed in the CH4 region that hadbeen identified by fine-mapping. They are characterized by a strong linkagedisequilibrium with incompatibility alleles, thus allowing the development ofefficient diagnostic markers predicting self-compatibility and fruit settingaccording to the presence of specific alleles or genotypes. SSR alleles specific to self-compatible Amelonado and Criollo varieties were also identified, thusallowing screening for self-compatible plants in cocoa populations.\n",
      "CI  - (c) The Author 2017. Published by Oxford University Press on behalf of theSociety for Experimental Biology.\n",
      "FAU - Lanaud, Claire\n",
      "AU  - Lanaud C\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "FAU - Fouet, Olivier\n",
      "AU  - Fouet O\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "FAU - Legavre, Thierry\n",
      "AU  - Legavre T\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "FAU - Lopes, Uilson\n",
      "AU  - Lopes U\n",
      "AD  - Centro de Pesquisas do Cacau (CEPEC), CEPLAC, Rod. Ilheus-Itabuna, km 22, Ilheus,BA 45605-350, Brazil.\n",
      "FAU - Sounigo, Olivier\n",
      "AU  - Sounigo O\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UR Bioagresseurs, BP 2572 Elig-Essono, Yaounde, Cameroun.\n",
      "AD  - Institut de Recherche Agricole pour le Developpement (IRAD) PO Box 2123, Yaounde,Cameroun.\n",
      "FAU - Eyango, Marie Claire\n",
      "AU  - Eyango MC\n",
      "AD  - Institut de Recherche Agricole pour le Developpement (IRAD)PO Box 2123, Yaounde, Cameroun.\n",
      "FAU - Mermaz, Benoit\n",
      "AU  - Mermaz B\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "FAU - Da Silva, Marcos Ramos\n",
      "AU  - Da Silva MR\n",
      "AD  - Centro de Pesquisas do Cacau (CEPEC), CEPLAC, Rod. Ilheus-Itabuna, km 22, Ilheus,BA 45605-350, Brazil.\n",
      "FAU - Loor Solorzano, Rey Gaston\n",
      "AU  - Loor Solorzano RG\n",
      "AD  - Instituto Nacional de Investigaciones Agropecuarias (INIAP), EET-Pichilingue. CP 24 Km 5 via Quevedo El Empalme, Quevedo, Los Rios, Ecuador.\n",
      "FAU - Argout, Xavier\n",
      "AU  - Argout X\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "FAU - Gyapay, Gabor\n",
      "AU  - Gyapay G\n",
      "AD  - Commissariat a l'Energie Antomique (CEA), Institut de Genomique (IG), Genoscope, Evry, France.\n",
      "FAU - Ebaiarrey, Herman Ebai\n",
      "AU  - Ebaiarrey HE\n",
      "AD  - Institut de Recherche Agricole pour le Developpement (IRAD)PO Box 2123, Yaounde, Cameroun.\n",
      "FAU - Colonges, Kelly\n",
      "AU  - Colonges K\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "FAU - Sanier, Christine\n",
      "AU  - Sanier C\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "FAU - Rivallan, Ronan\n",
      "AU  - Rivallan R\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "FAU - Mastin, Geraldine\n",
      "AU  - Mastin G\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "FAU - Cryer, Nicholas\n",
      "AU  - Cryer N\n",
      "AD  - Mondelez UK R&D Limited, PO Box 12, Bournville Place, Bournville Lane,Birmingham, B30 2LU, UK.\n",
      "FAU - Boccara, Michel\n",
      "AU  - Boccara M\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "FAU - Verdeil, Jean-Luc\n",
      "AU  - Verdeil JL\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "FAU - Efombagn Mousseni, Ives Bruno\n",
      "AU  - Efombagn Mousseni IB\n",
      "AD  - Institut de Recherche Agricole pour le Developpement (IRAD)PO Box 2123, Yaounde, Cameroun.\n",
      "FAU - Peres Gramacho, Karina\n",
      "AU  - Peres Gramacho K\n",
      "AD  - Centro de Pesquisas do Cacau (CEPEC), CEPLAC, Rod. Ilheus-Itabuna, km 22, Ilheus,BA 45605-350, Brazil.\n",
      "FAU - Clement, Didier\n",
      "AU  - Clement D\n",
      "AD  - Centre de Cooperation Internationale en Recherche Agronomique pour leDeveloppement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 MontpellierCedex 5, France.\n",
      "AD  - Centro de Pesquisas do Cacau (CEPEC), CEPLAC, Rod. Ilheus-Itabuna, km 22, Ilheus,BA 45605-350, Brazil.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PL  - England\n",
      "TA  - J Exp Bot\n",
      "JT  - Journal of experimental botany\n",
      "JID - 9882906\n",
      "SB  - IM\n",
      "MH  - Cacao/genetics/*physiology\n",
      "MH  - Chromosome Mapping\n",
      "MH  - *Genetic Linkage\n",
      "MH  - *Genome-Wide Association Study\n",
      "MH  - Self-Incompatibility in Flowering Plants/*genetics\n",
      "PMC - PMC5853246\n",
      "OTO - NOTNLM\n",
      "OT  - *Candidate genes\n",
      "OT  - *Theobroma cacao\n",
      "OT  - *diagnostic markers\n",
      "OT  - *fine mapping\n",
      "OT  - *gene expression\n",
      "OT  - *immunolocalization\n",
      "OT  - *self-incompatibility\n",
      "EDAT- 2017/10/20 06:00\n",
      "MHDA- 2018/05/15 06:00\n",
      "CRDT- 2017/10/20 06:00\n",
      "PHST- 2017/10/20 06:00 [entrez]\n",
      "PHST- 2017/10/20 06:00 [pubmed]\n",
      "PHST- 2018/05/15 06:00 [medline]\n",
      "AID - 4371616 [pii]\n",
      "AID - 10.1093/jxb/erx293 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Exp Bot. 2017 Oct 13;68(17):4775-4790. doi: 10.1093/jxb/erx293.\n",
      "\n",
      "PMID- 29045648\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180731\n",
      "LR  - 20190107\n",
      "IS  - 1460-2091 (Electronic)\n",
      "IS  - 0305-7453 (Linking)\n",
      "VI  - 73\n",
      "IP  - 1\n",
      "DP  - 2018 Jan 1\n",
      "TI  - Staphylococcus aureus bloodstream infections in Latin America: results of amultinational prospective cohort study.\n",
      "PG  - 212-222\n",
      "LID - 10.1093/jac/dkx350 [doi]\n",
      "AB  - Background: Substantial heterogeneity in the epidemiology and management ofStaphylococcus aureus bacteraemia (SAB) occurs in Latin America. We conducted aprospective cohort study in 24 hospitals from nine Latin American countries.Objectives: To assess the clinical impact of SAB in Latin America. Patients andmethods: We evaluated differences in the 30 day attributable mortality amongpatients with SAB due to MRSA compared with MSSA involving 84 days of follow-up. Adjusted relative risks were calculated using a generalized linear model.Results: A total of 1030 patients were included. MRSA accounted for 44.7% ofcases with a heterogeneous geographical distribution. MRSA infection wasassociated with higher 30 day attributable mortality [25% (78 of 312) versus13.2% (48 of 363), adjusted RR: 1.94, 95% CI: 1.38-2.73, P < 0.001] compared withMSSA in the multivariable analysis based on investigators' assessment, but not ina per-protocol analysis [13% (35 of 270) versus 8.1% (28 of 347), adjusted RR:1.10, 95% CI: 0.75-1.60, P = 0.616] or in a sensitivity analysis using 30 dayall-cause mortality [36% (132 of 367) versus 27.8% (123 of 442), adjusted RR:1.09, 95% CI: 0.96-1.23, P = 0.179]. MRSA infection was not associated withincreased length of hospital stay. Only 49% of MSSA bloodstream infections (BSI) received treatment with beta-lactams, but appropriate definitive treatment wasnot associated with lower mortality (adjusted RR: 0.93, 95% CI: 0.70-1.23, P =0.602). Conclusions: MRSA-BSIs in Latin America are not associated with higher 30day mortality or longer length of stay compared with MSSA. Management ofMSSA-BSIs was not optimal, but appropriate definitive therapy did not appear toinfluence mortality.\n",
      "CI  - (c) The Author 2017. Published by Oxford University Press on behalf of theBritish Society for Antimicrobial Chemotherapy. All rights reserved. ForPermissions, please email: journals.permissions@oup.com.\n",
      "FAU - Seas, Carlos\n",
      "AU  - Seas C\n",
      "AD  - Hospital Cayetano Heredia, Lima, Peru, Instituto de Medicina Tropical Alexandervon Humboldt, Universidad Peruana Cayetano Heredia, Lima, Peru.\n",
      "FAU - Garcia, Coralith\n",
      "AU  - Garcia C\n",
      "AD  - Hospital Cayetano Heredia, Lima, Peru, Instituto de Medicina Tropical Alexandervon Humboldt, Universidad Peruana Cayetano Heredia, Lima, Peru.\n",
      "FAU - Salles, Mauro J\n",
      "AU  - Salles MJ\n",
      "AD  - Division of Infectious Diseases, Department of Internal Medicine, Santa Casa deSao Paulo School of Medicine, Sao Paulo, Brazil.\n",
      "FAU - Labarca, Jaime\n",
      "AU  - Labarca J\n",
      "AD  - Department of Infectious Diseases, School of Medicine, Pontificia UniversidadCatolica de Chile, Santiago, Chile.\n",
      "FAU - Luna, Carlos\n",
      "AU  - Luna C\n",
      "AD  - Pulmonary Division, Department of Medicine, Jose de San Martin Hospital,University of Buenos Aires, Buenos Aires, Argentina.\n",
      "FAU - Alvarez-Moreno, Carlos\n",
      "AU  - Alvarez-Moreno C\n",
      "AD  - Grupo de Investigacion en Enfermedades Infecciosas, Facultad de Medicina,Universidad Nacional de Colombia, Bogota, Colombia.\n",
      "FAU - Mejia-Villatoro, Carlos\n",
      "AU  - Mejia-Villatoro C\n",
      "AD  - Clinica de Enfermedades Infecciosas, Hospital Roosevelt, Guatemala City,Guatemala.\n",
      "FAU - Zurita, Jeannete\n",
      "AU  - Zurita J\n",
      "AD  - Hospital Vozandes, Facultad de Medicina, Pontificia Universidad Catolica delEcuador, Quito, Ecuador.\n",
      "FAU - Guzman-Blanco, Manuel\n",
      "AU  - Guzman-Blanco M\n",
      "AD  - Centro Medico de Caracas, Hospital Vargas de Caracas, Caracas, Venezuela.\n",
      "FAU - Rodriguez-Noriega, Eduardo\n",
      "AU  - Rodriguez-Noriega E\n",
      "AD  - Hospital Civil de Guadalajara, Fray Antonio Alcalde, and Instituto de PatologiaInfecciosa y Experimental, Centro Universitario de Ciencias de la Salud,Universidad de Guadalajara, Guadalajara, Mexico.\n",
      "FAU - Reyes, Jinnethe\n",
      "AU  - Reyes J\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit and International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Arias, Cesar A\n",
      "AU  - Arias CA\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit and International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "AD  - Center for Antimicrobial Resistance and Microbial Genomics, University of Texas, McGovern School of Medicine at Houston, Houston, TX, USA.\n",
      "FAU - Carcamo, Cesar\n",
      "AU  - Carcamo C\n",
      "AD  - School of Public Health and Administration, Universidad Peruana Cayetano Heredia,Lima, Peru.\n",
      "FAU - Gotuzzo, Eduardo\n",
      "AU  - Gotuzzo E\n",
      "AD  - Hospital Cayetano Heredia, Lima, Peru, Instituto de Medicina Tropical Alexandervon Humboldt, Universidad Peruana Cayetano Heredia, Lima, Peru.\n",
      "CN  - Latin America Working Group on Bacterial Resistance\n",
      "LA  - eng\n",
      "GR  - K24 AI121296/AI/NIAID NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Observational Study\n",
      "PL  - England\n",
      "TA  - J Antimicrob Chemother\n",
      "JT  - The Journal of antimicrobial chemotherapy\n",
      "JID - 7513617\n",
      "RN  - 0 (Anti-Bacterial Agents)\n",
      "RN  - 6Q205EH1VU (Vancomycin)\n",
      "SB  - IM\n",
      "MH  - Anti-Bacterial Agents/therapeutic use\n",
      "MH  - Bacteremia/drug therapy/*epidemiology/microbiology\n",
      "MH  - Blood Culture\n",
      "MH  - Cohort Studies\n",
      "MH  - Cross Infection/drug therapy/*epidemiology/microbiology\n",
      "MH  - Female\n",
      "MH  - Humans\n",
      "MH  - Latin America/epidemiology\n",
      "MH  - Male\n",
      "MH  - Methicillin-Resistant Staphylococcus aureus/drug effects/*isolation &purification\n",
      "MH  - Middle Aged\n",
      "MH  - Prospective Studies\n",
      "MH  - Staphylococcal Infections/drug therapy/*epidemiology/mortality\n",
      "MH  - Vancomycin/therapeutic use\n",
      "PMC - PMC5890709\n",
      "IR  - Bruno D\n",
      "FIR - Bruno, Didier\n",
      "IR  - Efron E\n",
      "FIR - Efron, Ernesto\n",
      "IR  - Del Castillo M\n",
      "FIR - Del Castillo, Marcelo\n",
      "IR  - Mater Dei S\n",
      "FIR - Mater Dei, Sanatorio\n",
      "IR  - Guimaraes T\n",
      "FIR - Guimaraes, Thais\n",
      "IR  - Ceballos ME\n",
      "FIR - Ceballos, Maria Elena\n",
      "IR  - de Medicina E\n",
      "FIR - de Medicina, Escuela\n",
      "IR  - Dominguez I\n",
      "FIR - Dominguez, Isabel\n",
      "IR  - Beltran D\n",
      "FIR - Beltran, Daniela\n",
      "IR  - Riedel G\n",
      "FIR - Riedel, Gisela\n",
      "IR  - Valderrama SL\n",
      "FIR - Valderrama, Sandra Liliana\n",
      "IR  - Gualtero SM\n",
      "FIR - Gualtero, Sandra Milena\n",
      "IR  - Shaio C\n",
      "FIR - Shaio, Clinica\n",
      "IR  - Saavedra CH\n",
      "FIR - Saavedra, Carlos Humberto\n",
      "IR  - de Medicina F\n",
      "FIR - de Medicina, Facultad\n",
      "IR  - Aragon JC\n",
      "FIR - Aragon, Juan Carlos\n",
      "IR  - Guerrero F\n",
      "FIR - Guerrero, Fausto\n",
      "IR  - Silvestre MM\n",
      "FIR - Silvestre, Maria Monica\n",
      "IR  - Morfin-Otero R\n",
      "FIR - Morfin-Otero, Rayo\n",
      "IR  - Alcalde FA\n",
      "FIR - Alcalde, Fray Antonio\n",
      "IR  - Hidalgo J\n",
      "FIR - Hidalgo, Jose\n",
      "IR  - Hercilla L\n",
      "FIR - Hercilla, Luis\n",
      "IR  - Hernandez AMC\n",
      "FIR - Hernandez, Ana Maria Caceres\n",
      "IR  - Silva M\n",
      "FIR - Silva, Marisela\n",
      "IR  - Guzman AJ\n",
      "FIR - Guzman, Alfonso Jose\n",
      "EDAT- 2017/10/19 06:00\n",
      "MHDA- 2018/08/01 06:00\n",
      "CRDT- 2017/10/19 06:00\n",
      "PHST- 2017/03/31 00:00 [received]\n",
      "PHST- 2017/08/30 00:00 [accepted]\n",
      "PHST- 2017/10/19 06:00 [pubmed]\n",
      "PHST- 2018/08/01 06:00 [medline]\n",
      "PHST- 2017/10/19 06:00 [entrez]\n",
      "AID - 4555401 [pii]\n",
      "AID - 10.1093/jac/dkx350 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Antimicrob Chemother. 2018 Jan 1;73(1):212-222. doi: 10.1093/jac/dkx350.\n",
      "\n",
      "PMID- 28889342\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180614\n",
      "LR  - 20181113\n",
      "IS  - 1572-9702 (Electronic)\n",
      "IS  - 0168-8162 (Linking)\n",
      "VI  - 73\n",
      "IP  - 2\n",
      "DP  - 2017 Oct\n",
      "TI  - Life cycle of Amblyomma mixtum (Acari: Ixodidae) parasitizing different hostsunder laboratory conditions.\n",
      "PG  - 257-267\n",
      "LID - 10.1007/s10493-017-0178-y [doi]\n",
      "AB  - Amblyomma mixtum is a tick species in the Amblyomma cajennense complex. The knowngeographic range of A. mixtum extends from Texas in the USA to western Ecuadorand some islands in the Caribbean. Amblyomma mixtum is a vector of disease agentsof veterinary and public health importance. The objective of this study was todescribe the life cycle of A. mixtum under laboratory conditions. Bovines,rabbits and sheep were infested with larvae, nymphs, and adult ticks undercontrolled conditions to assess several biological parameters. Eggs, larvae,nymphs and adults were kept in an incubator (27 degrees C temperature and 80%relative humidity) when they were off the host. The average life cycle of A.mixtum was 88 and 79 days when fed on rabbits and cattle, respectively. Sheepwere found to be unsuitable because no ticks attached. The rabbit is a morepractical host to maintain a colony of A. mixtum under laboratory conditions. Thedata from this study can be considered as an example for the life cycle of A.mixtum. However, caution must be exercised when making comparisons to the biologyof A. mixtum in its natural habitat.\n",
      "FAU - Pina, Francisco Tobias Barradas\n",
      "AU  - Pina FTB\n",
      "AD  - Pos-Graduacao em Doencas Infecciosas e Parasitarias - Faculdade de Medicina, UFMS- Universidade Federal Mato Grosso do Sul, Campo Grande, MS, Brazil.\n",
      "AD  - Laboratorio de Biologia do Carrapato, Embrapa Gado de Corte, Campo Grande, MS,Brazil.\n",
      "AD  - Instituto Nacional de Investigaciones Forestales Agricolas y Pecuarias INIFAP,Mexico City, Mexico.\n",
      "FAU - da Silva Rodrigues, Vinicius\n",
      "AU  - da Silva Rodrigues V\n",
      "AD  - Pos-Graduacao em Doencas Infecciosas e Parasitarias - Faculdade de Medicina, UFMS- Universidade Federal Mato Grosso do Sul, Campo Grande, MS, Brazil.\n",
      "AD  - Laboratorio de Biologia do Carrapato, Embrapa Gado de Corte, Campo Grande, MS,Brazil.\n",
      "FAU - de Oliveira Souza Higa, Leandro\n",
      "AU  - de Oliveira Souza Higa L\n",
      "AD  - Pos-Graduacao em Doencas Infecciosas e Parasitarias - Faculdade de Medicina, UFMS- Universidade Federal Mato Grosso do Sul, Campo Grande, MS, Brazil.\n",
      "AD  - Laboratorio de Biologia do Carrapato, Embrapa Gado de Corte, Campo Grande, MS,Brazil.\n",
      "FAU - Garcia, Marcos Valerio\n",
      "AU  - Garcia MV\n",
      "AD  - Laboratorio de Biologia do Carrapato, Empresa Brasileira de PesquisaAgropecuaria, Embrapa Gado de Corte, Campo Grande, MS, Brazil.\n",
      "AD  - Bolsista DCR Fundect, MS - Governo do estado de Mato Grosso do Sul, Campo Grande,Brazil.\n",
      "FAU - Barros, Jacqueline Cavalcante\n",
      "AU  - Barros JC\n",
      "AD  - Empresa Brasileira de Pesquisa Agropecuaria, Embrapa Gado de Corte, Avenida RadioMaia, 830, Campo Grande, MS, CEP 79106-550, Brazil.\n",
      "FAU - de Leon, Adalberto Angel Perez\n",
      "AU  - de Leon AAP\n",
      "AD  - USDA-ARS, Knipling-Bushland U.S. Livestock Insects Research Laboratory, andVeterinary Pest Genomics Center, Kerrville, TX, USA.\n",
      "FAU - Andreotti, Renato\n",
      "AU  - Andreotti R\n",
      "AUID- ORCID: http://orcid.org/0000-0002-0739-2997\n",
      "AD  - Empresa Brasileira de Pesquisa Agropecuaria, Embrapa Gado de Corte, Avenida RadioMaia, 830, Campo Grande, MS, CEP 79106-550, Brazil. renato.andreotti@embrapa.br.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20170909\n",
      "PL  - Netherlands\n",
      "TA  - Exp Appl Acarol\n",
      "JT  - Experimental & applied acarology\n",
      "JID - 8507436\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Cattle\n",
      "MH  - Cattle Diseases/parasitology\n",
      "MH  - Female\n",
      "MH  - *Host-Parasite Interactions\n",
      "MH  - Ixodidae/*growth & development/physiology\n",
      "MH  - Larva/growth & development/physiology\n",
      "MH  - *Life Cycle Stages\n",
      "MH  - Male\n",
      "MH  - Nymph/growth & development/physiology\n",
      "MH  - Rabbits\n",
      "MH  - Sheep\n",
      "MH  - Sheep Diseases/parasitology\n",
      "MH  - Tick Infestations/parasitology/*veterinary\n",
      "OTO - NOTNLM\n",
      "OT  - Bovine\n",
      "OT  - Host suitability\n",
      "OT  - Laboratory colony\n",
      "OT  - Life cycle\n",
      "OT  - Rabbit\n",
      "EDAT- 2017/09/11 06:00\n",
      "MHDA- 2018/06/15 06:00\n",
      "CRDT- 2017/09/11 06:00\n",
      "PHST- 2017/06/29 00:00 [received]\n",
      "PHST- 2017/09/05 00:00 [accepted]\n",
      "PHST- 2017/09/11 06:00 [pubmed]\n",
      "PHST- 2018/06/15 06:00 [medline]\n",
      "PHST- 2017/09/11 06:00 [entrez]\n",
      "AID - 10.1007/s10493-017-0178-y [doi]\n",
      "AID - 10.1007/s10493-017-0178-y [pii]\n",
      "PST - ppublish\n",
      "SO  - Exp Appl Acarol. 2017 Oct;73(2):257-267. doi: 10.1007/s10493-017-0178-y. Epub2017 Sep 9.\n",
      "\n",
      "PMID- 28878794\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20201001\n",
      "IS  - 1664-462X (Print)\n",
      "IS  - 1664-462X (Linking)\n",
      "VI  - 8\n",
      "DP  - 2017\n",
      "TI  - A Tau Class Glutathione-S-Transferase is Involved in Trans-Resveratrol Transport Out of Grapevine Cells.\n",
      "PG  - 1457\n",
      "LID - 10.3389/fpls.2017.01457 [doi]\n",
      "AB  - Vitis vinifera cell cultures respond to pathogens and elicitors by synthesizingand extracellularly accumulating stilbenoid phytoalexins. Large amounts oftrans-resveratrol (t-R) are produced when a cell culture is elicited withmethylated cyclodextrins (MBCD), either alone or combined with methyl jasmonate(MeJA). t-R transport to the extracellular medium, which represents theapoplastic space, would place this antifungal defense right in the battlefield toefficiently fight against pathogen attack. Yet despite their physiologicalrelevance, these transport pathways are mostly unknown. A broad hypothesis-freeDIGE-based proteomic experiment of a temporal series of elicited grapevine cellcultures was performed to explore the expression profiles of t-R biosyntheticproteins and other co-expressing proteins potentially involved in such a cellresponse. A correlation between two tau class glutathione-S-transferases (GSTs)with several stilbene synthase and phenylalanine ammonia-lyase isoforms, and withthe t-R metabolite itself, was found and further assessed by a qRT-PCR geneexpression analysis. The best candidate, GSTU-2, was cloned from the cDNA of the MBCD + MeJA-elicited grapevine cells and used for Agrobacterium-mediatedgrapevine cell transformation. The non-elicited lines that overexpressed GSTU-2displayed an extracellular t-R accumulating phenotype, but stabilization of t-Rrequired the addition to culture medium of adsorbent compounds, e.g., PVP orbeta-cyclodextrin. The wild-type cell cultures accumulated no t-R, not even inthe presence of adsorbents. The transient expression of the GSTU-2-GFP fusionproteins in grapevine cells showed localisation in the plasma membrane, and theimmunoprecipitation of HA-tagged GSTU-2 revealed its interaction with HIR, aplasma membrane-bound protein. These findings are consistent with a functionalrole in transport. This is the first report providing several pieces ofexperimental evidence for the involvement of a specific tau class GST in t-Rtransport to the extracellular medium.\n",
      "FAU - Martinez-Marquez, Ascension\n",
      "AU  - Martinez-Marquez A\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of AlicanteAlicante, Spain.\n",
      "FAU - Martinez-Esteso, Maria J\n",
      "AU  - Martinez-Esteso MJ\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of AlicanteAlicante, Spain.\n",
      "FAU - Vilella-Anton, Maria T\n",
      "AU  - Vilella-Anton MT\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of AlicanteAlicante, Spain.\n",
      "FAU - Selles-Marchart, Susana\n",
      "AU  - Selles-Marchart S\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of AlicanteAlicante, Spain.\n",
      "FAU - Morante-Carriel, Jaime A\n",
      "AU  - Morante-Carriel JA\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of AlicanteAlicante, Spain.\n",
      "AD  - Biotechnology and Molecular Biology Group, Quevedo State TechnicalUniversityQuevedo, Ecuador.\n",
      "FAU - Hurtado, Elias\n",
      "AU  - Hurtado E\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of AlicanteAlicante, Spain.\n",
      "FAU - Palazon, Javier\n",
      "AU  - Palazon J\n",
      "AD  - Laboratory of Plant Physiology, Faculty of Pharmacy, University ofBarcelonaBarcelona, Spain.\n",
      "FAU - Bru-Martinez, Roque\n",
      "AU  - Bru-Martinez R\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of AlicanteAlicante, Spain.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20170821\n",
      "PL  - Switzerland\n",
      "TA  - Front Plant Sci\n",
      "JT  - Frontiers in plant science\n",
      "JID - 101568200\n",
      "PMC - PMC5573539\n",
      "OTO - NOTNLM\n",
      "OT  - DIGE proteomics\n",
      "OT  - Vitis vinifera\n",
      "OT  - cell culture\n",
      "OT  - glutathione-S-transferase\n",
      "OT  - qRT-PCR\n",
      "OT  - resveratrol\n",
      "OT  - transformation\n",
      "OT  - transport\n",
      "EDAT- 2017/09/08 06:00\n",
      "MHDA- 2017/09/08 06:01\n",
      "CRDT- 2017/09/08 06:00\n",
      "PHST- 2017/05/16 00:00 [received]\n",
      "PHST- 2017/08/04 00:00 [accepted]\n",
      "PHST- 2017/09/08 06:00 [entrez]\n",
      "PHST- 2017/09/08 06:00 [pubmed]\n",
      "PHST- 2017/09/08 06:01 [medline]\n",
      "AID - 10.3389/fpls.2017.01457 [doi]\n",
      "PST - epublish\n",
      "SO  - Front Plant Sci. 2017 Aug 21;8:1457. doi: 10.3389/fpls.2017.01457. eCollection2017.\n",
      "\n",
      "PMID- 28792418\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20171103\n",
      "LR  - 20171103\n",
      "IS  - 0177-5103 (Print)\n",
      "IS  - 0177-5103 (Linking)\n",
      "VI  - 125\n",
      "IP  - 3\n",
      "DP  - 2017 Aug 9\n",
      "TI  - Prevalence, diversity and co-occurrence of the white spot syndrome virus, monodonbaculovirus and Penaeus stylirostris densovirus in wild populations of Penaeusmonodon in the Philippines.\n",
      "PG  - 199-206\n",
      "LID - 10.3354/dao03150 [doi]\n",
      "AB  - The farming of the black tiger shrimp Penaeus monodon in the Philippines relieson wild broodstock. PCR was thus used to determine the prevalence of white spotsyndrome virus (WSSV), monodon baculovirus (MBV) and Penaeus stylirostrisdensovirus (PstDV) in a total of 178 shrimp from 6 geographically disparatelocations where broodstock are captured for use in hatcheries. PCR amplicons werealso sequenced to identify phylogenetic relationships of the virus haplotypesdetected. Shrimp from southeastern Luzon (Camarines Norte) had the highestprevalence of each of the 3 viruses and were frequently co-infected with 2 ormore viruses. No viruses were detected in shrimp from northwestern Luzon(Pangasinan). MBV was most prevalent and PstDV strains displayed the most geneticdiversity. WSSV was detected at 3 sites, and a VP28 gene sequence examined wasinvariant and consistent with strains found in many countries, includingThailand, China, Japan, Korea, Indonesia, Iran, Brazil and Mexico. WSSV openreading frame 94 gene sequence analysis identified location-specific repeattypes. MBV sequences were dissimilar to haplotypes detected in India. PstDVsequences were diverse and included 2 lineages detected either in Australia or inthe United States, Ecuador, Taiwan, China and Vietnam. The PCR data confirmedthat WSSV, MBV and PstDV are endemic in P. monodon in the Philippines but thatpopulations at some locations might remain free of infection.\n",
      "FAU - Orosco, Fredmoore L\n",
      "AU  - Orosco FL\n",
      "AD  - Marine Genomics and Molecular Genetics Laboratory, Marine Science Institute,University of the Philippines Diliman, Quezon City 1101, Phillippines.\n",
      "FAU - Lluisma, Arturo O\n",
      "AU  - Lluisma AO\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PL  - Germany\n",
      "TA  - Dis Aquat Organ\n",
      "JT  - Diseases of aquatic organisms\n",
      "JID - 8807037\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - *Animals, Wild\n",
      "MH  - Baculoviridae/*genetics/isolation & purification\n",
      "MH  - Densovirus/*genetics/isolation & purification\n",
      "MH  - *Genetic Variation\n",
      "MH  - Genome, Viral\n",
      "MH  - Host-Pathogen Interactions\n",
      "MH  - Penaeidae/*virology\n",
      "MH  - Philippines\n",
      "MH  - Phylogeny\n",
      "MH  - White spot syndrome virus 1/*genetics/isolation & purification\n",
      "OTO - NOTNLM\n",
      "OT  - *Diversity\n",
      "OT  - *MBV\n",
      "OT  - *Monodon baculovirus\n",
      "OT  - *Penaeus monodon\n",
      "OT  - *Penaeus stylirostris densovirus\n",
      "OT  - *Philippines\n",
      "OT  - *Prevalence\n",
      "OT  - *PstDV\n",
      "OT  - *WSSV\n",
      "OT  - *White spot syndrome virus\n",
      "EDAT- 2017/08/10 06:00\n",
      "MHDA- 2017/11/04 06:00\n",
      "CRDT- 2017/08/10 06:00\n",
      "PHST- 2017/08/10 06:00 [entrez]\n",
      "PHST- 2017/08/10 06:00 [pubmed]\n",
      "PHST- 2017/11/04 06:00 [medline]\n",
      "AID - 10.3354/dao03150 [doi]\n",
      "PST - ppublish\n",
      "SO  - Dis Aquat Organ. 2017 Aug 9;125(3):199-206. doi: 10.3354/dao03150.\n",
      "\n",
      "PMID- 28789679\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180518\n",
      "LR  - 20181113\n",
      "IS  - 1755-8794 (Electronic)\n",
      "IS  - 1755-8794 (Linking)\n",
      "VI  - 10\n",
      "IP  - 1\n",
      "DP  - 2017 Aug 8\n",
      "TI  - Consensus strategy in genes prioritization and combined bioinformatics analysisfor preeclampsia pathogenesis.\n",
      "PG  - 50\n",
      "LID - 10.1186/s12920-017-0286-x [doi]\n",
      "AB  - BACKGROUND: Preeclampsia is a multifactorial disease with unknown pathogenesis.Even when recent studies explored this disease using several bioinformaticstools, the main objective was not directed to pathogenesis. Additionally,consensus prioritization was proved to be highly efficient in the recognition of genes-disease association. However, not information is available about theconsensus ability to early recognize genes directly involved in pathogenesis.Therefore our aim in this study is to apply several theoretical approaches toexplore preeclampsia; specifically those genes directly involved in thepathogenesis. METHODS: We firstly evaluated the consensus between 12prioritization strategies to early recognize pathogenic genes related topreeclampsia. A communality analysis in the protein-protein interaction networkof previously selected genes was done including further enrichment analysis. The enrichment analysis includes metabolic pathways as well as gene ontology.Microarray data was also collected and used in order to confirm our results or asa strategy to weight the previously enriched pathways. RESULTS: The consensusprioritized gene list was rationally filtered to 476 genes using severalcriteria. The communality analysis showed an enrichment of communities connected with VEGF-signaling pathway. This pathway is also enriched considering themicroarray data. Our result point to VEGF, FLT1 and KDR as relevant pathogenicgenes, as well as those connected with NO metabolism. CONCLUSION: Our resultsrevealed that consensus strategy improve the detection and initial enrichment of pathogenic genes, at least in preeclampsia condition. Moreover the combination ofthe first percent of the prioritized genes with protein-protein interactionnetwork followed by communality analysis reduces the gene space. This approachactually identifies well known genes related with pathogenesis. However, geneslike HSP90, PAK2, CD247 and others included in the first 1% of the prioritizedlist need to be further explored in preeclampsia pathogenesis throughexperimental approaches.\n",
      "FAU - Tejera, Eduardo\n",
      "AU  - Tejera E\n",
      "AUID- ORCID: 0000-0002-1377-0413\n",
      "AD  - Facultad de Medicina, Universidad de Las Americas, Av. de los Granados E12-41yColimes esq, EC170125, Quito, Ecuador. edutp00@gmail.com.\n",
      "FAU - Cruz-Monteagudo, Maykel\n",
      "AU  - Cruz-Monteagudo M\n",
      "AD  - Department of Molecular and Cellular Pharmacology, Miller School of Medicine and Center for Computational Science, University of Miami, FL 33136, Miami, USA.\n",
      "AD  - Department of General Education, West Coast University-Miami Campus, Doral, FL33178, USA.\n",
      "AD  - CIQUP/Departamento de Quimica e Bioquimica, Faculdade de Ciencias, Universidadedo Porto, 4169-007, Porto, Portugal.\n",
      "AD  - REQUIMTE, Department of Chemistry and Biochemistry, Faculty of Sciences,University of Porto, 4169-007, Porto, Portugal.\n",
      "FAU - Burgos, German\n",
      "AU  - Burgos G\n",
      "AD  - Facultad de Medicina, Universidad de Las Americas, Av. de los Granados E12-41yColimes esq, EC170125, Quito, Ecuador.\n",
      "FAU - Sanchez, Maria-Eugenia\n",
      "AU  - Sanchez ME\n",
      "AD  - Facultad de Medicina, Universidad de Las Americas, Av. de los Granados E12-41yColimes esq, EC170125, Quito, Ecuador.\n",
      "FAU - Sanchez-Rodriguez, Aminael\n",
      "AU  - Sanchez-Rodriguez A\n",
      "AD  - Departamento de Ciencias Naturales, Universidad Tecnica Particular de Loja, CalleParis S/N, EC1101608, Loja, Ecuador.\n",
      "FAU - Perez-Castillo, Yunierkis\n",
      "AU  - Perez-Castillo Y\n",
      "AD  - Escuela de Ciencias Fisicas y Matematicas, Universidad de Las Americas, Quito,Ecuador.\n",
      "FAU - Borges, Fernanda\n",
      "AU  - Borges F\n",
      "AD  - CIQUP/Departamento de Quimica e Bioquimica, Faculdade de Ciencias, Universidadedo Porto, 4169-007, Porto, Portugal.\n",
      "FAU - Cordeiro, Maria Natalia Dias Soeiro\n",
      "AU  - Cordeiro MNDS\n",
      "AD  - REQUIMTE, Department of Chemistry and Biochemistry, Faculty of Sciences,University of Porto, 4169-007, Porto, Portugal.\n",
      "FAU - Paz-Y-Mino, Cesar\n",
      "AU  - Paz-Y-Mino C\n",
      "AD  - Centro de Investigaciones genetica y genomica, Facultad de Ciencias de la Salud, Universidad Tecnologica Equinoccial, Quito, Ecuador.\n",
      "FAU - Rebelo, Irene\n",
      "AU  - Rebelo I\n",
      "AD  - Faculty of Pharmacy, University of Porto, Porto, Portugal.\n",
      "AD  - UCIBIO@REQUIMTE, Caparica, Portugal.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20170808\n",
      "PL  - England\n",
      "TA  - BMC Med Genomics\n",
      "JT  - BMC medical genomics\n",
      "JID - 101319628\n",
      "SB  - IM\n",
      "MH  - *Computational Biology\n",
      "MH  - *Consensus\n",
      "MH  - Female\n",
      "MH  - Gene Expression Profiling\n",
      "MH  - Humans\n",
      "MH  - Metabolic Networks and Pathways/genetics\n",
      "MH  - Pre-Eclampsia/*etiology/*genetics/metabolism\n",
      "MH  - Pregnancy\n",
      "MH  - Protein Interaction Maps\n",
      "PMC - PMC5549357\n",
      "OTO - NOTNLM\n",
      "OT  - *Communality analysis\n",
      "OT  - *Consensus analysis\n",
      "OT  - *Early recognition\n",
      "OT  - *Gene periodization\n",
      "OT  - *Microarray analysis\n",
      "OT  - *Pathogenesis\n",
      "OT  - *Preeclampsia\n",
      "EDAT- 2017/08/10 06:00\n",
      "MHDA- 2018/05/19 06:00\n",
      "CRDT- 2017/08/10 06:00\n",
      "PHST- 2017/02/08 00:00 [received]\n",
      "PHST- 2017/07/28 00:00 [accepted]\n",
      "PHST- 2017/08/10 06:00 [entrez]\n",
      "PHST- 2017/08/10 06:00 [pubmed]\n",
      "PHST- 2018/05/19 06:00 [medline]\n",
      "AID - 10.1186/s12920-017-0286-x [doi]\n",
      "AID - 10.1186/s12920-017-0286-x [pii]\n",
      "PST - epublish\n",
      "SO  - BMC Med Genomics. 2017 Aug 8;10(1):50. doi: 10.1186/s12920-017-0286-x.\n",
      "\n",
      "PMID- 28760895\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180530\n",
      "LR  - 20210714\n",
      "IS  - 1098-6596 (Electronic)\n",
      "IS  - 0066-4804 (Linking)\n",
      "VI  - 61\n",
      "IP  - 10\n",
      "DP  - 2017 Oct\n",
      "TI  - A Prospective Cohort Multicenter Study of Molecular Epidemiology andPhylogenomics of Staphylococcus aureus Bacteremia in Nine Latin AmericanCountries.\n",
      "LID - e00816-17 [pii]\n",
      "LID - 10.1128/AAC.00816-17 [doi]\n",
      "AB  - Staphylococcus aureus is an important pathogen causing a spectrum of diseasesranging from mild skin and soft tissue infections to life-threatening conditions.Bloodstream infections are particularly important, and the treatment approach is complicated by the presence of methicillin-resistant S. aureus (MRSA) isolates.The emergence of new genetic lineages of MRSA has occurred in Latin America (LA) with the rise and dissemination of the community-associated USA300 Latin Americanvariant (USA300-LV). Here, we prospectively characterized bloodstream MRSArecovered from selected hospitals in 9 Latin American countries. All isolateswere typed by pulsed-field gel electrophoresis (PFGE) and subjected to antibioticsusceptibility testing. Whole-genome sequencing was performed on 96 MRSArepresentatives. MRSA represented 45% of all (1,185 S. aureus) isolates. Themajority of MRSA isolates belonged to clonal cluster (CC) 5. In Colombia andEcuador, most isolates (>/=72%) belonged to the USA300-LV lineage (CC8).Phylogenetic reconstructions indicated that MRSA isolates from participatinghospitals belonged to three major clades. Clade A grouped isolates with sequence type 5 (ST5), ST105, and ST1011 (mostly staphylococcal chromosomal cassette mec[SCCmec] I and II). Clade B included ST8, ST88, ST97, and ST72 strains (SCCmecIV, subtypes a, b, and c/E), and clade C grouped mostly Argentinian MRSAbelonging to ST30. In summary, CC5 MRSA was prevalent in bloodstream infectionsin LA with the exception of Colombia and Ecuador, where USA300-LV is now thedominant lineage. Clonal replacement appears to be a common phenomenon, andcontinuous surveillance is crucial to identify changes in the molecularepidemiology of MRSA.\n",
      "CI  - Copyright (c) 2017 American Society for Microbiology.\n",
      "FAU - Arias, Cesar A\n",
      "AU  - Arias CA\n",
      "AD  - Center for Antimicrobial Resistance and Microbial Genomics and Division ofInfectious Diseases, University of Texas McGovern Medical School at Houston,Houston, Texas, USA cesar.arias@uth.tmc.edu.\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Reyes, Jinnethe\n",
      "AU  - Reyes J\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Carvajal, Lina Paola\n",
      "AU  - Carvajal LP\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Rincon, Sandra\n",
      "AU  - Rincon S\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Diaz, Lorena\n",
      "AU  - Diaz L\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Panesso, Diana\n",
      "AU  - Panesso D\n",
      "AD  - Center for Antimicrobial Resistance and Microbial Genomics and Division ofInfectious Diseases, University of Texas McGovern Medical School at Houston,Houston, Texas, USA.\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Ibarra, Gabriel\n",
      "AU  - Ibarra G\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Rios, Rafael\n",
      "AU  - Rios R\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Munita, Jose M\n",
      "AU  - Munita JM\n",
      "AD  - Grupo de Genomica Microbiana, Instituto de Ciencias e Innovacion en Medicina,Clinica Alemana, Universidad del Desarrollo, Santiago, Chile.\n",
      "FAU - Salles, Mauro J\n",
      "AU  - Salles MJ\n",
      "AD  - Division of Infectious Diseases, Department of Internal Medicine, Santa Casa deSao Paulo School of Medicine, Sao Paulo, Brazil.\n",
      "FAU - Alvarez-Moreno, Carlos\n",
      "AU  - Alvarez-Moreno C\n",
      "AD  - Unidad Infectologia, Departamento de Medicina Interna, Facultad de Medicina,Universidad Nacional de Colombia, Clinica Universitaria Colombia, Colsanitas,Bogota, Colombia.\n",
      "FAU - Labarca, Jaime\n",
      "AU  - Labarca J\n",
      "AD  - Department of Infectious Diseases, School of Medicine, Pontificia UniversidadCatolica de Chile, Santiago, Chile.\n",
      "FAU - Garcia, Coralith\n",
      "AU  - Garcia C\n",
      "AD  - Hospital Cayetano Heredia, Instituto de Medicina Tropical Alexander von Humboldt,Universidad Peruana Cayetano Heredia, Lima, Peru.\n",
      "FAU - Luna, Carlos M\n",
      "AU  - Luna CM\n",
      "AD  - Pulmonary Division, Department of Medicine, Jose de San Martin Hospital,University of Buenos Aires, Buenos Aires, Argentina.\n",
      "FAU - Mejia-Villatoro, Carlos\n",
      "AU  - Mejia-Villatoro C\n",
      "AD  - Clinica de Enfermedades Infecciosas, Hospital Roosevelt, Guatemala City,Guatemala.\n",
      "FAU - Zurita, Jeannete\n",
      "AU  - Zurita J\n",
      "AD  - Hospital Vozandes, Facultad de Medicina, Pontificia Universidad Catolica delEcuador, Quito, Ecuador.\n",
      "FAU - Guzman-Blanco, Manuel\n",
      "AU  - Guzman-Blanco M\n",
      "AD  - Centro Medico de Caracas, Caracas, Venezuela.\n",
      "FAU - Rodriguez-Noriega, Eduardo\n",
      "AU  - Rodriguez-Noriega E\n",
      "AD  - Hospital Civil de Guadalajara, Fray Antonio Alcalde, Instituto de PatologiaInfecciosa y Experimental, Centro Universitario de Ciencias de la Salud,Universidad de Guadalajara, Guadalajara, Mexico.\n",
      "FAU - Narechania, Apurva\n",
      "AU  - Narechania A\n",
      "AD  - Sackler Institute for Comparative Genomics, American Museum of Natural History,New York, New York, USA.\n",
      "FAU - Rojas, Laura J\n",
      "AU  - Rojas LJ\n",
      "AD  - Department of Molecular Biology and Microbiology, Case Western ReserveUniversity, and Research Service, Louis Stokes Cleveland Department of VeteransAffairs, Cleveland, Ohio, USA.\n",
      "FAU - Planet, Paul J\n",
      "AU  - Planet PJ\n",
      "AD  - Sackler Institute for Comparative Genomics, American Museum of Natural History,New York, New York, USA.\n",
      "AD  - Department of Pediatrics, Division of Pediatric Infectious Diseases, ColumbiaUniversity, College of Physicians and Surgeons, New York, New York, USA.\n",
      "AD  - Department of Pediatrics, Division of Infectious Diseases, The Children'sHospital of Philadelphia, Philadelphia, Pennsylvania, USA.\n",
      "FAU - Weinstock, George M\n",
      "AU  - Weinstock GM\n",
      "AD  - The Jackson Laboratory for Genomic Medicine, Farmington, Connecticut, USA.\n",
      "FAU - Gotuzzo, Eduardo\n",
      "AU  - Gotuzzo E\n",
      "AD  - Hospital Cayetano Heredia, Instituto de Medicina Tropical Alexander von Humboldt,Universidad Peruana Cayetano Heredia, Lima, Peru.\n",
      "FAU - Seas, Carlos\n",
      "AU  - Seas C\n",
      "AD  - Hospital Cayetano Heredia, Instituto de Medicina Tropical Alexander von Humboldt,Universidad Peruana Cayetano Heredia, Lima, Peru.\n",
      "LA  - eng\n",
      "GR  - R01 AI137526/AI/NIAID NIH HHS/United States\n",
      "GR  - R21 AI121519/AI/NIAID NIH HHS/United States\n",
      "GR  - K24 AI121296/AI/NIAID NIH HHS/United States\n",
      "GR  - R01 AI093749/AI/NIAID NIH HHS/United States\n",
      "GR  - R21 AI114961/AI/NIAID NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Multicenter Study\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20170922\n",
      "PL  - United States\n",
      "TA  - Antimicrob Agents Chemother\n",
      "JT  - Antimicrobial agents and chemotherapy\n",
      "JID - 0315061\n",
      "RN  - 0 (Anti-Bacterial Agents)\n",
      "SB  - IM\n",
      "EIN - Antimicrob Agents Chemother. 2018 Feb 23;62(3):. PMID: 29475888\n",
      "MH  - Anti-Bacterial Agents/pharmacology\n",
      "MH  - Bacteremia/*epidemiology/microbiology\n",
      "MH  - Genome, Bacterial/genetics\n",
      "MH  - Humans\n",
      "MH  - Latin America\n",
      "MH  - Methicillin-Resistant Staphylococcus aureus/classification/*drugeffects/*genetics/isolation & purification\n",
      "MH  - Microbial Sensitivity Tests\n",
      "MH  - Molecular Epidemiology\n",
      "MH  - Multilocus Sequence Typing\n",
      "MH  - Prospective Studies\n",
      "MH  - Staphylococcal Infections/*epidemiology/microbiology\n",
      "PMC - PMC5610503\n",
      "OTO - NOTNLM\n",
      "OT  - *Latin America\n",
      "OT  - *Staphylococcus aureus\n",
      "OT  - *bacteremia\n",
      "EDAT- 2017/08/02 06:00\n",
      "MHDA- 2018/05/31 06:00\n",
      "CRDT- 2017/08/02 06:00\n",
      "PHST- 2017/04/18 00:00 [received]\n",
      "PHST- 2017/07/18 00:00 [accepted]\n",
      "PHST- 2017/08/02 06:00 [pubmed]\n",
      "PHST- 2018/05/31 06:00 [medline]\n",
      "PHST- 2017/08/02 06:00 [entrez]\n",
      "AID - AAC.00816-17 [pii]\n",
      "AID - 10.1128/AAC.00816-17 [doi]\n",
      "PST - epublish\n",
      "SO  - Antimicrob Agents Chemother. 2017 Sep 22;61(10). pii: AAC.00816-17. doi:10.1128/AAC.00816-17. Print 2017 Oct.\n",
      "\n",
      "PMID- 28583505\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170718\n",
      "LR  - 20181202\n",
      "IS  - 1872-8464 (Electronic)\n",
      "IS  - 0165-5876 (Linking)\n",
      "VI  - 98\n",
      "DP  - 2017 Jul\n",
      "TI  - Novel EYA1 variants causing Branchio-oto-renal syndrome.\n",
      "PG  - 59-63\n",
      "LID - S0165-5876(17)30178-7 [pii]\n",
      "LID - 10.1016/j.ijporl.2017.04.037 [doi]\n",
      "AB  - INTRODUCTION: Branchio-oto-renal (BOR) syndrome is an autosomal dominant genetic disorder characterized by second branchial arch anomalies, hearing impairment,and renal malformations. Pathogenic mutations have been discovered in severalgenes such as EYA1, SIX5, and SIX1. However, nearly half of those affected revealno pathogenic variant by traditional genetic testing. METHODS AND MATERIALS:Whole Exome sequencing and/or Sanger sequencing performed in 10 unrelatedfamilies from Turkey, Iran, Ecuador, and USA with BOR syndrome in this study.RESULTS: We identified causative DNA variants in six families including novelc.525delT, c.979T > C, and c.1768delG and a previously reported c.1779A > Tvariants in EYA1. Two large heterozygous deletions involving EYA1 were detectedin additional two families. Whole exome sequencing did not reveal a causativevariant in the remaining four families. CONCLUSIONS: A variety of DNA changesincluding large deletions underlie BOR syndrome in different populations, whichcan be detected with comprehensive genetic testing.\n",
      "CI  - Copyright (c) 2017 Elsevier B.V. All rights reserved.\n",
      "FAU - Klingbeil, Kyle D\n",
      "AU  - Klingbeil KD\n",
      "AD  - John P. Hussman Institute for Human Genomics, Miller School of Medicine,University of Miami, Miami, FL, USA.\n",
      "FAU - Greenland, Christopher M\n",
      "AU  - Greenland CM\n",
      "AD  - John P. Hussman Institute for Human Genomics, Miller School of Medicine,University of Miami, Miami, FL, USA.\n",
      "FAU - Arslan, Selcuk\n",
      "AU  - Arslan S\n",
      "AD  - Karadeniz Technical University, Faculty of Medicine, Department of ENT, Trabzon, Turkey.\n",
      "FAU - Llamos Paneque, Arianne\n",
      "AU  - Llamos Paneque A\n",
      "AD  - Medical Genetics Lab, FFAA Hospital, Quito, Ecuador.\n",
      "FAU - Gurkan, Hakan\n",
      "AU  - Gurkan H\n",
      "AD  - Trakya University, School of Medicine, Department of Medical Genetics, Edirne,Turkey.\n",
      "FAU - Demir Ulusal, Selma\n",
      "AU  - Demir Ulusal S\n",
      "AD  - Trakya University, School of Medicine, Department of Medical Genetics, Edirne,Turkey.\n",
      "FAU - Maroofian, Reza\n",
      "AU  - Maroofian R\n",
      "AD  - Institute of Biomedical and Clinical Science, University of Exeter MedicalSchool, RILD Wellcome Wolfson Centre, Exeter, UK.\n",
      "FAU - Carrera-Gonzalez, Andrea\n",
      "AU  - Carrera-Gonzalez A\n",
      "AD  - Medical Genetics Lab, FFAA Hospital, Quito, Ecuador.\n",
      "FAU - Montufar-Armendariz, Stefany\n",
      "AU  - Montufar-Armendariz S\n",
      "AD  - Medical Genetics Lab, FFAA Hospital, Quito, Ecuador.\n",
      "FAU - Paredes, Rosario\n",
      "AU  - Paredes R\n",
      "AD  - Medical Genetics Lab, FFAA Hospital, Quito, Ecuador.\n",
      "FAU - Elcioglu, Nursel\n",
      "AU  - Elcioglu N\n",
      "AD  - Department of Pediatric Genetics, Marmara University Medical School, Istanbul,Turkey.\n",
      "FAU - Menendez, Ibis\n",
      "AU  - Menendez I\n",
      "AD  - John P. Hussman Institute for Human Genomics, Miller School of Medicine,University of Miami, Miami, FL, USA.\n",
      "FAU - Behnam, Mahdiyeh\n",
      "AU  - Behnam M\n",
      "AD  - Medical Genetics Laboratory of Genome, Isfahan, Iran.\n",
      "FAU - Foster, Joseph 2nd\n",
      "AU  - Foster J 2nd\n",
      "AD  - John P. Hussman Institute for Human Genomics, Miller School of Medicine,University of Miami, Miami, FL, USA.\n",
      "FAU - Guo, Shengru\n",
      "AU  - Guo S\n",
      "AD  - John P. Hussman Institute for Human Genomics, Miller School of Medicine,University of Miami, Miami, FL, USA.\n",
      "FAU - Escarfuller, Sebastian\n",
      "AU  - Escarfuller S\n",
      "AD  - John P. Hussman Institute for Human Genomics, Miller School of Medicine,University of Miami, Miami, FL, USA.\n",
      "FAU - Cengiz, Filiz Basak\n",
      "AU  - Cengiz FB\n",
      "AD  - John P. Hussman Institute for Human Genomics, Miller School of Medicine,University of Miami, Miami, FL, USA.\n",
      "FAU - Duman, Duygu\n",
      "AU  - Duman D\n",
      "AD  - Division of Genetics, Department of Pediatrics, Ankara University School ofMedicine, Ankara, Turkey.\n",
      "FAU - Bademci, Guney\n",
      "AU  - Bademci G\n",
      "AD  - John P. Hussman Institute for Human Genomics, Miller School of Medicine,University of Miami, Miami, FL, USA.\n",
      "FAU - Tekin, Mustafa\n",
      "AU  - Tekin M\n",
      "AD  - John P. Hussman Institute for Human Genomics, Miller School of Medicine,University of Miami, Miami, FL, USA; Dr. John T. Macdonald Foundation Department of Human Genetics, Miller School of Medicine, University of Miami, Miami, FL,USA. Electronic address: mtekin@med.miami.edu.\n",
      "LA  - eng\n",
      "GR  - R01 DC009645/DC/NIDCD NIH HHS/United States\n",
      "GR  - R01 DC012836/DC/NIDCD NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "DEP - 20170426\n",
      "PL  - Ireland\n",
      "TA  - Int J Pediatr Otorhinolaryngol\n",
      "JT  - International journal of pediatric otorhinolaryngology\n",
      "JID - 8003603\n",
      "RN  - 0 (Intracellular Signaling Peptides and Proteins)\n",
      "RN  - 0 (Nuclear Proteins)\n",
      "RN  - EC 3.1.3.48 (EYA1 protein, human)\n",
      "RN  - EC 3.1.3.48 (Protein Tyrosine Phosphatases)\n",
      "SB  - IM\n",
      "MH  - Adult\n",
      "MH  - Branchio-Oto-Renal Syndrome/*genetics\n",
      "MH  - Child, Preschool\n",
      "MH  - Ecuador\n",
      "MH  - Female\n",
      "MH  - Humans\n",
      "MH  - Intracellular Signaling Peptides and Proteins/*genetics\n",
      "MH  - Iran\n",
      "MH  - Male\n",
      "MH  - Mutation\n",
      "MH  - Nuclear Proteins/*genetics\n",
      "MH  - Pedigree\n",
      "MH  - Protein Tyrosine Phosphatases/*genetics\n",
      "MH  - Sequence Analysis, DNA\n",
      "MH  - Turkey\n",
      "MH  - United States\n",
      "PMC - PMC5516569\n",
      "MID - NIHMS873537\n",
      "OTO - NOTNLM\n",
      "OT  - Branchial arch anomalies\n",
      "OT  - Branchiootorenal syndrome\n",
      "OT  - EYA1\n",
      "OT  - Hearing loss\n",
      "OT  - Whole exome sequencing\n",
      "EDAT- 2017/06/07 06:00\n",
      "MHDA- 2017/07/19 06:00\n",
      "CRDT- 2017/06/07 06:00\n",
      "PHST- 2017/04/06 00:00 [received]\n",
      "PHST- 2017/04/16 00:00 [accepted]\n",
      "PHST- 2017/06/07 06:00 [entrez]\n",
      "PHST- 2017/06/07 06:00 [pubmed]\n",
      "PHST- 2017/07/19 06:00 [medline]\n",
      "AID - S0165-5876(17)30178-7 [pii]\n",
      "AID - 10.1016/j.ijporl.2017.04.037 [doi]\n",
      "PST - ppublish\n",
      "SO  - Int J Pediatr Otorhinolaryngol. 2017 Jul;98:59-63. doi:10.1016/j.ijporl.2017.04.037. Epub 2017 Apr 26.\n",
      "\n",
      "PMID- 28572335\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180117\n",
      "LR  - 20220129\n",
      "IS  - 1095-9203 (Electronic)\n",
      "IS  - 0036-8075 (Linking)\n",
      "VI  - 356\n",
      "IP  - 6341\n",
      "DP  - 2017 Jun 2\n",
      "TI  - A genetic signature of the evolution of loss of flight in the Galapagoscormorant.\n",
      "LID - eaal3345 [pii]\n",
      "LID - 10.1126/science.aal3345 [doi]\n",
      "AB  - We have a limited understanding of the genetic and molecular basis ofevolutionary changes in the size and proportion of limbs. We studied wing andpectoral skeleton reduction leading to flightlessness in the Galapagos cormorant (Phalacrocorax harrisi). We sequenced and de novo assembled the genomes of fourcormorant species and applied a predictive and comparative genomics approach tofind candidate variants that may have contributed to the evolution offlightlessness. These analyses and cross-species experiments in Caenorhabditiselegans and in chondrogenic cell lines implicated variants in genes necessary fortranscriptional regulation and function of the primary cilium. Cilia areessential for Hedgehog signaling, and humans affected by skeletal ciliopathiessuffer from premature bone growth arrest, mirroring skeletal features associated with loss of flight.\n",
      "CI  - Copyright (c) 2017, American Association for the Advancement of Science.\n",
      "FAU - Burga, Alejandro\n",
      "AU  - Burga A\n",
      "AUID- ORCID: 0000-0002-0211-4357\n",
      "AD  - Department of Human Genetics, Department of Biological Chemistry, and HowardHughes Medical Institute, University of California, Los Angeles, CA, USA.aburga@mednet.ucla.edu lkruglyak@mednet.ucla.edu.\n",
      "FAU - Wang, Weiguang\n",
      "AU  - Wang W\n",
      "AUID- ORCID: 0000-0002-8604-5044\n",
      "AD  - Departments of Molecular, Cell and Developmental Biology and Orthopaedic Surgery,University of California and Orthopaedic Institute for Children, Los Angeles, CA,USA.\n",
      "FAU - Ben-David, Eyal\n",
      "AU  - Ben-David E\n",
      "AUID- ORCID: 0000-0003-0514-0400\n",
      "AD  - Department of Human Genetics, Department of Biological Chemistry, and HowardHughes Medical Institute, University of California, Los Angeles, CA, USA.\n",
      "FAU - Wolf, Paul C\n",
      "AU  - Wolf PC\n",
      "AD  - Wildlife Services, U.S. Department of Agriculture, Roseburg, OR, USA.\n",
      "FAU - Ramey, Andrew M\n",
      "AU  - Ramey AM\n",
      "AUID- ORCID: 0000-0002-3601-8400\n",
      "AD  - U.S. Geological Survey Alaska Science Center, Anchorage, AK, USA.\n",
      "FAU - Verdugo, Claudio\n",
      "AU  - Verdugo C\n",
      "AUID- ORCID: 0000-0002-2866-260X\n",
      "AD  - Instituto de Patologia Animal, Facultad de Ciencias Veterinarias, UniversidadAustral de Chile, Valdivia, Chile.\n",
      "FAU - Lyons, Karen\n",
      "AU  - Lyons K\n",
      "AUID- ORCID: 0000-0001-9420-5813\n",
      "AD  - Departments of Molecular, Cell and Developmental Biology and Orthopaedic Surgery,University of California and Orthopaedic Institute for Children, Los Angeles, CA,USA.\n",
      "FAU - Parker, Patricia G\n",
      "AU  - Parker PG\n",
      "AUID- ORCID: 0000-0003-0843-835X\n",
      "AD  - Department of Biology and Whitney Harris World Ecology Center, University ofMissouri, St. Louis, MO, USA.\n",
      "AD  - WildCare Institute, Saint Louis Zoo, St. Louis, MO, USA.\n",
      "FAU - Kruglyak, Leonid\n",
      "AU  - Kruglyak L\n",
      "AUID- ORCID: 0000-0002-8065-3057\n",
      "AD  - Department of Human Genetics, Department of Biological Chemistry, and HowardHughes Medical Institute, University of California, Los Angeles, CA, USA.aburga@mednet.ucla.edu lkruglyak@mednet.ucla.edu.\n",
      "LA  - eng\n",
      "SI  - Dryad/10.5061/dryad.8m2t5\n",
      "GR  - HHMI/Howard Hughes Medical Institute/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PL  - United States\n",
      "TA  - Science\n",
      "JT  - Science (New York, N.Y.)\n",
      "JID - 0404511\n",
      "RN  - 0 (Homeodomain Proteins)\n",
      "SB  - IM\n",
      "CIN - Science. 2017 Jun 2;356(6341):904-905. PMID: 28572349\n",
      "MH  - Animals\n",
      "MH  - *Biological Evolution\n",
      "MH  - Birds/classification/*genetics/physiology\n",
      "MH  - Bone and Bones/anatomy & histology\n",
      "MH  - Caenorhabditis elegans/genetics\n",
      "MH  - Cell Differentiation/genetics\n",
      "MH  - Cell Line\n",
      "MH  - Chondrogenesis/genetics\n",
      "MH  - Cilia/genetics\n",
      "MH  - Ecuador\n",
      "MH  - Gene Expression Regulation/genetics\n",
      "MH  - Genetic Variation\n",
      "MH  - Genome/*genetics\n",
      "MH  - Homeodomain Proteins/genetics\n",
      "MH  - Mutation\n",
      "MH  - Phylogeny\n",
      "MH  - Wings, Animal/anatomy & histology/*physiology\n",
      "PMC - PMC5567675\n",
      "MID - HHMIMS895192\n",
      "EDAT- 2017/06/03 06:00\n",
      "MHDA- 2018/01/18 06:00\n",
      "CRDT- 2017/06/03 06:00\n",
      "PHST- 2016/11/04 00:00 [received]\n",
      "PHST- 2017/04/13 00:00 [accepted]\n",
      "PHST- 2017/06/03 06:00 [entrez]\n",
      "PHST- 2017/06/03 06:00 [pubmed]\n",
      "PHST- 2018/01/18 06:00 [medline]\n",
      "AID - 356/6341/eaal3345 [pii]\n",
      "AID - 10.1126/science.aal3345 [doi]\n",
      "PST - ppublish\n",
      "SO  - Science. 2017 Jun 2;356(6341). pii: 356/6341/eaal3345. doi:10.1126/science.aal3345.\n",
      "\n",
      "PMID- 28475225\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20171220\n",
      "LR  - 20171220\n",
      "IS  - 1365-294X (Electronic)\n",
      "IS  - 0962-1083 (Linking)\n",
      "VI  - 26\n",
      "IP  - 19\n",
      "DP  - 2017 Oct\n",
      "TI  - The adaptive genomic landscape of beak morphology in Darwin's finches.\n",
      "PG  - 4978-4989\n",
      "LID - 10.1111/mec.14166 [doi]\n",
      "AB  - Beak shape in Darwin's ground finches (Geospiza) is emblematic of naturalselection and adaptive radiation, yet our understanding of the genetic basis ofbeak shape variation, and thus the genetic target of natural selection, is still evolving. Here we reveal the genomic architecture of beak shape variation usinggenomewide comparisons of four closely related and hybridizing species across 13 islands subject to parallel natural selection. Pairwise contrasts among specieswere used to identify a large number of genomic loci that are consistentlyrelated to species differences across a complex landscape. These loci areassociated with hundreds of genes that have enriched GO categories significantly associated with development. One genomic region of particular interest is asection of Chromosome 1A with many candidate genes and increased linkage. Thedistinct, pointed beak shape of the cactus finch is linked to an excess ofintermediate frequency alleles and increased heterozygosity in significant SNPs, but not across the rest of the genome. Alleles associated with pointier beaksamong species were associated with pointier-beaked populations within eachspecies, thus establishing a common basis for natural selection, speciesdivergence and adaptive radiation. The adaptive genomic landscape for Darwin'sfinches mirrors theoretical expectations based on morphological variation. Theimplication that a large number of genes are actively maintained to facilitatebeak variation across parallel populations with documented interspecies admixturechallenges our understanding of evolutionary processes in the wild.\n",
      "CI  - (c) 2017 John Wiley & Sons Ltd.\n",
      "FAU - Lawson, Lucinda P\n",
      "AU  - Lawson LP\n",
      "AUID- ORCID: http://orcid.org/0000-0003-3939-7829\n",
      "AD  - Department of Biological Sciences, University of Cincinnati, Cincinnati, OH, USA.\n",
      "FAU - Petren, Kenneth\n",
      "AU  - Petren K\n",
      "AD  - Department of Biological Sciences, University of Cincinnati, Cincinnati, OH, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20170522\n",
      "PL  - England\n",
      "TA  - Mol Ecol\n",
      "JT  - Molecular ecology\n",
      "JID - 9214478\n",
      "SB  - IM\n",
      "MH  - Adaptation, Biological/genetics\n",
      "MH  - Alleles\n",
      "MH  - Animals\n",
      "MH  - Beak/*anatomy & histology\n",
      "MH  - Ecuador\n",
      "MH  - Finches/*genetics\n",
      "MH  - Genetic Linkage\n",
      "MH  - Genetic Loci\n",
      "MH  - Genetics, Population\n",
      "MH  - Hybridization, Genetic\n",
      "MH  - Islands\n",
      "MH  - Linkage Disequilibrium\n",
      "MH  - Polymorphism, Single Nucleotide\n",
      "MH  - *Selection, Genetic\n",
      "OTO - NOTNLM\n",
      "OT  - Geospiza\n",
      "OT  - Galapagos\n",
      "OT  - population genomics\n",
      "OT  - quantitative traits\n",
      "EDAT- 2017/05/06 06:00\n",
      "MHDA- 2017/12/21 06:00\n",
      "CRDT- 2017/05/06 06:00\n",
      "PHST- 2015/10/27 00:00 [received]\n",
      "PHST- 2017/04/26 00:00 [revised]\n",
      "PHST- 2017/04/27 00:00 [accepted]\n",
      "PHST- 2017/05/06 06:00 [pubmed]\n",
      "PHST- 2017/12/21 06:00 [medline]\n",
      "PHST- 2017/05/06 06:00 [entrez]\n",
      "AID - 10.1111/mec.14166 [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Ecol. 2017 Oct;26(19):4978-4989. doi: 10.1111/mec.14166. Epub 2017 May 22.\n",
      "\n",
      "PMID- 28442558\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180309\n",
      "LR  - 20181113\n",
      "IS  - 1549-5469 (Electronic)\n",
      "IS  - 1088-9051 (Linking)\n",
      "VI  - 27\n",
      "IP  - 6\n",
      "DP  - 2017 Jun\n",
      "TI  - Gene flow, ancient polymorphism, and ecological adaptation shape the genomiclandscape of divergence among Darwin's finches.\n",
      "PG  - 1004-1015\n",
      "LID - 10.1101/gr.212522.116 [doi]\n",
      "AB  - Genomic comparisons of closely related species have identified \"islands\" oflocally elevated sequence divergence. Genomic islands may contain functionalvariants involved in local adaptation or reproductive isolation and may thereforeplay an important role in the speciation process. However, genomic islands canalso arise through evolutionary processes unrelated to speciation, andexamination of their properties can illuminate how new species evolve. Here, weperformed scans for regions of high relative divergence (FST) in 12 species pairsof Darwin's finches at different genetic distances. In each pair, we identifygenomic islands that are, on average, elevated in both relative divergence (FST) and absolute divergence (dXY). This signal indicates that haplotypes within thesegenomic regions became isolated from each other earlier than the rest of thegenome. Interestingly, similar numbers of genomic islands of elevated dXY areobserved in sympatric and allopatric species pairs, suggesting that recent geneflow is not a major factor in their formation. We find that two of the mostpronounced genomic islands contain the ALX1 and HMGA2 loci, which are associated with variation in beak shape and size, respectively, suggesting that they areinvolved in ecological adaptation. A subset of genomic island regions, including these loci, appears to represent anciently diverged haplotypes that evolved earlyduring the radiation of Darwin's finches. Comparative genomics data indicate thatthese loci, and genomic islands in general, have exceptionally low recombination rates, which may play a role in their establishment.\n",
      "CI  - (c) 2017 Han et al.; Published by Cold Spring Harbor Laboratory Press.\n",
      "FAU - Han, Fan\n",
      "AU  - Han F\n",
      "AD  - Department of Medical Biochemistry and Microbiology, Science for Life Laboratory,Uppsala University, 75123 Uppsala, Sweden.\n",
      "FAU - Lamichhaney, Sangeet\n",
      "AU  - Lamichhaney S\n",
      "AD  - Department of Medical Biochemistry and Microbiology, Science for Life Laboratory,Uppsala University, 75123 Uppsala, Sweden.\n",
      "FAU - Grant, B Rosemary\n",
      "AU  - Grant BR\n",
      "AD  - Department of Ecology and Evolutionary Biology, Princeton University, Princeton, New Jersey 08544-2016, USA.\n",
      "FAU - Grant, Peter R\n",
      "AU  - Grant PR\n",
      "AD  - Department of Ecology and Evolutionary Biology, Princeton University, Princeton, New Jersey 08544-2016, USA.\n",
      "FAU - Andersson, Leif\n",
      "AU  - Andersson L\n",
      "AD  - Department of Medical Biochemistry and Microbiology, Science for Life Laboratory,Uppsala University, 75123 Uppsala, Sweden.\n",
      "AD  - Department of Animal Breeding and Genetics, Swedish University of AgriculturalSciences, 75007 Uppsala, Sweden.\n",
      "AD  - Department of Veterinary Integrative Biosciences, Texas A&M University, CollegeStation, Texas 77843-4461, USA.\n",
      "FAU - Webster, Matthew T\n",
      "AU  - Webster MT\n",
      "AD  - Department of Medical Biochemistry and Microbiology, Science for Life Laboratory,Uppsala University, 75123 Uppsala, Sweden.\n",
      "LA  - eng\n",
      "SI  - Dryad/10.5061/dryad.7155d\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20170425\n",
      "PL  - United States\n",
      "TA  - Genome Res\n",
      "JT  - Genome research\n",
      "JID - 9518021\n",
      "SB  - IM\n",
      "MH  - Adaptation, Physiological/genetics\n",
      "MH  - Animal Distribution\n",
      "MH  - Animals\n",
      "MH  - Ecuador\n",
      "MH  - Finches/classification/*genetics\n",
      "MH  - *Gene Flow\n",
      "MH  - Genetic Loci\n",
      "MH  - *Genetic Speciation\n",
      "MH  - *Genome\n",
      "MH  - Genomic Islands\n",
      "MH  - Haplotypes\n",
      "MH  - Microsatellite Repeats\n",
      "MH  - *Phylogeny\n",
      "MH  - Polymorphism, Genetic\n",
      "MH  - Selection, Genetic\n",
      "MH  - Sympatry\n",
      "PMC - PMC5453315\n",
      "EDAT- 2017/04/27 06:00\n",
      "MHDA- 2018/03/10 06:00\n",
      "CRDT- 2017/04/27 06:00\n",
      "PHST- 2016/07/08 00:00 [received]\n",
      "PHST- 2017/02/14 00:00 [accepted]\n",
      "PHST- 2017/04/27 06:00 [pubmed]\n",
      "PHST- 2018/03/10 06:00 [medline]\n",
      "PHST- 2017/04/27 06:00 [entrez]\n",
      "AID - gr.212522.116 [pii]\n",
      "AID - 10.1101/gr.212522.116 [doi]\n",
      "PST - ppublish\n",
      "SO  - Genome Res. 2017 Jun;27(6):1004-1015. doi: 10.1101/gr.212522.116. Epub 2017 Apr25.\n",
      "\n",
      "PMID- 28367073\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20200930\n",
      "IS  - 1389-2029 (Print)\n",
      "IS  - 1389-2029 (Linking)\n",
      "VI  - 18\n",
      "IP  - 2\n",
      "DP  - 2017 Apr\n",
      "TI  - Natural Compounds As Modulators of Non-apoptotic Cell Death in Cancer Cells.\n",
      "PG  - 132-155\n",
      "LID - 10.2174/1389202917666160803150639 [doi]\n",
      "AB  - Cell death is an innate capability of cells to be removed from microenvironment, if and when they are damaged by multiple stresses. Cell death is often regulated by multiple molecular pathways and mechanism, including apoptosis, autophagy, andnecroptosis. The molecular network underlying these processes is oftenintertwined and one pathway can dynamically shift to another one acquiringcertain protein components, in particular upon treatment with various drugs. The strategy to treat human cancer ultimately relies on the ability of anticancertherapeutics to induce tumor-specific cell death, while leaving normal adjacentcells undamaged. However, tumor cells often develop the resistance to thedrug-induced cell death, thus representing a great challenge for the anticancerapproaches. Numerous compounds originated from the natural sources andbiopharmaceutical industries are applied today in clinics showing advantageousresults. However, some exhibit serious toxic side effects. Thus, novel effective therapeutic approaches in treating cancers are continued to be developed. Naturalcompounds with anticancer activity have gained a great interest among researchersand clinicians alike since they have shown more favorable safety and efficacythen the synthetic marketed drugs. Numerous studies in vitro and in vivo havefound that several natural compounds display promising anticancer potentials.This review underlines certain information regarding the role of naturalcompounds from plants, microorganisms and sea life forms, which are able toinduce non-apoptotic cell death in tumor cells, namely autophagy and necroptosis.\n",
      "FAU - Guaman-Ortiz, Luis Miguel\n",
      "AU  - Guaman-Ortiz LM\n",
      "AD  - 1 Departamento de Ciencias de la Salud, Universidad Tecnica Particular de Loja,Loja, Ecuador ; 2 Head and Neck Cancer Research Division, Johns HopkinsUniversity School of Medicine, Baltimore, MD, USA.\n",
      "FAU - Orellana, Maria Isabel Ramirez\n",
      "AU  - Orellana MI\n",
      "AD  - 1 Departamento de Ciencias de la Salud, Universidad Tecnica Particular de Loja,Loja, Ecuador ; 2 Head and Neck Cancer Research Division, Johns HopkinsUniversity School of Medicine, Baltimore, MD, USA.\n",
      "FAU - Ratovitski, Edward A\n",
      "AU  - Ratovitski EA\n",
      "AD  - 1 Departamento de Ciencias de la Salud, Universidad Tecnica Particular de Loja,Loja, Ecuador ; 2 Head and Neck Cancer Research Division, Johns HopkinsUniversity School of Medicine, Baltimore, MD, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PL  - United Arab Emirates\n",
      "TA  - Curr Genomics\n",
      "JT  - Current genomics\n",
      "JID - 100960527\n",
      "PMC - PMC5345338\n",
      "OTO - NOTNLM\n",
      "OT  - Autophagy\n",
      "OT  - Cancer\n",
      "OT  - Natural compounds\n",
      "OT  - Necroptosis\n",
      "OT  - Paraptosis\n",
      "EDAT- 2017/04/04 06:00\n",
      "MHDA- 2017/04/04 06:01\n",
      "CRDT- 2017/04/04 06:00\n",
      "PHST- 2015/10/27 00:00 [received]\n",
      "PHST- 2015/11/24 00:00 [revised]\n",
      "PHST- 2015/11/28 00:00 [accepted]\n",
      "PHST- 2017/04/04 06:00 [entrez]\n",
      "PHST- 2017/04/04 06:00 [pubmed]\n",
      "PHST- 2017/04/04 06:01 [medline]\n",
      "AID - 10.2174/1389202917666160803150639 [doi]\n",
      "AID - CG-18-132 [pii]\n",
      "PST - ppublish\n",
      "SO  - Curr Genomics. 2017 Apr;18(2):132-155. doi: 10.2174/1389202917666160803150639.\n",
      "\n",
      "PMID- 28367072\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20200930\n",
      "IS  - 1389-2029 (Print)\n",
      "IS  - 1389-2029 (Linking)\n",
      "VI  - 18\n",
      "IP  - 2\n",
      "DP  - 2017 Apr\n",
      "TI  - Natural Compounds as Modulators of Cell Cycle Arrest: Application for Anticancer Chemotherapies.\n",
      "PG  - 106-131\n",
      "LID - 10.2174/1389202917666160808125645 [doi]\n",
      "AB  - Natural compounds from various plants, microorganisms and marine species play an important role in the discovery novel components that can be successfully used innumerous biomedical applications, including anticancer therapeutics. Sinceuncontrolled and rapid cell division is a hallmark of cancer, unraveling themolecular mechanisms underlying mitosis is key to understanding how variousnatural compounds might function as inhibitors of cell cycle progression. Anumber of natural compounds that inhibit the cell cycle arrest have proveneffective for killing cancer cells in vitro, in vivo and in clinical settings.Significant advances that have been recently made in the understanding ofmolecular mechanisms underlying the cell cycle regulation using thechemotherapeutic agents is of great importance for improving the efficacy oftargeted therapeutics and overcoming resistance to anticancer drugs, especiallyof natural origin, which inhibit the activities of cyclins and cyclin-dependentkinases, as well as other proteins and enzymes involved in proper regulation ofcell cycle leading to controlled cell proliferation.\n",
      "FAU - Bailon-Moscoso, Natalia\n",
      "AU  - Bailon-Moscoso N\n",
      "AD  - Departamento de Ciencias de la Salud.\n",
      "FAU - Cevallos-Solorzano, Gabriela\n",
      "AU  - Cevallos-Solorzano G\n",
      "AD  - Departamento de Ciencias Naturales.\n",
      "FAU - Romero-Benavides, Juan Carlos\n",
      "AU  - Romero-Benavides JC\n",
      "AD  - Departamento de Quimica, Universidad Tecnica Particular de Loja, Loja, Ecuador.\n",
      "FAU - Orellana, Maria Isabel Ramirez\n",
      "AU  - Orellana MI\n",
      "AD  - Departamento de Ciencias de la Salud.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "PL  - United Arab Emirates\n",
      "TA  - Curr Genomics\n",
      "JT  - Current genomics\n",
      "JID - 100960527\n",
      "PMC - PMC5345333\n",
      "OTO - NOTNLM\n",
      "OT  - Cancer\n",
      "OT  - Cell cycle arrest\n",
      "OT  - Mitosis\n",
      "OT  - Natural compounds\n",
      "EDAT- 2017/04/04 06:00\n",
      "MHDA- 2017/04/04 06:01\n",
      "CRDT- 2017/04/04 06:00\n",
      "PHST- 2015/10/16 00:00 [received]\n",
      "PHST- 2015/11/13 00:00 [revised]\n",
      "PHST- 2015/11/20 00:00 [accepted]\n",
      "PHST- 2017/04/04 06:00 [entrez]\n",
      "PHST- 2017/04/04 06:00 [pubmed]\n",
      "PHST- 2017/04/04 06:01 [medline]\n",
      "AID - 10.2174/1389202917666160808125645 [doi]\n",
      "AID - CG-18-106 [pii]\n",
      "PST - ppublish\n",
      "SO  - Curr Genomics. 2017 Apr;18(2):106-131. doi: 10.2174/1389202917666160808125645.\n",
      "\n",
      "PMID- 28318380\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180212\n",
      "LR  - 20181202\n",
      "IS  - 1943-3700 (Electronic)\n",
      "IS  - 0090-3558 (Linking)\n",
      "VI  - 53\n",
      "IP  - 3\n",
      "DP  - 2017 Jul\n",
      "TI  - PUP MORTALITY AND EVIDENCE FOR PATHOGEN EXPOSURE IN GALAPAGOS SEA LIONS (ZALOPHUSWOLLEBAEKI) ON SAN CRISTOBAL ISLAND, GALAPAGOS, ECUADOR.\n",
      "PG  - 491-498\n",
      "LID - 10.7589/2016-05-092 [doi]\n",
      "AB  - The Galapagos sea lion ( Zalophus wollebaeki ), an endangered species,experiences high pup mortality (up to 100%) in years when El Nino events reducefood supply in the Galapagos Islands. Mortality of pups in non-El Nino years isestimated to be 5% in undisturbed colonies. From 2009 to 2012 we observed highpup mortality (up to 67%) in colonies close to the Galapagos capital, PuertoBaquerizo Moreno, where contact with humans, domestic animals, and rats isfrequent. Gross postmortem findings from 54 pups included hemorrhagic lesions in liver and congestion in lungs; histopathology suggested a possible associationwith infectious diseases. Evidence of Leptospira infection was found in five out of seven samples collected in 2010. Canine distemper viral (CDV) RNA was detectedin tissues from six sea lions (in 2011-12), four of which were confirmed bynucleotide sequencing. The absence of CDV antibodies in 109 juvenile animalstested in 2014 at urban and remote colonies could indicate that the CDV infectionobserved in 2011 was likely confined to a few animals. Our results indicated thatGalapagos sea lions have been exposed at least to two pathogens, Leptospira andCDV; however, the impact of these infections on the sea lions is unclear.\n",
      "FAU - Denkinger, Judith\n",
      "AU  - Denkinger J\n",
      "AD  - 1 Colegio de Ciencias Biologicas y Ambientales y Galapagos Science Center,Universidad San Francisco de Quito, Campus Cumbaya, Quito, Ecuador.\n",
      "FAU - Guevara, Nataly\n",
      "AU  - Guevara N\n",
      "AD  - 1 Colegio de Ciencias Biologicas y Ambientales y Galapagos Science Center,Universidad San Francisco de Quito, Campus Cumbaya, Quito, Ecuador.\n",
      "FAU - Ayala, Sofia\n",
      "AU  - Ayala S\n",
      "AD  - 1 Colegio de Ciencias Biologicas y Ambientales y Galapagos Science Center,Universidad San Francisco de Quito, Campus Cumbaya, Quito, Ecuador.\n",
      "FAU - Murillo, Juan Carlos\n",
      "AU  - Murillo JC\n",
      "AD  - 2 Universidad de la Peninsula Santa Elena, Salinas, Ecuador.\n",
      "FAU - Hirschfeld, Maximilian\n",
      "AU  - Hirschfeld M\n",
      "AD  - 1 Colegio de Ciencias Biologicas y Ambientales y Galapagos Science Center,Universidad San Francisco de Quito, Campus Cumbaya, Quito, Ecuador.\n",
      "FAU - Montero-Serra, Ignasi\n",
      "AU  - Montero-Serra I\n",
      "AD  - 1 Colegio de Ciencias Biologicas y Ambientales y Galapagos Science Center,Universidad San Francisco de Quito, Campus Cumbaya, Quito, Ecuador.\n",
      "AD  - 3 Departament d'Ecologia, Universitat de Barcelona, 08028 Barcelona, Spain.\n",
      "FAU - Fietz, Katharina\n",
      "AU  - Fietz K\n",
      "AD  - 1 Colegio de Ciencias Biologicas y Ambientales y Galapagos Science Center,Universidad San Francisco de Quito, Campus Cumbaya, Quito, Ecuador.\n",
      "AD  - 4 University of Hamburg, Zoological Museum and Biocenter Grindel,Martin-Luther-King Platz 3, 20146 Hamburg, Germany.\n",
      "FAU - Goldstein, Tracey\n",
      "AU  - Goldstein T\n",
      "AD  - 5 Wildlife Health Center, School of Veterinary Medicine, 1089 Veterinary MedicineDrive, University of California, Davis, California 95616, USA.\n",
      "FAU - Ackermann, Mark\n",
      "AU  - Ackermann M\n",
      "AD  - 6 Department of Veterinary Pathology, Iowa State University, 1800 ChristensenDrive, Ames, Iowa 50011, USA.\n",
      "FAU - Barragan, Veronica\n",
      "AU  - Barragan V\n",
      "AD  - 1 Colegio de Ciencias Biologicas y Ambientales y Galapagos Science Center,Universidad San Francisco de Quito, Campus Cumbaya, Quito, Ecuador.\n",
      "AD  - 9 The Center of Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona 86011, USA.\n",
      "FAU - Cabrera, Francisco\n",
      "AU  - Cabrera F\n",
      "AD  - 7 Universidad San Francisco de Quito, Colegio de Veterinaria, Campus Cumbaya,Quito, Ecuador.\n",
      "FAU - Chavez, Cristina\n",
      "AU  - Chavez C\n",
      "AD  - 1 Colegio de Ciencias Biologicas y Ambientales y Galapagos Science Center,Universidad San Francisco de Quito, Campus Cumbaya, Quito, Ecuador.\n",
      "FAU - Dubovi, Edward J\n",
      "AU  - Dubovi EJ\n",
      "AD  - 8 Animal Health Diagnostic Center, College of Veterinary Medicine, CornellUniversity, PO Box 5786, Ithaca, New York 14850, USA.\n",
      "FAU - Martinez, Jael\n",
      "AU  - Martinez J\n",
      "AD  - 1 Colegio de Ciencias Biologicas y Ambientales y Galapagos Science Center,Universidad San Francisco de Quito, Campus Cumbaya, Quito, Ecuador.\n",
      "FAU - Trueba, Gabriel\n",
      "AU  - Trueba G\n",
      "AD  - 1 Colegio de Ciencias Biologicas y Ambientales y Galapagos Science Center,Universidad San Francisco de Quito, Campus Cumbaya, Quito, Ecuador.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20170320\n",
      "PL  - United States\n",
      "TA  - J Wildl Dis\n",
      "JT  - Journal of wildlife diseases\n",
      "JID - 0244160\n",
      "RN  - 0 (Cristobal)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Ceramics\n",
      "MH  - Distemper Virus, Canine/*isolation & purification\n",
      "MH  - Ecuador\n",
      "MH  - Endangered Species\n",
      "MH  - Islands\n",
      "MH  - Leptospira/*isolation & purification\n",
      "MH  - Rats\n",
      "MH  - Sea Lions/microbiology/*virology\n",
      "OTO - NOTNLM\n",
      "OT  - *Canine distemper virus\n",
      "OT  - *Galapagos sea lion\n",
      "OT  - *Zalophus wollebaeki\n",
      "OT  - *leptospirosis\n",
      "OT  - *pup mortality\n",
      "EDAT- 2017/03/21 06:00\n",
      "MHDA- 2018/02/13 06:00\n",
      "CRDT- 2017/03/21 06:00\n",
      "PHST- 2017/03/21 06:00 [pubmed]\n",
      "PHST- 2018/02/13 06:00 [medline]\n",
      "PHST- 2017/03/21 06:00 [entrez]\n",
      "AID - 10.7589/2016-05-092 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Wildl Dis. 2017 Jul;53(3):491-498. doi: 10.7589/2016-05-092. Epub 2017 Mar 20.\n",
      "\n",
      "PMID- 28274840\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180404\n",
      "LR  - 20181113\n",
      "IS  - 1878-5832 (Electronic)\n",
      "IS  - 1359-6446 (Linking)\n",
      "VI  - 22\n",
      "IP  - 7\n",
      "DP  - 2017 Jul\n",
      "TI  - Systemic QSAR and phenotypic virtual screening: chasing butterflies in drugdiscovery.\n",
      "PG  - 994-1007\n",
      "LID - S1359-6446(17)30107-1 [pii]\n",
      "LID - 10.1016/j.drudis.2017.02.004 [doi]\n",
      "AB  - Current advances in systems biology suggest a new change of paradigm reinforcing the holistic nature of the drug discovery process. According to the principles ofsystems biology, a simple drug perturbing a network of targets can triggercomplex reactions. Therefore, it is possible to connect initial events with finaloutcomes and consequently prioritize those events, leading to a desired effect.Here, we introduce a new concept, 'Systemic Chemogenomics/QuantitativeStructure-Activity Relationship (QSAR)'. To elaborate on the concept, relevantinformation surrounding it is addressed. The concept is challenged byimplementing a systemic QSAR approach for phenotypic virtual screening (VS) ofcandidate ligands acting as neuroprotective agents in Parkinson's disease (PD).The results support the suitability of the approach for the phenotypicprioritization of drug candidates.\n",
      "CI  - Copyright (c) 2017 Elsevier Ltd. All rights reserved.\n",
      "FAU - Cruz-Monteagudo, Maykel\n",
      "AU  - Cruz-Monteagudo M\n",
      "AD  - CIQUP/Departamento de Quimica e Bioquimica, Faculdade de Ciencias, Universidadedo Porto, Porto 4169-007, Portugal. Electronic address: gmailkelcm@yahoo.es.\n",
      "FAU - Schurer, Stephan\n",
      "AU  - Schurer S\n",
      "AD  - Department of Pharmacology, Miller School of Medicine and Center forComputational Science, University of Miami, Miami, FL 33136, USA.\n",
      "FAU - Tejera, Eduardo\n",
      "AU  - Tejera E\n",
      "AD  - Instituto de Investigaciones Biomedicas (IIB), Universidad de Las Americas,170513 Quito, Ecuador.\n",
      "FAU - Perez-Castillo, Yunierkis\n",
      "AU  - Perez-Castillo Y\n",
      "AD  - Seccion Fisico Quimica y Matematicas, Departamento de Quimica, UniversidadTecnica Particular de Loja, San Cayetano Alto S/N, EC1101608 Loja, Ecuador.\n",
      "FAU - Medina-Franco, Jose L\n",
      "AU  - Medina-Franco JL\n",
      "AD  - Universidad Nacional Autonoma de Mexico, Departamento de Farmacia, Facultad deQuimica, Avenida Universidad 3000, Mexico City, 04510, Mexico.\n",
      "FAU - Sanchez-Rodriguez, Aminael\n",
      "AU  - Sanchez-Rodriguez A\n",
      "AD  - Departamento de Ciencias Naturales, Universidad Tecnica Particular de Loja, CalleParis S/N, EC1101608 Loja, Ecuador.\n",
      "FAU - Borges, Fernanda\n",
      "AU  - Borges F\n",
      "AD  - CIQUP/Departamento de Quimica e Bioquimica, Faculdade de Ciencias, Universidadedo Porto, Porto 4169-007, Portugal. Electronic address: fborges@fc.up.pt.\n",
      "LA  - eng\n",
      "GR  - U54 CA189205/CA/NCI NIH HHS/United States\n",
      "GR  - U54 HL127624/HL/NHLBI NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20170306\n",
      "PL  - England\n",
      "TA  - Drug Discov Today\n",
      "JT  - Drug discovery today\n",
      "JID - 9604391\n",
      "SB  - IM\n",
      "MH  - *Drug Discovery\n",
      "MH  - Genomics\n",
      "MH  - Humans\n",
      "MH  - Phenotype\n",
      "MH  - *Quantitative Structure-Activity Relationship\n",
      "PMC - PMC5487293\n",
      "MID - NIHMS857937\n",
      "EDAT- 2017/03/10 06:00\n",
      "MHDA- 2018/04/05 06:00\n",
      "CRDT- 2017/03/10 06:00\n",
      "PHST- 2016/11/16 00:00 [received]\n",
      "PHST- 2017/02/02 00:00 [revised]\n",
      "PHST- 2017/02/27 00:00 [accepted]\n",
      "PHST- 2017/03/10 06:00 [pubmed]\n",
      "PHST- 2018/04/05 06:00 [medline]\n",
      "PHST- 2017/03/10 06:00 [entrez]\n",
      "AID - S1359-6446(17)30107-1 [pii]\n",
      "AID - 10.1016/j.drudis.2017.02.004 [doi]\n",
      "PST - ppublish\n",
      "SO  - Drug Discov Today. 2017 Jul;22(7):994-1007. doi: 10.1016/j.drudis.2017.02.004.Epub 2017 Mar 6.\n",
      "\n",
      "PMID- 28239602\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "LR  - 20200930\n",
      "IS  - 2296-2360 (Print)\n",
      "IS  - 2296-2360 (Linking)\n",
      "VI  - 5\n",
      "DP  - 2017\n",
      "TI  - First Case of CD40LG Deficiency in Ecuador, Diagnosed after Whole ExomeSequencing in a Patient with Severe Cutaneous Histoplasmosis.\n",
      "PG  - 17\n",
      "LID - 10.3389/fped.2017.00017 [doi]\n",
      "AB  - Severe infections with Histoplasma capsulatum are commonly observed in patientwith secondary immunodeficiency disorders. We report a two and a half years oldboy previously healthy with disseminated cutaneous histoplasmosis. Using wholeexome sequencing, we found an indel mutation at the CD40LG gene, suggesting adiagnosis of hyper-IgM (HIGM) syndrome, even in the absence of the usual featuresfor the disease. Interestingly, the patient lives in a region endemic forhistoplasmosis. The unusual infections in our case suggest that in children with severe histoplasmosis and resident in endemic areas, HIGM syndrome should beconsidered as a diagnosis.\n",
      "FAU - Pedroza, Luis Alberto\n",
      "AU  - Pedroza LA\n",
      "AD  - Colegio de ciencias de la salud-Hospital de los Valles, Universidad San Franciscode Quito, Quito, Ecuador; Instituto de Microbiologia, Universidad San Franciscode Quito, Quito, Ecuador.\n",
      "FAU - Guerrero, Nina\n",
      "AU  - Guerrero N\n",
      "AD  - Colegio de ciencias de la salud-Hospital de los Valles, Universidad San Franciscode Quito , Quito , Ecuador.\n",
      "FAU - Stray-Pedersen, Asbjorg\n",
      "AU  - Stray-Pedersen A\n",
      "AD  - Norwegian National Unit for Newborn Screening, Oslo University Hospital, Oslo,Norway; Department of Pediatrics, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Tafur, Cristina\n",
      "AU  - Tafur C\n",
      "AD  - Hospital Pediatrico Baca Ortiz , Quito , Ecuador.\n",
      "FAU - Macias, Roque\n",
      "AU  - Macias R\n",
      "AD  - Hospital Pediatrico Baca Ortiz , Quito , Ecuador.\n",
      "FAU - Munoz, Greta\n",
      "AU  - Munoz G\n",
      "AD  - Hospital Pediatrico Baca Ortiz , Quito , Ecuador.\n",
      "FAU - Akdemir, Zeynep Coban\n",
      "AU  - Akdemir ZC\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics, Department of Molecular and HumanGenetics, Baylor College of Medicine , Houston, TX , USA.\n",
      "FAU - Jhangiani, Shalini N\n",
      "AU  - Jhangiani SN\n",
      "AD  - Department of Molecular and Human Genetics, Baylor College of Medicine , Houston,TX , USA.\n",
      "FAU - Watkin, Levi B\n",
      "AU  - Watkin LB\n",
      "AD  - Center for Human Immunobiology, Texas Children's Hospital, Department ofPediatrics, Baylor College of Medicine, Houston, TX, USA; Department ofPediatrics, Section of Immunology, Allergy, and Rheumatology, Baylor College ofMedicine, Texas Children's Hospital, Houston, TX, USA.\n",
      "FAU - Chinn, Ivan K\n",
      "AU  - Chinn IK\n",
      "AD  - Center for Human Immunobiology, Texas Children's Hospital, Department ofPediatrics, Baylor College of Medicine, Houston, TX, USA; Department ofPediatrics, Section of Immunology, Allergy, and Rheumatology, Baylor College ofMedicine, Texas Children's Hospital, Houston, TX, USA.\n",
      "FAU - Lupski, James R\n",
      "AU  - Lupski JR\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics, Department of Molecular and HumanGenetics, Baylor College of Medicine, Houston, TX, USA; Department of Molecularand Human Genetics, Baylor College of Medicine, Houston, TX, USA; Human GenomeSequencing Center, Baylor College of Medicine, Houston, TX, USA; Department ofPediatrics, Baylor College of Medicine, Texas Children's Hospital, Houston TX,USA.\n",
      "FAU - Orange, Jordan S\n",
      "AU  - Orange JS\n",
      "AD  - Center for Human Immunobiology, Texas Children's Hospital, Department ofPediatrics, Baylor College of Medicine, Houston, TX, USA; Department ofPediatrics, Section of Immunology, Allergy, and Rheumatology, Baylor College ofMedicine, Texas Children's Hospital, Houston, TX, USA.\n",
      "LA  - eng\n",
      "GR  - UM1 HG006542/HG/NHGRI NIH HHS/United States\n",
      "PT  - Case Reports\n",
      "DEP - 20170210\n",
      "PL  - Switzerland\n",
      "TA  - Front Pediatr\n",
      "JT  - Frontiers in pediatrics\n",
      "JID - 101615492\n",
      "PMC - PMC5300990\n",
      "OTO - NOTNLM\n",
      "OT  - CD40LG\n",
      "OT  - histoplasmosis\n",
      "OT  - hyper-IgM syndrome\n",
      "OT  - primary immunodeficiency diseases\n",
      "OT  - whole exome sequencing\n",
      "EDAT- 2017/02/28 06:00\n",
      "MHDA- 2017/02/28 06:01\n",
      "CRDT- 2017/02/28 06:00\n",
      "PHST- 2016/10/20 00:00 [received]\n",
      "PHST- 2017/01/23 00:00 [accepted]\n",
      "PHST- 2017/02/28 06:00 [entrez]\n",
      "PHST- 2017/02/28 06:00 [pubmed]\n",
      "PHST- 2017/02/28 06:01 [medline]\n",
      "AID - 10.3389/fped.2017.00017 [doi]\n",
      "PST - epublish\n",
      "SO  - Front Pediatr. 2017 Feb 10;5:17. doi: 10.3389/fped.2017.00017. eCollection 2017.\n",
      "\n",
      "PMID- 28105701\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180626\n",
      "LR  - 20180626\n",
      "IS  - 1469-185X (Electronic)\n",
      "IS  - 0006-3231 (Linking)\n",
      "VI  - 92\n",
      "IP  - 4\n",
      "DP  - 2017 Nov\n",
      "TI  - Climate change and alpine stream biology: progress, challenges, and opportunitiesfor the future.\n",
      "PG  - 2024-2045\n",
      "LID - 10.1111/brv.12319 [doi]\n",
      "AB  - In alpine regions worldwide, climate change is dramatically altering ecosystemsand affecting biodiversity in many ways. For streams, receding alpine glaciersand snowfields, paired with altered precipitation regimes, are driving shifts in hydrology, species distributions, basal resources, and threatening the veryexistence of some habitats and biota. Alpine streams harbour substantial species and genetic diversity due to significant habitat insularity and environmentalheterogeneity. Climate change is expected to affect alpine stream biodiversityacross many levels of biological resolution from micro- to macroscopic organisms and genes to communities. Herein, we describe the current state of alpine stream biology from an organism-focused perspective. We begin by reviewing sevenstandard and emerging approaches that combine to form the current state of thediscipline. We follow with a call for increased synthesis across existingapproaches to improve understanding of how these imperiled ecosystems areresponding to rapid environmental change. We then take a forward-lookingviewpoint on how alpine stream biologists can make better use of existing datasets through temporal comparisons, integrate remote sensing and geographicinformation system (GIS) technologies, and apply genomic tools to refineknowledge of underlying evolutionary processes. We conclude with comments aboutthe future of biodiversity conservation in alpine streams to confront thedaunting challenge of mitigating the effects of rapid environmental change inthese sentinel ecosystems.\n",
      "CI  - (c) 2017 Cambridge Philosophical Society.\n",
      "FAU - Hotaling, Scott\n",
      "AU  - Hotaling S\n",
      "AD  - Department of Biology, University of Kentucky, Lexington, KY 40506, U.S.A.\n",
      "FAU - Finn, Debra S\n",
      "AU  - Finn DS\n",
      "AD  - Department of Integrative Biology, Oregon State University, Corvallis, OR 97331, U.S.A.\n",
      "AD  - Departamento de Recursos Hidricos y Ciencias Ambientales, Universidad de Cuenca, Cuenca, Ecuador.\n",
      "FAU - Joseph Giersch, J\n",
      "AU  - Joseph Giersch J\n",
      "AD  - U.S. Geological Survey, Northern Rocky Mountain Science Center, West Glacier, MT 59936, U.S.A.\n",
      "FAU - Weisrock, David W\n",
      "AU  - Weisrock DW\n",
      "AD  - Department of Biology, University of Kentucky, Lexington, KY 40506, U.S.A.\n",
      "FAU - Jacobsen, Dean\n",
      "AU  - Jacobsen D\n",
      "AD  - Freshwater Biological Laboratory, Department of Biology, University ofCopenhagen, DK-2100, Copenhagen, Denmark.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "DEP - 20170120\n",
      "PL  - England\n",
      "TA  - Biol Rev Camb Philos Soc\n",
      "JT  - Biological reviews of the Cambridge Philosophical Society\n",
      "JID - 0414576\n",
      "SB  - IM\n",
      "MH  - *Altitude\n",
      "MH  - Animals\n",
      "MH  - *Biodiversity\n",
      "MH  - *Climate Change\n",
      "MH  - Databases, Factual\n",
      "MH  - Genomics\n",
      "MH  - *Rivers\n",
      "OTO - NOTNLM\n",
      "OT  - benthic\n",
      "OT  - biodiversity\n",
      "OT  - conservation biology\n",
      "OT  - ecology\n",
      "OT  - glacier recession\n",
      "OT  - global change\n",
      "OT  - lotic\n",
      "OT  - macroinvertebrate\n",
      "OT  - microbial ecology\n",
      "OT  - mountain\n",
      "EDAT- 2017/01/21 06:00\n",
      "MHDA- 2018/06/27 06:00\n",
      "CRDT- 2017/01/21 06:00\n",
      "PHST- 2016/02/23 00:00 [received]\n",
      "PHST- 2016/12/04 00:00 [revised]\n",
      "PHST- 2016/12/12 00:00 [accepted]\n",
      "PHST- 2017/01/21 06:00 [pubmed]\n",
      "PHST- 2018/06/27 06:00 [medline]\n",
      "PHST- 2017/01/21 06:00 [entrez]\n",
      "AID - 10.1111/brv.12319 [doi]\n",
      "PST - ppublish\n",
      "SO  - Biol Rev Camb Philos Soc. 2017 Nov;92(4):2024-2045. doi: 10.1111/brv.12319. Epub 2017 Jan 20.\n",
      "\n",
      "PMID- 27940644\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170726\n",
      "LR  - 20181202\n",
      "IS  - 1574-6941 (Electronic)\n",
      "IS  - 0168-6496 (Linking)\n",
      "VI  - 93\n",
      "IP  - 2\n",
      "DP  - 2017 Feb\n",
      "TI  - Rhizosphere microbiomes of potato cultivated in the High Andes show stable anddynamic core microbiomes with different responses to plant development.\n",
      "LID - fiw242 [pii]\n",
      "LID - 10.1093/femsec/fiw242 [doi]\n",
      "AB  - The rhizosphere hosts a rich microflora supporting plant nutrition and health. Weexamined bacterial rhizosphere microbiota of Solanum tuberosum grown in itscenter of origin, the Central Andean Highlands, at different vegetation stagesand sites at altitudes ranging from 3245 to 4070 m.a.s.l., differing in soilcharacteristics, climate and the agricultural practices by 454 sequence analysis of 16S rRNA genes. We observed that the taxonomic composition of bacteriarepeatedly occurring at particular stages of plant development was almostunaffected by highly diverse environmental conditions. A detailed statisticalanalysis on the operational taxonomic unit (OTU) level, representing bacterialspecies, revealed a complex community structure of the rhizosphere. We identifiedan opportunistic microbiome which comprises OTUs that occur randomly or underspecific environmental conditions. In contrast, core microbiome members werefound at all sites. The 'stable' component of the core microbiome consisted offew ubiquitous OTUs that were continuously abundant in all samples and vegetationstages, whereas the 'dynamic' component comprised OTUs that were enriched atspecific vegetation stages.\n",
      "CI  - (c) FEMS 2016. All rights reserved. For permissions, please e-mail:journals.permissions@oup.com.\n",
      "FAU - Pfeiffer, Stefan\n",
      "AU  - Pfeiffer S\n",
      "AD  - AIT Austrian Institute of Technology GmbH, Department of Health and Environment, Bioresources Unit, Konrad-Lorenz Strasse 24, A-3430 Tulln, Austria.\n",
      "FAU - Mitter, Birgit\n",
      "AU  - Mitter B\n",
      "AD  - AIT Austrian Institute of Technology GmbH, Department of Health and Environment, Bioresources Unit, Konrad-Lorenz Strasse 24, A-3430 Tulln, Austria.\n",
      "FAU - Oswald, Andreas\n",
      "AU  - Oswald A\n",
      "AD  - Integrated Crop Management Division, International Potato Center (CIP), LaMolina, Lima, Peru.\n",
      "AD  - Agroforestry and Sustainable Agriculture Program, CATIE, Turrialba, 30501 CostaRica.\n",
      "FAU - Schloter-Hai, Brigitte\n",
      "AU  - Schloter-Hai B\n",
      "AD  - Research Unit Environmental Genomics, Helmholtz Zentrum Munchen GmbH, GermanResearch Center for Environmental Health, D-85764 Neuherberg, Germany.\n",
      "FAU - Schloter, Michael\n",
      "AU  - Schloter M\n",
      "AD  - Research Unit Environmental Genomics, Helmholtz Zentrum Munchen GmbH, GermanResearch Center for Environmental Health, D-85764 Neuherberg, Germany.\n",
      "FAU - Declerck, Stephane\n",
      "AU  - Declerck S\n",
      "AD  - Earth and Life Institute, Applied Microbiology, Mycology, Universite catholiquede Louvain, Croix du Sud 2, 1348 Louvain-la-Neuve, Belgium.\n",
      "FAU - Sessitsch, Angela\n",
      "AU  - Sessitsch A\n",
      "AD  - AIT Austrian Institute of Technology GmbH, Department of Health and Environment, Bioresources Unit, Konrad-Lorenz Strasse 24, A-3430 Tulln, Austriaangela.sessitsch@ait.ac.at.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161208\n",
      "PL  - England\n",
      "TA  - FEMS Microbiol Ecol\n",
      "JT  - FEMS microbiology ecology\n",
      "JID - 8901229\n",
      "RN  - 0 (RNA, Ribosomal, 16S)\n",
      "RN  - 0 (Soil)\n",
      "SB  - IM\n",
      "MH  - Agriculture\n",
      "MH  - Bacteria/genetics\n",
      "MH  - Ecuador\n",
      "MH  - Microbiota/*genetics\n",
      "MH  - Plant Development\n",
      "MH  - RNA, Ribosomal, 16S/genetics\n",
      "MH  - *Rhizosphere\n",
      "MH  - Soil/chemistry\n",
      "MH  - *Soil Microbiology\n",
      "MH  - Solanum tuberosum/growth & development/*microbiology\n",
      "OTO - NOTNLM\n",
      "OT  - *16S rRNA\n",
      "OT  - *core microbiome\n",
      "OT  - *potato\n",
      "OT  - *rhizosphere\n",
      "EDAT- 2016/12/13 06:00\n",
      "MHDA- 2017/07/27 06:00\n",
      "CRDT- 2016/12/13 06:00\n",
      "PHST- 2016/12/04 00:00 [accepted]\n",
      "PHST- 2016/02/04 00:00 [received]\n",
      "PHST- 2016/12/13 06:00 [pubmed]\n",
      "PHST- 2017/07/27 06:00 [medline]\n",
      "PHST- 2016/12/13 06:00 [entrez]\n",
      "AID - fiw242 [pii]\n",
      "AID - 10.1093/femsec/fiw242 [doi]\n",
      "PST - ppublish\n",
      "SO  - FEMS Microbiol Ecol. 2017 Feb;93(2). pii: fiw242. doi: 10.1093/femsec/fiw242.Epub 2016 Dec 8.\n",
      "\n",
      "PMID- 27891694\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180130\n",
      "LR  - 20181202\n",
      "IS  - 1365-294X (Electronic)\n",
      "IS  - 0962-1083 (Linking)\n",
      "VI  - 26\n",
      "IP  - 3\n",
      "DP  - 2017 Feb\n",
      "TI  - Genetic variation in blue whales in the eastern pacific: implication for taxonomyand use of common wintering grounds.\n",
      "PG  - 740-751\n",
      "LID - 10.1111/mec.13940 [doi]\n",
      "AB  - Many aspects of blue whale biology are poorly understood. Some of the gaps in ourknowledge, such as those regarding their basic taxonomy and seasonal movements,directly affect our ability to monitor and manage blue whale populations. As astep towards filling in some of these gaps, microsatellite and mtDNA sequenceanalyses were conducted on blue whale samples from the Southern Hemisphere, theeastern tropical Pacific (ETP) and the northeast Pacific. The results indicatethat the ETP is differentially used by blue whales from the northern and southerneastern Pacific, with the former showing stronger affinity to the region offCentral America known as the Costa Rican Dome, and the latter favouring thewaters of Peru and Ecuador. Although the pattern of genetic variation throughout the Southern Hemisphere is compatible with the recently proposed subspeciesstatus of Chilean blue whales, some discrepancies remain between catch lengthsand lengths from aerial photography, and not all blue whales in Chilean waterscan be assumed to be of this type. Also, the range of the proposed Chileansubspecies, which extends to the Galapagos region of the ETP, at leastseasonally, perhaps should include the Costa Rican Dome and the eastern NorthPacific as well.\n",
      "CI  - (c) 2016 John Wiley & Sons Ltd.\n",
      "FAU - Leduc, R G\n",
      "AU  - Leduc RG\n",
      "AD  - Marine Mammal and Turtle Division, Southwest Fisheries Science Center, NationalMarine Fisheries Service, NOAA, 8901 La Jolla Shores Drive, La Jolla, CA, USA.\n",
      "FAU - Archer, F I\n",
      "AU  - Archer FI\n",
      "AD  - Marine Mammal and Turtle Division, Southwest Fisheries Science Center, NationalMarine Fisheries Service, NOAA, 8901 La Jolla Shores Drive, La Jolla, CA, USA.\n",
      "FAU - Lang, A R\n",
      "AU  - Lang AR\n",
      "AD  - Marine Mammal and Turtle Division, Southwest Fisheries Science Center, NationalMarine Fisheries Service, NOAA, 8901 La Jolla Shores Drive, La Jolla, CA, USA.\n",
      "FAU - Martien, K K\n",
      "AU  - Martien KK\n",
      "AD  - Marine Mammal and Turtle Division, Southwest Fisheries Science Center, NationalMarine Fisheries Service, NOAA, 8901 La Jolla Shores Drive, La Jolla, CA, USA.\n",
      "FAU - Hancock-Hanser, B\n",
      "AU  - Hancock-Hanser B\n",
      "AD  - Marine Mammal and Turtle Division, Southwest Fisheries Science Center, NationalMarine Fisheries Service, NOAA, 8901 La Jolla Shores Drive, La Jolla, CA, USA.\n",
      "FAU - Torres-Florez, J P\n",
      "AU  - Torres-Florez JP\n",
      "AD  - Departamento de Genetica e Evolucao, Universidade Federal de Sao Carlos, Rod.Washington Luis, km 235, Sao Carlos, SP, Brazil.\n",
      "AD  - Centro Ballena Azul/Blue Whale Center, Valdivia, Chile.\n",
      "FAU - Hucke-Gaete, R\n",
      "AU  - Hucke-Gaete R\n",
      "AD  - Centro Ballena Azul/Blue Whale Center, Valdivia, Chile.\n",
      "AD  - Instituto de Ciencias Marinas y Limnologicas, Universidad Austral de Chile,Casilla 567, Valdivia, Chile.\n",
      "FAU - Rosenbaum, H C\n",
      "AU  - Rosenbaum HC\n",
      "AD  - Ocean Giants Program, Wildlife Conservation Society - Global Conservation, 2300Southern Blvd., Bronx, NY, 10460, USA.\n",
      "AD  - Sackler Institute for Comparative Genomics, American Museum of Natural History,New York, NY, USA.\n",
      "FAU - van Waerebeek, K\n",
      "AU  - van Waerebeek K\n",
      "AD  - Peruvian Centre for Cetacean Research, Lima 20, Peru.\n",
      "FAU - Brownell, R L Jr\n",
      "AU  - Brownell RL Jr\n",
      "AD  - Marine Mammal and Turtle Division, Southwest Fisheries Science Center, NationalMarine Fisheries Service, NOAA, 8901 La Jolla Shores Drive, La Jolla, CA, USA.\n",
      "FAU - Taylor, B L\n",
      "AU  - Taylor BL\n",
      "AD  - Marine Mammal and Turtle Division, Southwest Fisheries Science Center, NationalMarine Fisheries Service, NOAA, 8901 La Jolla Shores Drive, La Jolla, CA, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161225\n",
      "PL  - England\n",
      "TA  - Mol Ecol\n",
      "JT  - Molecular ecology\n",
      "JID - 9214478\n",
      "RN  - 0 (DNA, Mitochondrial)\n",
      "SB  - IM\n",
      "MH  - Animal Migration\n",
      "MH  - Animals\n",
      "MH  - Balaenoptera/*genetics\n",
      "MH  - Central America\n",
      "MH  - Chile\n",
      "MH  - DNA, Mitochondrial/genetics\n",
      "MH  - Ecuador\n",
      "MH  - *Genetic Variation\n",
      "MH  - *Genetics, Population\n",
      "MH  - Microsatellite Repeats\n",
      "MH  - Pacific Ocean\n",
      "MH  - Peru\n",
      "OTO - NOTNLM\n",
      "OT  - Balaenoptera musculus\n",
      "OT  - conservation genetics\n",
      "OT  - eastern Pacific\n",
      "OT  - endangered species\n",
      "EDAT- 2016/11/29 06:00\n",
      "MHDA- 2018/01/31 06:00\n",
      "CRDT- 2016/11/29 06:00\n",
      "PHST- 2015/12/25 00:00 [received]\n",
      "PHST- 2016/09/29 00:00 [revised]\n",
      "PHST- 2016/11/16 00:00 [accepted]\n",
      "PHST- 2016/11/29 06:00 [pubmed]\n",
      "PHST- 2018/01/31 06:00 [medline]\n",
      "PHST- 2016/11/29 06:00 [entrez]\n",
      "AID - 10.1111/mec.13940 [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Ecol. 2017 Feb;26(3):740-751. doi: 10.1111/mec.13940. Epub 2016 Dec 25.\n",
      "\n",
      "PMID- 27863902\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170712\n",
      "LR  - 20181113\n",
      "IS  - 1471-5007 (Electronic)\n",
      "IS  - 1471-4922 (Linking)\n",
      "VI  - 33\n",
      "IP  - 4\n",
      "DP  - 2017 Apr\n",
      "TI  - Prediction and Prevention of Parasitic Diseases Using a Landscape GenomicsFramework.\n",
      "PG  - 264-275\n",
      "LID - S1471-4922(16)30191-X [pii]\n",
      "LID - 10.1016/j.pt.2016.10.008 [doi]\n",
      "AB  - Substantial heterogeneity exists in the dispersal, distribution and transmission of parasitic species. Understanding and predicting how such features are governedby the ecological variation of landscape they inhabit is the central goal ofspatial epidemiology. Genetic data can further inform functional connectivityamong parasite, host and vector populations in a landscape. Gene flow correlates with the spread of epidemiologically relevant phenotypes among parasite andvector populations (e.g., virulence, drug and pesticide resistance), as well asinvasion and re-invasion risk where parasite transmission is absent due tocurrent or past intervention measures. However, the formal integration of spatialand genetic data ('landscape genetics') is scarcely ever applied to parasites.Here, we discuss the specific challenges and practical prospects for the use oflandscape genetics and genomics to understand the biology and control ofparasitic disease and present a practical framework for doing so.\n",
      "CI  - Copyright (c) 2016 Elsevier Ltd. All rights reserved.\n",
      "FAU - Schwabl, Philipp\n",
      "AU  - Schwabl P\n",
      "AD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of Glasgow, Glasgow G12 8QQ, UK.\n",
      "FAU - Llewellyn, Martin S\n",
      "AU  - Llewellyn MS\n",
      "AD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of Glasgow, Glasgow G12 8QQ, UK. Electronic address: martin.llewellyn@glasgow.ac.uk.\n",
      "FAU - Landguth, Erin L\n",
      "AU  - Landguth EL\n",
      "AD  - Division of Biological Sciences, University of Montana, 32 Campus Drive,Missoula, MT, 59812, USA.\n",
      "FAU - Andersson, Bjorn\n",
      "AU  - Andersson B\n",
      "AD  - Department of Cell and Molecular Biology, Science for Life Laboratory, KarolinskaInstitutet, Berzelius vag 35, 171 77 Stockholm, Sweden.\n",
      "FAU - Kitron, Uriel\n",
      "AU  - Kitron U\n",
      "AD  - Department of Environmental Sciences, Emory University, Atlanta, GA, USA.\n",
      "FAU - Costales, Jaime A\n",
      "AU  - Costales JA\n",
      "AD  - Center for Research on Health in Latin America, School of Biological Sciences,Pontifical Catholic University of Ecuador, Quito, Ecuador.\n",
      "FAU - Ocana, Sofia\n",
      "AU  - Ocana S\n",
      "AD  - Center for Research on Health in Latin America, School of Biological Sciences,Pontifical Catholic University of Ecuador, Quito, Ecuador.\n",
      "FAU - Grijalva, Mario J\n",
      "AU  - Grijalva MJ\n",
      "AD  - Center for Research on Health in Latin America, School of Biological Sciences,Pontifical Catholic University of Ecuador, Quito, Ecuador; Infectious andTropical Disease Institute, Department of Biomedical Sciences, Heritage Collegeof Osteopathic Medicine, Ohio University, Athens, OH, USA.\n",
      "LA  - eng\n",
      "GR  - R15 AI105749/AI/NIAID NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "DEP - 20161116\n",
      "PL  - England\n",
      "TA  - Trends Parasitol\n",
      "JT  - Trends in parasitology\n",
      "JID - 100966034\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - *Genomics\n",
      "MH  - *Host-Parasite Interactions/genetics\n",
      "MH  - Humans\n",
      "MH  - Parasitic Diseases/epidemiology/*parasitology/*prevention & control/transmission\n",
      "MH  - Phenotype\n",
      "PMC - PMC5376512\n",
      "MID - NIHMS827616\n",
      "EDAT- 2016/11/20 06:00\n",
      "MHDA- 2017/07/14 06:00\n",
      "CRDT- 2016/11/20 06:00\n",
      "PHST- 2016/05/07 00:00 [received]\n",
      "PHST- 2016/09/10 00:00 [revised]\n",
      "PHST- 2016/10/19 00:00 [accepted]\n",
      "PHST- 2016/11/20 06:00 [pubmed]\n",
      "PHST- 2017/07/14 06:00 [medline]\n",
      "PHST- 2016/11/20 06:00 [entrez]\n",
      "AID - S1471-4922(16)30191-X [pii]\n",
      "AID - 10.1016/j.pt.2016.10.008 [doi]\n",
      "PST - ppublish\n",
      "SO  - Trends Parasitol. 2017 Apr;33(4):264-275. doi: 10.1016/j.pt.2016.10.008. Epub2016 Nov 16.\n",
      "\n",
      "PMID- 27825829\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20171127\n",
      "LR  - 20180228\n",
      "IS  - 1873-1899 (Electronic)\n",
      "IS  - 0734-9750 (Linking)\n",
      "VI  - 34\n",
      "IP  - 8\n",
      "DP  - 2016 Dec\n",
      "TI  - Metagenomics: Probing pollutant fate in natural and engineered ecosystems.\n",
      "PG  - 1413-1426\n",
      "LID - S0734-9750(16)30132-X [pii]\n",
      "LID - 10.1016/j.biotechadv.2016.10.006 [doi]\n",
      "AB  - Polluted environments are a reservoir of microbial species able to degrade or to convert pollutants to harmless compounds. The proper management of microbialresources requires a comprehensive characterization of their genetic pool toassess the fate of contaminants and increase the efficiency of bioremediationprocesses. Metagenomics offers appropriate tools to describe microbialcommunities in their whole complexity without lab-based cultivation of individualstrains. After a decade of use of metagenomics to study microbiomes, thescientific community has made significant progress in this field. In this review,we survey the main steps of metagenomics applied to environments contaminatedwith organic compounds or heavy metals. We emphasize technical solutions proposedto overcome encountered obstacles. We then compare two metagenomic approaches,i.e. library-based targeted metagenomics and direct sequencing of metagenomes. Inthe former, environmental DNA is cloned inside a host, and then clones ofinterest are selected based on (i) their expression of biodegradative functionsor (ii) sequence homology with probes and primers designed from relevant, alreadyknown sequences. The highest score for the discovery of novel genes anddegradation pathways has been achieved so far by functional screening of largeclone libraries. On the other hand, direct sequencing of metagenomes without acloning step has been more often applied to polluted environments forcharacterization of the taxonomic and functional composition of microbialcommunities and their dynamics. In this case, the analysis has focused on 16SrRNA genes and marker genes of biodegradation. Advances in next generationsequencing and in bioinformatic analysis of sequencing data have opened up newopportunities for assessing the potential of biodegradation by microbes, butannotation of collected genes is still hampered by a limited number of available reference sequences in databases. Although metagenomics is still facing technicaland computational challenges, our review of the recent literature highlights its value as an aid to efficiently monitor the clean-up of contaminated environments and develop successful strategies to mitigate the impact of pollutants onecosystems.\n",
      "CI  - Copyright (c) 2016 Elsevier Inc. All rights reserved.\n",
      "FAU - Bouhajja, Emna\n",
      "AU  - Bouhajja E\n",
      "AD  - Laboratoire de Genie Biologique, Earth and Life Institute, Universite Catholique de Louvain, Place Croix du Sud 2, boite L7.05.19, 1348 Louvain-la-Neuve, Belgium.\n",
      "FAU - Agathos, Spiros N\n",
      "AU  - Agathos SN\n",
      "AD  - Laboratoire de Genie Biologique, Earth and Life Institute, Universite Catholique de Louvain, Place Croix du Sud 2, boite L7.05.19, 1348 Louvain-la-Neuve, Belgium;School of Life Sciences and Biotechnology, Yachay Tech University, 100119 SanMiguel de Urcuqui, Ecuador.\n",
      "FAU - George, Isabelle F\n",
      "AU  - George IF\n",
      "AD  - Universite Libre de Bruxelles, Laboratoire d'Ecologie des Systemes Aquatiques,Campus de la Plaine CP 221, Boulevard du Triomphe, 1050 Brussels, Belgium.Electronic address: igeorge@ulb.ac.be.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20161105\n",
      "PL  - England\n",
      "TA  - Biotechnol Adv\n",
      "JT  - Biotechnology advances\n",
      "JID - 8403708\n",
      "RN  - 0 (Environmental Pollutants)\n",
      "SB  - IM\n",
      "MH  - *Biodegradation, Environmental\n",
      "MH  - *Ecosystem\n",
      "MH  - *Environmental Pollutants/analysis/chemistry/isolation & purification/metabolism\n",
      "MH  - Metabolic Networks and Pathways/genetics\n",
      "MH  - Metagenome/*genetics\n",
      "MH  - *Metagenomics\n",
      "MH  - Microbial Consortia/*genetics\n",
      "OTO - NOTNLM\n",
      "OT  - *Catabolic genes\n",
      "OT  - *Degradation pathways\n",
      "OT  - *Metagenomics\n",
      "OT  - *Molecular microbial ecology\n",
      "OT  - *Pollutant biodegradation\n",
      "EDAT- 2016/11/09 06:00\n",
      "MHDA- 2017/11/29 06:00\n",
      "CRDT- 2016/11/10 06:00\n",
      "PHST- 2016/03/14 00:00 [received]\n",
      "PHST- 2016/10/01 00:00 [revised]\n",
      "PHST- 2016/10/12 00:00 [accepted]\n",
      "PHST- 2016/11/09 06:00 [pubmed]\n",
      "PHST- 2017/11/29 06:00 [medline]\n",
      "PHST- 2016/11/10 06:00 [entrez]\n",
      "AID - S0734-9750(16)30132-X [pii]\n",
      "AID - 10.1016/j.biotechadv.2016.10.006 [doi]\n",
      "PST - ppublish\n",
      "SO  - Biotechnol Adv. 2016 Dec;34(8):1413-1426. doi: 10.1016/j.biotechadv.2016.10.006. Epub 2016 Nov 5.\n",
      "\n",
      "PMID- 27785541\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170117\n",
      "LR  - 20170117\n",
      "IS  - 1432-0614 (Electronic)\n",
      "IS  - 0175-7598 (Linking)\n",
      "VI  - 101\n",
      "IP  - 2\n",
      "DP  - 2017 Jan\n",
      "TI  - Identification of novel toluene monooxygenase genes in a hydrocarbon-pollutedsediment using sequence- and function-based screening of metagenomic libraries.\n",
      "PG  - 797-808\n",
      "LID - 10.1007/s00253-016-7934-5 [doi]\n",
      "AB  - The microbial potential for toluene degradation within sediments from a taroil-contaminated site in Flingern, Germany, was assessed using a metagenomicapproach. High molecular weight environmental DNA from contaminated sediments wasextracted, purified, and cloned into fosmid and BAC vectors and transformed into Escherichia coli. The fosmid library was screened by hybridization with a PCRamplicon of the alpha-subunit of the toluene 4-monooxygenase gene to identifygenes and pathways encoding toluene degradation. Fourteen clones were recoveredfrom the fosmid library, among which 13 were highly divergent from known tmoAgenes and several had the closest relatives among Acinetobacter species. The BAC library was transferred to the heterologous hosts Cupriavidus metallidurans(phylum Proteobacteria) and Edaphobacter aggregans (phylum Acidobacteria). Theresulting libraries were screened for expression of toluene degradation in thenon-degradative hosts. From expression in C. metallidurans, three novel toluenemonooxygenase-encoding operons were identified that were located on IncP1plasmids. The E. aggregans-hosted BAC library led to the isolation of a clonedgenetic locus putatively derived from an Acidobacteria taxon that contained genesinvolved in aerobic and anaerobic toluene degradation. These data suggest theimportant role of plasmids in the spread of toluene degradative capacity andindicate putative novel tmoA genes present in this hydrocarbon-pollutedenvironment.\n",
      "FAU - Bouhajja, E\n",
      "AU  - Bouhajja E\n",
      "AD  - Earth and Life Institute, Laboratoire de Genie Biologique, Universite catholique de Louvain, Place Croix du Sud 2, boite L7.05.19, 1348, Louvain-la-Neuve,Belgium.\n",
      "FAU - McGuire, M\n",
      "AU  - McGuire M\n",
      "AD  - Department of Biological Sciences, Auburn University, 101 Rouse Life ScienceBuilding, Auburn, Alabama, 36849, USA.\n",
      "FAU - Liles, M R\n",
      "AU  - Liles MR\n",
      "AD  - Department of Biological Sciences, Auburn University, 101 Rouse Life ScienceBuilding, Auburn, Alabama, 36849, USA.\n",
      "FAU - Bataille, G\n",
      "AU  - Bataille G\n",
      "AD  - Earth and Life Institute, Biodiversity Research Centre, Universite catholique de Louvain, Place Croix du Sud 4-5, Bte L.7.07.04, 1348, Louvain-la-Neuve, Belgium.\n",
      "FAU - Agathos, S N\n",
      "AU  - Agathos SN\n",
      "AD  - Earth and Life Institute, Laboratoire de Genie Biologique, Universite catholique de Louvain, Place Croix du Sud 2, boite L7.05.19, 1348, Louvain-la-Neuve,Belgium.\n",
      "AD  - School of Life Sciences and Biotechnology, Yachay Tech University, San Miguel de Urcuqui, Ecuador.\n",
      "FAU - George, I F\n",
      "AU  - George IF\n",
      "AD  - Laboratoire d'Ecologie des Systemes Aquatiques, Universite libre de Bruxelles,Campus de la Plaine CP 221, Boulevard du Triomphe, 1050, Brussels, Belgium.igeorge@ulb.ac.be.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161026\n",
      "PL  - Germany\n",
      "TA  - Appl Microbiol Biotechnol\n",
      "JT  - Applied microbiology and biotechnology\n",
      "JID - 8406612\n",
      "RN  - 0 (Environmental Pollutants)\n",
      "RN  - 3FPU23BG52 (Toluene)\n",
      "RN  - EC 1.- (Mixed Function Oxygenases)\n",
      "SB  - IM\n",
      "MH  - Cloning, Molecular\n",
      "MH  - *Environmental Microbiology\n",
      "MH  - Environmental Pollutants/*metabolism\n",
      "MH  - Genetic Testing\n",
      "MH  - Germany\n",
      "MH  - Metagenomics\n",
      "MH  - Mixed Function Oxygenases/*analysis/*genetics\n",
      "MH  - Nucleic Acid Hybridization\n",
      "MH  - Sequence Analysis, DNA\n",
      "MH  - Toluene/*metabolism\n",
      "OTO - NOTNLM\n",
      "OT  - Biodegradation\n",
      "OT  - Catabolic gene diversity\n",
      "OT  - Expression in heterologous hosts\n",
      "OT  - Macroarray\n",
      "OT  - Metagenomics\n",
      "OT  - Toluene 4-monooxygenase\n",
      "EDAT- 2016/10/28 06:00\n",
      "MHDA- 2017/01/18 06:00\n",
      "CRDT- 2016/10/28 06:00\n",
      "PHST- 2016/07/05 00:00 [received]\n",
      "PHST- 2016/10/12 00:00 [accepted]\n",
      "PHST- 2016/10/06 00:00 [revised]\n",
      "PHST- 2016/10/28 06:00 [pubmed]\n",
      "PHST- 2017/01/18 06:00 [medline]\n",
      "PHST- 2016/10/28 06:00 [entrez]\n",
      "AID - 10.1007/s00253-016-7934-5 [doi]\n",
      "AID - 10.1007/s00253-016-7934-5 [pii]\n",
      "PST - ppublish\n",
      "SO  - Appl Microbiol Biotechnol. 2017 Jan;101(2):797-808. doi:10.1007/s00253-016-7934-5. Epub 2016 Oct 26.\n",
      "\n",
      "PMID- 27363308\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20171129\n",
      "LR  - 20181202\n",
      "IS  - 1365-294X (Electronic)\n",
      "IS  - 0962-1083 (Linking)\n",
      "VI  - 25\n",
      "IP  - 21\n",
      "DP  - 2016 Nov\n",
      "TI  - Genomic variation at the tips of the adaptive radiation of Darwin's finches.\n",
      "PG  - 5282-5295\n",
      "LID - 10.1111/mec.13743 [doi]\n",
      "AB  - Adaptive radiation unfolds as selection acts on the genetic variation underlying functional traits. The nature of this variation can be revealed by studying thetips of an ongoing adaptive radiation. We studied genomic variation at the tipsof the Darwin's finch radiation; specifically focusing on polymorphism within,and variation among, three sympatric species of the genus Geospiza. Usingrestriction site-associated DNA (RAD-seq), we characterized 32 569single-nucleotide polymorphisms (SNPs), from which 11 outlier SNPs for beak andbody size were uncovered by a genomewide association study (GWAS). Principalcomponent analysis revealed that these 11 SNPs formed four statistically linkedgroups. Stepwise regression then revealed that the first PC score, which included6 of the 11 top SNPs, explained over 80% of the variation in beak size,suggesting that selection on these traits influences multiple correlated loci.The two SNPs most strongly associated with beak size were near genes associatedwith beak morphology across deeper branches of the radiation: delta-like 1homologue (DLK1) and high-mobility group AT-hook 2 (HMGA2). Our results suggestthat (i) key adaptive traits are associated with a small fraction of the genome(11 of 32 569 SNPs), (ii) SNPs linked to the candidate genes are dispersedthroughout the genome (on several chromosomes), and (iii) micro- andmacro-evolutionary variation (roots and tips of the radiation) involve someshared and some unique genomic regions.\n",
      "CI  - (c) 2016 John Wiley & Sons Ltd.\n",
      "FAU - Chaves, Jaime A\n",
      "AU  - Chaves JA\n",
      "AD  - Department of Biology, University of Miami, Coral Gables, FL, 33146, USA.jachaves@usfq.edu.ec.\n",
      "AD  - Universidad San Francisco de Quito, USFQ, Colegio de Ciencias Biologicas yAmbientales, y Extension Galapagos, Campus Cumbaya, Quito, Ecuador.jachaves@usfq.edu.ec.\n",
      "FAU - Cooper, Elizabeth A\n",
      "AU  - Cooper EA\n",
      "AD  - Department of Biology, University of Miami, Coral Gables, FL, 33146, USA.\n",
      "AD  - Department of Genetics and Biochemistry, Clemson University, Clemson, SC, 29634, USA.\n",
      "FAU - Hendry, Andrew P\n",
      "AU  - Hendry AP\n",
      "AD  - Redpath Museum, Department of Biology, McGill University, Montreal, QC, Canada.\n",
      "FAU - Podos, Jeffrey\n",
      "AU  - Podos J\n",
      "AD  - Department of Biology, University of Massachusetts Amherst, Amherst, MA, 01003,USA.\n",
      "FAU - De Leon, Luis F\n",
      "AU  - De Leon LF\n",
      "AUID- ORCID: 0000-0001-9317-420X\n",
      "AD  - Centro de Biodiversidad y Descubrimiento de Drogas, Instituto de Investigaciones Cientificas y Servicios de Alta Tecnologia (INDICASAT-AIP), Ciudad del Saber,Panama, Panama.\n",
      "AD  - Department of Biology, University of Massachusetts Boston, 100 Morrissey Blvd,Boston, MA, 02125, USA.\n",
      "FAU - Raeymaekers, Joost A M\n",
      "AU  - Raeymaekers JA\n",
      "AD  - Laboratory of Biodiversity and Evolutionary Genomics, University of Leuven,B-3000, Leuven, Belgium.\n",
      "AD  - Center for Biodiversity Dynamics, Department of Biology, Norwegian University of Science and Technology, N-7491, Trondheim, Norway.\n",
      "FAU - MacMillan, W Owen\n",
      "AU  - MacMillan WO\n",
      "AD  - Smithsonian Tropical Research Institute, Panama, Panama.\n",
      "FAU - Uy, J Albert C\n",
      "AU  - Uy JA\n",
      "AD  - Department of Biology, University of Miami, Coral Gables, FL, 33146, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160729\n",
      "PL  - England\n",
      "TA  - Mol Ecol\n",
      "JT  - Molecular ecology\n",
      "JID - 9214478\n",
      "SB  - IM\n",
      "CIN - Mol Ecol. 2016 Nov;25(21):5265-5266. PMID: 27785886\n",
      "MH  - Animals\n",
      "MH  - Beak\n",
      "MH  - *Biological Evolution\n",
      "MH  - Body Size\n",
      "MH  - Finches/*genetics\n",
      "MH  - Genetic Association Studies\n",
      "MH  - Genetic Linkage\n",
      "MH  - Genomics\n",
      "MH  - Polymorphism, Single Nucleotide\n",
      "MH  - Selection, Genetic\n",
      "OTO - NOTNLM\n",
      "OT  - *Darwin's finches\n",
      "OT  - *RAD-seq\n",
      "OT  - *adaptive radiation\n",
      "OT  - *beak size\n",
      "OT  - *genomic regions\n",
      "EDAT- 2016/10/28 06:00\n",
      "MHDA- 2017/12/01 06:00\n",
      "CRDT- 2016/07/02 06:00\n",
      "PHST- 2015/11/23 00:00 [received]\n",
      "PHST- 2016/06/27 00:00 [revised]\n",
      "PHST- 2016/06/28 00:00 [accepted]\n",
      "PHST- 2016/10/28 06:00 [pubmed]\n",
      "PHST- 2017/12/01 06:00 [medline]\n",
      "PHST- 2016/07/02 06:00 [entrez]\n",
      "AID - 10.1111/mec.13743 [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Ecol. 2016 Nov;25(21):5282-5295. doi: 10.1111/mec.13743. Epub 2016 Jul 29.\n",
      "\n",
      "PMID- 27776178\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170606\n",
      "LR  - 20190112\n",
      "IS  - 1932-6203 (Electronic)\n",
      "IS  - 1932-6203 (Linking)\n",
      "VI  - 11\n",
      "IP  - 10\n",
      "DP  - 2016\n",
      "TI  - Molecular Study of the Amazonian Macabea Cattle History.\n",
      "PG  - e0165398\n",
      "LID - 10.1371/journal.pone.0165398 [doi]\n",
      "AB  - Macabea cattle are the only Bos taurus breed that have adapted to the wettropical conditions of the Amazon. This breed has integrated into the culture of the indigenous Shuar-Asuar nations probably since its origins, being one of thefew European zoogenetic resources assimilated by the deep-jungle Amazoncommunities. Despite its potential for local endogenous sustainable development, this breed is currently endangered. The present study used molecular geneticstools to investigate the within- and between-breeds diversity, in order tocharacterize the breed population, define its associations with other breeds, andinfer its origin and evolution. The within-breed genetic diversity showed highvalues, as indicated by all genetic parameters, such as the mean number ofalleles (MNA = 7.25+/-2.03), the observed heterozygosity (Ho = 0.72+/-0.02) andthe expected heterozygosity (He = 0.72+/-0.02). The between-breeds diversityanalysis, which included factorial correspondence analysis, Reynolds geneticdistance, neighbor-joining analysis, and genetic structure analysis, showed that the Macabea breed belongs to the group of the American Creoles, with aSouthern-Spain origin. Our outcomes demonstrated that the Macabea breed has ahigh level of purity and null influences of exotic cosmopolitan breeds withEuropean or Asiatic origin. This breed is an important zoogenetic resource ofEcuador, with relevant and unique attributes; therefore, there is an urgent need to develop conservation strategies for the Macabea breed.\n",
      "FAU - Vargas, Julio\n",
      "AU  - Vargas J\n",
      "AD  - Universidad Nacional Amazonica. Rectoria. Puyo, Ecuador.\n",
      "FAU - Landi, Vincenzo\n",
      "AU  - Landi V\n",
      "AUID- ORCID: http://orcid.org/0000-0003-1385-8439\n",
      "AD  - Departamento de Genetica, Universidad de Cordoba, Cordoba, Espana.\n",
      "AD  - Animal Breeding Consulting S.L., Parque Cientifico Tecnologico de Cordoba,Espana.\n",
      "FAU - Martinez, Amparo\n",
      "AU  - Martinez A\n",
      "AD  - Departamento de Genetica, Universidad de Cordoba, Cordoba, Espana.\n",
      "AD  - Animal Breeding Consulting S.L., Parque Cientifico Tecnologico de Cordoba,Espana.\n",
      "FAU - Gomez, Mayra\n",
      "AU  - Gomez M\n",
      "AD  - Departamento de Genetica, Universidad de Cordoba, Cordoba, Espana.\n",
      "AD  - Animal Breeding Consulting S.L., Parque Cientifico Tecnologico de Cordoba,Espana.\n",
      "FAU - Camacho, Maria Esperanza\n",
      "AU  - Camacho ME\n",
      "AD  - Instituto de Investigacion y Formacion Agraria y Pesquera (IFAPA), Alameda delObispo, Cordoba, Espana.\n",
      "FAU - Alvarez, Luz Angela\n",
      "AU  - Alvarez LA\n",
      "AD  - Universidad Nacional de Colombia - Sede Palmira, Palmira Colombia.\n",
      "FAU - Aguirre, Lenin\n",
      "AU  - Aguirre L\n",
      "AD  - Universidad Nacional de Loja, Loja, Ecuador.\n",
      "FAU - Delgado, Juan Vicente\n",
      "AU  - Delgado JV\n",
      "AD  - Departamento de Genetica, Universidad de Cordoba, Cordoba, Espana.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20161024\n",
      "PL  - United States\n",
      "TA  - PLoS One\n",
      "JT  - PloS one\n",
      "JID - 101285081\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Biodiversity\n",
      "MH  - Cattle/*genetics\n",
      "MH  - Female\n",
      "MH  - Male\n",
      "MH  - Microsatellite Repeats/genetics\n",
      "MH  - South America\n",
      "PMC - PMC5077120\n",
      "COIS- The authors have the following interests. Vincenzo Landi, Mayra Gomez and Amparo Martinez are employed by Animal Breeding Consulting sl. Animal BreedingConsulting sl is involved in research programmes in cattle genetics/genomics and contributed to laboratory and statistical analysis for this work. There are nopatents, products in development or marketed products to declare. This does notalter the authors' adherence to all the PLOS ONE policies on sharing data andmaterials.\n",
      "EDAT- 2016/10/25 06:00\n",
      "MHDA- 2017/06/07 06:00\n",
      "CRDT- 2016/10/25 06:00\n",
      "PHST- 2016/06/24 00:00 [received]\n",
      "PHST- 2016/10/11 00:00 [accepted]\n",
      "PHST- 2016/10/25 06:00 [pubmed]\n",
      "PHST- 2017/06/07 06:00 [medline]\n",
      "PHST- 2016/10/25 06:00 [entrez]\n",
      "AID - 10.1371/journal.pone.0165398 [doi]\n",
      "AID - PONE-D-16-25312 [pii]\n",
      "PST - epublish\n",
      "SO  - PLoS One. 2016 Oct 24;11(10):e0165398. doi: 10.1371/journal.pone.0165398.eCollection 2016.\n",
      "\n",
      "PMID- 27655649\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20180222\n",
      "LR  - 20181202\n",
      "IS  - 1612-9210 (Electronic)\n",
      "IS  - 1612-9202 (Linking)\n",
      "VI  - 13\n",
      "IP  - 4\n",
      "DP  - 2016 Dec\n",
      "TI  - Impact of Anthropogenic Disturbance on Native and Invasive Trypanosomes ofRodents in Forested Uganda.\n",
      "PG  - 698-707\n",
      "AB  - Habitat disturbance and anthropogenic change are globally associated withextinctions and invasive species introductions. Less understood is the impact of environmental change on the parasites harbored by endangered, extinct, andintroduced species. To improve our understanding of the impacts of anthropogenic disturbance on such host-parasite interactions, we investigated an invasivetrypanosome (Trypanosoma lewisi). We screened 348 individual small mammals,representing 26 species, from both forested and non-forested habitats in ruralUganda. Using microscopy and PCR, we identified 18% of individuals (orderRodentia) as positive for trypanosomes. Further phylogenetic analyses revealedtwo trypanosomes circulating-T. lewisi and T. varani. T. lewisi was found inseven species both native and invasive, while T. varani was identified in onlythree native forest species. The lack of T. varani in non-forested habitatssuggests that it is a natural parasite of forest-dwelling rodents. Our findingssuggest that anthropogenic disturbance may lead to spillover of an invasiveparasite (T. lewisi) from non-native to native species, and lead to localco-extinction of a native parasite (T. varani) and native forest-dwelling hosts.\n",
      "FAU - Salzer, Johanna S\n",
      "AU  - Salzer JS\n",
      "AD  - Program in Population Biology, Ecology, and Evolution, Emory University, 400Dowman Dr., Math and Science Center 5th Floor, Atlanta, GA, 30322, USA.\n",
      "AD  - Department of Environmental Sciences, Emory University, Atlanta, GA, 30322, USA.\n",
      "AD  - Poxvirus and Rabies Branch, Division of High-Consequence Pathogens and Pathology,Centers for Disease Control and Prevention, Atlanta, GA, 30333, USA.\n",
      "FAU - Pinto, C Miguel\n",
      "AU  - Pinto CM\n",
      "AD  - Department of Mammalogy and Sackler Institute for Comparative Genomics, American Museum of Natural History, New York, NY, 10024, USA.\n",
      "AD  - Instituto de Ciencias Biologicas, Escuela Politecnica Nacional, PO Box17-01-2759, Quito, Ecuador.\n",
      "FAU - Grippi, Dylan C\n",
      "AU  - Grippi DC\n",
      "AD  - Program in Population Biology, Ecology, and Evolution, Emory University, 400Dowman Dr., Math and Science Center 5th Floor, Atlanta, GA, 30322, USA.\n",
      "FAU - Williams-Newkirk, Amanda Jo\n",
      "AU  - Williams-Newkirk AJ\n",
      "AD  - Program in Population Biology, Ecology, and Evolution, Emory University, 400Dowman Dr., Math and Science Center 5th Floor, Atlanta, GA, 30322, USA.\n",
      "AD  - Department of Environmental Sciences, Emory University, Atlanta, GA, 30322, USA.\n",
      "AD  - Rickettsial Zoonoses Branch, Division of Vector-borne Diseases, Centers forDisease Control and Prevention, Atlanta, GA, 30333, USA.\n",
      "FAU - Peterhans, Julian Kerbis\n",
      "AU  - Peterhans JK\n",
      "AD  - College of Professional Studies, Roosevelt University, Chicago, IL, 60605, USA.\n",
      "AD  - Division of Mammals, Field Museum of Natural History, Chicago, IL, 60605, USA.\n",
      "FAU - Rwego, Innocent B\n",
      "AU  - Rwego IB\n",
      "AD  - Department of Environmental Sciences, Emory University, Atlanta, GA, 30322, USA.\n",
      "AD  - Department of Environmental Health, Rollins School of Public Health, EmoryUniversity, Atlanta, GA, 30322, USA.\n",
      "AD  - Department of Biological Sciences, Makerere University, Kampala, Uganda.\n",
      "FAU - Carroll, Darin S\n",
      "AU  - Carroll DS\n",
      "AD  - Department of Environmental Sciences, Emory University, Atlanta, GA, 30322, USA.\n",
      "AD  - Poxvirus and Rabies Branch, Division of High-Consequence Pathogens and Pathology,Centers for Disease Control and Prevention, Atlanta, GA, 30333, USA.\n",
      "FAU - Gillespie, Thomas R\n",
      "AU  - Gillespie TR\n",
      "AD  - Program in Population Biology, Ecology, and Evolution, Emory University, 400Dowman Dr., Math and Science Center 5th Floor, Atlanta, GA, 30322, USA.thomas.gillespie@emory.edu.\n",
      "AD  - Department of Environmental Sciences, Emory University, Atlanta, GA, 30322, USA. thomas.gillespie@emory.edu.\n",
      "AD  - Department of Environmental Health, Rollins School of Public Health, EmoryUniversity, Atlanta, GA, 30322, USA. thomas.gillespie@emory.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160921\n",
      "PL  - United States\n",
      "TA  - Ecohealth\n",
      "JT  - EcoHealth\n",
      "JID - 101222144\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Humans\n",
      "MH  - Phylogeny\n",
      "MH  - *Rodent Diseases\n",
      "MH  - Rodentia/*parasitology\n",
      "MH  - Trypanosomiasis/*veterinary\n",
      "MH  - Uganda\n",
      "OTO - NOTNLM\n",
      "OT  - *Kibale National Park\n",
      "OT  - *Praomys\n",
      "OT  - *Rattus\n",
      "OT  - *Trypanosoma lewisi\n",
      "OT  - *Trypanosoma varani\n",
      "OT  - *disease\n",
      "OT  - *forest fragment\n",
      "OT  - *small mammals\n",
      "EDAT- 2016/09/23 06:00\n",
      "MHDA- 2018/02/23 06:00\n",
      "CRDT- 2016/09/23 06:00\n",
      "PHST- 2014/04/09 00:00 [received]\n",
      "PHST- 2016/08/11 00:00 [accepted]\n",
      "PHST- 2016/08/10 00:00 [revised]\n",
      "PHST- 2016/09/23 06:00 [pubmed]\n",
      "PHST- 2018/02/23 06:00 [medline]\n",
      "PHST- 2016/09/23 06:00 [entrez]\n",
      "AID - 10.1007/s10393-016-1160-6 [doi]\n",
      "AID - 10.1007/s10393-016-1160-6 [pii]\n",
      "PST - ppublish\n",
      "SO  - Ecohealth. 2016 Dec;13(4):698-707. doi: 10.1007/s10393-016-1160-6. Epub 2016 Sep 21.\n",
      "\n",
      "PMID- 27622673\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170616\n",
      "LR  - 20190202\n",
      "IS  - 1935-2735 (Electronic)\n",
      "IS  - 1935-2727 (Linking)\n",
      "VI  - 10\n",
      "IP  - 9\n",
      "DP  - 2016 Sep\n",
      "TI  - High Leptospira Diversity in Animals and Humans Complicates the Search for CommonReservoirs of Human Disease in Rural Ecuador.\n",
      "PG  - e0004990\n",
      "LID - 10.1371/journal.pntd.0004990 [doi]\n",
      "AB  - BACKGROUND: Leptospirosis is a zoonotic disease responsible for high morbidityaround the world, especially in tropical and low income countries. Rats arethought to be the main vector of human leptospirosis in urban settings. However, differences between urban and low-income rural communities provide additionalinsights into the epidemiology of the disease. METHODOLOGY/PRINCIPAL FINDINGS:Our study was conducted in two low-income rural communities near the coast ofEcuador. We detected and characterized infectious leptospira DNA in a widevariety of samples using new real time quantitative PCR assays and ampliconsequencing. We detected infectious leptospira in a high percentage of febrilepatients (14.7%). In contrast to previous studies on leptospirosis risk factors, higher positivity was not found in rats (3.0%) but rather in cows (35.8%) andpigs (21.1%). Six leptospira species were identified (L. borgpetersenii, Lkirschnerii, L santarosai, L. interrogans, L noguchii, and an intermediatespecies within the L. licerasiae and L. wolffii clade) and no significantdifferences in the species of leptospira present in each animal species wasdetected (chi2 = 9.89, adj.p-value = 0.27). CONCLUSIONS/SIGNIFICANCE: A largeportion of the world's human population lives in low-income, rural communities,however, there is limited information about leptospirosis transmission dynamicsin these settings. In these areas, exposure to peridomestic livestock isparticularly common and high prevalence of infectious leptospira in cows and pigssuggest that they may be the most important reservoir for human transmission.Genotyping clinical samples show that multiple species of leptospira are involvedin human disease. As these genotypes were also detected in samples from a varietyof animals, genotype data must be used in conjunction with epidemiological datato provide evidence of transmission and the importance of different potentialleptospirosis reservoirs.\n",
      "FAU - Barragan, Veronica\n",
      "AU  - Barragan V\n",
      "AUID- ORCID: 0000-0002-2205-3010\n",
      "AD  - The Center for Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona, United States of America.\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Chiriboga, Jorge\n",
      "AU  - Chiriboga J\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Miller, Erin\n",
      "AU  - Miller E\n",
      "AD  - The Center for Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona, United States of America.\n",
      "FAU - Olivas, Sonora\n",
      "AU  - Olivas S\n",
      "AD  - The Center for Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona, United States of America.\n",
      "FAU - Birdsell, Dawn\n",
      "AU  - Birdsell D\n",
      "AD  - The Center for Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona, United States of America.\n",
      "FAU - Hepp, Crystal\n",
      "AU  - Hepp C\n",
      "AD  - The Center for Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona, United States of America.\n",
      "AD  - Informatics and Computing Program, Northern Arizona University, Flagstaff,Arizona, United States of America.\n",
      "FAU - Hornstra, Heidie\n",
      "AU  - Hornstra H\n",
      "AD  - The Center for Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona, United States of America.\n",
      "FAU - Schupp, James M\n",
      "AU  - Schupp JM\n",
      "AD  - Ministerio de Salud Publica, Quito, Ecuador.\n",
      "FAU - Morales, Melba\n",
      "AU  - Morales M\n",
      "AD  - Ministerio de Salud Publica, Quito, Ecuador.\n",
      "FAU - Gonzalez, Manuel\n",
      "AU  - Gonzalez M\n",
      "AD  - Instituto Nacional de Salud Publica e Investigacion, Guayaquil, Ecuador.\n",
      "FAU - Reyes, Soraya\n",
      "AU  - Reyes S\n",
      "AD  - Ministerio de Salud Publica, Quito, Ecuador.\n",
      "FAU - de la Cruz, Carmen\n",
      "AU  - de la Cruz C\n",
      "AD  - Instituto Nacional de Salud Publica e Investigacion, Guayaquil, Ecuador.\n",
      "FAU - Keim, Paul\n",
      "AU  - Keim P\n",
      "AD  - The Center for Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona, United States of America.\n",
      "AD  - Translational Genomics Research Institute, Flagstaff, Arizona, United States ofAmerica.\n",
      "FAU - Hartskeerl, Rudy\n",
      "AU  - Hartskeerl R\n",
      "AD  - Biomedical Research, Royal Tropical Institute (KIT), Amsterdam, The Netherlands.\n",
      "FAU - Trueba, Gabriel\n",
      "AU  - Trueba G\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Pearson, Talima\n",
      "AU  - Pearson T\n",
      "AD  - The Center for Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona, United States of America.\n",
      "LA  - eng\n",
      "GR  - R15 AI101913/AI/NIAID NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20160913\n",
      "PL  - United States\n",
      "TA  - PLoS Negl Trop Dis\n",
      "JT  - PLoS neglected tropical diseases\n",
      "JID - 101291488\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Cattle\n",
      "MH  - Disease Reservoirs/microbiology\n",
      "MH  - Ecuador/epidemiology\n",
      "MH  - Genotype\n",
      "MH  - Humans\n",
      "MH  - Leptospira/*classification/*genetics\n",
      "MH  - Leptospirosis/*epidemiology/*transmission\n",
      "MH  - Livestock/microbiology\n",
      "MH  - Phylogeny\n",
      "MH  - Poverty\n",
      "MH  - Rats\n",
      "MH  - Regression Analysis\n",
      "MH  - Rural Population\n",
      "MH  - Sequence Analysis, DNA\n",
      "MH  - Swine\n",
      "MH  - Zoonoses/*epidemiology/microbiology\n",
      "PMC - PMC5021363\n",
      "COIS- The authors have declared that no competing interests exist.\n",
      "EDAT- 2016/09/14 06:00\n",
      "MHDA- 2017/06/18 06:00\n",
      "CRDT- 2016/09/14 06:00\n",
      "PHST- 2016/05/10 00:00 [received]\n",
      "PHST- 2016/08/19 00:00 [accepted]\n",
      "PHST- 2016/09/14 06:00 [entrez]\n",
      "PHST- 2016/09/14 06:00 [pubmed]\n",
      "PHST- 2017/06/18 06:00 [medline]\n",
      "AID - 10.1371/journal.pntd.0004990 [doi]\n",
      "AID - PNTD-D-16-00845 [pii]\n",
      "PST - epublish\n",
      "SO  - PLoS Negl Trop Dis. 2016 Sep 13;10(9):e0004990. doi:10.1371/journal.pntd.0004990. eCollection 2016 Sep.\n",
      "\n",
      "PMID- 27577878\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170719\n",
      "LR  - 20210216\n",
      "IS  - 1097-6825 (Electronic)\n",
      "IS  - 0091-6749 (Linking)\n",
      "VI  - 139\n",
      "IP  - 1\n",
      "DP  - 2017 Jan\n",
      "TI  - Primary immunodeficiency diseases: Genomic approaches delineate heterogeneousMendelian disorders.\n",
      "PG  - 232-245\n",
      "LID - S0091-6749(16)30624-8 [pii]\n",
      "LID - 10.1016/j.jaci.2016.05.042 [doi]\n",
      "AB  - BACKGROUND: Primary immunodeficiency diseases (PIDDs) are clinically andgenetically heterogeneous disorders thus far associated with mutations in morethan 300 genes. The clinical phenotypes derived from distinct genotypes canoverlap. Genetic etiology can be a prognostic indicator of disease severity andcan influence treatment decisions. OBJECTIVE: We sought to investigate theability of whole-exome screening methods to detect disease-causing variants inpatients with PIDDs. METHODS: Patients with PIDDs from 278 families from 22countries were investigated by using whole-exome sequencing. Computational copynumber variant (CNV) prediction pipelines and an exome-tiling chromosomalmicroarray were also applied to identify intragenic CNVs. Analytic approachesinitially focused on 475 known or candidate PIDD genes but were nonexclusive and further tailored based on clinical data, family history, and immunophenotyping.RESULTS: A likely molecular diagnosis was achieved in 110 (40%) unrelatedprobands. Clinical diagnosis was revised in about half (60/110) and managementwas directly altered in nearly a quarter (26/110) of families based on molecular findings. Twelve PIDD-causing CNVs were detected, including 7 smaller than 30 Kb that would not have been detected with conventional diagnostic CNV arrays.CONCLUSION: This high-throughput genomic approach enabled detection ofdisease-related variants in unexpected genes; permitted detection of low-gradeconstitutional, somatic, and revertant mosaicism; and provided evidence of amutational burden in mixed PIDD immunophenotypes.\n",
      "CI  - Copyright (c) 2016 American Academy of Allergy, Asthma & Immunology. Published byElsevier Inc. All rights reserved.\n",
      "FAU - Stray-Pedersen, Asbjorg\n",
      "AU  - Stray-Pedersen A\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Center for HumanImmunobiology of Texas Children's Hospital/Department of Pediatrics, BaylorCollege of Medicine, Houston, Tex; Department of Pediatrics, Division ofImmunology, Allergy, and Rheumatology, Baylor College of Medicine, and TexasChildren's Hospital, Houston, Tex; Norwegian National Unit for Newborn Screening,Oslo University Hospital, Oslo, Norway; Department of Pediatrics, Oslo UniversityHospital, Oslo, Norway. Electronic address: astraype@ous-hf.no.\n",
      "FAU - Sorte, Hanne Sormo\n",
      "AU  - Sorte HS\n",
      "AD  - Department of Medical Genetics, Oslo University Hospital, Oslo, Norway; Instituteof Clinical Medicine, University of Oslo, Oslo, Norway.\n",
      "FAU - Samarakoon, Pubudu\n",
      "AU  - Samarakoon P\n",
      "AD  - Department of Medical Genetics, Oslo University Hospital, Oslo, Norway; Instituteof Clinical Medicine, University of Oslo, Oslo, Norway.\n",
      "FAU - Gambin, Tomasz\n",
      "AU  - Gambin T\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Institute of ComputerScience, Warsaw University of Technology, Warsaw, Poland.\n",
      "FAU - Chinn, Ivan K\n",
      "AU  - Chinn IK\n",
      "AD  - Center for Human Immunobiology of Texas Children's Hospital/Department ofPediatrics, Baylor College of Medicine, Houston, Tex; Department of Pediatrics,Division of Immunology, Allergy, and Rheumatology, Baylor College of Medicine,and Texas Children's Hospital, Houston, Tex; Department of Pediatrics, BaylorCollege of Medicine, and Texas Children's Hospital, Houston, Tex.\n",
      "FAU - Coban Akdemir, Zeynep H\n",
      "AU  - Coban Akdemir ZH\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Erichsen, Hans Christian\n",
      "AU  - Erichsen HC\n",
      "AD  - Department of Pediatrics, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Forbes, Lisa R\n",
      "AU  - Forbes LR\n",
      "AD  - Center for Human Immunobiology of Texas Children's Hospital/Department ofPediatrics, Baylor College of Medicine, Houston, Tex; Department of Pediatrics,Division of Immunology, Allergy, and Rheumatology, Baylor College of Medicine,and Texas Children's Hospital, Houston, Tex; Department of Pediatrics, BaylorCollege of Medicine, and Texas Children's Hospital, Houston, Tex.\n",
      "FAU - Gu, Shen\n",
      "AU  - Gu S\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Yuan, Bo\n",
      "AU  - Yuan B\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Department of Molecularand Human Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Jhangiani, Shalini N\n",
      "AU  - Jhangiani SN\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Department of Molecularand Human Genetics, Baylor College of Medicine, Houston, Tex; Human GenomeSequencing Center, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Muzny, Donna M\n",
      "AU  - Muzny DM\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Department of Molecularand Human Genetics, Baylor College of Medicine, Houston, Tex; Human GenomeSequencing Center, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Rodningen, Olaug Kristin\n",
      "AU  - Rodningen OK\n",
      "AD  - Department of Medical Genetics, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Sheng, Ying\n",
      "AU  - Sheng Y\n",
      "AD  - Department of Medical Genetics, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Nicholas, Sarah K\n",
      "AU  - Nicholas SK\n",
      "AD  - Center for Human Immunobiology of Texas Children's Hospital/Department ofPediatrics, Baylor College of Medicine, Houston, Tex; Department of Pediatrics,Division of Immunology, Allergy, and Rheumatology, Baylor College of Medicine,and Texas Children's Hospital, Houston, Tex; Department of Pediatrics, BaylorCollege of Medicine, and Texas Children's Hospital, Houston, Tex.\n",
      "FAU - Noroski, Lenora M\n",
      "AU  - Noroski LM\n",
      "AD  - Department of Pediatrics, Division of Immunology, Allergy, and Rheumatology,Baylor College of Medicine, and Texas Children's Hospital, Houston, Tex;Department of Pediatrics, Baylor College of Medicine, and Texas Children'sHospital, Houston, Tex.\n",
      "FAU - Seeborg, Filiz O\n",
      "AU  - Seeborg FO\n",
      "AD  - Department of Pediatrics, Division of Immunology, Allergy, and Rheumatology,Baylor College of Medicine, and Texas Children's Hospital, Houston, Tex;Department of Pediatrics, Baylor College of Medicine, and Texas Children'sHospital, Houston, Tex.\n",
      "FAU - Davis, Carla M\n",
      "AU  - Davis CM\n",
      "AD  - Department of Pediatrics, Division of Immunology, Allergy, and Rheumatology,Baylor College of Medicine, and Texas Children's Hospital, Houston, Tex;Department of Pediatrics, Baylor College of Medicine, and Texas Children'sHospital, Houston, Tex.\n",
      "FAU - Canter, Debra L\n",
      "AU  - Canter DL\n",
      "AD  - Department of Pediatrics, Division of Immunology, Allergy, and Rheumatology,Baylor College of Medicine, and Texas Children's Hospital, Houston, Tex;Department of Pediatrics, Baylor College of Medicine, and Texas Children'sHospital, Houston, Tex.\n",
      "FAU - Mace, Emily M\n",
      "AU  - Mace EM\n",
      "AD  - Center for Human Immunobiology of Texas Children's Hospital/Department ofPediatrics, Baylor College of Medicine, Houston, Tex; Department of Pediatrics,Division of Immunology, Allergy, and Rheumatology, Baylor College of Medicine,and Texas Children's Hospital, Houston, Tex.\n",
      "FAU - Vece, Timothy J\n",
      "AU  - Vece TJ\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, and Texas Children'sHospital, Houston, Tex.\n",
      "FAU - Allen, Carl E\n",
      "AU  - Allen CE\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, and Texas Children'sHospital, Houston, Tex; Department of Pediatrics, Section of Hematology-Oncology,Baylor College of Medicine, Houston, Tex; Texas Children's Cancer and Hematology Center, Department of Pediatrics, Center for Cell and Gene Therapy, TexasChildren's Hospital and Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Abhyankar, Harshal A\n",
      "AU  - Abhyankar HA\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, and Texas Children'sHospital, Houston, Tex; Department of Pediatrics, Section of Hematology-Oncology,Baylor College of Medicine, Houston, Tex; Texas Children's Cancer and Hematology Center, Department of Pediatrics, Center for Cell and Gene Therapy, TexasChildren's Hospital and Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Boone, Philip M\n",
      "AU  - Boone PM\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Department of Molecularand Human Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Beck, Christine R\n",
      "AU  - Beck CR\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Department of Molecularand Human Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Wiszniewski, Wojciech\n",
      "AU  - Wiszniewski W\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Department of Molecularand Human Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Fevang, Borre\n",
      "AU  - Fevang B\n",
      "AD  - Institute of Clinical Medicine, University of Oslo, Oslo, Norway; Section ofClinical Immunology and Infectious Diseases, Oslo University HospitalRikshospitalet, Oslo, Norway.\n",
      "FAU - Aukrust, Pal\n",
      "AU  - Aukrust P\n",
      "AD  - Institute of Clinical Medicine, University of Oslo, Oslo, Norway; Section ofClinical Immunology and Infectious Diseases, Oslo University HospitalRikshospitalet, Oslo, Norway.\n",
      "FAU - Tjonnfjord, Geir E\n",
      "AU  - Tjonnfjord GE\n",
      "AD  - Institute of Clinical Medicine, University of Oslo, Oslo, Norway; Department ofHematology, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Gedde-Dahl, Tobias\n",
      "AU  - Gedde-Dahl T\n",
      "AD  - Department of Hematology, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Hjorth-Hansen, Henrik\n",
      "AU  - Hjorth-Hansen H\n",
      "AD  - Department of Hematology, St Olavs Hospital, Trondheim, Norway; Department ofCancer Research and Molecular Medicine, Norwegian University of Science andTechnology, Trondheim, Norway.\n",
      "FAU - Dybedal, Ingunn\n",
      "AU  - Dybedal I\n",
      "AD  - Department of Hematology, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Nordoy, Ingvild\n",
      "AU  - Nordoy I\n",
      "AD  - Institute of Clinical Medicine, University of Oslo, Oslo, Norway; Section ofClinical Immunology and Infectious Diseases, Oslo University HospitalRikshospitalet, Oslo, Norway.\n",
      "FAU - Jorgensen, Silje F\n",
      "AU  - Jorgensen SF\n",
      "AD  - Section of Clinical Immunology and Infectious Diseases, Oslo University Hospital Rikshospitalet, Oslo, Norway.\n",
      "FAU - Abrahamsen, Tore G\n",
      "AU  - Abrahamsen TG\n",
      "AD  - Department of Pediatrics, Oslo University Hospital, Oslo, Norway; Institute ofClinical Medicine, University of Oslo, Oslo, Norway.\n",
      "FAU - Overland, Torstein\n",
      "AU  - Overland T\n",
      "AD  - Department of Pediatrics, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Bechensteen, Anne Grete\n",
      "AU  - Bechensteen AG\n",
      "AD  - Department of Pediatrics, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Skogen, Vegard\n",
      "AU  - Skogen V\n",
      "AD  - Department of Infectious Diseases, Medical Clinic, University Hospital ofNorth-Norway, Tromso, Norway.\n",
      "FAU - Osnes, Liv T N\n",
      "AU  - Osnes LTN\n",
      "AD  - Department of Immunology and Transfusion Medicine, Oslo University Hospital,Oslo, Norway.\n",
      "FAU - Kulseth, Mari Ann\n",
      "AU  - Kulseth MA\n",
      "AD  - Department of Medical Genetics, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Prescott, Trine E\n",
      "AU  - Prescott TE\n",
      "AD  - Department of Medical Genetics, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Rustad, Cecilie F\n",
      "AU  - Rustad CF\n",
      "AD  - Department of Medical Genetics, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Heimdal, Ketil R\n",
      "AU  - Heimdal KR\n",
      "AD  - Department of Medical Genetics, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Belmont, John W\n",
      "AU  - Belmont JW\n",
      "AD  - Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Rider, Nicholas L\n",
      "AU  - Rider NL\n",
      "AD  - Center for Human Immunobiology of Texas Children's Hospital/Department ofPediatrics, Baylor College of Medicine, Houston, Tex; Department of Pediatrics,Division of Immunology, Allergy, and Rheumatology, Baylor College of Medicine,and Texas Children's Hospital, Houston, Tex; Department of Pediatrics, BaylorCollege of Medicine, and Texas Children's Hospital, Houston, Tex.\n",
      "FAU - Chinen, Javier\n",
      "AU  - Chinen J\n",
      "AD  - Department of Pediatrics, Division of Immunology, Allergy, and Rheumatology,Baylor College of Medicine, and Texas Children's Hospital, Houston, Tex;Department of Pediatrics, Baylor College of Medicine, and Texas Children'sHospital, Houston, Tex.\n",
      "FAU - Cao, Tram N\n",
      "AU  - Cao TN\n",
      "AD  - Center for Human Immunobiology of Texas Children's Hospital/Department ofPediatrics, Baylor College of Medicine, Houston, Tex; Department of Pediatrics,Division of Immunology, Allergy, and Rheumatology, Baylor College of Medicine,and Texas Children's Hospital, Houston, Tex; Department of Pediatrics, BaylorCollege of Medicine, and Texas Children's Hospital, Houston, Tex.\n",
      "FAU - Smith, Eric A\n",
      "AU  - Smith EA\n",
      "AD  - Department of Molecular and Cellular Biology, Baylor College of Medicine,Houston, Tex.\n",
      "FAU - Caldirola, Maria Soledad\n",
      "AU  - Caldirola MS\n",
      "AD  - Immunology Service, Ricardo Gutierrez Children's Hospital, Ciudad Autonoma deBuenos Aires, Buenos Aires, Argentina.\n",
      "FAU - Bezrodnik, Liliana\n",
      "AU  - Bezrodnik L\n",
      "AD  - Immunology Service, Ricardo Gutierrez Children's Hospital, Ciudad Autonoma deBuenos Aires, Buenos Aires, Argentina.\n",
      "FAU - Lugo Reyes, Saul Oswaldo\n",
      "AU  - Lugo Reyes SO\n",
      "AD  - Immunodeficiencies Research Unit, National Institute of Pediatrics, Coyoacan,Mexico City, Mexico.\n",
      "FAU - Espinosa Rosales, Francisco J\n",
      "AU  - Espinosa Rosales FJ\n",
      "AD  - Immunodeficiencies Research Unit, National Institute of Pediatrics, Coyoacan,Mexico City, Mexico.\n",
      "FAU - Guerrero-Cursaru, Nina Denisse\n",
      "AU  - Guerrero-Cursaru ND\n",
      "AD  - Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Pedroza, Luis Alberto\n",
      "AU  - Pedroza LA\n",
      "AD  - Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Poli, Cecilia M\n",
      "AU  - Poli CM\n",
      "AD  - Center for Human Immunobiology of Texas Children's Hospital/Department ofPediatrics, Baylor College of Medicine, Houston, Tex; Hospital Roberto del Rio,Universidad de Chile, Santiago, Chile.\n",
      "FAU - Franco, Jose L\n",
      "AU  - Franco JL\n",
      "AD  - Grupo de Inmunodeficiencias Primarias, Departamento de Microbiologia yParasitologia, Facultad de Medicina, Universidad de Antioquia UdeA, Medellin,Colombia.\n",
      "FAU - Trujillo Vargas, Claudia M\n",
      "AU  - Trujillo Vargas CM\n",
      "AD  - Grupo de Inmunodeficiencias Primarias, Departamento de Microbiologia yParasitologia, Facultad de Medicina, Universidad de Antioquia UdeA, Medellin,Colombia.\n",
      "FAU - Aldave Becerra, Juan Carlos\n",
      "AU  - Aldave Becerra JC\n",
      "AD  - Allergy and Clinical Immunology, Hospital Nacional Edgardo Rebagliati Martins,Lima, Peru.\n",
      "FAU - Wright, Nicola\n",
      "AU  - Wright N\n",
      "AD  - Alberta Children's Hospital, Calgary, Alberta, Canada.\n",
      "FAU - Issekutz, Thomas B\n",
      "AU  - Issekutz TB\n",
      "AD  - Department of Pediatrics, Dalhousie University, Izaak Walton Killam HealthCentre, Halifax, Nova Scotia, Canada.\n",
      "FAU - Issekutz, Andrew C\n",
      "AU  - Issekutz AC\n",
      "AD  - Department of Pediatrics, Dalhousie University, Izaak Walton Killam HealthCentre, Halifax, Nova Scotia, Canada.\n",
      "FAU - Abbott, Jordan\n",
      "AU  - Abbott J\n",
      "AD  - Department of Pediatrics, National Jewish Health, Denver, Colo.\n",
      "FAU - Caldwell, Jason W\n",
      "AU  - Caldwell JW\n",
      "AD  - Section of Pulmonary, Critical Care, Allergic and Immunological Diseases, WakeForest Baptist Medical Center, Medical Center Boulevard, Winston-Salem, NC.\n",
      "FAU - Bayer, Diana K\n",
      "AU  - Bayer DK\n",
      "AD  - Department of Pediatrics, Division of Pediatric Allergy/Immunology andPulmonology, University of Iowa Carver College of Medicine, Iowa City, Iowa.\n",
      "FAU - Chan, Alice Y\n",
      "AU  - Chan AY\n",
      "AD  - Department of Pediatrics, University of California, San Francisco, Calif.\n",
      "FAU - Aiuti, Alessandro\n",
      "AU  - Aiuti A\n",
      "AD  - San Raffaele Telethon Institute for Gene Therapy (SR-TIGET), and Vita-Salute San Raffaele University, Milan, Italy.\n",
      "FAU - Cancrini, Caterina\n",
      "AU  - Cancrini C\n",
      "AD  - University Department of Pediatrics, DPUO, Bambino Gesu Children's Hospital, and Tor Vergata University, Rome, Italy.\n",
      "FAU - Holmberg, Eva\n",
      "AU  - Holmberg E\n",
      "AD  - Department of Clinical Genetics, University Hospital of Umea, Umea, Sweden.\n",
      "FAU - West, Christina\n",
      "AU  - West C\n",
      "AD  - Department of Clinical Sciences, Pediatrics, Umea University, Umea, Sweden.\n",
      "FAU - Burstedt, Magnus\n",
      "AU  - Burstedt M\n",
      "AD  - Department of Clinical Genetics, University Hospital of Umea, Umea, Sweden.\n",
      "FAU - Karaca, Ender\n",
      "AU  - Karaca E\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Yesil, Gozde\n",
      "AU  - Yesil G\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Department of MedicalGenetics, Bezmi Alem Vakif University Faculty of Medicine, Istanbul, Turkey.\n",
      "FAU - Artac, Hasibe\n",
      "AU  - Artac H\n",
      "AD  - Department of Pediatric Immunology and Allergy, Selcuk University MedicalFaculty, Alaeddin Keykubat Kampusu, Konya, Turkey.\n",
      "FAU - Bayram, Yavuz\n",
      "AU  - Bayram Y\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Atik, Mehmed Musa\n",
      "AU  - Atik MM\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Eldomery, Mohammad K\n",
      "AU  - Eldomery MK\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Ehlayel, Mohammad S\n",
      "AU  - Ehlayel MS\n",
      "AD  - Department of Pediatrics, Section of Pediatric Allergy and Immunology, HamadMedical Corporation, Doha, Department of Paediatrics, Weill Cornell MedicalCollege, Ar-Rayyan, Qatar.\n",
      "FAU - Jolles, Stephen\n",
      "AU  - Jolles S\n",
      "AD  - Immunodeficiency Centre for Wales, University Hospital of Wales, Cardiff, Wales.\n",
      "FAU - Flato, Berit\n",
      "AU  - Flato B\n",
      "AD  - Department of Rheumatology, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Bertuch, Alison A\n",
      "AU  - Bertuch AA\n",
      "AD  - Department of Pediatrics, Baylor College of Medicine, and Texas Children'sHospital, Houston, Tex; Department of Pediatrics, Section of Hematology-Oncology,Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Hanson, I Celine\n",
      "AU  - Hanson IC\n",
      "AD  - Center for Human Immunobiology of Texas Children's Hospital/Department ofPediatrics, Baylor College of Medicine, Houston, Tex; Department of Pediatrics,Division of Immunology, Allergy, and Rheumatology, Baylor College of Medicine,and Texas Children's Hospital, Houston, Tex; Department of Pediatrics, BaylorCollege of Medicine, and Texas Children's Hospital, Houston, Tex.\n",
      "FAU - Zhang, Victor W\n",
      "AU  - Zhang VW\n",
      "AD  - Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Wong, Lee-Jun\n",
      "AU  - Wong LJ\n",
      "AD  - Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Hu, Jianhong\n",
      "AU  - Hu J\n",
      "AD  - Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Tex; Human Genome Sequencing Center, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Walkiewicz, Magdalena\n",
      "AU  - Walkiewicz M\n",
      "AD  - Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Yang, Yaping\n",
      "AU  - Yang Y\n",
      "AD  - Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Eng, Christine M\n",
      "AU  - Eng CM\n",
      "AD  - Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Boerwinkle, Eric\n",
      "AU  - Boerwinkle E\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Human Genome SequencingCenter, Baylor College of Medicine, Houston, Tex; Human Genetics Center,University of Texas School of Public Health, Houston, Tex.\n",
      "FAU - Gibbs, Richard A\n",
      "AU  - Gibbs RA\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Department of Molecularand Human Genetics, Baylor College of Medicine, Houston, Tex; Human GenomeSequencing Center, Baylor College of Medicine, Houston, Tex.\n",
      "FAU - Shearer, William T\n",
      "AU  - Shearer WT\n",
      "AD  - Department of Pediatrics, Division of Immunology, Allergy, and Rheumatology,Baylor College of Medicine, and Texas Children's Hospital, Houston, Tex;Department of Pediatrics, Baylor College of Medicine, and Texas Children'sHospital, Houston, Tex.\n",
      "FAU - Lyle, Robert\n",
      "AU  - Lyle R\n",
      "AD  - Department of Medical Genetics, Oslo University Hospital, Oslo, Norway.\n",
      "FAU - Orange, Jordan S\n",
      "AU  - Orange JS\n",
      "AD  - Center for Human Immunobiology of Texas Children's Hospital/Department ofPediatrics, Baylor College of Medicine, Houston, Tex; Department of Pediatrics,Division of Immunology, Allergy, and Rheumatology, Baylor College of Medicine,and Texas Children's Hospital, Houston, Tex; Department of Pediatrics, BaylorCollege of Medicine, and Texas Children's Hospital, Houston, Tex. Electronicaddress: orange@bcm.edu.\n",
      "FAU - Lupski, James R\n",
      "AU  - Lupski JR\n",
      "AD  - Baylor-Hopkins Center for Mendelian Genomics of the Department of Molecular andHuman Genetics, Baylor College of Medicine, Houston, Tex; Department ofPediatrics, Baylor College of Medicine, and Texas Children's Hospital, Houston,Tex; Department of Molecular and Human Genetics, Baylor College of Medicine,Houston, Tex; Human Genome Sequencing Center, Baylor College of Medicine,Houston, Tex. Electronic address: jlupski@bcm.tmc.edu.\n",
      "LA  - eng\n",
      "GR  - K23 NS078056/NS/NINDS NIH HHS/United States\n",
      "GR  - R01 AI120989/AI/NIAID NIH HHS/United States\n",
      "GR  - UM1 HG006542/HG/NHGRI NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20160716\n",
      "PL  - United States\n",
      "TA  - J Allergy Clin Immunol\n",
      "JT  - The Journal of allergy and clinical immunology\n",
      "JID - 1275002\n",
      "SB  - IM\n",
      "EIN - J Allergy Clin Immunol. 2018 Feb;141(2):832. PMID: 29413258\n",
      "MH  - Adolescent\n",
      "MH  - Adult\n",
      "MH  - Aged\n",
      "MH  - Child\n",
      "MH  - Child, Preschool\n",
      "MH  - DNA Copy Number Variations\n",
      "MH  - Female\n",
      "MH  - Genomics\n",
      "MH  - High-Throughput Nucleotide Sequencing\n",
      "MH  - Humans\n",
      "MH  - Immunologic Deficiency Syndromes/*genetics\n",
      "MH  - Infant\n",
      "MH  - Male\n",
      "MH  - Middle Aged\n",
      "MH  - Young Adult\n",
      "PMC - PMC5222743\n",
      "MID - NIHMS814036\n",
      "OTO - NOTNLM\n",
      "OT  - *Primary immunodeficiency disease\n",
      "OT  - *copy number variants\n",
      "OT  - *whole-exome sequencing\n",
      "EDAT- 2016/09/01 06:00\n",
      "MHDA- 2017/07/20 06:00\n",
      "CRDT- 2016/09/01 06:00\n",
      "PHST- 2016/02/02 00:00 [received]\n",
      "PHST- 2016/04/10 00:00 [revised]\n",
      "PHST- 2016/05/13 00:00 [accepted]\n",
      "PHST- 2016/09/01 06:00 [pubmed]\n",
      "PHST- 2017/07/20 06:00 [medline]\n",
      "PHST- 2016/09/01 06:00 [entrez]\n",
      "AID - 10.1016/j.jaci.2016.05.042 [doi]\n",
      "AID - S0091-6749(16)30624-8 [pii]\n",
      "PST - ppublish\n",
      "SO  - J Allergy Clin Immunol. 2017 Jan;139(1):232-245. doi: 10.1016/j.jaci.2016.05.042.Epub 2016 Jul 16.\n",
      "\n",
      "PMID- 27547576\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "DCOM- 20160822\n",
      "LR  - 20200930\n",
      "IS  - 2167-8359 (Print)\n",
      "IS  - 2167-8359 (Linking)\n",
      "VI  - 4\n",
      "DP  - 2016\n",
      "TI  - Further evidence of Chelonid herpesvirus 5 (ChHV5) latency: high levels of ChHV5 DNA detected in clinically healthy marine turtles.\n",
      "PG  - e2274\n",
      "LID - 10.7717/peerj.2274 [doi]\n",
      "AB  - The Chelonid herpesvirus 5 (ChHV5) has been consistently associated withfibropapillomatosis (FP), a transmissible neoplastic disease of marine turtles.Whether ChHV5 plays a causal role remains debated, partly because while FPtumours have been clearly documented to contain high concentrations of ChHV5 DNA,recent PCR-based studies have demonstrated that large proportions of asymptomaticmarine turtles are also carriers of ChHV5. We used a real-time PCR assay toquantify the levels of ChHV5 Glycoprotein B (gB) DNA in both tumour andnon-tumour skin tissues, from clinically affected and healthy turtles drawn from distant ocean basins across four species. In agreement with previous studies,higher ratios of viral to host DNA were consistently observed in tumour versusnon-tumour tissues in turtles with FP. Unexpectedly however, the levels of ChHV5 gB DNA in clinically healthy turtles were significantly higher than in non-tumourtissues from FP positive turtles. Thus, a large proportion of clinically healthy sea turtle populations worldwide across species carry ChHV5 gB DNA presumablythrough persistent latent infections. ChHV5 appears to be ubiquitous regardlessof the animals' clinical conditions. Hence, these results support the theory thatChHV5 is a near ubiquitous virus with latency characteristics requiringco-factors, possibly environmental or immune related, to induce FP.\n",
      "FAU - Alfaro-Nunez, Alonzo\n",
      "AU  - Alfaro-Nunez A\n",
      "AD  - Section for Evolutionary Genomics, Center for GeoGenetics, Natural History Museumof Denmark, University of Copenhagen, Copenhagen K, Denmark; Laboratorio deBiomedicina, Facultad de Ciencias de la Vida, Escuela Superior Politecnica delLitoral, Guayaquil, Ecuador.\n",
      "FAU - Bojesen, Anders Miki\n",
      "AU  - Bojesen AM\n",
      "AD  - Department of Veterinary Disease Biology, Veterinary Clinical Microbiology,Faculty of Health and Medical Sciences, University of Copenhagen , Frederiskberg , Copenhagen , Denmark.\n",
      "FAU - Bertelsen, Mads F\n",
      "AU  - Bertelsen MF\n",
      "AD  - Center for Zoo and Wild Animal Health, Copenhagen Zoo , Frederiskberg ,Copenhagen , Denmark.\n",
      "FAU - Wales, Nathan\n",
      "AU  - Wales N\n",
      "AD  - Section for Evolutionary Genomics, Center for GeoGenetics, Natural History Museumof Denmark, University of Copenhagen , Copenhagen K , Denmark.\n",
      "FAU - Balazs, George H\n",
      "AU  - Balazs GH\n",
      "AD  - Pacific Islands Fisheries Science Center, National Marine Fisheries Service ,Honolulu , HI , United States of America.\n",
      "FAU - Gilbert, M Thomas P\n",
      "AU  - Gilbert MT\n",
      "AD  - Section for Evolutionary Genomics, Center for GeoGenetics, Natural History Museumof Denmark, University of Copenhagen, Copenhagen K, Denmark; Trace andEnvironmental DNA Laboratory, School of Environment and Agriculture, CurtinUniversity of Technology, Perth, Perth, Australia.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160727\n",
      "PL  - United States\n",
      "TA  - PeerJ\n",
      "JT  - PeerJ\n",
      "JID - 101603425\n",
      "PMC - PMC4974929\n",
      "OTO - NOTNLM\n",
      "OT  - Asymptomatic\n",
      "OT  - Chelonid herpesvirus 5 (ChHV5)\n",
      "OT  - Clinically healthy\n",
      "OT  - Fibropapillomatosis (FP)\n",
      "OT  - Glycoprotein B\n",
      "OT  - Quantitative PCR\n",
      "OT  - Ubiquitous\n",
      "OT  - Viral loads\n",
      "EDAT- 2016/08/23 06:00\n",
      "MHDA- 2016/08/23 06:01\n",
      "CRDT- 2016/08/23 06:00\n",
      "PHST- 2016/03/25 00:00 [received]\n",
      "PHST- 2016/06/29 00:00 [accepted]\n",
      "PHST- 2016/08/23 06:00 [entrez]\n",
      "PHST- 2016/08/23 06:00 [pubmed]\n",
      "PHST- 2016/08/23 06:01 [medline]\n",
      "AID - 10.7717/peerj.2274 [doi]\n",
      "AID - 2274 [pii]\n",
      "PST - epublish\n",
      "SO  - PeerJ. 2016 Jul 27;4:e2274. doi: 10.7717/peerj.2274. eCollection 2016.\n",
      "\n",
      "PMID- 27547186\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "DCOM- 20160822\n",
      "LR  - 20200930\n",
      "IS  - 1663-9812 (Print)\n",
      "IS  - 1663-9812 (Linking)\n",
      "VI  - 7\n",
      "DP  - 2016\n",
      "TI  - Genetic Polymorphisms Associated to Folate Transport as Predictors of IncreasedRisk for Acute Lymphoblastic Leukemia in Mexican Children.\n",
      "PG  - 238\n",
      "LID - 10.3389/fphar.2016.00238 [doi]\n",
      "AB  - Acute lymphoblastic leukemia (ALL) is a frequent neoplasia occurring in children.The most commonly used drug for the treatment of ALL is methotrexate (MTX), ananti-folate agent. Previous studies suggest that folate transporters play a role in ALL prognosis and that genetic polymorphism of genes encoding folatetransporters may increase the risk of ALL. Therefore, the main goal of this studywas to determine the associations among six genetic polymorphisms in four genesrelated with the folate transporter pathway to determine a relationship with the occurrence of ALL in Mexican children. A case-control study was performed in 73ALL children and 133 healthy children from Northern and Northwestern Mexico.COL18A1 (rs2274808), SLC19A1 (rs2838956), ABCB1 (rs1045642 and rs1128503), andABCC5 (rs9838667 and rs3792585). Polymorphisms were assayed through qPCR. Ourresults showed an increased ALL risk in children carrying CT genotype (OR = 2.55,CI 95% 1.11-5.83, p = 0.0001) and TT genotype (OR = 21.05, CI 95% 5.62-78.87, p <0.0001) of COL18A1 rs2274808; in SLC19A1 rs2838956 AG carriers (OR = 44.69, CI95% 10.42-191.63, p = 0.0001); in ABCB1 rs1045642 TT carriers (OR = 13.76, CI 95%5.94-31.88, p = 0.0001); in ABCC5 rs9838667 AC carriers (OR = 2.61, CI 95%1.05-6.48, p < 0.05); and in ABCC5 rs3792585 CC carriers (OR = 9.99, CI 95%3.19-31.28, p = 0.004). Moreover, several combinations of genetic polymorphismswere found to be significantly associated with a risk for ALL. Finally, twocombinations of ABCC5 polymorphisms resulted in protection from this neoplasia.In conclusion, certain genetic polymorphisms related to the folate transportpathway, particularly COL18A1 rs2274808, SLC19A1 rs2838956, ABCB1 rs1045642, and ABCC5 rs3792585, were associated with an increased risk for ALL in Mexicanchildren.\n",
      "FAU - Zaruma-Torres, Fausto\n",
      "AU  - Zaruma-Torres F\n",
      "AD  - Pharmacogenomics Academia, National Polytechnic Institute-CIIDIRDurango, Mexico; School of Biochemistry and Pharmacy, University of CuencaCuenca, Ecuador.\n",
      "FAU - Lares-Asseff, Ismael\n",
      "AU  - Lares-Asseff I\n",
      "AD  - Pharmacogenomics Academia, National Polytechnic Institute-CIIDIR Durango, Mexico.\n",
      "FAU - Lima, Aurea\n",
      "AU  - Lima A\n",
      "AD  - CESPU, Instituto de Investigacao e Formacao Avancada em Ciencias e Tecnologias daSaude Gandra, Portugal.\n",
      "FAU - Reyes-Espinoza, Aaron\n",
      "AU  - Reyes-Espinoza A\n",
      "AD  - State Center of Cancerology of SSA Durango, Mexico.\n",
      "FAU - Loera-Castaneda, Veronica\n",
      "AU  - Loera-Castaneda V\n",
      "AD  - Pharmacogenomics Academia, National Polytechnic Institute-CIIDIR Durango, Mexico.\n",
      "FAU - Sosa-Macias, Martha\n",
      "AU  - Sosa-Macias M\n",
      "AD  - Pharmacogenomics Academia, National Polytechnic Institute-CIIDIR Durango, Mexico.\n",
      "FAU - Galaviz-Hernandez, Carlos\n",
      "AU  - Galaviz-Hernandez C\n",
      "AD  - Pharmacogenomics Academia, National Polytechnic Institute-CIIDIR Durango, Mexico.\n",
      "FAU - Arias-Pelaez, Maria C\n",
      "AU  - Arias-Pelaez MC\n",
      "AD  - Institute of Scientific Research of the University Juarez of State of DurangoDurango, Mexico.\n",
      "FAU - Reyes-Lopez, Miguel A\n",
      "AU  - Reyes-Lopez MA\n",
      "AD  - Center of Biotechnology Genomics, National Polytechnic Institute Reynosa, Mexico.\n",
      "FAU - Quinones, Luis A\n",
      "AU  - Quinones LA\n",
      "AD  - Pharmacological and Molecular Program, Laboratory of Chemical Carcinogenesis and Pharmacogenetics (CQF), Faculty of Medicine, ICBM, University of Chile Santiago, Chile.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160805\n",
      "PL  - Switzerland\n",
      "TA  - Front Pharmacol\n",
      "JT  - Frontiers in pharmacology\n",
      "JID - 101548923\n",
      "PMC - PMC4974492\n",
      "OTO - NOTNLM\n",
      "OT  - acute lymphoblastic leukemia\n",
      "OT  - folate transporters\n",
      "OT  - genetic polymorphisms\n",
      "OT  - methotrexate\n",
      "OT  - molecular epidemiology\n",
      "EDAT- 2016/08/23 06:00\n",
      "MHDA- 2016/08/23 06:01\n",
      "CRDT- 2016/08/23 06:00\n",
      "PHST- 2016/05/13 00:00 [received]\n",
      "PHST- 2016/07/21 00:00 [accepted]\n",
      "PHST- 2016/08/23 06:00 [entrez]\n",
      "PHST- 2016/08/23 06:00 [pubmed]\n",
      "PHST- 2016/08/23 06:01 [medline]\n",
      "AID - 10.3389/fphar.2016.00238 [doi]\n",
      "PST - epublish\n",
      "SO  - Front Pharmacol. 2016 Aug 5;7:238. doi: 10.3389/fphar.2016.00238. eCollection2016.\n",
      "\n",
      "PMID- 27542721\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170901\n",
      "LR  - 20190112\n",
      "IS  - 1471-2164 (Electronic)\n",
      "IS  - 1471-2164 (Linking)\n",
      "VI  - 17\n",
      "IP  - 1\n",
      "DP  - 2016 Aug 19\n",
      "TI  - Transcriptome analysis of smooth cordgrass (Spartina alterniflora Loisel), amonocot halophyte, reveals candidate genes involved in its adaptation tosalinity.\n",
      "PG  - 657\n",
      "LID - 10.1186/s12864-016-3017-3 [doi]\n",
      "AB  - BACKGROUND: Soil salinity affects growth and yield of crop plants. Plants respondto salinity by physiological and biochemical adjustments through a coordinatedregulation and expression of a cascade of genes. Recently, halophytes haveattracted attention of the biologists to understand their salt adaptationmechanisms. Spartina alterniflora (smooth cordgrass) is a Louisiana nativemonocot halophyte that can withstand salinity up to double the strength of seawater. To dissect the molecular mechanisms underlying its salinity adaptation,leaf and root transcriptome of S. alterniflora was sequenced using 454/GS-FLX.RESULTS: Altogether, 770,690 high quality reads with an average length 324-bpwere assembled de novo into 73,131 contigs (average 577-bp long) with 5.9Xsequence coverage. Most unigenes (95 %) annotated to proteins with knownfunctions, and had more than 90 % similarity to rice genes. About 28 % unigeneswere considered specific to S. alterniflora. Digital expression profiles revealedsignificant enrichment (P < 0.01) of transporters, vacuolar proton pump membersand transcription factors under salt stress, which suggested the role of ionhomeostasis and transcriptional regulation in the salinity adaptation of thisgrass. Also, 10,805 SSRs markers from 9457 unigenes were generated and validated through genetic diversity analysis among 13 accessions of S. alterniflora.CONCLUSIONS: The present study explores the transcriptome of S. alterniflora tounderstand the gene regulation under salt stress in halophytes. The sequencedtranscriptome (control and salt-regulated) of S. alterniflora provides a platformfor further gene finding studies in grasses. This study and our previouslypublished studies suggested that S. alterniflora is a rich reservoir of salttolerance genes that can be used to develop salt tolerant cereal crops,especially rice, a major food crop of global importance.\n",
      "FAU - Bedre, Renesh\n",
      "AU  - Bedre R\n",
      "AD  - School of Plant, Environmental and Soil Sciences, Louisiana State UniversityAgricultural Center, Baton Rouge, LA, 70803, USA.\n",
      "FAU - Mangu, Venkata Ramanarao\n",
      "AU  - Mangu VR\n",
      "AD  - School of Plant, Environmental and Soil Sciences, Louisiana State UniversityAgricultural Center, Baton Rouge, LA, 70803, USA.\n",
      "FAU - Srivastava, Subodh\n",
      "AU  - Srivastava S\n",
      "AD  - Department of Genetics and Biochemistry, Clemson University, Clemson, SC, 29634, USA.\n",
      "FAU - Sanchez, Luis Eduardo\n",
      "AU  - Sanchez LE\n",
      "AD  - School of Plant, Environmental and Soil Sciences, Louisiana State UniversityAgricultural Center, Baton Rouge, LA, 70803, USA.\n",
      "AD  - Current address: Centro de Investigaciones Biotecnologicas del Ecuador (CIBE), Km30.5 Via Perimetral, Guayaquil, Ecuador.\n",
      "FAU - Baisakh, Niranjan\n",
      "AU  - Baisakh N\n",
      "AD  - School of Plant, Environmental and Soil Sciences, Louisiana State UniversityAgricultural Center, Baton Rouge, LA, 70803, USA. nbaisakh@agcenter.lsu.edu.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160819\n",
      "PL  - England\n",
      "TA  - BMC Genomics\n",
      "JT  - BMC genomics\n",
      "JID - 100965258\n",
      "SB  - IM\n",
      "MH  - Adaptation, Biological/genetics\n",
      "MH  - Cluster Analysis\n",
      "MH  - Computational Biology/methods\n",
      "MH  - *Gene Expression Profiling\n",
      "MH  - Gene Expression Regulation, Plant\n",
      "MH  - *Genes, Plant\n",
      "MH  - Genome, Plant\n",
      "MH  - Genomics/methods\n",
      "MH  - Molecular Sequence Annotation\n",
      "MH  - Multigene Family\n",
      "MH  - Oryza/genetics\n",
      "MH  - Poaceae/*genetics\n",
      "MH  - Salinity\n",
      "MH  - Salt Tolerance/*genetics\n",
      "MH  - Stress, Physiological/genetics\n",
      "MH  - Synteny\n",
      "MH  - *Transcriptome\n",
      "PMC - PMC4992267\n",
      "OTO - NOTNLM\n",
      "OT  - *GS-FLX\n",
      "OT  - *Gene expression\n",
      "OT  - *Salinity\n",
      "OT  - *Spartina alterniflora\n",
      "OT  - *Transcriptome\n",
      "EDAT- 2016/08/21 06:00\n",
      "MHDA- 2017/09/02 06:00\n",
      "CRDT- 2016/08/21 06:00\n",
      "PHST- 2015/12/23 00:00 [received]\n",
      "PHST- 2016/08/13 00:00 [accepted]\n",
      "PHST- 2016/08/21 06:00 [entrez]\n",
      "PHST- 2016/08/21 06:00 [pubmed]\n",
      "PHST- 2017/09/02 06:00 [medline]\n",
      "AID - 10.1186/s12864-016-3017-3 [doi]\n",
      "AID - 10.1186/s12864-016-3017-3 [pii]\n",
      "PST - epublish\n",
      "SO  - BMC Genomics. 2016 Aug 19;17(1):657. doi: 10.1186/s12864-016-3017-3.\n",
      "\n",
      "PMID- 27321580\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20171121\n",
      "LR  - 20181202\n",
      "IS  - 1876-7737 (Electronic)\n",
      "IS  - 1874-3919 (Linking)\n",
      "VI  - 146\n",
      "DP  - 2016 Sep 2\n",
      "TI  - Peptidomic approach identifies cruzioseptins, a new family of potentantimicrobial peptides in the splendid leaf frog, Cruziohyla calcarifer.\n",
      "PG  - 1-13\n",
      "LID - 10.1016/j.jprot.2016.06.017 [doi]\n",
      "LID - S1874-3919(16)30256-1 [pii]\n",
      "AB  - UNLABELLED: Phyllomedusine frogs are an extraordinary source of biologicallyactive peptides. At least 8 families of antimicrobial peptides have been reportedin this frog clade, the dermaseptins being the most diverse. By a peptidomicapproach, integrating molecular cloning, Edman degradation sequencing and tandem mass spectrometry, a new family of antimicrobial peptides has been identified in Cruziohyla calcarifer. These 15 novel antimicrobial peptides of 20-32 residues inlength are named cruzioseptins. They are characterized by having a unique shared N-terminal sequence GFLD- and the sequence motifs -VALGAVSK- or -GKAAL(N/G/S)(V/A)V- in the middle of the peptide. Cruzioseptins have a broad spectrum ofantimicrobial activity and low haemolytic effect. The most potent cruzioseptinwas CZS-1 that had a MIC of 3.77muM against the Gram positive bacterium,Staphylococcus aureus and the yeast Candida albicans. In contrast, CZS-1 was3-fold less potent against the Gram negative bacterium, Escherichia coli (MIC15.11muM). CZS-1 reached 100% haemolysis at 120.87muM. Skin secretions fromunexplored species such as C. calcarifer continue to demonstrate the enormousmolecular diversity hidden in the amphibian skin. Some of these novel peptidesmay provide lead structures for the development of a new class of antibiotics andantifungals of therapeutic use. BIOLOGICAL SIGNIFICANCE: Through the combination of molecular cloning, Edman degradation sequencing, tandem mass spectrometry and MALDI-TOF MS we have identified a new family of 15 antimicrobial peptides in the skin secretion of Cruziohyla calcarifer. The novel family is named\"Cruzioseptins\" and contains cationic amphipathic peptides of 20-32 residues.They have a broad range of antimicrobial activity that also includes effectiveantifungals with low haemolytic activity. Therefore, C. calcarifer has proven to be a rich source of novel peptides, which could become leading structures for thedevelopment of novel antibiotics and antifungals of clinical application.\n",
      "CI  - Copyright (c) 2016 Elsevier B.V. All rights reserved.\n",
      "FAU - Proano-Bolanos, Carolina\n",
      "AU  - Proano-Bolanos C\n",
      "AD  - Natural Drug Discovery Group, School of Pharmacy, Queen's University Belfast, 97 Lisburn Road, BT9 7BL Belfast, Northern Ireland, UK.\n",
      "FAU - Zhou, Mei\n",
      "AU  - Zhou M\n",
      "AD  - Natural Drug Discovery Group, School of Pharmacy, Queen's University Belfast, 97 Lisburn Road, BT9 7BL Belfast, Northern Ireland, UK.\n",
      "FAU - Wang, Lei\n",
      "AU  - Wang L\n",
      "AD  - Natural Drug Discovery Group, School of Pharmacy, Queen's University Belfast, 97 Lisburn Road, BT9 7BL Belfast, Northern Ireland, UK.\n",
      "FAU - Coloma, Luis A\n",
      "AU  - Coloma LA\n",
      "AD  - Centro Jambatu de Investigacion y Conservacion de Anfibios, Fundacion Otonga,Geovanni Farina 566 y Baltra, San Rafael, Quito, Ecuador; Ikiam, UniversidadRegional Amazonica, Muyuna, Tena, Ecuador.\n",
      "FAU - Chen, Tianbao\n",
      "AU  - Chen T\n",
      "AD  - Natural Drug Discovery Group, School of Pharmacy, Queen's University Belfast, 97 Lisburn Road, BT9 7BL Belfast, Northern Ireland, UK.\n",
      "FAU - Shaw, Chris\n",
      "AU  - Shaw C\n",
      "AD  - Natural Drug Discovery Group, School of Pharmacy, Queen's University Belfast, 97 Lisburn Road, BT9 7BL Belfast, Northern Ireland, UK.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160615\n",
      "PL  - Netherlands\n",
      "TA  - J Proteomics\n",
      "JT  - Journal of proteomics\n",
      "JID - 101475056\n",
      "RN  - 0 (Amphibian Proteins)\n",
      "RN  - 0 (Anti-Infective Agents)\n",
      "RN  - 0 (Antimicrobial Cationic Peptides)\n",
      "RN  - 0 (Peptides)\n",
      "RN  - 136212-91-4 (dermaseptin)\n",
      "SB  - IM\n",
      "MH  - Amphibian Proteins\n",
      "MH  - Animals\n",
      "MH  - Anti-Infective Agents/*isolation & purification/pharmacology\n",
      "MH  - Antimicrobial Cationic Peptides\n",
      "MH  - Candida albicans/drug effects\n",
      "MH  - Cloning, Molecular\n",
      "MH  - Gram-Negative Bacteria/drug effects\n",
      "MH  - Gram-Positive Bacteria/drug effects\n",
      "MH  - Peptides/*analysis/*pharmacology\n",
      "MH  - Proteomics/methods\n",
      "MH  - *Ranidae\n",
      "MH  - Sequence Analysis, Protein\n",
      "MH  - Skin/metabolism\n",
      "MH  - Tandem Mass Spectrometry\n",
      "OTO - NOTNLM\n",
      "OT  - *Antimicrobial peptides\n",
      "OT  - *Cruzioseptins\n",
      "OT  - *Molecular cloning\n",
      "OT  - *Peptidomic\n",
      "OT  - *Skin secretions\n",
      "OT  - *Tandem mass spectrometry\n",
      "EDAT- 2016/06/21 06:00\n",
      "MHDA- 2017/11/29 06:00\n",
      "CRDT- 2016/06/21 06:00\n",
      "PHST- 2016/04/22 00:00 [received]\n",
      "PHST- 2016/05/29 00:00 [revised]\n",
      "PHST- 2016/06/13 00:00 [accepted]\n",
      "PHST- 2016/06/21 06:00 [entrez]\n",
      "PHST- 2016/06/21 06:00 [pubmed]\n",
      "PHST- 2017/11/29 06:00 [medline]\n",
      "AID - S1874-3919(16)30256-1 [pii]\n",
      "AID - 10.1016/j.jprot.2016.06.017 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Proteomics. 2016 Sep 2;146:1-13. doi: 10.1016/j.jprot.2016.06.017. Epub 2016Jun 15.\n",
      "\n",
      "PMID- 27185408\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20171113\n",
      "LR  - 20190112\n",
      "IS  - 2050-084X (Electronic)\n",
      "IS  - 2050-084X (Linking)\n",
      "VI  - 5\n",
      "DP  - 2016 May 17\n",
      "TI  - Mitochondrial genetic diversity, selection and recombination in a caninetransmissible cancer.\n",
      "LID - 10.7554/eLife.14552 [doi]\n",
      "LID - e14552 [pii]\n",
      "AB  - Canine transmissible venereal tumour (CTVT) is a clonally transmissible cancerthat originated approximately 11,000 years ago and affects dogs worldwide.Despite the clonal origin of the CTVT nuclear genome, CTVT mitochondrial genomes (mtDNAs) have been acquired by periodic capture from transient hosts. Wesequenced 449 complete mtDNAs from a global population of CTVTs, and show thatmtDNA horizontal transfer has occurred at least five times, delineating fivetumour clades whose distributions track two millennia of dog global migration.Negative selection has operated to prevent accumulation of deleterious mutations in captured mtDNA, and recombination has caused occasional mtDNA re-assortment.These findings implicate functional mtDNA as a driver of CTVT global metastaticspread, further highlighting the important role of mtDNA in cancer evolution.\n",
      "FAU - Strakova, Andrea\n",
      "AU  - Strakova A\n",
      "AD  - Department of Veterinary Medicine, University of Cambridge, Cambridge, UnitedKingdom.\n",
      "FAU - Ni Leathlobhair, Maire\n",
      "AU  - Ni Leathlobhair M\n",
      "AD  - Department of Veterinary Medicine, University of Cambridge, Cambridge, UnitedKingdom.\n",
      "FAU - Wang, Guo-Dong\n",
      "AU  - Wang GD\n",
      "AD  - State Key Laboratory of Genetic Resources and Evolution, Yunnan Laboratory ofMolecular Biology of Domestic Animals, Kunming Institute of Zoology, ChineseAcademy of Sciences, Kunming, China.\n",
      "FAU - Yin, Ting-Ting\n",
      "AU  - Yin TT\n",
      "AD  - State Key Laboratory of Genetic Resources and Evolution, Yunnan Laboratory ofMolecular Biology of Domestic Animals, Kunming Institute of Zoology, ChineseAcademy of Sciences, Kunming, China.\n",
      "FAU - Airikkala-Otter, Ilona\n",
      "AU  - Airikkala-Otter I\n",
      "AD  - International Training Center, Worldwide Veterinary Service, Aruvankadu, India.\n",
      "FAU - Allen, Janice L\n",
      "AU  - Allen JL\n",
      "AD  - Animal Management in Rural and Remote Indigenous Communities, Darwin, Australia.\n",
      "FAU - Allum, Karen M\n",
      "AU  - Allum KM\n",
      "AD  - World Vets, Fargo, United States.\n",
      "FAU - Bansse-Issa, Leontine\n",
      "AU  - Bansse-Issa L\n",
      "AD  - Stichting Dierenbescherming Suriname, Paramaribo, Suriname.\n",
      "FAU - Bisson, Jocelyn L\n",
      "AU  - Bisson JL\n",
      "AD  - Department of Veterinary Medicine, University of Cambridge, Cambridge, UnitedKingdom.\n",
      "FAU - Castillo Domracheva, Artemio\n",
      "AU  - Castillo Domracheva A\n",
      "AD  - Corozal Veterinary Hospital, University of Panama, Panama City, Panama.\n",
      "FAU - de Castro, Karina F\n",
      "AU  - de Castro KF\n",
      "AD  - Department of Clinical and Veterinary Surgery, Sao Paulo State University, SaoPaulo, Brazil.\n",
      "FAU - Corrigan, Anne M\n",
      "AU  - Corrigan AM\n",
      "AD  - St. George's University, True Blue, Grenada.\n",
      "FAU - Cran, Hugh R\n",
      "AU  - Cran HR\n",
      "AD  - The Nakuru District Veterinary Scheme Ltd, Nakuru, Kenya.\n",
      "FAU - Crawford, Jane T\n",
      "AU  - Crawford JT\n",
      "AD  - Animal Medical Centre, Belize City, Belize.\n",
      "FAU - Cutter, Stephen M\n",
      "AU  - Cutter SM\n",
      "AD  - Animal Management in Rural and Remote Indigenous Communities, Darwin, Australia.\n",
      "FAU - Delgadillo Keenan, Laura\n",
      "AU  - Delgadillo Keenan L\n",
      "AD  - Veterinary clinic Sr. Dog's, Guadalajara, Mexico.\n",
      "FAU - Donelan, Edward M\n",
      "AU  - Donelan EM\n",
      "AD  - Animal Management in Rural and Remote Indigenous Communities, Darwin, Australia.\n",
      "FAU - Faramade, Ibikunle A\n",
      "AU  - Faramade IA\n",
      "AD  - National Veterinary Research Institute, Vom, Nigeria.\n",
      "FAU - Flores Reynoso, Erika\n",
      "AU  - Flores Reynoso E\n",
      "AD  - International Fund for Animal Welfare, Quintana Roo, Mexico.\n",
      "FAU - Fotopoulou, Eleni\n",
      "AU  - Fotopoulou E\n",
      "AD  - Intermunicipal Stray Animals Care Centre, Perama, Greece.\n",
      "FAU - Fruean, Skye N\n",
      "AU  - Fruean SN\n",
      "AD  - Animal Protection Society of Samoa, Apia, Samoa.\n",
      "FAU - Gallardo-Arrieta, Fanny\n",
      "AU  - Gallardo-Arrieta F\n",
      "AD  - Faculty of Veterinary Science, University of Zulia, Maracaibo, Venezuela.\n",
      "FAU - Glebova, Olga\n",
      "AU  - Glebova O\n",
      "AD  - Veterinary clinic BIOCONTROL, Moscow, Russia.\n",
      "FAU - Hafelin Manrique, Rodrigo F\n",
      "AU  - Hafelin Manrique RF\n",
      "AD  - Veterinary clinic El Roble, Santiago de Chile, Chile.\n",
      "FAU - Henriques, Joaquim Jgp\n",
      "AU  - Henriques JJ\n",
      "AD  - OnevetGroup, Centro Veterinario Berna, Lisboa, Portugal.\n",
      "FAU - Ignatenko, Natalia\n",
      "AU  - Ignatenko N\n",
      "AD  - Veterinary clinic Zoovetservis, Kiev, Ukraine.\n",
      "FAU - Koenig, Debbie\n",
      "AU  - Koenig D\n",
      "AD  - World Vets, Fargo, United States.\n",
      "FAU - Lanza-Perea, Marta\n",
      "AU  - Lanza-Perea M\n",
      "AD  - St. George's University, True Blue, Grenada.\n",
      "FAU - Lobetti, Remo\n",
      "AU  - Lobetti R\n",
      "AD  - Bryanston Veterinary Hospital, Bryanston, South Africa.\n",
      "FAU - Lopez Quintana, Adriana M\n",
      "AU  - Lopez Quintana AM\n",
      "AD  - Veterinary Clinic Lopez Quintana, Maldonado, Uruguay.\n",
      "FAU - Losfelt, Thibault\n",
      "AU  - Losfelt T\n",
      "AD  - Clinique Veterinaire de Grand Fond, Saint Gilles les Bains, France.\n",
      "FAU - Marino, Gabriele\n",
      "AU  - Marino G\n",
      "AD  - Department of Veterinary Sciences, University of Messina, Messina, Italy.\n",
      "FAU - Martincorena, Inigo\n",
      "AU  - Martincorena I\n",
      "AD  - Wellcome Trust Sanger Institute, Hinxton, United Kingdom.\n",
      "FAU - Martinez Castaneda, Simon\n",
      "AU  - Martinez Castaneda S\n",
      "AD  - Facultad de Medicina Veterinaria y Zootecnia, Universidad Autonoma del Estado de Mexico, Toluca, Mexico.\n",
      "FAU - Martinez-Lopez, Mayra F\n",
      "AU  - Martinez-Lopez MF\n",
      "AD  - School of Veterinary Medicine, Universidad de las Americas, Quito, Ecuador.\n",
      "FAU - Meyer, Michael\n",
      "AU  - Meyer M\n",
      "AD  - Touray & Meyer Vet Clinic, Serrekunda, Gambia.\n",
      "FAU - Nakanwagi, Berna\n",
      "AU  - Nakanwagi B\n",
      "AD  - The Kampala Veterinary Surgery, Kampala, Uganda.\n",
      "FAU - De Nardi, Andrigo B\n",
      "AU  - De Nardi AB\n",
      "AD  - Department of Clinical and Veterinary Surgery, Sao Paulo State University, SaoPaulo, Brazil.\n",
      "FAU - Neunzig, Winifred\n",
      "AU  - Neunzig W\n",
      "AD  - World Vets, Fargo, United States.\n",
      "FAU - Nixon, Sally J\n",
      "AU  - Nixon SJ\n",
      "AD  - Vets Beyond Borders, The Rocks, Australia.\n",
      "FAU - Onsare, Marsden M\n",
      "AU  - Onsare MM\n",
      "AD  - Aniworld veterinary clinic, Kisumu, Kenya.\n",
      "FAU - Ortega-Pacheco, Antonio\n",
      "AU  - Ortega-Pacheco A\n",
      "AD  - Faculty of Veterinary Medicine, Autonomous University of Yucatan, Merida, Mexico.\n",
      "FAU - Peleteiro, Maria C\n",
      "AU  - Peleteiro MC\n",
      "AD  - Interdisciplinary Centre of Research in Animal Health, Faculty of VeterinaryMedicine, University of Lisbon, Lisboa, Portugal.\n",
      "FAU - Pye, Ruth J\n",
      "AU  - Pye RJ\n",
      "AD  - Vets Beyond Borders, The Rocks, Australia.\n",
      "FAU - Reece, John F\n",
      "AU  - Reece JF\n",
      "AD  - Help in Suffering, Jaipur, India.\n",
      "FAU - Rojas Gutierrez, Jose\n",
      "AU  - Rojas Gutierrez J\n",
      "AD  - Veterinary clinic Dr Jose Rojas, Los Andes, Chile.\n",
      "FAU - Sadia, Haleema\n",
      "AU  - Sadia H\n",
      "AD  - University of Veterinary and Animal Sciences, Lahore, Pakistan.\n",
      "FAU - Schmeling, Sheila K\n",
      "AU  - Schmeling SK\n",
      "AD  - Corozal Veterinary Clinic, Corozal Town, Belize.\n",
      "FAU - Shamanova, Olga\n",
      "AU  - Shamanova O\n",
      "AD  - Veterinary clinic Vetmaster, Ramenskoye, Russia.\n",
      "FAU - Ssuna, Richard K\n",
      "AU  - Ssuna RK\n",
      "AD  - Lilongwe Society for Protection and Care of Animals, Lilongwe, Malawi.\n",
      "FAU - Steenland-Smit, Audrey E\n",
      "AU  - Steenland-Smit AE\n",
      "AD  - Stichting Dierenbescherming Suriname, Paramaribo, Suriname.\n",
      "FAU - Svitich, Alla\n",
      "AU  - Svitich A\n",
      "AD  - State Hospital of Veterinary Medicine, Dniprodzerzhynsk, Ukraine.\n",
      "FAU - Thoya Ngoka, Ismail\n",
      "AU  - Thoya Ngoka I\n",
      "AD  - Kenya Society for Protection and Care of Animals, Nairobi, Kenya.\n",
      "FAU - Vitalaru, Bogdan A\n",
      "AU  - Vitalaru BA\n",
      "AD  - Clinical Sciences Department, Faculty of Veterinary Medicine, Bucharest, Romania.\n",
      "FAU - de Vos, Anna P\n",
      "AU  - de Vos AP\n",
      "AD  - Ladybrand Animal Clinic, Ladybrand, South Africa.\n",
      "FAU - de Vos, Johan P\n",
      "AU  - de Vos JP\n",
      "AD  - Veterinary Oncology Referral Centre De Ottenhorst, Terneuzen, Netherlands.\n",
      "FAU - Walkinton, Oliver\n",
      "AU  - Walkinton O\n",
      "AD  - Vets Beyond Borders, The Rocks, Australia.\n",
      "FAU - Wedge, David C\n",
      "AU  - Wedge DC\n",
      "AD  - Wellcome Trust Sanger Institute, Hinxton, United Kingdom.\n",
      "FAU - Wehrle-Martinez, Alvaro S\n",
      "AU  - Wehrle-Martinez AS\n",
      "AD  - Faculty of Veterinary Sciences, National University of Asuncion, San Lorenzo,Paraguay.\n",
      "FAU - van der Wel, Mirjam G\n",
      "AU  - van der Wel MG\n",
      "AD  - Animal Anti Cruelty League, Port Elizabeth, South Africa.\n",
      "FAU - Widdowson, Sophie Ae\n",
      "AU  - Widdowson SA\n",
      "AD  - Lilongwe Society for Protection and Care of Animals, Lilongwe, Malawi.\n",
      "FAU - Murchison, Elizabeth P\n",
      "AU  - Murchison EP\n",
      "AUID- ORCID: 0000-0001-7462-8907\n",
      "AD  - Department of Veterinary Medicine, University of Cambridge, Cambridge, UnitedKingdom.\n",
      "LA  - eng\n",
      "GR  - Wellcome Trust/United Kingdom\n",
      "GR  - 102942/Z/13/A/Wellcome Trust/United Kingdom\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20160517\n",
      "PL  - England\n",
      "TA  - Elife\n",
      "JT  - eLife\n",
      "JID - 101579614\n",
      "RN  - 0 (DNA, Mitochondrial)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - DNA, Mitochondrial/chemistry/genetics\n",
      "MH  - Dog Diseases/*genetics\n",
      "MH  - Dogs\n",
      "MH  - *Genetic Variation\n",
      "MH  - Mitochondria/*genetics\n",
      "MH  - *Recombination, Genetic\n",
      "MH  - *Selection, Genetic\n",
      "MH  - Sequence Analysis, DNA\n",
      "MH  - Venereal Tumors, Veterinary/*genetics\n",
      "PMC - PMC4869914\n",
      "OTO - NOTNLM\n",
      "OT  - *Dog\n",
      "OT  - *cancer biology\n",
      "OT  - *canine transmissible venereal tumour\n",
      "OT  - *evolutionary biology\n",
      "OT  - *genomics\n",
      "OT  - *mitochondria\n",
      "OT  - *transmissible cancer\n",
      "EDAT- 2016/05/18 06:00\n",
      "MHDA- 2017/11/14 06:00\n",
      "CRDT- 2016/05/18 06:00\n",
      "PHST- 2016/01/19 00:00 [received]\n",
      "PHST- 2016/04/08 00:00 [accepted]\n",
      "PHST- 2016/05/18 06:00 [entrez]\n",
      "PHST- 2016/05/18 06:00 [pubmed]\n",
      "PHST- 2017/11/14 06:00 [medline]\n",
      "AID - 10.7554/eLife.14552 [doi]\n",
      "PST - epublish\n",
      "SO  - Elife. 2016 May 17;5. doi: 10.7554/eLife.14552.\n",
      "\n",
      "PMID- 27151788\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "DCOM- 20160506\n",
      "LR  - 20201001\n",
      "IS  - 2169-8287 (Print)\n",
      "VI  - 4\n",
      "IP  - 3\n",
      "DP  - 2016 May 5\n",
      "TI  - Draft Genome Sequence of the First Pathogenic Leptospira Isolates from Ecuador.\n",
      "LID - 10.1128/genomeA.00271-16 [doi]\n",
      "LID - e00271-16 [pii]\n",
      "AB  - Pathogenic Leptospira spp. cause leptospirosis upon contact with mucosa throughwounds or ingestion, leading to headaches, fever, jaundice, kidney or liverfailure, or death in about 1.3 million people each year. Here, we present thedraft genomes of one L. santarosai isolate and two L. interrogans isolates fromEcuador.\n",
      "CI  - Copyright (c) 2016 Barragan et al.\n",
      "FAU - Barragan, Veronica\n",
      "AU  - Barragan V\n",
      "AD  - The Center for Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona, USA Instituto de Microbiologia, Colegio de CienciasBiologicas y Ambientales, Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Sahl, Jason W\n",
      "AU  - Sahl JW\n",
      "AD  - The Center for Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona, USA.\n",
      "FAU - Wiggins, Kristin\n",
      "AU  - Wiggins K\n",
      "AD  - Translational Genomics Research Institute, Flagstaff, Arizona, USA.\n",
      "FAU - Chiriboga, Jorge\n",
      "AU  - Chiriboga J\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Salinas, Ana\n",
      "AU  - Salinas A\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Cantos, Nancy E\n",
      "AU  - Cantos NE\n",
      "AD  - Ministerio de Salud Publica, Portoviejo, Ecuador.\n",
      "FAU - Loor, Mariana N\n",
      "AU  - Loor MN\n",
      "AD  - Ministerio de Salud Publica, Portoviejo, Ecuador.\n",
      "FAU - Intriago, Bertha I\n",
      "AU  - Intriago BI\n",
      "AD  - Ministerio de Salud Publica, Portoviejo, Ecuador.\n",
      "FAU - Morales, Melba\n",
      "AU  - Morales M\n",
      "AD  - Ministerio de Salud Publica, Portoviejo, Ecuador.\n",
      "FAU - Trueba, Gabriel\n",
      "AU  - Trueba G\n",
      "AD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,Universidad San Francisco de Quito, Quito, Ecuador.\n",
      "FAU - Pearson, Talima\n",
      "AU  - Pearson T\n",
      "AD  - The Center for Microbial Genetics and Genomics, Northern Arizona University,Flagstaff, Arizona, USA talima.pearson@nau.edu.\n",
      "LA  - eng\n",
      "GR  - R15 AI101913/AI/NIAID NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "DEP - 20160505\n",
      "PL  - United States\n",
      "TA  - Genome Announc\n",
      "JT  - Genome announcements\n",
      "JID - 101595808\n",
      "PMC - PMC4859170\n",
      "EDAT- 2016/05/07 06:00\n",
      "MHDA- 2016/05/07 06:01\n",
      "CRDT- 2016/05/07 06:00\n",
      "PHST- 2016/05/07 06:00 [entrez]\n",
      "PHST- 2016/05/07 06:00 [pubmed]\n",
      "PHST- 2016/05/07 06:01 [medline]\n",
      "AID - 4/3/e00271-16 [pii]\n",
      "AID - 10.1128/genomeA.00271-16 [doi]\n",
      "PST - epublish\n",
      "SO  - Genome Announc. 2016 May 5;4(3). pii: 4/3/e00271-16. doi:10.1128/genomeA.00271-16.\n",
      "\n",
      "PMID- 27102486\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20160510\n",
      "LR  - 20160422\n",
      "IS  - 1095-9203 (Electronic)\n",
      "IS  - 0036-8075 (Linking)\n",
      "VI  - 352\n",
      "IP  - 6284\n",
      "DP  - 2016 Apr 22\n",
      "TI  - A beak size locus in Darwin's finches facilitated character displacement during adrought.\n",
      "PG  - 470-4\n",
      "LID - 10.1126/science.aad8786 [doi]\n",
      "AB  - Ecological character displacement is a process of morphological divergence thatreduces competition for limited resources. We used genomic analysis toinvestigate the genetic basis of a documented character displacement event inDarwin's finches on Daphne Major in the Galapagos Islands: The medium groundfinch diverged from its competitor, the large ground finch, during a severedrought. We discovered a genomic region containing the HMGA2 gene that variessystematically among Darwin's finch species with different beak sizes. Twohaplotypes that diverged early in the radiation were involved in the characterdisplacement event: Genotypes associated with large beak size were at a strongselective disadvantage in medium ground finches (selection coefficient s = 0.59).Thus, a major locus has apparently facilitated a rapid ecological diversificationin the adaptive radiation of Darwin's finches.\n",
      "CI  - Copyright (c) 2016, American Association for the Advancement of Science.\n",
      "FAU - Lamichhaney, Sangeet\n",
      "AU  - Lamichhaney S\n",
      "AD  - Department of Medical Biochemistry and Microbiology, Uppsala University, Uppsala,Sweden.\n",
      "FAU - Han, Fan\n",
      "AU  - Han F\n",
      "AD  - Department of Medical Biochemistry and Microbiology, Uppsala University, Uppsala,Sweden.\n",
      "FAU - Berglund, Jonas\n",
      "AU  - Berglund J\n",
      "AD  - Department of Medical Biochemistry and Microbiology, Uppsala University, Uppsala,Sweden.\n",
      "FAU - Wang, Chao\n",
      "AU  - Wang C\n",
      "AD  - Department of Medical Biochemistry and Microbiology, Uppsala University, Uppsala,Sweden.\n",
      "FAU - Almen, Markus Sallman\n",
      "AU  - Almen MS\n",
      "AD  - Department of Medical Biochemistry and Microbiology, Uppsala University, Uppsala,Sweden.\n",
      "FAU - Webster, Matthew T\n",
      "AU  - Webster MT\n",
      "AD  - Department of Medical Biochemistry and Microbiology, Uppsala University, Uppsala,Sweden.\n",
      "FAU - Grant, B Rosemary\n",
      "AU  - Grant BR\n",
      "AD  - Department of Ecology and Evolutionary Biology, Princeton University, Princeton, NJ, USA.\n",
      "FAU - Grant, Peter R\n",
      "AU  - Grant PR\n",
      "AD  - Department of Ecology and Evolutionary Biology, Princeton University, Princeton, NJ, USA.\n",
      "FAU - Andersson, Leif\n",
      "AU  - Andersson L\n",
      "AD  - Department of Medical Biochemistry and Microbiology, Uppsala University, Uppsala,Sweden. Department of Animal Breeding and Genetics, Swedish University ofAgricultural Sciences, Uppsala, Sweden. Department of Veterinary IntegrativeBiosciences, Texas A&M University, College Station, TX, USA.leif.andersson@imbim.uu.se.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "PL  - United States\n",
      "TA  - Science\n",
      "JT  - Science (New York, N.Y.)\n",
      "JID - 0404511\n",
      "RN  - 0 (HMGA2 Protein)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Beak/*anatomy & histology\n",
      "MH  - Body Size/genetics\n",
      "MH  - *Droughts\n",
      "MH  - Ecuador\n",
      "MH  - Female\n",
      "MH  - Finches/*anatomy & histology/classification/*genetics\n",
      "MH  - Genomics\n",
      "MH  - Genotype\n",
      "MH  - HMGA2 Protein/genetics\n",
      "MH  - Haplotypes\n",
      "MH  - Organ Size/genetics\n",
      "MH  - Phylogeny\n",
      "MH  - *Quantitative Trait Loci\n",
      "MH  - *Selection, Genetic\n",
      "EDAT- 2016/04/23 06:00\n",
      "MHDA- 2016/05/11 06:00\n",
      "CRDT- 2016/04/23 06:00\n",
      "PHST- 2015/11/16 00:00 [received]\n",
      "PHST- 2016/03/14 00:00 [accepted]\n",
      "PHST- 2016/04/23 06:00 [entrez]\n",
      "PHST- 2016/04/23 06:00 [pubmed]\n",
      "PHST- 2016/05/11 06:00 [medline]\n",
      "AID - 352/6284/470 [pii]\n",
      "AID - 10.1126/science.aad8786 [doi]\n",
      "PST - ppublish\n",
      "SO  - Science. 2016 Apr 22;352(6284):470-4. doi: 10.1126/science.aad8786.\n",
      "\n",
      "PMID- 27093110\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170109\n",
      "LR  - 20181202\n",
      "IS  - 1557-8100 (Electronic)\n",
      "IS  - 1536-2310 (Linking)\n",
      "VI  - 20\n",
      "IP  - 4\n",
      "DP  - 2016 Apr\n",
      "TI  - An Open Letter in Support of Transformative Biotechnology and Social Innovation: SANKO University Innovation Summit in Medicine and Integrative Biology,Gaziantep, Turkey, May 5-7, 2016.\n",
      "PG  - 259-62\n",
      "LID - 10.1089/omi.2016.0038 [doi]\n",
      "FAU - Saygili, Eyup Ilker\n",
      "AU  - Saygili EI\n",
      "AD  - 1 Faculty of Medicine and Office of the President, SANKO University , Gaziantep, Turkey .\n",
      "FAU - Abou-Zeid, Alaa H\n",
      "AU  - Abou-Zeid AH\n",
      "AD  - 2 Faculty of Medicine, Cairo University , Cairo, Egypt .\n",
      "FAU - Akkin, Salih Murat\n",
      "AU  - Akkin SM\n",
      "AD  - 1 Faculty of Medicine and Office of the President, SANKO University , Gaziantep, Turkey .\n",
      "FAU - Aklillu, Eleni\n",
      "AU  - Aklillu E\n",
      "AD  - 3 Department of Physiology and Pharmacology, Karolinska Institutet , Solna,Sweden .\n",
      "FAU - Barlas, Ibrahim Omer\n",
      "AU  - Barlas IO\n",
      "AD  - 4 Faculty of Medicine, Department of Medical Biology and Genetics, MersinUniversity , Mersin, Turkey .\n",
      "FAU - Borda-Rodriguez, Alexander\n",
      "AU  - Borda-Rodriguez A\n",
      "AD  - 5 Development Policy and Practice Group, The Open University , Milton Keynes,United Kingdom .\n",
      "AD  - 6 Secretaria Nacional de Educacion Superior , Ciencia, Tecnologia e Innovacion,Quito, Ecuador .\n",
      "FAU - Boschele, Filiz Aydogan\n",
      "AU  - Boschele FA\n",
      "AD  - 7 Faculty of Communications, Marmara University , Istanbul, Turkey .\n",
      "FAU - Cetin, Zafer\n",
      "AU  - Cetin Z\n",
      "AD  - 1 Faculty of Medicine and Office of the President, SANKO University , Gaziantep, Turkey .\n",
      "FAU - Coskun, Enes\n",
      "AU  - Coskun E\n",
      "AD  - 8 Faculty of Life Sciences & Medicine, King's College London , London, UnitedKingdom .\n",
      "AD  - 9 Faculty of Medicine, Faculty of Engineering, Gaziantep University , Gaziantep, Turkey .\n",
      "FAU - Coskun, Yavuz\n",
      "AU  - Coskun Y\n",
      "AD  - 9 Faculty of Medicine, Faculty of Engineering, Gaziantep University , Gaziantep, Turkey .\n",
      "AD  - 10 Office of the President, Faculty of Engineering, Gaziantep University ,Gaziantep, Turkey .\n",
      "FAU - Dagli, Guner\n",
      "AU  - Dagli G\n",
      "AD  - 1 Faculty of Medicine and Office of the President, SANKO University , Gaziantep, Turkey .\n",
      "FAU - Dai, Turkan Ugur\n",
      "AU  - Dai TU\n",
      "AD  - 7 Faculty of Communications, Marmara University , Istanbul, Turkey .\n",
      "FAU - Dandara, Collet\n",
      "AU  - Dandara C\n",
      "AD  - 11 Division of Human Genetics, Department of Pathology & Institute of Infectious Disease and Molecular Medicine, Faculty of Health Sciences, University of CapeTown , South Africa .\n",
      "FAU - Dereli, Turkay\n",
      "AU  - Dereli T\n",
      "AD  - 12 Office of the President and Technology Transfer Office, Iskenderun TechnicalUniversity (ISTE) , Iskenderun, Turkey .\n",
      "AD  - 13 Department of Industrial Engineering, Faculty of Engineering, GaziantepUniversity , Gaziantep, Turkey .\n",
      "FAU - Elbeyli, Levent\n",
      "AU  - Elbeyli L\n",
      "AD  - 9 Faculty of Medicine, Faculty of Engineering, Gaziantep University , Gaziantep, Turkey .\n",
      "FAU - Endrenyi, Laszlo\n",
      "AU  - Endrenyi L\n",
      "AD  - 14 Faculty of Medicine, University of Toronto , Toronto, Ontario, Canada .\n",
      "FAU - Eyigun, Can Polat\n",
      "AU  - Eyigun CP\n",
      "AD  - 1 Faculty of Medicine and Office of the President, SANKO University , Gaziantep, Turkey .\n",
      "FAU - Georgakilas, Alexandros\n",
      "AU  - Georgakilas A\n",
      "AD  - 15 Department of Physics, National Technical University of Athens , Athens,Greece .\n",
      "FAU - Gunbulut, Bircan\n",
      "AU  - Gunbulut B\n",
      "AD  - 1 Faculty of Medicine and Office of the President, SANKO University , Gaziantep, Turkey .\n",
      "FAU - Gungor, Kivanc\n",
      "AU  - Gungor K\n",
      "AD  - 9 Faculty of Medicine, Faculty of Engineering, Gaziantep University , Gaziantep, Turkey .\n",
      "FAU - Guzelbey, Asim\n",
      "AU  - Guzelbey A\n",
      "AD  - 16 Independent Scholars, Gaziantep, Turkey .\n",
      "FAU - Hekim, Can\n",
      "AU  - Hekim C\n",
      "AD  - 17 Independent Scholars, Uppsala, Sweden .\n",
      "FAU - Huzair, Farah\n",
      "AU  - Huzair F\n",
      "AD  - 18 School of Social and Political Sciences, The University of Edinburgh ,Edinburgh, United Kingdom .\n",
      "FAU - Kimyon, Sabit\n",
      "AU  - Kimyon S\n",
      "AD  - 9 Faculty of Medicine, Faculty of Engineering, Gaziantep University , Gaziantep, Turkey .\n",
      "FAU - Karakas, Umit\n",
      "AU  - Karakas U\n",
      "AD  - 4 Faculty of Medicine, Department of Medical Biology and Genetics, MersinUniversity , Mersin, Turkey .\n",
      "FAU - Lin, Biaoyang\n",
      "AU  - Lin B\n",
      "AD  - 19 University of Washington , Seattle, Washington, USA.\n",
      "AD  - 20 Systems Biology Division, Zhejiang-California International NanosystemsInstitute (ZCNI) of Zhejiang University , Hangzhou, China .\n",
      "FAU - LLerena, Adrian\n",
      "AU  - LLerena A\n",
      "AD  - 21 Clinical Research Center, Extremadura University Hospital and Medical School ,Hospital Infanta Cristina Av de Elvas s/n, Badajoz, Spain .\n",
      "FAU - Masimirembwa, Collen\n",
      "AU  - Masimirembwa C\n",
      "AD  - 22 African Institute of Biomedical Science & Technology , Harare, Zimbabwe .\n",
      "FAU - McNally, Ruth\n",
      "AU  - McNally R\n",
      "AD  - 23 Anglia Ruskin University , Cambridge, United Kingdom .\n",
      "FAU - Mete, Alper\n",
      "AU  - Mete A\n",
      "AD  - 9 Faculty of Medicine, Faculty of Engineering, Gaziantep University , Gaziantep, Turkey .\n",
      "FAU - Sancar, Pesvin\n",
      "AU  - Sancar P\n",
      "AD  - 24 Independent Scholars, Istanbul, Turkey .\n",
      "FAU - Srivastava, Sanjeeva\n",
      "AU  - Srivastava S\n",
      "AD  - 25 Proteomics Laboratory, Indian Institute of Technology Bombay , Powai, Mumbai, India .\n",
      "FAU - Steuten, Lotte M\n",
      "AU  - Steuten LM\n",
      "AD  - 26 PANAXEA bv, University of Twente , Enschede, The Netherlands .\n",
      "AD  - 27 Fred Hutchinson Cancer Research Center, Hutchinson Institute for CancerOutcomes Research and School of Pharmacy, Pharmaceutical Outcomes Research andPolicy Program, University of Washington , Seattle, Washington, USA.\n",
      "FAU - Tanriover, Oylum\n",
      "AU  - Tanriover O\n",
      "AD  - 7 Faculty of Communications, Marmara University , Istanbul, Turkey .\n",
      "FAU - Tyfield, David\n",
      "AU  - Tyfield D\n",
      "AD  - 28 Centre for Mobilities Research (CeMoRe), Lancaster University , Lancaster,United Kingdom .\n",
      "AD  - 29 International Research & Innovation Centre for the Environment (I-RICE) ,Guangzhou, China .\n",
      "FAU - Tore, Volkan Ihsan\n",
      "AU  - Tore VI\n",
      "AD  - 1 Faculty of Medicine and Office of the President, SANKO University , Gaziantep, Turkey .\n",
      "FAU - Vuruskan, Deniz\n",
      "AU  - Vuruskan D\n",
      "AD  - 30 Gaziantep University Technopark and the Department of Textile Engineering ,Faculty of Engineering, Gaziantep University , Gaziantep, Turkey .\n",
      "FAU - Wang, Wei\n",
      "AU  - Wang W\n",
      "AD  - 31 Global Health Genomics, School of Medical Sciences and Health, Edith CowanUniversity , Perth, Australia .\n",
      "AD  - 32 Beijing Municipal Key laboratory of Clinical Epidemiology, School of PublicHealth, Capital Medical University , Beijing, China .\n",
      "FAU - Warnich, Louise\n",
      "AU  - Warnich L\n",
      "AD  - 33 Faculty of Science, Stellenbosch University , Stellenbosch, South Africa .\n",
      "FAU - Wonkam, Ambroise\n",
      "AU  - Wonkam A\n",
      "AD  - 11 Division of Human Genetics, Department of Pathology & Institute of Infectious Disease and Molecular Medicine, Faculty of Health Sciences, University of CapeTown , South Africa .\n",
      "FAU - Yildirim, Yusuf Ziya\n",
      "AU  - Yildirim YZ\n",
      "AD  - 1 Faculty of Medicine and Office of the President, SANKO University , Gaziantep, Turkey .\n",
      "FAU - Yilmaz, Ismet\n",
      "AU  - Yilmaz I\n",
      "AD  - 34 Faculty of Arts and Sciences, Department of Chemistry, Inonu University ,Malatya, Turkey .\n",
      "FAU - Sinav, Ahmet\n",
      "AU  - Sinav A\n",
      "AD  - 1 Faculty of Medicine and Office of the President, SANKO University , Gaziantep, Turkey .\n",
      "FAU - Hekim, Nezih\n",
      "AU  - Hekim N\n",
      "AD  - 24 Independent Scholars, Istanbul, Turkey .\n",
      "LA  - eng\n",
      "PT  - Congress\n",
      "PT  - Letter\n",
      "PL  - United States\n",
      "TA  - OMICS\n",
      "JT  - Omics : a journal of integrative biology\n",
      "JID - 101131135\n",
      "SB  - IM\n",
      "MH  - *Biotechnology\n",
      "MH  - *Organizational Innovation\n",
      "MH  - Turkey\n",
      "EDAT- 2016/04/20 06:00\n",
      "MHDA- 2017/01/10 06:00\n",
      "CRDT- 2016/04/20 06:00\n",
      "PHST- 2016/04/20 06:00 [entrez]\n",
      "PHST- 2016/04/20 06:00 [pubmed]\n",
      "PHST- 2017/01/10 06:00 [medline]\n",
      "AID - 10.1089/omi.2016.0038 [doi]\n",
      "PST - ppublish\n",
      "SO  - OMICS. 2016 Apr;20(4):259-62. doi: 10.1089/omi.2016.0038.\n",
      "\n",
      "PMID- 27038556\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170130\n",
      "LR  - 20181202\n",
      "IS  - 1873-6254 (Electronic)\n",
      "IS  - 0001-706X (Linking)\n",
      "VI  - 159\n",
      "DP  - 2016 Jul\n",
      "TI  - De novo assembly and characterization of the Trichuris trichiura adult wormtranscriptome using Ion Torrent sequencing.\n",
      "PG  - 132-41\n",
      "LID - 10.1016/j.actatropica.2016.03.036 [doi]\n",
      "LID - S0001-706X(16)30147-4 [pii]\n",
      "AB  - Infection with helminthic parasites, including the soil-transmitted helminthTrichuris trichiura (human whipworm), has been shown to modulate host immuneresponses and, consequently, to have an impact on the development andmanifestation of chronic human inflammatory diseases. De novo derivation ofhelminth proteomes from sequencing of transcriptomes will provide valuable datato aid identification of parasite proteins that could be evaluated as potentialimmunotherapeutic molecules in near future. Herein, we characterized thetranscriptome of the adult stage of the human whipworm T. trichiura, usingnext-generation sequencing technology and a de novo assembly strategy. Nearly17.6 million high-quality clean reads were assembled into 6414 contiguoussequences, with an N50 of 1606bp. In total, 5673 protein-encoding sequences were confidentially identified in the T. trichiura adult worm transcriptome; of these,1013 sequences represent potential newly discovered proteins for the species,most of which presenting orthologs already annotated in the related species T.suis. A number of transcripts representing probable novel non-coding transcripts for the species T. trichiura were also identified. Among the most abundanttranscripts, we found sequences that code for proteins involved in lipidtransport, such as vitellogenins, and several chitin-binding proteins. Through a cross-species expression analysis of gene orthologs shared by T. trichiura andthe closely related parasites T. suis and T. muris it was possible to findtwenty-six protein-encoding genes that are consistently highly expressed in theadult stages of the three helminth species. Additionally, twenty transcriptscould be identified that code for proteins previously detected by massspectrometry analysis of protein fractions of the whipworm somatic extract thatpresent immunomodulatory activities. Five of these transcripts were amongst themost highly expressed protein-encoding sequences in the T. trichiura adult worm. Besides, orthologs of proteins demonstrated to have potent immunomodulatoryproperties in related parasitic helminths were also predicted from the T.trichiura de novo assembled transcriptome.\n",
      "CI  - Copyright (c) 2016. Published by Elsevier B.V.\n",
      "FAU - Santos, Leonardo N\n",
      "AU  - Santos LN\n",
      "AD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil.\n",
      "FAU - Silva, Eduardo S\n",
      "AU  - Silva ES\n",
      "AD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil.\n",
      "FAU - Santos, Andre S\n",
      "AU  - Santos AS\n",
      "AD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil.\n",
      "FAU - De Sa, Pablo H\n",
      "AU  - De Sa PH\n",
      "AD  - Institute of Biological Sciences, Federal University of Para, Belem, PA, Brazil.\n",
      "FAU - Ramos, Rommel T\n",
      "AU  - Ramos RT\n",
      "AD  - Institute of Biological Sciences, Federal University of Para, Belem, PA, Brazil.\n",
      "FAU - Silva, Artur\n",
      "AU  - Silva A\n",
      "AD  - Institute of Biological Sciences, Federal University of Para, Belem, PA, Brazil.\n",
      "FAU - Cooper, Philip J\n",
      "AU  - Cooper PJ\n",
      "AD  - Institute of Infection and Immunity, St. George's University of London, London,UK; Centro de Investigacion en Enfermedades Infecciosas y Cronicas, PontificiaUniversidad Catolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Barreto, Mauricio L\n",
      "AU  - Barreto ML\n",
      "AD  - Institute of Public Health, Federal University of Bahia, Salvador, BA, Brazil;Centro de Pesquisas Goncalo Muniz, FIOCRUZ-BA, Salvador, BA, Brazil.\n",
      "FAU - Loureiro, Sebastiao\n",
      "AU  - Loureiro S\n",
      "AD  - Institute of Public Health, Federal University of Bahia, Salvador, BA, Brazil.\n",
      "FAU - Pinheiro, Carina S\n",
      "AU  - Pinheiro CS\n",
      "AD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil.\n",
      "FAU - Alcantara-Neves, Neuza M\n",
      "AU  - Alcantara-Neves NM\n",
      "AD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil. Electronic address: neuza@ufba.br.\n",
      "FAU - Pacheco, Luis G C\n",
      "AU  - Pacheco LG\n",
      "AD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil. Electronic address: luis.pacheco@ufba.br.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160331\n",
      "PL  - Netherlands\n",
      "TA  - Acta Trop\n",
      "JT  - Acta tropica\n",
      "JID - 0370374\n",
      "RN  - 0 (Antigens, Helminth)\n",
      "SB  - IM\n",
      "MH  - Adolescent\n",
      "MH  - Amino Acid Sequence\n",
      "MH  - Animals\n",
      "MH  - Antigens, Helminth/*genetics\n",
      "MH  - Child\n",
      "MH  - Child, Preschool\n",
      "MH  - Ecuador\n",
      "MH  - Female\n",
      "MH  - Humans\n",
      "MH  - Infant\n",
      "MH  - Infant, Newborn\n",
      "MH  - Male\n",
      "MH  - Transcriptome/*genetics\n",
      "MH  - Trichuriasis/*parasitology\n",
      "MH  - Trichuris/*genetics\n",
      "OTO - NOTNLM\n",
      "OT  - Functional genomics\n",
      "OT  - Next-generation sequencing\n",
      "OT  - Transcriptome\n",
      "OT  - Trichuris trichiura\n",
      "EDAT- 2016/04/04 06:00\n",
      "MHDA- 2017/01/31 06:00\n",
      "CRDT- 2016/04/04 06:00\n",
      "PHST- 2015/07/10 00:00 [received]\n",
      "PHST- 2016/03/23 00:00 [revised]\n",
      "PHST- 2016/03/30 00:00 [accepted]\n",
      "PHST- 2016/04/04 06:00 [entrez]\n",
      "PHST- 2016/04/04 06:00 [pubmed]\n",
      "PHST- 2017/01/31 06:00 [medline]\n",
      "AID - S0001-706X(16)30147-4 [pii]\n",
      "AID - 10.1016/j.actatropica.2016.03.036 [doi]\n",
      "PST - ppublish\n",
      "SO  - Acta Trop. 2016 Jul;159:132-41. doi: 10.1016/j.actatropica.2016.03.036. Epub 2016Mar 31.\n",
      "\n",
      "PMID- 26995696\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170130\n",
      "LR  - 20181202\n",
      "IS  - 1873-6254 (Electronic)\n",
      "IS  - 0001-706X (Linking)\n",
      "VI  - 159\n",
      "DP  - 2016 Jul\n",
      "TI  - Haemoproteus erythrogravidus n. sp. (Haemosporida, Haemoproteidae): Descriptionand molecular characterization of a widespread blood parasite of birds in SouthAmerica.\n",
      "PG  - 83-94\n",
      "LID - 10.1016/j.actatropica.2016.02.025 [doi]\n",
      "LID - S0001-706X(16)30087-0 [pii]\n",
      "AB  - The great diversity of birds and ecosystems in the Andean mountains has beenunderstudied in terms of their parasite species. We describe a new Haemoproteusparasite, H. (Parahaemoproteus) erythrogravidus infecting Zonotrichia capensis(Rufous-Collared Sparrow) in South America. The description of this bloodparasite species is supported by morphological and molecular data based on afragment of cytochrome b gene (cyt b) and complete mitochondrial genomesequences. The new species is closely related to H. (Parahaemoproteus) coatneyi, and it can be readily distinguished from the latter parasite due to morphology ofits blood stages, particularly 1) the formation of a marked protrusion onenvelope of infected erythrocytes by the majority of developing gametocytes, afeature which is unique for this Haemoproteus species and 2) the extremelyattenuated width of the growing dumbbell-shaped macro- and microgametocytes.Additionally, Haemoproteus erythrogravidus is shown to be a monophyletic taxonthat diverges from Haemoproteus coatneyi at the molecular level. We provide thecomplete mitochondrial DNA genome for both H. coatneyi and H. erythrogravidus.Molecular and morphological evidences indicate that H. erythrogravidus is presentin Ecuador and Colombia, and genetic lineages with 100% of identity for the cyt bgene were reported in Chile, Peru, and Venezuela. Our study also indicates thatH. erythrogravidus and H. coatneyi are sympatric sister taxa sharing Z. capensis as a host species across its distribution, which could be the result of sympatricspeciation or complex biogeographic processes. Further studies on thedistribution and evolutionary history of Z. capensis and its parasites H.erythrogravidus and H. coatneyi insight for our better understanding of thefactors and dynamics driving parasite speciation.\n",
      "CI  - Copyright (c) 2016 Elsevier B.V. All rights reserved.\n",
      "FAU - Mantilla, Juan S\n",
      "AU  - Mantilla JS\n",
      "AD  - Universidad Nacional de Colombia, Sede Bogota-Facultad de Ciencias, Departamento de Biologia, Grupo de Investigacion Caracterizacion genetica e inmunologia,Carrera 30 No. 45-03, Bogota 111321, Colombia.\n",
      "FAU - Gonzalez, Angie D\n",
      "AU  - Gonzalez AD\n",
      "AD  - Universidad Nacional de Colombia, Sede Bogota-Facultad de Ciencias, Departamento de Biologia, Grupo de Investigacion Caracterizacion genetica e inmunologia,Carrera 30 No. 45-03, Bogota 111321, Colombia.\n",
      "FAU - Lotta, Ingrid A\n",
      "AU  - Lotta IA\n",
      "AD  - Universidad Nacional de Colombia, Sede Bogota-Facultad de Ciencias, Departamento de Biologia, Grupo de Investigacion Caracterizacion genetica e inmunologia,Carrera 30 No. 45-03, Bogota 111321, Colombia.\n",
      "FAU - Moens, Michael\n",
      "AU  - Moens M\n",
      "AD  - Departamento de Zoologia y Antropologia Fisica, Facultad de Biologia, UniversidadComplutense de Madrid, Calle Jose Antonio Novais 12, 28040 Madrid, Spain.\n",
      "FAU - Pacheco, M Andreina\n",
      "AU  - Pacheco MA\n",
      "AD  - Institute for Genomics and Evolutionary Medicine (igem), Temple University,Philadelphia, Pennsylvania, USA.\n",
      "FAU - Escalante, Ananias A\n",
      "AU  - Escalante AA\n",
      "AD  - Institute for Genomics and Evolutionary Medicine (igem), Temple University,Philadelphia, Pennsylvania, USA.\n",
      "FAU - Valkiunas, Gediminas\n",
      "AU  - Valkiunas G\n",
      "AD  - Nature Research Centre, Akademijos 2, LT-08412 Vilnius, Lithuania.\n",
      "FAU - Moncada, Ligia I\n",
      "AU  - Moncada LI\n",
      "AD  - Universidad Nacional de Colombia, Sede Bogota-Facultad de Medicina, Departamento de Salud Publica-Grupo de Investigacion Infecciones y Salud en el Tropico,Carrera 30 No. 45-03, Bogota 111321, Colombia.\n",
      "FAU - Perez-Tris, Javier\n",
      "AU  - Perez-Tris J\n",
      "AD  - Departamento de Zoologia y Antropologia Fisica, Facultad de Biologia, UniversidadComplutense de Madrid, Calle Jose Antonio Novais 12, 28040 Madrid, Spain.\n",
      "FAU - Matta, Nubia E\n",
      "AU  - Matta NE\n",
      "AD  - Universidad Nacional de Colombia, Sede Bogota-Facultad de Ciencias, Departamento de Biologia, Grupo de Investigacion Caracterizacion genetica e inmunologia,Carrera 30 No. 45-03, Bogota 111321, Colombia. Electronic address:nemattac@unal.edu.co.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160316\n",
      "PL  - Netherlands\n",
      "TA  - Acta Trop\n",
      "JT  - Acta tropica\n",
      "JID - 0370374\n",
      "RN  - 0 (DNA, Mitochondrial)\n",
      "RN  - 0 (DNA, Protozoan)\n",
      "RN  - 9035-37-4 (Cytochromes b)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Bird Diseases/*parasitology\n",
      "MH  - Birds/*parasitology\n",
      "MH  - Cytochromes b/genetics\n",
      "MH  - DNA, Mitochondrial\n",
      "MH  - DNA, Protozoan/*genetics\n",
      "MH  - Genome, Mitochondrial\n",
      "MH  - Haemosporida/*classification/*genetics\n",
      "MH  - Parasites/*classification\n",
      "MH  - Phylogeny\n",
      "MH  - Protozoan Infections, Animal/*parasitology\n",
      "MH  - South America\n",
      "OTO - NOTNLM\n",
      "OT  - Avian malaria\n",
      "OT  - Blood parasite\n",
      "OT  - Haemoproteus\n",
      "OT  - Haemosporida\n",
      "OT  - Neotropics\n",
      "OT  - New species\n",
      "OT  - Zonotrichia capensis\n",
      "EDAT- 2016/03/21 06:00\n",
      "MHDA- 2017/01/31 06:00\n",
      "CRDT- 2016/03/21 06:00\n",
      "PHST- 2015/09/06 00:00 [received]\n",
      "PHST- 2016/02/15 00:00 [revised]\n",
      "PHST- 2016/02/29 00:00 [accepted]\n",
      "PHST- 2016/03/21 06:00 [entrez]\n",
      "PHST- 2016/03/21 06:00 [pubmed]\n",
      "PHST- 2017/01/31 06:00 [medline]\n",
      "AID - S0001-706X(16)30087-0 [pii]\n",
      "AID - 10.1016/j.actatropica.2016.02.025 [doi]\n",
      "PST - ppublish\n",
      "SO  - Acta Trop. 2016 Jul;159:83-94. doi: 10.1016/j.actatropica.2016.02.025. Epub 2016 Mar 16.\n",
      "\n",
      "PMID- 26977404\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "DCOM- 20160315\n",
      "LR  - 20201001\n",
      "IS  - 2213-2244 (Print)\n",
      "IS  - 2213-2244 (Linking)\n",
      "VI  - 5\n",
      "IP  - 1\n",
      "DP  - 2016 Apr\n",
      "TI  - Sexual differences in prevalence of a new species of trypanosome infectingtungara frogs.\n",
      "PG  - 40-7\n",
      "LID - 10.1016/j.ijppaw.2016.01.005 [doi]\n",
      "AB  - Trypanosomes are a diverse group of protozoan parasites of vertebratestransmitted by a variety of hematophagous invertebrate vectors. Anurantrypanosomes and their vectors have received relatively little attention eventhough these parasites have been reported from frog and toad species worldwide.Blood samples collected from tungara frogs (Engystomops pustulosus), aNeotropical anuran species heavily preyed upon by eavesdropping frog-bitingmidges (Corethrella spp.), were examined for trypanosomes. Our results revealedsexual differences in trypanosome prevalence with female frogs being rarelyinfected (<1%). This finding suggests this protozoan parasite may be transmitted by frog-biting midges that find their host using the mating calls produced bymale frogs. Following previous anuran trypanosome studies, we examined 18Sribosomal RNA gene to characterize and establish the phylogenetic relationship ofthe trypanosome species found in tungara frogs. A new species of gianttrypanosome, Trypanosoma tungarae n. sp., is described in this study. Overall themorphometric data revealed that the trypomastigotes of T. tungarae n. sp. aresimilar to other giant trypanosomes such as Trypanosoma rotatorium andTrypanosoma ranarum. Despite its slender and long cell shape, however, 18S rRNAgene sequences revealed that T. tungarae n. sp. is sister to the rounded-bodiedgiant trypanosome, Trypanosoma chattoni. Therefore, morphological convergenceexplains similar morphology among members of two non-closely related groups oftrypanosomes infecting frogs. The results from this study underscore the value ofcoupling morphological identification with molecular characterization of anurantrypanosomes.\n",
      "FAU - Bernal, Ximena E\n",
      "AU  - Bernal XE\n",
      "AD  - Department of Biological Sciences, Purdue University, West Lafayette, IN, 47907, USA; Smithsonian Tropical Research Institute, Apartado 2072, Balboa, Panama.\n",
      "FAU - Pinto, C Miguel\n",
      "AU  - Pinto CM\n",
      "AD  - Sackler Institute for Comparative Genomics and Department of Mammalogy, American Museum of Natural History, New York, NY, 10024, USA; Centro de Investigacion enEnfermedades Infecciosas, Pontificia Universidad Catolica del Ecuador, Quito,Ecuador; Division of Mammals, National Museum of Natural History, SmithsonianInstitution, Washington, DC, 20004, USA.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160121\n",
      "PL  - England\n",
      "TA  - Int J Parasitol Parasites Wildl\n",
      "JT  - International journal for parasitology. Parasites and wildlife\n",
      "JID - 101599824\n",
      "PMC - PMC4781969\n",
      "OTO - NOTNLM\n",
      "OT  - Corethrella\n",
      "OT  - Engystomops pustulosus\n",
      "OT  - Frog-biting midges\n",
      "OT  - Panama\n",
      "OT  - Physalaemus\n",
      "OT  - Species delimitation\n",
      "OT  - Trypanosome phylogeny\n",
      "EDAT- 2016/03/16 06:00\n",
      "MHDA- 2016/03/16 06:01\n",
      "CRDT- 2016/03/16 06:00\n",
      "PHST- 2015/08/24 00:00 [received]\n",
      "PHST- 2015/12/25 00:00 [revised]\n",
      "PHST- 2016/01/17 00:00 [accepted]\n",
      "PHST- 2016/03/16 06:00 [entrez]\n",
      "PHST- 2016/03/16 06:00 [pubmed]\n",
      "PHST- 2016/03/16 06:01 [medline]\n",
      "AID - 10.1016/j.ijppaw.2016.01.005 [doi]\n",
      "AID - S2213-2244(16)30005-0 [pii]\n",
      "PST - epublish\n",
      "SO  - Int J Parasitol Parasites Wildl. 2016 Jan 21;5(1):40-7. doi:10.1016/j.ijppaw.2016.01.005. eCollection 2016 Apr.\n",
      "\n",
      "PMID- 26961748\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20160930\n",
      "LR  - 20190110\n",
      "IS  - 1755-8794 (Electronic)\n",
      "IS  - 1755-8794 (Linking)\n",
      "VI  - 9\n",
      "DP  - 2016 Mar 9\n",
      "TI  - Efficient and biologically relevant consensus strategy for Parkinson's diseasegene prioritization.\n",
      "PG  - 12\n",
      "LID - 10.1186/s12920-016-0173-x [doi]\n",
      "AB  - BACKGROUND: The systemic information enclosed in microarray data encodes relevantclues to overcome the poorly understood combination of genetic and environmental factors in Parkinson's disease (PD), which represents the major obstacle tounderstand its pathogenesis and to develop disease-modifying therapeutics. While several gene prioritization approaches have been proposed, none dominate over therest. Instead, hybrid approaches seem to outperform individual approaches.METHODS: A consensus strategy is proposed for PD related gene prioritization frommRNA microarray data based on the combination of three independent prioritizationapproaches: Limma, machine learning, and weighted gene co-expression networks.RESULTS: The consensus strategy outperformed the individual approaches in termsof statistical significance, overall enrichment and early recognition ability. Inaddition to a significant biological relevance, the set of 50 genes prioritizedexhibited an excellent early recognition ability (6 of the top 10 genes aredirectly associated with PD). 40 % of the prioritized genes were previouslyassociated with PD including well-known PD related genes such as SLC18A2, TH orDRD2. Eight genes (CCNH, DLK1, PCDH8, SLIT1, DLD, PBX1, INSM1, and BMI1) werefound to be significantly associated to biological process affected in PD,representing potentially novel PD biomarkers or therapeutic targets.Additionally, several metrics of standard use in chemoinformatics are proposed toevaluate the early recognition ability of gene prioritization tools. CONCLUSIONS:The proposed consensus strategy represents an efficient and biologically relevantapproach for gene prioritization tasks providing a valuable decision-making tool for the study of PD pathogenesis and the development of disease-modifying PDtherapeutics.\n",
      "FAU - Cruz-Monteagudo, Maykel\n",
      "AU  - Cruz-Monteagudo M\n",
      "AD  - CIQUP/Departamento de Quimica e Bioquimica, Faculdade de Ciencias, Universidadedo Porto, Porto, 4169-007, Portugal. gmailkelcm@yahoo.es.\n",
      "AD  - Instituto de Investigaciones Biomedicas (IIB), Universidad de Las Americas,170513, Quito, Ecuador. gmailkelcm@yahoo.es.\n",
      "FAU - Borges, Fernanda\n",
      "AU  - Borges F\n",
      "AD  - CIQUP/Departamento de Quimica e Bioquimica, Faculdade de Ciencias, Universidadedo Porto, Porto, 4169-007, Portugal. fborges@fc.up.pt.\n",
      "FAU - Paz-Y-Mino, Cesar\n",
      "AU  - Paz-Y-Mino C\n",
      "AD  - Instituto de Investigaciones Biomedicas (IIB), Universidad de Las Americas,170513, Quito, Ecuador. cesar.pazymino@udla.edu.ec.\n",
      "FAU - Cordeiro, M Natalia D S\n",
      "AU  - Cordeiro MN\n",
      "AD  - REQUIMTE, Department of Chemistry and Biochemistry, Faculty of Sciences,University of Porto, 4169-007, Porto, Portugal. ncordeir@fc.up.pt.\n",
      "FAU - Rebelo, Irene\n",
      "AU  - Rebelo I\n",
      "AD  - REQUIMTE, Department of Biochemistry, Faculty of Pharmacy, University of Porto,4050-313, Porto, Portugal. irebelo@ff.up.pt.\n",
      "FAU - Perez-Castillo, Yunierkis\n",
      "AU  - Perez-Castillo Y\n",
      "AD  - Seccion Fisico Quimica y Matematicas, Departamento de Quimica, UniversidadTecnica Particular de Loja, San Cayetano Alto S/N, EC1101608, Loja, Ecuador.yunierkis@gmail.com.\n",
      "AD  - Molecular Simulation and Drug Design Group, Centro de Bioactivos Quimicos (CBQ), Central University of Las Villas, Santa Clara, 54830, Cuba. yunierkis@gmail.com.\n",
      "FAU - Helguera, Aliuska Morales\n",
      "AU  - Helguera AM\n",
      "AD  - Molecular Simulation and Drug Design Group, Centro de Bioactivos Quimicos (CBQ), Central University of Las Villas, Santa Clara, 54830, Cuba. aliuska@uclv.edu.cu.\n",
      "FAU - Sanchez-Rodriguez, Aminael\n",
      "AU  - Sanchez-Rodriguez A\n",
      "AD  - Departamento de Ciencias Naturales, Universidad Tecnica Particular de Loja, CalleParis S/N, EC1101608, Loja, Ecuador. asanchez2@utpl.edu.ec.\n",
      "FAU - Tejera, Eduardo\n",
      "AU  - Tejera E\n",
      "AD  - Instituto de Investigaciones Biomedicas (IIB), Universidad de Las Americas,170513, Quito, Ecuador. eduardo.tejera@udla.edu.ec.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20160309\n",
      "PL  - England\n",
      "TA  - BMC Med Genomics\n",
      "JT  - BMC medical genomics\n",
      "JID - 101319628\n",
      "SB  - IM\n",
      "MH  - Algorithms\n",
      "MH  - Case-Control Studies\n",
      "MH  - Gene Expression Regulation\n",
      "MH  - Gene Ontology\n",
      "MH  - Gene Regulatory Networks\n",
      "MH  - Genetic Association Studies\n",
      "MH  - *Genetic Predisposition to Disease\n",
      "MH  - Humans\n",
      "MH  - Machine Learning\n",
      "MH  - Oligonucleotide Array Sequence Analysis\n",
      "MH  - Parkinson Disease/*genetics\n",
      "MH  - Reproducibility of Results\n",
      "PMC - PMC4784386\n",
      "EDAT- 2016/03/11 06:00\n",
      "MHDA- 2016/10/01 06:00\n",
      "CRDT- 2016/03/11 06:00\n",
      "PHST- 2015/09/03 00:00 [received]\n",
      "PHST- 2016/03/01 00:00 [accepted]\n",
      "PHST- 2016/03/11 06:00 [entrez]\n",
      "PHST- 2016/03/11 06:00 [pubmed]\n",
      "PHST- 2016/10/01 06:00 [medline]\n",
      "AID - 10.1186/s12920-016-0173-x [doi]\n",
      "AID - 10.1186/s12920-016-0173-x [pii]\n",
      "PST - epublish\n",
      "SO  - BMC Med Genomics. 2016 Mar 9;9:12. doi: 10.1186/s12920-016-0173-x.\n",
      "\n",
      "PMID- 26947765\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20170901\n",
      "LR  - 20181202\n",
      "IS  - 1467-7652 (Electronic)\n",
      "IS  - 1467-7644 (Linking)\n",
      "VI  - 14\n",
      "IP  - 9\n",
      "DP  - 2016 Sep\n",
      "TI  - Production of highly bioactive resveratrol analogues pterostilbene andpiceatannol in metabolically engineered grapevine cell cultures.\n",
      "PG  - 1813-25\n",
      "LID - 10.1111/pbi.12539 [doi]\n",
      "AB  - Grapevine stilbenes, particularly trans-resveratrol, have a demonstratedpharmacological activity. Other natural stilbenes derived from resveratrol suchas pterostilbene or piceatannol, display higher oral bioavailability andbioactivity than the parent compound, but are far less abundant in naturalsources. Thus, to efficiently obtain these bioactive resveratrol derivatives,there is a need to develop new bioproduction systems. Grapevine cell cultures areable to produce large amounts of easily recoverable extracellular resveratrolwhen elicited with methylated cyclodextrins and methyl jasmonate. We devised thissystem as an interesting starting point of a metabolic engineering-based strategyto produce resveratrol derivatives using resveratrol-converting enzymes.Constitutive expression of either Vitis vinifera resveratrol O-methyltransferase (VvROMT) or human cytochrome P450 hydroxylase 1B1 (HsCYP1B1) led to pterostilbeneor piceatannol, respectively, after the engineered cell cultures were treatedwith the aforementioned elicitors. Functionality of both gene products was first assessed in planta by Nicotiana benthamiana agroinfiltration assays, in whichtobacco cells transiently expressed stilbene synthase and VvROMT or HsCYP1B1.Grapevine cell cultures transformed with VvROMT produced pterostilbene, which wasdetected in both intra- and extracellular compartments, at a level of micrograms per litre. Grapevine cell cultures transformed with HsCYP1B1 produced about 20mg/L culture of piceatannol, displaying a sevenfold increase in relation towild-type cultures, and reaching an extracellular distribution of up to 45% oftotal production. The results obtained demonstrate the feasibility of this novel system for the bioproduction of natural and more bioactive resveratrolderivatives and suggest new ways for the improvement of production yields.\n",
      "CI  - (c) 2016 The Authors. Plant Biotechnology Journal published by Society forExperimental Biology and The Association of Applied Biologists and John Wiley &Sons Ltd.\n",
      "FAU - Martinez-Marquez, Ascension\n",
      "AU  - Martinez-Marquez A\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of Alicante, Alicante, Spain.\n",
      "FAU - Morante-Carriel, Jaime A\n",
      "AU  - Morante-Carriel JA\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of Alicante, Alicante, Spain.\n",
      "AD  - Biotechnology and Molecular Biology Group, Quevedo State Technical University,Quevedo, Ecuador.\n",
      "FAU - Ramirez-Estrada, Karla\n",
      "AU  - Ramirez-Estrada K\n",
      "AD  - Laboratory of Plant Physiology, Faculty of Pharmacy, University of Barcelona,Barcelona, Spain.\n",
      "FAU - Cusido, Rosa M\n",
      "AU  - Cusido RM\n",
      "AD  - Laboratory of Plant Physiology, Faculty of Pharmacy, University of Barcelona,Barcelona, Spain.\n",
      "FAU - Palazon, Javier\n",
      "AU  - Palazon J\n",
      "AD  - Laboratory of Plant Physiology, Faculty of Pharmacy, University of Barcelona,Barcelona, Spain.\n",
      "FAU - Bru-Martinez, Roque\n",
      "AU  - Bru-Martinez R\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of Alicante, Alicante, Spain.\n",
      "LA  - eng\n",
      "SI  - GENBANK/BT019979.1\n",
      "SI  - GENBANK/NM_001281115\n",
      "SI  - GENBANK/XM_002264953\n",
      "PT  - Journal Article\n",
      "DEP - 20160307\n",
      "PL  - England\n",
      "TA  - Plant Biotechnol J\n",
      "JT  - Plant biotechnology journal\n",
      "JID - 101201889\n",
      "RN  - 0 (Stilbenes)\n",
      "RN  - 26R60S6A5I (pterostilbene)\n",
      "RN  - 6KS3LS0D4F (3,3',4,5'-tetrahydroxystilbene)\n",
      "RN  - Q369O8926L (Resveratrol)\n",
      "SB  - IM\n",
      "MH  - Cell Culture Techniques\n",
      "MH  - Gene Expression Regulation, Plant\n",
      "MH  - *Metabolic Engineering\n",
      "MH  - *Plants, Genetically Modified/genetics/metabolism\n",
      "MH  - Resveratrol\n",
      "MH  - Stilbenes/chemistry/*metabolism\n",
      "MH  - Vitis/cytology/*genetics/*metabolism\n",
      "PMC - PMC5069453\n",
      "OTO - NOTNLM\n",
      "OT  - *Vitis vinifera\n",
      "OT  - *grapevine cell culture\n",
      "OT  - *metabolic engineering\n",
      "OT  - *piceatannol\n",
      "OT  - *pterostilbene\n",
      "OT  - *resveratrol\n",
      "EDAT- 2016/03/08 06:00\n",
      "MHDA- 2017/09/02 06:00\n",
      "CRDT- 2016/03/08 06:00\n",
      "PHST- 2015/11/24 00:00 [received]\n",
      "PHST- 2016/01/11 00:00 [revised]\n",
      "PHST- 2016/01/11 00:00 [accepted]\n",
      "PHST- 2016/03/08 06:00 [entrez]\n",
      "PHST- 2016/03/08 06:00 [pubmed]\n",
      "PHST- 2017/09/02 06:00 [medline]\n",
      "AID - 10.1111/pbi.12539 [doi]\n",
      "PST - ppublish\n",
      "SO  - Plant Biotechnol J. 2016 Sep;14(9):1813-25. doi: 10.1111/pbi.12539. Epub 2016 Mar7.\n",
      "\n",
      "PMID- 26921298\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20171116\n",
      "LR  - 20181113\n",
      "IS  - 2160-1836 (Electronic)\n",
      "IS  - 2160-1836 (Linking)\n",
      "VI  - 6\n",
      "IP  - 5\n",
      "DP  - 2016 May 3\n",
      "TI  - Genomic Bayesian Prediction Model for Count Data with Genotype x EnvironmentInteraction.\n",
      "PG  - 1165-77\n",
      "LID - 10.1534/g3.116.028118 [doi]\n",
      "AB  - Genomic tools allow the study of the whole genome, and facilitate the study ofgenotype-environment combinations and their relationship with phenotype. However,most genomic prediction models developed so far are appropriate for Gaussianphenotypes. For this reason, appropriate genomic prediction models are needed forcount data, since the conventional regression models used on count data with alarge sample size ([Formula: see text]) and a small number of parameters (p)cannot be used for genomic-enabled prediction where the number of parameters (p) is larger than the sample size ([Formula: see text]). Here, we propose a Bayesianmixed-negative binomial (BMNB) genomic regression model for counts that takesinto account genotype by environment [Formula: see text] interaction. We alsoprovide all the full conditional distributions to implement a Gibbs sampler. Weevaluated the proposed model using a simulated data set, and a real wheat dataset from the International Maize and Wheat Improvement Center (CIMMYT) andcollaborators. Results indicate that our BMNB model provides a viable option for analyzing count data.\n",
      "CI  - Copyright (c) 2016 Montesinos-Lopez et al.\n",
      "FAU - Montesinos-Lopez, Abelardo\n",
      "AU  - Montesinos-Lopez A\n",
      "AD  - Departamento de Estadistica, Centro de Investigacion en Matematicas (CIMAT),Guanajuato, 36240, Mexico.\n",
      "FAU - Montesinos-Lopez, Osval A\n",
      "AU  - Montesinos-Lopez OA\n",
      "AD  - International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641,06600, Mexico, D.F., Mexico.\n",
      "FAU - Crossa, Jose\n",
      "AU  - Crossa J\n",
      "AD  - International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641,06600, Mexico, D.F., Mexico j.crossa@cgiar.org.\n",
      "FAU - Burgueno, Juan\n",
      "AU  - Burgueno J\n",
      "AD  - International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641,06600, Mexico, D.F., Mexico.\n",
      "FAU - Eskridge, Kent M\n",
      "AU  - Eskridge KM\n",
      "AD  - Department of Statistics, University of Nebraska, Lincoln, Nebraska 68583-0963.\n",
      "FAU - Falconi-Castillo, Esteban\n",
      "AU  - Falconi-Castillo E\n",
      "AD  - Instituto Nacional Autonomo de Investigaciones Agropecuarias (INIAP),Panamericana Sur Km 1, Quito, Ecuador.\n",
      "FAU - He, Xinyao\n",
      "AU  - He X\n",
      "AD  - International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641,06600, Mexico, D.F., Mexico.\n",
      "FAU - Singh, Pawan\n",
      "AU  - Singh P\n",
      "AD  - International Maize and Wheat Improvement Center (CIMMYT), Apdo. Postal 6-641,06600, Mexico, D.F., Mexico.\n",
      "FAU - Cichy, Karen\n",
      "AU  - Cichy K\n",
      "AD  - Sugarbeet and Bean Research Unit, USDA-ARS, East Lansing, Michigan 48824.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160503\n",
      "PL  - England\n",
      "TA  - G3 (Bethesda)\n",
      "JT  - G3 (Bethesda, Md.)\n",
      "JID - 101566598\n",
      "SB  - IM\n",
      "MH  - Algorithms\n",
      "MH  - *Bayes Theorem\n",
      "MH  - *Environment\n",
      "MH  - *Gene-Environment Interaction\n",
      "MH  - Genetic Association Studies\n",
      "MH  - *Genomics/methods\n",
      "MH  - *Genotype\n",
      "MH  - *Models, Genetic\n",
      "MH  - Models, Statistical\n",
      "MH  - Phenotype\n",
      "MH  - Triticum/genetics\n",
      "PMC - PMC4856070\n",
      "OTO - NOTNLM\n",
      "OT  - *Bayesian model\n",
      "OT  - *GenPred\n",
      "OT  - *Gibbs sampler\n",
      "OT  - *count data\n",
      "OT  - *genome enabled prediction\n",
      "OT  - *genomic selection\n",
      "OT  - *shared data resource\n",
      "EDAT- 2016/02/28 06:00\n",
      "MHDA- 2017/11/29 06:00\n",
      "CRDT- 2016/02/28 06:00\n",
      "PHST- 2016/02/28 06:00 [entrez]\n",
      "PHST- 2016/02/28 06:00 [pubmed]\n",
      "PHST- 2017/11/29 06:00 [medline]\n",
      "AID - g3.116.028118 [pii]\n",
      "AID - 10.1534/g3.116.028118 [doi]\n",
      "PST - epublish\n",
      "SO  - G3 (Bethesda). 2016 May 3;6(5):1165-77. doi: 10.1534/g3.116.028118.\n",
      "\n",
      "PMID- 26903960\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "DCOM- 20160223\n",
      "LR  - 20200930\n",
      "IS  - 1664-302X (Print)\n",
      "IS  - 1664-302X (Linking)\n",
      "VI  - 7\n",
      "DP  - 2016\n",
      "TI  - Back to the Future of Soil Metagenomics.\n",
      "PG  - 73\n",
      "LID - 10.3389/fmicb.2016.00073 [doi]\n",
      "FAU - Nesme, Joseph\n",
      "AU  - Nesme J\n",
      "AD  - Environmental Microbial Genomics Group, Laboratoire Ampere, Centre National de laRecherche Scientifique, UMR5005, Institut National de la Recherche Agronomique,USC1407, Ecole Centrale de Lyon, Universite de LyonEcully, France; Research Unit for Environmental Genomics, Helmholtz Zentrum Munchen Deutsches Forschungszentrumfur Gesundheit und Umwelt (GmbH)Neuherberg, Germany.\n",
      "FAU - Achouak, Wafa\n",
      "AU  - Achouak W\n",
      "AD  - Aix-Marseille Universite, CEA, Centre National de la Recherche Scientifique,Laboratoire d'Ecologie Microbienne de la Rhizosphere et Environnements Extremes, UMR 7265, Biologie Vegetale et de Microbiologie EnvironnementalesSaint-Paul-lez-Durance, France.\n",
      "FAU - Agathos, Spiros N\n",
      "AU  - Agathos SN\n",
      "AD  - Earth and Life Institute, Catholic University of LouvainLouvain-la-Neuve,Belgium; School of Life Sciences and Biotechnology, Yachay TechUniversityUrcuqui, Ecuador.\n",
      "FAU - Bailey, Mark\n",
      "AU  - Bailey M\n",
      "AD  - Natural Environment Research Council, Centre for Ecology and Hydrology Oxford,UK.\n",
      "FAU - Baldrian, Petr\n",
      "AU  - Baldrian P\n",
      "AD  - Laboratory of Environmental Microbiology, Institute of Microbiology of the Czech Academy of Sciences Praha, Czech Republic.\n",
      "FAU - Brunel, Dominique\n",
      "AU  - Brunel D\n",
      "AD  - Institut National de la Recherche Agronomique, US1279, Etude du Polymorphisme desGenomes Vegetaux, CEA, Institut de Genomique, Centre National de Genotypage Evry,France.\n",
      "FAU - Frostegard, Asa\n",
      "AU  - Frostegard A\n",
      "AD  - NMBU Nitrogen Group, Department of Chemistry, Biotechnology and Food Science,Norwegian University of Life Sciences Aas, Norway.\n",
      "FAU - Heulin, Thierry\n",
      "AU  - Heulin T\n",
      "AD  - Aix-Marseille Universite, CEA, Centre National de la Recherche Scientifique,Laboratoire d'Ecologie Microbienne de la Rhizosphere et Environnements Extremes, UMR 7265, Biologie Vegetale et de Microbiologie EnvironnementalesSaint-Paul-lez-Durance, France.\n",
      "FAU - Jansson, Janet K\n",
      "AU  - Jansson JK\n",
      "AD  - Earth and Biological Sciences Directorate, Pacific Northwest National Laboratory Richland, WA, USA.\n",
      "FAU - Jurkevitch, Edouard\n",
      "AU  - Jurkevitch E\n",
      "AD  - Department of Plant Pathology and Microbiology, The Faculty of Agriculture, Food and Environment, The Otto Warburg-Minerva Center in Agricultural Biotechnology,The Hebrew University of Jerusalem Rehovot, Israel.\n",
      "FAU - Kruus, Kristiina L\n",
      "AU  - Kruus KL\n",
      "AD  - Enzymology of Renewable Biomass, VTT, Technical Research Centre of Finland Espoo,Finland.\n",
      "FAU - Kowalchuk, George A\n",
      "AU  - Kowalchuk GA\n",
      "AD  - Ecology and Biodiversity, Institute of Environmental Biology, Utrecht University Utrecht, Netherlands.\n",
      "FAU - Lagares, Antonio\n",
      "AU  - Lagares A\n",
      "AD  - Departamento de Ciencia Biologicas, Facultad de Ciencias Exactas, Instituto deBiotecnologia y Biologia Molecular, Centro Cientifico Tecnologico-ConsejoNacional de Investigaciones Cientificas y Tecnicas, Universidad Nacional de LaPlata La Plata, Argentina.\n",
      "FAU - Lappin-Scott, Hilary M\n",
      "AU  - Lappin-Scott HM\n",
      "AD  - Department of Biosciences, Swansea University Swansea, UK.\n",
      "FAU - Lemanceau, Philippe\n",
      "AU  - Lemanceau P\n",
      "AD  - Institut National de la Recherche Agronomique, UMR 1347, Agroecologie, Universitede Bourgogne Dijon, France.\n",
      "FAU - Le Paslier, Denis\n",
      "AU  - Le Paslier D\n",
      "AD  - CEA/Direction des sciences du vivant/Institut de Genomique. Genoscope, CentreNational de la Recherche Scientifiue UMR 8030, Universite d'Evry Val d'EssonneEvry, France.\n",
      "FAU - Mandic-Mulec, Ines\n",
      "AU  - Mandic-Mulec I\n",
      "AD  - Department of Food Science and Technology, Biotechnical Faculty- University ofLjubljana Ljubljana, Slovenia.\n",
      "FAU - Murrell, J Colin\n",
      "AU  - Murrell JC\n",
      "AD  - School of Environmental Sciences, University of East Anglia Norwich, UK.\n",
      "FAU - Myrold, David D\n",
      "AU  - Myrold DD\n",
      "AD  - Department of Crop and Soil Science, Oregon State University Corvallis, OR, USA.\n",
      "FAU - Nalin, Renaud\n",
      "AU  - Nalin R\n",
      "AD  - NALINOV Dremil Lafage, France.\n",
      "FAU - Nannipieri, Paolo\n",
      "AU  - Nannipieri P\n",
      "AD  - Department of Agrifood and Environmental Science, University of FlorenceFlorence, Italy.\n",
      "FAU - Neufeld, Josh D\n",
      "AU  - Neufeld JD\n",
      "AD  - Department of Biology, University of Waterloo Waterloo, ON, Canada.\n",
      "FAU - O'Gara, Fergal\n",
      "AU  - O'Gara F\n",
      "AD  - BIOMERIT Research Centre, School of Microbiology, National University ofIrelandCork, Ireland; School of Biomedical Science, Curtin UniversityPerth, WA,Australia.\n",
      "FAU - Parnell, John J\n",
      "AU  - Parnell JJ\n",
      "AD  - National Ecological Observatory Network Boulder, CO, USA.\n",
      "FAU - Puhler, Alfred\n",
      "AU  - Puhler A\n",
      "AD  - Center for Biotechnology, Institute for Genome Research and Systems Biology,Genome Research of Industrial Microorganisms, Bielefeld University Bielefeld,Germany.\n",
      "FAU - Pylro, Victor\n",
      "AU  - Pylro V\n",
      "AD  - Genomics and Computational Biology Group, Rene Rachou Research Centre -CPqRR/FIOCRUZ Belo Horizonte, Brazil.\n",
      "FAU - Ramos, Juan L\n",
      "AU  - Ramos JL\n",
      "AD  - Department of Environmental Protection, Estacion Experimental del Zaidin, ConsejoSuperior de Investigaciones Cientificas Granada, Spain.\n",
      "FAU - Roesch, Luiz F W\n",
      "AU  - Roesch LF\n",
      "AD  - Federal University of Pampa Sao Gabriel, Brazil.\n",
      "FAU - Schloter, Michael\n",
      "AU  - Schloter M\n",
      "AD  - Research Unit for Environmental Genomics, Helmholtz Zentrum Munchen DeutschesForschungszentrum fur Gesundheit und Umwelt (GmbH) Neuherberg, Germany.\n",
      "FAU - Schleper, Christa\n",
      "AU  - Schleper C\n",
      "AD  - Archaea Biology and Ecogenomics Division, Department of Ecogenomics and SystemsBiology, University of Vienna Vienna, Austria.\n",
      "FAU - Sczyrba, Alexander\n",
      "AU  - Sczyrba A\n",
      "AD  - Center for Biotechnology and Faculty of Technology, Computational Metagenomics,Bielefeld University Bielefeld, Germany.\n",
      "FAU - Sessitsch, Angela\n",
      "AU  - Sessitsch A\n",
      "AD  - Health and Environment Department, Bioresources, AIT Austrian Institute ofTechnology GmbH Tulln, Austria.\n",
      "FAU - Sjoling, Sara\n",
      "AU  - Sjoling S\n",
      "AD  - School of Natural Sciences and Environmental Studies, Sodertorn UniversityHuddinge, Sweden.\n",
      "FAU - Sorensen, Jan\n",
      "AU  - Sorensen J\n",
      "AD  - Section of Genetics and Microbiology, Department of Plant and EnvironmentalMicrobiology, University of Copenhagen Frederiksberg, Denmark.\n",
      "FAU - Sorensen, Soren J\n",
      "AU  - Sorensen SJ\n",
      "AD  - Section of Microbiology, Department of Biology, University of CopenhagenCopenhagen, Denmark.\n",
      "FAU - Tebbe, Christoph C\n",
      "AU  - Tebbe CC\n",
      "AD  - Thunen-Institute of Biodiversity Braunschweig, Germany.\n",
      "FAU - Topp, Edward\n",
      "AU  - Topp E\n",
      "AD  - Agriculture and Agri-Food Canada, Department of Biology, University of WesternOntario London, ON, Canada.\n",
      "FAU - Tsiamis, George\n",
      "AU  - Tsiamis G\n",
      "AD  - Department of Environmental and Natural Resources Management, University ofPatras Agrinio, Greece.\n",
      "FAU - van Elsas, Jan Dirk\n",
      "AU  - van Elsas JD\n",
      "AD  - Department of Microbial Ecology, Groningen Institute for Evolutionary LifeSciences, University of Groningen Groningen, Netherlands.\n",
      "FAU - van Keulen, Geertje\n",
      "AU  - van Keulen G\n",
      "AD  - Institute of Life Science, Medical School, Swansea University Swansea, UK.\n",
      "FAU - Widmer, Franco\n",
      "AU  - Widmer F\n",
      "AD  - Molecular Ecology, Institute for Sustainability Sciences, Agroscope Zurich,Switzerland.\n",
      "FAU - Wagner, Michael\n",
      "AU  - Wagner M\n",
      "AD  - Department of Microbiology and Ecosystem Science, Division of Microbial Ecology, University of Vienna Vienna, Austria.\n",
      "FAU - Zhang, Tong\n",
      "AU  - Zhang T\n",
      "AD  - Environmental Biotechnology Laboratory, Department of Civil Engineering, TheUniversity of Hong Kong Hong Kong, China.\n",
      "FAU - Zhang, Xiaojun\n",
      "AU  - Zhang X\n",
      "AD  - Group of Microbial Ecology and Ecogenomics, State Key Laboratory of MicrobialMetabolism, School of Life Sciences and Biotechnology, Shanghai Jiao TongUniversity Shanghai, China.\n",
      "FAU - Zhao, Liping\n",
      "AU  - Zhao L\n",
      "AD  - Group of Microbial Ecology and Ecogenomics, State Key Laboratory of MicrobialMetabolism, School of Life Sciences and Biotechnology, Shanghai Jiao TongUniversity Shanghai, China.\n",
      "FAU - Zhu, Yong-Guan\n",
      "AU  - Zhu YG\n",
      "AD  - Institute of Urban Environment, Chinese Academy of Sciences Xiamen, China.\n",
      "FAU - Vogel, Timothy M\n",
      "AU  - Vogel TM\n",
      "AD  - Environmental Microbial Genomics Group, Laboratoire Ampere, Centre National de laRecherche Scientifique, UMR5005, Institut National de la Recherche Agronomique,USC1407, Ecole Centrale de Lyon, Universite de Lyon Ecully, France.\n",
      "FAU - Simonet, Pascal\n",
      "AU  - Simonet P\n",
      "AD  - Institute of Life Science, Medical School, Swansea University Swansea, UK.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20160210\n",
      "PL  - Switzerland\n",
      "TA  - Front Microbiol\n",
      "JT  - Frontiers in microbiology\n",
      "JID - 101548977\n",
      "PMC - PMC4748112\n",
      "OTO - NOTNLM\n",
      "OT  - metagenomic\n",
      "OT  - microbial ecology\n",
      "OT  - soil ecology\n",
      "OT  - soil microbiology\n",
      "OT  - terrestrial microbiology\n",
      "EDAT- 2016/02/24 06:00\n",
      "MHDA- 2016/02/24 06:01\n",
      "CRDT- 2016/02/24 06:00\n",
      "PHST- 2015/12/01 00:00 [received]\n",
      "PHST- 2016/01/15 00:00 [accepted]\n",
      "PHST- 2016/02/24 06:00 [entrez]\n",
      "PHST- 2016/02/24 06:00 [pubmed]\n",
      "PHST- 2016/02/24 06:01 [medline]\n",
      "AID - 10.3389/fmicb.2016.00073 [doi]\n",
      "PST - epublish\n",
      "SO  - Front Microbiol. 2016 Feb 10;7:73. doi: 10.3389/fmicb.2016.00073. eCollection2016.\n",
      "\n",
      "PMID- 26831713\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20161024\n",
      "LR  - 20161230\n",
      "IS  - 1940-6029 (Electronic)\n",
      "IS  - 1064-3745 (Linking)\n",
      "VI  - 1401\n",
      "DP  - 2016\n",
      "TI  - Alignment-Free Methods for the Detection and Specificity Prediction ofAdenylation Domains.\n",
      "PG  - 253-72\n",
      "LID - 10.1007/978-1-4939-3375-4_16 [doi]\n",
      "AB  - Identifying adenylation domains (A-domains) and their substrate specificity canaid the detection of nonribosomal peptide synthetases (NRPS) at genome/proteomelevel and allow inferring the structure of oligopeptides with relevant biologicalactivities. However, that is challenging task due to the high sequence diversity of A-domains (~10-40 % of amino acid identity) and their selectivity for 50different natural/unnatural amino acids. Altogether these characteristics maketheir detection and the prediction of their substrate specificity a realchallenge when using traditional sequence alignment methods, e.g., BLASTsearches. In this chapter we describe two workflows based on alignment-freemethods intended for the identification and substrate specificity prediction ofA-domains. To identify A-domains we introduce a graphical-numerical method,implemented in TI2BioP version 2.0 (topological indices to biopolymers), which ina first step uses protein four-color maps to represent A-domains. In a secondstep, simple topological indices (TIs), called spectral moments, are derived fromthe graphical representations of known A-domains (positive dataset) and ofunrelated but well-characterized sequences (negative set). Spectral moments arethen used as input predictors for statistical classification techniques to build alignment-free models. Finally, the resulting alignment-free models can be usedto explore entire proteomes for unannotated A-domains. In addition, thisgraphical-numerical methodology works as a sequence-search method that can beensemble with homology-based tools to deeply explore the A-domain signature andcope with the diversity of this class (Aguero-Chapin et al., PLoS One8(7):e65926, 2013). The second workflow for the prediction of A-domain'ssubstrate specificity is based on alignment-free models constructed bytransductive support vector machines (TSVMs) that incorporate information ofuncharacterized A-domains. The construction of the models was implemented in the NRPSpredictor and in a first step uses the physicochemical fingerprint of the 34 residues lining the active site of the phenylalanine-adenylation domain ofgramicidin synthetase A [PDB ID 1 amu] to derive a feature vector. Homologouspositions were extracted for A-domains with known and unknown substratespecificities and turned into feature vectors. At the same time, A-domains withknown specificities towards similar substrates were clustered by physicochemical properties of amino acids (AA). In a second step, support vector machines (SVMs) were optimized from feature vectors of characterized A-domains in each of theresulting clusters. Later, SVMs were used in the variant of TSVMs that integrate a fraction of uncharacterized A-domains during training to predict unknownspecificities. Finally, uncharacterized A-domains were scored by each of theconstructed alignment-free models (TSVM) representing each substrate specificity resulting from the clustering. The model producing the largest score for theuncharacterized A-domain assigns the substrate specificity to it (Rausch et al., Nucleic Acids Res 33:5799-5808, 2005).\n",
      "FAU - Aguero-Chapin, Guillermin\n",
      "AU  - Aguero-Chapin G\n",
      "AD  - CIMAR/CIIMAR, Centro Interdisciplinar de Investigacao Marinha e Ambiental,Universidade do Porto, Rua dos Bragas, 177, Porto, 4050-123, Portugal.\n",
      "AD  - Centro de Bioactivos Quimicos, Universidad Central \"Marta Abreu\" de Las Villas(UCLV), Santa Clara, 54830, Cuba.\n",
      "FAU - Perez-Machado, Gisselle\n",
      "AU  - Perez-Machado G\n",
      "AD  - Centro de Bioactivos Quimicos, Universidad Central \"Marta Abreu\" de Las Villas(UCLV), Santa Clara, 54830, Cuba.\n",
      "FAU - Sanchez-Rodriguez, Aminael\n",
      "AU  - Sanchez-Rodriguez A\n",
      "AD  - Departamento de Ciencias Naturales, Universidad Tecnica Particular de Loja, SanCayetano Alto, S/N, Loja, Ecuador.\n",
      "FAU - Santos, Miguel Machado\n",
      "AU  - Santos MM\n",
      "AD  - CIMAR/CIIMAR, Centro Interdisciplinar de Investigacao Marinha e Ambiental,Universidade do Porto, Rua dos Bragas, 177, Porto, 4050-123, Portugal.\n",
      "AD  - Departamento de Biologia, Faculdade de Ciencias, Universidade do Porto, Rua doCampo Alegre, Porto, 4169-007, Portugal.\n",
      "FAU - Antunes, Agostinho\n",
      "AU  - Antunes A\n",
      "AD  - CIMAR/CIIMAR, Centro Interdisciplinar de Investigacao Marinha e Ambiental,Universidade do Porto, Rua dos Bragas, 177, Porto, 4050-123, Portugal.aantunes@ciimar.up.pt.\n",
      "AD  - Departamento de Biologia, Faculdade de Ciencias, Universidade do Porto, Rua doCampo Alegre, Porto, 4169-007, Portugal. aantunes@ciimar.up.pt.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PL  - United States\n",
      "TA  - Methods Mol Biol\n",
      "JT  - Methods in molecular biology (Clifton, N.J.)\n",
      "JID - 9214969\n",
      "RN  - EC 6.3.2.- (Peptide Synthases)\n",
      "RN  - EC 6.3.2.- (non-ribosomal peptide synthase)\n",
      "SB  - IM\n",
      "MH  - Bacteria/chemistry/*enzymology/metabolism\n",
      "MH  - Catalytic Domain\n",
      "MH  - Computer Graphics\n",
      "MH  - Models, Biological\n",
      "MH  - Peptide Synthases/chemistry/*metabolism\n",
      "MH  - Protein Structure, Tertiary\n",
      "MH  - Proteomics/*methods\n",
      "MH  - Software\n",
      "MH  - Substrate Specificity\n",
      "MH  - *Support Vector Machine\n",
      "MH  - Workflow\n",
      "OTO - NOTNLM\n",
      "OT  - Adenylation domains\n",
      "OT  - Alignment-free models\n",
      "OT  - NRPS\n",
      "OT  - Topological Indices\n",
      "OT  - Transductive support vector machines\n",
      "EDAT- 2016/02/03 06:00\n",
      "MHDA- 2016/10/25 06:00\n",
      "CRDT- 2016/02/03 06:00\n",
      "PHST- 2016/02/03 06:00 [entrez]\n",
      "PHST- 2016/02/03 06:00 [pubmed]\n",
      "PHST- 2016/10/25 06:00 [medline]\n",
      "AID - 10.1007/978-1-4939-3375-4_16 [doi]\n",
      "PST - ppublish\n",
      "SO  - Methods Mol Biol. 2016;1401:253-72. doi: 10.1007/978-1-4939-3375-4_16.\n",
      "\n",
      "PMID- 26645579\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20161213\n",
      "LR  - 20181113\n",
      "IS  - 1557-7759 (Electronic)\n",
      "IS  - 1530-3667 (Linking)\n",
      "VI  - 15\n",
      "IP  - 12\n",
      "DP  - 2015 Dec\n",
      "TI  - Prevalence, Genetic Characterization, and 18S Small Subunit Ribosomal RNADiversity of Trypanosoma rangeli in Triatomine and Mammal Hosts in Endemic Areas for Chagas Disease in Ecuador.\n",
      "PG  - 732-42\n",
      "LID - 10.1089/vbz.2015.1794 [doi]\n",
      "AB  - Trypanosoma rangeli is a nonpathogenic parasite for humans; however, its medical importance relies in its similarity and overlapping distribution with Trypanosomacruzi, causal agent of Chagas disease in the Americas. The genetic diversity ofT. rangeli and its association with host species (triatomines and mammals) hasbeen identified along Central and the South America; however, it has not includeddata of isolates from Ecuador. This study reports infection with T. rangeli in 18genera of mammal hosts and five species of triatomines in three environments(domestic, peridomestic, and sylvatic). Higher infection rates were found in the sylvatic environment, in close association with Rhodnius ecuadoriensis. Theresults of this study extend the range of hosts infected with this parasite andthe geographic range of the T. rangeli genotype KP1(-)/lineage C in SouthAmerica. It was not possible to detect variation on T. rangeli from the centralcoastal region and southern Ecuador with the analysis of the small subunitribosomal RNA (SSU-rRNA) gene, even though these areas are ecologically differentand a phenotypic subdivision of R. ecuadoriensis has been found. R. ecuadoriensisis considered one of the most important vectors for Chagas disease transmissionin Ecuador due to its wide distribution and adaptability to diverse environments.An extensive knowledge of the trypanosomes circulating in this species oftriatomine, and associated mammal hosts, is important for delineatingtransmission dynamics and preventive measures in the endemic areas of Ecuador andNorthern Peru.\n",
      "FAU - Ocana-Mayorga, Sofia\n",
      "AU  - Ocana-Mayorga S\n",
      "AD  - 1 Center for Infectious and Chronic Disease Research, School of BiologicalSciences, Pontifical Catholic University of Ecuador, Quito, Ecuador , andTropical Disease Institute, Department of Biomedical Sciences, Heritage Collegeof Osteopathic Medicine, Ohio University , Athens, Ohio.\n",
      "FAU - Aguirre-Villacis, Fernanda\n",
      "AU  - Aguirre-Villacis F\n",
      "AD  - 2 Life Sciences Department, University of the Army Forces-ESPE, Sangolqui,Ecuador, and Center for Infectious and Chronic Disease Research, School ofBiological Sciences, Pontifical Catholic University of Ecuador , Quito, Ecuador .\n",
      "FAU - Pinto, C Miguel\n",
      "AU  - Pinto CM\n",
      "AD  - 3 Division of Mammals, National Museum of Natural History, SmithsonianInstitution, Washington, DC; Sackler Institute for Comparative Genomics, AmericanMuseum of Natural History, New York, New York; and Center for Infectious andChronic Disease Research, School of Biological Sciences, Pontifical CatholicUniversity of Ecuador , Quito, Ecuador .\n",
      "FAU - Vallejo, Gustavo A\n",
      "AU  - Vallejo GA\n",
      "AD  - 4 Laboratorio de Investigaciones en Parasitologia Tropical, Facultad de Ciencias,Departamento de Biologia, Universidad de Tolima , Ibague, Colombia .\n",
      "FAU - Grijalva, Mario J\n",
      "AU  - Grijalva MJ\n",
      "AD  - 5 Tropical Disease Institute, Department of Biomedical Sciences, Heritage Collegeof Osteopathic Medicine, Ohio University, Athens, Ohio, and Center for Infectiousand Chronic Disease Research, School of Biological Sciences, Pontifical Catholic University of Ecuador , Quito, Ecuador .\n",
      "LA  - eng\n",
      "GR  - 1 R15 AI077896-01/AI/NIAID NIH HHS/United States\n",
      "GR  - D43TW008261/TW/FIC NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20151208\n",
      "PL  - United States\n",
      "TA  - Vector Borne Zoonotic Dis\n",
      "JT  - Vector borne and zoonotic diseases (Larchmont, N.Y.)\n",
      "JID - 100965525\n",
      "RN  - 0 (DNA, Ribosomal)\n",
      "RN  - 0 (RNA, Ribosomal, 18S)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Base Sequence\n",
      "MH  - Chagas Disease/*epidemiology/parasitology/transmission\n",
      "MH  - DNA, Ribosomal/chemistry/genetics\n",
      "MH  - Ecuador/epidemiology\n",
      "MH  - Environment\n",
      "MH  - *Genetic Variation\n",
      "MH  - Geography\n",
      "MH  - Humans\n",
      "MH  - Insect Vectors/*parasitology\n",
      "MH  - Mammals\n",
      "MH  - Molecular Sequence Data\n",
      "MH  - Phenotype\n",
      "MH  - Prevalence\n",
      "MH  - RNA, Ribosomal, 18S/genetics\n",
      "MH  - Rhodnius/*parasitology\n",
      "MH  - Sequence Analysis, DNA\n",
      "MH  - Trypanosoma cruzi/*genetics/isolation & purification\n",
      "MH  - Trypanosoma rangeli/*genetics/isolation & purification\n",
      "MH  - Zoonoses\n",
      "PMC - PMC4700398\n",
      "OTO - NOTNLM\n",
      "OT  - Ecuador.\n",
      "OT  - KP1(-)\n",
      "OT  - Lineage C\n",
      "OT  - Loja Province\n",
      "OT  - Manabi Province\n",
      "OT  - Rhodnius ecuadoriensis\n",
      "OT  - SSU-rRNA\n",
      "OT  - Trypanosoma rangeli characterization\n",
      "EDAT- 2015/12/10 06:00\n",
      "MHDA- 2016/12/15 06:00\n",
      "CRDT- 2015/12/10 06:00\n",
      "PHST- 2015/12/10 06:00 [entrez]\n",
      "PHST- 2015/12/10 06:00 [pubmed]\n",
      "PHST- 2016/12/15 06:00 [medline]\n",
      "AID - 10.1089/vbz.2015.1794 [doi]\n",
      "PST - ppublish\n",
      "SO  - Vector Borne Zoonotic Dis. 2015 Dec;15(12):732-42. doi: 10.1089/vbz.2015.1794.Epub 2015 Dec 8.\n",
      "\n",
      "PMID- 26636962\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20160607\n",
      "LR  - 20211203\n",
      "IS  - 1553-7404 (Electronic)\n",
      "IS  - 1553-7390 (Linking)\n",
      "VI  - 11\n",
      "IP  - 12\n",
      "DP  - 2015 Dec\n",
      "TI  - Genomic Insights into the Ancestry and Demographic History of South America.\n",
      "PG  - e1005602\n",
      "LID - 10.1371/journal.pgen.1005602 [doi]\n",
      "AB  - South America has a complex demographic history shaped by multiple migration and admixture events in pre- and post-colonial times. Settled over 14,000 years agoby Native Americans, South America has experienced migrations of European andAfrican individuals, similar to other regions in the Americas. However, thetiming and magnitude of these events resulted in markedly different patterns ofadmixture throughout Latin America. We use genome-wide SNP data for 437 admixedindividuals from 5 countries (Colombia, Ecuador, Peru, Chile, and Argentina) toexplore the population structure and demographic history of South AmericanLatinos. We combined these data with population reference panels from Africa,Asia, Europe and the Americas to perform global ancestry analysis and infer thesubcontinental origin of the European and Native American ancestry components of the admixed individuals. By applying ancestry-specific PCA analyses we find that most of the European ancestry in South American Latinos is from the IberianPeninsula; however, many individuals trace their ancestry back to Italy,especially within Argentina. We find a strong gradient in the Native Americanancestry component of South American Latinos associated with country of originand the geography of local indigenous populations. For example, Native Americangenomic segments in Peruvians show greater affinities with Andean indigenouspeoples like Quechua and Aymara, whereas Native American haplotypes fromColombians tend to cluster with Amazonian and coastal tribes from northern South America. Using ancestry tract length analysis we modeled post-colonial SouthAmerican migration history as the youngest in Latin America during Europeancolonization (9-14 generations ago), with an additional strong pulse of European migration occurring between 3 and 9 generations ago. These genetic footprints canimpact our understanding of population-level differences in biomedical traitsand, thus, inform future medical genetic studies in the region.\n",
      "FAU - Homburger, Julian R\n",
      "AU  - Homburger JR\n",
      "AD  - Department of Genetics, Stanford University, Stanford, California, United States of America.\n",
      "FAU - Moreno-Estrada, Andres\n",
      "AU  - Moreno-Estrada A\n",
      "AD  - Department of Genetics, Stanford University, Stanford, California, United States of America.\n",
      "AD  - Laboratorio Nacional de Genomica para la Biodiversidad (LANGEBIO), CINVESTAV,Irapuato, Guanajuato, Mexico.\n",
      "FAU - Gignoux, Christopher R\n",
      "AU  - Gignoux CR\n",
      "AD  - Department of Genetics, Stanford University, Stanford, California, United States of America.\n",
      "FAU - Nelson, Dominic\n",
      "AU  - Nelson D\n",
      "AD  - McGill University and Genome Quebec Innovation Centre, Montreal, Quebec, Canada.\n",
      "AD  - Department of Human Genetics, McGill University, Montreal, Quebec, Canada.\n",
      "FAU - Sanchez, Elena\n",
      "AU  - Sanchez E\n",
      "AD  - Arthritis and Clinical Immunology, Oklahoma Medical Research Foundation, OklahomaCity, Oklahoma, United States of America.\n",
      "FAU - Ortiz-Tello, Patricia\n",
      "AU  - Ortiz-Tello P\n",
      "AD  - Department of Genetics, Stanford University, Stanford, California, United States of America.\n",
      "FAU - Pons-Estel, Bernardo A\n",
      "AU  - Pons-Estel BA\n",
      "AD  - Sanatorio Parque, Rosario, Argentina.\n",
      "FAU - Acevedo-Vasquez, Eduardo\n",
      "AU  - Acevedo-Vasquez E\n",
      "AD  - Facultad de Medicina, Universidad Nacional Mayor de San Marcos, Hospital NacionalGuillermo Almenara Irigoyen, Lima, Peru.\n",
      "FAU - Miranda, Pedro\n",
      "AU  - Miranda P\n",
      "AD  - Centro de Estudios Reumatologicos, Santiago, Chile.\n",
      "FAU - Langefeld, Carl D\n",
      "AU  - Langefeld CD\n",
      "AD  - Center for Public Health Genomics, Wake Forest School of Medicine, Winston-Salem,North Carolina, United States of America.\n",
      "FAU - Gravel, Simon\n",
      "AU  - Gravel S\n",
      "AD  - McGill University and Genome Quebec Innovation Centre, Montreal, Quebec, Canada.\n",
      "AD  - Department of Human Genetics, McGill University, Montreal, Quebec, Canada.\n",
      "FAU - Alarcon-Riquelme, Marta E\n",
      "AU  - Alarcon-Riquelme ME\n",
      "AD  - Arthritis and Clinical Immunology, Oklahoma Medical Research Foundation, OklahomaCity, Oklahoma, United States of America.\n",
      "AD  - GENYO, Centre for Genomics and Oncological Research: Pfizer/ University ofGranada/ Andalusian Regional Government, Granada, Spain.\n",
      "FAU - Bustamante, Carlos D\n",
      "AU  - Bustamante CD\n",
      "AD  - Department of Genetics, Stanford University, Stanford, California, United States of America.\n",
      "LA  - eng\n",
      "GR  - MOP-134855/CAPMC/ CIHR/Canada\n",
      "GR  - RC1AR058621/AR/NIAMS NIH HHS/United States\n",
      "GR  - T32HG000044/HG/NHGRI NIH HHS/United States\n",
      "GR  - R01CA141700/CA/NCI NIH HHS/United States\n",
      "GR  - R01 CA141700/CA/NCI NIH HHS/United States\n",
      "GR  - T32 HG000044/HG/NHGRI NIH HHS/United States\n",
      "GR  - 1R01GM090087/GM/NIGMS NIH HHS/United States\n",
      "GR  - U01 HG007419/HG/NHGRI NIH HHS/United States\n",
      "GR  - 5U01HG007419-02/HG/NHGRI NIH HHS/United States\n",
      "GR  - R01 GM090087/GM/NIGMS NIH HHS/United States\n",
      "GR  - RC1 AR058621/AR/NIAMS NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20151204\n",
      "PL  - United States\n",
      "TA  - PLoS Genet\n",
      "JT  - PLoS genetics\n",
      "JID - 101239074\n",
      "RN  - 0 (DNA, Mitochondrial)\n",
      "SB  - IM\n",
      "MH  - Argentina\n",
      "MH  - Blacks/genetics\n",
      "MH  - Colombia\n",
      "MH  - DNA, Mitochondrial/*genetics\n",
      "MH  - Ethnicity/*genetics\n",
      "MH  - *Genetics, Population\n",
      "MH  - *Genome, Human\n",
      "MH  - Genomics\n",
      "MH  - Haplotypes\n",
      "MH  - Humans\n",
      "MH  - Indians, North American/genetics\n",
      "MH  - Peru\n",
      "MH  - Racial Groups\n",
      "MH  - South America\n",
      "MH  - Whites/genetics\n",
      "PMC - PMC4670080\n",
      "EDAT- 2015/12/05 06:00\n",
      "MHDA- 2016/06/09 06:00\n",
      "CRDT- 2015/12/05 06:00\n",
      "PHST- 2015/03/31 00:00 [received]\n",
      "PHST- 2015/09/22 00:00 [accepted]\n",
      "PHST- 2015/12/05 06:00 [entrez]\n",
      "PHST- 2015/12/05 06:00 [pubmed]\n",
      "PHST- 2016/06/09 06:00 [medline]\n",
      "AID - 10.1371/journal.pgen.1005602 [doi]\n",
      "AID - PGENETICS-D-15-00796 [pii]\n",
      "PST - epublish\n",
      "SO  - PLoS Genet. 2015 Dec 4;11(12):e1005602. doi: 10.1371/journal.pgen.1005602.eCollection 2015 Dec.\n",
      "\n",
      "PMID- 26537019\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20160108\n",
      "LR  - 20181113\n",
      "IS  - 1479-7364 (Electronic)\n",
      "IS  - 1473-9542 (Linking)\n",
      "VI  - 9\n",
      "DP  - 2015 Nov 4\n",
      "TI  - Genomics is changing personal healthcare and medicine: the dawn of iPH(individualized preventive healthcare).\n",
      "PG  - 29\n",
      "LID - 10.1186/s40246-015-0052-0 [doi]\n",
      "AB  - This opinion piece focuses on the convergence of information technology (IT) inthe form of personal monitors, especially smart phones and possibly also smartwatches, individual genomic information and preventive healthcare and medicine.This may benefit each one of us not only individually but also society as a wholethrough iPH (individualized preventive healthcare). This shift driven by genomic and other technologies may well also change the relationship between patient and physician by empowering the former but giving him/her also much more individualresponsibility.\n",
      "FAU - Mehrian-Shai, Ruty\n",
      "AU  - Mehrian-Shai R\n",
      "AD  - Pediatric Hemato-Oncology, Chaim Sheba Medical Center, Ramat Gan, Israel.\n",
      "FAU - Reichardt, Juergen K V\n",
      "AU  - Reichardt JK\n",
      "AD  - Division of Tropical Health and Medicine, James Cook University, Townsville, QLD,Australia. jreichardt@yachaytech.edu.au.\n",
      "AD  - Present Address: Yachay Tech University, San Miguel de Urcuqui, Ecuador.jreichardt@yachaytech.edu.au.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20151104\n",
      "PL  - England\n",
      "TA  - Hum Genomics\n",
      "JT  - Human genomics\n",
      "JID - 101202210\n",
      "SB  - IM\n",
      "MH  - Education, Medical\n",
      "MH  - *Genome, Human\n",
      "MH  - *Genomics\n",
      "MH  - Humans\n",
      "MH  - Precision Medicine/economics/*methods\n",
      "MH  - Preventive Health Services/economics/methods\n",
      "PMC - PMC4634796\n",
      "EDAT- 2015/11/06 06:00\n",
      "MHDA- 2016/01/09 06:00\n",
      "CRDT- 2015/11/06 06:00\n",
      "PHST- 2015/09/29 00:00 [received]\n",
      "PHST- 2015/10/31 00:00 [accepted]\n",
      "PHST- 2015/11/06 06:00 [entrez]\n",
      "PHST- 2015/11/06 06:00 [pubmed]\n",
      "PHST- 2016/01/09 06:00 [medline]\n",
      "AID - 10.1186/s40246-015-0052-0 [doi]\n",
      "AID - 10.1186/s40246-015-0052-0 [pii]\n",
      "PST - epublish\n",
      "SO  - Hum Genomics. 2015 Nov 4;9:29. doi: 10.1186/s40246-015-0052-0.\n",
      "\n",
      "PMID- 26465748\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20160531\n",
      "LR  - 20201215\n",
      "IS  - 1932-6203 (Electronic)\n",
      "IS  - 1932-6203 (Linking)\n",
      "VI  - 10\n",
      "IP  - 10\n",
      "DP  - 2015\n",
      "TI  - Bats, Trypanosomes, and Triatomines in Ecuador: New Insights into the Diversity, Transmission, and Origins of Trypanosoma cruzi and Chagas Disease.\n",
      "PG  - e0139999\n",
      "LID - 10.1371/journal.pone.0139999 [doi]\n",
      "AB  - The generalist parasite Trypanosoma cruzi has two phylogenetic lineagesassociated almost exclusively with bats-Trypanosoma cruzi Tcbat and thesubspecies T. c. marinkellei. We present new information on the geneticvariation, geographic distribution, host associations, and potential vectors ofthese lineages. We conducted field surveys of bats and triatomines in southernEcuador, a country endemic for Chagas disease, and screened for trypanosomes bymicroscopy and PCR. We identified parasites at species and genotype levelsthrough phylogenetic approaches based on 18S ribosomal RNA (18S rRNA) andcytochrome b (cytb) genes and conducted a comparison of nucleotide diversity ofthe cytb gene. We document for the first time T. cruzi Tcbat and T. c.marinkellei in Ecuador, expanding their distribution in South America to thewestern side of the Andes. In addition, we found the triatomines Cavernicolapilosa and Triatoma dispar sharing shelters with bats. The comparisons ofnucleotide diversity revealed a higher diversity for T. c. marinkellei than anyof the T. c. cruzi genotypes associated with Chagas disease. Findings from thisstudy increased both the number of host species and known geographical ranges of both parasites and suggest potential vectors for these two trypanosomesassociated with bats in rural areas of southern Ecuador. The higher nucleotidediversity of T. c. marinkellei supports a long evolutionary relationship between T. cruzi and bats, implying that bats are the original hosts of this importantparasite.\n",
      "FAU - Pinto, C Miguel\n",
      "AU  - Pinto CM\n",
      "AD  - Centro de Investigacion en Enfermedades Infecciosas, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador; Sackler Institute for Comparative Genomics,American Museum of Natural History, New York, New York, United States of America;Department of Mammalogy, American Museum of Natural History, New York, New York, United States of America; The Graduate Center, The City University of New York,New York, New York, United States of America; Division of Mammals, NationalMuseum of Natural History, Smithsonian Institution, Washington, District ofColumbia, United States of America.\n",
      "FAU - Ocana-Mayorga, Sofia\n",
      "AU  - Ocana-Mayorga S\n",
      "AD  - Centro de Investigacion en Enfermedades Infecciosas, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador; Tropical Disease Institute, Department ofBiomedical Sciences, Heritage College of Osteopathic Medicine, Ohio University,Athens, Ohio, United States of America.\n",
      "FAU - Tapia, Elicio E\n",
      "AU  - Tapia EE\n",
      "AD  - Fundacion Otonga, Quito, Ecuador.\n",
      "FAU - Lobos, Simon E\n",
      "AU  - Lobos SE\n",
      "AD  - Centro de Investigacion en Enfermedades Infecciosas, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Zurita, Alejandra P\n",
      "AU  - Zurita AP\n",
      "AD  - Centro de Investigacion en Enfermedades Infecciosas, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Aguirre-Villacis, Fernanda\n",
      "AU  - Aguirre-Villacis F\n",
      "AD  - Centro de Investigacion en Enfermedades Infecciosas, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador.\n",
      "FAU - MacDonald, Amber\n",
      "AU  - MacDonald A\n",
      "AD  - Centro de Investigacion en Enfermedades Infecciosas, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Villacis, Anita G\n",
      "AU  - Villacis AG\n",
      "AD  - Centro de Investigacion en Enfermedades Infecciosas, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Lima, Luciana\n",
      "AU  - Lima L\n",
      "AD  - Departamento de Parasitologia, Instituto de Ciencias Biomedicas, Universidade de Sao Paulo, Sao Paulo, Sao Paulo, Brazil.\n",
      "FAU - Teixeira, Marta M G\n",
      "AU  - Teixeira MM\n",
      "AD  - Departamento de Parasitologia, Instituto de Ciencias Biomedicas, Universidade de Sao Paulo, Sao Paulo, Sao Paulo, Brazil.\n",
      "FAU - Grijalva, Mario J\n",
      "AU  - Grijalva MJ\n",
      "AD  - Centro de Investigacion en Enfermedades Infecciosas, Pontificia UniversidadCatolica del Ecuador, Quito, Ecuador; Tropical Disease Institute, Department ofBiomedical Sciences, Heritage College of Osteopathic Medicine, Ohio University,Athens, Ohio, United States of America.\n",
      "FAU - Perkins, Susan L\n",
      "AU  - Perkins SL\n",
      "AD  - Sackler Institute for Comparative Genomics, American Museum of Natural History,New York, New York, United States of America.\n",
      "LA  - eng\n",
      "SI  - GENBANK/KT829450\n",
      "SI  - GENBANK/KT829451\n",
      "SI  - GENBANK/KT829452\n",
      "SI  - GENBANK/KT829453\n",
      "SI  - GENBANK/KT829454\n",
      "SI  - GENBANK/KT829455\n",
      "SI  - GENBANK/KT829456\n",
      "SI  - GENBANK/KT829457\n",
      "SI  - GENBANK/KT829458\n",
      "SI  - GENBANK/KT829459\n",
      "SI  - GENBANK/KT829460\n",
      "SI  - GENBANK/KT829461\n",
      "SI  - GENBANK/KT829462\n",
      "SI  - GENBANK/KT829463\n",
      "SI  - GENBANK/KT829464\n",
      "SI  - GENBANK/KT829465\n",
      "SI  - GENBANK/KT829466\n",
      "SI  - GENBANK/KT829467\n",
      "SI  - GENBANK/KT829468\n",
      "SI  - GENBANK/KT829469\n",
      "SI  - GENBANK/KT829470\n",
      "SI  - GENBANK/KT829471\n",
      "SI  - GENBANK/KT829472\n",
      "GR  - D43 TW008261/TW/FIC NIH HHS/United States\n",
      "GR  - R15 AI077896/AI/NIAID NIH HHS/United States\n",
      "GR  - 1D43TW008261-01A1/TW/FIC NIH HHS/United States\n",
      "GR  - 1R15AI077896-01/AI/NIAID NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20151014\n",
      "PL  - United States\n",
      "TA  - PLoS One\n",
      "JT  - PloS one\n",
      "JID - 101285081\n",
      "RN  - 0 (RNA, Ribosomal, 18S)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Chagas Disease/*genetics/parasitology/transmission\n",
      "MH  - Chiroptera/*genetics/parasitology\n",
      "MH  - Ecuador\n",
      "MH  - Genotype\n",
      "MH  - Humans\n",
      "MH  - Molecular Sequence Data\n",
      "MH  - Phylogeny\n",
      "MH  - *Phylogeography\n",
      "MH  - RNA, Ribosomal, 18S/genetics\n",
      "MH  - Sequence Analysis, DNA\n",
      "MH  - Trypanosoma cruzi/*genetics/pathogenicity\n",
      "PMC - PMC4605636\n",
      "EDAT- 2015/10/16 06:00\n",
      "MHDA- 2016/06/01 06:00\n",
      "CRDT- 2015/10/15 06:00\n",
      "PHST- 2015/03/30 00:00 [received]\n",
      "PHST- 2015/09/21 00:00 [accepted]\n",
      "PHST- 2015/10/15 06:00 [entrez]\n",
      "PHST- 2015/10/16 06:00 [pubmed]\n",
      "PHST- 2016/06/01 06:00 [medline]\n",
      "AID - 10.1371/journal.pone.0139999 [doi]\n",
      "AID - PONE-D-14-58583 [pii]\n",
      "PST - epublish\n",
      "SO  - PLoS One. 2015 Oct 14;10(10):e0139999. doi: 10.1371/journal.pone.0139999.eCollection 2015.\n",
      "\n",
      "PMID- 26320872\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20160606\n",
      "LR  - 20220129\n",
      "IS  - 1943-7811 (Electronic)\n",
      "IS  - 1525-1578 (Linking)\n",
      "VI  - 17\n",
      "IP  - 5\n",
      "DP  - 2015 Sep\n",
      "TI  - Analytical Validation of Quantitative Real-Time PCR Methods for Quantification ofTrypanosoma cruzi DNA in Blood Samples from Chagas Disease Patients.\n",
      "PG  - 605-15\n",
      "LID - 10.1016/j.jmoldx.2015.04.010 [doi]\n",
      "LID - S1525-1578(15)00129-4 [pii]\n",
      "AB  - An international study was performed by 26 experienced PCR laboratories from 14countries to assess the performance of duplex quantitative real-time PCR (qPCR)strategies on the basis of TaqMan probes for detection and quantification ofparasitic loads in peripheral blood samples from Chagas disease patients. Twomethods were studied: Satellite DNA (SatDNA) qPCR and kinetoplastid DNA (kDNA)qPCR. Both methods included an internal amplification control. Reportable range, analytical sensitivity, limits of detection and quantification, and precisionwere estimated according to international guidelines. In addition, inclusivityand exclusivity were estimated with DNA from stocks representing the differentTrypanosoma cruzi discrete typing units and Trypanosoma rangeli and Leishmaniaspp. Both methods were challenged against 156 blood samples provided by theparticipant laboratories, including samples from acute and chronic patients with varied clinical findings, infected by oral route or vectorial transmission. kDNA qPCR showed better analytical sensitivity than SatDNA qPCR with limits ofdetection of 0.23 and 0.70 parasite equivalents/mL, respectively. Analyses ofclinical samples revealed a high concordance in terms of sensitivity andparasitic loads determined by both SatDNA and kDNA qPCRs. This effort is a major step toward international validation of qPCR methods for the quantification of T.cruzi DNA in human blood samples, aiming to provide an accurate surrogatebiomarker for diagnosis and treatment monitoring for patients with Chagasdisease.\n",
      "CI  - Copyright (c) 2015 American Society for Investigative Pathology and theAssociation for Molecular Pathology. Published by Elsevier Inc. All rightsreserved.\n",
      "FAU - Ramirez, Juan Carlos\n",
      "AU  - Ramirez JC\n",
      "AD  - Laboratory of Molecular Biology of Chagas Disease (LaBMECh), Instituto deInvestigaciones en Ingenieria Genetica y Biologia Molecular (INGEBI-CONICET),Buenos Aires, Argentina.\n",
      "FAU - Cura, Carolina Ines\n",
      "AU  - Cura CI\n",
      "AD  - Laboratory of Molecular Biology of Chagas Disease (LaBMECh), Instituto deInvestigaciones en Ingenieria Genetica y Biologia Molecular (INGEBI-CONICET),Buenos Aires, Argentina.\n",
      "FAU - da Cruz Moreira, Otacilio\n",
      "AU  - da Cruz Moreira O\n",
      "AD  - Laboratory of Molecular Biology of Endemic Diseases, Instituto OswaldoCruz/Fiocruz, Rio de Janeiro, Brazil.\n",
      "FAU - Lages-Silva, Eliane\n",
      "AU  - Lages-Silva E\n",
      "AD  - Laboratory of Discipline of Parasitology, Universidade Federal do TrianguloMineiro, Uberaba, Brazil.\n",
      "FAU - Juiz, Natalia\n",
      "AU  - Juiz N\n",
      "AD  - Laboratory of Molecular Biology of Chagas Disease (LaBMECh), Instituto deInvestigaciones en Ingenieria Genetica y Biologia Molecular (INGEBI-CONICET),Buenos Aires, Argentina.\n",
      "FAU - Velazquez, Elsa\n",
      "AU  - Velazquez E\n",
      "AD  - National Institute of Parasitology \"Dr. Mario Fatala Chaben\", Buenos Aires,Argentina.\n",
      "FAU - Ramirez, Juan David\n",
      "AU  - Ramirez JD\n",
      "AD  - Center for Research in Tropical Microbiology and Parasitology, Universidad de losAndes, Bogota, Colombia.\n",
      "FAU - Alberti, Anahi\n",
      "AU  - Alberti A\n",
      "AD  - Institute of Experimental Pathology, CONICET-Universidad Nacional de Salta,Salta, Argentina.\n",
      "FAU - Pavia, Paula\n",
      "AU  - Pavia P\n",
      "AD  - Laboratory of Molecular Parasitology, Pontificia Universidad Javeriana, Bogota,Colombia.\n",
      "FAU - Flores-Chavez, Maria Delmans\n",
      "AU  - Flores-Chavez MD\n",
      "AD  - National Center for Microbiology, Instituto de Salud Carlos III, Madrid, Spain.\n",
      "FAU - Munoz-Calderon, Arturo\n",
      "AU  - Munoz-Calderon A\n",
      "AD  - Institute of Tropical Medicine, Universidad Central de Venezuela, Caracas,Venezuela.\n",
      "FAU - Perez-Morales, Deyanira\n",
      "AU  - Perez-Morales D\n",
      "AD  - Biomedical Research Institute, Universidad Nacional Autonoma de Mexico, MexicoDF, Mexico.\n",
      "FAU - Santalla, Jose\n",
      "AU  - Santalla J\n",
      "AD  - Laboratory of Parasitology and Molecular Biology, Instituto Nacional deLaboratorios en Salud, La Paz, Bolivia.\n",
      "FAU - Marcos da Matta Guedes, Paulo\n",
      "AU  - Marcos da Matta Guedes P\n",
      "AD  - Department of Microbiology and Parasitology, Universidade Federal do Rio Grandedo Norte, Natal, Brazil.\n",
      "FAU - Peneau, Julie\n",
      "AU  - Peneau J\n",
      "AD  - Hospital and University Laboratory-CH AndrA(c)e Rosemon, Cayenne, French Guiana.\n",
      "FAU - Marcet, Paula\n",
      "AU  - Marcet P\n",
      "AD  - Division of Parasitic Diseases and Malaria, Centers for Disease Control andPrevention, Atlanta, Georgia.\n",
      "FAU - Padilla, Carlos\n",
      "AU  - Padilla C\n",
      "AD  - National Center for Public Health, Instituto Nacional de Salud, Lima, Peru.\n",
      "FAU - Cruz-Robles, David\n",
      "AU  - Cruz-Robles D\n",
      "AD  - Laboratory of Genomics, Instituto Nacional de Cardiologia \"Ignacio Chavez\",Mexico DF, Mexico.\n",
      "FAU - Valencia, Edward\n",
      "AU  - Valencia E\n",
      "AD  - Laboratory for Research in Infectious Diseases, Universidad Peruana CayetanoHeredia, Lima, Peru.\n",
      "FAU - Crisante, Gladys Elena\n",
      "AU  - Crisante GE\n",
      "AD  - Department of Biology, Universidad de los Andes, Merida, Venezuela.\n",
      "FAU - Greif, Gonzalo\n",
      "AU  - Greif G\n",
      "AD  - Molecular Biology Unit, Instituto Pasteur de Montevideo, Montevideo, Uruguay.\n",
      "FAU - Zulantay, Ines\n",
      "AU  - Zulantay I\n",
      "AD  - Basic Clinical Parasitology Laboratory, Universidad de Chile, Santiago, Chile.\n",
      "FAU - Costales, Jaime Alfredo\n",
      "AU  - Costales JA\n",
      "AD  - Research Center for Infectious Diseases, Pontificia Universidad Catolica deEcuador, Quito, Ecuador.\n",
      "FAU - Alvarez-Martinez, Miriam\n",
      "AU  - Alvarez-Martinez M\n",
      "AD  - Microbiology Department, Hospital Clinic and Barcelona Centre for InternationalHealth Research (CRESIB), Barcelona, Spain.\n",
      "FAU - Martinez, Norma Edith\n",
      "AU  - Martinez NE\n",
      "AD  - State Laboratory of Public Health, Acapulco, Mexico.\n",
      "FAU - Villarroel, Rodrigo\n",
      "AU  - Villarroel R\n",
      "AD  - Biomedical Department, Instituto de Salud PA masculineblica, Santiago, Chile.\n",
      "FAU - Villarroel, Sandro\n",
      "AU  - Villarroel S\n",
      "AD  - Laboratory of Molecular Biology, Universidad Mayor de San Simon, Cochabamba,Bolivia.\n",
      "FAU - Sanchez, Zunilda\n",
      "AU  - Sanchez Z\n",
      "AD  - Research Institute for Health Sciences, Universidad Nacional de Asuncion,Asuncion, Paraguay.\n",
      "FAU - Bisio, Margarita\n",
      "AU  - Bisio M\n",
      "AD  - Laboratory of Molecular Biology of Chagas Disease (LaBMECh), Instituto deInvestigaciones en Ingenieria Genetica y Biologia Molecular (INGEBI-CONICET),Buenos Aires, Argentina.\n",
      "FAU - Parrado, Rudy\n",
      "AU  - Parrado R\n",
      "AD  - Laboratory of Molecular Biology, Universidad Mayor de San Simon, Cochabamba,Bolivia.\n",
      "FAU - Maria da Cunha Galvao, Lucia\n",
      "AU  - Maria da Cunha Galvao L\n",
      "AD  - Department of Microbiology and Parasitology, Universidade Federal do Rio Grandedo Norte, Natal, Brazil.\n",
      "FAU - Jacome da Camara, Antonia Claudia\n",
      "AU  - Jacome da Camara AC\n",
      "AD  - Department of Microbiology and Parasitology, Universidade Federal do Rio Grandedo Norte, Natal, Brazil.\n",
      "FAU - Espinoza, Bertha\n",
      "AU  - Espinoza B\n",
      "AD  - Biomedical Research Institute, Universidad Nacional Autonoma de Mexico, MexicoDF, Mexico.\n",
      "FAU - Alarcon de Noya, Belkisyole\n",
      "AU  - Alarcon de Noya B\n",
      "AD  - Institute of Tropical Medicine, Universidad Central de Venezuela, Caracas,Venezuela.\n",
      "FAU - Puerta, Concepcion\n",
      "AU  - Puerta C\n",
      "AD  - Laboratory of Molecular Parasitology, Pontificia Universidad Javeriana, Bogota,Colombia.\n",
      "FAU - Riarte, Adelina\n",
      "AU  - Riarte A\n",
      "AD  - National Institute of Parasitology \"Dr. Mario Fatala Chaben\", Buenos Aires,Argentina.\n",
      "FAU - Diosque, Patricio\n",
      "AU  - Diosque P\n",
      "AD  - Institute of Experimental Pathology, CONICET-Universidad Nacional de Salta,Salta, Argentina.\n",
      "FAU - Sosa-Estani, Sergio\n",
      "AU  - Sosa-Estani S\n",
      "AD  - National Institute of Parasitology \"Dr. Mario Fatala Chaben\", Buenos Aires,Argentina.\n",
      "FAU - Guhl, Felipe\n",
      "AU  - Guhl F\n",
      "AD  - Center for Research in Tropical Microbiology and Parasitology, Universidad de losAndes, Bogota, Colombia.\n",
      "FAU - Ribeiro, Isabela\n",
      "AU  - Ribeiro I\n",
      "AD  - Drugs for Neglected Diseases Initiative (DNDi), Geneva, Switzerland.\n",
      "FAU - Aznar, Christine\n",
      "AU  - Aznar C\n",
      "AD  - Hospital and University Laboratory-CH AndrA(c)e Rosemon, Cayenne, French Guiana.\n",
      "FAU - Britto, Constanca\n",
      "AU  - Britto C\n",
      "AD  - Laboratory of Molecular Biology of Endemic Diseases, Instituto OswaldoCruz/Fiocruz, Rio de Janeiro, Brazil.\n",
      "FAU - Yadon, Zaida Estela\n",
      "AU  - Yadon ZE\n",
      "AD  - Communicable Diseases and Health Analysis Department, Pan American HealthOrganization/World Health Organization (PAHO/WHO), Rio de Janeiro, Brazil.\n",
      "FAU - Schijman, Alejandro G\n",
      "AU  - Schijman AG\n",
      "AD  - Laboratory of Molecular Biology of Chagas Disease (LaBMECh), Instituto deInvestigaciones en Ingenieria Genetica y Biologia Molecular (INGEBI-CONICET),Buenos Aires, Argentina. Electronic address: schijman@dna.uba.ar.\n",
      "LA  - eng\n",
      "GR  - 001/WHO_/World Health Organization/International\n",
      "GR  - CC999999/Intramural CDC HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Validation Study\n",
      "PL  - United States\n",
      "TA  - J Mol Diagn\n",
      "JT  - The Journal of molecular diagnostics : JMD\n",
      "JID - 100893612\n",
      "RN  - 0 (DNA, Protozoan)\n",
      "SB  - IM\n",
      "MH  - Chagas Disease/*blood/diagnosis/genetics/parasitology\n",
      "MH  - DNA, Protozoan/*analysis/isolation & purification\n",
      "MH  - Humans\n",
      "MH  - International Cooperation\n",
      "MH  - Laboratory Proficiency Testing\n",
      "MH  - Molecular Typing\n",
      "MH  - Parasitemia/blood/diagnosis/genetics\n",
      "MH  - Real-Time Polymerase Chain Reaction/*methods\n",
      "MH  - Sensitivity and Specificity\n",
      "MH  - Trypanosoma cruzi/*genetics/isolation & purification\n",
      "PMC - PMC4698797\n",
      "MID - NIHMS729152\n",
      "OID - NLM: HHSPA729152\n",
      "EDAT- 2015/09/01 06:00\n",
      "MHDA- 2016/06/09 06:00\n",
      "CRDT- 2015/09/01 06:00\n",
      "PHST- 2014/11/02 00:00 [received]\n",
      "PHST- 2015/02/01 00:00 [revised]\n",
      "PHST- 2015/04/03 00:00 [accepted]\n",
      "PHST- 2015/09/01 06:00 [entrez]\n",
      "PHST- 2015/09/01 06:00 [pubmed]\n",
      "PHST- 2016/06/09 06:00 [medline]\n",
      "AID - S1525-1578(15)00129-4 [pii]\n",
      "AID - 10.1016/j.jmoldx.2015.04.010 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Mol Diagn. 2015 Sep;17(5):605-15. doi: 10.1016/j.jmoldx.2015.04.010.\n",
      "\n",
      "PMID- 26240451\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "DCOM- 20150805\n",
      "LR  - 20200929\n",
      "IS  - 0031-5850 (Print)\n",
      "IS  - 0031-5850 (Linking)\n",
      "VI  - 34\n",
      "DP  - 2015 Jun\n",
      "TI  - Fungal Planet description sheets: 320-370.\n",
      "PG  - 167-266\n",
      "LID - 10.3767/003158515X688433 [doi]\n",
      "AB  - Novel species of fungi described in the present study include the following from Malaysia: Castanediella eucalypti from Eucalyptus pellita, Codinaea acacia fromAcacia mangium, Emarcea eucalyptigena from Eucalyptus brassiana, Myrtapenidiella eucalyptorum from Eucalyptus pellita, Pilidiella eucalyptigena from Eucalyptusbrassiana and Strelitziana malaysiana from Acacia mangium. Furthermore,Stachybotrys sansevieriicola is described from Sansevieria ehrenbergii(Tanzania), Phacidium grevilleae from Grevillea robusta (Uganda), Graphium jumulufrom Adansonia gregorii and Ophiostoma eucalyptigena from Eucalyptus marginata(Australia), Pleurophoma ossicola from bone and Plectosphaerella populi fromPopulus nigra (Germany), Colletotrichum neosansevieriae from Sansevieriatrifasciata, Elsinoe othonnae from Othonna quinquedentata and Zeloasperisporiumcliviae (Zeloasperisporiaceae fam. nov.) from Clivia sp. (South Africa),Neodevriesia pakbiae, Phaeophleospora hymenocallidis and Phaeophleosporahymenocallidicola on leaves of a fern (Thailand), Melanconium elaeidicola fromElaeis guineensis (Indonesia), Hormonema viticola from Vitis vinifera (CanaryIslands), Chlorophyllum pseudoglobossum from a grassland (India), Triadelphiadisseminata from an immunocompromised patient (Saudi Arabia), Colletotrichumabscissum from Citrus (Brazil), Polyschema sclerotigenum and Phialemoniumlimoniforme from human patients (USA), Cadophora viticola from Vitis vinifera(Spain), Entoloma flavovelutinum and Bolbitius aurantiorugosus from soil(Vietnam), Rhizopogon granuloflavus from soil (Cape Verde Islands), Tulasnellaeremophila from Euphorbia officinarum subsp. echinus (Morocco), Verrucostomamartinicensis from Danaea elliptica (French West Indies), Metschnikowia colchici from Colchicum autumnale (Bulgaria), Thelebolus microcarpus from soil (Argentina)and Ceratocystis adelpha from Theobroma cacao (Ecuador). Myrmecridium iridis(Myrmecridiales ord. nov., Myrmecridiaceae fam. nov.) is also described from Irissp. (The Netherlands). Novel genera include (Ascomycetes): Budhanggurabania from Cynodon dactylon (Australia), Soloacrosporiella, Xenocamarosporium,Neostrelitziana and Castanediella from Acacia mangium and Sabahriopsis fromEucalyptus brassiana (Malaysia), Readerielliopsis from basidiomata of Fuscoporia wahlbergii (French Guyana), Neoplatysporoides from Aloe ferox (Tanzania),Wojnowiciella, Chrysofolia and Neoeriomycopsis from Eucalyptus (Colombia),Neophaeomoniella from Eucalyptus globulus (USA), Pseudophaeomoniella from Oleaeuropaea (Italy), Paraphaeomoniella from Encephalartos altensteinii,Aequabiliella, Celerioriella and Minutiella from Prunus (South Africa).Tephrocybella (Basidiomycetes) represents a novel genus from wood (Italy).Morphological and culture characteristics along with ITS DNA barcodes areprovided for all taxa.\n",
      "FAU - Crous, P W\n",
      "AU  - Crous PW\n",
      "AD  - CBS-KNAW Fungal Biodiversity Centre, P.O. Box 85167, 3508 AD Utrecht, TheNetherlands; ; Forestry and Agricultural Biotechnology Institute (FABI),University of Pretoria, P. Bag X20, Pretoria, 0028, South Africa. ; Microbiology,Department of Biology, Utrecht University, Padualaan 8, 3584 CH Utrecht, TheNetherlands.\n",
      "FAU - Wingfield, M J\n",
      "AU  - Wingfield MJ\n",
      "AD  - Forestry and Agricultural Biotechnology Institute (FABI), University of Pretoria,P. Bag X20, Pretoria, 0028, South Africa.\n",
      "FAU - Guarro, J\n",
      "AU  - Guarro J\n",
      "AD  - Mycology Unit, Medical School and IISPV, Universitat Rovira i Virgili (URV), SantLlorenc 21, 43201 Reus, Tarragona, Spain.\n",
      "FAU - Hernandez-Restrepo, M\n",
      "AU  - Hernandez-Restrepo M\n",
      "AD  - CBS-KNAW Fungal Biodiversity Centre, P.O. Box 85167, 3508 AD Utrecht, TheNetherlands; ; Forestry and Agricultural Biotechnology Institute (FABI),University of Pretoria, P. Bag X20, Pretoria, 0028, South Africa.\n",
      "FAU - Sutton, D A\n",
      "AU  - Sutton DA\n",
      "AD  - Fungus Testing Laboratory, Department of Pathology, University of Texas HealthScience Center, 7703 Floyd Curl Dr., San Antonio, Texas 78229-3900, USA.\n",
      "FAU - Acharya, K\n",
      "AU  - Acharya K\n",
      "AD  - Molecular and Applied Mycology and Plant Pathology Laboratory, Department ofBotany, University of Calcutta, 35, Ballygunge Circular Road, Kolkata 700019,West Bengal, India.\n",
      "FAU - Barber, P A\n",
      "AU  - Barber PA\n",
      "AD  - Arbor Carbon, P.O. Box 1065, Willagee Central, 6156 Australia, and School ofVeterinary and Life Sciences, Murdoch University, Murdoch WA, 6150 Australia.\n",
      "FAU - Boekhout, T\n",
      "AU  - Boekhout T\n",
      "AD  - CBS-KNAW Fungal Biodiversity Centre, P.O. Box 85167, 3508 AD Utrecht, TheNetherlands;\n",
      "FAU - Dimitrov, R A\n",
      "AU  - Dimitrov RA\n",
      "AD  - Sofia University \"St. Kliment Ohridski\", 5 James Bourchier Blvd., Sofia 1164,Bulgaria.\n",
      "FAU - Duenas, M\n",
      "AU  - Duenas M\n",
      "AD  - Departamento de Micologia, Real Jardin Botanico-CSIC, Plaza de Murillo 2, 28014Madrid, Spain.\n",
      "FAU - Dutta, A K\n",
      "AU  - Dutta AK\n",
      "AD  - Molecular and Applied Mycology and Plant Pathology Laboratory, Department ofBotany, University of Calcutta, 35, Ballygunge Circular Road, Kolkata 700019,West Bengal, India.\n",
      "FAU - Gene, J\n",
      "AU  - Gene J\n",
      "AD  - Mycology Unit, Medical School and IISPV, Universitat Rovira i Virgili (URV), SantLlorenc 21, 43201 Reus, Tarragona, Spain.\n",
      "FAU - Gouliamova, D E\n",
      "AU  - Gouliamova DE\n",
      "AD  - The Stephan Angeloff Institute of Microbiology, Bulgarian Academy of Sciences,Acad. Georgi Bonchev 26, Sofia 1113, Bulgaria.\n",
      "FAU - Groenewald, M\n",
      "AU  - Groenewald M\n",
      "AD  - CBS-KNAW Fungal Biodiversity Centre, P.O. Box 85167, 3508 AD Utrecht, TheNetherlands;\n",
      "FAU - Lombard, L\n",
      "AU  - Lombard L\n",
      "AD  - CBS-KNAW Fungal Biodiversity Centre, P.O. Box 85167, 3508 AD Utrecht, TheNetherlands;\n",
      "FAU - Morozova, O V\n",
      "AU  - Morozova OV\n",
      "AD  - Joint Russian-Vietnamese Tropical Research and Technological Center, SouthBranch, Ho Chi Minh City, Vietnam. ; Komarov Botanical Institute of the RussianAcademy of Sciences, 197376, Prof. Popov Str. 2, Saint Petersburg, Russia.\n",
      "FAU - Sarkar, J\n",
      "AU  - Sarkar J\n",
      "AD  - Molecular and Applied Mycology and Plant Pathology Laboratory, Department ofBotany, University of Calcutta, 35, Ballygunge Circular Road, Kolkata 700019,West Bengal, India.\n",
      "FAU - Smith, M Th\n",
      "AU  - Smith MT\n",
      "AD  - CBS-KNAW Fungal Biodiversity Centre, P.O. Box 85167, 3508 AD Utrecht, TheNetherlands;\n",
      "FAU - Stchigel, A M\n",
      "AU  - Stchigel AM\n",
      "AD  - Mycology Unit, Medical School and IISPV, Universitat Rovira i Virgili (URV), SantLlorenc 21, 43201 Reus, Tarragona, Spain.\n",
      "FAU - Wiederhold, N P\n",
      "AU  - Wiederhold NP\n",
      "AD  - Fungus Testing Laboratory, Department of Pathology, University of Texas HealthScience Center, 7703 Floyd Curl Dr., San Antonio, Texas 78229-3900, USA.\n",
      "FAU - Alexandrova, A V\n",
      "AU  - Alexandrova AV\n",
      "AD  - Joint Russian-Vietnamese Tropical Research and Technological Center, SouthBranch, Ho Chi Minh City, Vietnam. ; Lomonosov Moscow State University, GSP-1,119991, Leninskie Gory, Moscow, Russia.\n",
      "FAU - Antelmi, I\n",
      "AU  - Antelmi I\n",
      "AD  - Dip. Scienze del Suolo, della Pianta e degli Alimenti, Universita degli Studi di Bari - Aldo Moro, Via Amendola 165/A, 70126 Bari, Italy.\n",
      "FAU - Armengol, J\n",
      "AU  - Armengol J\n",
      "AD  - Instituto Agroforestal Mediterraneo, Universitat Politecnica de Valencia, Camino de Vera s/n, 46022 Valencia, Spain.\n",
      "FAU - Barnes, I\n",
      "AU  - Barnes I\n",
      "AD  - Department of Genetics, Forestry and Agricultural Biotechnology Institute (FABI),University of Pretoria, Pretoria 0002, South Africa.\n",
      "FAU - Cano-Lira, J F\n",
      "AU  - Cano-Lira JF\n",
      "AD  - Mycology Unit, Medical School and IISPV, Universitat Rovira i Virgili (URV), SantLlorenc 21, 43201 Reus, Tarragona, Spain.\n",
      "FAU - Castaneda Ruiz, R F\n",
      "AU  - Castaneda Ruiz RF\n",
      "AD  - Instituto de Investigaciones Fundamentales en Agricultura tropical \"AlejandroHumboldt\", calle 1, esq. 2, C.P. 17200, Santiago de Las Vegas, La Habana, Cuba.\n",
      "FAU - Contu, M\n",
      "AU  - Contu M\n",
      "AD  - Via Marmilla 12, I-07026 Olbia (OT), Italy.\n",
      "FAU - Courtecuisse, Pr R\n",
      "AU  - Courtecuisse PR\n",
      "AD  - Laboratoire des sciences vegetales et fongiques, Faculte des sciencespharmaceutiques et biologiques B.P. 83, F59000 Lille Cedex, France.\n",
      "FAU - da Silveira, A L\n",
      "AU  - da Silveira AL\n",
      "AD  - Universidade Estadual Paulista Julio de Mesquita Filho (UNESP), Campusexperimental de Dracena, Dracena, Sao Paulo State, CEP: 17900-00, Brazil.\n",
      "FAU - Decock, C A\n",
      "AU  - Decock CA\n",
      "AD  - Mycotheque de l'Universite catholique de Louvain (MUCL, BCCMTM), Earth and LifeInstitute - ELIM - Mycology, Universite catholique de Louvain, Croix du Sud 2 bteL7.05.06, B-1348 Louvain-la-Neuve, Belgium.\n",
      "FAU - de Goes, A\n",
      "AU  - de Goes A\n",
      "AD  - Universidade Estadual Paulista Julio de Mesquita Filho (UNESP), Campusexperimental de Dracena, Dracena, Sao Paulo State, CEP: 17900-00, Brazil.\n",
      "FAU - Edathodu, J\n",
      "AU  - Edathodu J\n",
      "AD  - Section of Infectious Diseases, Department of Medicine and Department ofMicrobiology, King Faisal Hospital and Research Centre, Riyadh, Saudi Arabia.\n",
      "FAU - Ercole, E\n",
      "AU  - Ercole E\n",
      "AD  - Department of Life Sciences and Systems Biology, University of Torino, Viale P.A.Mattioli 25, I-10125 Torino, Italy.\n",
      "FAU - Firmino, A C\n",
      "AU  - Firmino AC\n",
      "AD  - Universidade Estadual Paulista Julio de Mesquita Filho (UNESP), Campusexperimental de Dracena, Dracena, Sao Paulo State, CEP: 17900-00, Brazil.\n",
      "FAU - Fourie, A\n",
      "AU  - Fourie A\n",
      "AD  - Department of Genetics, Forestry and Agricultural Biotechnology Institute (FABI),University of Pretoria, Pretoria 0002, South Africa.\n",
      "FAU - Fournier, J\n",
      "AU  - Fournier J\n",
      "AD  - Las Muros, 09420 Rimont, France.\n",
      "FAU - Furtado, E L\n",
      "AU  - Furtado EL\n",
      "AD  - Universidade Paulista Julio de Mesquita Filho (UNESP), Faculdade de CienciasAgronomicas, Botucatu, Sao Paulo State, CEP: 18610-307, Brazil.\n",
      "FAU - Geering, A D W\n",
      "AU  - Geering AD\n",
      "AD  - Queensland Alliance for Agriculture and Food Innovation, The University ofQueensland, GPO Box 267, Brisbane 4001, Queensland, Australia and PlantBiosecurity CRC, LPO Box 5012, Bruce 2617, ACT Australia.\n",
      "FAU - Gershenzon, J\n",
      "AU  - Gershenzon J\n",
      "AD  - Department of Biochemistry, Max Planck Institute for Chemical Ecology,Hans-Knoll-Strasse 8. D-07745, Jena, Germany.\n",
      "FAU - Giraldo, A\n",
      "AU  - Giraldo A\n",
      "AD  - Mycology Unit, Medical School and IISPV, Universitat Rovira i Virgili (URV), SantLlorenc 21, 43201 Reus, Tarragona, Spain.\n",
      "FAU - Gramaje, D\n",
      "AU  - Gramaje D\n",
      "AD  - Instituto de Ciencias de la Vid y del Vino (Gobierno de La Rioja-CSIC-Universidadde La Rioja), Ctra. de Burgos, Km. 6, 26007 Logrono, La Rioja, Spain.\n",
      "FAU - Hammerbacher, A\n",
      "AU  - Hammerbacher A\n",
      "AD  - Department of Biochemistry, Max Planck Institute for Chemical Ecology,Hans-Knoll-Strasse 8. D-07745, Jena, Germany.\n",
      "FAU - He, X-L\n",
      "AU  - He XL\n",
      "AD  - Soil and Fertilizer Institute, Sichuan Academy of Agricultural Sciences, Chengdu,China.\n",
      "FAU - Haryadi, D\n",
      "AU  - Haryadi D\n",
      "AD  - R & D Centre, Asian Agri Sumatra, Bahilang Estate, P.O.Box 35, Tebing Tinggi,Deli 20600, North Sumatra, Indonesia.\n",
      "FAU - Khemmuk, W\n",
      "AU  - Khemmuk W\n",
      "AD  - Queensland Alliance for Agriculture and Food Innovation, The University ofQueensland, GPO Box 267, Brisbane 4001, Queensland, Australia and PlantBiosecurity CRC, LPO Box 5012, Bruce 2617, ACT Australia.\n",
      "FAU - Kovalenko, A E\n",
      "AU  - Kovalenko AE\n",
      "AD  - Joint Russian-Vietnamese Tropical Research and Technological Center, SouthBranch, Ho Chi Minh City, Vietnam. ; Komarov Botanical Institute of the RussianAcademy of Sciences, 197376, Prof. Popov Str. 2, Saint Petersburg, Russia.\n",
      "FAU - Krawczynski, R\n",
      "AU  - Krawczynski R\n",
      "AD  - Schmiedestrasse 11, D-27248 Ehrenburg, Germany.\n",
      "FAU - Laich, F\n",
      "AU  - Laich F\n",
      "AD  - Instituto Canario de Investigaciones Agrarias, Ctra. El Boqueron s/n, 38270,Valle de Guerra, Tenerife, Spain.\n",
      "FAU - Lechat, C\n",
      "AU  - Lechat C\n",
      "AD  - Ascofrance, 64 route de Chize, 79360 Villiers en Bois, France.\n",
      "FAU - Lopes, U P\n",
      "AU  - Lopes UP\n",
      "AD  - Universidade Federal de Vicosa, Vicosa, Minas Gerais, Brazil.\n",
      "FAU - Madrid, H\n",
      "AU  - Madrid H\n",
      "AD  - Center for Genomics and Bioinformatics and Medicine Faculty, Mayor University,Camino La Piramide 5750, Huechuraba, Santiago, Chile.\n",
      "FAU - Malysheva, E F\n",
      "AU  - Malysheva EF\n",
      "AD  - Komarov Botanical Institute of the Russian Academy of Sciences, 197376, Prof.Popov Str. 2, Saint Petersburg, Russia.\n",
      "FAU - Marin-Felix, Y\n",
      "AU  - Marin-Felix Y\n",
      "AD  - Mycology Unit, Medical School and IISPV, Universitat Rovira i Virgili (URV), SantLlorenc 21, 43201 Reus, Tarragona, Spain.\n",
      "FAU - Martin, M P\n",
      "AU  - Martin MP\n",
      "AD  - Departamento de Micologia, Real Jardin Botanico-CSIC, Plaza de Murillo 2, 28014Madrid, Spain.\n",
      "FAU - Mostert, L\n",
      "AU  - Mostert L\n",
      "AD  - Department of Plant Pathology, University of Stellenbosch, Private Bag X1,Stellenbosch 7602, South Africa.\n",
      "FAU - Nigro, F\n",
      "AU  - Nigro F\n",
      "AD  - Dip. Scienze del Suolo, della Pianta e degli Alimenti, Universita degli Studi di Bari - Aldo Moro, Via Amendola 165/A, 70126 Bari, Italy.\n",
      "FAU - Pereira, O L\n",
      "AU  - Pereira OL\n",
      "AD  - Universidade Federal de Vicosa, Vicosa, Minas Gerais, Brazil.\n",
      "FAU - Picillo, B\n",
      "AU  - Picillo B\n",
      "AD  - Via Roma 139, I-81017 S. Angelo d'Alife (CE), Italy.\n",
      "FAU - Pinho, D B\n",
      "AU  - Pinho DB\n",
      "AD  - Universidade Federal de Vicosa, Vicosa, Minas Gerais, Brazil.\n",
      "FAU - Popov, E S\n",
      "AU  - Popov ES\n",
      "AD  - Joint Russian-Vietnamese Tropical Research and Technological Center, SouthBranch, Ho Chi Minh City, Vietnam. ; Komarov Botanical Institute of the RussianAcademy of Sciences, 197376, Prof. Popov Str. 2, Saint Petersburg, Russia.\n",
      "FAU - Rodas Pelaez, C A\n",
      "AU  - Rodas Pelaez CA\n",
      "AD  - Smurfit Kappa Colombia, Calle 15 # 18-109 Yumbo, Colombia.\n",
      "FAU - Rooney-Latham, S\n",
      "AU  - Rooney-Latham S\n",
      "AD  - California Department of Food and Agriculture, Plant Health and Pest PreventionServices, Plant Pest Diagnostics Lab, 3294 Meadowview Road Sacramento, CA95832-1448, USA.\n",
      "FAU - Sandoval-Denis, M\n",
      "AU  - Sandoval-Denis M\n",
      "AD  - Mycology Unit, Medical School and IISPV, Universitat Rovira i Virgili (URV), SantLlorenc 21, 43201 Reus, Tarragona, Spain.\n",
      "FAU - Shivas, R G\n",
      "AU  - Shivas RG\n",
      "AD  - Plant Pathology Herbarium, Department of Agriculture and Forestry, Dutton Park4102, Queensland, Australia and Plant Biosecurity CRC, LPO Box 5012, Bruce 2617, ACT Australia.\n",
      "FAU - Silva, V\n",
      "AU  - Silva V\n",
      "AD  - Center for Genomics and Bioinformatics and Medicine Faculty, Mayor University,Camino La Piramide 5750, Huechuraba, Santiago, Chile.\n",
      "FAU - Stoilova-Disheva, M M\n",
      "AU  - Stoilova-Disheva MM\n",
      "AD  - The Stephan Angeloff Institute of Microbiology, Bulgarian Academy of Sciences,Acad. Georgi Bonchev 26, Sofia 1113, Bulgaria.\n",
      "FAU - Telleria, M T\n",
      "AU  - Telleria MT\n",
      "AD  - Departamento de Micologia, Real Jardin Botanico-CSIC, Plaza de Murillo 2, 28014Madrid, Spain.\n",
      "FAU - Ullah, C\n",
      "AU  - Ullah C\n",
      "AD  - Department of Biochemistry, Max Planck Institute for Chemical Ecology,Hans-Knoll-Strasse 8. D-07745, Jena, Germany.\n",
      "FAU - Unsicker, S B\n",
      "AU  - Unsicker SB\n",
      "AD  - Department of Biochemistry, Max Planck Institute for Chemical Ecology,Hans-Knoll-Strasse 8. D-07745, Jena, Germany.\n",
      "FAU - van der Merwe, N A\n",
      "AU  - van der Merwe NA\n",
      "AD  - Department of Genetics, Forestry and Agricultural Biotechnology Institute (FABI),University of Pretoria, Pretoria 0002, South Africa.\n",
      "FAU - Vizzini, A\n",
      "AU  - Vizzini A\n",
      "AD  - Department of Life Sciences and Systems Biology, University of Torino, Viale P.A.Mattioli 25, I-10125 Torino, Italy.\n",
      "FAU - Wagner, H-G\n",
      "AU  - Wagner HG\n",
      "AD  - Lowensteinring 28, D-12353 Berlin / Buckow, Germany.\n",
      "FAU - Wong, P T W\n",
      "AU  - Wong PT\n",
      "AD  - University of Sydney, Plant Breeding Institute, 107 Cobbitty Rd, Cobbitty 2570,New South Wales, Australia.\n",
      "FAU - Wood, A R\n",
      "AU  - Wood AR\n",
      "AD  - ARC - Plant Protection Research Institute, P. Bag X5017, Stellenbosch 7599, SouthAfrica.\n",
      "FAU - Groenewald, J Z\n",
      "AU  - Groenewald JZ\n",
      "AD  - CBS-KNAW Fungal Biodiversity Centre, P.O. Box 85167, 3508 AD Utrecht, TheNetherlands;\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20150610\n",
      "PL  - Netherlands\n",
      "TA  - Persoonia\n",
      "JT  - Persoonia\n",
      "JID - 19540520R\n",
      "PMC - PMC4510277\n",
      "OTO - NOTNLM\n",
      "OT  - ITS DNA barcodes\n",
      "OT  - LSU\n",
      "OT  - novel fungal species\n",
      "OT  - systematics\n",
      "EDAT- 2015/08/05 06:00\n",
      "MHDA- 2015/08/05 06:01\n",
      "CRDT- 2015/08/05 06:00\n",
      "PHST- 2015/03/01 00:00 [received]\n",
      "PHST- 2015/05/28 00:00 [accepted]\n",
      "PHST- 2015/08/05 06:00 [entrez]\n",
      "PHST- 2015/08/05 06:00 [pubmed]\n",
      "PHST- 2015/08/05 06:01 [medline]\n",
      "AID - 10.3767/003158515X688433 [doi]\n",
      "PST - ppublish\n",
      "SO  - Persoonia. 2015 Jun;34:167-266. doi: 10.3767/003158515X688433. Epub 2015 Jun 10.\n",
      "\n",
      "PMID- 26226137\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20161219\n",
      "LR  - 20220210\n",
      "IS  - 1530-0366 (Electronic)\n",
      "IS  - 1098-3600 (Linking)\n",
      "VI  - 18\n",
      "IP  - 4\n",
      "DP  - 2016 Apr\n",
      "TI  - Comprehensive analysis via exome sequencing uncovers genetic etiology inautosomal recessive nonsyndromic deafness in a large multiethnic cohort.\n",
      "PG  - 364-71\n",
      "LID - 10.1038/gim.2015.89 [doi]\n",
      "AB  - PURPOSE: Autosomal recessive nonsyndromic deafness (ARNSD) is characterized by a high degree of genetic heterogeneity, with reported mutations in 58 differentgenes. This study was designed to detect deafness-causing variants in amultiethnic cohort with ARNSD by using whole-exome sequencing (WES). METHODS:After excluding mutations in the most common gene, GJB2, we performed WES in 160 multiplex families with ARNSD from Turkey, Iran, Mexico, Ecuador, and Puerto Ricoto screen for mutations in all known ARNSD genes. RESULTS: We detectedARNSD-causing variants in 90 (56%) families, 54% of which had not been previouslyreported. Identified mutations were located in 31 known ARNSD genes. The mostcommon genes with mutations were MYO15A (13%), MYO7A (11%), SLC26A4 (10%),TMPRSS3 (9%), TMC1 (8%), ILDR1 (6%), and CDH23 (4%). Nine mutations were detectedin multiple families with shared haplotypes, suggesting founder effects.CONCLUSION: We report on a large multiethnic cohort with ARNSD in whichcomprehensive analysis of all known ARNSD genes identifies causative DNA variantsin 56% of the families. In the remaining families, WES allows us to search forcausative variants in novel genes, thus improving our ability to explain theunderlying etiology in more families.Genet Med 18 4, 364-371.\n",
      "FAU - Bademci, Guney\n",
      "AU  - Bademci G\n",
      "AD  - Dr. John T. Macdonald Foundation Department of Human Genetics and John P. HussmanInstitute for Human Genomics, University of Miami Miller School of Medicine,Miami, Florida, USA.\n",
      "FAU - Foster, Joseph 2nd\n",
      "AU  - Foster J 2nd\n",
      "AD  - Dr. John T. Macdonald Foundation Department of Human Genetics and John P. HussmanInstitute for Human Genomics, University of Miami Miller School of Medicine,Miami, Florida, USA.\n",
      "FAU - Mahdieh, Nejat\n",
      "AU  - Mahdieh N\n",
      "AD  - Rajaie Cardiovascular Medical and Research Center, Iran University of MedicalSciences, Tehran, Iran.\n",
      "FAU - Bonyadi, Mortaza\n",
      "AU  - Bonyadi M\n",
      "AD  - Faculty of Natural Sciences, Center of Excellence for Biodiversity, University ofTabriz, Tabriz, Iran.\n",
      "FAU - Duman, Duygu\n",
      "AU  - Duman D\n",
      "AUID- ORCID: http://orcid.org/0000-0001-7583-0349\n",
      "AD  - Division of Genetics, Department of Pediatrics, Ankara University School ofMedicine, Ankara, Turkey.\n",
      "FAU - Cengiz, F Basak\n",
      "AU  - Cengiz FB\n",
      "AD  - Division of Genetics, Department of Pediatrics, Ankara University School ofMedicine, Ankara, Turkey.\n",
      "FAU - Menendez, Ibis\n",
      "AU  - Menendez I\n",
      "AD  - Dr. John T. Macdonald Foundation Department of Human Genetics and John P. HussmanInstitute for Human Genomics, University of Miami Miller School of Medicine,Miami, Florida, USA.\n",
      "FAU - Diaz-Horta, Oscar\n",
      "AU  - Diaz-Horta O\n",
      "AD  - Dr. John T. Macdonald Foundation Department of Human Genetics and John P. HussmanInstitute for Human Genomics, University of Miami Miller School of Medicine,Miami, Florida, USA.\n",
      "FAU - Shirkavand, Atefeh\n",
      "AU  - Shirkavand A\n",
      "AD  - Kawsar's Human Genetic Research Center, Tehran, Iran.\n",
      "FAU - Zeinali, Sirous\n",
      "AU  - Zeinali S\n",
      "AD  - Kawsar's Human Genetic Research Center, Tehran, Iran.\n",
      "AD  - Department of Molecular Medicine, Biotechnology Research Center, PasteurInstitute of Iran, Tehran, Iran.\n",
      "FAU - Subasioglu, Asli\n",
      "AU  - Subasioglu A\n",
      "AD  - Department of Medical Genetics, Izmir Katip Celebi University, Ataturk Trainingand Research Hospital, Izmir, Turkey.\n",
      "FAU - Tokgoz-Yilmaz, Suna\n",
      "AU  - Tokgoz-Yilmaz S\n",
      "AUID- ORCID: http://orcid.org/0000-0002-4656-099X\n",
      "AD  - Department of Audiology, Hacettepe University Health Sciences Faculty, Ankara,Turkey.\n",
      "FAU - Huesca-Hernandez, Fabiola\n",
      "AU  - Huesca-Hernandez F\n",
      "AD  - Genetic Service, National Institute of Rehabilitation, Mexico D.F., Mexico.\n",
      "FAU - de la Luz Arenas-Sordo, Maria\n",
      "AU  - de la Luz Arenas-Sordo M\n",
      "AD  - Genetic Service, National Institute of Rehabilitation, Mexico D.F., Mexico.\n",
      "FAU - Dominguez-Aburto, Juan\n",
      "AU  - Dominguez-Aburto J\n",
      "AD  - Genetic Service, National Institute of Rehabilitation, Mexico D.F., Mexico.\n",
      "FAU - Hernandez-Zamora, Edgar\n",
      "AU  - Hernandez-Zamora E\n",
      "AD  - Genetic Service, National Institute of Rehabilitation, Mexico D.F., Mexico.\n",
      "FAU - Montenegro, Paola\n",
      "AU  - Montenegro P\n",
      "AD  - Molecular Genetic Lab, FFAA Hospital, Quito, Ecuador.\n",
      "FAU - Paredes, Rosario\n",
      "AU  - Paredes R\n",
      "AD  - Molecular Genetic Lab, FFAA Hospital, Quito, Ecuador.\n",
      "FAU - Moreta, Germania\n",
      "AU  - Moreta G\n",
      "AD  - Molecular Genetic Lab, FFAA Hospital, Quito, Ecuador.\n",
      "FAU - Vinueza, Rodrigo\n",
      "AU  - Vinueza R\n",
      "AUID- ORCID: http://orcid.org/0000-0001-5254-3072\n",
      "AD  - Molecular Genetic Lab, FFAA Hospital, Quito, Ecuador.\n",
      "FAU - Villegas, Franklin\n",
      "AU  - Villegas F\n",
      "AUID- ORCID: http://orcid.org/0000-0001-7188-5967\n",
      "AD  - Molecular Genetic Lab, FFAA Hospital, Quito, Ecuador.\n",
      "FAU - Mendoza-Benitez, Santiago\n",
      "AU  - Mendoza-Benitez S\n",
      "AD  - Audiology Department, Cuernavaca General Hospital, Morelos, Mexico.\n",
      "FAU - Guo, Shengru\n",
      "AU  - Guo S\n",
      "AD  - Dr. John T. Macdonald Foundation Department of Human Genetics and John P. HussmanInstitute for Human Genomics, University of Miami Miller School of Medicine,Miami, Florida, USA.\n",
      "FAU - Bozan, Nazim\n",
      "AU  - Bozan N\n",
      "AD  - Department of Otolaryngology, Faculty of Medicine, Yuzuncu Yil University, Van,Turkey.\n",
      "FAU - Tos, Tulay\n",
      "AU  - Tos T\n",
      "AD  - Department of Medical Genetics, Dr. Sami Ulus Research and Training Hospital ofWomen's and Children's Health and Diseases, Ankara, Turkey.\n",
      "FAU - Incesulu, Armagan\n",
      "AU  - Incesulu A\n",
      "AD  - Department of Otolaryngology, Head and Neck Surgery, Eskisehir OsmangaziUniversity, Eskisehir, Turkey.\n",
      "FAU - Sennaroglu, Gonca\n",
      "AU  - Sennaroglu G\n",
      "AD  - Department of Audiology, Hacettepe University Health Sciences Faculty, Ankara,Turkey.\n",
      "FAU - Blanton, Susan H\n",
      "AU  - Blanton SH\n",
      "AD  - Dr. John T. Macdonald Foundation Department of Human Genetics and John P. HussmanInstitute for Human Genomics, University of Miami Miller School of Medicine,Miami, Florida, USA.\n",
      "FAU - Ozturkmen-Akay, Hatice\n",
      "AU  - Ozturkmen-Akay H\n",
      "AD  - Department of Radiology, Istanbul Zeynep Kamil Maternity and Children Trainingand Research Hospital, Istanbul, Turkey.\n",
      "FAU - Yildirim-Baylan, Muzeyyen\n",
      "AU  - Yildirim-Baylan M\n",
      "AD  - Department of Otorhinolaryngology, Dicle University School of Medicine,Diyarbakir, Turkey.\n",
      "FAU - Tekin, Mustafa\n",
      "AU  - Tekin M\n",
      "AD  - Dr. John T. Macdonald Foundation Department of Human Genetics and John P. HussmanInstitute for Human Genomics, University of Miami Miller School of Medicine,Miami, Florida, USA.\n",
      "LA  - eng\n",
      "GR  - R01 DC009645/DC/NIDCD NIH HHS/United States\n",
      "GR  - R01DC009645/DC/NIDCD NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "DEP - 20150730\n",
      "PL  - United States\n",
      "TA  - Genet Med\n",
      "JT  - Genetics in medicine : official journal of the American College of MedicalGenetics\n",
      "JID - 9815831\n",
      "RN  - Deafness, Autosomal Recessive\n",
      "RN  - Nonsyndromic Deafness\n",
      "SB  - IM\n",
      "EIN - Genet Med. 2016 Aug;18(8):859. PMID: 27494218\n",
      "MH  - Alleles\n",
      "MH  - Cohort Studies\n",
      "MH  - Deafness/*diagnosis/*genetics\n",
      "MH  - Ethnicity/genetics\n",
      "MH  - *Exome\n",
      "MH  - *Genes, Recessive\n",
      "MH  - Genotype\n",
      "MH  - Hearing Loss, Sensorineural/*diagnosis/*genetics\n",
      "MH  - *High-Throughput Nucleotide Sequencing\n",
      "MH  - Humans\n",
      "MH  - Mutation\n",
      "PMC - PMC4733433\n",
      "MID - NIHMS693478\n",
      "EDAT- 2015/08/01 06:00\n",
      "MHDA- 2016/12/20 06:00\n",
      "CRDT- 2015/07/31 06:00\n",
      "PHST- 2015/03/27 00:00 [received]\n",
      "PHST- 2015/05/19 00:00 [accepted]\n",
      "PHST- 2015/07/31 06:00 [entrez]\n",
      "PHST- 2015/08/01 06:00 [pubmed]\n",
      "PHST- 2016/12/20 06:00 [medline]\n",
      "AID - S1098-3600(21)04336-7 [pii]\n",
      "AID - 10.1038/gim.2015.89 [doi]\n",
      "PST - ppublish\n",
      "SO  - Genet Med. 2016 Apr;18(4):364-71. doi: 10.1038/gim.2015.89. Epub 2015 Jul 30.\n",
      "\n",
      "PMID- 26217358\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "DCOM- 20150728\n",
      "LR  - 20200930\n",
      "IS  - 1664-462X (Print)\n",
      "IS  - 1664-462X (Linking)\n",
      "VI  - 6\n",
      "DP  - 2015\n",
      "TI  - The role of proteomics in progressing insights into plant secondary metabolism.\n",
      "PG  - 504\n",
      "LID - 10.3389/fpls.2015.00504 [doi]\n",
      "AB  - The development of omics has enabled the genome-wide exploration of all kinds of biological processes at the molecular level. Almost every field of plant biology has been analyzed at the genomic, transcriptomic and proteomic level. Here wefocus on the particular contribution that proteomic technologies have made inprogressing knowledge and characterising plant secondary metabolism (SM) pathwayssince early expectations were created 15 years ago. We analyzed how three majorissues in the proteomic analysis of plant SM have been implemented in variousresearch studies. These issues are: (i) the selection of a suitable plantmaterial rich in secondary metabolites of interest, such as specialized tissuesand organs, and in vitro cell cultures; (ii) the proteomic strategy to accesstarget proteins, either a comprehensive or a differential analysis; (iii) theproteomic approach, represented by the hypothesis-free discovery proteomics andthe hypothesis-driven targeted proteomics. We also examine to what extent themost-advanced technologies have been incorporated into proteomic research inplant SM and highlight some cutting edge techniques that would strongly benefitthe progress made in this field.\n",
      "FAU - Martinez-Esteso, Maria J\n",
      "AU  - Martinez-Esteso MJ\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Multidisciplinary Institute for Environmental Studies \"RamonMargalef\", University of Alicante , Alicante, Spain.\n",
      "FAU - Martinez-Marquez, Ascension\n",
      "AU  - Martinez-Marquez A\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Multidisciplinary Institute for Environmental Studies \"RamonMargalef\", University of Alicante , Alicante, Spain.\n",
      "FAU - Selles-Marchart, Susana\n",
      "AU  - Selles-Marchart S\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Multidisciplinary Institute for Environmental Studies \"RamonMargalef\", University of Alicante , Alicante, Spain ; Biotechnology and MolecularBiology Group, Quevedo State Technical University , Quevedo, Ecuador.\n",
      "FAU - Morante-Carriel, Jaime A\n",
      "AU  - Morante-Carriel JA\n",
      "AD  - Proteomics and Genomics Division, Research Technical Facility, University ofAlicante , Alicante, Spain.\n",
      "FAU - Bru-Martinez, Roque\n",
      "AU  - Bru-Martinez R\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Multidisciplinary Institute for Environmental Studies \"RamonMargalef\", University of Alicante , Alicante, Spain.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "DEP - 20150707\n",
      "PL  - Switzerland\n",
      "TA  - Front Plant Sci\n",
      "JT  - Frontiers in plant science\n",
      "JID - 101568200\n",
      "PMC - PMC4493368\n",
      "OTO - NOTNLM\n",
      "OT  - DIGE\n",
      "OT  - MRM\n",
      "OT  - alkaloids\n",
      "OT  - cell cultures\n",
      "OT  - polyphenols\n",
      "OT  - secondary metabolism\n",
      "OT  - shotgun proteomics\n",
      "OT  - terpenoids\n",
      "EDAT- 2015/07/29 06:00\n",
      "MHDA- 2015/07/29 06:01\n",
      "CRDT- 2015/07/29 06:00\n",
      "PHST- 2015/02/28 00:00 [received]\n",
      "PHST- 2015/06/23 00:00 [accepted]\n",
      "PHST- 2015/07/29 06:00 [entrez]\n",
      "PHST- 2015/07/29 06:00 [pubmed]\n",
      "PHST- 2015/07/29 06:01 [medline]\n",
      "AID - 10.3389/fpls.2015.00504 [doi]\n",
      "PST - epublish\n",
      "SO  - Front Plant Sci. 2015 Jul 7;6:504. doi: 10.3389/fpls.2015.00504. eCollection2015.\n",
      "\n",
      "PMID- 26048971\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20160308\n",
      "LR  - 20190110\n",
      "IS  - 1537-6613 (Electronic)\n",
      "IS  - 0022-1899 (Linking)\n",
      "VI  - 212\n",
      "IP  - 12\n",
      "DP  - 2015 Dec 15\n",
      "TI  - Parallel Epidemics of Community-Associated Methicillin-Resistant Staphylococcusaureus USA300 Infection in North and South America.\n",
      "PG  - 1874-82\n",
      "LID - 10.1093/infdis/jiv320 [doi]\n",
      "AB  - BACKGROUND: The community-associated methicillin-resistant Staphylococcus aureus (CA-MRSA) epidemic in the United States is attributed to the spread of the USA300clone. An epidemic of CA-MRSA closely related to USA300 has occurred in northern South America (USA300 Latin-American variant, USA300-LV). Using phylogenomicanalysis, we aimed to understand the relationships between these 2 epidemics.METHODS: We sequenced the genomes of 51 MRSA clinical isolates collected between 1999 and 2012 from the United States, Colombia, Venezuela, and Ecuador.Phylogenetic analysis was used to infer the relationships and times since thedivergence of the major clades. RESULTS: Phylogenetic analyses revealed 2dominant clades that segregated by geographical region, had a putative commonancestor in 1975, and originated in 1989, in North America, and in 1985, in SouthAmerica. Emergence of these parallel epidemics coincides with the independentacquisition of the arginine catabolic mobile element (ACME) in North Americanisolates and a novel copper and mercury resistance (COMER) mobile element inSouth American isolates. CONCLUSIONS: Our results reveal the existence of 2parallel USA300 epidemics that shared a recent common ancestor. The simultaneous rapid dissemination of these 2 epidemic clades suggests the presence of shared,potentially convergent adaptations that enhance fitness and ability to spread.\n",
      "CI  - (c) The Author 2015. Published by Oxford University Press on behalf of theInfectious Diseases Society of America. All rights reserved. For Permissions,please e-mail: journals.permissions@oup.com.\n",
      "FAU - Planet, Paul J\n",
      "AU  - Planet PJ\n",
      "AD  - Division of Pediatric Infectious Diseases, Department of Pediatrics, ColumbiaUniversity, College of Physicians and Surgeons Sackler Institute for Comparative Genomics, American Museum of Natural History.\n",
      "FAU - Diaz, Lorena\n",
      "AU  - Diaz L\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Kolokotronis, Sergios-Orestis\n",
      "AU  - Kolokotronis SO\n",
      "AD  - Sackler Institute for Comparative Genomics, American Museum of Natural HistoryDepartment of Biological Sciences, Fordham University, Bronx, New York.\n",
      "FAU - Narechania, Apurva\n",
      "AU  - Narechania A\n",
      "AD  - Sackler Institute for Comparative Genomics, American Museum of Natural History.\n",
      "FAU - Reyes, Jinnethe\n",
      "AU  - Reyes J\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Xing, Galen\n",
      "AU  - Xing G\n",
      "AD  - Division of Pediatric Infectious Diseases, Department of Pediatrics, ColumbiaUniversity, College of Physicians and Surgeons.\n",
      "FAU - Rincon, Sandra\n",
      "AU  - Rincon S\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Smith, Hannah\n",
      "AU  - Smith H\n",
      "AD  - Division of Pediatric Infectious Diseases, Department of Pediatrics, ColumbiaUniversity, College of Physicians and Surgeons.\n",
      "FAU - Panesso, Diana\n",
      "AU  - Panesso D\n",
      "AD  - Division of Infectious Diseases, Department of Internal Medicine MolecularGenetics and Antimicrobial Resistance Unit, International Center for MicrobialGenomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Ryan, Chanelle\n",
      "AU  - Ryan C\n",
      "AD  - Division of Pediatric Infectious Diseases, Department of Pediatrics, ColumbiaUniversity, College of Physicians and Surgeons.\n",
      "FAU - Smith, Dylan P\n",
      "AU  - Smith DP\n",
      "AD  - Molecular Genetics and Antimicrobial Resistance Unit, International Center forMicrobial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "FAU - Guzman, Manuel\n",
      "AU  - Guzman M\n",
      "AD  - Centro Medico Caracas, Venezuela.\n",
      "FAU - Zurita, Jeannete\n",
      "AU  - Zurita J\n",
      "AD  - Hospital Vozandes, Pontificia Universidad Catolica, Quito, Ecuador.\n",
      "FAU - Sebra, Robert\n",
      "AU  - Sebra R\n",
      "AD  - Genome Center, Mount Sinai Hospital, New York City.\n",
      "FAU - Deikus, Gintaras\n",
      "AU  - Deikus G\n",
      "AD  - Genome Center, Mount Sinai Hospital, New York City.\n",
      "FAU - Nolan, Rathel L\n",
      "AU  - Nolan RL\n",
      "AD  - Division of Infectious Diseases, Department of Internal Medicine.\n",
      "FAU - Tenover, Fred C\n",
      "AU  - Tenover FC\n",
      "AD  - Cepheid, Sunnyvale, California.\n",
      "FAU - Weinstock, George M\n",
      "AU  - Weinstock GM\n",
      "AD  - The Jackson Laboratory for Genomic Medicine, Farmington, Connecticut.\n",
      "FAU - Robinson, D Ashley\n",
      "AU  - Robinson DA\n",
      "AD  - Division of Infectious Diseases, Department of Microbiology, University ofMississippi Medical Center, Jackson.\n",
      "FAU - Arias, Cesar A\n",
      "AU  - Arias CA\n",
      "AD  - Division of Infectious Diseases, Department of Internal Medicine Department ofMicrobiology and Molecular Genetics, University of Texas Medical School atHouston Molecular Genetics and Antimicrobial Resistance Unit, InternationalCenter for Microbial Genomics, Universidad El Bosque, Bogota, Colombia.\n",
      "LA  - eng\n",
      "GR  - K08 AI101005/AI/NIAID NIH HHS/United States\n",
      "GR  - R01 AI093749/AI/NIAID NIH HHS/United States\n",
      "GR  - R01 GM080602/GM/NIGMS NIH HHS/United States\n",
      "GR  - K08AI101005/AI/NIAID NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20150605\n",
      "PL  - United States\n",
      "TA  - J Infect Dis\n",
      "JT  - The Journal of infectious diseases\n",
      "JID - 0413675\n",
      "SB  - IM\n",
      "MH  - Community-Acquired Infections/*epidemiology/*microbiology\n",
      "MH  - *Epidemics\n",
      "MH  - Epidemiological Monitoring\n",
      "MH  - Genome, Bacterial\n",
      "MH  - Genotype\n",
      "MH  - Humans\n",
      "MH  - Methicillin-Resistant Staphylococcus aureus/*isolation & purification\n",
      "MH  - Molecular Epidemiology\n",
      "MH  - Molecular Typing\n",
      "MH  - North America/epidemiology\n",
      "MH  - Phylogeography\n",
      "MH  - Sequence Analysis, DNA\n",
      "MH  - South America/epidemiology\n",
      "MH  - Staphylococcal Infections/*epidemiology/*microbiology\n",
      "PMC - PMC4655856\n",
      "OTO - NOTNLM\n",
      "OT  - MRSA\n",
      "OT  - USA300\n",
      "OT  - USA300-LV\n",
      "OT  - epidemics\n",
      "EDAT- 2015/06/07 06:00\n",
      "MHDA- 2016/03/10 06:00\n",
      "CRDT- 2015/06/07 06:00\n",
      "PHST- 2014/12/16 00:00 [received]\n",
      "PHST- 2015/05/13 00:00 [accepted]\n",
      "PHST- 2015/06/07 06:00 [entrez]\n",
      "PHST- 2015/06/07 06:00 [pubmed]\n",
      "PHST- 2016/03/10 06:00 [medline]\n",
      "AID - jiv320 [pii]\n",
      "AID - 10.1093/infdis/jiv320 [doi]\n",
      "PST - ppublish\n",
      "SO  - J Infect Dis. 2015 Dec 15;212(12):1874-82. doi: 10.1093/infdis/jiv320. Epub 2015 Jun 5.\n",
      "\n",
      "PMID- 25977422\n",
      "OWN - NLM\n",
      "STAT- PubMed-not-MEDLINE\n",
      "DCOM- 20150515\n",
      "LR  - 20200930\n",
      "IS  - 2169-8287 (Print)\n",
      "VI  - 3\n",
      "IP  - 3\n",
      "DP  - 2015 May 14\n",
      "TI  - Draft Genome Sequence of Phenylobacterium immobile Strain E (DSM 1986), Isolated from Uncontaminated Soil in Ecuador.\n",
      "LID - 10.1128/genomeA.00420-15 [doi]\n",
      "LID - e00420-15 [pii]\n",
      "AB  - We report the draft genome sequence of 3.3 Mb and the sequence (19.2 kb) of anatural plasmid isolated from Phenylobacterium immobile strain E (DSM 1986), ableto degrade xenobiotic compounds as the sole carbon source. The sequences reveal alarge number of novel Rieske nonheme iron aromatic ring-hydroxylating oxygenases (RHOs).\n",
      "CI  - Copyright (c) 2015 Reznicek et al.\n",
      "FAU - Reznicek, Ondrej\n",
      "AU  - Reznicek O\n",
      "AD  - Institute of Technical Biochemistry, University of Stuttgart, Stuttgart, Germany.\n",
      "FAU - Luesken, Francisca\n",
      "AU  - Luesken F\n",
      "AD  - Research Unit Environmental Genomics, Helmholtz Centre Munich, Munich, Germany.\n",
      "FAU - Facey, Sandra J\n",
      "AU  - Facey SJ\n",
      "AD  - Institute of Technical Biochemistry, University of Stuttgart, Stuttgart, Germany.\n",
      "FAU - Hauer, Bernhard\n",
      "AU  - Hauer B\n",
      "AD  - Institute of Technical Biochemistry, University of Stuttgart, Stuttgart, Germany bernhard.hauer@itb.uni-stuttgart.de.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20150514\n",
      "PL  - United States\n",
      "TA  - Genome Announc\n",
      "JT  - Genome announcements\n",
      "JID - 101595808\n",
      "PMC - PMC4432328\n",
      "EDAT- 2015/05/16 06:00\n",
      "MHDA- 2015/05/16 06:01\n",
      "CRDT- 2015/05/16 06:00\n",
      "PHST- 2015/05/16 06:00 [entrez]\n",
      "PHST- 2015/05/16 06:00 [pubmed]\n",
      "PHST- 2015/05/16 06:01 [medline]\n",
      "AID - 3/3/e00420-15 [pii]\n",
      "AID - 10.1128/genomeA.00420-15 [doi]\n",
      "PST - epublish\n",
      "SO  - Genome Announc. 2015 May 14;3(3). pii: 3/3/e00420-15. doi:10.1128/genomeA.00420-15.\n",
      "\n",
      "PMID- 25928534\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20150630\n",
      "LR  - 20200411\n",
      "IS  - 1471-2350 (Electronic)\n",
      "IS  - 1471-2350 (Linking)\n",
      "VI  - 16\n",
      "DP  - 2015 Feb 25\n",
      "TI  - Novel domain-specific POU3F4 mutations are associated with X-linked deafness:examples from different populations.\n",
      "PG  - 9\n",
      "LID - 10.1186/s12881-015-0149-2 [doi]\n",
      "AB  - BACKGROUND: Mutations in the POU3F4 gene cause X-linked deafness type 3 (DFN3),which is characterized by inner ear anomalies. METHODS: Three Turkish, oneEcuadorian, and one Nigerian families were included based on either inner earanomalies detected in probands or X-linked family histories. Exome sequencingand/or Sanger sequencing were performed in order to identify the causative DNAvariants in these families. RESULTS: Four novel, c.707A>C (p.(Glu236Ala)),c.772delG (p.(Glu258ArgfsX30)), c.902C>T (p.(Pro301Leu)), c.987T>C(p.(Ile308Thr)), and one previously reported mutation c.346delG(p.(Ala116ProfsX26)) in POU3F4, were identified. All mutations identified arepredicted to affect the POU-specific or POU homeo domains of the protein andco-segregated with deafness in all families. CONCLUSIONS: Expanding the spectrum of POU3F4 mutations in different populations along with their associatedphenotypes provides better understanding of their clinical importance and will behelpful in clinical evaluation and counseling of the affected individuals.\n",
      "FAU - Bademci, Guney\n",
      "AU  - Bademci G\n",
      "AD  - John P. Hussmann Institute for Human Genomics and John T. Macdonald Foundation,Department of Human Genetics, Miller school of Medicine, University of Miami,1501 NW 10th Avenue, BRB-610 (M-860), Miami, FL, 33136, USA.g.bademci@med.miami.edu.\n",
      "FAU - Lasisi, Akeem\n",
      "AU  - Lasisi A\n",
      "AD  - John P. Hussmann Institute for Human Genomics and John T. Macdonald Foundation,Department of Human Genetics, Miller school of Medicine, University of Miami,1501 NW 10th Avenue, BRB-610 (M-860), Miami, FL, 33136, USA.akeemlasisi@gmail.com.\n",
      "AD  - Department of Otorhinolaryngology, College of Medicine, University of Ibadan,Ibadan, Nigeria. akeemlasisi@gmail.com.\n",
      "FAU - Yariz, Kemal O\n",
      "AU  - Yariz KO\n",
      "AD  - John P. Hussmann Institute for Human Genomics and John T. Macdonald Foundation,Department of Human Genetics, Miller school of Medicine, University of Miami,1501 NW 10th Avenue, BRB-610 (M-860), Miami, FL, 33136, USA. koyariz@gmail.com.\n",
      "FAU - Montenegro, Paola\n",
      "AU  - Montenegro P\n",
      "AD  - Departamento de Genetica, Hospital de Especialidades FFAA, Quito, Ecuador.paom14@gmail.com.\n",
      "FAU - Menendez, Ibis\n",
      "AU  - Menendez I\n",
      "AD  - John P. Hussmann Institute for Human Genomics and John T. Macdonald Foundation,Department of Human Genetics, Miller school of Medicine, University of Miami,1501 NW 10th Avenue, BRB-610 (M-860), Miami, FL, 33136, USA.IMenendez1@med.miami.edu.\n",
      "FAU - Vinueza, Rodrigo\n",
      "AU  - Vinueza R\n",
      "AD  - Departamento de Genetica, Hospital de Especialidades FFAA, Quito, Ecuador.genmolhg1@gmail.com.\n",
      "FAU - Paredes, Rosario\n",
      "AU  - Paredes R\n",
      "AD  - Departamento de Genetica, Hospital de Especialidades FFAA, Quito, Ecuador.charoparedes@yahoo.fr.\n",
      "FAU - Moreta, Germania\n",
      "AU  - Moreta G\n",
      "AD  - Departamento de Genetica, Hospital de Especialidades FFAA, Quito, Ecuador.germaniamoreta@yahoo.com.\n",
      "FAU - Subasioglu, Asli\n",
      "AU  - Subasioglu A\n",
      "AD  - John P. Hussmann Institute for Human Genomics and John T. Macdonald Foundation,Department of Human Genetics, Miller school of Medicine, University of Miami,1501 NW 10th Avenue, BRB-610 (M-860), Miami, FL, 33136, USA.asuzak78@hotmail.com.\n",
      "AD  - Department of Medical Genetics, Izmir Katip Celebi University, Ataturk Trainingand Research Hospital, Izmir, Turkey. asuzak78@hotmail.com.\n",
      "FAU - Blanton, Susan\n",
      "AU  - Blanton S\n",
      "AD  - John P. Hussmann Institute for Human Genomics and John T. Macdonald Foundation,Department of Human Genetics, Miller school of Medicine, University of Miami,1501 NW 10th Avenue, BRB-610 (M-860), Miami, FL, 33136, USA.SBlanton@med.miami.edu.\n",
      "FAU - Fitoz, Suat\n",
      "AU  - Fitoz S\n",
      "AD  - Department of Radiodiagnostics, Ankara University School of Medicine, Ankara,Turkey. sfitoz@yahoo.com.\n",
      "FAU - Incesulu, Armagan\n",
      "AU  - Incesulu A\n",
      "AD  - Department of Otorhinolaryngology, Eskisehir Osmangazi University School ofMedicine, Eskisehir, Turkey. armaganincesulu@yahoo.com.\n",
      "FAU - Sennaroglu, Levent\n",
      "AU  - Sennaroglu L\n",
      "AD  - Department of Otorhinolaryngology, Hacettepe University School of Medicine,Ankara, Turkey. lsennaroglu@gmail.com.\n",
      "FAU - Tekin, Mustafa\n",
      "AU  - Tekin M\n",
      "AD  - John P. Hussmann Institute for Human Genomics and John T. Macdonald Foundation,Department of Human Genetics, Miller school of Medicine, University of Miami,1501 NW 10th Avenue, BRB-610 (M-860), Miami, FL, 33136, USA.mtekin@med.miami.edu.\n",
      "LA  - eng\n",
      "GR  - R01 DC009645/DC/NIDCD NIH HHS/United States\n",
      "GR  - R01 DC012836/DC/NIDCD NIH HHS/United States\n",
      "GR  - R01DC012836/DC/NIDCD NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20150225\n",
      "PL  - England\n",
      "TA  - BMC Med Genet\n",
      "JT  - BMC medical genetics\n",
      "JID - 100968552\n",
      "RN  - 0 (POU Domain Factors)\n",
      "RN  - 0 (POU3F4 protein, human)\n",
      "SB  - IM\n",
      "MH  - Child, Preschool\n",
      "MH  - Cohort Studies\n",
      "MH  - Deafness/ethnology/*genetics\n",
      "MH  - Female\n",
      "MH  - Genetic Diseases, X-Linked/ethnology/*genetics\n",
      "MH  - Humans\n",
      "MH  - Male\n",
      "MH  - Models, Molecular\n",
      "MH  - POU Domain Factors/*chemistry/*genetics\n",
      "MH  - Pedigree\n",
      "MH  - Protein Structure, Tertiary\n",
      "PMC - PMC4422282\n",
      "EDAT- 2015/05/01 06:00\n",
      "MHDA- 2015/07/01 06:00\n",
      "CRDT- 2015/05/01 06:00\n",
      "PHST- 2014/09/12 00:00 [received]\n",
      "PHST- 2015/01/29 00:00 [accepted]\n",
      "PHST- 2015/05/01 06:00 [entrez]\n",
      "PHST- 2015/05/01 06:00 [pubmed]\n",
      "PHST- 2015/07/01 06:00 [medline]\n",
      "AID - 10.1186/s12881-015-0149-2 [doi]\n",
      "AID - 10.1186/s12881-015-0149-2 [pii]\n",
      "PST - epublish\n",
      "SO  - BMC Med Genet. 2015 Feb 25;16:9. doi: 10.1186/s12881-015-0149-2.\n",
      "\n",
      "PMID- 25880392\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20150825\n",
      "LR  - 20181113\n",
      "IS  - 1471-2164 (Electronic)\n",
      "IS  - 1471-2164 (Linking)\n",
      "VI  - 16\n",
      "DP  - 2015 Apr 1\n",
      "TI  - Genomic variation in tomato, from wild ancestors to contemporary breedingaccessions.\n",
      "PG  - 257\n",
      "LID - 10.1186/s12864-015-1444-1 [doi]\n",
      "AB  - BACKGROUND: Domestication modifies the genomic variation of species. Quantifying this variation provides insights into the domestication process, facilitates the management of resources used by breeders and germplasm centers, and enables thedesign of experiments to associate traits with genes. We described and analyzedthe genetic diversity of 1,008 tomato accessions including Solanum lycopersicumvar. lycopersicum (SLL), S. lycopersicum var. cerasiforme (SLC), and S.pimpinellifolium (SP) that were genotyped using 7,720 SNPs. Additionally, weexplored the allelic frequency of six loci affecting fruit weight and shape toinfer patterns of selection. RESULTS: Our results revealed a pattern of variationthat strongly supported a two-step domestication process, occasionalhybridization in the wild, and differentiation through human selection. Theseinterpretations were consistent with the observed allele frequencies for the six loci affecting fruit weight and shape. Fruit weight was strongly selected in SLC in the Andean region of Ecuador and Northern Peru prior to the domestication oftomato in Mesoamerica. Alleles affecting fruit shape were differentially selectedamong SLL genetic subgroups. Our results also clarified the biological status of SLC. True SLC was phylogenetically positioned between SP and SLL and its fruitmorphology was diverse. SLC and \"cherry tomato\" are not synonymous terms. Themorphologically-based term \"cherry tomato\" included some SLC, contemporaryvarieties, as well as many admixtures between SP and SLL. Contemporary SLL showeda moderate increase in nucleotide diversity, when compared with vintage groups.CONCLUSIONS: This study presents a broad and detailed representation of thegenomic variation in tomato. Tomato domestication seems to have followed a twostep-process; a first domestication in South America and a second step inMesoamerica. The distribution of fruit weight and shape alleles supports thatdomestication of SLC occurred in the Andean region. Our results also clarify the biological status of SLC as true phylogenetic group within tomato. We detectEcuadorian and Peruvian accessions that may represent a pool of unexploredvariation that could be of interest for crop improvement.\n",
      "FAU - Blanca, Jose\n",
      "AU  - Blanca J\n",
      "AD  - Institute for the Conservation and Improvement of Agricultural Biodiversity(COMAV), Polytechnic University of Valencia, Camino de Vera 8E, 46022, Valencia, Spain. jblanca@upv.es.\n",
      "FAU - Montero-Pau, Javier\n",
      "AU  - Montero-Pau J\n",
      "AD  - Institute for the Conservation and Improvement of Agricultural Biodiversity(COMAV), Polytechnic University of Valencia, Camino de Vera 8E, 46022, Valencia, Spain. jamonpa@upv.es.\n",
      "FAU - Sauvage, Christopher\n",
      "AU  - Sauvage C\n",
      "AD  - INRA, UR 1052 Unite de Genetique et Amelioration des Fruits et Legumes, DomaineSaint-Maurice, 67 Allee des Chenes CS60094, 84143, Montfavet Cedex, France.christopher.sauvage@avignon.inra.fr.\n",
      "FAU - Bauchet, Guillaume\n",
      "AU  - Bauchet G\n",
      "AD  - INRA, UR 1052 Unite de Genetique et Amelioration des Fruits et Legumes, DomaineSaint-Maurice, 67 Allee des Chenes CS60094, 84143, Montfavet Cedex, France.Guillaume.Bauchet@avignon.inra.fr.\n",
      "AD  - Syngenta seeds, 12 chemin de l'hobit, 31790, Saint-Sauveur, France.Guillaume.Bauchet@avignon.inra.fr.\n",
      "FAU - Illa, Eudald\n",
      "AU  - Illa E\n",
      "AD  - Department of Horticulture and Crop Science, The Ohio State University/OhioAgricultural Research and Development Center, Wooster, OH, 44691, USA.illaberenguer.1@osu.edu.\n",
      "FAU - Diez, Maria Jose\n",
      "AU  - Diez MJ\n",
      "AD  - Institute for the Conservation and Improvement of Agricultural Biodiversity(COMAV), Polytechnic University of Valencia, Camino de Vera 8E, 46022, Valencia, Spain. mdiezni@btc.upv.es.\n",
      "FAU - Francis, David\n",
      "AU  - Francis D\n",
      "AD  - Department of Horticulture and Crop Science, The Ohio State University/OhioAgricultural Research and Development Center, Wooster, OH, 44691, USA.francis.77@osu.edu.\n",
      "FAU - Causse, Mathilde\n",
      "AU  - Causse M\n",
      "AD  - INRA, UR 1052 Unite de Genetique et Amelioration des Fruits et Legumes, DomaineSaint-Maurice, 67 Allee des Chenes CS60094, 84143, Montfavet Cedex, France.mcausse@avignon.inra.fr.\n",
      "FAU - van der Knaap, Esther\n",
      "AU  - van der Knaap E\n",
      "AD  - Department of Horticulture and Crop Science, The Ohio State University/OhioAgricultural Research and Development Center, Wooster, OH, 44691, USA.vanderknaap.1@osu.edu.\n",
      "FAU - Canizares, Joaquin\n",
      "AU  - Canizares J\n",
      "AD  - Institute for the Conservation and Improvement of Agricultural Biodiversity(COMAV), Polytechnic University of Valencia, Camino de Vera 8E, 46022, Valencia, Spain. jcanizares@upv.es.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20150401\n",
      "PL  - England\n",
      "TA  - BMC Genomics\n",
      "JT  - BMC genomics\n",
      "JID - 100965258\n",
      "SB  - IM\n",
      "MH  - Breeding\n",
      "MH  - Evolution, Molecular\n",
      "MH  - Fruit/genetics\n",
      "MH  - Gene Frequency\n",
      "MH  - Genome, Plant\n",
      "MH  - Genomics\n",
      "MH  - Heterozygote\n",
      "MH  - Lycopersicon esculentum/*genetics\n",
      "MH  - *Polymorphism, Single Nucleotide\n",
      "PMC - PMC4404671\n",
      "EDAT- 2015/04/17 06:00\n",
      "MHDA- 2015/08/26 06:00\n",
      "CRDT- 2015/04/17 06:00\n",
      "PHST- 2014/10/08 00:00 [received]\n",
      "PHST- 2015/03/06 00:00 [accepted]\n",
      "PHST- 2015/04/17 06:00 [entrez]\n",
      "PHST- 2015/04/17 06:00 [pubmed]\n",
      "PHST- 2015/08/26 06:00 [medline]\n",
      "AID - 10.1186/s12864-015-1444-1 [doi]\n",
      "AID - 10.1186/s12864-015-1444-1 [pii]\n",
      "PST - epublish\n",
      "SO  - BMC Genomics. 2015 Apr 1;16:257. doi: 10.1186/s12864-015-1444-1.\n",
      "\n",
      "PMID- 25673391\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20150227\n",
      "LR  - 20181113\n",
      "IS  - 1476-4687 (Electronic)\n",
      "IS  - 0028-0836 (Linking)\n",
      "VI  - 518\n",
      "IP  - 7538\n",
      "DP  - 2015 Feb 12\n",
      "TI  - Darwin's finches join genome club.\n",
      "PG  - 147\n",
      "LID - 10.1038/518147a [doi]\n",
      "FAU - Marsh, Geoff\n",
      "AU  - Marsh G\n",
      "LA  - eng\n",
      "PT  - News\n",
      "PL  - England\n",
      "TA  - Nature\n",
      "JT  - Nature\n",
      "JID - 0410462\n",
      "RN  - 0 (Avian Proteins)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Avian Proteins/genetics\n",
      "MH  - Beak/*anatomy & histology\n",
      "MH  - *Biological Evolution\n",
      "MH  - Costa Rica\n",
      "MH  - Diet/veterinary\n",
      "MH  - Ecuador\n",
      "MH  - Finches/*anatomy & histology/classification/*genetics\n",
      "MH  - Genetic Variation/genetics\n",
      "MH  - Genome/*genetics\n",
      "MH  - Genomics\n",
      "MH  - Phylogeny\n",
      "MH  - Selection, Genetic\n",
      "EDAT- 2015/02/13 06:00\n",
      "MHDA- 2015/02/28 06:00\n",
      "CRDT- 2015/02/13 06:00\n",
      "PHST- 2015/02/13 06:00 [entrez]\n",
      "PHST- 2015/02/13 06:00 [pubmed]\n",
      "PHST- 2015/02/28 06:00 [medline]\n",
      "AID - 518147a [pii]\n",
      "AID - 10.1038/518147a [doi]\n",
      "PST - ppublish\n",
      "SO  - Nature. 2015 Feb 12;518(7538):147. doi: 10.1038/518147a.\n",
      "\n",
      "PMID- 25524569\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20151014\n",
      "LR  - 20181113\n",
      "IS  - 1432-184X (Electronic)\n",
      "IS  - 0095-3628 (Linking)\n",
      "VI  - 69\n",
      "IP  - 2\n",
      "DP  - 2015 Feb\n",
      "TI  - Metagenomic-based study of the phylogenetic and functional gene diversity inGalapagos land and marine iguanas.\n",
      "PG  - 444-56\n",
      "LID - 10.1007/s00248-014-0547-6 [doi]\n",
      "AB  - In this study, a metagenome-based analysis of the fecal samples from themacrophytic algae-consuming marine iguana (MI; Amblyrhynchus cristatus) andterrestrial biomass-consuming land iguanas (LI; Conolophus spp.) was conducted.Phylogenetic affiliations of the fecal microbiome were more similar between both iguanas than to other mammalian herbivorous hosts. However, functional genediversities in both MI and LI iguana hosts differed in relation to the diet,where the MI fecal microbiota had a functional diversity that clustered apartfrom the other terrestrial-biomass consuming reptilian and mammalian hosts. Afurther examination of the carbohydrate-degrading genes revealed that several of the prevalent glycosyl hydrolases (GH), glycosyl transferases (GT), carbohydrate binding modules (CBM), and carbohydrate esterases (CE) gene classes wereconserved among all examined herbivorous hosts, reiterating the important rolesthese genes play in the breakdown and metabolism of herbivorous diets. Genesencoding some classes of carbohydrate-degrading families, including GH2, GH13,GT2, GT4, CBM50, CBM48, CE4, and CE11, as well as genes associated with sulfurmetabolism and dehalogenation, were highly enriched or unique to the MI. Incontrast, gene sequences that relate to archaeal methanogenesis were detectedonly in LI fecal microbiome, and genes coding for GH13, GH66, GT2, GT4, CBM50,CBM13, CE4, and CE8 carbohydrate active enzymes were highly abundant in the LI.Bacterial populations were enriched on various carbohydrates substrates (e.g.,glucose, arabinose, xylose). The majority of the enriched bacterial populationsbelong to genera Clostridium spp. and Enterococcus spp. that likely accounted forthe high prevalence of GH13 and GH2, as well as the GT families (e.g., GT2, GT4, GT28, GT35, and GT51) that were ubiquitously present in the fecal microbiota ofall herbivorous hosts.\n",
      "FAU - Hong, Pei-Ying\n",
      "AU  - Hong PY\n",
      "AD  - Department of Animal Sciences, University of Illinois at Urbana Champaign,Urbana, IL, 61801, USA.\n",
      "FAU - Mao, Yuejian\n",
      "AU  - Mao Y\n",
      "FAU - Ortiz-Kofoed, Shannon\n",
      "AU  - Ortiz-Kofoed S\n",
      "FAU - Shah, Rushabh\n",
      "AU  - Shah R\n",
      "FAU - Cann, Isaac\n",
      "AU  - Cann I\n",
      "FAU - Mackie, Roderick I\n",
      "AU  - Mackie RI\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20141219\n",
      "PL  - United States\n",
      "TA  - Microb Ecol\n",
      "JT  - Microbial ecology\n",
      "JID - 7500663\n",
      "RN  - 0 (DNA, Archaeal)\n",
      "RN  - 0 (DNA, Bacterial)\n",
      "RN  - 0 (Fatty Acids, Volatile)\n",
      "RN  - 0 (RNA, Ribosomal, 16S)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Archaea/*classification/genetics/isolation & purification\n",
      "MH  - Bacteria/*classification/genetics/isolation & purification\n",
      "MH  - DNA, Archaeal/genetics\n",
      "MH  - DNA, Bacterial/genetics\n",
      "MH  - Ecuador\n",
      "MH  - Fatty Acids, Volatile/analysis\n",
      "MH  - Feces/microbiology\n",
      "MH  - Host-Pathogen Interactions\n",
      "MH  - Iguanas/*microbiology\n",
      "MH  - Islands\n",
      "MH  - *Metagenome\n",
      "MH  - Metagenomics\n",
      "MH  - Microbiota\n",
      "MH  - Multigene Family\n",
      "MH  - *Phylogeny\n",
      "MH  - RNA, Ribosomal, 16S/genetics\n",
      "MH  - Sequence Analysis, DNA\n",
      "EDAT- 2014/12/20 06:00\n",
      "MHDA- 2015/10/16 06:00\n",
      "CRDT- 2014/12/20 06:00\n",
      "PHST- 2014/07/23 00:00 [received]\n",
      "PHST- 2014/11/27 00:00 [accepted]\n",
      "PHST- 2014/12/20 06:00 [entrez]\n",
      "PHST- 2014/12/20 06:00 [pubmed]\n",
      "PHST- 2015/10/16 06:00 [medline]\n",
      "AID - 10.1007/s00248-014-0547-6 [doi]\n",
      "PST - ppublish\n",
      "SO  - Microb Ecol. 2015 Feb;69(2):444-56. doi: 10.1007/s00248-014-0547-6. Epub 2014 Dec19.\n",
      "\n",
      "PMID- 25435058\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20151029\n",
      "LR  - 20211203\n",
      "IS  - 1618-1301 (Electronic)\n",
      "IS  - 0018-442X (Linking)\n",
      "VI  - 66\n",
      "IP  - 1\n",
      "DP  - 2015 Feb\n",
      "TI  - Admixture and genetic relationships of Mexican Mestizos regarding Latin American and Caribbean populations based on 13 CODIS-STRs.\n",
      "PG  - 44-59\n",
      "LID - 10.1016/j.jchb.2014.08.005 [doi]\n",
      "LID - S0018-442X(14)00103-6 [pii]\n",
      "AB  - Short tandem repeats (STRs) of the combined DNA index system (CODIS) are probablythe most employed markers for human identification purposes. STR databasesgenerated to interpret DNA profiles are also helpful for anthropologicalpurposes. In this work, we report admixture, population structure, and geneticrelationships of Mexican Mestizos with respect to Latin American and Caribbeanpopulations based on 13 CODIS-STRs. In addition, new STR population data wereincluded from Tijuana, Baja California (Northwest, Mexico), which represents aninteresting case of elevated genetic flow as a bordering city with the USA.Inter-population analyses included CODIS-STR data from 11 Mexican Mestizo, 12Latin American and four Caribbean populations, in addition to European,Amerindian, and African genetic pools as ancestral references. We report allelefrequencies and statistical parameters of forensic interest (PD, PE, Het, PIC,typical PI), for 15 STRs in Tijuana, Baja California. This Mexican border citywas peculiar by the increase of African ancestry, and by presenting three STRs inHardy-Weinberg disequilibrium, probably explained by recurrent gene flow. TheAmerindian ancestry in Central and Southeast of Mexico was the greatest in Latin America (50.9-68.6%), only comparable with the North of Central America andEcuador (48.8-56.4%), whereas the European ancestry was prevalent in SouthAmerica (66.7-75%). The African ancestry in Mexico was the smallest (2.2-6.3%) inLatin America (>/= 2.6%), particularly regarding Brazil (21%), Honduras (62%),and the Caribbean (43.2-65.2%). CODIS-STRs allowed detecting significantpopulation structure in Latin America based on greater presence of European,Amerindian, and African ancestries in Central/South America, Mexican Mestizos,and the Caribbean, respectively.\n",
      "CI  - Copyright (c) 2014 Elsevier GmbH. All rights reserved.\n",
      "FAU - Salazar-Flores, J\n",
      "AU  - Salazar-Flores J\n",
      "AD  - Instituto de Investigacion en Genetica Molecular, Centro Universitario de laCienega, Universidad de Guadalajara (CUCI-UdeG), Av. Universidad #1115, CP 47810 Ocotlan, Jalisco, Mexico.\n",
      "FAU - Zuniga-Chiquette, F\n",
      "AU  - Zuniga-Chiquette F\n",
      "AD  - Laboratorio de Genetica Forense, Servicios Periciales de la Procuraduria General de Justicia del Estado de Baja California, BC, Mexico.\n",
      "FAU - Rubi-Castellanos, R\n",
      "AU  - Rubi-Castellanos R\n",
      "AD  - Laboratorio de Genetica-CIR Biomedicas, Universidad Autonoma de Yucatan (UADY),Merida, Yucatan, Mexico.\n",
      "FAU - Alvarez-Miranda, J L\n",
      "AU  - Alvarez-Miranda JL\n",
      "AD  - Laboratorio de Genetica Forense, Servicios Periciales de la Procuraduria General de Justicia del Estado de Baja California, BC, Mexico.\n",
      "FAU - Zetina-Hernandez, A\n",
      "AU  - Zetina-Hernandez A\n",
      "AD  - Laboratorio de Genetica Forense, Servicios Periciales de la Procuraduria General de Justicia del Estado de Baja California, BC, Mexico.\n",
      "FAU - Martinez-Sevilla, V M\n",
      "AU  - Martinez-Sevilla VM\n",
      "AD  - Instituto de Investigacion en Genetica Molecular, Centro Universitario de laCienega, Universidad de Guadalajara (CUCI-UdeG), Av. Universidad #1115, CP 47810 Ocotlan, Jalisco, Mexico.\n",
      "FAU - Gonzalez-Andrade, F\n",
      "AU  - Gonzalez-Andrade F\n",
      "AD  - Escuela de Medicina, Universidad Central del Ecuador, Quito, Ecuador.\n",
      "FAU - Corach, D\n",
      "AU  - Corach D\n",
      "AD  - Servicio de Huellas Digitales Geneticas, Facultad de Farmacia y Bioquimica,Universidad de Buenos Aires, Buenos Aires, Argentina.\n",
      "FAU - Vullo, C\n",
      "AU  - Vullo C\n",
      "AD  - Equipo Argentino de Antropologia Forense, Independencia 644 - 5C, Edif. EME1,Cordoba, Argentina.\n",
      "FAU - Alvarez, J C\n",
      "AU  - Alvarez JC\n",
      "AD  - Laboratory of Genetic Identification, Department of Legal Medicine, University ofGranada, Granada, Spain.\n",
      "FAU - Lorente, J A\n",
      "AU  - Lorente JA\n",
      "AD  - Laboratory of Genetic Identification, Department of Legal Medicine, University ofGranada, Granada, Spain.\n",
      "FAU - Sanchez-Diz, P\n",
      "AU  - Sanchez-Diz P\n",
      "AD  - Genomics Medicine Group, Galician Foundation of Genomic Medicine and Universityof Santiago de Compostela, CIBERER, Santiago de Compostela, Spain.\n",
      "FAU - Herrera, R J\n",
      "AU  - Herrera RJ\n",
      "AD  - Department of Biological Sciences, Florida International University, Miami, FL33199, USA.\n",
      "FAU - Cerda-Flores, R M\n",
      "AU  - Cerda-Flores RM\n",
      "AD  - Facultad de Enfermeria, Universidad Autonoma de Nuevo Leon (UANL), Monterrey,Nuevo Leon, Mexico.\n",
      "FAU - Munoz-Valle, J F\n",
      "AU  - Munoz-Valle JF\n",
      "AD  - Instituto de Investigacion en Ciencias Biomedicas, Centro Universitario deCiencias de la Salud, Universidad de Guadalajara (CUCS-UdeG), Guadalajara,Jalisco, Mexico.\n",
      "FAU - Rangel-Villalobos, H\n",
      "AU  - Rangel-Villalobos H\n",
      "AD  - Instituto de Investigacion en Genetica Molecular, Centro Universitario de laCienega, Universidad de Guadalajara (CUCI-UdeG), Av. Universidad #1115, CP 47810 Ocotlan, Jalisco, Mexico. Electronic address: hrangel13@hotmail.com.\n",
      "LA  - eng\n",
      "PT  - Comparative Study\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20141105\n",
      "PL  - Germany\n",
      "TA  - Homo\n",
      "JT  - Homo : internationale Zeitschrift fur die vergleichende Forschung am Menschen\n",
      "JID - 0374655\n",
      "RN  - 9007-49-2 (DNA)\n",
      "SB  - IM\n",
      "MH  - Blacks/genetics\n",
      "MH  - Caribbean Region\n",
      "MH  - Central America\n",
      "MH  - DNA/*genetics\n",
      "MH  - *DNA Fingerprinting\n",
      "MH  - *Databases, Nucleic Acid\n",
      "MH  - Gene Flow/*genetics\n",
      "MH  - Gene Frequency/genetics\n",
      "MH  - Humans\n",
      "MH  - Indians, North American/*genetics\n",
      "MH  - Latin America\n",
      "MH  - Mexico\n",
      "MH  - Microsatellite Repeats/*genetics\n",
      "MH  - South America\n",
      "MH  - Whites/genetics\n",
      "EDAT- 2014/12/02 06:00\n",
      "MHDA- 2015/10/30 06:00\n",
      "CRDT- 2014/12/02 06:00\n",
      "PHST- 2013/08/25 00:00 [received]\n",
      "PHST- 2014/08/28 00:00 [accepted]\n",
      "PHST- 2014/12/02 06:00 [entrez]\n",
      "PHST- 2014/12/02 06:00 [pubmed]\n",
      "PHST- 2015/10/30 06:00 [medline]\n",
      "AID - S0018-442X(14)00103-6 [pii]\n",
      "AID - 10.1016/j.jchb.2014.08.005 [doi]\n",
      "PST - ppublish\n",
      "SO  - Homo. 2015 Feb;66(1):44-59. doi: 10.1016/j.jchb.2014.08.005. Epub 2014 Nov 5.\n",
      "\n",
      "PMID- 25268381\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20150608\n",
      "LR  - 20211021\n",
      "IS  - 1932-6203 (Electronic)\n",
      "IS  - 1932-6203 (Linking)\n",
      "VI  - 9\n",
      "IP  - 9\n",
      "DP  - 2014\n",
      "TI  - High local diversity of Trypanosoma in a common bat species, and implications forthe biogeography and taxonomy of the T. cruzi clade.\n",
      "PG  - e108603\n",
      "LID - 10.1371/journal.pone.0108603 [doi]\n",
      "AB  - The Trypanosoma cruzi clade is a group of parasites that comprises T. cruzi sensulato and its closest relatives. Although several species have been confirmedphylogenetically to belong to this clade, it is uncertain how many more speciescan be expected to belong into this group. Here, we present the results of asurvey of trypanosome parasites of the bat Artibeus jamaicensis from the PanamaCanal Zone, an important seed disperser. Using a genealogical speciesdelimitation approach, the Poisson tree processes (PTP), we tentativelyidentified five species of trypanosomes - all belonging to the T. cruzi clade. A small monophyletic group of three putative Trypanosoma species places at the baseof the clade phylogeny, providing evidence for at least five independentcolonization events of these parasites into the New World. Artibeus jamaicensispresents a high diversity of these blood parasites and is the vertebrate with thehighest number of putative trypanosome species reported from a single locality.Our results emphasize the need for continued efforts to survey mammaliantrypanosomes.\n",
      "FAU - Cottontail, Veronika M\n",
      "AU  - Cottontail VM\n",
      "AD  - Institute of Experimental Ecology, University of Ulm, Ulm, Germany; Institute of Medical Microbiology and Hygiene, University of Ulm, Ulm, Germany.\n",
      "FAU - Kalko, Elisabeth K V\n",
      "AU  - Kalko EK\n",
      "AD  - Institute of Experimental Ecology, University of Ulm, Ulm, Germany; SmithsonianTropical Research Institute, Balboa, Panama.\n",
      "FAU - Cottontail, Iain\n",
      "AU  - Cottontail I\n",
      "AD  - kbo-Isar-Amper-Klinikum, Taufkirchen (Vils), Germany.\n",
      "FAU - Wellinghausen, Nele\n",
      "AU  - Wellinghausen N\n",
      "AD  - Institute of Medical Microbiology and Hygiene, University of Ulm, Ulm, Germany;Gaertner & Colleagues Laboratory, Ravensburg, Germany.\n",
      "FAU - Tschapka, Marco\n",
      "AU  - Tschapka M\n",
      "AD  - Institute of Experimental Ecology, University of Ulm, Ulm, Germany; SmithsonianTropical Research Institute, Balboa, Panama.\n",
      "FAU - Perkins, Susan L\n",
      "AU  - Perkins SL\n",
      "AD  - Sackler Institute for Comparative Genomics, American Museum of Natural History,New York, New York, United States of America.\n",
      "FAU - Pinto, C Miguel\n",
      "AU  - Pinto CM\n",
      "AD  - Sackler Institute for Comparative Genomics, American Museum of Natural History,New York, New York, United States of America; Department of Mammalogy, AmericanMuseum of Natural History, New York, New York, United States of America; TheGraduate Center, City University of New York, New York, New York, United Statesof America; Centro de Investigacion en Enfermedades Infecciosas, PontificiaUniversidad Catolica del Ecuador, Quito, Ecuador.\n",
      "LA  - eng\n",
      "SI  - GENBANK/KM406851\n",
      "SI  - GENBANK/KM406852\n",
      "SI  - GENBANK/KM406853\n",
      "SI  - GENBANK/KM406854\n",
      "SI  - GENBANK/KM406855\n",
      "SI  - GENBANK/KM406856\n",
      "SI  - GENBANK/KM406857\n",
      "SI  - GENBANK/KM406858\n",
      "SI  - GENBANK/KM406859\n",
      "SI  - GENBANK/KM406860\n",
      "SI  - GENBANK/KM406861\n",
      "SI  - GENBANK/KM406862\n",
      "SI  - GENBANK/KM406863\n",
      "SI  - GENBANK/KM406864\n",
      "SI  - GENBANK/KM406865\n",
      "SI  - GENBANK/KM406866\n",
      "SI  - GENBANK/KM406867\n",
      "SI  - GENBANK/KM406868\n",
      "SI  - GENBANK/KM406869\n",
      "SI  - GENBANK/KM406870\n",
      "SI  - GENBANK/KM406871\n",
      "SI  - GENBANK/KM406872\n",
      "SI  - GENBANK/KM406873\n",
      "SI  - GENBANK/KM406874\n",
      "SI  - GENBANK/KM406875\n",
      "SI  - GENBANK/KM406876\n",
      "SI  - GENBANK/KM406877\n",
      "SI  - GENBANK/KM406878\n",
      "SI  - GENBANK/KM406879\n",
      "SI  - GENBANK/KM406880\n",
      "SI  - GENBANK/KM406881\n",
      "SI  - GENBANK/KM406882\n",
      "SI  - GENBANK/KM406883\n",
      "SI  - GENBANK/KM406884\n",
      "SI  - GENBANK/KM406885\n",
      "SI  - GENBANK/KM406886\n",
      "SI  - GENBANK/KM406887\n",
      "SI  - GENBANK/KM406888\n",
      "SI  - GENBANK/KM406889\n",
      "SI  - GENBANK/KM406890\n",
      "SI  - GENBANK/KM406891\n",
      "SI  - GENBANK/KM406892\n",
      "SI  - GENBANK/KM406893\n",
      "SI  - GENBANK/KM406894\n",
      "SI  - GENBANK/KM406895\n",
      "SI  - GENBANK/KM406896\n",
      "SI  - GENBANK/KM406897\n",
      "SI  - GENBANK/KM406898\n",
      "SI  - GENBANK/KM406899\n",
      "SI  - GENBANK/KM406900\n",
      "SI  - GENBANK/KM406901\n",
      "SI  - GENBANK/KM406902\n",
      "SI  - GENBANK/KM406903\n",
      "SI  - GENBANK/KM406904\n",
      "SI  - GENBANK/KM406905\n",
      "SI  - GENBANK/KM406906\n",
      "SI  - GENBANK/KM406907\n",
      "SI  - GENBANK/KM406908\n",
      "SI  - GENBANK/KM406909\n",
      "SI  - GENBANK/KM406910\n",
      "SI  - GENBANK/KM406911\n",
      "SI  - GENBANK/KM406912\n",
      "SI  - GENBANK/KM406913\n",
      "SI  - GENBANK/KM406914\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20140930\n",
      "PL  - United States\n",
      "TA  - PLoS One\n",
      "JT  - PloS one\n",
      "JID - 101285081\n",
      "RN  - 0 (RNA, Ribosomal, 18S)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Base Sequence\n",
      "MH  - Chagas Disease/epidemiology/parasitology/*veterinary\n",
      "MH  - Chiroptera/*parasitology\n",
      "MH  - *Genes, Protozoan\n",
      "MH  - Genetic Variation\n",
      "MH  - Molecular Sequence Data\n",
      "MH  - Panama\n",
      "MH  - *Phylogeny\n",
      "MH  - Phylogeography\n",
      "MH  - RNA, Ribosomal, 18S/*genetics\n",
      "MH  - Trypanosoma cruzi/classification/*genetics/isolation & purification\n",
      "PMC - PMC4182490\n",
      "EDAT- 2014/10/01 06:00\n",
      "MHDA- 2015/06/09 06:00\n",
      "CRDT- 2014/10/01 06:00\n",
      "PHST- 2014/06/05 00:00 [received]\n",
      "PHST- 2014/08/11 00:00 [accepted]\n",
      "PHST- 2014/10/01 06:00 [entrez]\n",
      "PHST- 2014/10/01 06:00 [pubmed]\n",
      "PHST- 2015/06/09 06:00 [medline]\n",
      "AID - 10.1371/journal.pone.0108603 [doi]\n",
      "AID - PONE-D-14-24819 [pii]\n",
      "PST - epublish\n",
      "SO  - PLoS One. 2014 Sep 30;9(9):e108603. doi: 10.1371/journal.pone.0108603.eCollection 2014.\n",
      "\n",
      "PMID- 25091349\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20150625\n",
      "LR  - 20141008\n",
      "IS  - 1879-3150 (Electronic)\n",
      "IS  - 0041-0101 (Linking)\n",
      "VI  - 90\n",
      "DP  - 2014 Nov\n",
      "TI  - Proteomic and toxicological profiling of the venom of Bothrocophias campbelli, a pitviper species from Ecuador and Colombia.\n",
      "PG  - 15-25\n",
      "LID - 10.1016/j.toxicon.2014.07.012 [doi]\n",
      "LID - S0041-0101(14)00208-6 [pii]\n",
      "AB  - Detailed snake venom proteomes for nearly a hundred species in different pitvipergenera have accumulated using 'venomics' methodologies. However, venomcomposition for some lineages remains poorly known. Bothrocophias (toad-headedpitvipers) is a genus restricted to the northwestern portion of South America forwhich information on venom composition is lacking. Here, we describe the protein composition, toxicological profiling, and antivenom neutralization of the venomof Bothrocophias campbelli, a species distributed in Colombia and Ecuador. Ouranalyses show that its venom mainly consists of phospholipases A2 (43.1%), serineproteinases (21.3%), and metalloproteinases (15.8%). The low proportion ofmetalloproteinases and high amount of a Lys49 phospholipase A2 homologuecorrelate well with the low hemorrhagic and high myotoxic effects found. Overall,B. campbelli venom showed a simpler composition compared to other crotalines inthe region. A polyvalent antivenom prepared with a mixture of Bothrops asper,Crotalus simus, and Lachesis stenophrys venoms cross-recognized B. campbellivenom and neutralized its lethal effect in mice, albeit with a lower potency thanfor B. asper venom. Additional work comparing B. campbelli venom properties with those of related species could help understand the evolution of different venomprotein families during the South American radiation of New World pitvipers.\n",
      "CI  - Copyright (c) 2014 Elsevier Ltd. All rights reserved.\n",
      "FAU - Salazar-Valenzuela, David\n",
      "AU  - Salazar-Valenzuela D\n",
      "AD  - Department of Evolution, Ecology and Organismal Biology, The Ohio StateUniversity, 300 Aronoff Laboratory, 318 W. 12th Ave., Columbus, OH 43210-1293,USA; Escuela de Biologia, Pontificia Universidad Catolica del Ecuador, Avenida 12de Octubre y Roca, Apartado 17-01-2184, Quito, Ecuador.\n",
      "FAU - Mora-Obando, Diana\n",
      "AU  - Mora-Obando D\n",
      "AD  - Instituto Clodomiro Picado, Facultad de Microbiologia, Universidad de Costa Rica,San Jose 11501, Costa Rica.\n",
      "FAU - Fernandez, Maria Laura\n",
      "AU  - Fernandez ML\n",
      "AD  - Instituto Clodomiro Picado, Facultad de Microbiologia, Universidad de Costa Rica,San Jose 11501, Costa Rica.\n",
      "FAU - Loaiza-Lange, Amaru\n",
      "AU  - Loaiza-Lange A\n",
      "AD  - Escuela de Biologia, Pontificia Universidad Catolica del Ecuador, Avenida 12 deOctubre y Roca, Apartado 17-01-2184, Quito, Ecuador.\n",
      "FAU - Gibbs, H Lisle\n",
      "AU  - Gibbs HL\n",
      "AD  - Department of Evolution, Ecology and Organismal Biology, The Ohio StateUniversity, 300 Aronoff Laboratory, 318 W. 12th Ave., Columbus, OH 43210-1293,USA.\n",
      "FAU - Lomonte, Bruno\n",
      "AU  - Lomonte B\n",
      "AD  - Instituto Clodomiro Picado, Facultad de Microbiologia, Universidad de Costa Rica,San Jose 11501, Costa Rica. Electronic address: bruno.lomonte@ucr.ac.cr.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20140801\n",
      "PL  - England\n",
      "TA  - Toxicon\n",
      "JT  - Toxicon : official journal of the International Society on Toxinology\n",
      "JID - 1307333\n",
      "RN  - 0 (Viper Venoms)\n",
      "SB  - IM\n",
      "MH  - Amino Acid Sequence\n",
      "MH  - Animals\n",
      "MH  - Colombia\n",
      "MH  - Ecuador\n",
      "MH  - Female\n",
      "MH  - Male\n",
      "MH  - Mice\n",
      "MH  - Molecular Sequence Data\n",
      "MH  - *Proteomics\n",
      "MH  - Viper Venoms/*metabolism/toxicity\n",
      "MH  - Viperidae\n",
      "OTO - NOTNLM\n",
      "OT  - Bothrocophias campbelli\n",
      "OT  - Proteomic\n",
      "OT  - Snake venom\n",
      "OT  - Toxicity\n",
      "OT  - Venomics\n",
      "OT  - Viperidae\n",
      "EDAT- 2014/08/06 06:00\n",
      "MHDA- 2015/06/26 06:00\n",
      "CRDT- 2014/08/06 06:00\n",
      "PHST- 2014/06/16 00:00 [received]\n",
      "PHST- 2014/07/24 00:00 [accepted]\n",
      "PHST- 2014/08/06 06:00 [entrez]\n",
      "PHST- 2014/08/06 06:00 [pubmed]\n",
      "PHST- 2015/06/26 06:00 [medline]\n",
      "AID - S0041-0101(14)00208-6 [pii]\n",
      "AID - 10.1016/j.toxicon.2014.07.012 [doi]\n",
      "PST - ppublish\n",
      "SO  - Toxicon. 2014 Nov;90:15-25. doi: 10.1016/j.toxicon.2014.07.012. Epub 2014 Aug 1.\n",
      "\n",
      "PMID- 24929830\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20140904\n",
      "LR  - 20211021\n",
      "IS  - 1546-1718 (Electronic)\n",
      "IS  - 1061-4036 (Linking)\n",
      "VI  - 46\n",
      "IP  - 7\n",
      "DP  - 2014 Jul\n",
      "TI  - Whipworm genome and dual-species transcriptome analyses provide molecularinsights into an intimate host-parasite interaction.\n",
      "PG  - 693-700\n",
      "LID - 10.1038/ng.3010 [doi]\n",
      "AB  - Whipworms are common soil-transmitted helminths that cause debilitating chronicinfections in man. These nematodes are only distantly related to Caenorhabditiselegans and have evolved to occupy an unusual niche, tunneling through epithelialcells of the large intestine. We report here the whole-genome sequences of thehuman-infective Trichuris trichiura and the mouse laboratory model Trichurismuris. On the basis of whole-transcriptome analyses, we identify many genes that are expressed in a sex- or life stage-specific manner and characterize thetranscriptional landscape of a morphological region with unique biologicaladaptations, namely, bacillary band and stichosome, found only in whipworms andrelated parasites. Using RNA sequencing data from whipworm-infected mice, wedescribe the regulated T helper 1 (TH1)-like immune response of the chronicallyinfected cecum in unprecedented detail. In silico screening identified numerousnew potential drug targets against trichuriasis. Together, these genomes andassociated functional data elucidate key aspects of the molecular host-parasiteinteractions that define chronic whipworm infection.\n",
      "FAU - Foth, Bernardo J\n",
      "AU  - Foth BJ\n",
      "AD  - Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus,Hinxton, Cambridge CB10 1SA, UK.\n",
      "FAU - Tsai, Isheng J\n",
      "AU  - Tsai IJ\n",
      "AD  - Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus,Hinxton, Cambridge CB10 1SA, UK.\n",
      "AD  - Division of Parasitology, Department of Infectious Disease, Faculty of Medicine, University of Miyazaki, Miyazaki 889-1692, Japan.\n",
      "FAU - Reid, Adam J\n",
      "AU  - Reid AJ\n",
      "AD  - Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus,Hinxton, Cambridge CB10 1SA, UK.\n",
      "FAU - Bancroft, Allison J\n",
      "AU  - Bancroft AJ\n",
      "AD  - Faculty of Life Sciences, University of Manchester, Oxford Road, Manchester M139PT, UK.\n",
      "FAU - Nichol, Sarah\n",
      "AU  - Nichol S\n",
      "AD  - Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus,Hinxton, Cambridge CB10 1SA, UK.\n",
      "FAU - Tracey, Alan\n",
      "AU  - Tracey A\n",
      "AD  - Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus,Hinxton, Cambridge CB10 1SA, UK.\n",
      "FAU - Holroyd, Nancy\n",
      "AU  - Holroyd N\n",
      "AD  - Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus,Hinxton, Cambridge CB10 1SA, UK.\n",
      "FAU - Cotton, James A\n",
      "AU  - Cotton JA\n",
      "AD  - Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus,Hinxton, Cambridge CB10 1SA, UK.\n",
      "FAU - Stanley, Eleanor J\n",
      "AU  - Stanley EJ\n",
      "AD  - Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus,Hinxton, Cambridge CB10 1SA, UK.\n",
      "FAU - Zarowiecki, Magdalena\n",
      "AU  - Zarowiecki M\n",
      "AD  - Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus,Hinxton, Cambridge CB10 1SA, UK.\n",
      "FAU - Liu, Jimmy Z\n",
      "AU  - Liu JZ\n",
      "AD  - Statistical Genetics, Wellcome Trust Sanger Institute, Wellcome Trust GenomeCampus, Hinxton, Cambridge CB10 1SA, UK.\n",
      "FAU - Huckvale, Thomas\n",
      "AU  - Huckvale T\n",
      "AD  - Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus,Hinxton, Cambridge CB10 1SA, UK.\n",
      "FAU - Cooper, Philip J\n",
      "AU  - Cooper PJ\n",
      "AD  - Liverpool School of Tropical Medicine, Pembroke Place, Liverpool L3 5QA, UK.\n",
      "AD  - Centro de Investigacion en Enfermedades Infecciosas, Escuela de Biologia,Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\n",
      "FAU - Grencis, Richard K\n",
      "AU  - Grencis RK\n",
      "AD  - Faculty of Life Sciences, University of Manchester, Oxford Road, Manchester M139PT, UK.\n",
      "FAU - Berriman, Matthew\n",
      "AU  - Berriman M\n",
      "AD  - Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus,Hinxton, Cambridge CB10 1SA, UK.\n",
      "LA  - eng\n",
      "GR  - WT100290MA/WT_/Wellcome Trust/United Kingdom\n",
      "GR  - 100290/Wellcome Trust/United Kingdom\n",
      "GR  - 100714/Wellcome Trust/United Kingdom\n",
      "GR  - 088862/Z/09/Z/WT_/Wellcome Trust/United Kingdom\n",
      "GR  - 088862/Wellcome Trust/United Kingdom\n",
      "GR  - Wellcome Trust/United Kingdom\n",
      "GR  - WT083620MA/WT_/Wellcome Trust/United Kingdom\n",
      "GR  - 098051/WT_/Wellcome Trust/United Kingdom\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20140615\n",
      "PL  - United States\n",
      "TA  - Nat Genet\n",
      "JT  - Nature genetics\n",
      "JID - 9216904\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - *Gene Expression Profiling\n",
      "MH  - *Genome, Helminth\n",
      "MH  - Host-Parasite Interactions/*genetics/*immunology\n",
      "MH  - Humans\n",
      "MH  - Intestines/parasitology\n",
      "MH  - Male\n",
      "MH  - Mice\n",
      "MH  - Mice, Inbred C57BL\n",
      "MH  - Phylogeny\n",
      "MH  - Species Specificity\n",
      "MH  - Th1 Cells/*immunology\n",
      "MH  - Trichuriasis/*genetics/immunology/parasitology\n",
      "MH  - Trichuris/*genetics/immunology\n",
      "PMC - PMC5012510\n",
      "MID - EMS58674\n",
      "OID - NLM: EMS58674\n",
      "EDAT- 2014/06/16 06:00\n",
      "MHDA- 2014/09/05 06:00\n",
      "CRDT- 2014/06/16 06:00\n",
      "PHST- 2013/10/24 00:00 [received]\n",
      "PHST- 2014/05/22 00:00 [accepted]\n",
      "PHST- 2014/06/16 06:00 [entrez]\n",
      "PHST- 2014/06/16 06:00 [pubmed]\n",
      "PHST- 2014/09/05 06:00 [medline]\n",
      "AID - 10.1038/ng.3010 [doi]\n",
      "PST - ppublish\n",
      "SO  - Nat Genet. 2014 Jul;46(7):693-700. doi: 10.1038/ng.3010. Epub 2014 Jun 15.\n",
      "\n",
      "PMID- 24823669\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20150330\n",
      "LR  - 20220129\n",
      "IS  - 1549-5469 (Electronic)\n",
      "IS  - 1088-9051 (Linking)\n",
      "VI  - 24\n",
      "IP  - 8\n",
      "DP  - 2014 Aug\n",
      "TI  - Population genomics of parallel hybrid zones in the mimetic butterflies, H.melpomene and H. erato.\n",
      "PG  - 1316-33\n",
      "LID - 10.1101/gr.169292.113 [doi]\n",
      "AB  - Hybrid zones can be valuable tools for studying evolution and identifying genomicregions responsible for adaptive divergence and underlying phenotypic variation. Hybrid zones between subspecies of Heliconius butterflies can be very narrow and are maintained by strong selection acting on color pattern. The comimeticspecies, H. erato and H. melpomene, have parallel hybrid zones in which bothspecies undergo a change from one color pattern form to another. We userestriction-associated DNA sequencing to obtain several thousand genome-widesequence markers and use these to analyze patterns of population divergenceacross two pairs of parallel hybrid zones in Peru and Ecuador. We compare twoapproaches for analysis of this type of data-alignment to a reference genome and de novo assembly-and find that alignment gives the best results for species both closely (H. melpomene) and distantly (H. erato, approximately 15% divergent)related to the reference sequence. Our results confirm that the color patterncontrolling loci account for the majority of divergent regions across the genome,but we also detect other divergent regions apparently unlinked to color patterndifferences. We also use association mapping to identify previously unmappedcolor pattern loci, in particular the Ro locus. Finally, we identify a newcryptic population of H. timareta in Ecuador, which occurs at relatively lowaltitude and is mimetic with H. melpomene malleti.\n",
      "CI  - (c) 2014 Nadeau et al.; Published by Cold Spring Harbor Laboratory Press.\n",
      "FAU - Nadeau, Nicola J\n",
      "AU  - Nadeau NJ\n",
      "AUID- ORCID: http://orcid.org/0000-0002-9319-921X\n",
      "AD  - Department of Zoology, University of Cambridge, Cambridge CB2 3EJ, UnitedKingdom; Department of Animal and Plant Sciences, University of Sheffield,Sheffield, S10 2TN, United Kingdom;\n",
      "FAU - Ruiz, Mayte\n",
      "AU  - Ruiz M\n",
      "AD  - Department of Biology and Center for Applied Tropical Ecology and Conservation,University of Puerto Rico, Rio Piedras, San Juan, Puerto Rico 00921;\n",
      "FAU - Salazar, Patricio\n",
      "AU  - Salazar P\n",
      "AD  - Department of Zoology, University of Cambridge, Cambridge CB2 3EJ, UnitedKingdom; Centro de Investigacion en Biodiversidad y Cambio Climatico (BioCamb),Universidad Tecnologica Indoamerica, Quito, Ecuador;\n",
      "FAU - Counterman, Brian\n",
      "AU  - Counterman B\n",
      "AD  - Department of Biology, Mississippi State University, Mississippi 39762, USA;\n",
      "FAU - Medina, Jose Alejandro\n",
      "AU  - Medina JA\n",
      "AD  - High Performance Computing Facility, University of Puerto Rico, San Juan, Puerto Rico, 00921;\n",
      "FAU - Ortiz-Zuazaga, Humberto\n",
      "AU  - Ortiz-Zuazaga H\n",
      "AD  - High Performance Computing Facility, University of Puerto Rico, San Juan, Puerto Rico, 00921; Department of Computer Science, University of Puerto Rico, RioPiedras, San Juan, Puerto Rico 00921;\n",
      "FAU - Morrison, Anna\n",
      "AU  - Morrison A\n",
      "AD  - Department of Zoology, University of Cambridge, Cambridge CB2 3EJ, UnitedKingdom;\n",
      "FAU - McMillan, W Owen\n",
      "AU  - McMillan WO\n",
      "AUID- ORCID: http://orcid.org/0000-0003-2805-2745\n",
      "AD  - Smithsonian Tropical Research Institute, Apartado 0843-03092, Balboa, Ancon,Panama.\n",
      "FAU - Jiggins, Chris D\n",
      "AU  - Jiggins CD\n",
      "AUID- ORCID: http://orcid.org/0000-0002-7809-062X\n",
      "AD  - Department of Zoology, University of Cambridge, Cambridge CB2 3EJ, UnitedKingdom; Smithsonian Tropical Research Institute, Apartado 0843-03092, Balboa,Ancon, Panama c.jiggins@zoo.cam.ac.uk.\n",
      "FAU - Papa, Riccardo\n",
      "AU  - Papa R\n",
      "AD  - Department of Biology and Center for Applied Tropical Ecology and Conservation,University of Puerto Rico, Rio Piedras, San Juan, Puerto Rico 00921;\n",
      "LA  - eng\n",
      "GR  - BB/G008841/1/BB_/Biotechnology and Biological Sciences Research Council/UnitedKingdom\n",
      "GR  - BB/H01439X/1/BB_/Biotechnology and Biological Sciences Research Council/UnitedKingdom\n",
      "GR  - P20 GM103475/GM/NIGMS NIH HHS/United States\n",
      "GR  - P20GM103475/GM/NIGMS NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, N.I.H., Extramural\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20140513\n",
      "PL  - United States\n",
      "TA  - Genome Res\n",
      "JT  - Genome research\n",
      "JID - 9518021\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Butterflies/anatomy & histology/*genetics\n",
      "MH  - Chromosome Mapping\n",
      "MH  - Evolution, Molecular\n",
      "MH  - Genetic Loci\n",
      "MH  - Genetic Speciation\n",
      "MH  - Genetic Variation\n",
      "MH  - *Genome, Insect\n",
      "MH  - Phenotype\n",
      "MH  - Phylogeny\n",
      "MH  - Pigmentation\n",
      "MH  - Wings, Animal/anatomy & histology\n",
      "PMC - PMC4120085\n",
      "EDAT- 2014/05/16 06:00\n",
      "MHDA- 2015/03/31 06:00\n",
      "CRDT- 2014/05/15 06:00\n",
      "PHST- 2014/05/15 06:00 [entrez]\n",
      "PHST- 2014/05/16 06:00 [pubmed]\n",
      "PHST- 2015/03/31 06:00 [medline]\n",
      "AID - gr.169292.113 [pii]\n",
      "AID - 10.1101/gr.169292.113 [doi]\n",
      "PST - ppublish\n",
      "SO  - Genome Res. 2014 Aug;24(8):1316-33. doi: 10.1101/gr.169292.113. Epub 2014 May 13.\n",
      "\n",
      "PMID- 24603342\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20141117\n",
      "LR  - 20210201\n",
      "IS  - 1878-0326 (Electronic)\n",
      "IS  - 1872-4973 (Linking)\n",
      "VI  - 10\n",
      "DP  - 2014 May\n",
      "TI  - GHEP-ISFG collaborative exercise on mixture profiles of autosomal STRs(GHEP-MIX01, GHEP-MIX02 and GHEP-MIX03): results and evaluation.\n",
      "PG  - 64-72\n",
      "LID - S1872-4973(14)00025-8 [pii]\n",
      "LID - 10.1016/j.fsigen.2014.01.009 [doi]\n",
      "AB  - One of the main objectives of the Spanish and Portuguese-Speaking Group of theInternational Society for Forensic Genetics (GHEP-ISFG) is to promote andcontribute to the development and dissemination of scientific knowledge in thearea of forensic genetics. Due to this fact, GHEP-ISFG holds different workingcommissions that are set up to develop activities in scientific aspects ofgeneral interest. One of them, the Mixture Commission of GHEP-ISFG, has organizedannually, since 2009, a collaborative exercise on analysis and interpretation of autosomal short tandem repeat (STR) mixture profiles. Until now, three exercises have been organized (GHEP-MIX01, GHEP-MIX02 and GHEP-MIX03), with 32, 24 and 17participant laboratories respectively. The exercise aims to give a general visionby addressing, through the proposal of mock cases, aspects related to the editionof mixture profiles and the statistical treatment. The main conclusions obtained from these exercises may be summarized as follows. Firstly, the data show anincreased tendency of the laboratories toward validation of DNA mixture profiles analysis following international recommendations (ISO/IEC 17025:2005). Secondly, the majority of discrepancies are mainly encountered in stutters positions(53.4%, 96.0% and 74.9%, respectively for the three editions). On the other hand,the results submitted reveal the importance of performing duplicate analysis byusing different kits in order to reduce errors as much as possible. Regarding thestatistical aspect (GHEP-MIX02 and 03), all participants employed the likelihood ratio (LR) parameter to evaluate the statistical compatibility and the formulasemployed were quite similar. When the hypotheses to evaluate the LR value werelocked by the coordinators (GHEP-MIX02) the results revealed a minor number ofdiscrepancies that were mainly due to clerical reasons. However, the GHEP-MIX03exercise allowed the participants to freely come up with their own hypotheses to calculate the LR value. In this situation the laboratories reported severaloptions to explain the mock cases proposed and therefore significant differences between the final LR values were obtained. Complete information concerning thebackground of the criminal case is a critical aspect in order to select theadequate hypotheses to calculate the LR value. Although this should be a task forthe judicial court to decide, it is important for the expert to account for thedifferent possibilities and scenarios, and also offer this expertise to thejudge. In addition, continuing education in the analysis and interpretation ofmixture DNA profiles may also be a priority for the vast majority of forensiclaboratories.\n",
      "CI  - Copyright (c) 2014 Elsevier Ireland Ltd. All rights reserved.\n",
      "FAU - Crespillo, M\n",
      "AU  - Crespillo M\n",
      "AD  - Mixture Commission of the GHEP-ISFG (The Spanish and Portuguese Speaking Working Group of the International Society for Forensic Genetics), Spain; INTCF -National Institute of Toxicology and Forensic Science, Department of Barcelona,Spain. Electronic address: manuel.crespillo@mju.es.\n",
      "FAU - Barrio, P A\n",
      "AU  - Barrio PA\n",
      "AD  - Mixture Commission of the GHEP-ISFG (The Spanish and Portuguese Speaking Working Group of the International Society for Forensic Genetics), Spain; INTCF -National Institute of Toxicology and Forensic Science, Department of Barcelona,Spain.\n",
      "FAU - Luque, J A\n",
      "AU  - Luque JA\n",
      "AD  - Mixture Commission of the GHEP-ISFG (The Spanish and Portuguese Speaking Working Group of the International Society for Forensic Genetics), Spain; INTCF -National Institute of Toxicology and Forensic Science, Department of Barcelona,Spain.\n",
      "FAU - Alves, C\n",
      "AU  - Alves C\n",
      "AD  - IPATIMUP - Institute of Molecular Pathology and Immunology of the University ofPorto, Porto, Portugal.\n",
      "FAU - Aler, M\n",
      "AU  - Aler M\n",
      "AD  - Seccion de Genetica Forense y Criminalistica, Servicio de Laboratorio, Valencia, Spain.\n",
      "FAU - Alessandrini, F\n",
      "AU  - Alessandrini F\n",
      "AD  - Department of Biomedical Sciences and Public Health, Universita Politecnica delleMarche, Italy.\n",
      "FAU - Andrade, L\n",
      "AU  - Andrade L\n",
      "AD  - Servico de Genetica e Biologia Forenses, Delegacao do Centro, Instituto Nacional de Medicina Legal e Ciencias Forenses, I.P. Coimbra, Portugal.\n",
      "FAU - Barretto, R M\n",
      "AU  - Barretto RM\n",
      "AD  - Universidade Estadual Paulista/UNESP, Laboratorio de Investigacao de Paternidade,Sao Paulo, SP, Brazil.\n",
      "FAU - Bofarull, A\n",
      "AU  - Bofarull A\n",
      "AD  - INTCF - National Institute of Toxicology and Forensic Science, Department ofBarcelona, Spain.\n",
      "FAU - Costa, S\n",
      "AU  - Costa S\n",
      "AD  - Genetic Forensic Service, North Branch of the National Institute of LegalMedicine and Forensic Sciences, IP, Porto, Portugal.\n",
      "FAU - Garcia, M A\n",
      "AU  - Garcia MA\n",
      "AD  - Servicio de Criminalistica de la Guardia Civil, Laboratorio Central deCriminalistica, Departamento de Biologia, Madrid, Spain.\n",
      "FAU - Garcia, O\n",
      "AU  - Garcia O\n",
      "AD  - Forensic Science Unit, Forensic Genetics Section, Basque Country Police, Erandio,Bizkaia, Spain.\n",
      "FAU - Gaviria, A\n",
      "AU  - Gaviria A\n",
      "AD  - Laboratorio de Genetica Molecular, Cruz Roja Ecuatoriana, Quito, Ecuador.\n",
      "FAU - Gladys, A\n",
      "AU  - Gladys A\n",
      "AD  - Laboratorio de Analisis Comparativo de ADN, de la Suprema Corte de Justicia deBuenos Aires, Argentina.\n",
      "FAU - Gorostiza, A\n",
      "AU  - Gorostiza A\n",
      "AD  - Laboratorio Identificacion Genetica, GENOMICA SAU, Grupo Zeltia, Madrid, Spain.\n",
      "FAU - Hernandez, A\n",
      "AU  - Hernandez A\n",
      "AD  - INTCF - National Institute of Toxicology and Forensic Science, Delegation ofCanarias, Santa Cruz de Tenerife, Spain.\n",
      "FAU - Pinero, M Herrera\n",
      "AU  - Pinero MH\n",
      "AD  - Laboratorio Genda SA., Ciudad Autonoma de Buenos Aires, Argentina.\n",
      "FAU - Hombreiro, L\n",
      "AU  - Hombreiro L\n",
      "AD  - Laboratorio Territorial de Biologia - ADN, Brigada de Policia Cientifica,Jefatura Superior de Policia de Galicia, La Coruna, Spain.\n",
      "FAU - Ibarra, A A\n",
      "AU  - Ibarra AA\n",
      "AD  - Laboratorio de Identificacion Genetica - IdentiGEN, Universidad de Antioquia,Colombia.\n",
      "FAU - Jimenez, M J\n",
      "AU  - Jimenez MJ\n",
      "AD  - Unitat Central de Laboratori Biologic, Area Central de Criminalistica, Divisio dePolicia Cientifica, Policia de la Generalitat - Mossos d'Esquadra, Barcelona,Spain.\n",
      "FAU - Luque, G M\n",
      "AU  - Luque GM\n",
      "AD  - INTCF - National Institute of Toxicology and Forensic Science, Department ofMadrid, Spain.\n",
      "FAU - Madero, P\n",
      "AU  - Madero P\n",
      "AD  - Centro de Analisis Geneticos, Zaragoza, Spain.\n",
      "FAU - Martinez-Jarreta, B\n",
      "AU  - Martinez-Jarreta B\n",
      "AD  - Laboratorio de Genetica y Identificacion humana, Facultad de Medicina,Universidad de Zaragoza, Spain.\n",
      "FAU - Masciovecchio, M V\n",
      "AU  - Masciovecchio MV\n",
      "AD  - Bioquimica, IACA Laboratorios, Bahia Blanca, Argentina.\n",
      "FAU - Modesti, N M\n",
      "AU  - Modesti NM\n",
      "AD  - Instituto de Genetica Forense, Poder Judicial de Cordoba, Argentina.\n",
      "FAU - Moreno, F\n",
      "AU  - Moreno F\n",
      "AD  - Unidad de Genetica Forense, Servicio Medico Legal, Santiago de Chile, Chile.\n",
      "FAU - Pagano, S\n",
      "AU  - Pagano S\n",
      "AD  - Laboratorio de Analisis de ADN para el CODIS, Direccion Nacional de PoliciaTecnica, Uruguay.\n",
      "FAU - Pedrosa, S\n",
      "AU  - Pedrosa S\n",
      "AD  - Navarra de Servicios y Tecnologias S.A.U. (NASERTIC), Pamplona, Spain.\n",
      "FAU - Plaza, G\n",
      "AU  - Plaza G\n",
      "AD  - NEODIAGNOSTICA, S.L., Lleida, Spain.\n",
      "FAU - Prat, E\n",
      "AU  - Prat E\n",
      "AD  - Laboratorio de ADN, Comisaria General de Policia Cientifica, Madrid, Spain.\n",
      "FAU - Puente, J\n",
      "AU  - Puente J\n",
      "AD  - LabGenetics, Laboratorio de Genetica Clinica S.L., Madrid, Spain.\n",
      "FAU - Rendo, F\n",
      "AU  - Rendo F\n",
      "AD  - Sequencing and Genotyping Unit, Genomics Service - SGIker, University of theBasque Country, UPV/EHU, Bilbao, Spain.\n",
      "FAU - Ribeiro, T\n",
      "AU  - Ribeiro T\n",
      "AD  - Servico de Genetica e Biologia Forenses, Instituto Nacional de Medicina Legal eCiencias Forenses, Delegacao Sul, Lisboa, Portugal.\n",
      "FAU - Sala, A\n",
      "AU  - Sala A\n",
      "AD  - Servicio de Huellas Digitales Geneticas, Fac. Farmacia y Bioquimica, Universityde Buenos Aires, Argentina.\n",
      "FAU - Santamaria, E\n",
      "AU  - Santamaria E\n",
      "AD  - INTCF - National Institute of Toxicology and Forensic Science, Department ofSevilla, Spain.\n",
      "FAU - Saragoni, V G\n",
      "AU  - Saragoni VG\n",
      "AD  - Unidad de Genetica Forense, Departamento de Laboratorios, Servicio Medico Legalde Santiago de Chile, Santiago, Chile.\n",
      "FAU - Whittle, M R\n",
      "AU  - Whittle MR\n",
      "AD  - Genomic Engenharia Molecular, Sao Paulo, Brazil.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20140206\n",
      "PL  - Netherlands\n",
      "TA  - Forensic Sci Int Genet\n",
      "JT  - Forensic science international. Genetics\n",
      "JID - 101317016\n",
      "SB  - IM\n",
      "MH  - Humans\n",
      "MH  - *Microsatellite Repeats\n",
      "MH  - Surveys and Questionnaires\n",
      "OTO - NOTNLM\n",
      "OT  - Autosomal STRs\n",
      "OT  - Collaborative exercise\n",
      "OT  - DNA analysis\n",
      "OT  - Mixture interpretation\n",
      "OT  - Spanish and Portuguese-Speaking Working Group\n",
      "EDAT- 2014/03/08 06:00\n",
      "MHDA- 2014/11/18 06:00\n",
      "CRDT- 2014/03/08 06:00\n",
      "PHST- 2013/11/16 00:00 [received]\n",
      "PHST- 2013/12/30 00:00 [revised]\n",
      "PHST- 2014/01/27 00:00 [accepted]\n",
      "PHST- 2014/03/08 06:00 [entrez]\n",
      "PHST- 2014/03/08 06:00 [pubmed]\n",
      "PHST- 2014/11/18 06:00 [medline]\n",
      "AID - S1872-4973(14)00025-8 [pii]\n",
      "AID - 10.1016/j.fsigen.2014.01.009 [doi]\n",
      "PST - ppublish\n",
      "SO  - Forensic Sci Int Genet. 2014 May;10:64-72. doi: 10.1016/j.fsigen.2014.01.009.Epub 2014 Feb 6.\n",
      "\n",
      "PMID- 24556246\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20141203\n",
      "LR  - 20181202\n",
      "IS  - 1096-0309 (Electronic)\n",
      "IS  - 0003-2697 (Linking)\n",
      "VI  - 452\n",
      "DP  - 2014 May 1\n",
      "TI  - RNA isolation from loquat and other recalcitrant woody plants with high qualityand yield.\n",
      "PG  - 46-53\n",
      "LID - 10.1016/j.ab.2014.02.010 [doi]\n",
      "LID - S0003-2697(14)00059-1 [pii]\n",
      "AB  - RNA isolation is difficult in plants that contain large amounts ofpolysaccharides and polyphenol compounds. To date, no commercial kit has beendeveloped for the isolation of high-quality RNA from tissues with thesecharacteristics, especially for fruit. The common protocols for RNA isolation aretedious and usually result in poor yields when applied to recalcitrant planttissues. Here an efficient RNA isolation protocol based on cetyltrimethylammoniumbromide (CTAB) and two successive precipitations with 10 M lithium chloride(LiCl) was developed specifically for loquat fruits, but it was proved to workefficiently in other tissues of loquat and woody plants. The RNA isolated by thisimproved protocol was not only of high purity and integrity (A260/A280 ratiosranged from 1.90 to 2.04 and A260/A230 ratios were>2.0) but also of high yield(up to 720 mug on average [coefficient of variation=21%] total RNA per gram freshtissue). The protocol was tested on loquat fruit (different stages ofdevelopment, postharvest, ripening, and bruising), leaf, root, flower, stem, and bud; quince fruit and root; grapevine cells in liquid culture; and rose petals.The RNA obtained with this method is amenable to enzymatic treatments and can be efficiently applied for research on gene characterization, expression, andfunction.\n",
      "CI  - Copyright (c) 2014 Elsevier Inc. All rights reserved.\n",
      "FAU - Morante-Carriel, Jaime\n",
      "AU  - Morante-Carriel J\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of Alicante, 03690 San Vicente delRaspeig, Alicante, Spain; Biotechnology and Molecular Biology Group, QuevedoState Technical University, EC-120501 Quevedo, Ecuador.\n",
      "FAU - Selles-Marchart, Susana\n",
      "AU  - Selles-Marchart S\n",
      "AD  - Research Technical Facility, Proteomics and Genomics Division, University ofAlicante, 03690 San Vicente del Raspeig, Alicante, Spain.\n",
      "FAU - Martinez-Marquez, Ascension\n",
      "AU  - Martinez-Marquez A\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of Alicante, 03690 San Vicente delRaspeig, Alicante, Spain.\n",
      "FAU - Martinez-Esteso, Maria Jose\n",
      "AU  - Martinez-Esteso MJ\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of Alicante, 03690 San Vicente delRaspeig, Alicante, Spain.\n",
      "FAU - Luque, Ignacio\n",
      "AU  - Luque I\n",
      "AD  - Instituto de Bioquimica Vegetal y Fotosintesis, CSIC and Universidad de Sevilla, 41092 Seville, Spain.\n",
      "FAU - Bru-Martinez, Roque\n",
      "AU  - Bru-Martinez R\n",
      "AD  - Plant Proteomics and Functional Genomics Group, Department of Agrochemistry andBiochemistry, Faculty of Science, University of Alicante, 03690 San Vicente delRaspeig, Alicante, Spain. Electronic address: roque.bru@ua.es.\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20140217\n",
      "PL  - United States\n",
      "TA  - Anal Biochem\n",
      "JT  - Analytical biochemistry\n",
      "JID - 0370535\n",
      "RN  - 0 (Cetrimonium Compounds)\n",
      "RN  - 0 (RNA, Plant)\n",
      "RN  - Z7FF1XKL7A (Cetrimonium)\n",
      "SB  - IM\n",
      "MH  - Cetrimonium\n",
      "MH  - Cetrimonium Compounds/chemistry\n",
      "MH  - Chemical Fractionation/*methods\n",
      "MH  - Eriobotrya/*chemistry/genetics/growth & development\n",
      "MH  - Genome, Plant/genetics\n",
      "MH  - RNA, Plant/chemistry/*isolation & purification\n",
      "MH  - Wood/*chemistry/genetics/growth & development\n",
      "OTO - NOTNLM\n",
      "OT  - Fruits\n",
      "OT  - Polyphenols\n",
      "OT  - Polysaccharides\n",
      "OT  - RNA isolation\n",
      "OT  - Woody plants\n",
      "EDAT- 2014/02/22 06:00\n",
      "MHDA- 2014/12/15 06:00\n",
      "CRDT- 2014/02/22 06:00\n",
      "PHST- 2013/11/26 00:00 [received]\n",
      "PHST- 2014/02/06 00:00 [revised]\n",
      "PHST- 2014/02/07 00:00 [accepted]\n",
      "PHST- 2014/02/22 06:00 [entrez]\n",
      "PHST- 2014/02/22 06:00 [pubmed]\n",
      "PHST- 2014/12/15 06:00 [medline]\n",
      "AID - S0003-2697(14)00059-1 [pii]\n",
      "AID - 10.1016/j.ab.2014.02.010 [doi]\n",
      "PST - ppublish\n",
      "SO  - Anal Biochem. 2014 May 1;452:46-53. doi: 10.1016/j.ab.2014.02.010. Epub 2014 Feb 17.\n",
      "\n",
      "PMID- 24342523\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20150423\n",
      "LR  - 20211021\n",
      "IS  - 1474-760X (Electronic)\n",
      "IS  - 1474-7596 (Linking)\n",
      "VI  - 14\n",
      "IP  - 12\n",
      "DP  - 2013 Dec 16\n",
      "TI  - Population genomics of the endangered giant Galapagos tortoise.\n",
      "PG  - R136\n",
      "LID - 10.1186/gb-2013-14-12-r136 [doi]\n",
      "AB  - BACKGROUND: The giant Galapagos tortoise, Chelonoidis nigra, is a large-sizedterrestrial chelonian of high patrimonial interest. The species recentlycolonized a small continental archipelago, the Galapagos Islands, where it hasbeen facing novel environmental conditions and limited resource availability. To explore the genomic consequences of this ecological shift, we analyze thetranscriptomic variability of five individuals of C. nigra, and compare it tosimilar data obtained from several continental species of turtles. RESULTS:Having clarified the timing of divergence in the Chelonoidis genus, we report in C. nigra a very low level of genetic polymorphism, signatures of a weakenedefficacy of purifying selection, and an elevated mutation load in coding andregulatory sequences. These results are consistent with the hypothesis of anextremely low long-term effective population size in this insular species.Functional evolutionary analyses reveal a reduced diversity of immunity genes in C. nigra, in line with the hypothesis of attenuated pathogen diversity inislands, and an increased selective pressure on genes involved in response tostress, potentially related to the climatic instability of its environment andits elongated lifespan. Finally, we detect no population structure orhomozygosity excess in our five-individual sample. CONCLUSIONS: These resultsenlighten the molecular evolution of an endangered taxon in a stressfulenvironment and point to island endemic species as a promising model for thestudy of the deleterious effects on genome evolution of a reduced long-termpopulation size.\n",
      "FAU - Loire, Etienne\n",
      "AU  - Loire E\n",
      "FAU - Chiari, Ylenia\n",
      "AU  - Chiari Y\n",
      "FAU - Bernard, Aurelien\n",
      "AU  - Bernard A\n",
      "FAU - Cahais, Vincent\n",
      "AU  - Cahais V\n",
      "FAU - Romiguier, Jonathan\n",
      "AU  - Romiguier J\n",
      "FAU - Nabholz, Benoit\n",
      "AU  - Nabholz B\n",
      "FAU - Lourenco, Joao Miguel\n",
      "AU  - Lourenco JM\n",
      "FAU - Galtier, Nicolas\n",
      "AU  - Galtier N\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20131216\n",
      "PL  - England\n",
      "TA  - Genome Biol\n",
      "JT  - Genome biology\n",
      "JID - 100960660\n",
      "RN  - 0 (DNA, Mitochondrial)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - DNA, Mitochondrial/analysis\n",
      "MH  - Ecuador\n",
      "MH  - *Endangered Species\n",
      "MH  - Evolution, Molecular\n",
      "MH  - Gene Expression Profiling\n",
      "MH  - Genetic Drift\n",
      "MH  - *Genome\n",
      "MH  - Metagenomics/*methods\n",
      "MH  - Phylogeny\n",
      "MH  - Polymorphism, Single Nucleotide\n",
      "MH  - Population Density\n",
      "MH  - Selection, Genetic\n",
      "MH  - Stress, Physiological\n",
      "MH  - Turtles/classification/*genetics\n",
      "PMC - PMC4053747\n",
      "EDAT- 2013/12/18 06:00\n",
      "MHDA- 2015/04/24 06:00\n",
      "CRDT- 2013/12/18 06:00\n",
      "PHST- 2013/05/28 00:00 [received]\n",
      "PHST- 2013/12/16 00:00 [accepted]\n",
      "PHST- 2013/12/18 06:00 [entrez]\n",
      "PHST- 2013/12/18 06:00 [pubmed]\n",
      "PHST- 2015/04/24 06:00 [medline]\n",
      "AID - gb-2013-14-12-r136 [pii]\n",
      "AID - 10.1186/gb-2013-14-12-r136 [doi]\n",
      "PST - epublish\n",
      "SO  - Genome Biol. 2013 Dec 16;14(12):R136. doi: 10.1186/gb-2013-14-12-r136.\n",
      "\n",
      "PMID- 22615968\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20120920\n",
      "LR  - 20211021\n",
      "IS  - 1932-6203 (Electronic)\n",
      "IS  - 1932-6203 (Linking)\n",
      "VI  - 7\n",
      "IP  - 5\n",
      "DP  - 2012\n",
      "TI  - Salmonella strains isolated from Galapagos iguanas show spatial structuring ofserovar and genomic diversity.\n",
      "PG  - e37302\n",
      "LID - 10.1371/journal.pone.0037302 [doi]\n",
      "AB  - It is thought that dispersal limitation primarily structures host-associatedbacterial populations because host distributions inherently limit transmissionopportunities. However, enteric bacteria may disperse great distances duringfood-borne outbreaks. It is unclear if such rapid long-distance dispersal events happen regularly in natural systems or if these events represent an anthropogenicexception. We characterized Salmonella enterica isolates from the feces offree-living Galapagos land and marine iguanas from five sites on four islandsusing serotyping and genomic fingerprinting. Each site hosted unique and nearlyexclusive serovar assemblages. Genomic fingerprint analysis offered a morecomplex model of S. enterica biogeography, with evidence of both unique strainpools and of spatial population structuring along a geographic gradient. Thesefindings suggest that even relatively generalist enteric bacteria may be stronglydispersal limited in a natural system with strong barriers, such as oceanicdivides. Yet, these differing results seen on two typing methods also suggeststhat genomic variation is less dispersal limited, allowing for differentecological processes to shape biogeographical patterns of the core and flexibleportions of this bacterial species' genome.\n",
      "FAU - Lankau, Emily W\n",
      "AU  - Lankau EW\n",
      "AD  - Department of Animal Sciences and College of Veterinary Medicine, University ofIllinois at Urbana-Champaign, Urbana, Illinois, United States of America.\n",
      "FAU - Cruz Bedon, Lenin\n",
      "AU  - Cruz Bedon L\n",
      "FAU - Mackie, Roderick I\n",
      "AU  - Mackie RI\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, Non-P.H.S.\n",
      "DEP - 20120516\n",
      "PL  - United States\n",
      "TA  - PLoS One\n",
      "JT  - PloS one\n",
      "JID - 101285081\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Ecuador\n",
      "MH  - Feces/microbiology\n",
      "MH  - *Genetic Variation\n",
      "MH  - Genome, Bacterial\n",
      "MH  - Genomics\n",
      "MH  - Iguanas/*microbiology\n",
      "MH  - Prevalence\n",
      "MH  - Salmonella Infections, Animal/epidemiology/*genetics\n",
      "MH  - Salmonella enterica/classification/*genetics/isolation & purification\n",
      "MH  - Serotyping\n",
      "PMC - PMC3353930\n",
      "EDAT- 2012/05/23 06:00\n",
      "MHDA- 2012/09/21 06:00\n",
      "CRDT- 2012/05/23 06:00\n",
      "PHST- 2012/01/06 00:00 [received]\n",
      "PHST- 2012/04/19 00:00 [accepted]\n",
      "PHST- 2012/05/23 06:00 [entrez]\n",
      "PHST- 2012/05/23 06:00 [pubmed]\n",
      "PHST- 2012/09/21 06:00 [medline]\n",
      "AID - 10.1371/journal.pone.0037302 [doi]\n",
      "AID - PONE-D-12-00823 [pii]\n",
      "PST - ppublish\n",
      "SO  - PLoS One. 2012;7(5):e37302. doi: 10.1371/journal.pone.0037302. Epub 2012 May 16.\n",
      "\n",
      "PMID- 22038073\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20120313\n",
      "LR  - 20120131\n",
      "IS  - 1432-8798 (Electronic)\n",
      "IS  - 0304-8608 (Linking)\n",
      "VI  - 157\n",
      "IP  - 2\n",
      "DP  - 2012 Feb\n",
      "TI  - Genomic sequence of infectious hypodermal and hematopoietic necrosis virus(IHHNV) KLV-2010-01 originating from the first Korean outbreak in culturedLitopenaeus vannamei.\n",
      "PG  - 369-73\n",
      "LID - 10.1007/s00705-011-1155-0 [doi]\n",
      "AB  - Due to the need to track and monitor genetic diversity, the genome of theinfectious hypodermal and hematopoietic necrosis virus (IHHNV) strain KLV-2010-01in cultured Litopenaeus vannamei shrimp that originated from the first Koreanoutbreak in 2010 was sequenced and analyzed. The genome, with a length of 3914nucleotides, was sequenced from the Korean IHHNV. The genome encoded three large and overlapping open reading frames: ORF1 (NS-1) of 2001 bp, ORF2 (NS-2) of 1092 bp and ORF3 (capsid protein) of 990 bp. The overall organization, size andpredicted amino acid sequence of the three ORFs in Korean IHHNV were highlysimilar to those of members of the infectious IHHNV group, and the most closelyrelated strains were IHHNVs described from Ecuador and Hawaii. Additionally,phylogenetic analysis showed that the Korean IHHNV was clustered with lineage IIIin the infectious IHHNV group and was most similar to IHHNV isolates fromEcuador, China and Taiwan.\n",
      "FAU - Kim, J H\n",
      "AU  - Kim JH\n",
      "AD  - Laboratory of Aquatic Animal Medicine, College of Veterinary Medicine andResearch Institute for Veterinary Science, Seoul National University, Seoul151-742, Republic of Korea.\n",
      "FAU - Kim, H K\n",
      "AU  - Kim HK\n",
      "FAU - Nguyen, V G\n",
      "AU  - Nguyen VG\n",
      "FAU - Park, B K\n",
      "AU  - Park BK\n",
      "FAU - Choresca, C H\n",
      "AU  - Choresca CH\n",
      "FAU - Shin, S P\n",
      "AU  - Shin SP\n",
      "FAU - Han, J E\n",
      "AU  - Han JE\n",
      "FAU - Jun, J W\n",
      "AU  - Jun JW\n",
      "FAU - Park, S C\n",
      "AU  - Park SC\n",
      "LA  - eng\n",
      "SI  - GENBANK/JN377975\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "DEP - 20111030\n",
      "PL  - Austria\n",
      "TA  - Arch Virol\n",
      "JT  - Archives of virology\n",
      "JID - 7506870\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Base Sequence\n",
      "MH  - Densovirinae/classification/*genetics/*isolation & purification\n",
      "MH  - Disease Outbreaks/*veterinary\n",
      "MH  - Genome, Viral\n",
      "MH  - *Genomics\n",
      "MH  - Molecular Sequence Data\n",
      "MH  - Open Reading Frames\n",
      "MH  - Penaeidae/growth & development/*virology\n",
      "MH  - Phylogeny\n",
      "MH  - Republic of Korea/epidemiology\n",
      "EDAT- 2011/11/01 06:00\n",
      "MHDA- 2012/03/14 06:00\n",
      "CRDT- 2011/11/01 06:00\n",
      "PHST- 2011/09/20 00:00 [received]\n",
      "PHST- 2011/10/17 00:00 [accepted]\n",
      "PHST- 2011/11/01 06:00 [entrez]\n",
      "PHST- 2011/11/01 06:00 [pubmed]\n",
      "PHST- 2012/03/14 06:00 [medline]\n",
      "AID - 10.1007/s00705-011-1155-0 [doi]\n",
      "PST - ppublish\n",
      "SO  - Arch Virol. 2012 Feb;157(2):369-73. doi: 10.1007/s00705-011-1155-0. Epub 2011 Oct30.\n",
      "\n",
      "PMID- 19844075\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20100520\n",
      "LR  - 20100217\n",
      "IS  - 1662-8063 (Electronic)\n",
      "IS  - 1662-4246 (Linking)\n",
      "VI  - 13\n",
      "IP  - 3\n",
      "DP  - 2010\n",
      "TI  - Ecuador: public health genomics.\n",
      "PG  - 171-80\n",
      "LID - 10.1159/000249817 [doi]\n",
      "AB  - Ecuador has a heterogeneous population of almost 14 million people and a complex health care system provided through provincial and national health programs bygovernment and private hospitals. There are public health facilities at regional and territorial level. Ecuador has a small cadre of genetic professionals thatprovide clinical genetic services in a few private medical centers in the maincities. Prenatal screening is offered exclusively in a few individual hospitals, with variable uptake as part of prenatal care. Surveillance of the effect ofprenatal screening and diagnosis on the birth prevalence of congenital anomalies is limited by gaps and variations in surveillance systems. Newborn screeningprograms are almost inexistent. There is broad variation in optionalparticipation in laboratory quality assurance schemes, and there are noregulatory frameworks that are directly pertinent to genetic testing services or population genetics. Health technology assessment in Ecuador is conducted by adiverse collection of organizations, several of which have produced reportsrelated to genetics.\n",
      "CI  - Copyright 2009 S. Karger AG, Basel.\n",
      "FAU - Gonzalez-Andrade, F\n",
      "AU  - Gonzalez-Andrade F\n",
      "AD  - Department of Medicine, Metropolitan Hospital, Quito, Ecuador.fabriciogonzaleza@yahoo.es\n",
      "FAU - Lopez-Pulles, R\n",
      "AU  - Lopez-Pulles R\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "DEP - 20091016\n",
      "PL  - Switzerland\n",
      "TA  - Public Health Genomics\n",
      "JT  - Public health genomics\n",
      "JID - 101474167\n",
      "SB  - IM\n",
      "MH  - Community Health Services\n",
      "MH  - Congenital Abnormalities/*diagnosis\n",
      "MH  - Ecuador\n",
      "MH  - Ethics, Medical\n",
      "MH  - Genetic Testing/methods\n",
      "MH  - Genomics/*methods\n",
      "MH  - Geography\n",
      "MH  - Health Policy\n",
      "MH  - Health Priorities\n",
      "MH  - Health Services Accessibility\n",
      "MH  - Humans\n",
      "MH  - Infant, Newborn\n",
      "MH  - Neonatal Screening/*methods\n",
      "MH  - Prenatal Diagnosis/*methods\n",
      "MH  - Public Health/*methods\n",
      "EDAT- 2009/10/22 06:00\n",
      "MHDA- 2010/05/21 06:00\n",
      "CRDT- 2009/10/22 06:00\n",
      "PHST- 2008/12/18 00:00 [accepted]\n",
      "PHST- 2009/10/22 06:00 [entrez]\n",
      "PHST- 2009/10/22 06:00 [pubmed]\n",
      "PHST- 2010/05/21 06:00 [medline]\n",
      "AID - 000249817 [pii]\n",
      "AID - 10.1159/000249817 [doi]\n",
      "PST - ppublish\n",
      "SO  - Public Health Genomics. 2010;13(3):171-80. doi: 10.1159/000249817. Epub 2009 Oct 16.\n",
      "\n",
      "PMID- 19177531\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20090528\n",
      "LR  - 20131121\n",
      "IS  - 1098-1004 (Electronic)\n",
      "IS  - 1059-7794 (Linking)\n",
      "VI  - 30\n",
      "IP  - 3\n",
      "DP  - 2009 Mar\n",
      "TI  - Ten novel HMGCL mutations in 24 patients of different origin with3-hydroxy-3-methyl-glutaric aciduria.\n",
      "PG  - E520-9\n",
      "LID - 10.1002/humu.20966 [doi]\n",
      "AB  - 3-Hydroxy-3-methylglutaric aciduria is a rare autosomal recessive geneticdisorder that affects ketogenesis and L-leucine catabolism. The clinical acutesymptoms include vomiting, convulsions, metabolic acidosis, hypoketotichypoglycaemia and lethargy. To date, 33 mutations in 100 patients have beenreported in the HMGCL gene. In this study 10 new mutations in 24 patients aredescribed. They include: 5 missense mutations: c.109G>A, c.425C>T, c.521G>A,c.575T>C and c.598A>T, 2 nonsense mutations: c.242G>A and c.559G>T, one smalldeletion: c.853delC, and 2 mutations in intron regions: c.497+4A>G andc.750+1G>A. Two prevalent mutations were detected, 109G>T (E37X) in 38% ofdisease alleles analyzed and c.504_505delCT in 10% of them. Although patients aremainly of European origin (71%) and mostly Spanish (54%), the group is ethnicallydiverse and includes, for the first time, patients from Pakistan, Palestine andEcuador. We also present a simple, efficient method to express the enzyme and we analyze the possible functional effects of missense mutations. The finding thatall identified missense mutations cause a >95% decrease in the enzyme activity,indicates that the disease appears only in very severe genotypes.\"\n",
      "CI  - 2009 Wiley-Liss, Inc.\n",
      "FAU - Menao, Sebastian\n",
      "AU  - Menao S\n",
      "AD  - Laboratory of Clinical Genetics and Functional Genomics, Department ofPharmacology-Physiology, Medical School, University of Zaragoza, Zaragoza, Spain.\n",
      "FAU - Lopez-Vinas, Eduardo\n",
      "AU  - Lopez-Vinas E\n",
      "FAU - Mir, Cecilia\n",
      "AU  - Mir C\n",
      "FAU - Puisac, Beatriz\n",
      "AU  - Puisac B\n",
      "FAU - Gratacos, Esther\n",
      "AU  - Gratacos E\n",
      "FAU - Arnedo, Maria\n",
      "AU  - Arnedo M\n",
      "FAU - Carrasco, Patricia\n",
      "AU  - Carrasco P\n",
      "FAU - Moreno, Susana\n",
      "AU  - Moreno S\n",
      "FAU - Ramos, Monica\n",
      "AU  - Ramos M\n",
      "FAU - Gil, Maria Concepcion\n",
      "AU  - Gil MC\n",
      "FAU - Pie, Angeles\n",
      "AU  - Pie A\n",
      "FAU - Ribes, Antonia\n",
      "AU  - Ribes A\n",
      "FAU - Perez-Cerda, Celia\n",
      "AU  - Perez-Cerda C\n",
      "FAU - Ugarte, Magdalena\n",
      "AU  - Ugarte M\n",
      "FAU - Clayton, Peter T\n",
      "AU  - Clayton PT\n",
      "FAU - Korman, Stanley H\n",
      "AU  - Korman SH\n",
      "FAU - Serra, Dolors\n",
      "AU  - Serra D\n",
      "FAU - Asins, Guillermina\n",
      "AU  - Asins G\n",
      "FAU - Ramos, Feliciano J\n",
      "AU  - Ramos FJ\n",
      "FAU - Gomez-Puertas, Paulino\n",
      "AU  - Gomez-Puertas P\n",
      "FAU - Hegardt, Fausto G\n",
      "AU  - Hegardt FG\n",
      "FAU - Casals, Nuria\n",
      "AU  - Casals N\n",
      "FAU - Pie, Juan\n",
      "AU  - Pie J\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PL  - United States\n",
      "TA  - Hum Mutat\n",
      "JT  - Human mutation\n",
      "JID - 9215429\n",
      "RN  - CLA99KCD53 (Meglutol)\n",
      "RN  - EC 4.1.3.- (Oxo-Acid-Lyases)\n",
      "RN  - EC 4.1.3.4 (3-hydroxy-3-methylglutaryl-coenzyme A lyase)\n",
      "SB  - IM\n",
      "MH  - Alleles\n",
      "MH  - Amino Acid Sequence\n",
      "MH  - Arabs/genetics\n",
      "MH  - Catalytic Domain/genetics\n",
      "MH  - DNA Mutational Analysis\n",
      "MH  - Ecuador\n",
      "MH  - Europe\n",
      "MH  - Gene Frequency\n",
      "MH  - Genotype\n",
      "MH  - Humans\n",
      "MH  - Kinetics\n",
      "MH  - Meglutol/*metabolism\n",
      "MH  - Metabolism, Inborn Errors/ethnology/*genetics/pathology\n",
      "MH  - Models, Molecular\n",
      "MH  - Molecular Sequence Data\n",
      "MH  - *Mutation\n",
      "MH  - Mutation, Missense\n",
      "MH  - Oxo-Acid-Lyases/chemistry/*genetics/metabolism\n",
      "MH  - Pakistan\n",
      "MH  - Protein Structure, Tertiary\n",
      "MH  - Sequence Homology, Amino Acid\n",
      "EDAT- 2009/01/30 09:00\n",
      "MHDA- 2009/05/29 09:00\n",
      "CRDT- 2009/01/30 09:00\n",
      "PHST- 2009/01/30 09:00 [entrez]\n",
      "PHST- 2009/01/30 09:00 [pubmed]\n",
      "PHST- 2009/05/29 09:00 [medline]\n",
      "AID - 10.1002/humu.20966 [doi]\n",
      "PST - ppublish\n",
      "SO  - Hum Mutat. 2009 Mar;30(3):E520-9. doi: 10.1002/humu.20966.\n",
      "\n",
      "PMID- 19018989\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20090331\n",
      "LR  - 20211020\n",
      "IS  - 1364-3703 (Electronic)\n",
      "IS  - 1364-3703 (Linking)\n",
      "VI  - 9\n",
      "IP  - 5\n",
      "DP  - 2008 Sep\n",
      "TI  - Moniliophthora perniciosa, the causal agent of witches' broom disease of cacao:what's new from this old foe?\n",
      "PG  - 577-88\n",
      "LID - 10.1111/j.1364-3703.2008.00496.x [doi]\n",
      "AB  - UNLABELLED: Moniliophthora perniciosa (=Crinipellis perniciosa) causes one of thethree main fungal diseases of Theobroma cacao (cacao), the source of chocolate.This pathogen causes Witches' broom disease (WBD) and has brought about severeeconomic losses in all of the cacao-growing regions to which it has spread withyield reductions that range from 50 to 90%. Cacao production in South Americareflects the severity of this pathogen, as the yields in most of the infectedregions have not returned to pre-outbreak levels, even with the introduction ofresistant varieties. In this review we give a brief historical account andsummarize the current state of knowledge focusing on developments in the areas ofsystematics, fungal physiology, biochemistry, genomics and gene expression in an attempt to highlight this disease. Moniliophthora perniciosa is a hemibiotrophic fungus with two distinct growth phases. The ability to culture a biotrophic-like phase in vitro along with new findings derived from the nearly complete genomeand expression studies clearly show that these different fungal growth phasesfunction under distinct metabolic parameters. These new findings have greatlyimproved our understanding of this fungal/host interaction and we may be at thecrossroads of understanding how hemibiotrophic fungal plant pathogens causedisease in other crops. HISTORICAL SUMMARY OF WBD: The first WDB symptoms appear to have been described in the diaries of Alexandre Rodrigues Ferreira (described as lagartao; meaning big lizard) from his observations of cacao trees in 1785 and1787 in Amazonia, which is consistent with the generally accepted idea that M.perniciosa, like its main host T. cacao, evolved in this region. The diseasesubsequently arrived in Surinam in 1895. WBD moved rapidly, spreading to Guyanain 1906, Ecuador in 1918, Trinidad in 1928, Colombia in 1929 and Grenada in 1948.In each case, cacao production was catastrophically affected with yieldreductions of 50-90%. After the arrival of M. perniciosa in Bahia in 1989, Brazilwent from being the world's 3rd largest producer of cacao (347 000 tonnes in1988-1990; c. 15% of the total world production at that time) to a net importer(141 000 tonnes in 1998-2000). Fortunately for chocolate lovers, other regions ofthe world such as West Africa and South East Asia have not yet been affected bythis disease and have expanded production to meet growing world demand (predictedto reach 3 700 000 tonnes by 2010). CLASSIFICATION: Moniliophthora perniciosa(Stahel) Aime & Phillips-Mora: super-kingdom Eukaryota; kingdom Fungi; phylumBasidiomycota; subphylum Agaricomycotina; class Agaricomycetes; subclassAgaricomycetidae; order Agaricales; family Marasmiaceae; genus Moniliophthora.USEFUL WEBSITES: http://www.lge.ibi.unicamp.br/vassoura/,http://nt.ars-grin.gov/taxadescriptions/keys/TrichodermaIndex.cfm,http://www.worldcocoafoundation.org/info-center/research-updates.asp,http://www.ars.usda.gov/ba/psi/spcl.\n",
      "FAU - Meinhardt, Lyndel W\n",
      "AU  - Meinhardt LW\n",
      "AD  - Sustainable Perennial Crops Laboratory, USDA-ARS, 10300 Baltimore Ave., Bldg.001, Beltsville, MD 20705-2350, USA. lyndel.meinhardt@ars.usda.gov\n",
      "FAU - Rincones, Johana\n",
      "AU  - Rincones J\n",
      "FAU - Bailey, Bryan A\n",
      "AU  - Bailey BA\n",
      "FAU - Aime, M Catherine\n",
      "AU  - Aime MC\n",
      "FAU - Griffith, Gareth W\n",
      "AU  - Griffith GW\n",
      "FAU - Zhang, Dapeng\n",
      "AU  - Zhang D\n",
      "FAU - Pereira, Goncalo A G\n",
      "AU  - Pereira GA\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "PL  - England\n",
      "TA  - Mol Plant Pathol\n",
      "JT  - Molecular plant pathology\n",
      "JID - 100954969\n",
      "SB  - IM\n",
      "MH  - Agaricales/*physiology\n",
      "MH  - Cacao/*microbiology\n",
      "MH  - Host-Parasite Interactions\n",
      "MH  - Plant Diseases/*microbiology\n",
      "RF  - 91\n",
      "PMC - PMC6640444\n",
      "EDAT- 2008/11/21 09:00\n",
      "MHDA- 2009/04/01 09:00\n",
      "CRDT- 2008/11/21 09:00\n",
      "PHST- 2008/11/21 09:00 [pubmed]\n",
      "PHST- 2009/04/01 09:00 [medline]\n",
      "PHST- 2008/11/21 09:00 [entrez]\n",
      "AID - MPP496 [pii]\n",
      "AID - 10.1111/j.1364-3703.2008.00496.x [doi]\n",
      "PST - ppublish\n",
      "SO  - Mol Plant Pathol. 2008 Sep;9(5):577-88. doi: 10.1111/j.1364-3703.2008.00496.x.\n",
      "\n",
      "PMID- 17187350\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 20070131\n",
      "LR  - 20160407\n",
      "IS  - 0265-9247 (Print)\n",
      "IS  - 0265-9247 (Linking)\n",
      "VI  - 29\n",
      "IP  - 1\n",
      "DP  - 2007 Jan\n",
      "TI  - How to tweak a beak: molecular techniques for studying the evolution of size and shape in Darwin's finches and other birds.\n",
      "PG  - 1-6\n",
      "AB  - A flurry of technological advances in molecular, cellular and developmentalbiology during the past decade has provided a clearer understanding of mechanismsunderlying phenotypic diversification. Building upon such momentum, a recentpaper tackles one of the foremost topics in evolution, that is the origin ofspecies-specific beak morphology in Darwin's finches.1 Previous work involvingboth domesticated and wild birds implicated a well-known signaling pathway (i.e. bone morphogenetic proteins) and one population of progenitor cells in particular(i.e. cranial neural crest), as primary factors for establishing beak size andshape. But these results were limited in their ability to explain fully themorphogenetic bases of patterned outgrowth. So in a quest to identify novel geneswhose expression correlated with differences in beak anatomy among Darwin'sfinches, a DNA microarray approach was undertaken using tissues harvested fromthe Galapagos Islands. The results are striking and point to a protein calledcalmodulin, which is a mediator of cellular calcium signaling, as a keydeterminant of beak length.\n",
      "FAU - Schneider, Richard A\n",
      "AU  - Schneider RA\n",
      "AD  - Department of Orthopaedic Surgery, University of California at San Francisco, 533Parnassus Avenue, U-453, San Francisco, CA 94143, USA. rich.schneider@ucsf.edu\n",
      "LA  - eng\n",
      "PT  - Journal Article\n",
      "PT  - Review\n",
      "PL  - United States\n",
      "TA  - Bioessays\n",
      "JT  - BioEssays : news and reviews in molecular, cellular and developmental biology\n",
      "JID - 8510851\n",
      "RN  - 0 (Bone Morphogenetic Protein 4)\n",
      "RN  - 0 (Bone Morphogenetic Proteins)\n",
      "RN  - 0 (Calmodulin)\n",
      "SB  - IM\n",
      "MH  - Animals\n",
      "MH  - Beak/anatomy & histology\n",
      "MH  - Birds/*anatomy & histology/*genetics\n",
      "MH  - Bone Morphogenetic Protein 4\n",
      "MH  - Bone Morphogenetic Proteins/genetics\n",
      "MH  - Calmodulin/genetics\n",
      "MH  - Ecuador\n",
      "MH  - Evolution, Molecular\n",
      "MH  - Finches/*anatomy & histology/*genetics\n",
      "MH  - Gene Expression\n",
      "MH  - Genomics\n",
      "MH  - Species Specificity\n",
      "RF  - 44\n",
      "EDAT- 2006/12/26 09:00\n",
      "MHDA- 2007/02/01 09:00\n",
      "CRDT- 2006/12/26 09:00\n",
      "PHST- 2006/12/26 09:00 [pubmed]\n",
      "PHST- 2007/02/01 09:00 [medline]\n",
      "PHST- 2006/12/26 09:00 [entrez]\n",
      "AID - 10.1002/bies.20517 [doi]\n",
      "PST - ppublish\n",
      "SO  - Bioessays. 2007 Jan;29(1):1-6. doi: 10.1002/bies.20517.\n",
      "\n",
      "PMID- 8661097\n",
      "OWN - NLM\n",
      "STAT- MEDLINE\n",
      "DCOM- 19960930\n",
      "LR  - 20211203\n",
      "IS  - 0888-7543 (Print)\n",
      "IS  - 0888-7543 (Linking)\n",
      "VI  - 35\n",
      "IP  - 1\n",
      "DP  - 1996 Jul 1\n",
      "TI  - The gene for the Ellis-van Creveld syndrome is located on chromosome 4p16.\n",
      "PG  - 1-5\n",
      "AB  - Ellis-van Creveld syndrome (EVC) is an autosomal recessive disorder characterizedby disproportionate dwarfism, polydactyly, and congenital heart disease. Thisrare disorder is found with increased frequency among the Old Order Amishcommunity in Lancaster County, Pennsylvania. We have used linkage analysis tolocalize the gene responsible for the EVC phenotype in nine interrelated Amishpedigrees and three unrelated families from Mexico, Ecuador, and Brazil. We nowreport the linkage for the Ellis-van Creveld syndrome gene to markers on thedistal short arm of human chromosome 4, with Zmax = 6.91 at theta = 0.02 formarker HOX7, in a region proximal to the FGFR3 gene responsible for theachondroplasia phenotype.\n",
      "FAU - Polymeropoulos, M H\n",
      "AU  - Polymeropoulos MH\n",
      "AD  - Gene Mapping Unit, Laboratory of Genetic Disease Research, National Center forHuman Genome Research, National Institutes of Health, Building 49, Bethesda,Maryland, 20892, USA.\n",
      "FAU - Ide, S E\n",
      "AU  - Ide SE\n",
      "FAU - Wright, M\n",
      "AU  - Wright M\n",
      "FAU - Goodship, J\n",
      "AU  - Goodship J\n",
      "FAU - Weissenbach, J\n",
      "AU  - Weissenbach J\n",
      "FAU - Pyeritz, R E\n",
      "AU  - Pyeritz RE\n",
      "FAU - Da Silva, E O\n",
      "AU  - Da Silva EO\n",
      "FAU - Ortiz De Luna, R I\n",
      "AU  - Ortiz De Luna RI\n",
      "FAU - Francomano, C A\n",
      "AU  - Francomano CA\n",
      "LA  - eng\n",
      "GR  - FDR 692/FD/FDA HHS/United States\n",
      "GR  - HL 35877/HL/NHLBI NIH HHS/United States\n",
      "GR  - P01 HG00373/HG/NHGRI NIH HHS/United States\n",
      "PT  - Journal Article\n",
      "PT  - Research Support, Non-U.S. Gov't\n",
      "PT  - Research Support, U.S. Gov't, P.H.S.\n",
      "PL  - United States\n",
      "TA  - Genomics\n",
      "JT  - Genomics\n",
      "JID - 8800135\n",
      "SB  - IM\n",
      "MH  - Brazil/epidemiology\n",
      "MH  - Chromosome Mapping\n",
      "MH  - Chromosomes, Human, Pair 4/*genetics\n",
      "MH  - Consanguinity\n",
      "MH  - Ecuador/epidemiology\n",
      "MH  - Ellis-Van Creveld Syndrome/ethnology/*genetics\n",
      "MH  - Ethnicity/*genetics\n",
      "MH  - Genes, Recessive\n",
      "MH  - Genetic Linkage\n",
      "MH  - Haplotypes/genetics\n",
      "MH  - Humans\n",
      "MH  - Mexico/epidemiology\n",
      "MH  - Pedigree\n",
      "MH  - Pennsylvania/epidemiology\n",
      "EDAT- 1996/07/01 00:00\n",
      "MHDA- 1996/07/01 00:01\n",
      "CRDT- 1996/07/01 00:00\n",
      "PHST- 1996/07/01 00:00 [pubmed]\n",
      "PHST- 1996/07/01 00:01 [medline]\n",
      "PHST- 1996/07/01 00:00 [entrez]\n",
      "AID - S0888-7543(96)90315-8 [pii]\n",
      "AID - 10.1006/geno.1996.0315 [doi]\n",
      "PST - ppublish\n",
      "SO  - Genomics. 1996 Jul 1;35(1):1-5. doi: 10.1006/geno.1996.0315.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Escriba aquí su código para el ejercicio 2\n",
    "import miningscience\n",
    "import re\n",
    "a=miningscience.download_pubmed('Ecuador genomics')\n",
    "b=re.findall(r'[DIMP]{4}\\-.*\\d\\n',a)\n",
    "print ('El número de artículos es: ', len(b))\n",
    "print ('El PMID del primero artículo es: ',b[0])\n",
    "print (a)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Ejercicio 3 [1 punto]\n",
    "\n",
    "Utilice la función ```mining_pubs``` para crear los siguientes dataframes: \n",
    "\n",
    "* pub_year\n",
    "\n",
    "* pub_authors\n",
    "\n",
    "* coutries\n",
    "\n",
    "Muestre los dos primeros y los dos últimos registros de cada dataframe. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "jp-MarkdownHeadingCollapsed": true,
    "tags": []
   },
   "source": [
    "## Ejercicio 4 [2 puntos]\n",
    "\n",
    "Realice la visualización de los datos del ejercicio 3. \n",
    "\n",
    "* ```pubyear```: utilizando el conteo artículos por año, calcule la media y la desviación estandar (asumiendo que la distribución es normal). Realice un gráfico de la distribución normal de sus datos. Coloque el título, leyenda de los ejes \"x\" e \"y\", y la leyenda de la curva (utilice las expresiones matemáticas tipos latex de ser necesario). \n",
    "\n",
    "* ```pub_authors```: realice un boxplot del conteo de autores por artículo. Coloque las leyendas adecuadas para cada eje y el título.  \n",
    "\n",
    "* ```countries```: realice una nube de puntos (adaptación de su tarea), resalte al menos 3 países con flechas. Coloque las leyendas adecuadas para cada eje y el título. \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "90\n",
      "[1, 6, 35, 59, 25, 93, 2, 42, 5, 7, 200, 3, 98, 35, 52, 48, 89, 18, 7, 25, 172, 36, 356, 11, 2, 257, 7, 85, 4, 228, 2, 7, 3, 1, 12, 3, 1, 21, 2, 3, 6, 50, 1, 15, 72, 43, 7, 1, 2, 1, 1, 2, 71, 2, 1, 1, 2, 1, 6, 40, 1, 7, 13, 37, 12, 5, 111, 2, 1, 5, 4, 34, 9, 1, 31, 1, 2, 3, 1, 2, 31, 1, 5, 2, 544, 5, 11, 1, 1, 29]\n",
      "90\n",
      "['Andorra', 'United Arab Emirates ', 'The Netherlands', 'Argentina', 'Austria', 'Australia', 'Bangladesh', 'Belgium', 'Bulgaria', 'Bolivia', 'Brazil', 'Belize', 'Canada', 'Switzerland', 'Chile', 'China', 'Colombia', 'Costa Rica', 'Cuba', 'Czech Republic', 'Germany', 'Denmark', 'Ecuador', 'Estonia', 'Egypt', 'Spain', 'Finland', 'France', 'Gabon', 'United Kingdom', 'Grenada', 'Georgia', 'French Guiana', 'Gambia', 'Greece', 'Guatemala', 'Croatia', 'Hungary', 'Indonesia', 'Ireland', 'Israel', 'India', 'Iraq', 'Iran', 'Italy', 'Japan', 'Kenya', 'Kuwait', 'Laos', 'Sri Lanka', 'Lithuania', 'Malawi', 'Mexico', 'Malaysia', 'Mozambique', 'New Caledonia', 'Nigeria', 'Nicaragua', 'Netherlands', 'Norway', 'Nepal', 'New Zealand', 'Panama', 'Peru', 'Pakistan', 'Poland', 'Portugal', 'Paraguay', 'Qatar', 'Romania', 'Serbia', 'Russia', 'Saudi Arabia', 'Seychelles', 'Sweden', 'Singapore', 'Slovenia', 'Slovakia', 'Senegal', 'Suriname', 'Turkey', 'Taiwan', 'Ukraine', 'Uganda', 'United States of America', 'Uruguay', 'Venezuela', 'Vietnam', 'Samoa', 'South Africa']\n"
     ]
    }
   ],
   "source": [
    "# Escriba aquí su código para el ejercicio 3\n",
    "#9 lineas\n",
    "#print (a)\n",
    "#z=re.sub(r')z=re.findall(r'[DIMP]{4}.*\\d\\n',x)\n",
    "contentsse = re.sub(r'\\s+[Eceinlort]{10}\\s+[aders]{7}.*','',a)\n",
    "contentssc = re.sub(r'\\s[\\w._%+-]+@[\\w.-]+\\.[a-zA-Z]{1,4}','',contentsse)\n",
    "contentsna = re.sub(r'\\..*\\d.*\\,',',',contentssc)\n",
    "contentsnu = re.sub(r'\\..*\\d.*','',contentsna)\n",
    "x=contentsnu[1:].split('PMID-')\n",
    "ID=[]\n",
    "AU=[]\n",
    "DP=[]\n",
    "pre_AD=[]\n",
    "for PMID in x:\n",
    "    w=PMID.split('\\n')\n",
    "    AUc=0\n",
    "    ID.append(w[0])\n",
    "    for fila in w:\n",
    "        v=fila.split(' ')\n",
    "        if v[0] == 'DP':\n",
    "            DP.append(v[3])\n",
    "        if v[0] == 'AU':\n",
    "            AUc=AUc+1\n",
    "        if v[0] == 'AD':\n",
    "            u=fila.split(',')\n",
    "            pre_AD.append(u[-1])\n",
    "    AU.append(AUc)\n",
    "ID.pop(0)\n",
    "AU.pop(0)\n",
    "pre_AD.pop(-5)\n",
    "\n",
    "AP=[]\n",
    "for dire in pre_AD:\n",
    "    s=dire\n",
    "    t=dire.split(' ')\n",
    "    if t[0] == 'AD':\n",
    "        s=t[-1]   \n",
    "    AP.append(s)\n",
    "\n",
    "b=0\n",
    "AC =[0]*len(AP)\n",
    "\n",
    "for obj in AP:\n",
    "    bytes(obj,encoding=\"utf8\")\n",
    "    if obj != '.' or '':\n",
    "        g=obj\n",
    "        if g[0] == ' ':\n",
    "            g = re.sub (r'^\\s','',g)\n",
    "        if g[-1] == '.':\n",
    "            g = re.sub (r'\\.$','',g)\n",
    "        g = re.sub (r'\\.$','',g)\n",
    "    AC[b]=g\n",
    "    b=b+1\n",
    "\n",
    "q=0\n",
    "AD=[0]*len(AP)\n",
    "for g in AC:\n",
    "    if g == \"USA\":\n",
    "        g='United States of America'\n",
    "    if g == 'UK':\n",
    "        g='United Kingdom'\n",
    "    if g == \"GA\":\n",
    "        g= 'Gabon'\n",
    "    if g == 'CO':\n",
    "        g='Colombia'\n",
    "    if g == 'CA':\n",
    "        g='Canada'\n",
    "    AD[q]=g\n",
    "    q=q+1\n",
    "Countries_molde=['Andorra','United Arab Emirates ','Afghanistan','Antigua and Barbuda','Anguilla','Albania','Armenia',\n",
    "                 'The Netherlands','Angola','Antarctica','Argentina','American Samoa','Austria','Australia','Aruba',\n",
    "                 'Azerbaijan','Bosnia and Herzegovina','Barbados','Bangladesh','Belgium','Burkina Faso','Bulgaria','Bahrain',\n",
    "                 'Burundi','Benin','Bermuda','Brunei','Bolivia','Brazil','Bahamas','Bhutan','Bouvet Island','Botswana','Belarus',\n",
    "                 'Belize','Canada','Cocos [Keeling] Islands','Congo [DRC]','Central African Republic','Congo [Republic]',\n",
    "                 'Switzerland',\"Côte d'Ivoire\",'Cook Islands','Chile','Cameroon','China','Colombia','Costa Rica','Cuba',\n",
    "                 'Cape Verde','Christmas Island','Cyprus','Czech Republic','Germany','Djibouti','Denmark','Dominica',\n",
    "                 'Dominican Republic','Algeria','Ecuador','Estonia','Egypt','Western Sahara','Eritrea','Spain','Ethiopia',\n",
    "                 'Finland','Fiji','Falkland Islands [Islas Malvinas]','Micronesia','Faroe Islands','France','Gabon',\n",
    "                 'United Kingdom','Grenada','Georgia','French Guiana','Guernsey','Ghana','Gibraltar','Greenland','Gambia','Guinea',\n",
    "                 'Guadeloupe','Equatorial Guinea','Greece','South Georgia and the South Sandwich Islands','Guatemala','Guam',\n",
    "                 'Guinea-Bissau','Guyana','Gaza Strip','Hong Kong','Heard Island and McDonald Islands','Honduras','Croatia',\n",
    "                 'Haiti','Hungary','Indonesia','Ireland','Israel','Isle of Man','India','British Indian Ocean Territory','Iraq',\n",
    "                 'Iran','Iceland','Italy','Jersey','Jamaica','Jordan','Japan','Kenya','Kyrgyzstan','Cambodia','Kiribati','Comoros',\n",
    "                 'Saint Kitts and Nevis','North Korea','South Korea','Kuwait','Cayman Islands','Kazakhstan','Laos','Lebanon',\n",
    "                 'Saint Lucia','Liechtenstein','Sri Lanka','Liberia','Lesotho','Lithuania','Luxembourg','Latvia','Libya','Morocco',\n",
    "                 'Monaco','Moldova','Montenegro','Madagascar','Marshall Islands','Macedonia [FYROM]','Mali','Myanmar [Burma]',\n",
    "                 'Mongolia','Macau','Northern Mariana Islands','Martinique','Mauritania','Montserrat','Malta','Mauritius','Maldives',\n",
    "                 'Malawi','Mexico','Malaysia','Mozambique','Namibia','New Caledonia','Niger','Norfolk Island','Nigeria','Nicaragua',\n",
    "                 'Netherlands','Norway','Nepal','Nauru','Niue','New Zealand','Oman','Panama','Peru','French Polynesia',\n",
    "                 'Papua New Guinea','Philippines','Pakistan','Poland','Saint Pierre and Miquelon','Pitcairn Islands','Puerto Rico',\n",
    "                 'Palestinian Territories','Portugal','Palau','Paraguay','Qatar','Réunion','Romania','Serbia','Russia','Rwanda',\n",
    "                 'Saudi Arabia','Solomon Islands','Seychelles','Sudan','Sweden','Singapore','Saint Helena','Slovenia',\n",
    "                 'Svalbard and Jan Mayen','Slovakia','Sierra Leone','San Marino','Senegal','Somalia','Suriname',\n",
    "                 'São Tomé and Príncipe','El Salvador','Syria','Swaziland','Turks and Caicos Islands','Chad',\n",
    "                 'French Southern Territories','Togo','Thailand','Tajikistan','Tokelau','Timor-Leste','Turkmenistan','Tunisia','Tonga',\n",
    "                 'Turkey','Trinidad and Tobago','Tuvalu','Taiwan','Tanzania','Ukraine','Uganda','U.S. Minor Outlying Islands',\n",
    "                 'United States of America','Uruguay','Uzbekistan','Vatican City','Saint Vincent and the Grenadines','Venezuela',\n",
    "                 'British Virgin Islands','U.S. Virgin Islands','Vietnam','Vanuatu','Wallis and Futuna','Samoa','Kosovo','Yemen',\n",
    "                 'Mayotte','South Africa','Zambia','Zimbabwe']\n",
    "#print(len (Countries_molde))\n",
    "o=AD\n",
    "f=Countries_molde\n",
    "i=len(f)\n",
    "ADh=[0]*i\n",
    "k=0\n",
    "for elem in f:\n",
    "    d=0\n",
    "    for comp in o:\n",
    "        if elem == str(comp):\n",
    "            d=d+1\n",
    "    ADh[k]=d\n",
    "    k=k+1\n",
    "\n",
    "#print(AD)\n",
    "#n=0\n",
    "#for elim[n] in ADh:\n",
    " #   print(elim)\n",
    "  #  print('we)')\n",
    "   # if str(elim) == '0':\n",
    "    #   # print('YES')\n",
    "      #  ADh.pop(n)\n",
    "     #   print(ADh)\n",
    "       # Countries_molde.pop(n)\n",
    "        #print(Countries_molde)\n",
    "   # else:\n",
    "    #    n=n+1\n",
    "    #print('ne')\n",
    "    #print(n)\n",
    "AD=[]\n",
    "CO=[]\n",
    "n=0\n",
    "for elem in ADh:\n",
    "    if str(elem) != '0':\n",
    "        AD.append(elem)\n",
    "        m=Countries_molde[n]\n",
    "        CO.append(m)\n",
    "    n=n+1\n",
    "        \n",
    "print(len(AD))\n",
    "print (AD)\n",
    "print (len(CO))\n",
    "print (CO)\n",
    "#print (len (ID))\n",
    "#print (ID)\n",
    "#print (len(DP))\n",
    "#print(DP)\n",
    "#print (len (AU))\n",
    "#print (AU)\n",
    "#print (len(pre_AD))\n",
    "#print (len(AD) )\n",
    "#print (AD)\n",
    "\n",
    "#UK, USA U.S.A, IA, OK, GA, FL, NM\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Escriba aquí su código para el ejercicio 4"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Ejercicio 5 [1 punto]\n",
    "\n",
    "Interprete los resultados del **ejercicio 4**"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Escriba la respuesta del ejercicio 5\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Ejercicio 6 [2 puntos]\n",
    "\n",
    "Para algún gen de interés, realice lo siguiente: \n",
    "\n",
    "1. Una búsqueda en la página del [NCBI nucleotide](https://www.ncbi.nlm.nih.gov/nucleotide/).\n",
    "\n",
    "2. Descargue el ```Accession List``` de su búsqueda.\n",
    "\n",
    "3. Cargue el ```Accession List``` en este notebook y haga una descarga de la data tipo ```genbank``` utilizando los diez primeros IDs de accesión. \n",
    "\n",
    "4. Arme un árbol filogenético para los resultados del paso 3.\n",
    "\n",
    "5. Interprete el árbol del paso 4. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Escriba aquí su código para el ejercicio 6\n",
    "#Búsqueda en NCBI nucleotido de Dermatophilus congolensis 16S rRNA\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Escriba aquí la interpretación del árbol**"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Ejercicio 7 [1 punto]\n",
    "\n",
    "\n",
    "1. Cree en GitHub un repositorio de nombre ```GBI6_ExamenPython```. \n",
    "2. Cree un archivo ```Readme.md``` que debe tener lo siguiente: \n",
    "- Datos personales\n",
    "- Características del computador\n",
    "- Versión de Python/Anaconda y de cada uno de los módulos/paquetes y utilizados\n",
    "- Explicación de la data utilizada\n",
    "\n",
    "3. Cree un directorio de nombre ```data``` donde deben estar guardado la data utilizada \n",
    "4. Realice al menos 1 control de la versión (commits) por cada ejercicio (del 1 al 6), con un mensaje que inicie como: \n",
    "\n",
    "```sh \n",
    "Carlitos Alimaña ha realizado .....  \n",
    "```"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  },
  "latex_envs": {
   "LaTeX_envs_menu_present": true,
   "autoclose": false,
   "autocomplete": true,
   "bibliofile": "biblio.bib",
   "cite_by": "apalike",
   "current_citInitial": 1,
   "eqLabelWithNumbers": true,
   "eqNumInitial": 1,
   "hotkeys": {
    "equation": "Ctrl-E",
    "itemize": "Ctrl-I"
   },
   "labels_anchors": false,
   "latex_user_defs": false,
   "report_style_numbering": false,
   "user_envs_cfg": false
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": false,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {
    "height": "1152.67px",
    "left": "41px",
    "top": "110px",
    "width": "253.458px"
   },
   "toc_section_display": true,
   "toc_window_display": false
  },
  "varInspector": {
   "cols": {
    "lenName": 16,
    "lenType": 16,
    "lenVar": 40
   },
   "kernels_config": {
    "python": {
     "delete_cmd_postfix": "",
     "delete_cmd_prefix": "del ",
     "library": "var_list.py",
     "varRefreshCmd": "print(var_dic_list())"
    },
    "r": {
     "delete_cmd_postfix": ") ",
     "delete_cmd_prefix": "rm(",
     "library": "var_list.r",
     "varRefreshCmd": "cat(var_dic_list()) "
    }
   },
   "types_to_exclude": [
    "module",
    "function",
    "builtin_function_or_method",
    "instance",
    "_Feature"
   ],
   "window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
